# List of supported software {: #list_software }

EasyBuild supports 2898 different software packages (incl. toolchains, bundles):

<a href="#a">a</a> - <a href="#b">b</a> - <a href="#c">c</a> - <a href="#d">d</a> - <a href="#e">e</a> - <a href="#f">f</a> - <a href="#g">g</a> - <a href="#h">h</a> - <a href="#i">i</a> - <a href="#j">j</a> - <a href="#k">k</a> - <a href="#l">l</a> - <a href="#m">m</a> - <a href="#n">n</a> - <a href="#o">o</a> - <a href="#p">p</a> - <a href="#q">q</a> - <a href="#r">r</a> - <a href="#s">s</a> - <a href="#t">t</a> - <a href="#u">u</a> - <a href="#v">v</a> - <a href="#w">w</a> - <a href="#x">x</a> - <a href="#y">y</a> - <a href="#z">z</a>


<a anchor="3"/>
### *3*


<a href="#3to2">3to2</a>


<a anchor="3to2"/>
### *3to2*

lib3to2 is a set of fixers that are intended to backport code written for Python version 3.x  into Python version 2.x.

*homepage*: https://pypi.python.org/pypi/3to2

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``1.1.1``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``1.1.1``|``-Python-2.7.13``|``intel/2017a``

<a anchor="4"/>
### *4*


<a href="#4ti2">4ti2</a>


<a anchor="4ti2"/>
### *4ti2*

A software package for algebraic, geometric and combinatorial problems on linear spaces

*homepage*: https://4ti2.github.io/

version  |toolchain
---------|-------------------------------------
``1.6.9``|``GCC/8.2.0-2.31.1``, ``intel/2018b``

<a anchor="a"/>
### *A*


<a href="#abaqus">ABAQUS</a> - <a href="#abinit">ABINIT</a> - <a href="#abra2">ABRA2</a> - <a href="#abricate">ABRicate</a> - <a href="#abseil">Abseil</a> - <a href="#abtem">abTEM</a> - <a href="#abyss">ABySS</a> - <a href="#ack">ack</a> - <a href="#act">ACT</a> - <a href="#actc">ACTC</a> - <a href="#ada">ada</a> - <a href="#adapterremoval">AdapterRemoval</a> - <a href="#adda">ADDA</a> - <a href="#adf">ADF</a> - <a href="#adios">ADIOS</a> - <a href="#adjusttext">adjustText</a> - <a href="#admixture">ADMIXTURE</a> - <a href="#adol-c">ADOL-C</a> - <a href="#advisor">Advisor</a> - <a href="#afni">AFNI</a> - <a href="#agat">AGAT</a> - <a href="#agfusion">AGFusion</a> - <a href="#aiohttp">aiohttp</a> - <a href="#aladin">ALADIN</a> - <a href="#albacore">Albacore</a> - <a href="#albumentations">Albumentations</a> - <a href="#alevin-fry">alevin-fry</a> - <a href="#alfa">ALFA</a> - <a href="#alfred">Alfred</a> - <a href="#allelecount">alleleCount</a> - <a href="#allinea">Allinea</a> - <a href="#allpaths-lg">ALLPATHS-LG</a> - <a href="#almosthere">almosthere</a> - <a href="#alpha">Alpha</a> - <a href="#alphafold">AlphaFold</a> - <a href="#alps">ALPS</a> - <a href="#alsa-lib">alsa-lib</a> - <a href="#amapvox">AMAPVox</a> - <a href="#amara">Amara</a> - <a href="#amask">amask</a> - <a href="#amber">Amber</a> - <a href="#ambermini">AmberMini</a> - <a href="#ambertools">AmberTools</a> - <a href="#amd-libm">AMD-LibM</a> - <a href="#amd-rng">AMD-RNG</a> - <a href="#amd-securerng">AMD-SecureRNG</a> - <a href="#amd-uprof">AMD-uProf</a> - <a href="#amos">AMOS</a> - <a href="#amphora2">AMPHORA2</a> - <a href="#ampl-mp">AMPL-MP</a> - <a href="#amplimap">amplimap</a> - <a href="#amptk">AMPtk</a> - <a href="#amrplusplus">AmrPlusPlus</a> - <a href="#ams">AMS</a> - <a href="#anaconda2">Anaconda2</a> - <a href="#anaconda3">Anaconda3</a> - <a href="#anci">aNCI</a> - <a href="#andi">andi</a> - <a href="#angel">ANGEL</a> - <a href="#angsd">angsd</a> - <a href="#anicalculator">ANIcalculator</a> - <a href="#annif">Annif</a> - <a href="#annovar">annovar</a> - <a href="#ansys">ANSYS</a> - <a href="#ansys_cfd">ANSYS_CFD</a> - <a href="#ant">ant</a> - <a href="#antismash">antiSMASH</a> - <a href="#antlr">ANTLR</a> - <a href="#ants">ANTs</a> - <a href="#anvio">anvio</a> - <a href="#any2fasta">any2fasta</a> - <a href="#aocc">AOCC</a> - <a href="#aomp">AOMP</a> - <a href="#apbs">APBS</a> - <a href="#apex">apex</a> - <a href="#apr">APR</a> - <a href="#apr-util">APR-util</a> - <a href="#aptasuite">AptaSUITE</a> - <a href="#aragorn">ARAGORN</a> - <a href="#arb">Arb</a> - <a href="#arcade-learning-environment">Arcade-Learning-Environment</a> - <a href="#arcashla">arcasHLA</a> - <a href="#arch">ARCH</a> - <a href="#archive-zip">Archive-Zip</a> - <a href="#archr">ArchR</a> - <a href="#archspec">archspec</a> - <a href="#argos">ARGoS</a> - <a href="#argtable">argtable</a> - <a href="#aria2">aria2</a> - <a href="#arlequin">Arlequin</a> - <a href="#armadillo">Armadillo</a> - <a href="#arosics">arosics</a> - <a href="#arpack++">ARPACK++</a> - <a href="#arpack-ng">arpack-ng</a> - <a href="#arrayfire">ArrayFire</a> - <a href="#arriba">Arriba</a> - <a href="#arrow">Arrow</a> - <a href="#arrow-r">arrow-R</a> - <a href="#art">ART</a> - <a href="#artic-ncov2019">artic-ncov2019</a> - <a href="#arts">ARTS</a> - <a href="#arviz">ArviZ</a> - <a href="#arwen">ARWEN</a> - <a href="#asap">ASAP</a> - <a href="#asap3">ASAP3</a> - <a href="#ase">ASE</a> - <a href="#ashs">ASHS</a> - <a href="#aspera-cli">Aspera-CLI</a> - <a href="#aspera-connect">Aspera-Connect</a> - <a href="#assimp">assimp</a> - <a href="#assimulo">Assimulo</a> - <a href="#astrid">ASTRID</a> - <a href="#astropy">astropy</a> - <a href="#at-spi2-atk">at-spi2-atk</a> - <a href="#at-spi2-core">at-spi2-core</a> - <a href="#atk">ATK</a> - <a href="#atlas">ATLAS</a> - <a href="#atomate">atomate</a> - <a href="#atompaw">AtomPAW</a> - <a href="#atools">atools</a> - <a href="#atropos">atropos</a> - <a href="#atsas">ATSAS</a> - <a href="#attr">attr</a> - <a href="#attrdict">attrdict</a> - <a href="#augur">augur</a> - <a href="#augustus">AUGUSTUS</a> - <a href="#austin">Austin</a> - <a href="#auto-07p">AUTO-07p</a> - <a href="#autoconf">Autoconf</a> - <a href="#autoconf-archive">Autoconf-archive</a> - <a href="#autodock">AutoDock</a> - <a href="#autodock-gpu">AutoDock-GPU</a> - <a href="#autodock-vina">AutoDock-Vina</a> - <a href="#autogenes">AutoGeneS</a> - <a href="#autogrid">AutoGrid</a> - <a href="#automake">Automake</a> - <a href="#automap">AutoMap</a> - <a href="#autopep8">autopep8</a> - <a href="#autotools">Autotools</a> - <a href="#avogadro2">Avogadro2</a> - <a href="#awscli">awscli</a>


<a anchor="abaqus"/>
### *ABAQUS*

Finite Element Analysis software for modeling, visualization and best-in-class implicit and explicit  dynamics FEA.

*homepage*: http://www.simulia.com/products/abaqus_fea.html

version   |versionsuffix    |toolchain
----------|-----------------|----------
``6.12.1``|``-linux-x86_64``|``system``
``6.13.5``|``-linux-x86_64``|``system``
``6.14.1``|``-linux-x86_64``|``system``
``2017``  |``-hotfix-1721`` |``system``
``2018``  |``-hotfix-1806`` |``system``
``2020``  |                 |``system``
``2021``  |``-hotfix-2132`` |``system``
``2022``  |                 |``system``
``2022``  |``-hotfix-2214`` |``system``
``2022``  |``-hotfix-2223`` |``system``

<a anchor="abinit"/>
### *ABINIT*

ABINIT is a package whose main program allows one to find the total energy, charge density and electronic structure of systems made of electrons and nuclei (molecules and periodic solids) within Density Functional Theory (DFT), using pseudopotentials and a planewave or wavelet basis. 

*homepage*: https://www.abinit.org/

version   |versionsuffix           |toolchain
----------|------------------------|------------------------------------------------
``7.0.3`` |``-x86_64_linux_gnu4.5``|``system``
``7.0.5`` |``-x86_64_linux_gnu4.5``|``system``
``7.2.1`` |``-x86_64_linux_gnu4.5``|``system``
``7.10.5``|                        |``intel/2016.02-GCC-4.9``
``7.10.5``|``-libxc``              |``intel/2016.02-GCC-4.9``
``8.0.8`` |                        |``intel/2016a``
``8.0.8b``|                        |``foss/2016b``, ``intel/2016b``
``8.2.2`` |                        |``foss/2016b``, ``intel/2016b``
``8.4.4`` |                        |``intel/2017b``
``8.6.3`` |                        |``intel/2018a``
``8.10.2``|                        |``intel/2018b``
``8.10.3``|                        |``intel/2018b``
``9.2.1`` |                        |``intel/2020a``
``9.4.1`` |                        |``foss/2020b``, ``intel/2020a``, ``intel/2020b``
``9.4.2`` |                        |``foss/2021a``, ``intel/2021a``
``9.6.2`` |                        |``intel/2021a``, ``intel/2021b``

<a anchor="abra2"/>
### *ABRA2*

Assembly Based ReAligner

*homepage*: https://github.com/mozack/abra2

version |toolchain
--------|-----------------------------
``2.22``|``iccifort/2019.5.281``
``2.23``|``GCC/10.2.0``, ``GCC/9.3.0``

<a anchor="abricate"/>
### *ABRicate*

Mass screening of contigs for antimicrobial and virulence genes

*homepage*: https://github.com/tseemann/abricate

version  |versionsuffix   |toolchain
---------|----------------|---------------
``0.9.9``|                |``gompi/2019b``
``0.9.9``|``-Perl-5.28.1``|``gompi/2019a``
``1.0.0``|                |``gompi/2021a``

<a anchor="abseil"/>
### *Abseil*

Abseil is an open-source collection of C++ library code designed to augment the C++ standard library. The Abseil library code is collected from Google's own C++ code base, has been extensively tested and used in production, and is the same code we depend on in our daily coding lives.

*homepage*: https://abseil.io/

version       |toolchain
--------------|------------------
``20210324.2``|``GCCcore/11.2.0``

<a anchor="abtem"/>
### *abTEM*

abTEM provides a Python API for running simulations of Transmission Electron Microscopy images.

*homepage*: https://github.com/jacobjma/abTEM

version     |versionsuffix  |toolchain
------------|---------------|----------------------------------
``1.0.0b24``|``-ASE-3.22.0``|``fosscuda/2020b``
``1.0.0b26``|``-ASE-3.22.0``|``foss/2020b``, ``fosscuda/2020b``

<a anchor="abyss"/>
### *ABySS*

Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler

*homepage*: https://www.bcgsc.ca/platform/bioinfo/software/abyss

version  |toolchain
---------|----------------------------------------------------------------
``1.9.0``|``foss/2016a``
``2.0.2``|``foss/2016b``, ``foss/2018a``, ``gompi/2019a``, ``intel/2016b``
``2.0.3``|``foss/2017b``, ``intel/2017b``
``2.1.5``|``foss/2019b``
``2.2.5``|``foss/2020b``

<a anchor="ack"/>
### *ack*

ack is a tool like grep, optimized for programmers

*homepage*: https://beyondgrep.com

version  |toolchain
---------|------------------
``2.14`` |``system``
``3.4.0``|``GCCcore/10.2.0``
``3.5.0``|``GCCcore/10.3.0``

<a anchor="act"/>
### *ACT*

ACT is a Java application for displaying pairwise comparisons between two or more DNA sequences.  It can be used to identify and analyse regions of similarity and difference between genomes  and to explore conservation of synteny, in the context of the entire sequences and their annotation.  It can read complete EMBL, GENBANK and GFF entries or sequences in FASTA or raw format. 

*homepage*: https://sanger-pathogens.github.io/Artemis/ACT/

version   |versionsuffix|toolchain
----------|-------------|----------
``18.0.2``|``-Java-11`` |``system``
``18.0.3``|``-Java-11`` |``system``

<a anchor="actc"/>
### *ACTC*

ACTC converts independent triangles into triangle strips or fans.

*homepage*: https://sourceforge.net/projects/actc

version|toolchain
-------|-----------------------------------------------------------------------------------------------------------------------------------
``1.1``|``GCCcore/10.2.0``, ``GCCcore/11.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``intel/2016b``

<a anchor="ada"/>
### *ada*

Performs discrete, real, and gentle boost under both exponential and logistic loss on a given data set.

*homepage*: https://cran.r-project.org/web/packages/ada/index.html

version  |versionsuffix|toolchain
---------|-------------|---------------
``2.0-5``|``-R-3.4.0`` |``intel/2017a``

<a anchor="adapterremoval"/>
### *AdapterRemoval*

AdapterRemoval searches for and removes remnant adapter sequences  from High-Throughput Sequencing (HTS) data and (optionally) trims low quality bases  from the 3' end of reads following adapter removal.

*homepage*: https://github.com/MikkelSchubert/adapterremoval

version  |toolchain
---------|----------------------------------------------
``2.2.0``|``foss/2016b``
``2.2.2``|``foss/2018b``
``2.3.1``|``foss/2018b``
``2.3.2``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``
``2.3.3``|``GCC/11.3.0``

<a anchor="adda"/>
### *ADDA*

ADDA is an open-source parallel implementation of the discrete dipole approximation, capable to simulate light scattering by particles of arbitrary shape and composition in a wide range of particle sizes.

*homepage*: https://github.com/adda-team/adda/wiki

version  |toolchain
---------|--------------
``1.3b4``|``foss/2019a``

<a anchor="adf"/>
### *ADF*

ADF is an accurate, parallelized, powerful computational chemistry program to understand and  predict chemical structure and reactivity with density functional theory (DFT).

*homepage*: https://www.scm.com/ADF/

version                         |versionsuffix|toolchain
--------------------------------|-------------|---------------
``2009.01a.pc64_linux.intelmpi``|             |``system``
``2014.02``                     |             |``system``
``2014.11.r48287``              |             |``intel/2016a``
``2016.101``                    |             |``system``
``2019.303``                    |``-intelmpi``|``system``

<a anchor="adios"/>
### *ADIOS*

The Adaptable IO System (ADIOS) provides a simple, flexible way for scientists to describe the data in their code that may need to be written, read, or processed outside of the running simulation.

*homepage*: https://www.olcf.ornl.gov/center-projects/adios/

version     |versionsuffix     |toolchain
------------|------------------|--------------
``1.13.1``  |``-Python-2.7.15``|``foss/2019a``
``1.13.1``  |``-Python-3.8.2`` |``foss/2020a``
``20210804``|``-Python-3.8.2`` |``foss/2020a``

<a anchor="adjusttext"/>
### *adjustText*

A small library for automatically adjustment of text position in matplotlib plots to minimize overlaps.

*homepage*: https://github.com/Phlya/adjustText

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.7.3``|                 |``foss/2021b``
``0.7.3``|``-Python-3.6.6``|``foss/2018b``
``0.7.3``|``-Python-3.7.2``|``intel/2019a``

<a anchor="admixture"/>
### *ADMIXTURE*

ADMIXTURE is a software tool for maximum likelihood estimation of individual ancestries from  multilocus SNP genotype datasets. It uses the same statistical model as STRUCTURE but calculates estimates much  more rapidly using a fast numerical optimization algorithm.

*homepage*: https://dalexander.github.io/admixture/

version  |toolchain
---------|----------
``1.3.0``|``system``

<a anchor="adol-c"/>
### *ADOL-C*

The package ADOL-C (Automatic Differentiation by OverLoading in C++) facilitates the evaluation of first and higher derivatives of vector functions that are defined by computer programs written in C or C++. The resulting derivative evaluation routines may be called from C/C++, Fortran, or any other language that can be linked with C.  

*homepage*: https://projects.coin-or.org/ADOL-C

version  |toolchain
---------|---------------
``2.7.0``|``gompi/2019a``
``2.7.2``|``gompi/2020a``

<a anchor="advisor"/>
### *Advisor*

Vectorization Optimization and Thread Prototyping  - Vectorize & thread code or performance “dies”  - Easy workflow + data + tips = faster code faster  - Prioritize, Prototype & Predict performance gain  

*homepage*: https://software.intel.com/intel-advisor-xe

version         |toolchain
----------------|----------
``2016_update2``|``system``
``2017_update1``|``system``
``2018_update1``|``system``
``2018_update3``|``system``
``2019_update2``|``system``
``2019_update3``|``system``
``2019_update5``|``system``
``2021.2.0``    |``system``
``2021.4.0``    |``system``
``2022.1.0``    |``system``

<a anchor="afni"/>
### *AFNI*

AFNI is a set of C programs for processing, analyzing, and displaying functional MRI (FMRI) data -  a technique for mapping human brain activity.

*homepage*: http://afni.nimh.nih.gov/

version     |versionsuffix     |toolchain
------------|------------------|-------------------------------
``18.1.09`` |``-Python-3.6.4`` |``intel/2018a``
``18.3.00`` |``-Python-3.6.6`` |``foss/2018b``
``19.0.01`` |``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``20160329``|``-Python-2.7.11``|``intel/2016a``

<a anchor="agat"/>
### *AGAT*

AGAT: Another GTF/GFF Analysis Toolkit. Suite of tools to handle gene annotations  in any GTF/GFF format.

*homepage*: https://agat.readthedocs.io/en/latest/

version  |toolchain
---------|--------------
``0.9.2``|``GCC/11.2.0``

<a anchor="agfusion"/>
### *AGFusion*

AGFusion is a python package for annotating gene fusions  from the human or mouse genomes.

*homepage*: https://github.com/murphycj/AGFusion

version|versionsuffix    |toolchain
-------|-----------------|--------------
``1.2``|``-Python-3.7.2``|``foss/2019a``

<a anchor="aiohttp"/>
### *aiohttp*

Asynchronous HTTP client/server framework for asyncio and Python.

*homepage*: https://github.com/aio-libs/aiohttp

version  |versionsuffix    |toolchain
---------|-----------------|--------------------------------------
``3.5.4``|``-Python-3.6.6``|``foss/2018b``
``3.8.1``|                 |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``3.8.3``|                 |``GCCcore/11.3.0``

<a anchor="aladin"/>
### *ALADIN*

ALADIN was entirely built on the notion of compatibility with its mother system, IFS/ARPEG.  The latter, a joint development between the European Centre for Medium-Range Weather Forecasts (ECMWF) and  Meteo-France, was only meant to consider global Numerical Weather Prediction applications; hence the idea,  for ALADIN, to complement the IFS/ARPEGE project with a limited area model (LAM) version, while keeping the  differences between the two softwares as small as possible.

*homepage*: http://www.cnrm.meteo.fr/aladin/

version        |toolchain
---------------|---------------
``36t1_op2bf1``|``intel/2016a``

<a anchor="albacore"/>
### *Albacore*

Albacore is a software project that provides an entry point to the Oxford Nanopore basecalling  algorithms.

*homepage*: https://community.nanoporetech.com/protocols/albacore-offline-basecalli/v/abec_2003_v1_revz_29nov2016

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``2.0.2``|``-Python-3.6.1``|``intel/2017a``

<a anchor="albumentations"/>
### *Albumentations*

Albumentations is a Python library for fast and flexible image augmentations

*homepage*: https://albumentations.ai

version  |versionsuffix   |toolchain
---------|----------------|--------------
``1.1.0``|                |``foss/2021b``
``1.1.0``|``-CUDA-11.3.1``|``foss/2021a``
``1.3.0``|                |``foss/2022a``
``1.3.0``|``-CUDA-11.7.0``|``foss/2022a``

<a anchor="alevin-fry"/>
### *alevin-fry*

alevin-fry is an efficient and flexible tool for processing single-cell sequencing data,  currently focused on single-cell transcriptomics and feature barcoding.

*homepage*: https://github.com/COMBINE-lab/alevin-fry

version  |toolchain
---------|------------------
``0.4.3``|``GCCcore/11.2.0``
``0.6.0``|``GCCcore/10.3.0``

<a anchor="alfa"/>
### *ALFA*

ALFA provides a global overview of features distribution composing NGS dataset(s). Given a set of  aligned reads (BAM files) and an annotation file (GTF format), the tool produces plots of the raw and normalized  distributions of those reads among genomic categories (stop codon, 5'-UTR, CDS, intergenic, etc.) and biotypes  (protein coding genes, miRNA, tRNA, etc.). Whatever the sequencing technique, whatever the organism.

*homepage*: https://github.com/biocompibens/ALFA

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.1.1``|``-Python-3.7.2``|``foss/2019a``

<a anchor="alfred"/>
### *Alfred*

Alfred is an efficient and versatile command-line application  that computes multi-sample quality control metrics in a read-group aware manner.  Alfred supports read counting, feature annotation and haplotype-resolved consensus  computation using multiple sequence alignments.

*homepage*: https://www.gear-genomics.com/alfred/

version  |toolchain
---------|--------------
``0.2.6``|``GCC/11.2.0``

<a anchor="allelecount"/>
### *alleleCount*

The alleleCount package primarily exists to prevent code duplication between some other projects, specifically AscatNGS and Battenberg. As of v4 the perl  code wraps the C implementation of allele counting code for BAM/CRAM processing. 

*homepage*: http://cancerit.github.io/alleleCount/

version  |toolchain
---------|-----------------
``4.0.0``|``GCCcore/6.4.0``

<a anchor="allinea"/>
### *Allinea*

The Allinea environment is an essential toolkit for developers and computational scientists looking to get results faster.

*homepage*: http://www.allinea.com

version                        |toolchain
-------------------------------|----------
``4.1-32834-Redhat-5.7-x86_64``|``system``
``4.1-32834-Redhat-6.0-x86_64``|``system``
``6.1.1-Ubuntu-14.04-x86_64``  |``system``

<a anchor="allpaths-lg"/>
### *ALLPATHS-LG*

ALLPATHS-LG, the new short read genome assembler.

*homepage*: http://www.broadinstitute.org/software/allpaths-lg/blog/

version  |toolchain
---------|--------------
``52488``|``foss/2016a``

<a anchor="almosthere"/>
### *almosthere*

Progress indicator C library. ATHR is a simple yet powerful progress indicator library that works on Windows,  Linux, and macOS. It is non-blocking as the progress update is done via a  dedicated, lightweight thread, as to not impair the performance of the calling program.

*homepage*: https://github.com/horta/almosthere

version   |toolchain
----------|------------------
``1.0.1`` |``GCCcore/7.3.0``
``1.0.10``|``GCCcore/9.3.0``
``2.0.2`` |``GCCcore/10.2.0``

<a anchor="alpha"/>
### *Alpha*

Alpha is a tool designed for detailed comparative study of bacteriophage genomes.

*homepage*: https://www.lirmm.fr/~swenson/alpha/alpha.htm

version     |versionsuffix     |toolchain
------------|------------------|--------------
``20200430``|``-Python-2.7.16``|``foss/2019b``

<a anchor="alphafold"/>
### *AlphaFold*

AlphaFold can predict protein structures with atomic accuracy even where no similar structure is known

*homepage*: https://deepmind.com/research/case-studies/alphafold

version  |versionsuffix        |toolchain
---------|---------------------|----------------------------------
``2.0.0``|                     |``foss/2020b``, ``fosscuda/2020b``
``2.0.1``|                     |``foss/2020b``, ``fosscuda/2020b``
``2.1.1``|                     |``fosscuda/2020b``
``2.1.2``|                     |``foss/2021a``
``2.1.2``|``-CUDA-11.3.1``     |``foss/2021a``
``2.1.2``|``-TensorFlow-2.5.0``|``foss/2020b``, ``fosscuda/2020b``
``2.2.2``|                     |``foss/2021a``
``2.2.2``|``-CUDA-11.3.1``     |``foss/2021a``

<a anchor="alps"/>
### *ALPS*

The ALPS project (Algorithms and Libraries for Physics Simulations) is an open source effort  aiming at providing high-end simulation codes for strongly correlated quantum mechanical systems as well as   C++ libraries for simplifying the development of such code. 

*homepage*: http://alps.comp-phys.org/

version   |versionsuffix     |toolchain
----------|------------------|---------------
``2.2.b4``|``-Python-2.7.11``|``intel/2016a``
``2.3.0`` |``-Python-2.7.12``|``foss/2016b``
``2.3.0`` |``-Python-3.5.2`` |``foss/2016b``

<a anchor="alsa-lib"/>
### *alsa-lib*

The Advanced Linux Sound Architecture (ALSA) provides audio and MIDI functionality  to the Linux operating system.

*homepage*: https://www.alsa-project.org

version  |toolchain
---------|--------------------------------------
``1.2.4``|``GCCcore/9.3.0``
``1.2.8``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="amapvox"/>
### *AMAPVox*

LiDAR data voxelisation software

*homepage*: https://amap-dev.cirad.fr/projects/amapvox

version  |versionsuffix|toolchain
---------|-------------|----------
``1.9.4``|``-Java-11`` |``system``

<a anchor="amara"/>
### *Amara*

Library for XML processing in Python, designed to balance the native idioms of Python  with the native character of XML.

*homepage*: https://pypi.org/project/Amara

version    |versionsuffix     |toolchain
-----------|------------------|-------------------------------
``1.2.0.2``|``-Python-2.7.15``|``foss/2019a``, ``intel/2019a``

<a anchor="amask"/>
### *amask*

amask is a set of tools to to determine the affinity of MPI processes and OpenMP threads  in a parallel environment.

*homepage*: https://github.com/TACC/amask

version     |toolchain
------------|--------------
``20171106``|``foss/2018a``
``20190404``|``foss/2018b``

<a anchor="amber"/>
### *Amber*

Amber (originally Assisted Model Building with Energy Refinement) is software for performing  molecular dynamics and structure prediction.

*homepage*: https://ambermd.org/amber.html

version  |versionsuffix                                    |toolchain
---------|-------------------------------------------------|------------------------------------------------------------------------
``14``   |``-AmberTools-15-patchlevel-13-13``              |``intel/2016a``
``16``   |``-AmberTools-16-patchlevel-5-14``               |``iomkl/2016.07``
``16``   |``-AmberTools-16-patchlevel-5-14-CUDA``          |``iomkl/2016.09-GCC-4.9.3-2.25``
``16``   |``-AmberTools-16-patchlevel-5-14-serial``        |``iomkl/2016.07``
``16``   |``-AmberTools-17-patchlevel-10-15``              |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``16``   |``-AmberTools-17-patchlevel-10-15-Python-2.7.14``|``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``16``   |``-AmberTools-17-patchlevel-8-12``               |``intel/2017b``
``18``   |``-AmberTools-18-patchlevel-10-8``               |``foss/2018b``, ``fosscuda/2018b``, ``intel/2017b``
``18``   |``-AmberTools-19-patchlevel-12-17-Python-2.7.16``|``foss/2019b``, ``fosscuda/2019b``
``20.11``|``-AmberTools-20.15-Python-3.8.2``               |``foss/2020a``, ``fosscuda/2020a``
``20.11``|``-AmberTools-21.3``                             |``foss/2020b``, ``fosscuda/2020b``
``22.0`` |``-AmberTools-22.3``                             |``foss/2021b``

<a anchor="ambermini"/>
### *AmberMini*

A stripped-down set of just antechamber, sqm, and tleap.

*homepage*: https://github.com/choderalab/ambermini

version    |toolchain
-----------|--------------------------------
``16.16.0``|``intel/2017b``, ``intel/2020a``

<a anchor="ambertools"/>
### *AmberTools*

AmberTools consists of several independently developed packages that work well by themselves,  and with Amber itself. The suite can also be used to carry out complete molecular dynamics simulations,  with either explicit water or generalized Born solvent models.

*homepage*: https://ambermd.org/

version  |versionsuffix     |toolchain
---------|------------------|--------------------------------
``17``   |                  |``intel/2017b``, ``intel/2018a``
``17``   |``-Python-2.7.14``|``foss/2018a``
``20``   |``-Python-3.8.2`` |``intel/2020a``
``21``   |                  |``foss/2021a``, ``intel/2021b``
``21.12``|                  |``foss/2021b``
``22.3`` |                  |``foss/2021b``

<a anchor="amd-libm"/>
### *AMD-LibM*

AMD LibM is a software library containing a collection of basic math functions optimized for x86-64 processor based machines.

*homepage*: https://developer.amd.com/amd-cpu-libraries/amd-math-library-libm/

version    |toolchain
-----------|------------------
``3.2.2``  |``GCC/7.3.0-2.30``
``3.6.0-4``|``GCC/9.3.0``

<a anchor="amd-rng"/>
### *AMD-RNG*

AMD Random Number Generator Library is a pseudorandom number generator library.

*homepage*: https://developer.amd.com/amd-cpu-libraries/rng-library/

version  |toolchain
---------|------------------
``1.0``  |``GCC/7.3.0-2.30``
``2.2-4``|``GCC/9.3.0``

<a anchor="amd-securerng"/>
### *AMD-SecureRNG*

The AMD Secure Random Number Generator (RNG) is a library that provides APIs to access the cryptographically secure random numbers generated by AMD’s hardware-based random number generator implementation.

*homepage*: https://developer.amd.com/amd-cpu-libraries/rng-library/

version  |toolchain
---------|------------------
``1.0``  |``GCC/7.3.0-2.30``
``2.2-4``|``GCC/9.3.0``

<a anchor="amd-uprof"/>
### *AMD-uProf*

AMD uProf is a performance analysis tool for applications running on Windows, Linux & FreeBSD  operating systems. It allows developers to better understand the runtime performance of their application and  to identify ways to improve its performance.

*homepage*: https://developer.amd.com/amd-uprof/

version    |toolchain
-----------|----------
``3.4.502``|``system``
``3.5.671``|``system``

<a anchor="amos"/>
### *AMOS*

The AMOS consortium is committed to the development of open-source whole genome assembly software

*homepage*: http://amos.sourceforge.net

version  |toolchain
---------|------------------------------
``3.1.0``|``foss/2018b``, ``foss/2021b``

<a anchor="amphora2"/>
### *AMPHORA2*

An Automated Phylogenomic Inference Pipeline for Bacterial and Archaeal Sequences.

*homepage*: https://github.com/wu-lab-uva/AMPHORA2

version     |versionsuffix             |toolchain
------------|--------------------------|---------------
``20190730``|``-Java-13-pthreads-avx2``|``gompi/2020b``

<a anchor="ampl-mp"/>
### *AMPL-MP*

An open-source library for mathematical programming. 

*homepage*: https://github.com/ampl/mp

version  |toolchain
---------|-----------------
``3.1.0``|``GCCcore/6.4.0``

<a anchor="amplimap"/>
### *amplimap*

amplimap is a command-line tool to automate the processing and analysis of data from targeted  next-generation sequencing (NGS) experiments with PCR-based amplicons or capture-based enrichment systems.

*homepage*: https://amplimap.readthedocs.io/en/latest/

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.4.16``|``-Python-3.6.6``|``foss/2018b``

<a anchor="amptk"/>
### *AMPtk*

AMPtk is a series of scripts to process NGS amplicon data using USEARCH and VSEARCH, it can also be used to process any NGS amplicon data and includes databases setup for analysis of fungal ITS, fungal LSU, bacterial 16S, and insect COI amplicons.

*homepage*: https://amptk.readthedocs.io

version  |toolchain
---------|--------------
``1.5.4``|``foss/2021b``

<a anchor="amrplusplus"/>
### *AmrPlusPlus*

AmrPlusPlus v2.0 can process the raw data from the sequencer, identify the fragments of DNA, and count them. It also provides a count of the polymorphisms that occur in each DNA fragment with respect to the reference database. 

*homepage*: https://megares.meglab.org/amrplusplus/latest/html/v2

version         |toolchain
----------------|-------------
``2.0-20200114``|``GCC/8.3.0``

<a anchor="ams"/>
### *AMS*

The Amsterdam Modeling Suite (AMS) provides a comprehensive set of modules for computational chemistry and materials science, from quantum mechanics to fluid thermodynamics. 

*homepage*: https://www.scm.com/amsterdam-modeling-suite/

version     |versionsuffix|toolchain
------------|-------------|---------------
``2020.102``|``-intelmpi``|``iimpi/2020b``
``2022.102``|``-intelmpi``|``iimpi/2021b``

<a anchor="anaconda2"/>
### *Anaconda2*

Built to complement the rich, open source Python community, the Anaconda platform provides an enterprise-ready data analytics platform  that empowers companies to adopt a modern open data science analytics architecture. 

*homepage*: https://www.anaconda.com

version    |toolchain
-----------|----------
``4.0.0``  |``system``
``4.2.0``  |``system``
``4.4.0``  |``system``
``5.0.1``  |``system``
``5.1.0``  |``system``
``5.3.0``  |``system``
``2018.12``|``system``
``2019.03``|``system``
``2019.07``|``system``
``2019.10``|``system``

<a anchor="anaconda3"/>
### *Anaconda3*

Built to complement the rich, open source Python community, the Anaconda platform provides an enterprise-ready data analytics platform  that empowers companies to adopt a modern open data science analytics architecture. 

*homepage*: https://www.anaconda.com

version    |toolchain
-----------|----------
``4.0.0``  |``system``
``4.2.0``  |``system``
``4.4.0``  |``system``
``5.0.1``  |``system``
``5.1.0``  |``system``
``5.3.0``  |``system``
``2018.12``|``system``
``2019.03``|``system``
``2019.07``|``system``
``2019.10``|``system``
``2020.02``|``system``
``2020.07``|``system``
``2020.11``|``system``
``2021.05``|``system``
``2021.11``|``system``
``2022.05``|``system``

<a anchor="anci"/>
### *aNCI*

Non-covalent interaction (NCI) for MD trajectories

*homepage*: https://www.lct.jussieu.fr/pagesperso/contrera/nci-MD.html

version|toolchain
-------|-----------------------
``2.0``|``iccifort/2019.5.281``

<a anchor="andi"/>
### *andi*

This is the andi program for estimating the evolutionary distance between closely related genomes. These distances can be used to rapidly infer phylogenies for big sets of genomes. Because andi does not compute full alignments, it is so efficient that it scales even up to thousands of bacterial genomes.

*homepage*: https://github.com/evolbioinf/andi/

version |toolchain
--------|--------------
``0.13``|``foss/2018b``

<a anchor="angel"/>
### *ANGEL*

ANGEL: Robust Open Reading Frame prediction

*homepage*: https://github.com/PacificBiosciences/ANGEL

version|versionsuffix    |toolchain
-------|-----------------|--------------
``3.0``|``-Python-3.7.2``|``foss/2019a``

<a anchor="angsd"/>
### *angsd*

Program for analysing NGS data.

*homepage*: http://www.popgen.dk/angsd

version  |toolchain
---------|--------------------------------------
``0.910``|``foss/2016a``
``0.921``|``foss/2018a``
``0.925``|``foss/2018b``
``0.933``|``GCC/8.3.0``, ``iccifort/2019.5.281``
``0.935``|``GCC/10.2.0``

<a anchor="anicalculator"/>
### *ANIcalculator*

This tool will calculate the bidirectional average nucleotide identity (gANI) and  Alignment Fraction (AF) between two genomes. Required input is the full path to the fna file  (nucleotide sequence of genes in fasta format) of each query genome. Either the rRNA and tRNA genes  can be excluded, or provided in a list with the -ignoreList option. This is necessary as the presence of tRNA and/or rRNA genes in the fna will artificially inflate the ANI.

*homepage*: https://ani.jgi.doe.gov/html/home.php

version|toolchain
-------|------------------
``1.0``|``GCCcore/10.3.0``

<a anchor="annif"/>
### *Annif*

Annif is a multi-algorithm automated subject indexing tool for libraries, archives and museums.

*homepage*: https://github.com/NatLibFi/Annif

version   |versionsuffix    |toolchain
----------|-----------------|-------------------------------
``0.40.0``|``-Python-3.7.2``|``foss/2019a``, ``intel/2019a``

<a anchor="annovar"/>
### *annovar*

ANNOVAR is an efficient software tool to utilize update-to-date information  to functionally annotate genetic variants detected from diverse genomes (including human  genome hg18, hg19, hg38, as well as mouse, worm, fly, yeast and many others).

*homepage*: http://annovar.openbioinformatics.org/en/latest/

version      |versionsuffix   |toolchain
-------------|----------------|------------------
``2016Feb01``|``-Perl-5.22.1``|``foss/2016a``
``20191024`` |``-Perl-5.28.1``|``GCCcore/8.2.0``
``20200607`` |``-Perl-5.34.0``|``GCCcore/11.2.0``

<a anchor="ansys"/>
### *ANSYS*

ANSYS simulation software enables organizations to confidently predict      how their products will operate in the real world. We believe that every product is      a promise of something greater. 

*homepage*: https://www.ansys.com

version   |toolchain
----------|----------
``15.0``  |``system``
``2022R2``|``system``

<a anchor="ansys_cfd"/>
### *ANSYS_CFD*

ANSYS computational fluid dynamics (CFD) simulation software allows you to predict, with confidence,  the impact of fluid flows on your product throughout design and manufacturing as well as during end use.  ANSYS renowned CFD analysis tools include the widely used and well-validated ANSYS Fluent and ANSYS CFX.

*homepage*: http://www.ansys.com/Products/Simulation+Technology/Fluid+Dynamics

version |toolchain
--------|----------
``16.2``|``system``
``17.0``|``system``

<a anchor="ant"/>
### *ant*

Apache Ant is a Java library and command-line tool whose mission is to drive processes described in  build files as targets and extension points dependent upon each other. The main known usage of Ant is the build of  Java applications.

*homepage*: http://ant.apache.org/

version    |versionsuffix      |toolchain
-----------|-------------------|----------
``1.8.4``  |``-Java-1.7.0_10`` |``system``
``1.8.4``  |``-Java-1.7.0_21`` |``system``
``1.9.0``  |``-Java-1.7.0_15`` |``system``
``1.9.0``  |``-Java-1.7.0_21`` |``system``
``1.9.3``  |``-Java-1.7.0_60`` |``system``
``1.9.3``  |``-Java-1.7.0_79`` |``system``
``1.9.6``  |``-Java-1.7.0_75`` |``system``
``1.9.6``  |``-Java-1.7.0_79`` |``system``
``1.9.6``  |``-Java-1.7.0_80`` |``system``
``1.9.6``  |``-Java-1.8.0_66`` |``system``
``1.9.6``  |``-Java-1.8.0_72`` |``system``
``1.9.6``  |``-Java-1.8.0_77`` |``system``
``1.9.7``  |``-Java-1.8.0_92`` |``system``
``1.10.0`` |``-Java-1.8.0_112``|``system``
``1.10.1`` |``-Java-1.8``      |``system``
``1.10.1`` |``-Java-1.8.0_121``|``system``
``1.10.1`` |``-Java-1.8.0_144``|``system``
``1.10.1`` |``-Java-1.8.0_152``|``system``
``1.10.1`` |``-Java-1.8.0_162``|``system``
``1.10.5`` |``-Java-1.8``      |``system``
``1.10.6`` |``-Java-1.8``      |``system``
``1.10.7`` |``-Java-11``       |``system``
``1.10.8`` |``-Java-11``       |``system``
``1.10.9`` |``-Java-11``       |``system``
``1.10.11``|``-Java-11``       |``system``
``1.10.11``|``-Java-13``       |``system``
``1.10.12``|``-Java-11``       |``system``
``1.10.12``|``-Java-17``       |``system``

<a anchor="antismash"/>
### *antiSMASH*

antiSMASH allows the rapid genome-wide identification, annotation and analysis of secondary  metabolite biosynthesis gene clusters in bacterial and fungal genomes.

*homepage*: https://github.com/antismash/antismash

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``5.1.2``|``-Python-3.7.4``|``foss/2019b``
``5.2.0``|                 |``foss/2020b``
``6.0.1``|                 |``foss/2020b``

<a anchor="antlr"/>
### *ANTLR*

ANTLR, ANother Tool for Language Recognition, (formerly PCCTS)  is a language tool that provides a framework for constructing recognizers,  compilers, and translators from grammatical descriptions containing  Java, C#, C++, or Python actions.

*homepage*: http://www.antlr2.org/

version  |versionsuffix     |toolchain
---------|------------------|--------------------------------------------------------------------------------------------------------------------
``2.7.7``|                  |``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``foss/2017b``, ``foss/2018b``, ``foss/2019a``, ``intel/2017b``
``2.7.7``|``-Java-11``      |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``2.7.7``|``-Python-2.7.11``|``foss/2016a``
``2.7.7``|``-Python-2.7.13``|``intel/2017a``
``2.7.7``|``-Python-2.7.14``|``foss/2018a``, ``intel/2017b``
``2.7.7``|``-Python-3.6.4`` |``intel/2018a``

<a anchor="ants"/>
### *ANTs*

ANTs extracts information from complex datasets that include imaging. ANTs is useful for managing,  interpreting and visualizing multidimensional data.

*homepage*: https://stnava.github.io/ANTs/

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``2.2.0``|``-Python-2.7.12``|``foss/2016b``
``2.3.0``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``2.3.1``|``-Python-3.6.6`` |``foss/2018b``
``2.3.2``|``-Python-3.7.4`` |``foss/2019b``
``2.3.5``|                  |``foss/2021a``

<a anchor="anvio"/>
### *anvio*

An analysis and visualization platform for 'omics data.

*homepage*: https://merenlab.org/software/anvio/

version|versionsuffix    |toolchain
-------|-----------------|---------------
``6.1``|``-Python-3.7.4``|``intel/2019b``

<a anchor="any2fasta"/>
### *any2fasta*

Convert various sequence formats to FASTA

*homepage*: https://github.com/tseemann/any2fasta

version  |versionsuffix   |toolchain
---------|----------------|------------------------------------------------------------------------------------------------
``0.4.2``|                |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``0.4.2``|``-Perl-5.28.1``|``GCCcore/8.2.0``

<a anchor="aocc"/>
### *AOCC*

AMD Optimized C/C++ & Fortran compilers (AOCC) based on LLVM 13.0

*homepage*: https://developer.amd.com/amd-aocc/

version  |toolchain
---------|----------------------------------------------------------
``2.3.0``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``3.0.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``3.1.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``3.2.0``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``4.0.0``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``

<a anchor="aomp"/>
### *AOMP*

AMD fork of LLVM, setup for OpenMP offloading to Accelerators

*homepage*: https://github.com/ROCm-Developer-Tools/aomp

version   |toolchain
----------|-------------------------------------
``13.0-2``|``GCCcore/10.2.0``, ``gcccuda/2020a``

<a anchor="apbs"/>
### *APBS*

APBS is a software package for modeling biomolecular solvation   through solution of the Poisson-Boltzmann equation (PBE), one of the most popular   continuum models for describing electrostatic interactions between molecular solutes   in salty, aqueous media. 

*homepage*: http://www.poissonboltzmann.org/apbs

version|versionsuffix           |toolchain
-------|------------------------|----------
``1.4``|``-linux-static-x86_64``|``system``

<a anchor="apex"/>
### *apex*

A PyTorch Extension: Tools for easy mixed precision and distributed training in Pytorch

*homepage*: https://github.com/nvidia/apex

version     |versionsuffix    |toolchain
------------|-----------------|------------------
``20200325``|``-Python-3.7.4``|``fosscuda/2019b``
``20210420``|                 |``fosscuda/2020b``

<a anchor="apr"/>
### *APR*

Apache Portable Runtime (APR) libraries.

*homepage*: https://apr.apache.org/

version  |toolchain
---------|--------------------------------------------------------------------------------------------------------------------
``1.6.3``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``iomkl/2018a``
``1.7.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/8.2.0``, ``GCCcore/9.3.0``

<a anchor="apr-util"/>
### *APR-util*

Apache Portable Runtime (APR) util libraries.

*homepage*: http://apr.apache.org/

version  |toolchain
---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.6.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/9.3.0``, ``iomkl/2018a``

<a anchor="aptasuite"/>
### *AptaSUITE*

A full-featured bioinformatics software collection for the  comprehensive analysis of aptamers in HT-SELEX experiments 

*homepage*: https://drivenbyentropy.github.io/

version  |versionsuffix|toolchain
---------|-------------|----------
``0.9.4``|``-Java-11`` |``system``

<a anchor="aragorn"/>
### *ARAGORN*

a program to detect tRNA genes and tmRNA genes in nucleotide sequences

*homepage*: http://www.ansikte.se/ARAGORN/

version   |toolchain
----------|---------------------------------------
``1.2.38``|``foss/2016b``, ``iccifort/2019.5.281``
``1.2.41``|``foss/2021b``

<a anchor="arb"/>
### *Arb*

Arb is a C library for arbitrary-precision interval arithmetic.  It has full support for both real and complex numbers. The library is thread-safe,  portable, and extensively tested.

*homepage*: https://arblib.org/

version   |toolchain
----------|--------------------------------------------------------------------------------
``2.16.0``|``GCC/7.3.0-2.30``, ``GCC/8.2.0-2.31.1``, ``iccifort/2018.3.222-GCC-7.3.0-2.30``
``2.17.0``|``GCC/8.3.0``
``2.19.0``|``GCC/10.3.0``
``2.22.1``|``GCC/11.2.0``

<a anchor="arcade-learning-environment"/>
### *Arcade-Learning-Environment*

The Arcade Learning Environment (ALE) is a simple framework that allows researchers and hobbyists to develop AI agents for Atari 2600 games. It is built on top of the Atari 2600 emulator Stella and separates the details of emulation from agent design. This video depicts over 50 games currently supported in the ALE.

*homepage*: https://github.com/mgbellemare/Arcade-Learning-Environment

version  |toolchain
---------|--------------
``0.7.3``|``foss/2021b``

<a anchor="arcashla"/>
### *arcasHLA*

arcasHLA performs high resolution genotyping for HLA class I and class II genes from RNA sequencing, supporting both paired and single-end samples.

*homepage*: https://github.com/RabadanLab/arcasHLA

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.2.0``|``-Python-3.7.4``|``foss/2019b``

<a anchor="arch"/>
### *ARCH*

Autoregressive Conditional Heteroskedasticity (ARCH) and other tools for financial econometrics,  written in Python (with Cython and/or Numba used to improve performance).

*homepage*: https://pypi.org/project/arch

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``4.5.0``|``-Python-3.6.4``|``intel/2018a``

<a anchor="archive-zip"/>
### *Archive-Zip*

Provide an interface to ZIP archive files.

*homepage*: https://metacpan.org/pod/Archive::Zip

version |toolchain
--------|------------------------------------------------------------------------------
``1.68``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="archr"/>
### *ArchR*

ArchR is a full-featured R package for processing and analyzing single-cell ATAC-seq data.

*homepage*: https://www.archrproject.com

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.0.1``|``-R-4.1.2`` |``foss/2021b``

<a anchor="archspec"/>
### *archspec*

A library for detecting, labeling, and reasoning about microarchitectures

*homepage*: https://github.com/archspec/archspec

version  |versionsuffix    |toolchain
---------|-----------------|--------------------------------------
``0.1.0``|``-Python-3.7.4``|``GCCcore/8.3.0``
``0.1.0``|``-Python-3.8.2``|``GCCcore/9.3.0``
``0.1.2``|                 |``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``0.1.3``|                 |``GCCcore/11.2.0``
``0.1.4``|                 |``GCCcore/11.3.0``

<a anchor="argos"/>
### *ARGoS*

A parallel, multi-engine simulator for heterogeneous swarm robotics

*homepage*: http://www.argos-sim.info

version         |versionsuffix |toolchain
----------------|--------------|--------------
``3.0.0-beta53``|``-Lua-5.2.4``|``foss/2018b``
``3.0.0-beta59``|              |``GCC/11.2.0``

<a anchor="argtable"/>
### *argtable*

Argtable is an ANSI C library for parsing GNU style  command line options with a minimum of fuss. 

*homepage*: http://argtable.sourceforge.net/

version |toolchain
--------|--------------------------------------------------------------------------------------------------------------------------
``2.13``|``GCCcore/10.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``foss/2016b``, ``foss/2018b``, ``intel/2018a``, ``intel/2018b``

<a anchor="aria2"/>
### *aria2*

aria2 is a lightweight multi-protocol & multi-source command-line download utility.

*homepage*: https://aria2.github.io

version   |toolchain
----------|------------------
``1.35.0``|``GCCcore/10.3.0``
``1.36.0``|``GCCcore/11.3.0``

<a anchor="arlequin"/>
### *Arlequin*

Arlequin: An Integrated Software for Population Genetics Data Analysis

*homepage*: http://cmpg.unibe.ch/software/arlequin35/Arlequin35.html

version    |toolchain
-----------|--------------
``3.5.2.2``|``foss/2019b``

<a anchor="armadillo"/>
### *Armadillo*

Armadillo is an open-source C++ linear algebra library (matrix maths) aiming towards  a good balance between speed and ease of use. Integer, floating point and complex numbers are supported,  as well as a subset of trigonometric and statistics functions.

*homepage*: https://arma.sourceforge.net/

version    |versionsuffix     |toolchain
-----------|------------------|-------------------------------
``7.600.2``|``-Python-2.7.12``|``foss/2016b``
``7.800.0``|``-Python-2.7.12``|``intel/2016b``
``7.950.1``|``-Python-2.7.12``|``intel/2016b``
``8.300.1``|                  |``foss/2017b``, ``intel/2017b``
``8.400.0``|                  |``foss/2018a``
``9.600.5``|                  |``foss/2018b``
``9.700.2``|                  |``foss/2019a``
``9.880.1``|                  |``foss/2020a``
``9.900.1``|                  |``foss/2019b``, ``foss/2020a``
``10.5.3`` |                  |``foss/2020b``
``10.7.5`` |                  |``foss/2021a``

<a anchor="arosics"/>
### *arosics*

AROSICS is a python package to perform automatic subpixel co-registration of two satellite image datasets based on an image matching approach working in the frequency domain, combined with a multistage workflow for effective detection of false-positives.

*homepage*: https://github.com/GFZ/arosics

version  |toolchain
---------|--------------
``1.7.6``|``foss/2021a``

<a anchor="arpack++"/>
### *ARPACK++*

Arpackpp is a C++ interface to the ARPACK Fortran package,  which implements the implicit restarted Arnoldi method for  iteratively solving large-scale sparse eigenvalue problems.

*homepage*: https://github.com/m-reuter/arpackpp

version       |toolchain
--------------|--------------
``2018.03.26``|``foss/2017b``

<a anchor="arpack-ng"/>
### *arpack-ng*

ARPACK is a collection of Fortran77 subroutines designed to solve large scale eigenvalue problems.

*homepage*: https://github.com/opencollab/arpack-ng

version  |toolchain
---------|--------------------------------------------------------------------------------
``3.3.0``|``foss/2016a``, ``intel/2016a``
``3.4.0``|``foss/2016b``, ``foss/2017a``, ``intel/2016b``, ``intel/2017a``
``3.5.0``|``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``intel/2017a``, ``intel/2017b``
``3.6.2``|``intel/2018a``
``3.7.0``|``foss/2019a``, ``foss/2019b``, ``foss/2020a``, ``intel/2019b``, ``intel/2020a``
``3.8.0``|``foss/2020b``, ``foss/2021a``, ``foss/2021b``, ``foss/2022a``

<a anchor="arrayfire"/>
### *ArrayFire*

ArrayFire is a general-purpose library that simplifies the process of  developing software that targets parallel and massively-parallel architectures  including CPUs, GPUs, and other hardware acceleration devices. 

*homepage*: https://arrayfire.com/

version  |versionsuffix   |toolchain
---------|----------------|--------------
``3.6.4``|                |``foss/2018b``
``3.6.4``|``-CUDA-9.2.88``|``foss/2018b``

<a anchor="arriba"/>
### *Arriba*

Arriba is a command-line tool for the detection of gene fusions from RNA-Seq data.  It was developed for the use in a clinical research setting. Therefore, short runtimes and high  sensitivity were important design criteria.

*homepage*: https://github.com/suhrig/arriba

version  |toolchain
---------|------------------------------
``1.1.0``|``foss/2018b``
``2.1.0``|``GCC/10.2.0``, ``GCC/10.3.0``
``2.2.1``|``GCC/11.2.0``
``2.3.0``|``GCC/11.2.0``

<a anchor="arrow"/>
### *Arrow*

Apache Arrow (incl. PyArrow Python bindings), a cross-language development platform  for in-memory data.

*homepage*: https://arrow.apache.org

version   |versionsuffix     |toolchain
----------|------------------|---------------------------------------------------
``0.7.1`` |``-Python-3.6.3`` |``intel/2017b``
``0.12.0``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``0.12.0``|``-Python-3.6.6`` |``intel/2018b``
``0.16.0``|``-Python-3.7.4`` |``foss/2019b``, ``intel/2019b``
``0.17.1``|                  |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``
``0.17.1``|``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``6.0.0`` |                  |``foss/2021a``, ``foss/2021b``
``6.0.1`` |                  |``foss/2021a``
``8.0.0`` |                  |``foss/2022.05``, ``foss/2022a``

<a anchor="arrow-r"/>
### *arrow-R*

R interface to the Apache Arrow C++ library

*homepage*: https://cran.r-project.org/web/packages/arrow

version    |versionsuffix|toolchain
-----------|-------------|--------------
``0.17.1`` |``-R-4.0.0`` |``foss/2020a``
``6.0.0.2``|``-R-4.1.0`` |``foss/2021a``
``6.0.0.2``|``-R-4.1.2`` |``foss/2021b``
``6.0.0.2``|``-R-4.2.0`` |``foss/2021b``
``8.0.0``  |``-R-4.2.1`` |``foss/2022a``

<a anchor="art"/>
### *ART*

ART is a set of simulation tools to generate synthetic next-generation sequencing reads

*homepage*: http://www.niehs.nih.gov/research/resources/software/biostatistics/art/

version       |toolchain
--------------|----------------------------------
``2016.06.05``|``GCCcore/6.4.0``, ``intel/2016b``

<a anchor="artic-ncov2019"/>
### *artic-ncov2019*

Initial implementation of an ARTIC bioinformatics platform  for nanopore sequencing of nCoV2019 novel coronavirus.

*homepage*: https://github.com/artic-network/artic-ncov2019

version       |versionsuffix    |toolchain
--------------|-----------------|--------------
``2020.04.13``|``-Python-3.6.6``|``foss/2018b``
``2021.06.24``|                 |``foss/2020b``

<a anchor="arts"/>
### *ARTS*

ARTS is a radiative transfer model for the millimeter and sub-millimeter  spectral range. There are a number of models mostly developed explicitly  for the different sensors.

*homepage*: http://www.radiativetransfer.org/

version   |toolchain
----------|---------------
``2.2.64``|``gompi/2019a``

<a anchor="arviz"/>
### *ArviZ*

Exploratory analysis of Bayesian models with Python

*homepage*: https://github.com/arviz-devs/arviz

version   |versionsuffix    |toolchain
----------|-----------------|-------------------------------
``0.7.0`` |``-Python-3.7.4``|``foss/2019b``, ``intel/2019b``
``0.11.1``|                 |``intel/2020b``
``0.11.4``|                 |``foss/2021b``, ``intel/2021b``
``0.12.1``|                 |``foss/2022a``, ``intel/2022a``

<a anchor="arwen"/>
### *ARWEN*

ARWEN, tRNA detection in metazoan mitochondrial sequences

*homepage*: http://mbio-serv2.mbioekol.lu.se/ARWEN

version  |toolchain
---------|-----------------
``1.2.3``|``GCCcore/7.3.0``

<a anchor="asap"/>
### *ASAP*

ASAP focuses on fast and fluid image viewing with an easy-to-use interface  for making annotations. It consists of two main components: an IO library for reading and writing  multi-resolution images and a viewer component for visualizing such images.

*homepage*: https://computationalpathologygroup.github.io/ASAP/

version|versionsuffix   |toolchain
-------|----------------|--------------
``2.0``|``-CUDA-11.3.1``|``foss/2021a``
``2.1``|                |``foss/2022a``

<a anchor="asap3"/>
### *ASAP3*

ASAP is a calculator for doing large-scale classical molecular dynamics within the Campos Atomic Simulation Environment (ASE).

*homepage*: https://wiki.fysik.dtu.dk/asap/

version    |versionsuffix    |toolchain
-----------|-----------------|------------------------------------------------
``3.10.7`` |``-Python-3.5.2``|``foss/2016b``
``3.10.7`` |``-Python-3.6.2``|``foss/2017b``
``3.10.8`` |``-Python-3.5.2``|``foss/2016b``
``3.10.8`` |``-Python-3.6.2``|``foss/2017b``
``3.10.8`` |``-Python-3.6.3``|``foss/2017b``
``3.10.10``|``-Python-3.6.6``|``foss/2018b``, ``intel/2018b``, ``iomkl/2018b``
``3.11.10``|``-Python-3.7.2``|``foss/2019a``, ``intel/2019a``
``3.12.2`` |``-ASE-3.21.1``  |``foss/2020b``, ``intel/2020b``
``3.12.7`` |``-ASE-3.21.1``  |``foss/2020b``, ``intel/2020b``

<a anchor="ase"/>
### *ASE*

ASE is a python package providing an open source Atomic Simulation Environment  in the Python scripting language.

*homepage*: https://wiki.fysik.dtu.dk/ase/

version       |versionsuffix     |toolchain
--------------|------------------|------------------------------------------------------------------------------------
``3.9.1.4567``|``-Python-2.7.11``|``intel/2016a``
``3.10.0``    |``-Python-2.7.11``|``intel/2016.02-GCC-4.9``
``3.11.0``    |``-Python-2.7.12``|``intel/2016b``
``3.13.0``    |``-Python-2.7.12``|``foss/2016b``
``3.13.0``    |``-Python-2.7.13``|``intel/2017a``
``3.15.0``    |``-Python-2.7.12``|``foss/2016b``
``3.15.0``    |``-Python-2.7.14``|``intel/2017b``
``3.15.0``    |``-Python-3.5.2`` |``foss/2016b``
``3.15.0``    |``-Python-3.6.2`` |``foss/2017b``
``3.15.0``    |``-Python-3.6.3`` |``foss/2017b``
``3.16.2``    |``-Python-3.6.4`` |``iomkl/2018.02``, ``iomkl/2018a``
``3.16.2``    |``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``, ``iomkl/2018b``
``3.17.0``    |``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``, ``iomkl/2018b``
``3.17.0``    |``-Python-3.7.2`` |``foss/2019a``, ``intel/2019a``
``3.18.0``    |``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``3.18.0``    |``-Python-3.7.2`` |``foss/2019a``, ``intel/2019a``
``3.19.0``    |``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``3.19.0``    |``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``
``3.19.0``    |``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``3.20.1``    |                  |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``
``3.20.1``    |``-Python-3.7.4`` |``foss/2019b``, ``intel/2019b``
``3.20.1``    |``-Python-3.8.2`` |``intel/2020a``
``3.21.1``    |                  |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``
``3.21.1``    |``-Python-3.8.2`` |``foss/2020a``
``3.22.0``    |                  |``foss/2020b``, ``foss/2021a``, ``fosscuda/2020b``, ``intel/2020b``, ``intel/2021a``
``3.22.1``    |                  |``foss/2021b``, ``foss/2022a``, ``intel/2021b``, ``intel/2022a``

<a anchor="ashs"/>
### *ASHS*

Automatic Segmentation of Hippocampal Subfields (ASHS) 

*homepage*: https://sites.google.com/site/hipposubfields/home

version            |toolchain
-------------------|----------
``rev103_20140612``|``system``

<a anchor="aspera-cli"/>
### *Aspera-CLI*

IBM Aspera Command-Line Interface (the Aspera CLI) is a collection of Aspera tools for performing high-speed, secure data transfers from the command line. The Aspera CLI is for users and organizations who want to automate their transfer workflows.

*homepage*: https://asperasoft.com

version  |versionsuffix    |toolchain
---------|-----------------|----------
``3.7.2``|``.354.010c3b8`` |``system``
``3.9.0``|``.1326.6985b21``|``system``
``3.9.6``|``.1467.159c5b1``|``system``

<a anchor="aspera-connect"/>
### *Aspera-Connect*

Connect is an install-on-demand Web browser plug-in that facilitates high-speed uploads and  downloads with an Aspera transfer server.

*homepage*: http://downloads.asperasoft.com/connect2/

version  |toolchain
---------|----------
``3.6.1``|``system``
``3.9.6``|``system``

<a anchor="assimp"/>
### *assimp*

Open Asset Import Library (assimp) is a library to import and export various  3d-model-formats including scene-post-processing to generate missing render data. 

*homepage*: https://github.com/assimp/assimp

version  |toolchain
---------|-----------------
``5.0.1``|``GCCcore/8.3.0``

<a anchor="assimulo"/>
### *Assimulo*

Assimulo is a simulation package for solving ordinary differential equations.

*homepage*: https://jmodelica.org/assimulo/

version|versionsuffix     |toolchain
-------|------------------|---------------
``2.9``|``-Python-2.7.15``|``intel/2018b``

<a anchor="astrid"/>
### *ASTRID*

ASTRID-2 is a method for estimating species trees from gene trees.

*homepage*: https://github.com/pranjalv123/ASTRID

version  |toolchain
---------|---------------
``2.2.1``|``gompi/2019a``

<a anchor="astropy"/>
### *astropy*

The Astropy Project is a community effort to develop a common  core package for Astronomy in Python and foster an ecosystem of interoperable  astronomy packages.  The Astropy community is committed to supporting diversity and inclusion.

*homepage*: https://www.astropy.org/

version   |versionsuffix     |toolchain
----------|------------------|-----------------------------------------------
``2.0.12``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``2.0.14``|                  |``foss/2019a``
``4.0.1`` |``-Python-3.7.4`` |``foss/2019b``
``4.0.1`` |``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``4.2.1`` |                  |``intelcuda/2020b``
``4.3.1`` |                  |``foss/2021a``, ``foss/2021b``, ``intel/2021a``
``5.0.4`` |                  |``foss/2021a``
``5.1.1`` |                  |``foss/2022a``

<a anchor="at-spi2-atk"/>
### *at-spi2-atk*

AT-SPI 2 toolkit bridge

*homepage*: https://wiki.gnome.org/Accessibility

version   |toolchain
----------|--------------------------------------------------------------------------------------------------
``2.26.3``|``fosscuda/2018b``
``2.32.0``|``GCCcore/8.2.0``
``2.34.1``|``GCCcore/8.3.0``
``2.34.2``|``GCCcore/9.3.0``
``2.38.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``

<a anchor="at-spi2-core"/>
### *at-spi2-core*

Assistive Technology Service Provider Interface. 

*homepage*: https://wiki.gnome.org/Accessibility

version   |toolchain
----------|------------------
``2.26.3``|``fosscuda/2018b``
``2.32.0``|``GCCcore/8.2.0``
``2.34.0``|``GCCcore/8.3.0``
``2.36.0``|``GCCcore/9.3.0``
``2.38.0``|``GCCcore/10.2.0``
``2.40.2``|``GCCcore/10.3.0``
``2.40.3``|``GCCcore/11.2.0``
``2.44.1``|``GCCcore/11.3.0``
``2.46.0``|``GCCcore/12.2.0``

<a anchor="atk"/>
### *ATK*

ATK provides the set of accessibility interfaces that are implemented by other  toolkits and applications. Using the ATK interfaces, accessibility tools have  full access to view and control running applications. 

*homepage*: https://developer.gnome.org/atk/

version   |toolchain
----------|-----------------------------------------------------------------------------
``2.18.0``|``intel/2016a``
``2.20.0``|``foss/2016a``, ``intel/2016a``
``2.22.0``|``foss/2016b``, ``intel/2016b``
``2.26.0``|``intel/2017a``
``2.26.1``|``foss/2018b``, ``intel/2017b``
``2.27.1``|``foss/2017b``, ``intel/2017b``
``2.28.1``|``foss/2018a``, ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018a``
``2.32.0``|``GCCcore/8.2.0``
``2.34.1``|``GCCcore/8.3.0``
``2.36.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``
``2.38.0``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``

<a anchor="atlas"/>
### *ATLAS*

ATLAS (Automatically Tuned Linear Algebra Software) is the application of  the AEOS (Automated Empirical Optimization of Software) paradigm, with the present emphasis   on the Basic Linear Algebra Subprograms (BLAS), a widely used, performance-critical, linear   algebra kernel library.

*homepage*: http://math-atlas.sourceforge.net

version   |versionsuffix    |toolchain
----------|-----------------|------------------
``3.10.2``|``-LAPACK-3.6.1``|``GCC/5.4.0-2.26``

<a anchor="atomate"/>
### *atomate*

atomate has implementations of FireWorks workflows for Materials Science.

*homepage*: https://pythonhosted.org/atomate/

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.4.4``|``-Python-2.7.13``|``intel/2017a``

<a anchor="atompaw"/>
### *AtomPAW*

AtomPAW is a Projector-Augmented Wave Dataset Generator that  can be used both as a standalone program and a library.

*homepage*: http://users.wfu.edu/natalie/papers/pwpaw/man.html

version    |toolchain
-----------|---------------
``4.1.0.5``|``intel/2018b``
``4.1.0.6``|``intel/2018b``

<a anchor="atools"/>
### *atools*

Tools to make using job arrays a lot more convenient.

*homepage*: https://github.com/gjbex/atools

version  |versionsuffix     |toolchain
---------|------------------|--------------------------------------
``1.4.2``|``-Python-2.7.12``|``intel/2016b``
``1.4.6``|``-Python-2.7.16``|``GCCcore/8.3.0``
``1.4.8``|``-Python-2.7.18``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.5.1``|                  |``GCCcore/11.2.0``

<a anchor="atropos"/>
### *atropos*

Atropos is tool for specific, sensitive, and speedy trimming of NGS reads. 

*homepage*: https://atropos.readthedocs.io

version   |versionsuffix    |toolchain
----------|-----------------|---------------
``1.1.21``|``-Python-3.6.6``|``intel/2018b``

<a anchor="atsas"/>
### *ATSAS*

ATSAS is a program suite for small-angle scattering data analysis from biological macromolecules.

*homepage*: http://www.embl-hamburg.de/ExternalInfo/Research/Sax/software.html

version    |versionsuffix  |toolchain
-----------|---------------|----------
``2.5.1-1``|``.el6.x86_64``|``system``
``2.5.1-1``|``.sl5.x86_64``|``system``
``2.7.1-1``|``.el7.x86_64``|``system``

<a anchor="attr"/>
### *attr*

Commands for Manipulating Filesystem Extended Attributes

*homepage*: https://savannah.nongnu.org/projects/attr

version   |toolchain
----------|--------------------------------------
``2.4.47``|``GCCcore/8.2.0``
``2.4.48``|``GCCcore/9.3.0``
``2.5.1`` |``GCCcore/10.3.0``, ``GCCcore/11.2.0``

<a anchor="attrdict"/>
### *attrdict*

AttrDict is a Python library that provides mapping objects that allow their elements  to be accessed both as keys and as attributes.

*homepage*: https://github.com/bcj/AttrDict

version  |versionsuffix    |toolchain
---------|-----------------|-----------------
``2.0.1``|``-Python-3.7.4``|``GCCcore/8.3.0``

<a anchor="augur"/>
### *augur*

Pipeline components for real-time phylodynamic analysis

*homepage*: https://github.com/nextstrain/augur

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``7.0.2``|``-Python-3.7.4``|``intel/2019b``

<a anchor="augustus"/>
### *AUGUSTUS*

AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences

*homepage*: https://bioinf.uni-greifswald.de/augustus/

version  |versionsuffix     |toolchain
---------|------------------|--------------------------------------------------------------
``3.2.3``|``-Python-2.7.13``|``intel/2017a``
``3.3``  |                  |``foss/2018a``
``3.3.2``|                  |``intel/2019a``
``3.3.2``|``-Python-2.7.13``|``intel/2017a``
``3.3.2``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``3.3.2``|``-Python-2.7.15``|``intel/2018b``
``3.3.3``|                  |``foss/2019b``, ``intel/2019b``
``3.4.0``|                  |``foss/2020a``, ``foss/2020b``, ``foss/2021a``, ``foss/2021b``

<a anchor="austin"/>
### *Austin*

Austin is a Python frame stack sampler for CPython written in pure C.

*homepage*: https://github.com/P403n1x87/austin

version  |toolchain
---------|------------------------------
``3.2.0``|``GCCcore/11.2.0``, ``system``

<a anchor="auto-07p"/>
### *AUTO-07p*

AUTO is a publicly available software for continuation and bifurcation problems in ordinary differential equations originally written in 1980 and widely used in the dynamical systems community.

*homepage*: https://github.com/auto-07p

version  |toolchain
---------|--------------
``0.9.3``|``foss/2021a``

<a anchor="autoconf"/>
### *Autoconf*

Autoconf is an extensible package of M4 macros that produce shell scripts  to automatically configure software source code packages. These scripts can adapt the  packages to many kinds of UNIX-like systems without manual user intervention. Autoconf  creates a configuration script for a package from a template file that lists the  operating system features that the package can use, in the form of M4 macro calls.

*homepage*: https://www.gnu.org/software/autoconf/

version |toolchain
--------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.68``|``foss/2016b``
``2.69``|``GCC/4.8.2``, ``GCC/4.8.4``, ``GCC/4.9.2``, ``GCC/4.9.3``, ``GCC/4.9.3-2.25``, ``GCC/5.2.0``, ``GCC/5.4.0-2.26``, ``GCCcore/10.2.0``, ``GCCcore/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.1.0``, ``GCCcore/6.2.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.2.0``, ``GCCcore/9.3.0``, ``GNU/4.9.2-2.25``, ``GNU/4.9.3-2.25``, ``GNU/5.1.0-2.25``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
``2.71``|``FCC/4.5.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``system``

<a anchor="autoconf-archive"/>
### *Autoconf-archive*

The GNU Autoconf Archive is a collection of more than 500 macros for GNU Autoconf that have been contributed as free software by friendly supporters of the cause from all over the Internet. Every single one of those macros can be re-used without imposing any restrictions whatsoever on the licensing of the generated configure script. In particular, it is possible to use all those macros in configure scripts that are meant for non-free software. This policy is unusual for a Free Software Foundation project. The FSF firmly believes that software ought to be free, and software licenses like the GPL are specifically designed to ensure that derivative work based on free software must be free as well. In case of Autoconf, however, an exception has been made, because Autoconf is at such a pivotal position in the software development tool chain that the benefits from having this tool available as widely as possible outweigh the disadvantage that some authors may choose to use it, too, for proprietary software. 

*homepage*: https://www.gnu.org/software/autoconf-archive

version       |toolchain
--------------|--------------------
``2019.01.06``|``GCC/8.2.0-2.31.1``
``2021.02.19``|``GCCcore/10.2.0``

<a anchor="autodock"/>
### *AutoDock*

AutoDock is a suite of automated docking tools. It is designed to  predict how small molecules, such as substrates or drug candidates, bind to  a receptor of known 3D structure.

*homepage*: http://autodock.scripps.edu/

version    |toolchain
-----------|--------------
``4.2.5.1``|``GCC/5.2.0``
``4.2.6``  |``GCC/10.3.0``

<a anchor="autodock-gpu"/>
### *AutoDock-GPU*

OpenCL and Cuda accelerated version of AutoDock. It leverages its embarrasingly parallelizable LGA by processing ligand-receptor poses in parallel over multiple compute units. AutoDock is a suite of automated docking tools. It is designed to predict how small molecules, such as substrates or drug candidates, bind to a receptor of known 3D structure.

*homepage*: https://github.com/ccsb-scripps/AutoDock-GPU

version  |versionsuffix   |toolchain
---------|----------------|--------------
``1.5.3``|``-CUDA-11.3.1``|``GCC/10.3.0``

<a anchor="autodock-vina"/>
### *AutoDock-Vina*

AutoDock Vina is an open-source program for doing molecular docking.

*homepage*: https://vina.scripps.edu/

version  |versionsuffix |toolchain
---------|--------------|------------------------------
``1.1.2``|``-linux_x86``|``system``
``1.2.3``|              |``foss/2021a``, ``foss/2021b``

<a anchor="autogenes"/>
### *AutoGeneS*

AutoGeneS automatically extracts informative genes and reveals the cellular heterogeneity of bulk RNA samples.

*homepage*: https://github.com/theislab/AutoGeneS

version  |toolchain
---------|--------------
``1.0.4``|``foss/2020b``

<a anchor="autogrid"/>
### *AutoGrid*

AutoDock is a suite of automated docking tools. It is designed to   predict how small molecules, such as substrates or drug candidates, bind to   a receptor of known 3D structure.

*homepage*: http://autodock.scripps.edu/

version    |toolchain
-----------|-------------
``4.2.5.1``|``GCC/5.2.0``

<a anchor="automake"/>
### *Automake*

Automake: GNU Standards-compliant Makefile generator

*homepage*: https://www.gnu.org/software/automake/automake.html

version   |toolchain
----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.11.3``|``foss/2016b``
``1.14``  |``GCC/4.8.2``, ``intel/2016a``
``1.14.1``|``GCC/4.8.2``
``1.15``  |``GCC/4.8.4``, ``GCC/4.9.2``, ``GCC/4.9.3``, ``GCC/4.9.3-2.25``, ``GCC/5.2.0``, ``GCC/5.4.0-2.26``, ``GCCcore/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.1.0``, ``GCCcore/6.2.0``, ``GCCcore/6.3.0``, ``GNU/4.9.2-2.25``, ``GNU/4.9.3-2.25``, ``GNU/5.1.0-2.25``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
``1.15.1``|``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.3.0``
``1.16.1``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.2.0``, ``GCCcore/9.3.0``
``1.16.2``|``GCCcore/10.2.0``
``1.16.3``|``FCC/4.5.0``, ``GCCcore/10.3.0``
``1.16.4``|``GCCcore/11.2.0``
``1.16.5``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``system``

<a anchor="automap"/>
### *AutoMap*

Tool to find regions of homozygosity (ROHs) from sequencing data.

*homepage*: https://github.com/mquinodo/AutoMap

version|versionsuffix|toolchain
-------|-------------|--------------
``1.0``|``-20200324``|``foss/2019b``

<a anchor="autopep8"/>
### *autopep8*

A tool that automatically formats Python code to conform to the PEP 8 style guide.

*homepage*: https://github.com/hhatto/autopep8

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``1.4.4``|``-Python-3.6.4``|``intel/2018a``

<a anchor="autotools"/>
### *Autotools*

This bundle collect the standard GNU build tools: Autoconf, Automake  and libtool 

*homepage*: https://autotools.io

version     |toolchain
------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``20150119``|``GCC/4.9.2``
``20150215``|``GCC/4.8.4``, ``GCC/4.9.2``, ``GCC/4.9.3``, ``GCC/4.9.3-2.25``, ``GCC/5.2.0``, ``GCC/5.4.0-2.26``, ``GCCcore/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.1.0``, ``GCCcore/6.2.0``, ``GCCcore/6.3.0``, ``GNU/4.9.2-2.25``, ``GNU/4.9.3-2.25``, ``GNU/5.1.0-2.25``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``gimkl/2017a``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
``20170619``|``GCCcore/6.4.0``, ``GCCcore/7.2.0``
``20180311``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.2.0``, ``GCCcore/9.3.0``
``20200321``|``GCCcore/10.2.0``
``20210128``|``FCC/4.5.0``, ``GCCcore/10.3.0``
``20210726``|``GCCcore/11.2.0``
``20220317``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``

<a anchor="avogadro2"/>
### *Avogadro2*

Avogadro is an advanced molecule editor and visualizer designed for cross-platform      use in computational chemistry, molecular modeling, bioinformatics, materials science,      and related areas. 

*homepage*: https://two.avogadro.cc/index.html

version   |versionsuffix    |toolchain
----------|-----------------|----------
``1.97.0``|``-linux-x86_64``|``system``

<a anchor="awscli"/>
### *awscli*

Universal Command Line Environment for AWS

*homepage*: https://pypi.python.org/pypi/awscli

version     |versionsuffix     |toolchain
------------|------------------|-----------------
``1.11.1``  |``-Python-2.7.12``|``intel/2016b``
``1.11.56`` |``-Python-2.7.12``|``foss/2016b``
``1.16.290``|``-Python-3.6.6`` |``foss/2018b``
``1.17.7``  |``-Python-3.7.4`` |``GCCcore/8.3.0``
``1.18.89`` |``-Python-3.8.2`` |``GCCcore/9.3.0``
``2.0.55``  |``-Python-3.8.2`` |``GCCcore/9.3.0``

<a anchor="b"/>
### *B*


<a href="#babelstream">BabelStream</a> - <a href="#babl">babl</a> - <a href="#bader">Bader</a> - <a href="#bagel">BAGEL</a> - <a href="#bali-phy">BAli-Phy</a> - <a href="#bam-readcount">bam-readcount</a> - <a href="#bambi">Bambi</a> - <a href="#bamfilters">bamFilters</a> - <a href="#bamm">BAMM</a> - <a href="#bamm">BamM</a> - <a href="#bamsurgeon">BAMSurgeon</a> - <a href="#bamtofastq">bamtofastq</a> - <a href="#bamtools">BamTools</a> - <a href="#bamutil">BamUtil</a> - <a href="#bandage">Bandage</a> - <a href="#barrnap">barrnap</a> - <a href="#basemap">basemap</a> - <a href="#bash">Bash</a> - <a href="#bashplotlib">bashplotlib</a> - <a href="#bat">bat</a> - <a href="#batmeth2">BatMeth2</a> - <a href="#bayesass">BayesAss</a> - <a href="#bayescan">BayeScan</a> - <a href="#bayescenv">BayeScEnv</a> - <a href="#bayestraits">BayesTraits</a> - <a href="#bazel">Bazel</a> - <a href="#bbcp">bbcp</a> - <a href="#bbftp">bbFTP</a> - <a href="#bbmap">BBMap</a> - <a href="#bc">bc</a> - <a href="#bcalm">BCALM</a> - <a href="#bcbio-gff">bcbio-gff</a> - <a href="#bcel">BCEL</a> - <a href="#bcftools">BCFtools</a> - <a href="#bcgtree">bcgTree</a> - <a href="#bcl-convert">bcl-convert</a> - <a href="#bcl2fastq2">bcl2fastq2</a> - <a href="#bcolz">bcolz</a> - <a href="#bdbag">BDBag</a> - <a href="#beagle">Beagle</a> - <a href="#beagle-lib">beagle-lib</a> - <a href="#beast">Beast</a> - <a href="#beautifulsoup">BeautifulSoup</a> - <a href="#bedops">BEDOPS</a> - <a href="#bedtools">BEDTools</a> - <a href="#beef">BEEF</a> - <a href="#behave">behave</a> - <a href="#berkeleygw">BerkeleyGW</a> - <a href="#bfast">BFAST</a> - <a href="#bfc">BFC</a> - <a href="#bgen">bgen</a> - <a href="#bgen-reader">bgen-reader</a> - <a href="#bibtexparser">bibtexparser</a> - <a href="#big-scape">BiG-SCAPE</a> - <a href="#bigdft">BigDFT</a> - <a href="#binsanity">BinSanity</a> - <a href="#binutils">binutils</a> - <a href="#bio-db-hts">Bio-DB-HTS</a> - <a href="#bio-eutilities">Bio-EUtilities</a> - <a href="#bio-samtools">Bio-SamTools</a> - <a href="#bio-searchio-hmmer">Bio-SearchIO-hmmer</a> - <a href="#bioawk">bioawk</a> - <a href="#biobambam2">biobambam2</a> - <a href="#biogeme">biogeme</a> - <a href="#biom-format">biom-format</a> - <a href="#biomart-perl">biomart-perl</a> - <a href="#bioperl">BioPerl</a> - <a href="#biopp">BioPP</a> - <a href="#biopython">Biopython</a> - <a href="#bioservices">BioServices</a> - <a href="#birdnet">BirdNET</a> - <a href="#biscuit">biscuit</a> - <a href="#bismark">Bismark</a> - <a href="#bison">Bison</a> - <a href="#bitarray">bitarray</a> - <a href="#blacs">BLACS</a> - <a href="#blasr">BLASR</a> - <a href="#blasr_libcpp">blasr_libcpp</a> - <a href="#blast">BLAST</a> - <a href="#blast+">BLAST+</a> - <a href="#blat">BLAT</a> - <a href="#blender">Blender</a> - <a href="#blis">BLIS</a> - <a href="#blitz++">Blitz++</a> - <a href="#blobtools">BlobTools</a> - <a href="#blosc">Blosc</a> - <a href="#blosc2">Blosc2</a> - <a href="#blt">BLT</a> - <a href="#bmtagger">bmtagger</a> - <a href="#bnpy">bnpy</a> - <a href="#boinc">BOINC</a> - <a href="#bokeh">bokeh</a> - <a href="#boltztrap">BoltzTraP</a> - <a href="#bonito">Bonito</a> - <a href="#bonmin">Bonmin</a> - <a href="#bonnie++">Bonnie++</a> - <a href="#boost">Boost</a> - <a href="#boost.mpi">Boost.MPI</a> - <a href="#boost.python">Boost.Python</a> - <a href="#boost_histogram">boost_histogram</a> - <a href="#boto3">boto3</a> - <a href="#bottleneck">Bottleneck</a> - <a href="#bowtie">Bowtie</a> - <a href="#bowtie2">Bowtie2</a> - <a href="#bpipe">Bpipe</a> - <a href="#bpp">bpp</a> - <a href="#bpytop">bpytop</a> - <a href="#bracken">Bracken</a> - <a href="#braker">BRAKER</a> - <a href="#breakdancer">BreakDancer</a> - <a href="#breseq">breseq</a> - <a href="#brotli">Brotli</a> - <a href="#brotli-python">Brotli-python</a> - <a href="#brunsli">Brunsli</a> - <a href="#bsddb3">bsddb3</a> - <a href="#bsmapz">BSMAPz</a> - <a href="#bsoft">Bsoft</a> - <a href="#bsseeker2">BSseeker2</a> - <a href="#buddy">BuDDy</a> - <a href="#bufrlib">BUFRLIB</a> - <a href="#buildenv">buildenv</a> - <a href="#buildingspy">buildingspy</a> - <a href="#bullet">Bullet</a> - <a href="#busco">BUSCO</a> - <a href="#bustools">BUStools</a> - <a href="#bwa">BWA</a> - <a href="#bwa-meth">bwa-meth</a> - <a href="#bwakit">bwakit</a> - <a href="#bwidget">bwidget</a> - <a href="#bwise">BWISE</a> - <a href="#bx-python">bx-python</a> - <a href="#bxh_xcede_tools">BXH_XCEDE_TOOLS</a> - <a href="#byacc">byacc</a> - <a href="#byobu">byobu</a> - <a href="#bzip2">bzip2</a>


<a anchor="babelstream"/>
### *BabelStream*

STREAM, for lots of devices written in many programming models

*homepage*: https://uob-hpc.github.io/BabelStream

version|versionsuffix|toolchain
-------|-------------|--------------
``3.4``|``-omp``     |``GCC/11.2.0``

<a anchor="babl"/>
### *babl*

babl is pixel encoding and color space conversion engine in C

*homepage*: https://www.gegl.org/babl

version   |toolchain
----------|------------------
``0.1.86``|``GCCcore/10.3.0``

<a anchor="bader"/>
### *Bader*

A fast algorithm for doing Bader's analysis on a charge density grid.

*homepage*: http://theory.cm.utexas.edu/henkelman/code/bader/

version |toolchain
--------|---------------------------------------
``1.02``|``intel/2018a``
``1.03``|``intel/2018b``
``1.04``|``GCC/11.2.0``, ``iccifort/2020.4.304``

<a anchor="bagel"/>
### *BAGEL*

BAGEL (Brilliantly Advanced General Electronic-structure Library) is a parallel electronic-structure program.

*homepage*: http://www.nubakery.org

version  |toolchain
---------|---------------
``1.1.1``|``intel/2016b``

<a anchor="bali-phy"/>
### *BAli-Phy*

BAli-Phy estimates multiple sequence alignments and evolutionary trees from DNA, amino acid,  or codon sequences.

*homepage*: http://www.bali-phy.org/

version  |toolchain
---------|----------
``3.6.0``|``system``

<a anchor="bam-readcount"/>
### *bam-readcount*

Count DNA sequence reads in BAM files

*homepage*: https://github.com/genome/bam-readcount

version  |toolchain
---------|---------------------------------------------
``0.8.0``|``GCC/11.2.0``, ``GCC/9.3.0``, ``foss/2018b``

<a anchor="bambi"/>
### *Bambi*

Bambi is a high-level Bayesian model-building interface written in Python. It works with the probabilistic programming frameworks PyMC3 and is designed to make it extremely easy to fit Bayesian mixed-effects models common in biology, social sciences and other disciplines.

*homepage*: https://bambinos.github.io/bambi

version  |toolchain
---------|-------------------------------
``0.7.1``|``foss/2021b``, ``intel/2021b``

<a anchor="bamfilters"/>
### *bamFilters*

A utility tool to filter alignments from bam files,  using identity percent, low complexity and read coverage.

*homepage*: https://github.com/institut-de-genomique/bamFilters

version       |toolchain
--------------|--------------
``2022-06-30``|``GCC/11.3.0``

<a anchor="bamm"/>
### *BAMM*

BAMM is oriented entirely towards detecting and quantifying heterogeneity in evolutionary rates.  It uses reversible jump Markov chain Monte Carlo to automatically explore a vast universe of candidate models of  lineage diversification and trait evolution. 

*homepage*: http://bamm-project.org/

version  |toolchain
---------|--------------
``2.5.0``|``foss/2016b``

<a anchor="bamm"/>
### *BamM*

BamM is a c library, wrapped in python, that parses BAM files 

*homepage*: http://ecogenomics.github.io/BamM/

version  |versionsuffix     |toolchain
---------|------------------|--------------
``1.7.3``|``-Python-2.7.12``|``foss/2016b``
``1.7.3``|``-Python-2.7.14``|``foss/2018a``

<a anchor="bamsurgeon"/>
### *BAMSurgeon*

Tools for adding mutations to existing .bam files, used for testing mutation callers

*homepage*: https://github.com/adamewing/bamsurgeon

version|versionsuffix     |toolchain
-------|------------------|-------------
``1.2``|``-Python-2.7.16``|``GCC/8.3.0``

<a anchor="bamtofastq"/>
### *bamtofastq*

Convert 10x BAM files to the original FASTQs compatible with 10x pipelines.

*homepage*: https://github.com/10XGenomics/bamtofastq

version  |toolchain
---------|------------------
``1.4.0``|``GCCcore/10.3.0``

<a anchor="bamtools"/>
### *BamTools*

BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files.

*homepage*: https://github.com/pezmaster31/bamtools

version  |toolchain
---------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.4.0``|``foss/2016b``
``2.4.1``|``intel/2017a``
``2.5.0``|``foss/2016b``, ``intel/2017b``
``2.5.1``|``GCC/10.2.0``, ``GCC/8.2.0-2.31.1``, ``GCC/8.3.0``, ``GCC/9.3.0``, ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``iccifort/2019.5.281``, ``iccifort/2020.4.304``, ``intel/2017b``, ``intel/2018b``
``2.5.2``|``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/11.3.0``

<a anchor="bamutil"/>
### *BamUtil*

BamUtil is a repository that contains several programs   that perform operations on SAM/BAM files. All of these programs   are built into a single executable, bam.

*homepage*: http://genome.sph.umich.edu/wiki/BamUtil

version   |toolchain
----------|---------------
``1.0.13``|``intel/2016b``
``1.0.14``|``intel/2018a``

<a anchor="bandage"/>
### *Bandage*

Bandage is a program for visualising de novo assembly graphs

*homepage*: http://rrwick.github.io/Bandage

version  |versionsuffix|toolchain
---------|-------------|------------------
``0.8.1``|``_Centos``  |``system``
``0.8.1``|``_Ubuntu``  |``system``
``0.9.0``|             |``GCCcore/11.2.0``

<a anchor="barrnap"/>
### *barrnap*

Barrnap (BAsic Rapid Ribosomal RNA Predictor) predicts the location of ribosomal RNA genes in genomes.

*homepage*: https://github.com/tseemann/barrnap

version|toolchain
-------|----------------------------------------------------------------------
``0.9``|``GCC/8.2.0-2.31.1``, ``foss/2018b``, ``gompi/2020b``, ``gompi/2021b``

<a anchor="basemap"/>
### *basemap*

The matplotlib basemap toolkit is a library for plotting  2D data on maps in Python

*homepage*: https://matplotlib.org/basemap/

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.0.7``|``-Python-2.7.13``|``intel/2017a``
``1.0.7``|``-Python-3.6.3`` |``intel/2017b``
``1.0.7``|``-Python-3.6.4`` |``intel/2018a``
``1.2.0``|``-Python-3.6.6`` |``intel/2018b``
``1.2.2``|``-Python-3.8.2`` |``foss/2020a``
``1.3.6``|                  |``foss/2022a``

<a anchor="bash"/>
### *Bash*

Bash is an sh-compatible command language interpreter that executes commands    read from the standard input or from a file.  Bash also incorporates useful features from the    Korn and C shells (ksh and csh).

*homepage*: http://www.gnu.org/software/bash

version|toolchain
-------|-------------
``4.3``|``GCC/4.9.2``

<a anchor="bashplotlib"/>
### *bashplotlib*

bashplotlib is a python package and command line tool for making basic plots in the terminal.  It's a quick way to visualize data when you don't have a GUI.

*homepage*: https://github.com/glamp/bashplotlib

version  |toolchain
---------|------------------
``0.6.5``|``GCCcore/10.3.0``

<a anchor="bat"/>
### *bat*

The BAT Python package supports the processing and analysis of Bro data with Pandas, scikit-learn, and Spark

*homepage*: https://pypi.python.org/pypi/bat

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.3.3``|``-Python-3.6.3``|``intel/2017b``

<a anchor="batmeth2"/>
### *BatMeth2*

An Integrated Package for Bisulfite DNA Methylation Data Analysis with Indel-sensitive Mapping.

*homepage*: https://github.com/GuoliangLi-HZAU/BatMeth2

version|toolchain
-------|--------------
``2.1``|``foss/2019b``

<a anchor="bayesass"/>
### *BayesAss*

BayesAss: Bayesian Inference of Recent Migration Using Multilocus Genotypes

*homepage*: http://www.rannala.org/?page_id=245

version  |toolchain
---------|--------------
``3.0.4``|``foss/2016a``

<a anchor="bayescan"/>
### *BayeScan*

BayeScan aims at identifying candidate loci under natural selection from genetic data,  using differences in allele frequencies between populations.

*homepage*: http://cmpg.unibe.ch/software/BayeScan/

version|toolchain
-------|-----------------------------------------------
``2.1``|``foss/2016a``, ``foss/2018a``, ``intel/2018a``

<a anchor="bayescenv"/>
### *BayeScEnv*

BayeScEnv is a Fst-based, genome-scan method that uses environmental variables to detect local adaptation.

*homepage*: https://github.com/devillemereuil/bayescenv

version|toolchain
-------|------------------------------------------------------
``1.1``|``GCC/8.3.0``, ``foss/2016a``, ``iccifort/2019.5.281``

<a anchor="bayestraits"/>
### *BayesTraits*

BayesTraits is a computer package for performing analyses of trait   evolution among groups of species for which a phylogeny or sample of phylogenies is   available. This new package incoporates our earlier and separate programes Multistate,   Discrete and Continuous. BayesTraits can be applied to the analysis of traits that adopt   a finite number of discrete states, or to the analysis of continuously varying traits.   Hypotheses can be tested about models of evolution, about ancestral states and about   correlations among pairs of traits. 

*homepage*: http://www.evolution.reading.ac.uk/BayesTraitsV1.html

version        |versionsuffix    |toolchain
---------------|-----------------|----------
``1.0-linux32``|                 |``system``
``2.0``        |``-Beta-Linux64``|``system``

<a anchor="bazel"/>
### *Bazel*

Bazel is a build tool that builds code quickly and reliably. It is used to build the majority of Google's software.

*homepage*: https://bazel.io/

version   |versionsuffix|toolchain
----------|-------------|-----------------------------------------------------------------------------
``0.4.4`` |             |``system``
``0.7.0`` |             |``GCCcore/6.4.0``
``0.10.0``|             |``GCCcore/6.4.0``
``0.11.0``|             |``GCCcore/6.4.0``
``0.11.1``|             |``GCCcore/6.4.0``
``0.12.0``|             |``GCCcore/6.4.0``
``0.16.0``|             |``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``0.18.0``|             |``GCCcore/7.3.0``
``0.20.0``|             |``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``0.25.2``|             |``GCCcore/8.2.0``
``0.26.1``|             |``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``0.29.1``|             |``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``0.29.1``|``-Java-1.8``|``GCCcore/9.3.0``
``1.1.0`` |             |``GCCcore/8.3.0``
``2.0.0`` |             |``GCCcore/10.2.0``, ``GCCcore/8.3.0``
``3.4.1`` |             |``GCCcore/8.3.0``
``3.6.0`` |             |``GCCcore/9.3.0``
``3.7.1`` |             |``GCCcore/8.3.0``
``3.7.2`` |             |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/8.3.0``
``4.2.2`` |             |``GCCcore/11.2.0``
``5.1.1`` |             |``GCCcore/11.3.0``

<a anchor="bbcp"/>
### *bbcp*

BBCP is an alternative to Gridftp when transferring large amounts of data,  capable of breaking up your transfer into multiple simultaneous transferring streams,  thereby transferring data much faster than single-streaming utilities such as SCP and SFTP.  See details at http://pcbunn.cithep.caltech.edu/bbcp/using_bbcp.htm  or http://www.nics.tennessee.edu/computing-resources/data-transfer/bbcp

*homepage*: http://www.slac.stanford.edu/~abh/bbcp/

version          |versionsuffix     |toolchain
-----------------|------------------|----------
``12.01.30.00.0``|``-amd64_linux26``|``system``

<a anchor="bbftp"/>
### *bbFTP*

bbFTP is a file transfer software. It implements its own transfer protocol,   which is optimized for large files (larger than 2GB) and secure as it does not read the   password in a file and encrypts the connection information. bbFTP main features are:   * Encoded username and password at connection * SSH and Certificate authentication modules   * Multi-stream transfer * Big windows as defined in RFC1323 * On-the-fly data compression   * Automatic retry * Customizable time-outs * Transfer simulation   * AFS authentication integration * RFIO interface

*homepage*: https://software.in2p3.fr/bbftp/

version  |toolchain
---------|----------------------------------------------
``3.2.1``|``GCCcore/9.3.0``, ``intel/2016a``, ``system``

<a anchor="bbmap"/>
### *BBMap*

BBMap short read aligner, and other bioinformatic tools.

*homepage*: https://sourceforge.net/projects/bbmap/

version   |versionsuffix      |toolchain
----------|-------------------|-------------------------------
``36.62`` |``-Java-1.8.0_112``|``intel/2016b``
``37.93`` |                   |``foss/2018a``, ``intel/2018a``
``38.26`` |                   |``foss/2018b``
``38.50b``|                   |``GCC/8.2.0-2.31.1``
``38.76`` |                   |``GCC/8.2.0-2.31.1``
``38.79`` |                   |``GCC/8.3.0``
``38.87`` |                   |``iccifort/2020.1.217``
``38.90`` |                   |``GCC/10.2.0``, ``GCC/9.3.0``
``38.96`` |                   |``GCC/10.3.0``
``38.98`` |                   |``GCC/11.2.0``
``39.01`` |                   |``GCC/11.3.0``

<a anchor="bc"/>
### *bc*

bc is an arbitrary precision numeric processing language.

*homepage*: https://www.gnu.org/software/bc/

version    |toolchain
-----------|-------------
``1.06.95``|``GCC/4.8.2``

<a anchor="bcalm"/>
### *BCALM*

de Bruijn graph compaction in low memory

*homepage*: https://github.com/GATB/bcalm

version  |toolchain
---------|--------------
``2.2.0``|``foss/2018a``

<a anchor="bcbio-gff"/>
### *bcbio-gff*

Read and write Generic Feature Format (GFF) with Biopython integration. 

*homepage*: https://github.com/chapmanb/bcbb/tree/master/gff

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.6.6``|``-Python-3.8.2``|``foss/2020a``
``0.6.7``|                 |``foss/2021a``

<a anchor="bcel"/>
### *BCEL*

The Byte Code Engineering Library (Apache Commons BCEL™) is intended to give  users a convenient way to analyze, create, and manipulate (binary) Java class  files (those ending with .class). 

*homepage*: https://commons.apache.org/proper/commons-bcel/

version  |versionsuffix|toolchain
---------|-------------|----------
``5.2``  |``-Java-1.8``|``system``
``6.5.0``|``-Java-1.8``|``system``

<a anchor="bcftools"/>
### *BCFtools*

Samtools is a suite of programs for interacting with high-throughput sequencing data.  BCFtools - Reading/writing BCF2/VCF/gVCF files and calling/filtering/summarising SNP and short indel sequence  variants

*homepage*: http://www.htslib.org/

version   |toolchain
----------|-----------------------------------------------------------------------------------------
``1.3``   |``foss/2016a``, ``intel/2016a``
``1.3.1`` |``foss/2016b``
``1.6``   |``foss/2016b``, ``foss/2017b``, ``intel/2017b``
``1.8``   |``GCC/6.4.0-2.28``
``1.9``   |``foss/2018a``, ``foss/2018b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2018b``
``1.10.2``|``GCC/8.3.0``, ``GCC/9.3.0``, ``iccifort/2019.5.281``
``1.11``  |``GCC/10.2.0``
``1.12``  |``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/9.3.0``
``1.14``  |``GCC/11.2.0``
``1.15.1``|``GCC/11.3.0``

<a anchor="bcgtree"/>
### *bcgTree*

Automatized phylogenetic tree building from bacterial core genomes.

*homepage*: https://github.com/molbiodiv/bcgTree

version   |versionsuffix   |toolchain
----------|----------------|---------------
``1.0.10``|``-Perl-5.26.1``|``intel/2018a``
``1.1.0`` |``-Perl-5.28.0``|``intel/2018b``

<a anchor="bcl-convert"/>
### *bcl-convert*

The Illumina BCL Convert v4.0 is a standalone local software app that converts the  Binary Base Call (BCL) files produced by Illumina sequencing systems to FASTQ files.

*homepage*: https://support.illumina.com/sequencing/sequencing_software/bcl-convert.html

version    |versionsuffix |toolchain
-----------|--------------|----------
``4.0.3-2``|``el7.x86_64``|``system``

<a anchor="bcl2fastq2"/>
### *bcl2fastq2*

bcl2fastq Conversion Software both demultiplexes data and converts BCL files generated by  Illumina sequencing systems to standard FASTQ file formats for downstream analysis.

*homepage*: https://support.illumina.com/sequencing/sequencing_software/bcl2fastq-conversion-software.html

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------------------------------------------------------------------------------------
``2.19.1``|``-Python-2.7.12``|``foss/2016b``
``2.20.0``|                  |``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/8.3.0``, ``GCC/9.3.0``, ``foss/2018b``, ``intel/2019a``
``2.20.0``|``-Python-2.7.12``|``foss/2016b``
``2.20.0``|``-Python-2.7.14``|``intel/2017b``

<a anchor="bcolz"/>
### *bcolz*

bcolz provides columnar, chunked data containers that can be compressed either in-memory and on-disk.  Column storage allows for efficiently querying tables, as well as for cheap column addition and removal.  It is based on NumPy, and uses it as the standard data container to communicate with bcolz objects,  but it also comes with support for import/export facilities to/from HDF5/PyTables tables and pandas dataframes.

*homepage*: https://bcolz.blosc.org/en/latest/

version  |versionsuffix     |toolchain
---------|------------------|--------------
``1.1.1``|``-Python-2.7.13``|``foss/2017a``
``1.2.1``|                  |``foss/2020b``
``1.2.1``|``-Python-3.8.2`` |``foss/2020a``

<a anchor="bdbag"/>
### *BDBag*

The bdbag utilities are a collection of software programs for working with BagIt packages that conform to the Bagit and Bagit/RO profiles.

*homepage*: https://github.com/fair-research/bdbag

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.4.1``|``-Python-2.7.14``|``intel/2018a``
``1.4.1``|``-Python-3.6.4`` |``intel/2018a``
``1.6.3``|                  |``intel/2021b``

<a anchor="beagle"/>
### *Beagle*

Beagle is a software package for phasing genotypes and for imputing  ungenotyped markers.

*homepage*: https://faculty.washington.edu/browning/beagle/beagle.html

version            |versionsuffix|toolchain
-------------------|-------------|----------
``5.4.22Jul22.46e``|``-Java-11`` |``system``

<a anchor="beagle-lib"/>
### *beagle-lib*

beagle-lib is a high-performance library that can perform the core calculations at the heart of most  Bayesian and Maximum Likelihood phylogenetics packages.

*homepage*: https://github.com/beagle-dev/beagle-lib

version  |versionsuffix   |toolchain
---------|----------------|--------------------------------------------------------------------------------------------------------------------------------
``2.1.2``|                |``foss/2016a``, ``foss/2017a``
``3.0.1``|                |``foss/2018a``, ``intel/2018a``
``3.0.2``|                |``foss/2018b``
``3.0.2``|``-CUDA-9.2.88``|``foss/2018b``
``3.1.2``|                |``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/8.2.0-2.31.1``, ``GCC/9.3.0``, ``gcccuda/2019b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``

<a anchor="beast"/>
### *Beast*

BEAST is a cross-platform program for Bayesian MCMC analysis of molecular   sequences. It is entirely orientated towards rooted, time-measured phylogenies inferred using   strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies   but is also a framework for testing evolutionary hypotheses without conditioning on a single   tree topology. BEAST uses MCMC to average over tree space, so that each tree is weighted   proportional to its posterior probability. 

*homepage*: http://beast2.org

version   |toolchain
----------|------------------------------------
``1.8.4`` |``system``
``1.10.1``|``intel/2018a``
``1.10.4``|``GCC/10.2.0``, ``GCC/8.2.0-2.31.1``
``2.4.0`` |``foss/2016a``
``2.4.7`` |``foss/2017a``
``2.5.0`` |``foss/2018a``
``2.5.1`` |``foss/2018b``
``2.5.2`` |``GCC/8.2.0-2.31.1``
``2.6.3`` |``gcccuda/2019b``
``2.6.4`` |``GCC/10.2.0``
``2.6.7`` |``GCC/10.3.0``

<a anchor="beautifulsoup"/>
### *BeautifulSoup*

Beautiful Soup is a Python library designed for quick turnaround projects like screen-scraping.

*homepage*: https://www.crummy.com/software/BeautifulSoup

version   |versionsuffix    |toolchain
----------|-----------------|----------------------------------------------------------
``4.6.0`` |``-Python-3.6.3``|``intel/2017b``
``4.6.3`` |``-Python-3.6.4``|``intel/2018a``
``4.7.1`` |``-Python-3.6.6``|``intel/2018b``
``4.8.0`` |                 |``GCCcore/8.2.0``
``4.9.1`` |                 |``GCCcore/8.3.0``
``4.9.1`` |``-Python-3.8.2``|``GCCcore/9.3.0``
``4.9.3`` |                 |``GCCcore/10.2.0``
``4.10.0``|                 |``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="bedops"/>
### *BEDOPS*

BEDOPS is an open-source command-line toolkit that performs highly efficient and  scalable Boolean and other set operations, statistical calculations, archiving, conversion and  other management of genomic data of arbitrary scale. Tasks can be easily split by chromosome for  distributing whole-genome analyses across a computational cluster.

*homepage*: http://bedops.readthedocs.io/en/latest/index.html

version   |toolchain
----------|-------------------------------
``2.4.1`` |``GCC/4.8.4``
``2.4.2`` |``GCC/4.8.2``
``2.4.20``|``system``
``2.4.26``|``system``
``2.4.30``|``foss/2016b``
``2.4.32``|``foss/2018a``, ``intel/2018a``
``2.4.35``|``foss/2018b``
``2.4.41``|``foss/2021b``

<a anchor="bedtools"/>
### *BEDTools*

BEDTools: a powerful toolset for genome arithmetic. The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps and computing coverage. The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF, and SAM/BAM.

*homepage*: https://bedtools.readthedocs.io/

version   |toolchain
----------|---------------------------------------------------------------------------------------
``2.25.0``|``foss/2016a``
``2.26.0``|``GCCcore/6.4.0``, ``foss/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``
``2.27.1``|``GCCcore/6.4.0``, ``foss/2016b``, ``foss/2018b``, ``intel/2017a``, ``intel/2018a``
``2.28.0``|``GCC/8.2.0-2.31.1``, ``foss/2018b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``2.29.2``|``GCC/8.3.0``, ``GCC/9.3.0``
``2.30.0``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/11.3.0``, ``iccifort/2020.4.304``

<a anchor="beef"/>
### *BEEF*

BEEF is a library-based implementation of the Bayesian Error Estimation Functional, suitable for linking against by Fortran- or C-based DFT codes. A description of BEEF can be found at http://dx.doi.org/10.1103/PhysRevB.85.235149.

*homepage*: https://confluence.slac.stanford.edu/display/SUNCAT/BEEF+Functional+Software

version  |toolchain
---------|------------------------------------------------
``0.1.1``|``iccifort/2019.5.281``, ``iccifort/2020.4.304``

<a anchor="behave"/>
### *behave*

behave: Behavior-driven development (or BDD) is an agile software development technique that encourages collaboration between developers, QA and non-technical or business participants in a software project.

*homepage*: http://pythonhosted.org/behave

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.2.5``|``-Python-2.7.12``|``foss/2016b``
``1.2.6``|``-Python-3.6.4`` |``intel/2018a``

<a anchor="berkeleygw"/>
### *BerkeleyGW*

The BerkeleyGW Package is a set of computer codes that calculates the quasiparticle  properties and the optical responses of a large variety of materials from bulk periodic crystals to  nanostructures such as slabs, wires and molecules.

*homepage*: https://www.berkeleygw.org

version      |versionsuffix    |toolchain
-------------|-----------------|----------------------------------------------------------------
``1.0.6``    |                 |``intel/2016.02-GCC-4.9``
``1.1-beta2``|                 |``intel/2016.02-GCC-4.9``
``1.2.0``    |                 |``intel/2017a``, ``intel/2018a``
``2.0.0``    |                 |``foss/2017b``, ``foss/2018b``, ``intel/2017b``, ``intel/2018a``
``2.1.0``    |``-Python-3.7.4``|``foss/2019b``, ``intel/2019b``
``3.0.1``    |                 |``foss/2020b``, ``foss/2021a``

<a anchor="bfast"/>
### *BFAST*

BFAST facilitates the fast and accurate mapping of short reads to reference sequences.  Some advantages of BFAST include:  1) Speed: enables billions of short reads to be mapped quickly.   2) Accuracy: A priori probabilities for mapping reads with defined set of variants.   3) An easy way to measurably tune accuracy at the expense of speed.

*homepage*: http://bfast.sourceforge.net/

version   |toolchain
----------|--------------
``0.7.0a``|``foss/2016b``

<a anchor="bfc"/>
### *BFC*

BFC is a standalone high-performance tool for correcting sequencing errors from Illumina sequencing data. It is specifically designed for high-coverage whole-genome human data, though also performs well for small genomes.

*homepage*: https://github.com/lh3/bfc

version|toolchain
-------|--------------
``1``  |``foss/2018a``

<a anchor="bgen"/>
### *bgen*

A BGEN file format reader. It fully supports the BGEN format specifications 1.2 and 1.3.

*homepage*: https://github.com/limix/bgen

version  |toolchain
---------|------------------
``3.0.2``|``GCCcore/7.3.0``
``3.0.3``|``GCCcore/9.3.0``
``4.1.3``|``GCCcore/10.2.0``

<a anchor="bgen-reader"/>
### *bgen-reader*

A bgen file format reader. This python package is a wrapper around the bgen library, a low-memory footprint reader that efficiently reads bgen files. It fully supports the bgen format specifications: 1.2 and 1.3; as well as their optional compressed formats.

*homepage*: https://github.com/limix/bgen-reader-py

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``3.0.2``|``-Python-3.6.6``|``foss/2018b``

<a anchor="bibtexparser"/>
### *bibtexparser*

Bibtex parser in Python 2.7 and 3.x

*homepage*: https://github.com/sciunto-org/python-bibtexparser

version  |toolchain
---------|-----------------
``1.1.0``|``GCCcore/8.2.0``

<a anchor="big-scape"/>
### *BiG-SCAPE*

BiG-SCAPE and CORASON provide a set of tools to explore the diversity of biosynthetic gene clusters  (BGCs) across large numbers of genomes, by constructing BGC sequence similarity networks, grouping BGCs into gene  cluster families, and exploring gene cluster diversity linked to enzyme phylogenies.

*homepage*: https://bigscape-corason.secondarymetabolites.org/index.html

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``1.0.1``|``-Python-3.7.4``|``intel/2019b``

<a anchor="bigdft"/>
### *BigDFT*

BigDFT: electronic structure calculation based on Daubechies wavelets. bigdft-suite is a set of different packages to run bigdft.

*homepage*: https://gitlab.com/l_sim/bigdft-suite

version  |toolchain
---------|--------------
``1.9.1``|``foss/2021b``

<a anchor="binsanity"/>
### *BinSanity*

BinSanity contains a suite a scripts designed to cluster contigs generated from metagenomic assembly into putative genomes.

*homepage*: https://github.com/edgraham/BinSanity/wiki

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.3.5``|``-Python-3.7.4``|``foss/2019b``

<a anchor="binutils"/>
### *binutils*

binutils: GNU binary utilities

*homepage*: https://directory.fsf.org/project/binutils/

version   |toolchain
----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.25``  |``GCC/4.9.2``, ``GCC/4.9.2-binutils-2.25``, ``GCC/4.9.3``, ``GCC/4.9.3-binutils-2.25``, ``GCC/5.1.0-binutils-2.25``, ``GCCcore/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/4.9.4``, ``system``
``2.25.1``|``system``
``2.26``  |``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/5.5.0``, ``GCCcore/6.3.0``, ``system``
``2.27``  |``GCCcore/6.1.0``, ``GCCcore/6.2.0``, ``GCCcore/6.3.0``, ``system``
``2.28``  |``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.1.0``, ``system``
``2.29``  |``GCCcore/7.2.0``, ``GCCcore/system``, ``system``
``2.30``  |``GCCcore/7.3.0``, ``GCCcore/8.1.0``, ``system``
``2.31.1``|``GCCcore/7.4.0``, ``GCCcore/8.2.0``, ``system``
``2.32``  |``GCCcore/8.3.0``, ``GCCcore/9.1.0``, ``GCCcore/9.2.0``, ``system``
``2.34``  |``GCCcore/10.1.0``, ``GCCcore/9.3.0``, ``system``
``2.35``  |``GCCcore/10.2.0``, ``system``
``2.36.1``|``FCC/4.5.0``, ``GCCcore/10.3.0``, ``GCCcore/11.1.0``, ``GCCcore/8.4.0``, ``GCCcore/9.4.0``, ``system``
``2.37``  |``GCCcore/11.2.0``, ``system``
``2.38``  |``GCCcore/11.3.0``, ``GCCcore/12.1.0``, ``GCCcore/9.5.0``, ``system``
``2.39``  |``GCCcore/12.2.0``, ``system``

<a anchor="bio-db-hts"/>
### *Bio-DB-HTS*

Read files using HTSlib including BAM/CRAM, Tabix and BCF database files

*homepage*: https://metacpan.org/release/Bio-DB-HTS

version |versionsuffix   |toolchain
--------|----------------|----------------------------------------------
``2.11``|``-Perl-5.26.0``|``foss/2017b``, ``intel/2017b``
``2.11``|``-Perl-5.26.1``|``intel/2018a``
``2.11``|``-Perl-5.28.0``|``foss/2018b``
``3.01``|                |``GCC/10.2.0``, ``GCC/11.2.0``, ``GCC/11.3.0``
``3.01``|``-Perl-5.28.1``|``GCC/8.2.0-2.31.1``

<a anchor="bio-eutilities"/>
### *Bio-EUtilities*

BioPerl low-level API for retrieving and storing data from NCBI eUtils

*homepage*: https://github.com/bioperl/bio-eutilities

version |toolchain
--------|-----------------
``1.76``|``GCCcore/8.3.0``

<a anchor="bio-samtools"/>
### *Bio-SamTools*

This is a Perl interface to the SAMtools sequence alignment interface.

*homepage*: http://search.cpan.org/~lds/Bio-SamTools/

version |versionsuffix   |toolchain
--------|----------------|---------------
``1.43``|``-Perl-5.24.1``|``intel/2017a``

<a anchor="bio-searchio-hmmer"/>
### *Bio-SearchIO-hmmer*

Code to parse output from hmmsearch, hmmscan, phmmer and nhmmer, compatible with both version 2 and version 3 of the HMMER package from http://hmmer.org.

*homepage*: https://metacpan.org/pod/Bio::SearchIO::hmmer3

version  |toolchain
---------|------------------------------
``1.7.3``|``GCC/10.2.0``, ``GCC/10.3.0``

<a anchor="bioawk"/>
### *bioawk*

Bioawk is an extension to Brian Kernighan's awk,  adding the support of several common biological data formats,  including optionally gzip'ed BED, GFF, SAM, VCF, FASTA/Q and TAB-delimited formats with column names. 

*homepage*: https://github.com/lh3/bioawk

version|toolchain
-------|----------------------------------------------
``1.0``|``GCC/10.3.0``, ``GCC/11.2.0``, ``foss/2018b``

<a anchor="biobambam2"/>
### *biobambam2*

Tools for processing BAM files

*homepage*: https://github.com/gt1/biobambam2

version   |toolchain
----------|---------------
``2.0.87``|``intel/2018a``

<a anchor="biogeme"/>
### *biogeme*

Biogeme is a open source Python package designed for the maximum likelihood estimation of parametric models in general, with a special emphasis on discrete choice models.

*homepage*: https://pypi.python.org/pypi/biogeme

version   |toolchain
----------|--------------
``3.1.2`` |``foss/2021a``
``3.2.6`` |``foss/2022a``
``3.2.8`` |``foss/2021a``
``3.2.10``|``foss/2022a``

<a anchor="biom-format"/>
### *biom-format*

The BIOM file format (canonically pronounced biome) is designed to be  a general-use format for representing biological sample by observation  contingency tables. BIOM is a recognized standard for the Earth Microbiome  Project and is a Genomics Standards Consortium supported project. 

*homepage*: https://biom-format.org

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``2.1.10``|``-Python-3.8.2``|``foss/2020a``
``2.1.12``|                 |``foss/2021b``

<a anchor="biomart-perl"/>
### *biomart-perl*

The BioMart Perl API allows you to go a step further with BioMart   and integrate BioMart Perl Code into custom Perl scripts.

*homepage*: https://useast.ensembl.org/info/data/biomart/biomart_perl_api.html

version        |versionsuffix   |toolchain
---------------|----------------|-----------------
``0.7_e6db561``|``-Perl-5.26.0``|``GCCcore/6.4.0``

<a anchor="bioperl"/>
### *BioPerl*

Bioperl is the product of a community effort to produce Perl code which is useful in biology.  Examples include Sequence objects, Alignment objects and database searching objects.

*homepage*: https://bioperl.org/

version    |versionsuffix   |toolchain
-----------|----------------|------------------------------------------------------------------------------
``1.6.924``|``-Perl-5.20.3``|``intel/2016a``
``1.6.924``|``-Perl-5.22.1``|``foss/2016a``
``1.7.0``  |``-Perl-5.24.0``|``foss/2016b``
``1.7.1``  |``-Perl-5.24.0``|``intel/2016b``
``1.7.1``  |``-Perl-5.24.1``|``intel/2017a``
``1.7.2``  |                |``GCCcore/8.3.0``
``1.7.2``  |``-Perl-5.26.0``|``foss/2017b``, ``intel/2017b``
``1.7.2``  |``-Perl-5.26.1``|``intel/2018a``
``1.7.2``  |``-Perl-5.28.0``|``foss/2018b``, ``intel/2018b``
``1.7.2``  |``-Perl-5.28.1``|``GCCcore/8.2.0``
``1.7.7``  |                |``GCCcore/9.3.0``
``1.7.8``  |                |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="biopp"/>
### *BioPP*

Bio++ is a set of C++ libraries for Bioinformatics, including sequence analysis, phylogenetics, molecular evolution and population genetics. Bio++ is Object Oriented and is designed to be both easy to use and computer efficient. Bio++ intends to help programmers to write computer expensive programs, by providing them a set of re-usable tools. 

*homepage*: https://github.com/BioPP/bpp-core

version  |toolchain
---------|-----------------------------------
``2.4.1``|``GCC/8.2.0-2.31.1``, ``GCC/9.3.0``

<a anchor="biopython"/>
### *Biopython*

Biopython is a set of freely available tools for biological  computation written in Python by an international team of developers. It is  a distributed collaborative effort to develop Python libraries and  applications which address the needs of current and future work in  bioinformatics. 

*homepage*: https://www.biopython.org

version |versionsuffix     |toolchain
--------|------------------|---------------------------------------------------------------
``1.65``|``-Python-2.7.11``|``foss/2016a``
``1.68``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``1.68``|``-Python-3.5.2`` |``foss/2016b``, ``intel/2016b``
``1.70``|``-Python-2.7.13``|``foss/2017a``
``1.70``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``1.70``|``-Python-3.6.1`` |``intel/2017a``
``1.70``|``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``1.71``|``-Python-2.7.14``|``intel/2018a``
``1.71``|``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``1.72``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``1.72``|``-Python-3.6.6`` |``foss/2018b``
``1.73``|                  |``foss/2019a``, ``fosscuda/2019a``, ``intel/2019a``
``1.73``|``-Python-3.6.6`` |``foss/2018b``
``1.74``|                  |``foss/2019a``
``1.75``|``-Python-2.7.16``|``foss/2019b``
``1.75``|``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``
``1.76``|``-Python-2.7.18``|``foss/2020b``
``1.78``|                  |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``
``1.78``|``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``1.79``|                  |``foss/2021a``, ``foss/2021b``, ``foss/2022a``, ``intel/2021b``

<a anchor="bioservices"/>
### *BioServices*

Bioservices is a Python package that provides access to many Bioinformatices Web Services (e.g.,  UniProt) and a framework to easily implement Web Services wrappers (based on WSDL/SOAP or REST protocols).

*homepage*: https://bioservices.readthedocs.io/

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.7.9``|``-Python-3.8.2``|``foss/2020a``

<a anchor="birdnet"/>
### *BirdNET*

BirdNET is a research platform that aims at recognizing birds by sound at scale. We support various hardware and operating systems such as Arduino microcontrollers, the Raspberry Pi,  smartphones, web browsers, workstation PCs, and even cloud services. BirdNET is a citizen science  platform as well as an analysis software for extremely large collections of audio. BirdNET aims to provide innovative tools for conservationists, biologists, and birders alike.

*homepage*: https://birdnet.cornell.edu/

version     |versionsuffix    |toolchain
------------|-----------------|------------------
``20201214``|``-Python-3.7.4``|``fosscuda/2019b``

<a anchor="biscuit"/>
### *biscuit*

Utilities to help analyze bisulfite-treated sequence data  

*homepage*: https://github.com/zwdzwd/biscuit

version  |toolchain
---------|--------------
``0.1.4``|``foss/2016a``

<a anchor="bismark"/>
### *Bismark*

A tool to map bisulfite converted sequence reads and determine cytosine methylation states

*homepage*: https://www.bioinformatics.babraham.ac.uk/projects/bismark/

version   |toolchain
----------|-------------------------------
``0.19.0``|``intel/2017b``
``0.20.1``|``foss/2018b``, ``intel/2018b``
``0.23.1``|``foss/2021b``

<a anchor="bison"/>
### *Bison*

Bison is a general-purpose parser generator that converts an annotated  context-free grammar into a deterministic LR or generalized LR (GLR) parser  employing LALR(1) parser tables. 

*homepage*: https://www.gnu.org/software/bison

version  |toolchain
---------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.7``  |``GCC/4.8.1``, ``GCC/4.8.4``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``system``
``3.0.2``|``GCC/4.8.2``
``3.0.3``|``GCC/4.9.2``
``3.0.4``|``GCC/4.9.2``, ``GCC/4.9.2-binutils-2.25``, ``GCC/4.9.3``, ``GCC/4.9.3-2.25``, ``GCC/4.9.3-binutils-2.25``, ``GCC/5.1.0-binutils-2.25``, ``GCCcore/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/4.9.4``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/5.5.0``, ``GCCcore/6.1.0``, ``GCCcore/6.2.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.1.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.1.0``, ``GCCcore/system``, ``GNU/4.9.3-2.25``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``gimkl/2017a``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
``3.0.5``|``GCCcore/5.5.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.1.0``, ``GCCcore/8.2.0``, ``GCCcore/system``, ``system``
``3.2.2``|``GCCcore/7.4.0``
``3.3.2``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/8.4.0``, ``GCCcore/9.1.0``, ``GCCcore/9.2.0``, ``system``
``3.5.2``|``system``
``3.5.3``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``, ``system``
``3.6.1``|``GCCcore/10.1.0``
``3.7.1``|``GCCcore/10.2.0``, ``system``
``3.7.6``|``FCC/4.5.0``, ``GCCcore/10.3.0``, ``GCCcore/11.1.0``, ``GCCcore/11.2.0``, ``GCCcore/9.4.0``, ``system``
``3.8.2``|``GCCcore/11.3.0``, ``GCCcore/12.1.0``, ``GCCcore/12.2.0``, ``GCCcore/9.5.0``, ``system``

<a anchor="bitarray"/>
### *bitarray*

bitarray provides an object type which efficiently represents an array of booleans

*homepage*: https://github.com/ilanschnell/bitarray

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.8.3``|``-Python-2.7.15``|``intel/2018b``
``0.8.3``|``-Python-3.6.6`` |``intel/2018b``
``1.2.1``|``-Python-3.7.4`` |``foss/2019b``
``1.5.3``|``-Python-2.7.16``|``GCC/8.3.0``

<a anchor="blacs"/>
### *BLACS*

The BLACS (Basic Linear Algebra Communication Subprograms) project is  an ongoing investigation whose purpose is to create a linear algebra oriented message passing interface  that may be implemented efficiently and uniformly across a large range of distributed memory platforms.

*homepage*: http://www.netlib.org/blacs/

version|toolchain
-------|-------------------
``1.1``|``gmvapich2/2016a``

<a anchor="blasr"/>
### *BLASR*

The PacBio® long read aligner

*homepage*: https://github.com/PacificBiosciences/blasr

version     |toolchain
------------|---------------
``2.2``     |``intel/2016b``
``5.3.3``   |``gompi/2019a``
``20170330``|``intel/2017a``

<a anchor="blasr_libcpp"/>
### *blasr_libcpp*

Blasr_libcpp is a library used by blasr and other executables such as samtoh5, loadPulses for  analyzing PacBio sequences

*homepage*: https://github.com/PacificBiosciences/blasr_libcpp

version     |toolchain
------------|---------------
``20170426``|``intel/2017a``

<a anchor="blast"/>
### *BLAST*

Basic Local Alignment Search Tool, or BLAST, is an algorithm for comparing primary biological  sequence information, such as the amino-acid sequences of different proteins or the nucleotides of DNA sequences.

*homepage*: http://blast.ncbi.nlm.nih.gov/

version   |versionsuffix    |toolchain
----------|-----------------|----------
``2.2.26``|``-Linux_x86_64``|``system``
``2.8.1`` |``-Linux_x86_64``|``system``
``2.10.0``|``-Linux_x86_64``|``system``
``2.10.1``|``-Linux_x86_64``|``system``
``2.11.0``|``-Linux_x86_64``|``system``

<a anchor="blast+"/>
### *BLAST+*

Basic Local Alignment Search Tool, or BLAST, is an algorithm  for comparing primary biological sequence information, such as the amino-acid  sequences of different proteins or the nucleotides of DNA sequences.

*homepage*: https://blast.ncbi.nlm.nih.gov/

version   |versionsuffix     |toolchain
----------|------------------|-----------------------------------------------------------------------------------
``2.2.31``|                  |``system``
``2.3.0`` |``-Python-2.7.11``|``foss/2016a``
``2.6.0`` |``-Python-2.7.12``|``foss/2016b``
``2.6.0`` |``-Python-2.7.13``|``foss/2017a``, ``intel/2017a``
``2.6.0`` |``-Python-2.7.14``|``intel/2017b``
``2.7.1`` |                  |``foss/2018a``, ``foss/2018b``, ``intel/2018a``, ``intel/2018b``
``2.7.1`` |``-Python-2.7.14``|``intel/2017b``
``2.8.1`` |                  |``foss/2018b``
``2.9.0`` |                  |``gompi/2019a``, ``gompi/2019b``, ``gompi/2021b``, ``iimpi/2019a``, ``iimpi/2019b``
``2.10.1``|                  |``gompi/2020a``, ``iimpi/2020a``
``2.11.0``|                  |``gompi/2019b``, ``gompi/2020a``, ``gompi/2020b``, ``gompi/2021a``
``2.12.0``|                  |``gompi/2021b``
``2.13.0``|                  |``gompi/2022a``

<a anchor="blat"/>
### *BLAT*

BLAT on DNA is designed to quickly find sequences of 95% and greater similarity  of length 25 bases or more.

*homepage*: http://genome.ucsc.edu/FAQ/FAQblat.html

version|toolchain
-------|--------------------------------------------------------------------------------------------------------------------
``3.5``|``GCC/8.2.0-2.31.1``, ``GCC/8.3.0``, ``GCC/9.3.0``, ``foss/2016b``, ``foss/2018b``, ``intel/2016b``, ``intel/2017a``

<a anchor="blender"/>
### *Blender*

Blender is the free and open source 3D creation suite. It supports  the entirety of the 3D pipeline-modeling, rigging, animation, simulation, rendering,  compositing and motion tracking, even video editing and game creation.

*homepage*: https://www.blender.org/

version  |versionsuffix                |toolchain
---------|-----------------------------|-------------------------------
``2.77a``|``-Python-3.5.2``            |``intel/2016b``
``2.79`` |``-Python-3.6.1``            |``intel/2017a``
``2.79b``|``-Python-3.6.6``            |``intel/2018b``
``2.79b``|``-Python-3.6.6-CUDA-9.2.88``|``foss/2018b``
``2.81`` |``-Python-3.7.4``            |``foss/2019b``, ``intel/2019b``
``3.0.0``|``-linux-x64``               |``system``
``3.1.2``|``-linux-x64``               |``system``
``3.2.2``|``-linux-x64``               |``system``
``3.3.1``|``-linux-x86_64-CUDA-11.7.0``|``system``

<a anchor="blis"/>
### *BLIS*

AMD's fork of BLIS. BLIS is a portable software framework for instantiating high-performance BLAS-like dense linear algebra libraries.

*homepage*: https://developer.amd.com/amd-cpu-libraries/blas-library/

version  |versionsuffix|toolchain
---------|-------------|------------------------------------------------------------
``0.3.2``|             |``GCC/7.3.0-2.30``
``0.6.0``|             |``GCC/8.3.0-2.32``
``0.8.0``|             |``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``0.8.1``|             |``GCC/10.3.0``, ``GCC/11.2.0``, ``GCCcore/10.3.0``
``0.9.0``|             |``GCC/11.3.0``, ``GCC/12.2.0``, ``intel-compilers/2022.1.0``
``1.2``  |``-amd``     |``GCC/7.3.0-2.30``
``2.2``  |``-amd``     |``GCCcore/9.3.0``
``3.0``  |``-amd``     |``GCCcore/10.3.0``
``3.0.1``|``-amd``     |``GCC/11.2.0``, ``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``3.1``  |``-amd``     |``GCCcore/11.2.0``

<a anchor="blitz++"/>
### *Blitz++*

Blitz++ is a (LGPLv3+) licensed meta-template library for array manipulation  in C++ with a speed comparable to Fortran implementations, while preserving an  object-oriented interface 

*homepage*: https://github.com/blitzpp/blitz

version  |toolchain
---------|---------------------------------------------------------
``0.10`` |``GCCcore/6.4.0``, ``foss/2016a``
``1.0.2``|``GCCcore/10.2.0``, ``GCCcore/11.3.0``, ``GCCcore/9.3.0``

<a anchor="blobtools"/>
### *BlobTools*

A modular command-line solution for visualisation,  quality control and taxonomic partitioning of genome datasets. 

*homepage*: https://blobtools.readme.io/docs

version     |versionsuffix     |toolchain
------------|------------------|--------------
``20180528``|``-Python-2.7.15``|``foss/2018b``

<a anchor="blosc"/>
### *Blosc*

Blosc, an extremely fast, multi-threaded, meta-compressor library

*homepage*: https://www.blosc.org/

version   |toolchain
----------|------------------------------------------------------------------
``1.11.1``|``intel/2016b``
``1.12.1``|``GCCcore/6.4.0``, ``foss/2016b``, ``foss/2017a``, ``intel/2017a``
``1.14.2``|``GCCcore/6.4.0``, ``foss/2016a``
``1.14.4``|``GCCcore/7.3.0``
``1.17.0``|``GCCcore/8.2.0``
``1.17.1``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.21.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``1.21.1``|``GCCcore/11.2.0``
``1.21.3``|``GCCcore/11.3.0``

<a anchor="blosc2"/>
### *Blosc2*

Blosc, an extremely fast, multi-threaded, meta-compressor library

*homepage*: https://www.blosc.org/

version  |toolchain
---------|------------------
``2.0.3``|``GCCcore/10.2.0``
``2.0.4``|``GCCcore/10.3.0``
``2.4.3``|``GCCcore/11.3.0``

<a anchor="blt"/>
### *BLT*

BLT is an extension to the Tk toolkit, adding new widgets,  geometry managers, and miscellaneous commands.

*homepage*: http://blt.sourceforge.net/

version     |toolchain
------------|------------------
``20181223``|``GCCcore/11.2.0``

<a anchor="bmtagger"/>
### *bmtagger*

Best Match Tagger for removing human reads from metagenomics datasets

*homepage*: ftp://ftp.ncbi.nlm.nih.gov/pub/agarwala/bmtagger/

version  |toolchain
---------|------------------------------------------------
``3.101``|``foss/2018b``, ``gompi/2019a``, ``gompi/2020b``

<a anchor="bnpy"/>
### *bnpy*

Bayesian nonparametric machine learning for python provides  code for training popular clustering models on large datasets. The focus is  on Bayesian nonparametric models based on the Dirichlet process, but it also  provides parametric counterparts.

*homepage*: https://github.com/bnpy/bnpy

version  |versionsuffix     |toolchain
---------|------------------|--------------
``0.1.6``|``-Python-2.7.15``|``foss/2018b``

<a anchor="boinc"/>
### *BOINC*

BOINC is a program that lets you donate your idle computer time to science projects  like SETI@home, Climateprediction.net, Rosetta@home, World Community Grid, and many others.

*homepage*: https://boinc.berkeley.edu

version   |versionsuffix|toolchain
----------|-------------|-------------
``7.2.42``|``-client``  |``GCC/4.8.2``

<a anchor="bokeh"/>
### *bokeh*

Statistical and novel interactive HTML plots for Python

*homepage*: https://github.com/bokeh/bokeh

version    |versionsuffix     |toolchain
-----------|------------------|------------------------------------------------------------------------
``0.12.3`` |``-Python-2.7.12``|``intel/2016b``
``0.12.3`` |``-Python-3.5.2`` |``intel/2016b``
``0.12.15``|``-Python-3.6.4`` |``intel/2018a``
``1.0.4``  |``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``1.3.4``  |``-Python-3.7.2`` |``foss/2019a``
``1.4.0``  |``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``
``2.0.2``  |``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``2.2.3``  |                  |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``, ``intelcuda/2020b``
``2.4.1``  |                  |``foss/2021a``
``2.4.2``  |                  |``foss/2021b``
``2.4.3``  |                  |``foss/2022a``

<a anchor="boltztrap"/>
### *BoltzTraP*

Boltzmann Transport Properties (BoltzTraP) is a program for calculating the semi-classic transport  coefficients.

*homepage*: http://www.icams.de/content/departments/cmat/boltztrap/

version  |toolchain
---------|---------------
``1.2.5``|``intel/2016a``

<a anchor="bonito"/>
### *Bonito*

Convolution Basecaller for Oxford Nanopore Reads

*homepage*: https://github.com/nanoporetech/bonito

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------
``0.1.0``|``-Python-3.7.4``|``foss/2019b``, ``fosscuda/2019b``
``0.1.4``|``-Python-3.7.4``|``fosscuda/2019b``
``0.2.0``|``-Python-3.7.4``|``fosscuda/2019b``
``0.2.2``|``-Python-3.7.4``|``fosscuda/2019b``
``0.3.2``|``-Python-3.7.4``|``fosscuda/2019b``
``0.3.5``|``-Python-3.7.4``|``fosscuda/2019b``
``0.3.8``|                 |``fosscuda/2020b``
``0.4.0``|                 |``fosscuda/2020b``

<a anchor="bonmin"/>
### *Bonmin*

Ipopt (Interior Point OPTimizer, pronounced eye-pea-Opt) is a software package for  large-scale nonlinear optimization.

*homepage*: https://coin-or.github.io/Ipopt

version  |toolchain
---------|---------------
``1.8.7``|``intel/2019a``

<a anchor="bonnie++"/>
### *Bonnie++*

Enhanced performance Test of Filesystem I/O

*homepage*: https://www.coker.com.au/bonnie++

version  |toolchain
---------|--------------
``1.97`` |``foss/2016a``
``2.00a``|``GCC/10.3.0``

<a anchor="boost"/>
### *Boost*

Boost provides free peer-reviewed portable C++ source libraries.

*homepage*: https://www.boost.org/

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------------------------------------------
``1.54.0``|``-Python-2.7.12``|``foss/2016b``
``1.55.0``|                  |``system``
``1.55.0``|``-Python-2.7.11``|``foss/2016a``
``1.57.0``|``-Python-2.7.10``|``gimkl/2.11.5``
``1.58.0``|                  |``intel/2017a``
``1.58.0``|``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``1.58.0``|``-serial``       |``GCC/4.9.2``
``1.59.0``|``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``1.60.0``|                  |``foss/2016a``, ``intel/2016a``
``1.60.0``|``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``1.60.0``|``-Python-3.5.1`` |``foss/2016a``
``1.61.0``|                  |``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``1.61.0``|``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``1.61.0``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``1.61.0``|``-Python-3.5.1`` |``foss/2016a``
``1.62.0``|``-Python-2.7.12``|``intel/2016b``
``1.63.0``|                  |``foss/2017a``
``1.63.0``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``1.63.0``|``-Python-2.7.13``|``foss/2017a``, ``intel/2017a``
``1.63.0``|``-Python-2.7.14``|``foss/2018a``, ``intel/2017b``
``1.63.0``|``-Python-3.5.2`` |``foss/2016b``
``1.64.0``|                  |``gompi/2019a``, ``gompic/2019a``, ``intel/2017a``
``1.64.0``|``-Python-2.7.13``|``intel/2017a``
``1.65.0``|``-Python-2.7.13``|``intel/2017a``
``1.65.1``|                  |``foss/2017a``, ``foss/2017b``, ``intel/2017a``, ``intel/2017b``
``1.65.1``|``-Python-2.7.13``|``intel/2017a``
``1.65.1``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``1.65.1``|``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``1.66.0``|                  |``foss/2018a``, ``intel/2017b``, ``intel/2018.01``, ``intel/2018a``
``1.66.0``|``-Python-2.7.14``|``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``1.66.0``|``-Python-3.6.3`` |``intel/2018.01``
``1.66.0``|``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``1.66.0``|``-no_mpi``       |``GCCcore/6.4.0``
``1.67.0``|                  |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018a``, ``intel/2018b``
``1.67.0``|``-Python-2.7.14``|``foss/2018a``
``1.68.0``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``1.68.0``|``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``1.69.0``|                  |``intel/2019.01``
``1.70.0``|                  |``gompi/2019a``, ``gompic/2019a``, ``iimpi/2019a``, ``iimpic/2019a``
``1.71.0``|                  |``gompi/2019b``, ``gompic/2019b``, ``iimpi/2019b``, ``iimpic/2019b``
``1.72.0``|                  |``gompi/2020a``, ``gompic/2020a``, ``iimpi/2020a``
``1.72.0``|``-no_mpi``       |``GCCcore/9.3.0``
``1.74.0``|                  |``GCC/10.2.0``, ``iccifort/2020.4.304``
``1.75.0``|                  |``GCC/11.2.0``
``1.76.0``|                  |``GCC/10.3.0``, ``intel-compilers/2021.2.0``
``1.77.0``|                  |``GCC/11.2.0``, ``intel-compilers/2021.4.0``
``1.79.0``|                  |``GCC/11.2.0``, ``GCC/11.3.0``

<a anchor="boost.mpi"/>
### *Boost.MPI*

Boost provides free peer-reviewed portable C++ source libraries.

*homepage*: https://www.boost.org/

version   |toolchain
----------|---------------
``1.77.0``|``gompi/2021b``
``1.79.0``|``gompi/2022a``

<a anchor="boost.python"/>
### *Boost.Python*

Boost.Python is a C++ library which enables seamless interoperability between C++  and the Python programming language.

*homepage*: https://boostorg.github.io/python

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------------------------------------------
``1.64.0``|                  |``gompi/2019a``, ``gompic/2019a``
``1.65.1``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``1.66.0``|``-Python-2.7.14``|``intel/2018a``
``1.66.0``|``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``1.67.0``|``-Python-2.7.15``|``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``1.67.0``|``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``1.70.0``|                  |``gompi/2019a``, ``gompic/2019a``, ``iimpi/2019a``, ``iimpic/2019a``
``1.71.0``|                  |``gompi/2019b``, ``gompic/2019b``, ``iimpi/2019b``, ``iimpic/2019b``
``1.72.0``|                  |``gompi/2020a``, ``gompic/2020a``, ``iimpi/2020a``
``1.74.0``|                  |``GCC/10.2.0``
``1.76.0``|                  |``GCC/10.3.0``
``1.77.0``|                  |``GCC/11.2.0``
``1.79.0``|                  |``GCC/11.3.0``

<a anchor="boost_histogram"/>
### *boost_histogram*

Boost-histogram is a Python package providing Python bindings for Boost.Histogram.

*homepage*: https://boost-histogram.readthedocs.io

version  |toolchain
---------|--------------
``1.2.1``|``foss/2021a``

<a anchor="boto3"/>
### *boto3*

Boto3 is the Amazon Web Services (AWS) Software Development Kit (SDK) for Python, which allows Python developers to write software that makes use of services like Amazon S3 and Amazon EC2.

*homepage*: https://github.com/boto/boto3

version    |toolchain
-----------|--------------------------------------
``1.20.13``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.26.37``|``GCCcore/11.3.0``

<a anchor="bottleneck"/>
### *Bottleneck*

Fast NumPy array functions written in C

*homepage*: https://kwgoodman.github.io/bottleneck-doc

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``1.2.1``|``-Python-3.6.4``|``intel/2018a``
``1.3.2``|``-Python-3.8.2``|``foss/2020a``

<a anchor="bowtie"/>
### *Bowtie*

Bowtie is an ultrafast, memory-efficient short read aligner.  It aligns short DNA sequences (reads) to the human genome.

*homepage*: http://bowtie-bio.sourceforge.net/index.shtml

version    |toolchain
-----------|-------------------------------------------------------------------------------------------------------------------------------------
``1.1.2``  |``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``foss/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2018a``
``1.2.1.1``|``foss/2016b``, ``intel/2017b``
``1.2.2``  |``foss/2018b``, ``intel/2017b``, ``intel/2018a``
``1.2.3``  |``GCC/8.2.0-2.31.1``, ``GCC/8.3.0``, ``GCC/9.3.0``, ``foss/2018b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``iccifort/2019.5.281``
``1.3.0``  |``GCC/10.2.0``, ``GCC/9.3.0``
``1.3.1``  |``GCC/11.2.0``

<a anchor="bowtie2"/>
### *Bowtie2*

Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads  to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s  of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes.  Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome,  its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.

*homepage*: https://bowtie-bio.sourceforge.net/bowtie2/index.shtml

version    |toolchain
-----------|------------------------------------------------------------------------------------------------------
``2.2.8``  |``foss/2016a``
``2.2.9``  |``foss/2016a``, ``intel/2016b``
``2.3.2``  |``foss/2016b``, ``foss/2017a``, ``intel/2017a``
``2.3.3.1``|``intel/2017b``
``2.3.4``  |``intel/2017b``
``2.3.4.1``|``foss/2017b``, ``intel/2017b``, ``intel/2018a``
``2.3.4.2``|``foss/2018b``, ``intel/2018b``
``2.3.4.3``|``foss/2017b``
``2.3.5.1``|``GCC/8.2.0-2.31.1``, ``GCC/8.3.0``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``iccifort/2019.5.281``
``2.4.1``  |``GCC/9.3.0``
``2.4.2``  |``GCC/10.2.0``, ``GCC/9.3.0``
``2.4.4``  |``GCC/10.3.0``, ``GCC/11.2.0``
``2.4.5``  |``GCC/11.3.0``

<a anchor="bpipe"/>
### *Bpipe*

Bpipe - a tool for running and managing bioinformatics pipelines

*homepage*: http://docs.bpipe.org/

version    |toolchain
-----------|---------------
``0.9.9.2``|``intel/2017a``

<a anchor="bpp"/>
### *bpp*

The aim of this project is to implement a versatile high-performance version of the BPP software. 

*homepage*: https://github.com/bpp/bpp

version  |toolchain
---------|--------------
``4.3.8``|``GCC/8.3.0``
``4.4.0``|``GCC/10.3.0``

<a anchor="bpytop"/>
### *bpytop*

Resource monitor that shows usage and stats for processor, memory, disks, network and processes.

*homepage*: https://github.com/aristocratos/bpytop/

version   |toolchain
----------|------------------
``1.0.60``|``GCCcore/10.2.0``
``1.0.67``|``GCCcore/10.3.0``

<a anchor="bracken"/>
### *Bracken*

Bracken (Bayesian Reestimation of Abundance with KrakEN)  is a highly accurate statistical method that computes the abundance of  species in DNA sequences from a metagenomics sample. Braken uses the  taxonomy labels assigned by Kraken, a highly accurate metagenomics  classification algorithm, to estimate the number of reads originating  from each species present in a sample. Kraken classifies reads to the  best matching location in the taxonomic tree, but does not estimate  abundances of species. We use the Kraken database itself to derive  probabilities that describe how much sequence from each genome is  identical to other genomes in the database, and combine this information  with the assignments for a particular sample to estimate abundance at  the species level, the genus level, or above. Combined with the Kraken  classifier, Bracken produces accurate species- and genus-level abundance  estimates even when a sample contains two or more near-identical species.  NOTE: Bracken is compatible with both Kraken 1 and Kraken 2. However, the  default kmer length is different depending on the version of Kraken used.  If you use Kraken 1 defaults, specify 31 as the kmer length. If you use  Kraken 2 defaults, specify 35 as the kmer length.

*homepage*: https://ccb.jhu.edu/software/bracken/

version  |toolchain
---------|------------------
``2.6.0``|``GCCcore/9.3.0``
``2.6.2``|``GCCcore/11.2.0``
``2.7``  |``GCCcore/11.2.0``

<a anchor="braker"/>
### *BRAKER*

BRAKER is a pipeline for fully automated prediction of protein coding genes with GeneMark-ES/ET  and AUGUSTUS in novel eukaryotic genomes.

*homepage*: https://github.com/Gaius-Augustus/BRAKER

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``2.1.2``|                 |``intel/2019a``
``2.1.5``|``-Python-3.7.4``|``intel/2019b``
``2.1.6``|                 |``foss/2021b``

<a anchor="breakdancer"/>
### *BreakDancer*

BreakDancer is a Perl/C++ package that provides genome-wide detection of structural variants from  next generation paired-end sequencing reads

*homepage*: http://gmt.genome.wustl.edu/packages/breakdancer

version  |toolchain
---------|---------------
``1.4.5``|``intel/2017a``

<a anchor="breseq"/>
### *breseq*

breseq is a computational pipeline for the analysis of short-read re-sequencing data

*homepage*: https://barricklab.org/breseq

version   |versionsuffix|toolchain
----------|-------------|---------------
``0.35.0``|``-R-3.6.0`` |``intel/2019a``
``0.35.4``|``-R-4.0.0`` |``foss/2020a``
``0.36.1``|             |``foss/2021b``

<a anchor="brotli"/>
### *Brotli*

Brotli is a generic-purpose lossless compression algorithm that compresses data using a combination  of a modern variant of the LZ77 algorithm, Huffman coding and 2nd order context modeling, with a compression ratio  comparable to the best currently available general-purpose compression methods. It is similar in speed with deflate  but offers more dense compression. The specification of the Brotli Compressed Data Format is defined in RFC 7932.

*homepage*: https://github.com/google/brotli

version  |toolchain
---------|---------------------------------------------------------------------------------------------------------------------
``1.0.9``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/8.3.0``

<a anchor="brotli-python"/>
### *Brotli-python*

Brotli is a generic-purpose lossless compression algorithm that compresses data using a combination  of a modern variant of the LZ77 algorithm, Huffman coding and 2nd order context modeling, with a compression ratio  comparable to the best currently available general-purpose compression methods. It is similar in speed with deflate  but offers more dense compression. The specification of the Brotli Compressed Data Format is defined in RFC 7932.

*homepage*: https://github.com/google/brotli

version  |toolchain
---------|------------------
``1.0.9``|``GCCcore/10.3.0``

<a anchor="brunsli"/>
### *Brunsli*

Brunsli is a lossless JPEG repacking library.

*homepage*: https://github.com/google/brunsli/

version|toolchain
-------|--------------------------------------
``0.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``

<a anchor="bsddb3"/>
### *bsddb3*

bsddb3 is a nearly complete Python binding of the Oracle/Sleepycat C API for the Database Environment, Database, Cursor, Log Cursor, Sequence and Transaction objects.

*homepage*: https://pypi.org/project/bsddb3/

version  |versionsuffix     |toolchain
---------|------------------|--------------------------------------
``6.2.6``|                  |``GCCcore/8.2.0``
``6.2.6``|``-Python-2.7.15``|``fosscuda/2018b``
``6.2.9``|                  |``GCCcore/10.2.0``, ``GCCcore/11.3.0``

<a anchor="bsmapz"/>
### *BSMAPz*

Updated and optimized fork of BSMAP. BSMAPz is a short reads mapping program for bisulfite sequencing in DNA methylation study.

*homepage*: https://github.com/zyndagj/BSMAPz

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.1.1``|``-Python-2.7.16``|``intel/2019b``

<a anchor="bsoft"/>
### *Bsoft*

Bsoft is a collection of programs and a platform for development of software  for image and molecular processing in structural biology. Problems in  structural biology are approached with a highly modular design, allowing fast  development of new algorithms without the burden of issues such as file I/O.  It provides an easily accessible interface, a resource that can be and has  been used in other packages. 

*homepage*: https://lsbr.niams.nih.gov/bsoft/

version  |toolchain
---------|--------------
``2.0.2``|``foss/2017b``
``2.0.7``|``GCC/9.3.0``

<a anchor="bsseeker2"/>
### *BSseeker2*

BS-Seeker2 is a seamless and versatile pipeline for accurately and fast mapping the bisulfite-treated reads.

*homepage*: http://pellegrini-legacy.mcdb.ucla.edu/bs_seeker2

version  |versionsuffix     |toolchain
---------|------------------|--------------------------------------
``2.1.8``|``-Python-2.7.16``|``GCC/8.3.0``, ``iccifort/2019.5.281``

<a anchor="buddy"/>
### *BuDDy*

A Binary Decision Diagram library, with many highly efficient vectorized BDD operations, dynamic  variable reordering, automated garbage collection, a C++ interface with automatic reference counting, and much  more.

*homepage*: http://buddy.sourceforge.net/

version|toolchain
-------|------------------
``2.4``|``GCCcore/10.2.0``

<a anchor="bufrlib"/>
### *BUFRLIB*

NCEP BUFRLIB software to encode or decode BUFR messages. It is not intended to be a primer on the BUFR code form itself.

*homepage*: https://www.emc.ncep.noaa.gov/index.php?branch=BUFRLIB

version     |toolchain
------------|-----------------------
``11.3.0.2``|``iccifort/2020.1.217``

<a anchor="buildenv"/>
### *buildenv*

This module sets a group of environment variables for compilers, linkers, maths libraries, etc., that  you can use to easily transition between toolchains when building your software. To query the variables being set  please use: module show <this module name>

*homepage*: None

version    |versionsuffix   |toolchain
-----------|----------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``default``|                |``FCC/4.5.0``, ``Fujitsu/21.05``, ``foss/2017b``, ``foss/2018b``, ``foss/2019b``, ``foss/2020a``, ``foss/2020b``, ``foss/2021a``, ``foss/2021b``, ``foss/2022a``, ``fosscuda/2019b``, ``fosscuda/2020a``, ``fosscuda/2020b``, ``intel/2016b``, ``intel/2017a``, ``intel/2019b``, ``intel/2020a``, ``intel/2020b``, ``intel/2021a``, ``intel/2021b``, ``intel/2022a``, ``intelcuda/2019b``, ``intelcuda/2020a``, ``intelcuda/2020b``
``default``|``-CUDA-11.3.1``|``foss/2021a``
``default``|``-CUDA-11.4.1``|``foss/2021b``
``default``|``-CUDA-11.7.0``|``foss/2022a``

<a anchor="buildingspy"/>
### *buildingspy*

Python modules for automating Modelica simulations and for running unit test for the Buildings library

*homepage*: https://simulationresearch.lbl.gov/modelica/buildingspy

version  |toolchain
---------|--------------
``4.0.0``|``foss/2022a``

<a anchor="bullet"/>
### *Bullet*

Bullet professional 3D Game Multiphysics Library provides state  of the art collision detection, soft body and rigid body dynamics.

*homepage*: http://bulletphysics.org/

version   |toolchain
----------|-------------------------------
``2.83.7``|``foss/2016a``, ``intel/2016a``

<a anchor="busco"/>
### *BUSCO*

BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs

*homepage*: https://busco.ezlab.org/

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.22`` |``-Python-2.7.13``|``intel/2017a``
``2.0.1``|``-Python-2.7.13``|``intel/2017a``
``3.0.2``|``-Python-2.7.15``|``intel/2018b``
``4.0.5``|``-Python-3.7.4`` |``foss/2019b``
``4.0.6``|                  |``foss/2020b``
``5.0.0``|                  |``foss/2020b``
``5.1.2``|                  |``foss/2020b``
``5.4.2``|                  |``foss/2021a``
``5.4.3``|                  |``foss/2021b``

<a anchor="bustools"/>
### *BUStools*

bustools is a program for manipulating BUS files for single cell RNA-Seq datasets.  It can be used to error correct barcodes, collapse  UMIs, produce gene count or transcript compatibility  count matrices, and is useful for many other tasks. See the kallisto | bustools  website for examples  and instructions on how to use bustools as part of a single-cell RNA-seq workflow.

*homepage*: https://github.com/BUStools/bustools

version   |toolchain
----------|---------------------------------
``0.40.0``|``GCCcore/9.3.0``, ``foss/2018b``

<a anchor="bwa"/>
### *BWA*

Burrows-Wheeler Aligner (BWA) is an efficient program that aligns  relatively short nucleotide sequences against a long reference sequence such as the human genome.

*homepage*: http://bio-bwa.sourceforge.net/

version    |toolchain
-----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``0.7.12`` |``foss/2016b``
``0.7.13`` |``foss/2016a``, ``intel/2016a``
``0.7.15`` |``GCCcore/5.4.0``, ``GCCcore/6.4.0``, ``foss/2016a``, ``foss/2016b``, ``intel/2016b``, ``intel/2017a``
``0.7.16a``|``foss/2016b``, ``intel/2017a``
``0.7.17`` |``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/8.2.0-2.31.1``, ``GCC/8.3.0``, ``GCC/9.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``iccifort/2019.5.281``, ``iccifort/2020.4.304``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``

<a anchor="bwa-meth"/>
### *bwa-meth*

Fast and accurante alignment of BS-Seq reads.

*homepage*: https://github.com/brentp/bwa-meth

version  |toolchain
---------|-----------------------------------------------------------------
``0.2.2``|``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``iccifort/2019.5.281``

<a anchor="bwakit"/>
### *bwakit*

Bwakit is a self-consistent installation-free package of scripts and precompiled binaries, providing an end-to-end solution to read mapping.

*homepage*: https://github.com/lh3/bwa/tree/master/bwakit

version   |versionsuffix |toolchain
----------|--------------|----------
``0.7.15``|``_x64-linux``|``system``

<a anchor="bwidget"/>
### *bwidget*

The BWidget Toolkit is a high-level Widget Set for Tcl/Tk built using native Tcl/Tk 8.x namespaces.

*homepage*: https://core.tcl-lang.org/bwidget/home

version   |toolchain
----------|--------------------------------------------------------
``1.9.13``|``GCCcore/8.2.0``
``1.9.14``|``GCCcore/10.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.9.15``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="bwise"/>
### *BWISE*

de Bruijn Workflow using Integral information of Short pair End reads

*homepage*: https://github.com/Malfoy/BWISE

version     |versionsuffix    |toolchain
------------|-----------------|--------------
``20180820``|``-Python-3.6.4``|``foss/2018a``

<a anchor="bx-python"/>
### *bx-python*

The bx-python project is a Python library and associated set of scripts to allow for rapid  implementation of genome scale analyses.

*homepage*: https://github.com/bxlab/bx-python

version   |versionsuffix     |toolchain
----------|------------------|---------------
``0.7.4`` |``-Python-2.7.12``|``foss/2016b``
``0.7.4`` |``-Python-2.7.13``|``intel/2017a``
``0.8.1`` |``-Python-2.7.14``|``intel/2018a``
``0.8.2`` |``-Python-3.6.6`` |``foss/2018b``
``0.8.4`` |                  |``foss/2019a``
``0.8.8`` |``-Python-3.7.4`` |``foss/2019b``
``0.8.9`` |``-Python-3.8.2`` |``foss/2020a``
``0.8.11``|                  |``foss/2021a``
``0.8.13``|                  |``foss/2021b``

<a anchor="bxh_xcede_tools"/>
### *BXH_XCEDE_TOOLS*

A collection of data processing and image analysis  tools for data in BXH or XCEDE format. This includes data format  encapsulation/conversion, event-related analysis, QA tools, and more.  These tools form the basis of the fBIRN QA procedures and are also  distributed as part of the fBIRN Data Upload Scripts.

*homepage*: http://www.nitrc.org/projects/bxh_xcede_tools/

version   |toolchain
----------|----------
``1.11.1``|``system``

<a anchor="byacc"/>
### *byacc*

Berkeley Yacc (byacc) is generally conceded to be the best yacc variant  available.  In contrast to bison, it is written to avoid dependencies  upon a particular compiler. 

*homepage*: http://invisible-island.net/byacc/byacc.html

version     |toolchain
------------|-------------------------------
``20160324``|``intel/2016a``
``20160606``|``foss/2016b``, ``intel/2016b``
``20170709``|``GCCcore/6.4.0``

<a anchor="byobu"/>
### *byobu*

Byobu is an elegant enhancement of the otherwise functional, plain, practical GNU Screen. Byobu includes an enhanced profile, configuration utilities, and system status notifications for the GNU screen window manager as well as the Tmux terminal multiplexer

*homepage*: https://byobu.org

version  |toolchain
---------|-------------
``5.133``|``GCC/8.3.0``

<a anchor="bzip2"/>
### *bzip2*

bzip2 is a freely available, patent free, high-quality data compressor. It  typically compresses files to within 10% to 15% of the best available  techniques (the PPM family of statistical compressors), whilst being around  twice as fast at compression and six times faster at decompression. 

*homepage*: https://sourceware.org/bzip2

version  |toolchain
---------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.0.6``|``GCC/4.8.1``, ``GCC/4.8.2``, ``GCC/4.8.4``, ``GCC/4.9.2``, ``GCC/4.9.3-2.25``, ``GCC/5.4.0-2.26``, ``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GNU/4.9.3-2.25``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``gimkl/2017a``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
``1.0.8``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``system``

<a anchor="c"/>
### *C*


<a href="#c-ares">c-ares</a> - <a href="#c3d">C3D</a> - <a href="#cadaver">cadaver</a> - <a href="#cadical">CaDiCaL</a> - <a href="#cafe5">CAFE5</a> - <a href="#caffe">Caffe</a> - <a href="#cairo">cairo</a> - <a href="#cairomm">cairomm</a> - <a href="#calcam">Calcam</a> - <a href="#calendrical">Calendrical</a> - <a href="#calib">Calib</a> - <a href="#campari">CAMPARI</a> - <a href="#cantera">Cantera</a> - <a href="#canu">canu</a> - <a href="#canvas">Canvas</a> - <a href="#cap3">CAP3</a> - <a href="#capnproto">CapnProto</a> - <a href="#captum">captum</a> - <a href="#cargo">Cargo</a> - <a href="#carma">Carma</a> - <a href="#carputils">carputils</a> - <a href="#cartopy">Cartopy</a> - <a href="#casper">CaSpER</a> - <a href="#castep">CASTEP</a> - <a href="#castxml">CastXML</a> - <a href="#cat-bat">CAT-BAT</a> - <a href="#catch2">Catch2</a> - <a href="#category_encoders">category_encoders</a> - <a href="#catmap">CatMAP</a> - <a href="#causallift">causallift</a> - <a href="#causalml">causalml</a> - <a href="#caveman">CaVEMan</a> - <a href="#caviar">CAVIAR</a> - <a href="#cbc">Cbc</a> - <a href="#cblas">CBLAS</a> - <a href="#ccache">ccache</a> - <a href="#ccfits">CCfits</a> - <a href="#ccl">CCL</a> - <a href="#cclib">cclib</a> - <a href="#cctools">cctools</a> - <a href="#cd-hit">CD-HIT</a> - <a href="#cdat">CDAT</a> - <a href="#cdbfasta">cdbfasta</a> - <a href="#cdbtools">CDBtools</a> - <a href="#cddlib">cddlib</a> - <a href="#cdna_cupcake">cDNA_Cupcake</a> - <a href="#cdo">CDO</a> - <a href="#cdo-bindings">cdo-bindings</a> - <a href="#cdsapi">cdsapi</a> - <a href="#cell2location">cell2location</a> - <a href="#cellmix">CellMix</a> - <a href="#cellranger">CellRanger</a> - <a href="#cellranger-arc">CellRanger-ARC</a> - <a href="#cellranger-atac">CellRanger-ATAC</a> - <a href="#cellrank">CellRank</a> - <a href="#celltypist">CellTypist</a> - <a href="#centrifuge">Centrifuge</a> - <a href="#cereal">Cereal</a> - <a href="#cesm-deps">CESM-deps</a> - <a href="#cfdemcoupling">CFDEMcoupling</a> - <a href="#cfitsio">CFITSIO</a> - <a href="#cftime">cftime</a> - <a href="#cgal">CGAL</a> - <a href="#cget">cget</a> - <a href="#cgl">Cgl</a> - <a href="#cgmaptools">CGmapTools</a> - <a href="#cgns">CGNS</a> - <a href="#charls">CharLS</a> - <a href="#charmm">CHARMM</a> - <a href="#charmm">charmm</a> - <a href="#chase">CHASE</a> - <a href="#check">Check</a> - <a href="#check">check</a> - <a href="#checkm">CheckM</a> - <a href="#cheetah">Cheetah</a> - <a href="#chemaxon-marvin">Chemaxon-Marvin</a> - <a href="#chemps2">CheMPS2</a> - <a href="#cherab">CHERAB</a> - <a href="#chewbbaca">chewBBACA</a> - <a href="#chi2comb">chi2comb</a> - <a href="#chimera">Chimera</a> - <a href="#chimpipe">ChimPipe</a> - <a href="#chromaprint">Chromaprint</a> - <a href="#cicero">cicero</a> - <a href="#cif2cell">CIF2Cell</a> - <a href="#circexplorer">CIRCexplorer</a> - <a href="#circexplorer2">CIRCexplorer2</a> - <a href="#circos">Circos</a> - <a href="#ciri">CIRI</a> - <a href="#ciri-long">CIRI-long</a> - <a href="#cistem">cisTEM</a> - <a href="#cite-seq-count">CITE-seq-Count</a> - <a href="#clang">Clang</a> - <a href="#clang-aomp">Clang-AOMP</a> - <a href="#clang-python-bindings">Clang-Python-bindings</a> - <a href="#clapack">CLAPACK</a> - <a href="#clease">CLEASE</a> - <a href="#clhep">CLHEP</a> - <a href="#climetlab">CliMetLab</a> - <a href="#clisp">CLISP</a> - <a href="#clonalframeml">ClonalFrameML</a> - <a href="#cloog">CLooG</a> - <a href="#cloudcompare">CloudCompare</a> - <a href="#clp">Clp</a> - <a href="#clustal-omega">Clustal-Omega</a> - <a href="#clustalw2">ClustalW2</a> - <a href="#cluster-buster">Cluster-Buster</a> - <a href="#clustershell">ClusterShell</a> - <a href="#cmake">CMake</a> - <a href="#cmaverse">CMAverse</a> - <a href="#cmdstanr">CmdStanR</a> - <a href="#cmocean">cmocean</a> - <a href="#cmseq">CMSeq</a> - <a href="#cnt-ilp">CNT-ILP</a> - <a href="#cnvkit">CNVkit</a> - <a href="#cnvnator">CNVnator</a> - <a href="#co-phylog">Co-phylog</a> - <a href="#cobrapy">COBRApy</a> - <a href="#cocoalib">CoCoALib</a> - <a href="#codan">CodAn</a> - <a href="#code-server">code-server</a> - <a href="#codex2">CODEX2</a> - <a href="#cogent">Cogent</a> - <a href="#coin">Coin</a> - <a href="#coinutils">CoinUtils</a> - <a href="#colossalai">colossalai</a> - <a href="#commet">Commet</a> - <a href="#comparem">CompareM</a> - <a href="#compress-raw-zlib">Compress-Raw-Zlib</a> - <a href="#comsol">COMSOL</a> - <a href="#con3f">Con3F</a> - <a href="#concoct">CONCOCT</a> - <a href="#configparser">configparser</a> - <a href="#configurable-http-proxy">configurable-http-proxy</a> - <a href="#conn">CONN</a> - <a href="#connectomeworkbench">ConnectomeWorkbench</a> - <a href="#control-freec">Control-FREEC</a> - <a href="#coordgenlibs">CoordgenLibs</a> - <a href="#coot">Coot</a> - <a href="#core-counter">core-counter</a> - <a href="#coreutils">Coreutils</a> - <a href="#corner">corner</a> - <a href="#coverage">coverage</a> - <a href="#cowsay">cowsay</a> - <a href="#cp2k">CP2K</a> - <a href="#cpb">CPB</a> - <a href="#cplex">CPLEX</a> - <a href="#cppe">CPPE</a> - <a href="#cppunit">CppUnit</a> - <a href="#cppy">cppy</a> - <a href="#cpu_features">cpu_features</a> - <a href="#cram">cram</a> - <a href="#cramtools">cramtools</a> - <a href="#craycce">CrayCCE</a> - <a href="#craygnu">CrayGNU</a> - <a href="#crayintel">CrayIntel</a> - <a href="#craypgi">CrayPGI</a> - <a href="#crb-blast">crb-blast</a> - <a href="#crest">CREST</a> - <a href="#crf++">CRF++</a> - <a href="#crispr-dav">CRISPR-DAV</a> - <a href="#crispresso2">CRISPResso2</a> - <a href="#cromwell">cromwell</a> - <a href="#crossmap">CrossMap</a> - <a href="#crosstalkz">CrossTalkZ</a> - <a href="#crpropa">CRPropa</a> - <a href="#crumble">Crumble</a> - <a href="#cryocare">cryoCARE</a> - <a href="#cryodrgn">cryoDRGN</a> - <a href="#cryptominisat">CryptoMiniSat</a> - <a href="#crystfel">CrystFEL</a> - <a href="#csb">CSB</a> - <a href="#csbdeep">CSBDeep</a> - <a href="#csblast">CSBLAST</a> - <a href="#cscope">cscope</a> - <a href="#csvkit">csvkit</a> - <a href="#ctags">ctags</a> - <a href="#ctffind">ctffind</a> - <a href="#ctpl">CTPL</a> - <a href="#cube">Cube</a> - <a href="#cubegui">CubeGUI</a> - <a href="#cubelib">CubeLib</a> - <a href="#cubewriter">CubeWriter</a> - <a href="#cuda">CUDA</a> - <a href="#cuda-samples">CUDA-Samples</a> - <a href="#cudacompat">CUDAcompat</a> - <a href="#cudacore">CUDAcore</a> - <a href="#cudnn">cuDNN</a> - <a href="#cufflinks">Cufflinks</a> - <a href="#cunit">CUnit</a> - <a href="#cupy">CuPy</a> - <a href="#curl">cURL</a> - <a href="#custodian">custodian</a> - <a href="#cutadapt">cutadapt</a> - <a href="#cutensor">cuTENSOR</a> - <a href="#cvs">CVS</a> - <a href="#cvxopt">CVXOPT</a> - <a href="#cvxpy">CVXPY</a> - <a href="#cwipi">CWIPI</a> - <a href="#cwltool">cwltool</a> - <a href="#cxxopts">cxxopts</a> - <a href="#cysignals">cysignals</a> - <a href="#cython">Cython</a> - <a href="#cytoolz">cytoolz</a> - <a href="#cytoscape">Cytoscape</a> - <a href="#cytosim">cytosim</a> - <a href="#cyvcf2">cyvcf2</a>


<a anchor="c-ares"/>
### *c-ares*

c-ares is a C library for asynchronous DNS requests (including name resolves)

*homepage*: https://c-ares.org/

version   |toolchain
----------|--------------------------------------
``1.17.2``|``GCCcore/10.3.0``
``1.18.1``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="c3d"/>
### *C3D*

Convert3D Medical Image Processing Tool

*homepage*: https://sourceforge.net/projects/c3d/

version  |toolchain
---------|----------
``1.0.0``|``system``

<a anchor="cadaver"/>
### *cadaver*

cadaver is a command-line WebDAV client for Unix.

*homepage*: http://www.webdav.org/cadaver/

version   |toolchain
----------|---------------
``0.23.3``|``intel/2017a``

<a anchor="cadical"/>
### *CaDiCaL*

CaDiCaL is a simplified satisfiability solver. The original goal of the development of CaDiCaL was to obtain a CDCL solver, which is easy to understand and change, while at the same time not being much slower than other state-of-the-art CDCL solvers.

*homepage*: https://github.com/arminbiere/cadical

version  |toolchain
---------|-------------
``1.3.0``|``GCC/9.3.0``

<a anchor="cafe5"/>
### *CAFE5*

Software for Computational Analysis of gene Family Evolution  The purpose of CAFE is to analyze changes in gene family size in a way that accounts for phylogenetic history and provides a statistical foundation for evolutionary inferences. The program uses a birth and death process to model gene gain and loss across a user-specified phylogenetic tree. The distribution of family sizes generated under this model can provide a basis for assessing the significance of the observed family size differences among taxa.

*homepage*: https://github.com/hahnlab/CAFE5

version  |toolchain
---------|--------------
``5.0.0``|``GCC/10.2.0``

<a anchor="caffe"/>
### *Caffe*

Caffe is a deep learning framework made with expression, speed,  and modularity in mind. It is developed by the Berkeley Vision  and Learning Center (BVLC) and community contributors. 

*homepage*: https://github.com/BVLC/caffe

version|versionsuffix                 |toolchain
-------|------------------------------|---------------
``1.0``|``-CUDA-9.1.85-Python-2.7.14``|``intel/2017b``
``1.0``|``-Python-2.7.13``            |``intel/2017a``
``1.0``|``-Python-2.7.14``            |``intel/2017b``
``rc3``|``-CUDA-7.5.18-Python-2.7.11``|``foss/2016a``

<a anchor="cairo"/>
### *cairo*

Cairo is a 2D graphics library with support for multiple output devices.  Currently supported output targets include the X Window System (via both Xlib and XCB), Quartz, Win32, image buffers,  PostScript, PDF, and SVG file output. Experimental backends include OpenGL, BeOS, OS/2, and DirectFB

*homepage*: https://cairographics.org

version    |versionsuffix   |toolchain
-----------|----------------|-------------------------------------------------------------------------------------------------------------------
``1.14.6`` |                |``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``1.14.6`` |``-GLib-2.48.0``|``foss/2016a``, ``intel/2016a``
``1.14.8`` |                |``intel/2017a``
``1.14.10``|                |``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``intel/2017b``
``1.14.12``|                |``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``1.16.0`` |                |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.17.4`` |                |``GCCcore/11.3.0``, ``GCCcore/12.2.0``

<a anchor="cairomm"/>
### *cairomm*

The Cairomm package provides a C++ interface to Cairo.

*homepage*: http://cairographics.org

version   |toolchain
----------|------------------------------------
``1.12.2``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``

<a anchor="calcam"/>
### *Calcam*

Calcam is a Python package providing tools for spatial calibration  of cameras, i.e. determining the mapping between pixel coordinates in an image and  real-world 3D sight lines & coordinates.

*homepage*: https://euratom-software.github.io/calcam

version  |versionsuffix     |toolchain
---------|------------------|---------------
``2.1.0``|``-Python-2.7.14``|``intel/2018a``
``2.1.0``|``-Python-3.6.4`` |``intel/2018a``

<a anchor="calendrical"/>
### *Calendrical*

Calendrical module is for calendrical calculations.

*homepage*: https://www.funaba.org/code#calendrical

version   |versionsuffix    |toolchain
----------|-----------------|---------------
``2.0.1`` |``-Python-3.6.3``|``intel/2017b``
``2.0.2a``|``-Python-3.6.4``|``intel/2018a``
``2.0.2a``|``-Python-3.6.6``|``intel/2018b``

<a anchor="calib"/>
### *Calib*

Calib clusters paired-end reads using their barcodes and sequences. Calib is suitable for amplicon sequencing where a molecule is tagged, then PCR amplified with high depth, also known as Unique Molecule Identifier (UMI) sequencing.

*homepage*: https://github.com/vpc-ccg/calib

version  |toolchain
---------|-------------
``0.3.4``|``GCC/9.3.0``

<a anchor="campari"/>
### *CAMPARI*

CAMPARI is a joint package for performing and analyzing molecular simulations, in particular of systems of biological relevance. It focuses on a wide availability of algorithms for (advanced) sampling and is capable of combining Monte Carlo and molecular dynamics in seamless fashion.

*homepage*: http://campari.sourceforge.net/V4/index.html

version|toolchain
-------|---------------
``4.0``|``intel/2020b``

<a anchor="cantera"/>
### *Cantera*

Chemical kinetics, thermodynamics, and transport tool suite

*homepage*: https://github.com/Cantera/cantera

version  |versionsuffix     |toolchain
---------|------------------|------------------------------------------------
``2.2.1``|``-Python-2.7.12``|``intel/2016b``
``2.3.0``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``2.3.0``|``-Python-2.7.13``|``intel/2017a``
``2.3.0``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``, ``intel/2018a``
``2.4.0``|``-Python-2.7.14``|``intel/2018a``

<a anchor="canu"/>
### *canu*

Canu is a fork of the Celera Assembler designed for high-noise single-molecule sequencing

*homepage*: https://canu.readthedocs.io

version  |versionsuffix   |toolchain
---------|----------------|--------------------------------------
``1.4``  |                |``foss/2016b``
``1.7``  |                |``intel/2018a``
``1.8``  |``-Perl-5.26.0``|``foss/2017b``, ``intel/2017b``
``1.8``  |``-Perl-5.28.0``|``foss/2018b``
``1.8``  |``-Perl-5.28.1``|``GCCcore/8.2.0``
``1.9``  |``-Java-11``    |``GCCcore/8.3.0``
``2.1.1``|                |``GCCcore/10.2.0``
``2.1.1``|``-Java-11``    |``GCCcore/9.3.0``
``2.2``  |                |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``2.2``  |``-Java-11``    |``GCCcore/10.3.0``

<a anchor="canvas"/>
### *Canvas*

Copy number variant (CNV) calling from DNA sequencing data

*homepage*: https://github.com/Illumina/canvas

version        |toolchain
---------------|----------
``1.39.0.1598``|``system``

<a anchor="cap3"/>
### *CAP3*

CAP3 assembly program 

*homepage*: http://seq.cs.iastate.edu/

version                  |toolchain
-------------------------|----------
``20071221-intel-x86``   |``system``
``20071221-intel-x86_64``|``system``
``20071221-opteron``     |``system``

<a anchor="capnproto"/>
### *CapnProto*

Cap’n Proto is an insanely fast data interchange format and capability-based RPC system.

*homepage*: https://capnproto.org

version   |toolchain
----------|----------------------------------------------------------
``0.6.1`` |``GCCcore/6.4.0``
``0.7.0`` |``GCCcore/7.3.0``
``0.8.0`` |``GCCcore/9.3.0``
``0.9.1`` |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``0.10.2``|``GCCcore/11.3.0``

<a anchor="captum"/>
### *captum*

Captum is a model interpretability and understanding library for PyTorch. Captum means comprehension in Latin and contains general purpose implementations of integrated gradients, saliency maps, smoothgrad, vargrad and others for PyTorch models. It has quick integration for models built with domain-specific libraries such as torchvision, torchtext, and others.

*homepage*: https://captum.ai/

version  |versionsuffix   |toolchain
---------|----------------|--------------
``0.5.0``|                |``foss/2022a``
``0.5.0``|``-CUDA-11.7.0``|``foss/2022a``

<a anchor="cargo"/>
### *Cargo*

The Rust package manager

*homepage*: https://crates.io/

version   |toolchain
----------|--------------
``0.13.0``|``foss/2016b``

<a anchor="carma"/>
### *Carma*

Carma - A molecular dynamics analysis program

*homepage*: http://utopia.duth.gr/~glykos/Carma.html

version |toolchain
--------|--------------
``2.01``|``foss/2019b``

<a anchor="carputils"/>
### *carputils*

carputils is a Python framework for generating and running openCARP examples.

*homepage*: https://git.opencarp.org/openCARP/carputils

version     |versionsuffix    |toolchain
------------|-----------------|--------------
``20200915``|``-Python-3.8.2``|``foss/2020a``
``20210513``|                 |``foss/2020b``

<a anchor="cartopy"/>
### *Cartopy*

Cartopy is a Python package designed to make drawing maps for data analysis and visualisation easy.

*homepage*: https://scitools.org.uk/cartopy/docs/latest/

version         |versionsuffix    |toolchain
----------------|-----------------|-------------------------------
``0.18.0``      |``-Python-3.7.4``|``foss/2019b``
``0.18.0``      |``-Python-3.8.2``|``foss/2020a``
``0.19.0.post1``|                 |``foss/2020b``, ``intel/2020b``
``0.20.0``      |                 |``foss/2021a``
``0.20.3``      |                 |``foss/2021b``, ``foss/2022a``

<a anchor="casper"/>
### *CaSpER*

CaSpER is a signal processing approach for identification, visualization, and integrative analysis  of focal and large-scale CNV events in multiscale resolution using either bulk or single-cell RNA sequencing data. 

*homepage*: https://github.com/akdess/CaSpER

version|toolchain
-------|--------------
``2.0``|``foss/2019b``

<a anchor="castep"/>
### *CASTEP*

CASTEP is an electronic structure materials modelling code based on density functional theory (DFT),  with functionality including geometry optimization molecular dynamics, phonons, NMR chemical shifts and much more.

*homepage*: http://www.castep.org

version   |toolchain
----------|--------------------------------------------------------------------------------------
``21.1.1``|``CrayCCE/19.06``, ``CrayGNU/19.06``, ``foss/2019b``, ``intel/2019b``, ``iomkl/2019b``

<a anchor="castxml"/>
### *CastXML*

CastXML is a C-family abstract syntax tree XML output tool.

*homepage*: https://github.com/CastXML/CastXML

version     |toolchain
------------|-----------------
``0.4.3``   |``GCCcore/8.3.0``
``20160617``|``foss/2016a``
``20180806``|``foss/2018a``

<a anchor="cat-bat"/>
### *CAT-BAT*

Tool for taxonomic classification of contigs and metagenome-assembled genomes (MAGs).

*homepage*: https://github.com/dutilh/CAT

version  |toolchain
---------|--------------
``5.2.3``|``GCC/10.3.0``

<a anchor="catch2"/>
### *Catch2*

A modern, C++-native, header-only,   test framework for unit-tests, TDD and BDD   - using C++11, C++14, C++17 and later     (or C++03 on the Catch1.x branch) 

*homepage*: https://github.com/catchorg/Catch2

version   |toolchain
----------|----------
``2.9.1`` |``system``
``2.11.0``|``system``
``2.13.4``|``system``

<a anchor="category_encoders"/>
### *category_encoders*

A set of scikit-learn-style transformers for encoding categorical variables into numeric by means of different techniques.

*homepage*: https://github.com/scikit-learn-contrib/category_encoders

version  |toolchain
---------|--------------
``2.4.1``|``foss/2021b``

<a anchor="catmap"/>
### *CatMAP*

Catalyst Micro-kinetic Analysis Package for automated creation of micro-kinetic models used  in catalyst screening.

*homepage*: https://catmap.readthedocs.io/

version     |versionsuffix     |toolchain
------------|------------------|---------------
``20170927``|``-Python-2.7.14``|``intel/2017b``

<a anchor="causallift"/>
### *causallift*

CausalLift: Python package for Uplift Modeling in real-world business; applicable for both A/B testing and observational data 

*homepage*: https://github.com/Minyus/causallift

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.0.6``|``-Python-3.8.2``|``foss/2020a``

<a anchor="causalml"/>
### *causalml*

Causal ML: A Python Package for Uplift Modeling and Causal Inference with ML 

*homepage*: https://github.com/uber/causalml

version           |versionsuffix    |toolchain
------------------|-----------------|--------------
``0.3.0-20180610``|``-Python-3.7.2``|``foss/2019a``
``0.8.0-20200909``|``-Python-3.8.2``|``foss/2020a``

<a anchor="caveman"/>
### *CaVEMan*

SNV expectation maximisation based mutation calling algorithm   aimed at detecting somatic mutations in paired (tumour/normal) cancer samples.  Supports both bam and cram format via htslib

*homepage*: http://cancerit.github.io/CaVEMan/

version   |toolchain
----------|--------------
``1.13.2``|``foss/2018a``

<a anchor="caviar"/>
### *CAVIAR*

CAusal Variants Identication in Associated Regions. A statistical framework  that quantifies the probability of each variant to be causal while allowing an arbitrary  number of causal variants.

*homepage*: https://github.com/fhormoz/caviar

version         |toolchain
----------------|--------------
``2.2-20190419``|``foss/2019b``

<a anchor="cbc"/>
### *Cbc*

Cbc (Coin-or branch and cut) is an open-source mixed integer linear programming solver written in C++. It can be used as a callable library or using a stand-alone executable.

*homepage*: https://github.com/coin-or/Cbc

version   |toolchain
----------|------------------------------
``2.10.3``|``foss/2018b``
``2.10.5``|``foss/2020b``, ``foss/2021a``

<a anchor="cblas"/>
### *CBLAS*

C interface to the BLAS

*homepage*: https://www.netlib.org/blas/

version     |toolchain
------------|------------------------------------------------
``20110120``|``foss/2016b``, ``intel/2019b``, ``intel/2020a``

<a anchor="ccache"/>
### *ccache*

Ccache (or “ccache”) is a compiler cache. It speeds up recompilation by caching previous compilations and detecting when the same compilation is being done again

*homepage*: https://ccache.dev/

version   |versionsuffix|toolchain
----------|-------------|------------------
``3.2.5`` |             |``system``
``3.3.1`` |             |``system``
``3.3.3`` |             |``system``
``3.3.4`` |``-f90``     |``system``
``3.7.11``|             |``system``
``4.2.1`` |             |``system``
``4.6.1`` |             |``GCCcore/11.2.0``
``4.6.3`` |             |``system``

<a anchor="ccfits"/>
### *CCfits*

CCfits is an object oriented interface to the cfitsio library. It is designed to make the capabilities of cfitsio available to programmers working in C++.

*homepage*: https://heasarc.gsfc.nasa.gov/fitsio/CCfits/

version|toolchain
-------|-----------------
``2.5``|``GCCcore/9.3.0``

<a anchor="ccl"/>
### *CCL*

Clozure CL (often called CCL for short) is a free Common Lisp  implementation with a long history. Some distinguishing features of the implementation  include fast compilation speed, native threads, a precise, generational, compacting  garbage collector, and a convenient foreign-function interface.

*homepage*: https://ccl.clozure.com/

version   |toolchain
----------|------------------
``1.11.5``|``system``
``1.12``  |``GCCcore/9.3.0``
``1.12.1``|``GCCcore/10.3.0``

<a anchor="cclib"/>
### *cclib*

cclib is a Python library that provides parsers for computational chemistry log files.  It also provides a platform to implement algorithms in a package-independent manner. 

*homepage*: https://cclib.github.io/

version  |versionsuffix    |toolchain
---------|-----------------|-------------------------------
``1.5``  |``-Python-3.5.2``|``foss/2016b``
``1.6.3``|``-Python-3.8.2``|``foss/2020a``, ``intel/2020a``
``1.7.2``|                 |``foss/2021b``

<a anchor="cctools"/>
### *cctools*

The Cooperating Computing Tools (CCTools) help you to design and deploy  scalable applications that run on hundreds or thousands of machines at once. 

*homepage*: https://ccl.cse.nd.edu/

version   |toolchain
----------|-----------------
``7.0.22``|``GCCcore/8.3.0``

<a anchor="cd-hit"/>
### *CD-HIT*

CD-HIT is a very widely used program for clustering and  comparing protein or nucleotide sequences.

*homepage*: https://github.com/weizhongli/cdhit

version  |versionsuffix |toolchain
---------|--------------|---------------------------------------------------------------------------------------------------------------------
``4.6.4``|``-2015-0603``|``GNU/4.9.3-2.25``
``4.6.6``|              |``foss/2016b``
``4.6.8``|              |``foss/2018b``, ``intel/2017a``, ``intel/2018a``
``4.8.1``|              |``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/8.3.0``, ``GCC/9.3.0``, ``foss/2018b``, ``iccifort/2019.5.281``

<a anchor="cdat"/>
### *CDAT*

CDAT is a powerful and complete front-end to a rich set of  visual-data exploration and analysis capabilities well suited for data analysis problems.

*homepage*: https://github.com/CDAT/cdat

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``8.2.1``|``-Python-3.8.2``|``foss/2020a``

<a anchor="cdbfasta"/>
### *cdbfasta*

Fasta file indexing and retrival tool

*homepage*: https://sourceforge.net/projects/cdbfasta

version |toolchain
--------|--------------------------------------------------------------------------------
``0.99``|``GCC/8.3.0``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``iccifort/2019.5.281``

<a anchor="cdbtools"/>
### *CDBtools*

CDB (Constant DataBase) indexing and retrieval tools for FASTA files

*homepage*: http://compbio.dfci.harvard.edu/tgi

version |toolchain
--------|--------------
``0.99``|``GCC/10.2.0``

<a anchor="cddlib"/>
### *cddlib*

An efficient implementation of the Double Description Method

*homepage*: https://github.com/cddlib/cddlib

version  |toolchain
---------|-----------------
``0.94i``|``GCCcore/8.2.0``
``0.94j``|``GCCcore/8.3.0``

<a anchor="cdna_cupcake"/>
### *cDNA_Cupcake*

cDNA_Cupcake is a miscellaneous collection of Python and R scripts used for analyzing sequencing data.

*homepage*: https://github.com/Magdoll/cDNA_Cupcake

version   |versionsuffix     |toolchain
----------|------------------|---------------
``5.8``   |``-Python-2.7.14``|``intel/2018a``
``24.2.0``|``-Python-3.8.2`` |``foss/2020a``
``26.0.0``|                  |``foss/2021a``

<a anchor="cdo"/>
### *CDO*

CDO is a collection of command line Operators to manipulate and analyse Climate and NWP model Data.

*homepage*: https://code.zmaw.de/projects/cdo

version   |toolchain
----------|------------------------------------------------------------------
``1.7.2`` |``intel/2016b``
``1.8.1`` |``intel/2017a``
``1.9.1`` |``intel/2017b``
``1.9.2`` |``intel/2017b``
``1.9.5`` |``intel/2018a``, ``intel/2018b``, ``iomkl/2018b``
``1.9.8`` |``intel/2019b``
``1.9.10``|``gompi/2019b``, ``gompi/2020b``, ``gompi/2021a``, ``iimpi/2021b``
``2.0.5`` |``gompi/2021b``
``2.0.6`` |``gompi/2022a``
``2.1.1`` |``gompi/2021a``

<a anchor="cdo-bindings"/>
### *cdo-bindings*

Python interface to CDO.

*homepage*: https://github.com/Try2Code/cdo-bindings

version  |toolchain
---------|--------------
``1.5.7``|``foss/2021b``

<a anchor="cdsapi"/>
### *cdsapi*

Climate Data Store API

*homepage*: https://pypi.org/project/cdsapi

version  |versionsuffix    |toolchain
---------|-----------------|------------------
``0.1.4``|                 |``foss/2019a``
``0.1.4``|``-Python-3.6.6``|``foss/2018b``
``0.3.0``|                 |``GCCcore/9.3.0``
``0.5.1``|                 |``GCCcore/11.3.0``

<a anchor="cell2location"/>
### *cell2location*

Comprehensive mapping of tissue cell architecture via integrated  single cell and spatial transcriptomics (cell2location model)

*homepage*: https://github.com/BayraktarLab/cell2location/

version       |toolchain
--------------|------------------
``0.05-alpha``|``fosscuda/2020b``

<a anchor="cellmix"/>
### *CellMix*

A Comprehensive Toolbox for Gene Expression Deconvolution

*homepage*: http://web.cbio.uct.ac.za/~renaud/CRAN/web/CellMix

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.6.2``|``-R-3.5.1`` |``foss/2018b``

<a anchor="cellranger"/>
### *CellRanger*

Cell Ranger is a set of analysis pipelines that process Chromium  single-cell RNA-seq output to align reads, generate gene-cell matrices and perform  clustering and gene expression analysis.

*homepage*: https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger

version  |toolchain
---------|----------
``3.0.0``|``system``
``3.0.2``|``system``
``3.1.0``|``system``
``4.0.0``|``system``
``5.0.0``|``system``
``5.0.1``|``system``
``6.0.0``|``system``
``6.0.1``|``system``
``6.0.2``|``system``
``6.1.2``|``system``
``7.0.0``|``system``

<a anchor="cellranger-arc"/>
### *CellRanger-ARC*

Cell Ranger ARC is a set of analysis pipelines that process  Chromium Single Cell Multiome ATAC + Gene Expression sequencing data to generate a  variety of analyses pertaining to gene expression, chromatin accessibility and  their linkage. Furthermore, since the ATAC and gene expression measurements are on  the very same cell, we are able to perform analyses that link chromatin  accessibility and gene expression.

*homepage*: https://support.10xgenomics.com/single-cell-multiome-atac-gex/software/pipelines/latest/what-is-cell-ranger-arc

version  |toolchain
---------|----------
``1.0.1``|``system``
``2.0.0``|``system``
``2.0.1``|``system``

<a anchor="cellranger-atac"/>
### *CellRanger-ATAC*

Cell Ranger ATAC is a set of analysis pipelines that process  Chromium Single Cell ATAC data.

*homepage*: https://support.10xgenomics.com/single-cell-atac/software/pipelines/latest/what-is-cell-ranger-atac

version  |toolchain
---------|----------
``1.2.0``|``system``
``2.0.0``|``system``
``2.1.0``|``system``

<a anchor="cellrank"/>
### *CellRank*

CellRank is a toolkit to uncover cellular dynamics based on  Markov state modeling of single-cell data.  It contains two main modules: kernels compute cell-cell transition probabilities and estimators generate hypothesis based on these. 

*homepage*: https://cellrank.readthedocs.io/en/stable/

version  |toolchain
---------|--------------
``1.4.0``|``foss/2021a``

<a anchor="celltypist"/>
### *CellTypist*

A tool for semi-automatic cell type annotation

*homepage*: https://www.celltypist.org/

version  |toolchain
---------|--------------
``1.0.0``|``foss/2021b``

<a anchor="centrifuge"/>
### *Centrifuge*

Classifier for metagenomic sequences

*homepage*: https://ccb.jhu.edu/software/centrifuge

version       |toolchain
--------------|--------------------------------
``1.0.3``     |``foss/2018b``
``1.0.4``     |``gompi/2020b``, ``gompi/2021a``
``1.0.4-beta``|``foss/2018b``, ``gompi/2020a``

<a anchor="cereal"/>
### *Cereal*

cereal is a header-only C++11 serialization library. cereal takes arbitrary data types and reversibly  turns them into different representations, such as compact binary encodings, XML, or JSON. cereal was designed to be fast, light-weight, and easy to extend - it has no external dependencies and can be easily bundled with other code or  used standalone.

*homepage*: https://uscilab.github.io/cereal/

version  |toolchain
---------|----------
``1.3.0``|``system``

<a anchor="cesm-deps"/>
### *CESM-deps*

CESM is a fully-coupled, community, global climate model that provides state-of-the-art  computer simulations of the Earth's past, present, and future climate states.

*homepage*: https://www.cesm.ucar.edu/models/cesm2/

version|toolchain
-------|----------------------------------------------------------------
``2``  |``foss/2018b``, ``foss/2021b``, ``intel/2018b``, ``iomkl/2018b``

<a anchor="cfdemcoupling"/>
### *CFDEMcoupling*

CFDEMcoupling is an open source CFD-DEM engine. It provides the possibility to couple  the DEM engine LIGGGHTS to a CFD framework.

*homepage*: https://www.cfdem.com/cfdemrcoupling-open-source-cfd-dem-framework

version  |toolchain
---------|--------------
``3.8.0``|``foss/2018a``

<a anchor="cfitsio"/>
### *CFITSIO*

CFITSIO is a library of C and Fortran subroutines for reading and writing data files in FITS (Flexible Image Transport System) data format.

*homepage*: https://heasarc.gsfc.nasa.gov/fitsio/

version |toolchain
--------|----------------------------------------------------------
``3.38``|``foss/2016a``, ``intel/2016a``
``3.41``|``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``intel/2016b``
``3.42``|``GCCcore/6.4.0``, ``intel/2017b``
``3.45``|``GCCcore/7.3.0``, ``intel/2018b``
``3.47``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``3.48``|``GCCcore/9.3.0``
``3.49``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``

<a anchor="cftime"/>
### *cftime*

Time-handling functionality from netcdf4-python

*homepage*: https://github.com/Unidata/cftime

version    |versionsuffix     |toolchain
-----------|------------------|-------------------------------
``1.0.0``  |``-Python-3.6.4`` |``intel/2018a``
``1.0.0b1``|``-Python-3.6.2`` |``foss/2017b``
``1.0.1``  |``-Python-2.7.15``|``intel/2018b``
``1.0.1``  |``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``

<a anchor="cgal"/>
### *CGAL*

The goal of the CGAL Open Source Project is to provide easy access to efficient  and reliable geometric algorithms in the form of a C++ library.

*homepage*: https://www.cgal.org/

version   |versionsuffix     |toolchain
----------|------------------|------------------------------------------------------------------
``4.8``   |``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``4.8.1`` |                  |``foss/2016b``, ``intel/2016b``
``4.8.1`` |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``4.9``   |``-Python-2.7.12``|``intel/2016b``
``4.9``   |``-Python-2.7.13``|``intel/2017a``
``4.11``  |``-Python-2.7.13``|``intel/2017a``
``4.11``  |``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``4.11``  |``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``4.11.1``|``-Python-2.7.14``|``foss/2018a``, ``intel/2018a``
``4.11.1``|``-Python-2.7.15``|``foss/2018b``
``4.11.1``|``-Python-3.6.4`` |``foss/2018a``
``4.11.1``|``-Python-3.6.6`` |``foss/2018b``
``4.14``  |``-Python-3.7.2`` |``foss/2019a``, ``intel/2019a``
``4.14.1``|``-Python-3.7.4`` |``foss/2019b``, ``intel/2019b``
``4.14.3``|                  |``gompi/2021a``, ``gompi/2021b``, ``gompi/2022a``, ``iimpi/2021a``
``4.14.3``|``-Python-3.8.2`` |``gompi/2020a``, ``iimpi/2020a``
``5.2``   |                  |``gompi/2020b``

<a anchor="cget"/>
### *cget*

Cmake package retrieval. This can be used to download and install cmake packages

*homepage*: https://cget.readthedocs.io/en/latest/index.html

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.1.6``|``-Python-3.6.4``|``foss/2018a``

<a anchor="cgl"/>
### *Cgl*

The COIN-OR Cut Generation Library (Cgl) is a collection of cut generators that can be used with other COIN-OR packages that make use of cuts, such as, among others, the linear solver Clp or the mixed integer linear programming solvers Cbc or BCP. Cgl uses the abstract class OsiSolverInterface (see Osi) to use or communicate with a solver. It does not directly call a solver.

*homepage*: https://github.com/coin-or/Cgl

version   |toolchain
----------|------------------------------
``0.60.2``|``foss/2018b``
``0.60.3``|``foss/2020b``, ``foss/2021a``

<a anchor="cgmaptools"/>
### *CGmapTools*

Command-line Toolset for Bisulfite Sequencing Data Analysis

*homepage*: https://cgmaptools.github.io/

version  |toolchain
---------|---------------
``0.1.2``|``intel/2019b``

<a anchor="cgns"/>
### *CGNS*

The CGNS system is designed to facilitate the exchange of data between sites and applications,  and to help stabilize the archiving of aerodynamic data.

*homepage*: https://cgns.github.io/

version  |toolchain
---------|-------------------
``3.3.1``|``foss/2016b``
``4.1.0``|``intelcuda/2019b``

<a anchor="charls"/>
### *CharLS*

CharLS is a C++ implementation of the JPEG-LS standard for lossless and near-lossless image compression and decompression. JPEG-LS is a low-complexity image compression standard that matches JPEG 2000 compression ratios.

*homepage*: https://github.com/team-charls/charls

version  |toolchain
---------|------------------------------------
``2.0.0``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``2.1.0``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``2.2.0``|``GCCcore/10.2.0``
``2.3.4``|``GCCcore/10.3.0``

<a anchor="charmm"/>
### *CHARMM*

CHARMM (Chemistry at HARvard Macromolecular Mechanics) is a versatile and widely used molecular simulation program with broad application to many-particle systems.

*homepage*: http://www.charmm.org

version |toolchain
--------|-------------------------------
``37b2``|``foss/2016a``, ``intel/2016a``

<a anchor="charmm"/>
### *charmm*

CHARMM (Chemistry at HARvard Macromolecular Mechanics) is a versatile and widely used molecular simulation program with broad application to many-particle systems. charmm provides all the functionality of CHARMM except its performance enhancements.

*homepage*: http://www.charmm.org

version |toolchain
--------|-------------------------------
``43b2``|``foss/2017b``, ``intel/2017b``

<a anchor="chase"/>
### *CHASE*

Case-control HAplotype Sharing analyses.   Haplotype sharing analyses for genome-wide association studies.

*homepage*: http://people.duke.edu/~asallen/Software.html

version     |toolchain
------------|----------
``20130626``|``system``

<a anchor="check"/>
### *Check*

Check is a unit testing framework for C. It features a simple interface for defining unit tests, putting little in the way of the developer. Tests are run in a separate address space, so both assertion failures and code errors that cause segmentation faults or other signals can be caught. Test results are reportable in the following: Subunit, TAP, XML, and a generic logging format.

*homepage*: https://libcheck.github.io/check/

version   |toolchain
----------|-----------------------------------------------------------------------------
``0.15.2``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``

<a anchor="check"/>
### *check*

Check is a unit testing framework for C.  It features a simple interface for defining unit tests,  putting little in the way of the developer. Tests are run  in a separate address space, so both assertion failures  and code errors that cause segmentation faults or other  signals can be caught. Test results are reportable in  the following: Subunit, TAP, XML, and a generic logging format.

*homepage*: https://libcheck.github.io/check/

version   |toolchain
----------|-----------------
``0.12.0``|``GCCcore/6.4.0``

<a anchor="checkm"/>
### *CheckM*

CheckM provides a set of tools for assessing the quality of genomes recovered from isolates,  single cells, or metagenomes.

*homepage*: https://github.com/Ecogenomics/CheckM

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``1.0.13``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``1.0.13``|``-Python-2.7.15``|``foss/2018b``
``1.0.13``|``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``1.0.18``|``-Python-2.7.15``|``foss/2019a``
``1.0.18``|``-Python-2.7.18``|``foss/2020b``
``1.1.2`` |``-Python-3.7.4`` |``foss/2019b``, ``intel/2019b``
``1.1.3`` |                  |``foss/2021a``, ``foss/2021b``
``1.1.3`` |``-Python-3.8.2`` |``intel/2020a``

<a anchor="cheetah"/>
### *Cheetah*

Cheetah is an open source template engine and code generation tool.

*homepage*: https://cheetahtemplate.org

version  |versionsuffix     |toolchain
---------|------------------|---------------
``2.4.4``|``-Python-2.7.15``|``intel/2018b``

<a anchor="chemaxon-marvin"/>
### *Chemaxon-Marvin*

Marvin suite is a chemically intelligent desktop toolkit built to help you draw, edit, publish, render, import and export your chemical structures and as well as allowing you to convert between various chemical and graphical file formats. It is free for individual, academic and non-commercial use.

*homepage*: https://chemaxon.com/products/marvin

version  |toolchain
---------|----------
``21.14``|``system``

<a anchor="chemps2"/>
### *CheMPS2*

CheMPS2 is a scientific library which contains a spin-adapted implementation of the density matrix renormalization group (DMRG) for ab initio quantum chemistry.

*homepage*: https://github.com/SebWouters/CheMPS2

version    |toolchain
-----------|---------------------------------------------------------------------------------
``1.6``    |``intel/2016a``
``1.7-rc2``|``intel/2016a``
``1.7.1``  |``intel/2016a``
``1.7.2``  |``intel/2016a``
``1.8``    |``intel/2016b``
``1.8.8``  |``intel/2018b``
``1.8.9``  |``foss/2018b``, ``foss/2019a``, ``intel/2018b``, ``intel/2019a``, ``intel/2019b``
``1.8.11`` |``foss/2021b``
``1.8.12`` |``foss/2022a``

<a anchor="cherab"/>
### *CHERAB*

CHERAB is a python library for forward modelling diagnostics  based on spectroscopic plasma emission.

*homepage*: https://cherab.github.io/documentation/index.html

version  |versionsuffix    |toolchain
---------|-----------------|-------------------------------
``1.2.0``|``-Python-3.6.6``|``intel/2018b``
``1.3.0``|                 |``foss/2020b``, ``intel/2020b``

<a anchor="chewbbaca"/>
### *chewBBACA*

chewBBACA stands for "BSR-Based Allele Calling Algorithm". chewBBACA is a comprehensive pipeline including a set of functions for the creation and validation of whole genome and core genome MultiLocus Sequence Typing (wg/cgMLST) schemas, providing an allele calling algorithm based on Blast Score Ratio that can be run in multiprocessor settings and a set of functions to visualize and validate allele variation in the loci.

*homepage*: https://github.com/B-UMMI/chewBBACA

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``2.5.5``|``-Python-3.8.2``|``intel/2020a``

<a anchor="chi2comb"/>
### *chi2comb*

Cumulative density function of linear combinations of independent chi-square random variables and a standard Normal distribution. As of now, this is basically a repackaging of the davies function implemented in the CompQuadForm library for R.

*homepage*: https://github.com/limix/chi2comb

version  |toolchain
---------|-----------------
``0.0.3``|``GCCcore/7.3.0``

<a anchor="chimera"/>
### *Chimera*

UCSF Chimera is a highly extensible program for interactive visualization   and analysis of molecular structures and related data, including density maps, supramolecular   assemblies, sequence alignments, docking results, trajectories, and conformational ensembles. 

*homepage*: https://www.cgl.ucsf.edu/chimera/

version |versionsuffix    |toolchain
--------|-----------------|----------
``1.10``|``-linux_x86_64``|``system``
``1.16``|``-linux_x86_64``|``system``

<a anchor="chimpipe"/>
### *ChimPipe*

ChimPipe is a computational method for the detection of novel transcription-induced  chimeric transcripts and fusion genes from Illumina Paired-End RNA-seq data. It combines junction  spanning and paired-end read information to accurately detect chimeric splice junctions at base-pair  resolution.

*homepage*: https://chimpipe.readthedocs.org/

version  |versionsuffix     |toolchain
---------|------------------|--------------
``0.9.5``|                  |``foss/2018b``
``0.9.5``|``-Python-2.7.12``|``foss/2016b``

<a anchor="chromaprint"/>
### *Chromaprint*

Chromaprint is the core component of the AcoustID project. It's a client-side library  that implements a custom algorithm for extracting fingerprints from any audio source.

*homepage*: https://acoustid.org/chromaprint

version  |toolchain
---------|-----------------
``1.4.3``|``GCCcore/8.2.0``

<a anchor="cicero"/>
### *cicero*

Cicero is an R package that provides tools for analyzing single-cell chromatin accessibility  experiments.

*homepage*: https://cole-trapnell-lab.github.io/cicero-release

version     |versionsuffix        |toolchain
------------|---------------------|--------------
``1.3.4.11``|``-R-4.0.3-Monocle3``|``foss/2020b``

<a anchor="cif2cell"/>
### *CIF2Cell*

CIF2Cell is a tool to generate the geometrical setup for various electronic structure codes from a CIF (Crystallographic Information Framework) file. The program currently supports output for a number of popular electronic structure programs, including ABINIT, ASE, CASTEP, CP2K, CPMD, CRYSTAL09, Elk, EMTO, Exciting, Fleur, FHI-aims, Hutsepot, MOPAC, Quantum Espresso, RSPt, Siesta, SPR-KKR, VASP. Also exports some related formats like .coo, .cfg and .xyz-files.

*homepage*: https://sourceforge.net/projects/cif2cell

version   |versionsuffix     |toolchain
----------|------------------|-----------------
``1.2.10``|``-Python-2.7.16``|``GCCcore/8.3.0``

<a anchor="circexplorer"/>
### *CIRCexplorer*

CIRCexplorer2 is a comprehensive and integrative circular RNA analysis toolset.

*homepage*: https://circexplorer2.readthedocs.io/

version   |versionsuffix     |toolchain
----------|------------------|---------------
``1.1.10``|``-Python-2.7.14``|``intel/2017b``

<a anchor="circexplorer2"/>
### *CIRCexplorer2*

CIRCexplorer2 is a comprehensive and integrative circular RNA analysis toolset.

*homepage*: http://circexplorer2.readthedocs.io/

version  |versionsuffix     |toolchain
---------|------------------|---------------
``2.3.2``|``-Python-2.7.14``|``intel/2017b``
``2.3.3``|``-Python-2.7.14``|``intel/2018a``
``2.3.8``|``-Python-2.7.18``|``foss/2020b``

<a anchor="circos"/>
### *Circos*

Circos is a software package for visualizing data and information.  It visualizes data in a circular layout - this makes Circos ideal for exploring  relationships between objects or positions.

*homepage*: http://www.circos.ca/

version   |versionsuffix   |toolchain
----------|----------------|-------------------------------------
``0.69-5``|``-Perl-5.24.0``|``foss/2016b``
``0.69-6``|``-Perl-5.26.1``|``GCCcore/6.4.0``
``0.69-6``|``-Perl-5.28.0``|``GCCcore/7.3.0``
``0.69-9``|                |``GCCcore/11.3.0``, ``GCCcore/9.3.0``

<a anchor="ciri"/>
### *CIRI*

CircRNA Identifier. A de novo circular RNA identification tool

*homepage*: https://sourceforge.net/projects/ciri/

version  |versionsuffix   |toolchain
---------|----------------|---------------
``2.0.6``|``-Perl-5.26.0``|``intel/2017b``

<a anchor="ciri-long"/>
### *CIRI-long*

Circular RNA Identification for Long-Reads Nanopore Sequencing Data

*homepage*: https://github.com/bioinfo-biols/CIRI-long/

version  |toolchain
---------|--------------
``1.0.2``|``foss/2020b``

<a anchor="cistem"/>
### *cisTEM*

cisTEM is user-friendly software to process cryo-EM images of macromolecular complexes  and obtain high-resolution 3D reconstructions from them. 

*homepage*: https://cistem.org/

version       |toolchain
--------------|--------------
``1.0.0-beta``|``foss/2018a``

<a anchor="cite-seq-count"/>
### *CITE-seq-Count*

A python package that allows to count antibody TAGS from a CITE-seq and/or cell hashing experiment.

*homepage*: https://github.com/Hoohm/CITE-seq-Count

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.4.3``|``-Python-3.6.6``|``foss/2018b``
``1.4.3``|``-Python-3.7.4``|``foss/2019b``

<a anchor="clang"/>
### *Clang*

C, C++, Objective-C compiler, based on LLVM.  Does not  include C++ standard library -- use libstdc++ from GCC.

*homepage*: https://clang.llvm.org/

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------------
``3.3``   |                  |``GCC/4.8.1``
``3.4``   |                  |``GCC/4.8.2``
``3.4.1`` |                  |``GCC/4.8.2``
``3.4.2`` |                  |``GCC/4.8.2``
``3.6.0`` |                  |``GCC/4.9.2``
``3.6.1`` |                  |``GCC/4.9.2``
``3.7.0`` |                  |``GNU/4.9.3-2.25``
``3.7.1`` |                  |``GCC/4.9.3-2.25``, ``foss/2016a``
``3.8.0`` |                  |``GCC/4.9.3-2.25``
``3.8.1`` |                  |``GCC/5.4.0-2.26``, ``foss/2016b``
``5.0.0`` |                  |``GCC/6.4.0-2.28``
``5.0.1`` |                  |``GCC/6.4.0-2.28``
``6.0.1`` |                  |``GCC/6.4.0-2.28``, ``GCC/7.3.0-2.30``
``7.0.0`` |                  |``GCC/6.4.0-2.28``
``7.0.1`` |                  |``GCC/7.3.0-2.30``
``8.0.0`` |                  |``GCCcore/8.2.0``
``8.0.0`` |``-CUDA-10.1.105``|``GCCcore/8.2.0``
``8.0.1`` |``-CUDA-10.1.105``|``GCC/8.2.0-2.31.1``
``8.0.1`` |``-CUDA-10.1.243``|``GCC/8.3.0``
``9.0.1`` |                  |``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``9.0.1`` |``-CUDA-10.1.243``|``GCC/8.3.0``
``10.0.0``|                  |``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``10.0.1``|                  |``GCCcore/9.3.0``
``11.0.0``|                  |``GCCcore/9.3.0``
``11.0.1``|                  |``GCCcore/10.2.0``, ``gcccuda/2020b``
``12.0.1``|                  |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``12.0.1``|``-CUDA-11.3.1``  |``GCCcore/10.3.0``
``13.0.1``|                  |``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``13.0.1``|``-CUDA-11.4.1``  |``GCCcore/11.2.0``

<a anchor="clang-aomp"/>
### *Clang-AOMP*

AOMP is an open source Clang/LLVM based compiler with added support for the OpenMP® API on Radeon™ GPUs.

*homepage*: https://github.com/RadeonOpenCompute/llvm-project

version  |toolchain
---------|------------------
``4.5.0``|``GCCcore/11.2.0``

<a anchor="clang-python-bindings"/>
### *Clang-Python-bindings*

Python bindings for libclang

*homepage*: https://clang.llvm.org

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------------
``8.0.0`` |``-Python-2.7.15``|``GCCcore/8.2.0``
``10.0.1``|``-Python-3.8.2`` |``GCCcore/9.3.0``
``13.0.1``|                  |``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="clapack"/>
### *CLAPACK*

C version of LAPACK

*homepage*: http://www.netlib.org/clapack

version  |toolchain
---------|---------------------------------------------------------------------------
``3.2.1``|``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``, ``intel/2017a``

<a anchor="clease"/>
### *CLEASE*

CLuster Expansion in Atomic Simulation Environment (CLEASE) is a package that automates the cumbersome setup and construction procedure of cluster expansion (CE). It provides a comprehensive list of tools for specifying parameters for CE, generating training structures, fitting effective cluster interaction (ECI) values and running Monte Carlo simulations.

*homepage*: https://gitlab.com/computationalmaterials/clease

version   |toolchain
----------|---------------
``0.10.6``|``intel/2021a``

<a anchor="clhep"/>
### *CLHEP*

The CLHEP project is intended to be a set of HEP-specific foundation and  utility classes such as random generators, physics vectors, geometry and linear algebra.  CLHEP is structured in a set of packages independent of any external package.

*homepage*: https://proj-clhep.web.cern.ch/proj-clhep/

version    |toolchain
-----------|----------------------------------------------------------------
``2.1.1.0``|``intel/2016a``
``2.1.3.1``|``intel/2016a``
``2.2.0.8``|``intel/2016a``
``2.3.1.1``|``intel/2016a``
``2.3.4.3``|``foss/2017b``, ``intel/2017b``
``2.4.0.0``|``intel/2017b``
``2.4.1.0``|``foss/2017b``, ``foss/2018b``, ``intel/2017b``, ``intel/2018b``
``2.4.1.3``|``foss/2019b``, ``foss/2020a``
``2.4.4.0``|``GCC/10.2.0``, ``GCC/11.2.0``
``2.4.5.1``|``GCC/11.2.0``

<a anchor="climetlab"/>
### *CliMetLab*

CliMetLab is a Python package aiming at simplifying access to climate and meteorological datasets, allowing users to focus on science instead of technical issues such as data access and data formats. It is mostly intended to be used in Jupyter notebooks, and be interoperable with all popular data analytic packages, such as Numpy, Pandas, Xarray, SciPy, Matplotlib, etc. as well as machine learning frameworks, such as Tensorflow, Keras or PyTorch.

*homepage*: https://climetlab.readthedocs.io/

version   |toolchain
----------|--------------
``0.12.6``|``foss/2022a``

<a anchor="clisp"/>
### *CLISP*

Common Lisp is a high-level, general-purpose, object-oriented, dynamic,  functional programming language. 

*homepage*: https://clisp.sourceforge.io/

version |toolchain
--------|------------------------------------
``2.49``|``GCCcore/6.4.0``, ``GCCcore/9.3.0``

<a anchor="clonalframeml"/>
### *ClonalFrameML*

Efficient Inference of Recombination in Whole Bacterial Genomes

*homepage*: https://github.com/xavierdidelot/ClonalFrameML

version |toolchain
--------|--------------
``1.11``|``foss/2016b``

<a anchor="cloog"/>
### *CLooG*

CLooG is a free software and library to generate code for scanning Z-polyhedra. That is, it finds a  code (e.g. in C, FORTRAN...) that reaches each integral point of one or more parameterized polyhedra. CLooG has been  originally written to solve the code generation problem for optimizing compilers based on the polytope model.  Nevertheless it is used now in various area e.g. to build control automata for high-level synthesis or to find the  best polynomial approximation of a function. CLooG may help in any situation where scanning polyhedra matters. While  the user has full control on generated code quality, CLooG is designed to avoid control overhead and to produce a  very effective code.

*homepage*: http://www.bastoul.net/cloog/index.php

version   |toolchain
----------|-------------
``0.18.1``|``GCC/4.8.2``

<a anchor="cloudcompare"/>
### *CloudCompare*

3D point cloud and mesh processing software

*homepage*: https://www.cloudcompare.org/

version   |toolchain
----------|--------------
``2.12.4``|``foss/2021b``

<a anchor="clp"/>
### *Clp*

Clp (Coin-or linear programming) is an open-source linear programming solver. It is primarily meant to be used as a callable library, but a basic, stand-alone executable version is also available.

*homepage*: https://github.com/coin-or/Clp

version   |toolchain
----------|------------------------------
``1.17.3``|``foss/2018b``
``1.17.6``|``foss/2020b``, ``foss/2021a``
``1.17.7``|``foss/2021b``

<a anchor="clustal-omega"/>
### *Clustal-Omega*

Clustal Omega is a multiple sequence alignment  program for proteins. It produces biologically meaningful multiple  sequence alignments of divergent sequences. Evolutionary relationships  can be seen via viewing Cladograms or Phylograms 

*homepage*: http://www.clustal.org/omega/

version  |toolchain
---------|-------------------------------------------------------------------------------------------------------------
``1.2.0``|``foss/2016b``
``1.2.4``|``GCC/10.3.0``, ``GCC/8.3.0``, ``foss/2018b``, ``intel-compilers/2021.2.0``, ``intel/2018a``, ``intel/2018b``

<a anchor="clustalw2"/>
### *ClustalW2*

ClustalW2 is a general purpose multiple sequence alignment program for DNA or proteins.

*homepage*: https://www.ebi.ac.uk/Tools/msa/clustalw2/

version|toolchain
-------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.1``|``GCC/10.3.0``, ``GCC/11.2.0``, ``foss/2016b``, ``foss/2018b``, ``foss/2021a``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2017b``, ``intel/2018b``, ``intel/2020a``

<a anchor="cluster-buster"/>
### *Cluster-Buster*

Cluster-Buster: Finding dense clusters of motifs in DNA sequences

*homepage*: https://github.com/weng-lab/cluster-buster/

version     |toolchain
------------|---------------
``20160106``|``intel/2016a``

<a anchor="clustershell"/>
### *ClusterShell*

ClusterShell is an event-driven open source Python library, designed to run local or distant commands  in parallel on server farms or on large Linux clusters.

*homepage*: https://cea-hpc.github.io/clustershell/

version  |toolchain
---------|----------
``1.7.3``|``system``

<a anchor="cmake"/>
### *CMake*

CMake, the cross-platform, open-source build system.  CMake is a family of tools designed to build, test and package software.

*homepage*: https://www.cmake.org

version   |toolchain
----------|-------------------------------------------------------------------------------------------------------------------------------------------------
``2.8.11``|``GCC/4.8.1``
``2.8.12``|``GCC/4.8.1``, ``GCC/4.8.2``
``3.0.0`` |``GCC/4.8.3``
``3.1.0`` |``GCC/4.9.2``
``3.1.3`` |``GCC/4.9.2``, ``system``
``3.2.1`` |``GCC/4.9.2``, ``GNU/4.9.3-2.25``
``3.3.1`` |``system``
``3.3.2`` |``GNU/4.9.3-2.25``, ``gimkl/2.11.5``
``3.4.1`` |``GCC/4.9.2``, ``GCCcore/4.9.3``, ``foss/2016a``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``
``3.4.3`` |``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016a``
``3.5.1`` |``intel/2016a``
``3.5.2`` |``GCC/4.9.3-2.25``, ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``, ``system``
``3.6.1`` |``GCC/5.4.0-2.26``, ``GCCcore/4.9.3``, ``foss/2016b``, ``intel/2016b``, ``system``
``3.6.2`` |``GCCcore/5.4.0``, ``foss/2016b``, ``intel/2016b``
``3.7.1`` |``GCCcore/5.4.0``, ``GCCcore/6.2.0``, ``foss/2016b``, ``intel/2016b``
``3.7.2`` |``GCCcore/6.3.0``, ``foss/2016b``, ``intel/2016b``
``3.8.0`` |``GCCcore/6.3.0``
``3.8.1`` |``GCCcore/6.3.0``
``3.8.2`` |``GCCcore/6.3.0``
``3.9.1`` |``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``system``
``3.9.4`` |``GCCcore/6.4.0``
``3.9.5`` |``GCCcore/6.4.0``
``3.9.6`` |``system``
``3.10.0``|``GCCcore/6.4.0``
``3.10.1``|``GCCcore/6.4.0``
``3.10.2``|``GCCcore/6.4.0``, ``GCCcore/7.2.0``
``3.10.3``|``GCCcore/6.4.0``, ``GCCcore/7.2.0``
``3.11.1``|``GCCcore/6.4.0``
``3.11.4``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``3.12.1``|``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``system``
``3.13.3``|``GCCcore/8.2.0``
``3.15.1``|``system``
``3.15.3``|``GCCcore/8.3.0``
``3.16.4``|``GCCcore/9.3.0``
``3.18.4``|``GCCcore/10.2.0``, ``system``
``3.20.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``3.21.1``|``GCCcore/11.2.0``
``3.22.1``|``GCCcore/11.2.0``
``3.23.1``|``GCCcore/11.3.0``
``3.24.3``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``

<a anchor="cmaverse"/>
### *CMAverse*

The R package CMAverse provides a suite of functions for reproducible causal mediation analysis including cmdag for DAG visualization, cmest for statistical modeling and cmsens for sensitivity analysis.

*homepage*: https://bs1125.github.io/CMAverse/

version     |toolchain
------------|--------------
``20220112``|``foss/2021b``

<a anchor="cmdstanr"/>
### *CmdStanR*

CmdStanR is a lightweight interface to Stan for R users

*homepage*: https://mc-stan.org/cmdstanr

version  |versionsuffix|toolchain
---------|-------------|--------------
``0.5.0``|``-R-4.1.2`` |``foss/2021b``

<a anchor="cmocean"/>
### *cmocean*

This package contains colormaps for commonly-used  oceanographic variables. Most of the colormaps started from  matplotlib colormaps, but have now been adjusted using the viscm  tool to be perceptually uniform.

*homepage*: https://github.com/matplotlib/cmocean

version|toolchain
-------|--------------
``2.0``|``foss/2022a``

<a anchor="cmseq"/>
### *CMSeq*

CMSeq is a set of commands to provide an interface to .bam files for coverage and sequence consensus.

*homepage*: https://github.com/SegataLab/cmseq/

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.0.3``|``-Python-3.8.2``|``foss/2020a``

<a anchor="cnt-ilp"/>
### *CNT-ILP*

Integer Linear Program for the Copy-Number Tree Problem 

*homepage*: https://compbio.cs.brown.edu/projects/cnt-ilp/

version     |toolchain
------------|--------------------
``20171031``|``GCC/8.2.0-2.31.1``

<a anchor="cnvkit"/>
### *CNVkit*

A command-line toolkit and Python library for detecting copy  number variants and alterations genome-wide from high-throughput sequencing.

*homepage*: https://github.com/etal/cnvkit

version  |versionsuffix            |toolchain
---------|-------------------------|---------------
``0.9.2``|``-Python-2.7.14``       |``intel/2017b``
``0.9.3``|``-Python-3.6.4``        |``intel/2018a``
``0.9.6``|``-Python-3.7.2-R-3.6.0``|``foss/2019a``
``0.9.8``|``-R-4.0.3``             |``foss/2020b``

<a anchor="cnvnator"/>
### *CNVnator*

a tool for CNV discovery and genotyping from depth-of-coverage by mapped reads 

*homepage*: https://github.com/abyzovlab/CNVnator

version  |toolchain
---------|--------------
``0.3.3``|``foss/2016b``

<a anchor="co-phylog"/>
### *Co-phylog*

Co-phylog: an assembly-free phylogenomic approach for closely related organisms H Yi, L Jin Nucleic acids research 41 (7), e75-e75

*homepage*: https://github.com/yhg926/co-phylog

version     |toolchain
------------|------------------
``20201012``|``GCC/7.3.0-2.30``

<a anchor="cobrapy"/>
### *COBRApy*

COBRApy is a package for constraint-based modeling of metabolic networks.

*homepage*: https://opencobra.github.io/cobrapy/

version   |toolchain
----------|--------------
``0.26.0``|``foss/2021a``

<a anchor="cocoalib"/>
### *CoCoALib*

CoCoALib is a free GPL3 C++ library for doing Computations in Commutative Algebra.

*homepage*: http://cocoa.dima.unige.it/cocoalib 

version    |toolchain
-----------|--------------------
``0.99601``|``GCC/8.2.0-2.31.1``
``0.99700``|``GCC/8.3.0``

<a anchor="codan"/>
### *CodAn*

CodAn (Coding sequence Annotator) is a computational tool designed to characterize the CDS and UTR regions on transcripts from any Eukaryote species.

*homepage*: https://github.com/pedronachtigall/CodAn

version|toolchain
-------|--------------
``1.2``|``foss/2021b``

<a anchor="code-server"/>
### *code-server*

Run VS Code on any machine anywhere and access it in the browser.

*homepage*: https://github.com/cdr/code-server

version  |toolchain
---------|----------
``3.7.3``|``system``

<a anchor="codex2"/>
### *CODEX2*

Full-spectrum copy number variation detection by high-throughput DNA sequencing

*homepage*: https://github.com/yuchaojiang/CODEX2

version     |versionsuffix|toolchain
------------|-------------|---------------
``20180227``|``-R-3.4.3`` |``intel/2017b``

<a anchor="cogent"/>
### *Cogent*

Cogent is a tool for reconstructing the coding genome using high-quality full-length transcriptome sequences. It is designed to be used on Iso-Seq data and in cases where there is no reference genome or the ref genome is highly incomplete. 

*homepage*: https://github.com/Magdoll/Cogent

version  |toolchain
---------|--------------
``8.0.0``|``foss/2021a``

<a anchor="coin"/>
### *Coin*

Coin is an OpenGL-based, 3D graphics library that has its roots in the Open Inventor 2.1 API, which Coin still is compatible with.

*homepage*: https://coin3d.github.io

version  |toolchain
---------|------------------------------
``4.0.0``|``GCC/10.3.0``, ``GCC/11.2.0``

<a anchor="coinutils"/>
### *CoinUtils*

CoinUtils (Coin-OR Utilities) is an open-source collection of classes and functions that are generally useful to more than one COIN-OR project.

*homepage*: https://github.com/coin-or/CoinUtils

version   |toolchain
----------|----------------------------------
``2.11.3``|``GCCcore/7.3.0``, ``foss/2018b``
``2.11.4``|``GCC/10.3.0``, ``GCCcore/10.2.0``
``2.11.6``|``GCC/11.2.0``

<a anchor="colossalai"/>
### *colossalai*

Colossal-AI: A Unified Deep Learning System for Big Model Era

*homepage*: https://colossalai.org/

version  |versionsuffix   |toolchain
---------|----------------|--------------
``0.1.8``|``-CUDA-11.3.1``|``foss/2021a``

<a anchor="commet"/>
### *Commet*

COMMET ("COmpare Multiple METagenomes") provides a global similarity overview between all datasets of  a large metagenomic project. 

*homepage*: https://colibread.inria.fr/software/commet/

version     |versionsuffix     |toolchain
------------|------------------|--------------
``20150415``|``-Python-2.7.11``|``foss/2016a``

<a anchor="comparem"/>
### *CompareM*

A toolbox for comparative genomics.

*homepage*: https://github.com/dparks1134/CompareM

version   |versionsuffix     |toolchain
----------|------------------|--------------
``0.0.23``|``-Python-2.7.15``|``foss/2018b``
``0.1.2`` |                  |``foss/2021b``

<a anchor="compress-raw-zlib"/>
### *Compress-Raw-Zlib*

Low-Level Interface to zlib or zlib-ng compression library

*homepage*: https://metacpan.org/pod/Compress::Raw::Zlib

version  |toolchain
---------|------------------
``2.202``|``GCCcore/11.3.0``

<a anchor="comsol"/>
### *COMSOL*

COMSOL Multiphysics is a general-purpose software platform, based on advanced numerical methods, for modeling and simulating physics-based problems.  

*homepage*: https://www.comsol.com

version      |toolchain
-------------|----------
``5.4.0.225``|``system``

<a anchor="con3f"/>
### *Con3F*

Con3F is a Python package to read, manipulate and convert force field files

*homepage*: https://github.ugent.be/CMM/con3f

version         |versionsuffix    |toolchain
----------------|-----------------|---------------
``1.0-20190329``|``-Python-3.7.2``|``intel/2019a``

<a anchor="concoct"/>
### *CONCOCT*

Clustering cONtigs with COverage and ComposiTion (CONCOCT) is a program for unsupervised binning of metagenomic contigs by using nucleotide composition, coverage data in multiple samples and linkage data from paired end reads.

*homepage*: https://concoct.readthedocs.io

version  |versionsuffix     |toolchain
---------|------------------|--------------
``1.0.0``|``-Python-2.7.14``|``foss/2017b``
``1.0.0``|``-Python-2.7.15``|``foss/2018b``
``1.0.0``|``-Python-3.6.3`` |``foss/2017b``
``1.1.0``|``-Python-2.7.15``|``foss/2019a``
``1.1.0``|``-Python-2.7.18``|``foss/2020b``

<a anchor="configparser"/>
### *configparser*

configparser is a Python library that brings the updated configparser from Python 3.5 to Python 2.6-3.5

*homepage*: https://docs.python.org/3/library/configparser.html

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``3.5.0``|``-Python-2.7.11``|``foss/2016a``
``3.5.0``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``3.5.0``|``-Python-3.5.2`` |``intel/2016b``
``3.5.0``|``-Python-3.6.3`` |``intel/2017b``

<a anchor="configurable-http-proxy"/>
### *configurable-http-proxy*

HTTP proxy for node.js including a REST API for updating the routing table.  Developed as a part of the Jupyter Hub multi-user server.

*homepage*: https://github.com/jupyterhub/configurable-http-proxy

version  |versionsuffix    |toolchain
---------|-----------------|------------------
``1.3.0``|``-nodejs-4.4.7``|``foss/2016a``
``3.1.1``|``-nodejs-8.9.4``|``foss/2017a``
``4.2.1``|                 |``GCCcore/10.2.0``
``4.4.0``|                 |``GCCcore/10.3.0``

<a anchor="conn"/>
### *CONN*

CONN is an open-source Matlab/SPM-based cross-platform software  for the computation, display, and analysis of functional connectivity Magnetic  Resonance Imaging (fcMRI). CONN is used to analyze resting state data (rsfMRI)  as well as task-related designs. 

*homepage*: https://web.conn-toolbox.org/home

version|versionsuffix    |toolchain
-------|-----------------|----------
``21a``|``-MATLAB-2021a``|``system``

<a anchor="connectomeworkbench"/>
### *ConnectomeWorkbench*

Connectome Workbench is an open source, freely available visualization  and discovery tool used to map neuroimaging data, especially data generated by the  Human Connectome Project.

*homepage*: https://www.humanconnectome.org/software/connectome-workbench

version  |versionsuffix  |toolchain
---------|---------------|--------------------------------------------------
``1.2.2``|               |``system``
``1.3.2``|               |``GCCcore/8.2.0``, ``foss/2017b``, ``intel/2017b``
``1.4.2``|``-rh_linux64``|``system``
``1.5.0``|               |``GCCcore/10.3.0``

<a anchor="control-freec"/>
### *Control-FREEC*

Copy number and genotype annotation from whole  genome and whole exome sequencing data.

*homepage*: https://github.com/BoevaLab/FREEC

version |toolchain
--------|----------------------------------------
``11.5``|``GCC/7.3.0-2.30``, ``GCC/8.2.0-2.31.1``
``11.6``|``GCC/10.2.0``

<a anchor="coordgenlibs"/>
### *CoordgenLibs*

Schrodinger-developed 2D Coordinate Generation

*homepage*: https://github.com/schrodinger/coordgenlibs

version  |toolchain
---------|------------------------------------------------------------------
``1.3.2``|``gompi/2019a``, ``iimpi/2019a``
``3.0.1``|``gompi/2019b``, ``gompi/2021a``, ``gompi/2022a``, ``iimpi/2020a``

<a anchor="coot"/>
### *Coot*

Coot is for macromolecular model building, model completion  and validation, particularly suitable for protein modelling using X-ray data.

*homepage*: http://www2.mrc-lmb.cam.ac.uk/Personal/pemsley/coot

version  |versionsuffix                              |toolchain
---------|-------------------------------------------|----------
``0.8.1``|``-binary-Linux-x86_64-rhel-6-python-gtk2``|``system``

<a anchor="core-counter"/>
### *core-counter*

Tool to check available cores and OMP threads

*homepage*: https://github.com/gjbex/core-counter

version  |toolchain
---------|----------
``1.1.1``|``system``

<a anchor="coreutils"/>
### *Coreutils*

The GNU Core Utilities are the basic file, shell and text manipulation utilities of the GNU operating system. These are the core utilities which are expected to exist on every operating system. 

*homepage*: https://www.gnu.org/software/coreutils/

version |toolchain
--------|------------------------------------
``8.23``|``GCC/4.9.2``
``8.27``|``GCCcore/5.4.0``
``8.29``|``GCCcore/6.4.0``
``8.32``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``9.0`` |``GCCcore/11.2.0``

<a anchor="corner"/>
### *corner*

Make some beautiful corner plots.

*homepage*: https://corner.readthedocs.io/en/latest/

version  |versionsuffix     |toolchain
---------|------------------|--------------
``2.0.1``|``-Python-2.7.15``|``foss/2019a``
``2.0.1``|``-Python-3.7.2`` |``foss/2019a``

<a anchor="coverage"/>
### *coverage*

Coverage.py is a tool for measuring code coverage of Python programs.  It monitors your program, noting which parts of the code have been executed,  then analyzes the source to identify code that could have been executed but was not. 

*homepage*: https://coverage.readthedocs.io

version  |versionsuffix     |toolchain
---------|------------------|--------------------------------------
``4.5.1``|``-Python-2.7.14``|``intel/2017b``
``5.5``  |                  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``

<a anchor="cowsay"/>
### *cowsay*

Configurable talking characters in ASCII art

*homepage*: https://github.com/tnalpgge/rank-amateur-cowsay

version |toolchain
--------|----------
``3.04``|``system``

<a anchor="cp2k"/>
### *CP2K*

CP2K is a freely available (GPL) program, written in Fortran 95, to perform atomistic and molecular  simulations of solid state, liquid, molecular and biological systems. It provides a general framework for different  methods such as e.g. density functional theory (DFT) using a mixed Gaussian and plane waves approach (GPW), and  classical pair and many-body potentials. 

*homepage*: https://www.cp2k.org/

version|versionsuffix|toolchain
-------|-------------|-------------------------------------------------------------------------------------------------
``3.0``|             |``intel/2016a``, ``intel/2016b``, ``intel/2017b``, ``intel/2018a``
``3.0``|``-psmp``    |``intel/2016b``
``4.1``|             |``intel/2016b``
``4.1``|``-psmp``    |``foss/2016b``
``5.1``|             |``foss/2018a``, ``foss/2020a``, ``foss/2020b``, ``intel/2017b``, ``intel/2018a``, ``intel/2020a``
``6.1``|             |``foss/2019a``, ``intel/2018a``, ``intel/2020a``
``7.1``|             |``foss/2020a``, ``foss/2020b``, ``intel/2020a``, ``intel/2020b``
``7.1``|``-psmp``    |``foss/2020b``
``8.1``|             |``foss/2020b``
``8.2``|             |``foss/2021a``, ``intel/2021a``

<a anchor="cpb"/>
### *CPB*

CPB is a novel two-step Pearson correlation based biclustering approach to mine genes that are co-regulated with   a given reference gene in order to discover genes that function in a common biological process.   In the first step, the algorithm identifies subsets of genes with high correlation, reducing false negatives   with a nonparametric filtering scheme.   In the second step, biclusters from multiple datasets are used to extract and rank gene correlation information.

*homepage*: http://tda.gatech.edu/software/cpb/index.html

version      |versionsuffix     |toolchain
-------------|------------------|--------------
``11-4-2011``|``-Python-2.7.13``|``foss/2017a``

<a anchor="cplex"/>
### *CPLEX*

IBM ILOG CPLEX Optimizer's mathematical programming technology enables  analytical decision support for improving efficiency,  reducing costs, and increasing profitability.

*homepage*: https://www.ibm.com/analytics/cplex-optimizer

version  |versionsuffix    |toolchain
---------|-----------------|-----------------
``12.9`` |                 |``GCCcore/8.2.0``
``12.10``|``-Python-3.7.4``|``GCCcore/8.3.0``

<a anchor="cppe"/>
### *CPPE*

CPPE is an open-source, light-weight C++ and Python library for Polarizable Embedding (PE)1,2 calculations. It provides an easy-to-use API to implement PE for ground-state self-consistent field (SCF) calculations and post-SCF methods. A convenient Python interface is also available.

*homepage*: https://github.com/maxscheurer/cppe

version  |toolchain
---------|--------------
``0.3.1``|``GCC/11.3.0``

<a anchor="cppunit"/>
### *CppUnit*

CppUnit is the C++ port of the famous JUnit framework for unit testing. 

*homepage*: https://freedesktop.org/wiki/Software/cppunit/

version   |toolchain
----------|----------------------------------------------------------------------------
``1.12.1``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``foss/2016a``
``1.15.1``|``GCCcore/10.3.0``, ``GCCcore/11.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

<a anchor="cppy"/>
### *cppy*

A small C++ header library which makes it easier to write Python extension modules. The primary feature is a PyObject smart pointer which automatically handles reference counting and provides convenience methods for performing common object operations.

*homepage*: https://github.com/nucleic/cppy

version  |toolchain
---------|--------------------------------------
``1.1.0``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.2.1``|``GCCcore/11.3.0``

<a anchor="cpu_features"/>
### *cpu_features*

A cross-platform C library to retrieve CPU features (such as available instructions) at runtime.

*homepage*: https://github.com/google/cpu_features

version  |toolchain
---------|------------------
``0.6.0``|``GCCcore/10.2.0``

<a anchor="cram"/>
### *cram*

Cram is a functional testing framework for command line applications.

*homepage*: https://bitheap.org/cram

version|toolchain
-------|-----------------
``0.7``|``GCCcore/8.2.0``

<a anchor="cramtools"/>
### *cramtools*

CRAMTools is a set of Java tools and APIs for efficient compression of sequence  read data. Although this is intended as a stable version the code is released as early access. Parts of the CRAMTools are experimental and may not be supported in the future.

*homepage*: https://github.com/enasequence/cramtools/

version|versionsuffix     |toolchain
-------|------------------|----------
``2.0``|``-Java-1.7.0_80``|``system``
``3.0``|``-Java-1.7.0_80``|``system``

<a anchor="craycce"/>
### *CrayCCE*

Toolchain using Cray compiler wrapper, using PrgEnv-cray (PE release: June 2019). 

*homepage*: https://pubs.cray.com/discover

version  |toolchain
---------|----------
``19.06``|``system``

<a anchor="craygnu"/>
### *CrayGNU*

Toolchain using Cray compiler wrapper, using PrgEnv-gnu module (PE release: June 2019). 

*homepage*: https://pubs.cray.com/discover

version  |toolchain
---------|----------
``19.06``|``system``

<a anchor="crayintel"/>
### *CrayIntel*

Toolchain using Cray compiler wrapper, using PrgEnv-intel (PE release: June 2019). 

*homepage*: https://pubs.cray.com/discover

version  |toolchain
---------|----------
``19.06``|``system``

<a anchor="craypgi"/>
### *CrayPGI*

Toolchain using Cray compiler wrapper, PrgEnv-pgi compiler (PE release: June 2019). 

*homepage*: https://pubs.cray.com/discover

version  |toolchain
---------|----------
``19.06``|``system``

<a anchor="crb-blast"/>
### *crb-blast*

Conditional Reciprocal Best BLAST - high confidence ortholog assignment.  CRB-BLAST is a novel method for finding orthologs between one set of sequences and another.  This is particularly useful in genome and transcriptome annotation.

*homepage*: https://github.com/cboursnell/crb-blast

version  |versionsuffix  |toolchain
---------|---------------|-------------------------------------------------
``0.6.9``|               |``gompi/2021a``, ``gompi/2021b``, ``intel/2017a``
``0.6.9``|``-Ruby-2.6.1``|``foss/2018b``

<a anchor="crest"/>
### *CREST*

CREST is an utility/driver program for the xtb program. Originally it was designed  as conformer sampling program, hence the abbreviation Conformer–Rotamer Ensemble Sampling Tool,  but now offers also some utility functions for calculations with the GFNn–xTB methods. Generally  the program functions as an IO based OMP scheduler (i.e., calculations are performed by the xtb  program) and tool for the creation and analysation of structure ensembles. 

*homepage*: https://xtb-docs.readthedocs.io/en/latest/crest.html

version |toolchain
--------|---------------
``2.11``|``intel/2021a``

<a anchor="crf++"/>
### *CRF++*

CRF++ is a simple, customizable, and open source implementation of  Conditional Random Fields (CRFs) for segmenting/labeling sequential data. CRF++ is  designed for generic purpose and will be applied to a variety of NLP tasks, such as  Named Entity Recognition, Information Extraction and Text Chunking. 

*homepage*: https://taku910.github.io/crfpp/

version |toolchain
--------|---------------------------------------------------------
``0.58``|``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2018b``

<a anchor="crispr-dav"/>
### *CRISPR-DAV*

CRISPR-DAV is a pipeline to analyze amplicon-based NGS data of CRISPR clones in a high throughput manner.

*homepage*: https://github.com/pinetree1/crispr-dav/

version  |toolchain
---------|--------------
``2.3.4``|``foss/2020b``

<a anchor="crispresso2"/>
### *CRISPResso2*

CRISPResso2 is a software pipeline designed to enable rapid and intuitive interpretation of genome editing experiments.

*homepage*: https://github.com/pinellolab/CRISPResso2/

version   |versionsuffix     |toolchain
----------|------------------|--------------
``2.0.44``|``-Python-2.7.16``|``foss/2019b``
``2.1.2`` |``-Python-2.7.18``|``foss/2020b``
``2.2.1`` |                  |``foss/2020b``

<a anchor="cromwell"/>
### *cromwell*

Cromwell is a Workflow Management System geared towards scientific workflows.

*homepage*: http://cromwell.readthedocs.io/

version|versionsuffix|toolchain
-------|-------------|----------
``56`` |``-Java-11`` |``system``

<a anchor="crossmap"/>
### *CrossMap*

CrossMap is a program for genome coordinates conversion  between different assemblies (such as hg18 (NCBI36) <=> hg19 (GRCh37)).  It supports commonly used file formats including BAM, CRAM, SAM, Wiggle,  BigWig, BED, GFF, GTF and VCF.

*homepage*: http://crossmap.sourceforge.net

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.3.9``|``-Python-3.7.2``|``foss/2019a``

<a anchor="crosstalkz"/>
### *CrossTalkZ*

CrossTalkZ is a statistical method and software to assess the significance of crosstalk enrichment  between pairs of gene or protein groups in large biological networks. 

*homepage*: http://sonnhammer.sbc.su.se/download/software/CrossTalkZ/

version|toolchain
-------|--------------
``1.4``|``foss/2016a``

<a anchor="crpropa"/>
### *CRPropa*

CRPropa is a publicly available code to study the propagation of ultra high energy nuclei up to iron  on their voyage through an extra galactic environment.

*homepage*: https://crpropa.desy.de

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``3.1.5``|``-Python-3.7.2``|``foss/2019a``
``3.1.6``|``-Python-3.8.2``|``foss/2020a``

<a anchor="crumble"/>
### *Crumble*

Exploration of controlled loss of quality values for compressing CRAM files

*homepage*: https://github.com/jkbonfield/crumble

version  |toolchain
---------|--------------
``0.8.3``|``GCC/11.2.0``

<a anchor="cryocare"/>
### *cryoCARE*

This package is a memory efficient implementation of cryoCARE.  This setup trains a denoising U-Net for tomographic reconstruction according to   the Noise2Noise training paradigm. Therefore the user has to provide two        tomograms of the same sample. The simplest way to achieve this is with direct-   detector movie-frames.  You can use Warp to generate two reconstructed tomograms based on the even/odd    frames. Alternatively, the movie-frames can be split in two halves (e.g. with    MotionCor2 -SplitSum 1 or with IMOD alignframes -debug 10000) from which two     identical, up to random noise, tomograms can be reconstructed.  These two (even and odd) tomograms can be used as input to this cryoCARE         implementation.

*homepage*: https://github.com/juglab/cryoCARE_pip

version  |versionsuffix   |toolchain
---------|----------------|--------------
``0.2.1``|``-CUDA-11.3.1``|``foss/2021a``

<a anchor="cryodrgn"/>
### *cryoDRGN*

cryoDRGN: Deep Reconstructing Generative Networks for cryo-EM heterogeneous  reconstruction.  CryoDRGN is a neural network based algorithm for heterogeneous cryo-EM reconstruction. In  particular, the method models a continuous distribution over 3D structures by using a neural  network based representation for the volume. 

*homepage*: https://cb.csail.mit.edu/cb/cryodrgn/

version       |versionsuffix   |toolchain
--------------|----------------|------------------
``0.3.2``     |                |``fosscuda/2020b``
``0.3.5``     |``-CUDA-11.3.1``|``foss/2021a``
``1.0.0-beta``|``-CUDA-11.3.1``|``foss/2021a``

<a anchor="cryptominisat"/>
### *CryptoMiniSat*

CryptoMiniSat is an advanced SAT solver

*homepage*: https://github.com/msoos/cryptominisat

version  |versionsuffix     |toolchain
---------|------------------|---------------
``5.0.1``|``-Python-2.7.12``|``intel/2016b``

<a anchor="crystfel"/>
### *CrystFEL*

CrystFEL is a suite of programs for processing diffraction data acquired "serially" in a "snapshot" manner, such as when using the technique of Serial Femtosecond Crystallography (SFX) with a free-electron laser source.

*homepage*: http://www.desy.de/~twhite/crystfel/

version  |toolchain
---------|-------------------------------
``0.8.0``|``foss/2019a``, ``intel/2019a``

<a anchor="csb"/>
### *CSB*

Computational Structural Biology Toolbox CSB is a python library and application framework, which can be used to solve    problems in the field of structural bioinformatics. If you are a                 bioinformatician, software engineer or a researcher working in this field,       chances are you may find something useful here. Our package consists of a few    major components: 1. Core class library - object-oriented, granular, with an emphasis on design    and clean interfaces. 2. Application framework - console applications ("protocols"), which consume     objects from the core library in order to build something executable (and        hopefully useful). 3. Test framework - ensures that the library actually works. 

*homepage*: https://github.com/csb-toolbox

version  |toolchain
---------|--------------
``1.2.5``|``foss/2020b``

<a anchor="csbdeep"/>
### *CSBDeep*

CSBDeep is a toolbox for Content-aware Image Restoration (CARE).

*homepage*: https://csbdeep.bioimagecomputing.com/

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------
``0.4.1``|``-Python-3.7.2``|``foss/2019a``, ``fosscuda/2019a``

<a anchor="csblast"/>
### *CSBLAST*

Context-specific extension of BLAST that significantly improves sensitivity and alignment quality.

*homepage*: https://github.com/soedinglab/csblast/

version  |toolchain
---------|-----------------
``2.2.3``|``GCCcore/8.3.0``

<a anchor="cscope"/>
### *cscope*

Cscope is a developer's tool for browsing source code.

*homepage*: http://cscope.sourceforge.net/

version |toolchain
--------|----------
``15.9``|``system``

<a anchor="csvkit"/>
### *csvkit*

csvkit is a suite of command-line tools for converting to and working with CSV,  the king of tabular file formats.

*homepage*: https://github.com/wireservice/csvkit

version  |versionsuffix    |toolchain
---------|-----------------|-----------------
``1.0.4``|                 |``GCCcore/8.2.0``
``1.0.5``|``-Python-3.7.4``|``GCCcore/8.3.0``

<a anchor="ctags"/>
### *ctags*

Ctags generates an index (or tag) file of language objects found in source files that allows these  items to be quickly and easily located by a text editor or other utility.

*homepage*: http://ctags.sourceforge.net/

version|toolchain
-------|----------
``5.8``|``system``

<a anchor="ctffind"/>
### *ctffind*

Program for finding CTFs of electron micrographs.

*homepage*: https://grigoriefflab.umassmed.edu/ctffind4

version   |versionsuffix   |toolchain
----------|----------------|--------------------------------------------------------------------------------------
``4.1.13``|                |``foss/2019a``, ``fosscuda/2019a``, ``fosscuda/2019b``
``4.1.14``|                |``foss/2019b``, ``foss/2021b``, ``foss/2022a``, ``fosscuda/2019b``, ``fosscuda/2020b``
``4.1.14``|``-CUDA-11.3.1``|``foss/2021a``

<a anchor="ctpl"/>
### *CTPL*

Modern and efficient C++ Thread Pool Library

*homepage*: https://github.com/vit-vit/CTPL.git

version  |toolchain
---------|--------------
``0.0.2``|``GCC/11.3.0``

<a anchor="cube"/>
### *Cube*

Cube, which is used as performance report explorer for Scalasca and  Score-P, is a generic tool for displaying a multi-dimensional performance space  consisting of the dimensions (i) performance metric, (ii) call path, and (iii) system  resource. Each dimension can be represented as a tree, where non-leaf nodes of the tree  can be collapsed or expanded to achieve the desired level of granularity.

*homepage*: http://www.scalasca.org/software/cube-4.x/download.html

version  |toolchain
---------|--------------
``4.3.4``|``foss/2016a``

<a anchor="cubegui"/>
### *CubeGUI*

Cube, which is used as performance report explorer for Scalasca and Score-P,  is a generic tool for displaying a multi-dimensional performance space  consisting of the dimensions (i) performance metric, (ii) call path, and  (iii) system resource. Each dimension can be represented as a tree, where  non-leaf nodes of the tree can be collapsed or expanded to achieve the  desired level of granularity.   This module provides the Cube graphical report explorer. 

*homepage*: https://www.scalasca.org/software/cube-4.x/download.html

version  |toolchain
---------|--------------------------------------
``4.4.4``|``GCCcore/8.2.0``, ``GCCcore/9.3.0``
``4.6``  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``

<a anchor="cubelib"/>
### *CubeLib*

Cube, which is used as performance report explorer for Scalasca and Score-P,  is a generic tool for displaying a multi-dimensional performance space  consisting of the dimensions (i) performance metric, (ii) call path, and  (iii) system resource. Each dimension can be represented as a tree, where  non-leaf nodes of the tree can be collapsed or expanded to achieve the  desired level of granularity.   This module provides the Cube general purpose C++ library component and  command-line tools. 

*homepage*: https://www.scalasca.org/software/cube-4.x/download.html

version  |toolchain
---------|-------------------------------------------------------
``4.4.4``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``4.6``  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``

<a anchor="cubewriter"/>
### *CubeWriter*

Cube, which is used as performance report explorer for Scalasca and Score-P,  is a generic tool for displaying a multi-dimensional performance space  consisting of the dimensions (i) performance metric, (ii) call path, and  (iii) system resource. Each dimension can be represented as a tree, where  non-leaf nodes of the tree can be collapsed or expanded to achieve the  desired level of granularity.   This module provides the Cube high-performance C writer library component. 

*homepage*: https://www.scalasca.org/software/cube-4.x/download.html

version  |toolchain
---------|---------------------------------------------------------------------------
``4.4.3``|``GCCcore/10.2.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``4.6``  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``

<a anchor="cuda"/>
### *CUDA*

CUDA (formerly Compute Unified Device Architecture) is a parallel  computing platform and programming model created by NVIDIA and implemented by the  graphics processing units (GPUs) that they produce. CUDA gives developers access  to the virtual instruction set and memory of the parallel computational elements in CUDA GPUs.

*homepage*: https://developer.nvidia.com/cuda-toolkit

version          |toolchain
-----------------|--------------------------------------------------------------------------
``5.5.22``       |``GCC/4.8.2``, ``system``
``6.0.37``       |``system``
``6.5.14``       |``system``
``7.0.28``       |``system``
``7.5.18``       |``GCC/4.9.4-2.25``, ``iccifort/2016.3.210-GCC-4.9.3-2.25``, ``system``
``8.0.44``       |``GCC/5.4.0-2.26``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``, ``system``
``8.0.61``       |``system``
``8.0.61_375.26``|``GCC/5.4.0-2.26``
``9.0.176``      |``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``, ``system``
``9.1.85``       |``GCC/6.4.0-2.28``, ``system``
``9.2.88``       |``GCC/6.4.0-2.28``, ``GCC/7.3.0-2.30``, ``system``
``9.2.148.1``    |``system``
``10.0.130``     |``system``
``10.1.105``     |``GCC/8.2.0-2.31.1``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``system``
``10.1.168``     |``system``
``10.1.243``     |``GCC/8.3.0``, ``iccifort/2019.5.281``, ``system``
``10.2.89``      |``GCC/8.3.0``
``11.0.2``       |``GCC/9.3.0``, ``iccifort/2020.1.217``
``11.1.1``       |``GCC/10.2.0``, ``iccifort/2020.4.304``
``11.3.1``       |``system``
``11.4.1``       |``system``
``11.4.2``       |``system``
``11.5.0``       |``system``
``11.5.1``       |``system``
``11.6.0``       |``system``
``11.7.0``       |``system``
``11.8.0``       |``system``
``12.0.0``       |``system``

<a anchor="cuda-samples"/>
### *CUDA-Samples*

Samples for CUDA Developers which demonstrates features in CUDA Toolkit

*homepage*: https://github.com/NVIDIA/cuda-samples

version |versionsuffix   |toolchain
--------|----------------|--------------
``11.3``|``-CUDA-11.3.1``|``GCC/10.3.0``

<a anchor="cudacompat"/>
### *CUDAcompat*

Using the CUDA Forward Compatibility package,  system administrators can run applications built using a newer toolkit  even when an older driver that does not satisfy the minimum required driver version  is installed on the system.  This forward compatibility allows the CUDA deployments in data centers and enterprises  to benefit from the faster release cadence and the latest features and performance of CUDA Toolkit. 

*homepage*: https://docs.nvidia.com/deploy/cuda-compatibility/index.html

version |versionsuffix |toolchain
--------|--------------|----------
``11``  |              |``system``
``11.6``|              |``system``
``11.6``|``-510.85.02``|``system``
``11.7``|              |``system``
``11.7``|``-515.65.01``|``system``

<a anchor="cudacore"/>
### *CUDAcore*

CUDA (formerly Compute Unified Device Architecture) is a parallel  computing platform and programming model created by NVIDIA and implemented by the  graphics processing units (GPUs) that they produce. CUDA gives developers access  to the virtual instruction set and memory of the parallel computational elements in CUDA GPUs.

*homepage*: https://developer.nvidia.com/cuda-toolkit

version   |toolchain
----------|----------
``11.0.2``|``system``
``11.1.1``|``system``
``11.2.1``|``system``
``11.2.2``|``system``
``11.3.0``|``system``
``11.4.0``|``system``
``11.5.1``|``system``

<a anchor="cudnn"/>
### *cuDNN*

The NVIDIA CUDA Deep Neural Network library (cuDNN) is a GPU-accelerated library of primitives for deep neural networks.

*homepage*: https://developer.nvidia.com/cudnn

version      |versionsuffix     |toolchain
-------------|------------------|-----------------------------------------------------------
``4.0``      |                  |``system``
``5.0``      |``-CUDA-7.5.18``  |``system``
``5.0-rc``   |                  |``system``
``5.1``      |``-CUDA-8.0.44``  |``system``
``6.0``      |``-CUDA-8.0.61``  |``system``
``6.0.21``   |``-CUDA-7.5.18``  |``system``
``6.0.21``   |``-CUDA-8.0.44``  |``system``
``7.0.2``    |``-CUDA-9.0.176`` |``system``
``7.0.5``    |``-CUDA-8.0.44``  |``system``
``7.0.5``    |``-CUDA-9.0.176`` |``system``
``7.0.5``    |``-CUDA-9.1.85``  |``system``
``7.0.5.15`` |                  |``fosscuda/2017b``, ``fosscuda/2018a``, ``intelcuda/2017b``
``7.1.4.18`` |                  |``fosscuda/2018b``
``7.4.2.24`` |                  |``gcccuda/2019a``
``7.4.2.24`` |``-CUDA-10.0.130``|``system``
``7.5.0.56`` |``-CUDA-10.0.130``|``system``
``7.6.2.24`` |``-CUDA-10.1.243``|``system``
``7.6.4.38`` |                  |``gcccuda/2019a``, ``gcccuda/2019b``
``7.6.4.38`` |``-CUDA-10.0.130``|``system``
``8.0.4.30`` |``-CUDA-11.0.2``  |``system``
``8.0.4.30`` |``-CUDA-11.1.1``  |``system``
``8.0.5.39`` |``-CUDA-11.1.1``  |``system``
``8.1.0.77`` |``-CUDA-11.2.1``  |``system``
``8.1.1.33`` |``-CUDA-11.2.1``  |``system``
``8.2.1.32`` |``-CUDA-11.3.1``  |``system``
``8.2.2.26`` |``-CUDA-11.4.0``  |``system``
``8.2.2.26`` |``-CUDA-11.4.1``  |``system``
``8.4.0.27`` |``-CUDA-11.6.0``  |``system``
``8.4.1.50`` |``-CUDA-11.6.0``  |``system``
``8.4.1.50`` |``-CUDA-11.7.0``  |``system``
``8.6.0.163``|``-CUDA-11.8.0``  |``system``
``8.7.0.84`` |``-CUDA-11.8.0``  |``system``

<a anchor="cufflinks"/>
### *Cufflinks*

Transcript assembly, differential expression, and differential regulation for RNA-Seq

*homepage*: http://cole-trapnell-lab.github.io/cufflinks/

version     |toolchain
------------|-------------------------------------------------------------------------------------------------
``2.2.1``   |``foss/2016a``, ``foss/2016b``, ``foss/2018b``, ``gompi/2019b``, ``intel/2017b``, ``intel/2018a``
``20190706``|``GCC/10.2.0``, ``GCC/11.2.0``, ``gompi/2019a``

<a anchor="cunit"/>
### *CUnit*

Automated testing framework for C.

*homepage*: https://sourceforge.net/projects/cunit/

version  |toolchain
---------|---------------------------------------------------------
``2.1-3``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/6.4.0``

<a anchor="cupy"/>
### *CuPy*

CuPy is an open-source array library accelerated with NVIDIA CUDA.

*homepage*: https://cupy.dev

version  |versionsuffix    |toolchain
---------|-----------------|------------------
``8.2.0``|``-Python-3.7.4``|``fosscuda/2019b``
``8.5.0``|                 |``fosscuda/2020b``

<a anchor="curl"/>
### *cURL*

libcurl is a free and easy-to-use client-side URL transfer library,  supporting DICT, FILE, FTP, FTPS, Gopher, HTTP, HTTPS, IMAP, IMAPS, LDAP,  LDAPS, POP3, POP3S, RTMP, RTSP, SCP, SFTP, SMTP, SMTPS, Telnet and TFTP.  libcurl supports SSL certificates, HTTP POST, HTTP PUT, FTP uploading, HTTP  form based upload, proxies, cookies, user+password authentication (Basic,  Digest, NTLM, Negotiate, Kerberos), file transfer resume, http proxy tunneling  and more. 

*homepage*: https://curl.haxx.se

version   |toolchain
----------|-----------------------------------------------------------------------------------
``7.33.0``|``GCC/4.8.2``
``7.34.0``|``GCC/4.8.2``
``7.40.0``|``GCC/4.9.2``
``7.46.0``|``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``
``7.47.0``|``foss/2016a``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``
``7.49.1``|``GCCcore/5.4.0``, ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``7.52.1``|``gimkl/2017a``, ``intel/2016b``
``7.53.1``|``GCCcore/6.3.0``
``7.54.0``|``GCCcore/6.3.0``
``7.55.1``|``GCCcore/6.4.0``
``7.56.0``|``GCCcore/6.4.0``
``7.56.1``|``GCCcore/6.4.0``
``7.58.0``|``GCCcore/6.4.0``
``7.59.0``|``GCCcore/6.4.0``
``7.60.0``|``GCCcore/7.2.0``, ``GCCcore/7.3.0``
``7.63.0``|``GCCcore/8.2.0``
``7.66.0``|``GCCcore/8.3.0``
``7.69.1``|``GCCcore/9.3.0``
``7.72.0``|``GCCcore/10.2.0``
``7.76.0``|``GCCcore/10.3.0``
``7.78.0``|``GCCcore/11.2.0``
``7.83.0``|``GCCcore/11.3.0``
``7.84.0``|``GCCcore/12.1.0``
``7.86.0``|``GCCcore/12.2.0``

<a anchor="custodian"/>
### *custodian*

A simple JIT job management framework in Python.

*homepage*: https://pypi.python.org/pypi/custodian

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.1.0``|``-Python-2.7.13``|``intel/2017a``

<a anchor="cutadapt"/>
### *cutadapt*

Cutadapt finds and removes adapter sequences, primers, poly-A tails and  other types of unwanted sequence from your high-throughput sequencing reads.

*homepage*: https://opensource.scilifelab.se/projects/cutadapt/

version  |versionsuffix     |toolchain
---------|------------------|------------------------------------------------
``1.9.1``|``-Python-2.7.11``|``foss/2016a``
``1.9.1``|``-Python-2.7.12``|``foss/2016b``
``1.14`` |``-Python-2.7.13``|``foss/2017a``, ``intel/2017a``
``1.15`` |``-Python-3.5.2`` |``foss/2016b``
``1.16`` |``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``, ``intel/2018a``
``1.16`` |``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``1.16`` |``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``1.18`` |                  |``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``1.18`` |``-Python-2.7.15``|``foss/2018b``
``1.18`` |``-Python-2.7.18``|``GCC/10.2.0``
``1.18`` |``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``2.1``  |``-Python-3.6.6`` |``foss/2018b``
``2.7``  |``-Python-3.7.4`` |``GCCcore/8.3.0``
``2.8``  |``-Python-3.7.4`` |``GCCcore/8.3.0``
``2.10`` |``-Python-3.7.4`` |``GCCcore/8.3.0``
``2.10`` |``-Python-3.8.2`` |``GCCcore/9.3.0``
``3.4``  |                  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``3.5``  |                  |``GCCcore/11.2.0``
``4.2``  |                  |``GCCcore/11.3.0``

<a anchor="cutensor"/>
### *cuTENSOR*

The cuTENSOR Library is a GPU-accelerated tensor linear algebra library providing tensor contraction,  reduction and elementwise operations.

*homepage*: https://developer.nvidia.com/cutensor

version    |versionsuffix   |toolchain
-----------|----------------|-----------------
``1.2.2.5``|                |``gcccuda/2019b``
``1.2.2.5``|``-CUDA-11.1.1``|``system``
``1.6.0.3``|``-CUDA-11.3.1``|``system``
``1.6.1.5``|``-CUDA-11.7.0``|``system``

<a anchor="cvs"/>
### *CVS*

CVS is a version control system, an important component of  Source Configuration Management (SCM).

*homepage*: https://savannah.nongnu.org/projects/cvs

version    |toolchain
-----------|-----------------------------------------------------------------------
``1.11.23``|``GCC/4.8.2``, ``GCCcore/11.2.0``, ``GCCcore/4.9.3``, ``GCCcore/6.4.0``

<a anchor="cvxopt"/>
### *CVXOPT*

CVXOPT is a free software package for convex optimization based on the Python programming language.  Its main purpose is to make the development of software for convex optimization applications straightforward by  building on Python's extensive standard library and on the strengths of Python as a high-level programming language. 

*homepage*: https://cvxopt.org

version  |versionsuffix     |toolchain
---------|------------------|------------------------------
``1.1.9``|``-Python-2.7.13``|``intel/2017a``
``1.2.1``|``-Python-3.6.4`` |``intel/2018a``
``1.2.3``|                  |``foss/2019a``
``1.2.3``|``-Python-3.6.6`` |``intel/2018b``
``1.2.4``|``-Python-3.7.4`` |``intel/2019b``
``1.2.6``|                  |``foss/2020b``, ``foss/2021a``

<a anchor="cvxpy"/>
### *CVXPY*

CVXPY is a Python-embedded modeling language for convex optimization problems.  It allows you to express your problem in a natural way that follows the math,  rather than in the restrictive standard form required by solvers. 

*homepage*: https://www.cvxpy.org/

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``1.0.24``|                 |``foss/2019a``
``1.0.28``|``-Python-3.7.4``|``foss/2019b``

<a anchor="cwipi"/>
### *CWIPI*

CWIPI (Coupling With Interpolation Parallel Interface) library helps in chaining and coupling codes. Provides exchanges of interpolated fields through a non-compliant geometric interface and allows control of the coupling algorithm using control parameters. CWIPI takes advantage of the distribution of the definition of the coupling algorithm in the different codes. 

*homepage*: https://w3.onera.fr/cwipi/

version   |toolchain
----------|---------------
``0.12.0``|``gompi/2021a``

<a anchor="cwltool"/>
### *cwltool*

Common workflow language (CWL) reference implementation.

*homepage*: https://github.com/common-workflow-language/cwltool

version               |toolchain
----------------------|--------------
``3.1.20221018083734``|``foss/2021a``

<a anchor="cxxopts"/>
### *cxxopts*

cxxopts is a lightweight C++ command line option parser

*homepage*: https://github.com/jarro2783/cxxopts

version  |toolchain
---------|----------
``3.0.0``|``system``

<a anchor="cysignals"/>
### *cysignals*

The cysignals package provides mechanisms to handle  interrupts (and other signals and errors) in Cython code.

*homepage*: https://pypi.org/project/cysignals/

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``1.10.2``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``1.10.2``|``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``

<a anchor="cython"/>
### *Cython*

Cython is an optimising static compiler for both the Python programming language and the extended Cython programming language (based on Pyrex). 

*homepage*: https://cython.org/

version    |versionsuffix     |toolchain
-----------|------------------|-------------------------------
``0.23.4`` |``-Python-2.7.10``|``gimkl/2.11.5``
``0.24.1`` |``-Python-2.7.11``|``foss/2016a``
``0.25.2`` |``-Python-2.7.12``|``foss/2016b``
``0.25.2`` |``-Python-3.6.4`` |``intel/2018a``
``0.27.3`` |``-Python-2.7.15``|``GCCcore/8.2.0``
``0.29.10``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``0.29.10``|``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``0.29.22``|                  |``GCCcore/10.2.0``
``3.0a5``  |                  |``GCCcore/10.2.0``

<a anchor="cytoolz"/>
### *cytoolz*

Cython implementation of the toolz package, which provides high performance utility functions  for iterables, functions, and dictionaries.

*homepage*: https://github.com/pytoolz/cytoolz

version   |versionsuffix    |toolchain
----------|-----------------|-----------------
``0.10.1``|``-Python-3.6.6``|``foss/2018b``
``0.10.1``|``-Python-3.7.2``|``GCCcore/8.2.0``

<a anchor="cytoscape"/>
### *Cytoscape*

Cytoscape is an open source software platform for visualizing  complex networks and integrating these with any type of attribute data.  A lot of Apps are available for various kinds of problem domains,  including bioinformatics, social network analysis, and semantic web.

*homepage*: https://cytoscape.org/

version  |versionsuffix|toolchain
---------|-------------|----------
``3.9.1``|``-Java-11`` |``system``

<a anchor="cytosim"/>
### *cytosim*

Cytosim is a cytoskeleton simulation engine written in  C++ working on Mac OS, GNU/Linux and Windows (with Cygwin).

*homepage*: https://github.com/nedelec/cytosim

version     |versionsuffix|toolchain
------------|-------------|---------------
``20190117``|``-mkl``     |``gomkl/2019a``

<a anchor="cyvcf2"/>
### *cyvcf2*

cython + htslib == fast VCF and BCF processing

*homepage*: https://github.com/brentp/cyvcf2

version    |versionsuffix    |toolchain
-----------|-----------------|-------------------------------
``0.10.10``|``-Python-3.6.6``|``foss/2018b``
``0.11.5`` |                 |``foss/2019a``, ``intel/2019a``

<a anchor="d"/>
### *D*


<a href="#dadi">dadi</a> - <a href="#dagitty">dagitty</a> - <a href="#dakota">Dakota</a> - <a href="#dalton">Dalton</a> - <a href="#damageproto">damageproto</a> - <a href="#dammit">dammit</a> - <a href="#danpos2">DANPOS2</a> - <a href="#das_tool">DAS_Tool</a> - <a href="#dask">dask</a> - <a href="#datamash">datamash</a> - <a href="#davix">davix</a> - <a href="#db">DB</a> - <a href="#db_file">DB_File</a> - <a href="#dbd-mysql">DBD-mysql</a> - <a href="#dbg2olc">DBG2OLC</a> - <a href="#dbus">DBus</a> - <a href="#dbus-glib">dbus-glib</a> - <a href="#dcm2niix">dcm2niix</a> - <a href="#dcmtk">DCMTK</a> - <a href="#dd">dd</a> - <a href="#deal.ii">deal.II</a> - <a href="#deap">deap</a> - <a href="#decona">decona</a> - <a href="#deconf">deconf</a> - <a href="#deconica">DeconICA</a> - <a href="#deepdiff">deepdiff</a> - <a href="#deeplabcut">DeepLabCut</a> - <a href="#deepmedic">deepmedic</a> - <a href="#deepmod2">DeepMod2</a> - <a href="#deepsurv">DeepSurv</a> - <a href="#deeptools">deepTools</a> - <a href="#delly">Delly</a> - <a href="#demixt">DeMixT</a> - <a href="#dendropy">DendroPy</a> - <a href="#destiny">destiny</a> - <a href="#detectron2">Detectron2</a> - <a href="#detonate">detonate</a> - <a href="#devito">Devito</a> - <a href="#dfa">DFA</a> - <a href="#dft-d3">DFT-D3</a> - <a href="#dft-d4">DFT-D4</a> - <a href="#dftb+">DFTB+</a> - <a href="#dftd3-lib">dftd3-lib</a> - <a href="#dgl">DGL</a> - <a href="#dial">DIAL</a> - <a href="#diamond">DIAMOND</a> - <a href="#dice">Dice</a> - <a href="#dicom2nifti">dicom2nifti</a> - <a href="#dicombrowser">DicomBrowser</a> - <a href="#diffutils">Diffutils</a> - <a href="#dijitso">dijitso</a> - <a href="#dill">dill</a> - <a href="#dirac">DIRAC</a> - <a href="#distributed">distributed</a> - <a href="#distributedstream">DistributedStream</a> - <a href="#dl_poly_4">DL_POLY_4</a> - <a href="#dl_poly_classic">DL_POLY_Classic</a> - <a href="#dlb">dlb</a> - <a href="#dlib">dlib</a> - <a href="#dm-reverb">dm-reverb</a> - <a href="#dm-tree">dm-tree</a> - <a href="#dmcfun">DMCfun</a> - <a href="#dmtcp">DMTCP</a> - <a href="#dolfin">DOLFIN</a> - <a href="#doris">Doris</a> - <a href="#dosageconvertor">DosageConvertor</a> - <a href="#dotnet-core-runtime">dotNET-Core-Runtime</a> - <a href="#dotnet-sdk">dotNET-SDK</a> - <a href="#double-conversion">double-conversion</a> - <a href="#doubletfinder">DoubletFinder</a> - <a href="#doxygen">Doxygen</a> - <a href="#dragmap">DRAGMAP</a> - <a href="#drake">Drake</a> - <a href="#drep">dRep</a> - <a href="#drop">DROP</a> - <a href="#dropest">dropEst</a> - <a href="#dsa">DSA</a> - <a href="#dsfmt">dSFMT</a> - <a href="#dsrc">DSRC</a> - <a href="#dsuite">Dsuite</a> - <a href="#dtcmp">dtcmp</a> - <a href="#dtcwt">dtcwt</a> - <a href="#dualsphysics">DualSPHysics</a> - <a href="#dubstepr">DUBStepR</a> - <a href="#dune-core">dune-core</a> - <a href="#dune-fem">dune-fem</a> - <a href="#dxpy">dxpy</a> - <a href="#dymat">DyMat</a>


<a anchor="dadi"/>
### *dadi*

∂a∂i implements methods for demographic history and selection inference from genetic data,  based on diffusion approximations to the allele frequency spectrum.

*homepage*: https://bitbucket.org/gutenkunstlab/dadi

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.7.0``|``-Python-2.7.12``|``intel/2016b``

<a anchor="dagitty"/>
### *dagitty*

A port of the web-based software 'DAGitty', available at <http://dagitty.net>, for  analyzing structural causal models (also known as directed acyclic graphs or DAGs). This package  computes covariate adjustment sets for estimating causal effects, enumerates instrumental variables,  derives testable implications (d-separation and vanishing tetrads), generates equivalent models, and  includes a simple facility for data simulation.

*homepage*: https://cran.r-project.org/web/packages/dagitty/

version  |versionsuffix|toolchain
---------|-------------|--------------
``0.2-2``|``-R-3.5.1`` |``foss/2018b``

<a anchor="dakota"/>
### *Dakota*

The Dakota project delivers both state-of-the-art research and robust, usable  software for optimization and UQ. Broadly, the Dakota software's advanced  parametric analyses enable design exploration, model calibration, risk  analysis, and quantification of margins and uncertainty with computational  models. 

*homepage*: https://dakota.sandia.gov/

version   |toolchain
----------|--------------
``6.16.0``|``foss/2021b``

<a anchor="dalton"/>
### *Dalton*

The Dalton code is a powerful tool for a wide range of molecular properties  at different levels of theory.  Any published work arising from use of one of the Dalton2016 programs  must acknowledge that by a proper reference,  https://www.daltonprogram.org/www/citation.html.

*homepage*: https://daltonprogram.org/

version   |versionsuffix|toolchain
----------|-------------|---------------
``2016``  |``-i8``      |``intel/2017b``
``2020.0``|             |``foss/2021a``

<a anchor="damageproto"/>
### *damageproto*

X protocol and ancillary headers for xinerama

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version  |toolchain
---------|-------------------------------
``1.2.1``|``foss/2016a``, ``intel/2016a``

<a anchor="dammit"/>
### *dammit*

dammit is a simple de novo transcriptome annotator. It was born out of  the observations that annotation is mundane and annoying,  all the individual pieces of the process exist already, and the existing  solutions are overly complicated or rely on crappy non-free software.

*homepage*: http://www.camillescott.org/dammit/

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.3.2``|``-Python-2.7.13``|``intel/2017a``

<a anchor="danpos2"/>
### *DANPOS2*

A toolkit for Dynamic Analysis of Nucleosome and Protein Occupancy by Sequencing, version 2

*homepage*: https://sites.google.com/site/danposdoc

version  |versionsuffix     |toolchain
---------|------------------|--------------
``2.2.2``|``-Python-2.7.12``|``foss/2016b``

<a anchor="das_tool"/>
### *DAS_Tool*

DAS Tool is an automated method that integrates the results of a flexible number of binning  algorithms to calculate an optimized, non-redundant set of bins from a single assembly.

*homepage*: https://github.com/cmks/DAS_Tool

version  |versionsuffix             |toolchain
---------|--------------------------|--------------
``1.1.1``|``-R-3.5.1-Python-2.7.15``|``foss/2018b``
``1.1.1``|``-R-4.1.2``              |``foss/2021b``
``1.1.3``|``-R-4.1.0``              |``foss/2021a``

<a anchor="dask"/>
### *dask*

Dask natively scales Python. Dask provides advanced parallelism for analytics, enabling performance  at scale for the tools you love.

*homepage*: https://dask.org/

version      |versionsuffix     |toolchain
-------------|------------------|------------------------------------------------------------------------
``0.8.2``    |``-Python-2.7.11``|``intel/2016a``
``0.8.2``    |``-Python-3.5.1`` |``intel/2016a``
``0.11.0``   |``-Python-2.7.11``|``foss/2016a``
``0.11.0``   |``-Python-2.7.12``|``intel/2016b``
``0.11.0``   |``-Python-3.5.2`` |``intel/2016b``
``0.12.0``   |``-Python-2.7.12``|``intel/2016b``
``0.12.0``   |``-Python-3.5.2`` |``foss/2016b``, ``intel/2016b``
``0.16.0``   |``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``0.16.0``   |``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``0.17.0``   |``-Python-2.7.13``|``foss/2017a``, ``intel/2017a``
``0.17.0``   |``-Python-3.6.1`` |``intel/2017a``
``0.17.2``   |``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``0.19.4``   |``-Python-3.6.6`` |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``1.0.0``    |``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``1.1.4``    |``-Python-2.7.15``|``fosscuda/2018b``
``2.3.0``    |``-Python-3.7.2`` |``foss/2019a``
``2.8.0``    |``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``
``2.18.1``   |``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``2021.2.0`` |                  |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``, ``intelcuda/2020b``
``2021.9.1`` |                  |``foss/2021a``
``2022.1.0`` |                  |``foss/2021b``
``2022.10.0``|                  |``foss/2022a``

<a anchor="datamash"/>
### *datamash*

GNU datamash performs basic numeric, textual and statistical operations on input data files

*homepage*: https://www.gnu.org/software/datamash/

version|toolchain
-------|--------------------------------------------------------
``1.3``|``foss/2018a``
``1.5``|``GCCcore/10.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.3.0``

<a anchor="davix"/>
### *davix*

The davix project aims to make file management over HTTP-based protocols simple.  The focus is on high-performance remote I/O and data management of large collections of files.  Currently, there is support for the WebDav (link is external), Amazon S3 (link is external),  Microsoft Azure (link is external), and HTTP (link is external) protocols.

*homepage*: https://dmc.web.cern.ch/projects/davix/home

version  |toolchain
---------|-----------------
``0.6.6``|``intel/2017a``
``0.7.5``|``GCCcore/8.3.0``

<a anchor="db"/>
### *DB*

Berkeley DB enables the development of custom data management solutions,  without the overhead traditionally associated with such custom projects.

*homepage*: http://www.oracle.com/technetwork/products/berkeleydb

version    |toolchain
-----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``4.8.30`` |``intel/2016a``
``6.2.23`` |``foss/2016a``
``6.2.32`` |``GCCcore/6.4.0``, ``intel/2017a``
``18.1.25``|``GCCcore/7.3.0``
``18.1.32``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``18.1.40``|``FCC/4.5.0``, ``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.1.0``, ``GCCcore/12.2.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``

<a anchor="db_file"/>
### *DB_File*

Perl5 access to Berkeley DB version 1.x.

*homepage*: https://perldoc.perl.org/DB_File.html

version  |versionsuffix   |toolchain
---------|----------------|------------------
``1.835``|                |``GCCcore/9.3.0``
``1.835``|``-Perl-5.20.3``|``intel/2016a``
``1.835``|``-Perl-5.22.1``|``foss/2016a``
``1.855``|                |``GCCcore/10.2.0``
``1.856``|                |``GCCcore/10.3.0``
``1.857``|                |``GCCcore/11.2.0``
``1.858``|                |``GCCcore/11.3.0``

<a anchor="dbd-mysql"/>
### *DBD-mysql*

Perl binding for MySQL

*homepage*: https://metacpan.org/pod/distribution/DBD-mysql/lib/DBD/mysql.pm

version  |versionsuffix   |toolchain
---------|----------------|----------------------------------------------
``4.032``|``-Perl-5.22.2``|``intel/2016a``
``4.033``|``-Perl-5.24.0``|``intel/2016b``
``4.042``|``-Perl-5.24.1``|``intel/2017a``
``4.046``|``-Perl-5.26.0``|``foss/2017b``, ``intel/2017b``
``4.046``|``-Perl-5.26.1``|``intel/2018a``
``4.048``|``-Perl-5.28.0``|``foss/2018b``
``4.050``|                |``GCC/10.2.0``, ``GCC/11.2.0``, ``GCC/11.3.0``
``4.050``|``-Perl-5.28.1``|``foss/2019a``

<a anchor="dbg2olc"/>
### *DBG2OLC*

DBG2OLC:Efficient Assembly of Large Genomes Using Long Erroneous Reads of the Third Generation   Sequencing Technologies

*homepage*: https://github.com/yechengxi/DBG2OLC

version     |toolchain
------------|----------------------------------------------------------
``20170208``|``intel/2016b``, ``intel/2017a``
``20180221``|``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``

<a anchor="dbus"/>
### *DBus*

D-Bus is a message bus system, a simple way for applications to talk  to one another.  In addition to interprocess communication, D-Bus helps  coordinate process lifecycle; it makes it simple and reliable to code  a "single instance" application or daemon, and to launch applications  and daemons on demand when their services are needed. 

*homepage*: https://dbus.freedesktop.org/

version    |toolchain
-----------|----------------------------------------------------------
``1.10.8`` |``foss/2016a``, ``intel/2016a``
``1.10.12``|``intel/2016b``
``1.10.20``|``GCCcore/6.4.0``
``1.11.20``|``intel/2017a``
``1.13.0`` |``intel/2017b``
``1.13.6`` |``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``1.13.8`` |``GCCcore/8.2.0``
``1.13.12``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.13.18``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.14.0`` |``GCCcore/11.3.0``
``1.15.2`` |``GCCcore/12.2.0``

<a anchor="dbus-glib"/>
### *dbus-glib*

D-Bus is a message bus system, a simple way for applications to talk to one another.

*homepage*: https://dbus.freedesktop.org/doc/dbus-glib

version  |toolchain
---------|------------------------------------------------------------------------
``0.106``|``foss/2016a``, ``intel/2016a``
``0.108``|``intel/2016b``, ``intel/2017a``
``0.110``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``intel/2017b``
``0.112``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="dcm2niix"/>
### *dcm2niix*

dcm2niix is designed to convert neuroimaging data from the DICOM format to the NIfTI format.

*homepage*: https://github.com/rordenlab/dcm2niix

version         |toolchain
----------------|------------------------------------
``1.0.20180622``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``1.0.20190902``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``1.0.20200331``|``GCCcore/8.3.0``
``1.0.20201102``|``GCCcore/8.3.0``
``1.0.20211006``|``GCCcore/10.3.0``

<a anchor="dcmtk"/>
### *DCMTK*

DCMTK is a collection of libraries and applications implementing large parts the DICOM standard. It includes software for examining, constructing and converting DICOM image files, handling offline media, sending and receiving images over a network connection, as well as demonstrative image storage and worklist servers.

*homepage*: https://dicom.offis.de/dcmtk

version  |toolchain
---------|--------------------------------------
``3.6.3``|``GCCcore/7.3.0``
``3.6.5``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``3.6.6``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``3.6.7``|``GCCcore/11.3.0``

<a anchor="dd"/>
### *dd*

dd is a package for working with binary decision diagrams that includes both a pure Python implementation and Cython bindings to C libraries (CUDD, Sylvan, BuDDy). The Python and Cython modules implement the same API, so the same user code runs with both. All the standard operations on BDDs are available, including dynamic variable reordering using sifting, garbage collection, dump/load from files, plotting, and a parser of quantified Boolean expressions. This module includes bindings for: CUDD v3.0.0, Sylvan v1.0.0

*homepage*: https://github.com/tulip-control/dd

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.5.6``|``-Python-3.8.2``|``foss/2020a``

<a anchor="deal.ii"/>
### *deal.II*

deal.II is a C++ program library targeted at the computational solution of  partial differential equations using adaptive finite elements.

*homepage*: https://www.dealii.org

version  |toolchain
---------|-------------------------------
``9.1.1``|``foss/2019a``, ``intel/2019a``
``9.3.3``|``foss/2021a``

<a anchor="deap"/>
### *deap*

DEAP is a novel evolutionary computation framework for rapid prototyping and testing of ideas.  It seeks to make algorithms explicit and data structures transparent.

*homepage*: http://deap.readthedocs.org/en/master/

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.9.2``|``-Python-2.7.12``|``intel/2016b``

<a anchor="decona"/>
### *decona*

fastq to polished sequenses: pipeline suitable for mixed samples and long (Nanopore) reads

*homepage*: https://github.com/Saskia-Oosterbroek/decona

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.1.2``|``-Python-3.7.4``|``foss/2019b``

<a anchor="deconf"/>
### *deconf*

decomposition (deconfounding) of OMICS datasets in heterogeneous tissues

*homepage*: http://web.cbio.uct.ac.za/~renaud/CRAN/

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.0.1``|``-R-3.5.1`` |``foss/2018b``

<a anchor="deconica"/>
### *DeconICA*

Deconvolution of transcriptome through Immune Component Analysis (DeconICA) is an R package for identifying immune-related signals in transcriptome through deconvolution or unsupervised source separation methods.

*homepage*: https://urszulaczerwinska.github.io/DeconICA

version  |versionsuffix|toolchain
---------|-------------|--------------
``0.1.0``|``-R-3.5.1`` |``foss/2018b``

<a anchor="deepdiff"/>
### *deepdiff*

DeepDiff: Deep Difference of dictionaries, iterables and almost any other object recursively.

*homepage*: https://zepworks.com/deepdiff/current/

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``3.3.0``|``-Python-2.7.15``|``intel/2018b``
``3.3.0``|``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``4.0.6``|``-Python-3.7.2`` |``GCCcore/8.2.0``
``5.0.2``|``-Python-3.7.4`` |``GCCcore/8.3.0``
``5.7.0``|                  |``GCCcore/11.2.0``
``5.8.1``|                  |``GCCcore/11.3.0``

<a anchor="deeplabcut"/>
### *DeepLabCut*

Markerless tracking of user-defined features with deep learning

*homepage*: http://www.mousemotorlab.org/deeplabcut

version    |versionsuffix   |toolchain
-----------|----------------|--------------
``2.2.0.6``|``-CUDA-11.3.1``|``foss/2021a``

<a anchor="deepmedic"/>
### *deepmedic*

Efficient Multi-Scale 3D Convolutional Neural Network for Segmentation of 3D Medical Scans.

*homepage*: https://biomedia.doc.ic.ac.uk/software/deepmedic/

version  |versionsuffix   |toolchain
---------|----------------|--------------
``0.8.2``|                |``foss/2021a``
``0.8.2``|``-CUDA-11.3.1``|``foss/2021a``

<a anchor="deepmod2"/>
### *DeepMod2*

DeepMod2 is a computational tool for detecting DNA methylation and modifications  from Oxford Nanopore reads. 

*homepage*: https://github.com/WGLab/DeepMod2

version  |toolchain
---------|--------------
``0.0.1``|``foss/2021a``

<a anchor="deepsurv"/>
### *DeepSurv*

DeepSurv is a deep learning approach to survival analysis. 

*homepage*: https://github.com/jaredleekatzman/DeepSurv

version           |versionsuffix    |toolchain
------------------|-----------------|------------------
``2.0.0-20180922``|``-Python-3.6.6``|``fosscuda/2018b``

<a anchor="deeptools"/>
### *deepTools*

deepTools is a suite of python tools particularly developed for the efficient analysis of  high-throughput sequencing data, such as ChIP-seq, RNA-seq or MNase-seq.

*homepage*: https://deeptools.readthedocs.io/

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``2.5.4``|``-Python-3.6.3``|``intel/2017b``
``3.3.1``|``-Python-3.6.6``|``foss/2018b``
``3.3.1``|``-Python-3.7.4``|``intel/2019b``
``3.3.1``|``-Python-3.8.2``|``foss/2020a``
``3.5.0``|                 |``foss/2021a``
``3.5.1``|                 |``foss/2021b``

<a anchor="delly"/>
### *Delly*

Delly is an integrated structural variant (SV) prediction method that can discover, genotype and visualize deletions, tandem duplications, inversions and translocations at single-nucleotide resolution in short-read massively parallel sequencing data. It uses paired-ends, split-reads and read-depth to sensitively and accurately delineate genomic rearrangements throughout the genome.

*homepage*: https://github.com/dellytools/delly/

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.7.8``|``-linux_x86_64``|``system``
``0.8.7``|                 |``gompi/2020b``
``1.1.5``|                 |``GCC/11.3.0``

<a anchor="demixt"/>
### *DeMixT*

Cell type-specific deconvolution of heterogeneous tumor samples with two or three components  using expression data from RNAseq or microarray platforms.

*homepage*: https://bioinformatics.mdanderson.org/main/DeMixT

version  |versionsuffix|toolchain
---------|-------------|--------------
``0.2.1``|``-R-3.5.1`` |``foss/2018b``

<a anchor="dendropy"/>
### *DendroPy*

A Python library for phylogenetics and phylogenetic computing:  reading, writing, simulation, processing and manipulation of phylogenetic trees  (phylogenies) and characters.

*homepage*: https://dendropy.org/

version  |versionsuffix     |toolchain
---------|------------------|------------------------------------------------------------------------------
``4.4.0``|                  |``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``intel/2019a``
``4.4.0``|``-Python-2.7.15``|``intel/2018b``
``4.5.2``|                  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``4.5.2``|``-Python-2.7.18``|``GCCcore/10.2.0``

<a anchor="destiny"/>
### *destiny*

R packages to create and plot diffusion maps.

*homepage*: https://bioconductor.org/packages/destiny

version  |versionsuffix|toolchain
---------|-------------|---------------
``2.5.6``|``-R-3.4.0`` |``intel/2017a``

<a anchor="detectron2"/>
### *Detectron2*

Detectron2 is Facebook AI Research's next generation library that provides state-of-the-art detection and segmentation algorithms. It is the successor of Detectron and maskrcnn-benchmark. It supports a number of computer vision research projects and production applications in Facebook.

*homepage*: https://github.com/facebookresearch/detectron2

version|versionsuffix   |toolchain
-------|----------------|--------------
``0.6``|                |``foss/2021a``
``0.6``|``-CUDA-11.3.1``|``foss/2021a``

<a anchor="detonate"/>
### *detonate*

DETONATE (DE novo TranscriptOme rNa-seq Assembly with or without the Truth Evaluation)  consists of two component packages, RSEM-EVAL and REF-EVAL. Both packages are mainly intended to be used  to evaluate de novo transcriptome assemblies, although REF-EVAL can be used to compare sets of any kinds  of genomic sequences.

*homepage*: http://deweylab.biostat.wisc.edu/detonate/

version |toolchain
--------|---------------
``1.11``|``intel/2017b``

<a anchor="devito"/>
### *Devito*

Devito is a domain-specific Language (DSL) and code generation framework for  performing optimised Finite Difference (FD) computation from high-level symbolic problem definitions.  Devito performs automated code generation and Just-In-time (JIT) compilation based on symbolic  equations defined in SymPy to create and execute highly optimised Finite Difference stencil kernels on  multiple computer platforms.

*homepage*: https://www.devitoproject.org

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``4.6.1``|``-Python-3.8.2``|``foss/2020a``

<a anchor="dfa"/>
### *DFA*

Python library for modeling DFAs, Moore Machines, and Transition Systems.

*homepage*: https://github.com/mvcisback/dfa

version  |versionsuffix    |toolchain
---------|-----------------|------------------
``0.3.4``|``-Python-3.8.2``|``GCCcore/9.3.0``
``2.1.2``|                 |``GCCcore/10.2.0``

<a anchor="dft-d3"/>
### *DFT-D3*

DFT-D3 implements a dispersion correction for density functionals, Hartree-Fock and semi-empirical  quantum chemical methods.

*homepage*: http://www.thch.uni-bonn.de/tc/index.php?section=downloads&subsection=DFT-D3&lang=english

version  |toolchain
---------|-------------------------------------------------------------------------------------
``3.2.0``|``GCC/8.3.0``, ``iccifort/2020.4.304``, ``intel-compilers/2021.2.0``, ``intel/2019a``

<a anchor="dft-d4"/>
### *DFT-D4*

Generally Applicable Atomic-Charge Dependent London Dispersion Correction.

*homepage*: https://www.chemie.uni-bonn.de/pctc/mulliken-center/software/dftd4

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``3.2.0``|``-Python-3.7.4``|``intel/2019b``

<a anchor="dftb+"/>
### *DFTB+*

DFTB+ is a fast and efficient versatile quantum mechanical simulation package. It is based on the Density Functional Tight Binding (DFTB) method, containing almost all of the useful extensions which have been developed for the DFTB framework so far.  Using DFTB+ you can carry out quantum mechanical simulations like with ab-initio density functional theory based packages, but in an approximate way gaining typically around two order of magnitude in speed.

*homepage*: https://www.dftb-plus.info

version  |versionsuffix         |toolchain
---------|----------------------|---------------
``1.3.1``|                      |``intel/2017a``
``17.1`` |``-Python-2.7.14``    |``intel/2017b``
``19.1`` |``-Python-2.7.16``    |``foss/2019b``
``19.1`` |``-Python-2.7.16-mpi``|``foss/2019b``
``21.1`` |                      |``intel/2021a``

<a anchor="dftd3-lib"/>
### *dftd3-lib*

This is a repackaged version of the DFTD3 program by S. Grimme and his coworkers. The original program (V3.1 Rev 1) was downloaded at 2016-04-03. It has been converted to free format and encapsulated into modules.

*homepage*: https://github.com/dftbplus/dftd3-lib

version|toolchain
-------|-------------------------------------------
``0.9``|``GCC/8.3.0``, ``intel-compilers/2021.2.0``

<a anchor="dgl"/>
### *DGL*

DGL is an easy-to-use, high performance and scalable Python package for deep learning on graphs. DGL is framework agnostic, meaning if a deep graph model is a component of an end-to-end application, the rest of the logics can be implemented in any major frameworks, such as PyTorch, Apache MXNet or TensorFlow.

*homepage*: https://www.dgl.ai

version  |versionsuffix                  |toolchain
---------|-------------------------------|------------------
``0.6.1``|``-Python-3.7.4-PyTorch-1.8.1``|``fosscuda/2019b``

<a anchor="dial"/>
### *DIAL*

DIAL (De novo Identification of Alleles) is a collection of programs  to automate the discovery of alleles for a species where we lack a reference sequence.   The SNPs/alleles are specifically selected for a low error rate in genotyping assays.   

*homepage*: http://www.bx.psu.edu/miller_lab/

version       |toolchain
--------------|--------------
``2011.06.06``|``foss/2016a``

<a anchor="diamond"/>
### *DIAMOND*

Accelerated BLAST compatible local sequence aligner

*homepage*: https://github.com/bbuchfink/diamond

version   |toolchain
----------|----------------------------------------------------------------
``0.9.22``|``foss/2018a``, ``foss/2018b``, ``intel/2018a``, ``intel/2018b``
``0.9.24``|``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``0.9.30``|``GCC/8.3.0``, ``iccifort/2019.5.281``
``0.9.36``|``GCC/9.3.0``
``2.0.4`` |``GCC/9.3.0``
``2.0.6`` |``GCC/7.3.0-2.30``
``2.0.7`` |``GCC/10.2.0``
``2.0.11``|``GCC/10.3.0``
``2.0.13``|``GCC/10.3.0``, ``GCC/11.2.0``

<a anchor="dice"/>
### *Dice*

Dice contains code for performing SHCI, VMC, GFMC, DMC, FCIQMC, stochastic MRCI and SC-NEVPT2, and AFQMC calculations with a focus on ab initio systems.

*homepage*: https://github.com/sanshar/Dice

version     |toolchain
------------|--------------
``20221025``|``foss/2022a``

<a anchor="dicom2nifti"/>
### *dicom2nifti*

Python library for converting dicom files to nifti

*homepage*: https://github.com/icometrix/dicom2nifti

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``2.2.12``|``-Python-3.8.2``|``foss/2020a``
``2.3.0`` |                 |``foss/2020b``

<a anchor="dicombrowser"/>
### *DicomBrowser*

DicomBrowser is an application for inspecting and modifying DICOM metadata in many files at once.

*homepage*: http://nrg.wustl.edu/software/dicom-browser/

version    |versionsuffix     |toolchain
-----------|------------------|----------
``1.7.0b5``|``-Java-1.7.0_80``|``system``

<a anchor="diffutils"/>
### *Diffutils*

Diffutils: GNU diff utilities - find the differences between files

*homepage*: http://www.gnu.org/software/diffutils/diffutils.html

version|toolchain
-------|-------------
``3.3``|``GCC/4.8.2``

<a anchor="dijitso"/>
### *dijitso*

dijitso is a Python module for distributed just-in-time shared library building.

*homepage*: https://bitbucket.org/fenics-project/dijitso

version     |versionsuffix    |toolchain
------------|-----------------|--------------
``2019.1.0``|``-Python-3.7.4``|``foss/2019b``

<a anchor="dill"/>
### *dill*

dill extends python's pickle module for serializing and de-serializing python objects to the majority  of the built-in python types. Serialization is the process of converting an object to a byte stream, and the inverse  of which is converting a byte stream back to on python object hierarchy.

*homepage*: https://pypi.org/project/dill/

version  |toolchain
---------|--------------------------------------
``0.3.0``|``GCCcore/8.2.0``
``0.3.3``|``GCCcore/10.2.0``, ``GCCcore/8.3.0``
``0.3.4``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``

<a anchor="dirac"/>
### *DIRAC*

DIRAC: Program for Atomic and Molecular Direct Iterative Relativistic All-electron Calculations

*homepage*: http://www.diracprogram.org

version |versionsuffix               |toolchain
--------|----------------------------|-------------------------------
``19.0``|``-Python-2.7.18-int64``    |``intel/2020a``
``19.0``|``-Python-2.7.18-mpi-int64``|``intel/2020a``
``22.0``|                            |``foss/2021a``, ``intel/2021a``

<a anchor="distributed"/>
### *distributed*

Dask.distributed is a lightweight library for distributed computing in Python.  It extends both the concurrent.futures and dask APIs to moderate sized clusters.

*homepage*: https://distributed.readthedocs.io/

version   |versionsuffix     |toolchain
----------|------------------|---------------
``1.14.3``|``-Python-2.7.12``|``intel/2016b``
``1.14.3``|``-Python-3.5.2`` |``intel/2016b``
``1.21.6``|``-Python-3.6.4`` |``intel/2018a``

<a anchor="distributedstream"/>
### *DistributedStream*

A MPI distributed stream benchmark, useful to identifying nodes with poor memory performance and characterising memory bandwidth variation over systems.

*homepage*: https://github.com/adrianjhpc/DistributedStream

version|toolchain
-------|---------------
``1.0``|``gompi/2021a``

<a anchor="dl_poly_4"/>
### *DL_POLY_4*

DL_POLY is a general purpose classical molecular dynamics (MD) simulation software

*homepage*: https://www.scd.stfc.ac.uk/Pages/DL_POLY.aspx

version  |toolchain
---------|-------------------------------
``5.0.0``|``foss/2020b``, ``intel/2020b``

<a anchor="dl_poly_classic"/>
### *DL_POLY_Classic*

DL_POLY Classic is a freely available molecular dynamics program developed  from the DL_POLY_2 package. This version does not install the java gui.

*homepage*: http://ccpforge.cse.rl.ac.uk/gf/project/dl_poly_classic/

version |versionsuffix    |toolchain
--------|-----------------|-------------------------------
``1.9`` |                 |``intel/2016b``
``1.9`` |``-PLUMED-2.2.3``|``intel/2016b``
``1.10``|                 |``foss/2019b``, ``intel/2019b``

<a anchor="dlb"/>
### *dlb*

DLB is a dynamic library designed to speed up HPC hybrid applications (i.e., two levels of parallelism) by improving the load balance of the outer level of parallelism (e.g., MPI) by dynamically redistributing the computational resources at the inner level of parallelism (e.g., OpenMP). at run time. 

*homepage*: https://pm.bsc.es/dlb/

version|toolchain
-------|--------------------------------
``3.2``|``gompi/2022a``, ``iimpi/2022a``

<a anchor="dlib"/>
### *dlib*

Dlib is a modern C++ toolkit containing machine learning algorithms and tools for creating complex software in C++ to solve real world problems. It is used in both industry and academia in a wide range of domains including robotics, embedded devices, mobile phones, and large high performance computing environments.

*homepage*: https://github.com/davisking/dlib

version  |versionsuffix   |toolchain
---------|----------------|--------------
``19.22``|                |``foss/2021a``
``19.22``|``-CUDA-11.3.1``|``foss/2021a``

<a anchor="dm-reverb"/>
### *dm-reverb*

Reverb is an efficient and easy-to-use data storage and transport system designed for machine learning research. Reverb is primarily used as an experience replay system for distributed reinforcement learning algorithms but the system also supports multiple data structure representations such as FIFO, LIFO, and priority queues.

*homepage*: https://github.com/deepmind/reverb

version  |toolchain
---------|--------------
``0.2.0``|``foss/2020b``
``0.7.0``|``foss/2021b``

<a anchor="dm-tree"/>
### *dm-tree*

dm-tree provides tree, a library for working with nested data structures. In a way, tree generalizes the builtin map function which only supports flat sequences, and allows to apply a function to each "leaf" preserving the overall structure.

*homepage*: https://github.com/deepmind/tree

version  |toolchain
---------|--------------------------------------
``0.1.1``|``GCCcore/8.3.0``
``0.1.5``|``GCCcore/10.2.0``
``0.1.6``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``

<a anchor="dmcfun"/>
### *DMCfun*

Diffusion Model of Conflict (DMC) in Reaction Time Tasks

*homepage*: https://cran.r-project.org/web/packages/DMCfun

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.3.0``|``-R-3.6.2`` |``foss/2019b``

<a anchor="dmtcp"/>
### *DMTCP*

DMTCP is a tool to transparently checkpoint the state of multiple simultaneous applications, including multi-threaded and distributed applications. It operates directly on the user binary executable, without any Linux kernel modules or other kernel modifications.

*homepage*: http://dmtcp.sourceforge.net/index.html

version  |toolchain
---------|-------------------------------------------------
``2.4.5``|``system``
``2.5.0``|``foss/2016a``
``2.5.1``|``system``
``2.5.2``|``GCCcore/8.3.0``, ``foss/2016b``, ``foss/2018b``
``2.6.0``|``GCCcore/8.2.0``, ``GCCcore/9.3.0``

<a anchor="dolfin"/>
### *DOLFIN*

DOLFIN is the C++/Python interface of FEniCS, providing a consistent PSE   (Problem Solving Environment) for ordinary and partial differential equations.

*homepage*: https://bitbucket.org/fenics-project/dolfin

version           |versionsuffix    |toolchain
------------------|-----------------|--------------
``2018.1.0.post1``|``-Python-3.6.4``|``foss/2018a``
``2019.1.0.post0``|``-Python-3.7.4``|``foss/2019b``

<a anchor="doris"/>
### *Doris*

Delft object-oriented radar interferometric software

*homepage*: http://doris.tudelft.nl/

version      |toolchain
-------------|-------------------------------
``4.02``     |``intel/2017a``
``4.04beta4``|``foss/2018a``, ``intel/2017a``
``4.06beta2``|``intel/2017a``

<a anchor="dosageconvertor"/>
### *DosageConvertor*

DosageConvertor is a C++ tool to convert dosage files (in VCF format) from Minimac3/4  to other formats such as MaCH or PLINK. Please note that this tool CANNOT handle missing values in  the input files and may NOT work for non-Minimac3/4 VCF files.

*homepage*: http://genome.sph.umich.edu/wiki/DosageConvertor

version  |toolchain
---------|--------------
``1.0.4``|``GCC/10.2.0``

<a anchor="dotnet-core-runtime"/>
### *dotNET-Core-Runtime*

.NET is a free, cross-platform, open source developer platform for building many different types  of applications.

*homepage*: https://www.microsoft.com/net/

version   |toolchain
----------|------------------
``2.0.7`` |``GCCcore/6.4.0``
``5.0.17``|``GCCcore/10.3.0``
``6.0.1`` |``GCCcore/11.2.0``

<a anchor="dotnet-sdk"/>
### *dotNET-SDK*

.NET is a free, cross-platform, open source developer platform for building many different types  of applications.

*homepage*: https://www.microsoft.com/net/

version    |versionsuffix |toolchain
-----------|--------------|----------
``3.1.300``|``-linux-x64``|``system``
``6.0.101``|``-linux-x64``|``system``

<a anchor="double-conversion"/>
### *double-conversion*

Efficient binary-decimal and decimal-binary conversion routines for IEEE doubles.

*homepage*: https://github.com/google/double-conversion

version  |toolchain
---------|-----------------------------------------------------------------------------
``3.0.3``|``foss/2018a``
``3.1.4``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``3.1.5``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``
``3.2.0``|``GCCcore/11.3.0``
``3.2.1``|``GCCcore/12.2.0``

<a anchor="doubletfinder"/>
### *DoubletFinder*

R package for detecting doublets in single-cell RNA sequencing data

*homepage*: https://github.com/chris-mcginnis-ucsf/DoubletFinder

version  |versionsuffix|toolchain
---------|-------------|--------------
``2.0.3``|``-R-4.0.0`` |``foss/2020a``

<a anchor="doxygen"/>
### *Doxygen*

Doxygen is a documentation system for C++, C, Java, Objective-C, Python,  IDL (Corba and Microsoft flavors), Fortran, VHDL, PHP, C#, and to some  extent D. 

*homepage*: https://www.doxygen.org

version    |toolchain
-----------|-------------------------------------------------------------------------------------------------------------------------------------------------------
``1.8.9.1``|``GCC/4.9.2``
``1.8.10`` |``GNU/4.9.3-2.25``, ``intel/2016.02-GCC-4.9``
``1.8.11`` |``GCC/4.9.2``, ``GCCcore/5.4.0``, ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``
``1.8.13`` |``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``gimkl/2017a``
``1.8.14`` |``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``
``1.8.15`` |``GCCcore/8.2.0``
``1.8.16`` |``GCCcore/8.3.0``
``1.8.17`` |``GCCcore/9.3.0``
``1.8.20`` |``GCCcore/10.2.0``
``1.9.1``  |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.9.4``  |``GCCcore/11.3.0``
``1.9.5``  |``GCCcore/12.2.0``

<a anchor="dragmap"/>
### *DRAGMAP*

Dragmap is the Dragen mapper/aligner Open Source Software.

*homepage*: https://github.com/Illumina/DRAGMAP

version  |toolchain
---------|--------------
``1.3.0``|``foss/2021b``

<a anchor="drake"/>
### *Drake*

Drake is a simple-to-use, extensible, text-based data workflow tool that organizes command execution around data and its dependencies.

*homepage*: https://github.com/Factual/drake

version  |versionsuffix|toolchain
---------|-------------|----------
``1.0.3``|``-Java-1.8``|``system``

<a anchor="drep"/>
### *dRep*

dRep is a python program which performs rapid pair-wise comparison of genome sets. One of it’s major purposes is for genome de-replication, but it can do a lot more.

*homepage*: https://drep.readthedocs.io/en/latest/

version  |toolchain
---------|--------------
``3.0.0``|``foss/2021a``

<a anchor="drop"/>
### *DROP*

Pipeline to find aberrant events in RNA-Seq data, useful for diagnosis of rare disorders

*homepage*: https://gagneurlab-drop.readthedocs.io

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.0.3``|``-R-4.0.3`` |``foss/2020b``
``1.1.0``|``-R-4.0.3`` |``foss/2020b``
``1.1.1``|``-R-4.1.2`` |``foss/2021b``

<a anchor="dropest"/>
### *dropEst*

Pipeline for initial analysis of droplet-based single-cell RNA-seq data

*homepage*: https://github.com/hms-dbmi/dropEst

version  |versionsuffix|toolchain
---------|-------------|---------------
``0.7.1``|``-R-3.4.3`` |``intel/2017b``

<a anchor="dsa"/>
### *DSA*

Digital Sorting Algorithm

*homepage*: https://github.com/zhandong/DSA

version|versionsuffix|toolchain
-------|-------------|--------------
``1.0``|``-R-3.5.1`` |``foss/2018b``

<a anchor="dsfmt"/>
### *dSFMT*

Double precision SIMD-oriented Fast Mersenne Twister.

*homepage*: https://github.com/MersenneTwister-Lab/dSFMT

version  |toolchain
---------|------------------
``2.2.5``|``GCCcore/10.2.0``

<a anchor="dsrc"/>
### *DSRC*

DNA Sequence Reads Compression is an application designed for compression of data files  containing reads from DNA sequencing in FASTQ format. The amount of such files can be huge, e.g., a few  (or tens) of gigabytes, so a need for a robust data compression tool is clear. Usually universal compression  programs like gzip or bzip2 are used for this purpose, but it is obvious that a specialized tool can work better.

*homepage*: http://sun.aei.polsl.pl/dsrc

version  |versionsuffix    |toolchain
---------|-----------------|----------
``2.0rc``|``-linux-64-bit``|``system``

<a anchor="dsuite"/>
### *Dsuite*

Fast calculation of the ABBA-BABA statistics across many populations/species

*homepage*: https://github.com/millanek/Dsuite

version     |toolchain
------------|--------------------------------------------
``20190713``|``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``20210718``|``GCC/10.3.0``, ``intel-compilers/2021.2.0``

<a anchor="dtcmp"/>
### *dtcmp*

The Datatype Comparison (DTCMP) Library provides pre-defined and user-defined comparison operations to compare the values of two items which can be arbitrary MPI datatypes. Using these comparison operations, the library provides various routines for manipulating data, which may be distributed over the processes of an MPI communicator.

*homepage*: https://github.com/LLNL/dtcmp

version  |toolchain
---------|------------------------------------------------------------------
``1.1.0``|``gompi/2019a``, ``gompi/2020a``, ``iimpi/2019a``, ``iimpi/2020a``
``1.1.2``|``gompi/2020b``

<a anchor="dtcwt"/>
### *dtcwt*

Dual-Tree Complex Wavelet Transform library for Python

*homepage*: https://github.com/rjw57/dtcwt

version   |versionsuffix     |toolchain
----------|------------------|---------------
``0.12.0``|``-Python-2.7.15``|``foss/2018b``
``0.12.0``|``-Python-3.7.4`` |``intel/2019b``

<a anchor="dualsphysics"/>
### *DualSPHysics*

DualSPHysics is based on the Smoothed Particle Hydrodynamics model named SPHysics.   The code is developed to study free-surface flow phenomena where Eulerian methods  can be difficult to apply, such as waves or impact of dam-breaks on off-shore  structures. DualSPHysics is a set of C++, CUDA and Java codes designed to deal  with real-life engineering problems.

*homepage*: https://dual.sphysics.org/

version    |versionsuffix        |toolchain
-----------|---------------------|--------------
``5.0.175``|                     |``GCC/11.2.0``
``5.0.175``|``-CUDA-%(cudaver)s``|``GCC/11.2.0``

<a anchor="dubstepr"/>
### *DUBStepR*

DUBStepR (Determining the Underlying Basis using Step-wise Regression) is a feature selection algorithm for cell type identification in single-cell RNA-sequencing data.

*homepage*: https://github.com/prabhakarlab/DUBStepR

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.2.0``|``-R-4.1.2`` |``foss/2021b``

<a anchor="dune-core"/>
### *dune-core*

The Dune core modules build the stable basis of Dune.  They follow a consistent release cycle and have high requirements regarding stability and backwards compatibility.  These modules build the foundation for higher-level components. 

*homepage*: https://www.dune-project.org/groups/core/

version        |toolchain
---------------|--------------
``2.8.0.post1``|``foss/2020b``

<a anchor="dune-fem"/>
### *dune-fem*

DUNE-FEM is a discretization module based on DUNE containing all the building blocks required to implement efficient solvers for a wide range of (systems of non linear) partial differential equations. DUNE-FEM can also be used through an extensive Python interface which brings all components of DUNE-FEM and the DUNE core modules to Python. 

*homepage*: https://www.dune-project.org/modules/dune-fem/

version    |toolchain
-----------|--------------
``2.8.0.6``|``foss/2020b``

<a anchor="dxpy"/>
### *dxpy*

DNAnexus Platform API bindings for Python

*homepage*: http://autodoc.dnanexus.com/

version    |versionsuffix     |toolchain
-----------|------------------|---------------
``0.266.1``|``-Python-2.7.14``|``intel/2018a``

<a anchor="dymat"/>
### *DyMat*

Read and process result files from Dymola and OpenModelica with Python.

*homepage*: https://www.j-raedler.de/projects/DyMat

version|versionsuffix  |toolchain
-------|---------------|--------------
``0.7``|``-2020-12-12``|``foss/2021b``

<a anchor="e"/>
### *E*


<a href="#e-antic">E-ANTIC</a> - <a href="#ea-utils">ea-utils</a> - <a href="#earthengine-api">earthengine-api</a> - <a href="#easybuild">EasyBuild</a> - <a href="#easyqc">EasyQC</a> - <a href="#ecbuild">ecBuild</a> - <a href="#eccodes">ecCodes</a> - <a href="#eccodes-python">eccodes-python</a> - <a href="#ed2">ED2</a> - <a href="#edlib">edlib</a> - <a href="#egglib">EggLib</a> - <a href="#eggnog-mapper">eggnog-mapper</a> - <a href="#egttools">EGTtools</a> - <a href="#eht-imaging">eht-imaging</a> - <a href="#eigen">Eigen</a> - <a href="#eigenexa">EigenExa</a> - <a href="#eigensoft">EIGENSOFT</a> - <a href="#einops">einops</a> - <a href="#elastix">elastix</a> - <a href="#elbencho">elbencho</a> - <a href="#elfio">ELFIO</a> - <a href="#elfutils">elfutils</a> - <a href="#elk">Elk</a> - <a href="#elpa">ELPA</a> - <a href="#elph">ELPH</a> - <a href="#elprep">elprep</a> - <a href="#elsi">ELSI</a> - <a href="#elsi-rci">ELSI-RCI</a> - <a href="#emacs">Emacs</a> - <a href="#eman2">EMAN2</a> - <a href="#emboss">EMBOSS</a> - <a href="#embree">Embree</a> - <a href="#emcee">emcee</a> - <a href="#emu">EMU</a> - <a href="#enabrowsertool">enaBrowserTool</a> - <a href="#enchant">enchant</a> - <a href="#enchant-2">enchant-2</a> - <a href="#ensemblcoreapi">EnsEMBLCoreAPI</a> - <a href="#entrypoints">entrypoints</a> - <a href="#epd">EPD</a> - <a href="#epic">EPIC</a> - <a href="#episcanpy">epiScanpy</a> - <a href="#ericscript">EricScript</a> - <a href="#esl-bundle">ESL-Bundle</a> - <a href="#esm-2">ESM-2</a> - <a href="#esmf">ESMF</a> - <a href="#esmpy">ESMPy</a> - <a href="#esmvaltool">ESMValTool</a> - <a href="#espeak-ng">eSpeak-NG</a> - <a href="#essentia">Essentia</a> - <a href="#ete">ETE</a> - <a href="#etsf_io">ETSF_IO</a> - <a href="#eudev">eudev</a> - <a href="#evidentialgene">EvidentialGene</a> - <a href="#evmix">evmix</a> - <a href="#exabayes">ExaBayes</a> - <a href="#examl">ExaML</a> - <a href="#excel-writer-xlsx">Excel-Writer-XLSX</a> - <a href="#exiftool">ExifTool</a> - <a href="#exiv2">exiv2</a> - <a href="#exonerate">Exonerate</a> - <a href="#expat">expat</a> - <a href="#expect">expect</a> - <a href="#expecttest">expecttest</a> - <a href="#express">eXpress</a> - <a href="#extrae">Extrae</a>


<a anchor="e-antic"/>
### *E-ANTIC*

E-ANTIC is a C/C++ library to deal with real embedded number fields built on top of ANTIC (https://github.com/wbhart/antic). Its aim is to have as fast as possible exact arithmetic operations and comparisons.

*homepage*: https://github.com/videlec/e-antic

version  |toolchain
---------|--------------------
``0.1.2``|``GCC/8.2.0-2.31.1``
``0.1.5``|``GCC/8.3.0``

<a anchor="ea-utils"/>
### *ea-utils*

Command-line tools for processing biological sequencing data. Barcode demultiplexing, adapter trimming, etc.  Primarily written to support an Illumina based pipeline -  but should work with any FASTQs.

*homepage*: http://expressionanalysis.github.io/ea-utils/

version     |toolchain
------------|-----------------------------------------------
``1.04.807``|``foss/2016a``, ``foss/2016b``, ``intel/2016b``

<a anchor="earthengine-api"/>
### *earthengine-api*

Python and JavaScript bindings for calling the Earth Engine API

*homepage*: https://github.com/google/earthengine-api

version    |versionsuffix     |toolchain
-----------|------------------|---------------
``0.1.143``|``-Python-2.7.14``|``intel/2018a``

<a anchor="easybuild"/>
### *EasyBuild*

EasyBuild is a software build and installation framework  written in Python that allows you to install software in a structured,  repeatable and robust way.

*homepage*: https://easybuilders.github.io/easybuild

version   |toolchain
----------|----------
``1.0.0`` |``system``
``1.0.1`` |``system``
``1.0.2`` |``system``
``1.1.0`` |``system``
``1.2.0`` |``system``
``1.3.0`` |``system``
``1.4.0`` |``system``
``1.5.0`` |``system``
``1.6.0`` |``system``
``1.7.0`` |``system``
``1.8.0`` |``system``
``1.8.1`` |``system``
``1.8.2`` |``system``
``1.9.0`` |``system``
``1.10.0``|``system``
``1.11.0``|``system``
``1.11.1``|``system``
``1.12.0``|``system``
``1.12.1``|``system``
``1.13.0``|``system``
``1.14.0``|``system``
``1.15.0``|``system``
``1.15.1``|``system``
``1.15.2``|``system``
``1.16.0``|``system``
``1.16.1``|``system``
``1.16.2``|``system``
``2.0.0`` |``system``
``2.1.0`` |``system``
``2.1.1`` |``system``
``2.2.0`` |``system``
``2.3.0`` |``system``
``2.4.0`` |``system``
``2.5.0`` |``system``
``2.6.0`` |``system``
``2.7.0`` |``system``
``2.8.0`` |``system``
``2.8.1`` |``system``
``2.8.2`` |``system``
``2.9.0`` |``system``
``3.0.0`` |``system``
``3.0.1`` |``system``
``3.0.2`` |``system``
``3.1.0`` |``system``
``3.1.1`` |``system``
``3.1.2`` |``system``
``3.2.0`` |``system``
``3.2.1`` |``system``
``3.3.0`` |``system``
``3.3.1`` |``system``
``3.4.0`` |``system``
``3.4.1`` |``system``
``3.5.0`` |``system``
``3.5.1`` |``system``
``3.5.2`` |``system``
``3.5.3`` |``system``
``3.6.0`` |``system``
``3.6.1`` |``system``
``3.6.2`` |``system``
``3.7.0`` |``system``
``3.7.1`` |``system``
``3.8.0`` |``system``
``3.8.1`` |``system``
``3.9.0`` |``system``
``3.9.1`` |``system``
``3.9.2`` |``system``
``3.9.3`` |``system``
``3.9.4`` |``system``
``4.0.0`` |``system``
``4.0.1`` |``system``
``4.1.0`` |``system``
``4.1.1`` |``system``
``4.1.2`` |``system``
``4.2.0`` |``system``
``4.2.1`` |``system``
``4.2.2`` |``system``
``4.3.0`` |``system``
``4.3.1`` |``system``
``4.3.2`` |``system``
``4.3.3`` |``system``
``4.3.4`` |``system``
``4.4.0`` |``system``
``4.4.1`` |``system``
``4.4.2`` |``system``
``4.5.0`` |``system``
``4.5.1`` |``system``
``4.5.2`` |``system``
``4.5.3`` |``system``
``4.5.4`` |``system``
``4.5.5`` |``system``
``4.6.0`` |``system``
``4.6.1`` |``system``
``4.6.2`` |``system``

<a anchor="easyqc"/>
### *EasyQC*

EasyQC is an R-package that provides advanced functionality to  (1) perform file-level QC of single genome-wide association (GWA) data-sets  (2) conduct quality control across several GWA data-sets (meta-level QC)  (3) simplify data-handling of large-scale GWA data-sets.

*homepage*: http://www.uni-regensburg.de/medizin/epidemiologie-praeventivmedizin/genetische-epidemiologie/software/

version|versionsuffix|toolchain
-------|-------------|---------------
``9.2``|``-R-3.3.1`` |``intel/2016b``

<a anchor="ecbuild"/>
### *ecBuild*

A CMake-based build system, consisting of a collection of CMake macros and functions that ease the managing of software build systems 

*homepage*: https://ecbuild.readthedocs.io/

version  |toolchain
---------|----------
``3.7.0``|``system``

<a anchor="eccodes"/>
### *ecCodes*

ecCodes is a package developed by ECMWF which provides an application programming interface and  a set of tools for decoding and encoding messages in the following formats: WMO FM-92 GRIB edition 1 and edition 2,  WMO FM-94 BUFR edition 3 and edition 4, WMO GTS abbreviated header (only decoding).

*homepage*: https://software.ecmwf.int/wiki/display/ECC/ecCodes+Home

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------
``2.7.3`` |                  |``intel/2018a``
``2.7.3`` |``-Python-2.7.14``|``intel/2017b``
``2.8.2`` |                  |``intel/2018a``
``2.9.2`` |                  |``intel/2018b``, ``iomkl/2018b``
``2.12.5``|                  |``gompi/2019a``
``2.15.0``|                  |``gompi/2019b``, ``iimpi/2019b``
``2.17.0``|                  |``foss/2018b``, ``gompi/2019b``
``2.18.0``|                  |``gompi/2020a``
``2.20.0``|                  |``gompi/2020b``
``2.22.1``|                  |``gompi/2021a``
``2.24.2``|                  |``gompi/2021b``, ``iimpi/2021b``
``2.27.0``|                  |``gompi/2022a``

<a anchor="eccodes-python"/>
### *eccodes-python*

Python 3 interface to decode and encode GRIB and BUFR files via the ECMWF ecCodes library.

*homepage*: https://github.com/ecmwf/eccodes-python

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.0.0``|``-Python-3.7.4``|``foss/2019b``
``1.0.0``|``-Python-3.8.2``|``foss/2020a``
``1.1.0``|                 |``foss/2020b``

<a anchor="ed2"/>
### *ED2*

The Ecosystem Demography Biosphere Model (ED2) is an integrated terrestrial biosphere model  incorporating hydrology, land-surface biophysics, vegetation dynamics, and soil carbon and nitrogen biogeochemistry

*homepage*: https://github.com/EDmodel/ED2

version     |versionsuffix|toolchain
------------|-------------|---------------
``20170201``|             |``intel/2017a``
``20170201``|``-serial``  |``intel/2017a``

<a anchor="edlib"/>
### *edlib*

Lightweight, super fast library for sequence alignment using edit (Levenshtein) distance.

*homepage*: https://martinsos.github.io/edlib

version        |versionsuffix    |toolchain
---------------|-----------------|----------------------------------------------
``1.3.8.post1``|``-Python-3.7.4``|``GCC/8.3.0``, ``iccifort/2019.5.281``
``1.3.8.post1``|``-Python-3.8.2``|``GCC/9.3.0``
``1.3.8.post2``|``-Python-3.8.2``|``iccifort/2020.1.217``
``1.3.9``      |                 |``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``

<a anchor="egglib"/>
### *EggLib*

EggLib is a C++/Python library and program package for evolutionary genetics and genomics.

*homepage*: http://egglib.sourceforge.net/

version   |toolchain
----------|---------------
``2.1.10``|``intel/2016a``

<a anchor="eggnog-mapper"/>
### *eggnog-mapper*

EggNOG-mapper is a tool for fast functional annotation of novel sequences. It uses precomputed orthologous groups and phylogenies from the eggNOG database (http://eggnog5.embl.de) to transfer functional information from fine-grained orthologs only. Common uses of eggNOG-mapper include the annotation of novel genomes, transcriptomes or even metagenomic gene catalogs.

*homepage*: https://github.com/eggnogdb/eggnog-mapper

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.0.3``|``-Python-2.7.14``|``intel/2018a``
``2.1.4``|                  |``foss/2020b``
``2.1.7``|                  |``foss/2021b``

<a anchor="egttools"/>
### *EGTtools*

EGTtools provides a centralized repository with analytical and numerical methods to study/model game theoretical problems under the Evolutionary Game Theory (EGT) framework.

*homepage*: https://github.com/Socrats/EGTTools

version        |toolchain
---------------|--------------
``0.1.10.dev2``|``foss/2021b``
``0.1.11``     |``foss/2022a``

<a anchor="eht-imaging"/>
### *eht-imaging*

Python modules for simulating and manipulating VLBI data and producing images with regularized maximum likelihood methods.

*homepage*: https://github.com/achael/eht-imaging

version  |toolchain
---------|--------------
``1.2.2``|``foss/2021a``

<a anchor="eigen"/>
### *Eigen*

Eigen is a C++ template library for linear algebra: matrices, vectors, numerical solvers,  and related algorithms.

*homepage*: https://eigen.tuxfamily.org

version   |toolchain
----------|----------------------------------------------------------
``3.2.3`` |``foss/2016a``, ``foss/2016b``
``3.2.5`` |``system``
``3.2.6`` |``system``
``3.2.7`` |``foss/2016a``, ``intel/2016a``
``3.2.8`` |``foss/2016a``, ``intel/2016a``, ``system``
``3.2.9`` |``foss/2016b``, ``intel/2016b``
``3.2.10``|``intel/2016b``
``3.3.2`` |``foss/2016b``, ``intel/2016b``
``3.3.3`` |``GCCcore/6.3.0``, ``intel/2016b``
``3.3.4`` |``system``
``3.3.5`` |``system``
``3.3.7`` |``GCCcore/9.3.0``, ``system``
``3.3.8`` |``GCCcore/10.2.0``
``3.3.9`` |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``3.4.0`` |``GCCcore/10.2.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="eigenexa"/>
### *EigenExa*

EigenExa, a part of KMATHLIB, is a high performance eigen-solver.

*homepage*: https://www.r-ccs.riken.jp/labs/lpnctrt/projects/eigenexa/

version |toolchain
--------|---------------
``2.11``|``intel/2020b``

<a anchor="eigensoft"/>
### *EIGENSOFT*

The EIGENSOFT package combines functionality from our population genetics methods (Patterson et al.  2006)  and our EIGENSTRAT stratification correction method (Price et al. 2006). The EIGENSTRAT method uses principal  components  analysis to explicitly model ancestry differences between cases and controls along continuous axes of  variation;  the resulting correction is specific to a candidate marker’s variation in frequency across ancestral  populations,  minimizing spurious associations while maximizing power to detect true associations. The EIGENSOFT  package has a built-in plotting script and supports multiple file formats and quantitative phenotypes.

*homepage*: http://www.hsph.harvard.edu/alkes-price/software/

version  |toolchain
---------|-------------------------------------------------------------------------------
``6.0.1``|``foss/2016a``
``6.1.1``|``foss/2016a``
``6.1.4``|``foss/2016b``
``7.2.1``|``foss/2018b``, ``foss/2019a``, ``foss/2019b``, ``foss/2021a``, ``intel/2019a``

<a anchor="einops"/>
### *einops*

Flexible and powerful tensor operations for readable and reliable code. Supports numpy, pytorch, tensorflow, jax, and others.

*homepage*: https://einops.rocks/

version  |toolchain
---------|--------------------------------------
``0.3.2``|``GCCcore/10.2.0``
``0.4.1``|``GCCcore/10.3.0``, ``GCCcore/11.3.0``

<a anchor="elastix"/>
### *elastix*

elastix: a toolbox for rigid and nonrigid registration of images. 

*homepage*: http://elastix.isi.uu.nl/

version  |toolchain
---------|--------------
``4.9.0``|``foss/2018a``

<a anchor="elbencho"/>
### *elbencho*

A distributed storage benchmark for files, objects & blocks with support for GPUs

*homepage*: https://github.com/breuner/elbencho

version  |toolchain
---------|--------------
``2.0-3``|``GCC/10.3.0``

<a anchor="elfio"/>
### *ELFIO*

ELFIO is a header-only C++ library intended  for reading and generating files in the ELF binary format.

*homepage*: http://elfio.sourceforge.net

version|toolchain
-------|----------
``3.9``|``system``

<a anchor="elfutils"/>
### *elfutils*

The elfutils project provides libraries and tools for ELF files  and DWARF data. 

*homepage*: https://elfutils.org/

version  |toolchain
---------|----------------------------------------------------------------------------
``0.182``|``GCCcore/9.3.0``
``0.183``|``GCCcore/10.2.0``
``0.185``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``0.187``|``GCCcore/11.3.0``

<a anchor="elk"/>
### *Elk*

An all-electron full-potential linearised augmented-plane wave (FP-LAPW) code with many advanced features. Written originally at Karl-Franzens-Universität Graz as a milestone of the EXCITING EU Research and Training Network, the code is designed to be as simple as possible so that new developments in the field of density functional theory (DFT) can be added quickly and reliably. 

*homepage*: http://elk.sourceforge.net/

version   |toolchain
----------|---------------
``4.0.15``|``intel/2016b``
``4.3.6`` |``intel/2017a``
``6.3.2`` |``intel/2019b``
``7.0.12``|``foss/2020b``
``7.2.42``|``foss/2021a``
``8.5.2`` |``foss/2022a``

<a anchor="elpa"/>
### *ELPA*

Eigenvalue SoLvers for Petaflop-Applications.

*homepage*: https://elpa.rzg.mpg.de

version            |toolchain
-------------------|---------------------------------------------------------------------------------
``2015.02.002``    |``foss/2018a``, ``gimkl/2017a``, ``intel/2018a``
``2016.05.004``    |``intel/2016b``, ``intel/2017a``
``2016.11.001.pre``|``foss/2018b``, ``intel/2018b``
``2017.11.001``    |``foss/2018b``, ``intel/2018a``, ``intel/2018b``
``2018.05.001``    |``foss/2018b``, ``intel/2018b``
``2018.11.001``    |``intel/2019a``
``2019.11.001``    |``foss/2019b``, ``foss/2020a``, ``intel/2019b``, ``intel/2020a``, ``iomkl/2019b``
``2020.05.001``    |``intel/2020a``
``2020.11.001``    |``foss/2020b``, ``intel/2020b``
``2021.05.001``    |``foss/2021a``, ``foss/2021b``, ``intel/2021a``, ``intel/2021b``
``2021.05.002``    |``intel/2020b``
``2021.11.001``    |``foss/2021b``, ``foss/2022a``, ``intel/2021b``, ``intel/2022a``

<a anchor="elph"/>
### *ELPH*

ELPH is a general-purpose Gibbs sampler for finding motifs in a set   of DNA or protein sequences. The program takes as input a set containing anywhere from   a few dozen to thousands of sequences, and searches through them for the most common motif,   assuming that each sequence contains one copy of the motif. We have used ELPH to find   patterns such as ribosome binding sites (RBSs) and exon splicing enhancers (ESEs). 

*homepage*: http://ccb.jhu.edu/software/ELPH/index.shtml

version  |toolchain
---------|--------------
``1.0.1``|``foss/2018b``

<a anchor="elprep"/>
### *elprep*

elPrep is a high-performance tool for analyzing .sam/.bam files (up to and including variant calling)  in sequencing pipelines.

*homepage*: https://github.com/ExaScience/elprep

version  |toolchain
---------|----------
``5.1.1``|``system``

<a anchor="elsi"/>
### *ELSI*

ELSI provides and enhances scalable, open-source software library solutions for  electronic structure calculations in  materials science, condensed matter physics, chemistry, and many other fields.  ELSI focuses on methods that solve or circumvent eigenvalue problems in electronic structure theory.  The ELSI infrastructure should also be useful for other challenging eigenvalue problems. 

*homepage*: https://wordpress.elsi-interchange.org/

version  |versionsuffix|toolchain
---------|-------------|-------------------------------
``2.5.0``|             |``intel/2019b``
``2.5.0``|``-PEXSI``   |``intel/2019b``
``2.6.4``|``-PEXSI``   |``foss/2020b``, ``intel/2020b``
``2.7.1``|``-PEXSI``   |``foss/2021a``, ``intel/2021a``

<a anchor="elsi-rci"/>
### *ELSI-RCI*

ELSI-RCI provides and enhances open-source software packages which iteratively solve or circumvent eigenvalue problems in self-consistent field calculations based on the Kohn-Sham density-functional theory. 

*homepage*: https://wordpress.elsi-interchange.org/

version  |toolchain
---------|---------------------------------------------------------------------------------------------------------------------------
``0.1.0``|``GCC/10.3.0``, ``GCC/11.2.0``, ``foss/2020b``, ``intel-compilers/2021.2.0``, ``intel-compilers/2021.4.0``, ``intel/2020b``

<a anchor="emacs"/>
### *Emacs*

GNU Emacs is an extensible, customizable text editor--and more.  At its core is an interpreter for Emacs Lisp, a dialect of the Lisp programming  language with extensions to support text editing.

*homepage*: https://www.gnu.org/software/emacs/

version |versionsuffix|toolchain
--------|-------------|-------------------------------------------------------
``24.3``|             |``GCC/4.8.3``
``24.3``|``-bare``    |``GCC/4.8.3``
``24.4``|             |``GCC/4.9.2``
``24.5``|             |``GCC/4.9.3-2.25``
``25.1``|             |``foss/2016a``
``25.3``|             |``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``26.3``|             |``GCCcore/8.3.0``
``27.1``|             |``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``27.2``|             |``GCCcore/11.2.0``
``28.1``|             |``GCCcore/10.2.0``

<a anchor="eman2"/>
### *EMAN2*

EMAN2 is a broadly based greyscale scientific image processing suite with a  primary focus on processing data from transmission electron microscopes. 

*homepage*: https://blake.bcm.edu/emanwiki/EMAN2

version  |versionsuffix                  |toolchain
---------|-------------------------------|----------------------------------
``2.3``  |                               |``system``
``2.3``  |``-Python-2.7.15``             |``foss/2019a``, ``fosscuda/2019a``
``2.21a``|``-Python-2.7.14-Boost-1.63.0``|``foss/2018a``

<a anchor="emboss"/>
### *EMBOSS*

EMBOSS is 'The European Molecular Biology Open Software Suite'.  EMBOSS is a free Open Source software analysis package specially developed for  the needs of the molecular biology (e.g. EMBnet) user community.

*homepage*: http://emboss.sourceforge.net/

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------
``6.6.0``|                 |``GCC/8.2.0-2.31.1``, ``foss/2016b``, ``foss/2018b``, ``foss/2021a``, ``foss/2021b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2017a``, ``intel/2018b``
``6.6.0``|``-Java-11``     |``GCC/8.3.0``
``6.6.0``|``-Java-13``     |``GCC/10.2.0``
``6.6.0``|``-X11-20170314``|``intel/2017a``

<a anchor="embree"/>
### *Embree*

Intel® Embree is a collection of high-performance ray tracing kernels, developed at Intel.  The target users of Intel® Embree are graphics application engineers who want to improve the performance of  their photo-realistic rendering application by leveraging Embree's performance-optimized ray tracing kernels.  The kernels are optimized for the latest Intel® processors with support for SSE, AVX, AVX2, and AVX-512 instructions.  Intel® Embree supports runtime code selection to choose the traversal and build algorithms that best matches  the instruction set of your CPU. We recommend using Intel® Embree through its API to get the highest benefit  from future improvements. Intel® Embree is released as Open Source under the Apache 2.0 license.

*homepage*: https://embree.github.io/

version   |toolchain
----------|--------------------------------------
``3.4.0`` |``iccifort/2018.1.163-GCC-6.4.0-2.28``
``3.13.4``|``system``

<a anchor="emcee"/>
### *emcee*

Emcee is an extensible, pure-Python implementation of  Goodman & Weare's Affine Invariant Markov chain Monte Carlo (MCMC) Ensemble sampler.  It's designed for Bayesian parameter estimation and it's really sweet! 

*homepage*: https://dfm.io/emcee

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``2.2.1``|                  |``foss/2019a``
``2.2.1``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``2.2.1``|``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``

<a anchor="emu"/>
### *EMU*

EMU infers population structure in the presence of missingness and works for both haploid, psuedo-haploid and diploid genotype datasets 

*homepage*: http://www.popgen.dk/software/index.php/EMU

version |versionsuffix    |toolchain
--------|-----------------|--------------
``0.66``|``-Python-3.7.4``|``foss/2019b``

<a anchor="enabrowsertool"/>
### *enaBrowserTool*

enaBrowserTools is a set of scripts that interface with the ENA  web services to download data from ENA easily, without any knowledge of scripting required.

*homepage*: https://github.com/enasequence/enaBrowserTools/

version  |versionsuffix    |toolchain
---------|-----------------|------------------
``1.5.4``|``-Python-3.7.2``|``GCCcore/8.2.0``
``1.6``  |                 |``GCCcore/10.3.0``

<a anchor="enchant"/>
### *enchant*

Enchant is a library (and command-line program) that wraps a number of different spelling libraries  and programs with a consistent interface.

*homepage*: https://abiword.github.io/enchant/

version  |toolchain
---------|---------------
``1.6.1``|``intel/2017a``

<a anchor="enchant-2"/>
### *enchant-2*

Enchant aims to provide a simple but comprehensive abstraction for dealing with different spell checking libraries in a consistent way. A client, such as a text editor or word processor, need not know anything about a specific spell-checker, and since all back-ends are plugins, new spell-checkers can be added without needing any change to the program using Enchant.

*homepage*: https://github.com/AbiWord/enchant

version  |toolchain
---------|--------------------------------------
``2.3.3``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="ensemblcoreapi"/>
### *EnsEMBLCoreAPI*

The Ensembl Core Perl API and SQL schema

*homepage*: https://www.ensembl.org/info/docs/api/index.html

version           |versionsuffix   |toolchain
------------------|----------------|--------------
``96.0-r20190601``|``-Perl-5.28.1``|``foss/2019a``

<a anchor="entrypoints"/>
### *entrypoints*

Entry points are a way for Python packages to advertise objects with some common interface.

*homepage*: https://github.com/takluyver/entrypoints

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``0.2.2``|``-Python-2.7.11``|``foss/2016a``
``0.2.2``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``0.2.2``|``-Python-3.5.1`` |``foss/2016a``
``0.2.2``|``-Python-3.5.2`` |``intel/2016b``

<a anchor="epd"/>
### *EPD*

The Enthought Python Distribution provides scientists with a comprehensive set of tools to perform  rigorous data analysis and visualization. Python, distinguished by its flexibility, coherence, and ease-of-use,  is rapidly becoming the programming language of choice for researchers worldwide.  EPD extends this capacity with a powerful collection of Python libraries to enable interactive technical computing and  cross-platform rapid application development.

*homepage*: http://www.enthought.com/products/edudownload.php

version      |toolchain
-------------|----------
``7.3-2-rh5``|``system``

<a anchor="epic"/>
### *EPIC*

Package implementing EPIC method to estimate the proportion of immune, stromal, endothelial  and cancer or other cells from bulk gene expression data.

*homepage*: https://gfellerlab.shinyapps.io/EPIC_1-1

version|versionsuffix|toolchain
-------|-------------|--------------
``1.1``|``-R-3.5.1`` |``foss/2018b``

<a anchor="episcanpy"/>
### *epiScanpy*

EpiScanpy is a toolkit to analyse single-cell open chromatin (scATAC-seq) and single-cell DNA methylation (for example scBS-seq) data. EpiScanpy is the epigenomic extension of the very popular scRNA-seq analysis tool Scanpy (Genome Biology, 2018) [Wolf18].

*homepage*: https://github.com/colomemaria/episcanpy

version  |toolchain
---------|--------------
``0.3.1``|``foss/2021a``

<a anchor="ericscript"/>
### *EricScript*

EricScript is a computational framework for the discovery of gene fusions in paired end RNA-seq data.

*homepage*: https://sites.google.com/site/bioericscript/home

version  |versionsuffix|toolchain
---------|-------------|---------------
``0.5.5``|``-R-3.4.0`` |``intel/2017a``

<a anchor="esl-bundle"/>
### *ESL-Bundle*

The ESL Bundle is a collection of libraries and utilities broadly used in  electronic structure calculations, put together to make their use easier by  researchers and scientific software developers. ESL stands for Electronic  Structure Library, an initiative which distributes quality software and  promotes open standards for high-performance computing applications in the  field of electronic structure calculations. 

*homepage*: https://esl.cecam.org

version  |toolchain
---------|-------------------------------
``0.6.1``|``foss/2020b``, ``intel/2020b``

<a anchor="esm-2"/>
### *ESM-2*

ESM-2 outperforms all tested single-sequence protein language models  across a range of structure prediction tasks. ESMFold harnesses the ESM-2 language model to generate  accurate structure predictions end to end directly from the sequence of a protein.

*homepage*: https://github.com/facebookresearch/esm

version  |versionsuffix   |toolchain
---------|----------------|--------------
``2.0.0``|                |``foss/2021a``
``2.0.0``|``-CUDA-11.3.1``|``foss/2021a``

<a anchor="esmf"/>
### *ESMF*

The Earth System Modeling Framework (ESMF) is a suite of software tools for developing  high-performance, multi-component Earth science modeling applications.

*homepage*: https://www.earthsystemcog.org/projects/esmf/

version     |versionsuffix   |toolchain
------------|----------------|---------------------------------------------------------------------------------
``6.3.0rp1``|                |``intel/2017a``
``6.3.0rp1``|``-HDF5-1.8.18``|``intel/2017a``
``7.0.0``   |                |``foss/2016a``
``7.0.2``   |                |``intel/2017b``
``7.1.0r``  |                |``foss/2018b``, ``foss/2019a``, ``intel/2018a``, ``intel/2018b``, ``iomkl/2018b``
``8.0.0``   |                |``foss/2019b``, ``intel/2019b``
``8.0.1``   |                |``foss/2020a``, ``foss/2020b``, ``intel/2020a``, ``intel/2020b``
``8.1.1``   |                |``foss/2021a``, ``intel/2021a``
``8.2.0``   |                |``foss/2021b``, ``intel/2021b``
``8.3.0``   |                |``foss/2022a``, ``intel/2022a``

<a anchor="esmpy"/>
### *ESMPy*

Earth System Modeling Framework (ESMF) Python Interface

*homepage*: https://earthsystemmodeling.org/esmpy

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``8.0.0``|``-Python-3.7.4``|``intel/2019b``
``8.0.1``|                 |``intel/2020b``
``8.0.1``|``-Python-3.8.2``|``foss/2020a``

<a anchor="esmvaltool"/>
### *ESMValTool*

The Earth System Model eValuation Tool (ESMValTool)  is a community diagnostics and performance metrics tool  for the evaluation of Earth System Models (ESMs) that  allows for routine comparison of single or multiple models,  either against predecessor versions or against observations.

*homepage*: https://www.esmvaltool.org/

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.1.0``|``-Python-2.7.14``|``intel/2017b``

<a anchor="espeak-ng"/>
### *eSpeak-NG*

The eSpeak NG is a compact open source software text-to-speech synthesizer for Linux, Windows, Android and other operating systems. It supports more than 100 languages and accents. It is based on the eSpeak engine created by Jonathan Duddington. 

*homepage*: https://github.com/espeak-ng/espeak-ng

version |toolchain
--------|---------------
``1.50``|``gompi/2020a``

<a anchor="essentia"/>
### *Essentia*

Open-source library and tools for audio and music analysis, description and synthesis

*homepage*: https://essentia.upf.edu

version      |versionsuffix     |toolchain
-------------|------------------|--------------
``2.1_beta5``|``-Python-2.7.15``|``foss/2019a``

<a anchor="ete"/>
### *ETE*

A Python framework for the analysis and visualization of trees

*homepage*: http://etetoolkit.org

version     |versionsuffix     |toolchain
------------|------------------|----------------------------------------------
``3.0.0b36``|``-Python-2.7.12``|``intel/2016b``
``3.1.1``   |``-Python-3.6.6`` |``foss/2018b``
``3.1.2``   |                  |``foss/2020b``, ``foss/2021a``, ``foss/2021b``
``3.1.2``   |``-Python-3.8.2`` |``foss/2020a``

<a anchor="etsf_io"/>
### *ETSF_IO*

A library of F90 routines to read/write the ETSF file  format has been written. It is called ETSF_IO and available under LGPL. 

*homepage*: http://www.etsf.eu/resources/software/libraries_and_tools

version  |toolchain
---------|-------------------------------------------------------------------------------------------------
``1.0.4``|``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``

<a anchor="eudev"/>
### *eudev*

eudev is a fork of systemd-udev with the goal of obtaining better  compatibility with existing software such as OpenRC and Upstart,  older kernels, various toolchains and anything else required by  users and various distributions. 

*homepage*: https://wiki.gentoo.org/wiki/Project:Eudev

version  |toolchain
---------|-------------------------------------------------
``3.1.5``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``3.2``  |``GCCcore/4.9.3``
``3.2.2``|``GCCcore/6.4.0``

<a anchor="evidentialgene"/>
### *EvidentialGene*

EvidentialGene is a genome informatics project for  "Evidence Directed Gene Construction for Eukaryotes",  for constructing high quality, accurate gene sets for  animals and plants (any eukaryotes), being developed by  Don Gilbert at Indiana University, gilbertd at indiana edu.

*homepage*: http://arthropods.eugenes.org/EvidentialGene/

version       |versionsuffix   |toolchain
--------------|----------------|---------------
``2018.01.01``|``-Perl-5.24.1``|``intel/2017a``
``2022.01.14``|                |``gompi/2021b``

<a anchor="evmix"/>
### *evmix*

evmix: Extreme Value Mixture Modelling,  Threshold Estimation and Boundary Corrected Kernel Density Estimation

*homepage*: https://cran.r-project.org/web/packages/evmix

version|versionsuffix|toolchain
-------|-------------|---------------
``2.6``|``-R-3.3.1`` |``intel/2016b``

<a anchor="exabayes"/>
### *ExaBayes*

ExaBayes is a software package for Bayesian tree inference.  It is particularly suitable for large-scale analyses on computer clusters

*homepage*: https://cme.h-its.org/exelixis/web/software/exabayes/

version|toolchain
-------|--------------
``1.5``|``foss/2016b``

<a anchor="examl"/>
### *ExaML*

Exascale Maximum Likelihood (ExaML) code for phylogenetic inference using MPI

*homepage*: https://github.com/stamatak/ExaML/

version   |versionsuffix  |toolchain
----------|---------------|---------------
``3.0.22``|``-hybrid-avx``|``gompi/2021a``

<a anchor="excel-writer-xlsx"/>
### *Excel-Writer-XLSX*

The Excel::Writer::XLSX module can be used to create an Excel file in the 2007+ XLSX format. Multiple worksheets can be added to a workbook and formatting can be applied to cells. Text, numbers, and formulas can be written to the cells.

*homepage*: https://metacpan.org/pod/Excel::Writer::XLSX

version |toolchain
--------|--------------
``1.09``|``foss/2020b``

<a anchor="exiftool"/>
### *ExifTool*

Perl module (Image::ExifTool) and program (exiftool)  to read EXIF information from images

*homepage*: https://owl.phy.queensu.ca/~phil/exiftool/

version  |toolchain
---------|-----------------
``12.00``|``GCCcore/9.3.0``

<a anchor="exiv2"/>
### *exiv2*

Exiv2 is a Cross-platform C++ library and a command line utility to manage image metadata.

*homepage*: https://exiv2.org

version   |toolchain
----------|------------------
``0.27.4``|``GCCcore/10.3.0``

<a anchor="exonerate"/>
### *Exonerate*

Exonerate is a generic tool for pairwise sequence comparison.  It allows you to align sequences using a many alignment models, using either   exhaustive dynamic programming, or a variety of heuristics. 

*homepage*: https://www.ebi.ac.uk/about/vertebrate-genomics/software/exonerate

version  |toolchain
---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.4.0``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/6.4.0-2.28``, ``GCC/8.3.0``, ``foss/2016a``, ``foss/2016b``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``iccifort/2019.5.281``, ``intel/2017a``

<a anchor="expat"/>
### *expat*

Expat is an XML parser library written in C. It is a stream-oriented parser in which an application registers handlers for things the parser might find in the XML document (like start tags).

*homepage*: https://libexpat.github.io

version  |toolchain
---------|-------------------------------------------------------------------------------------------------------------------------
``2.1.0``|``GCC/4.9.2``, ``foss/2016a``, ``intel/2016a``
``2.1.1``|``foss/2016a``, ``intel/2016a``
``2.2.0``|``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2017a``, ``intel/2016b``
``2.2.4``|``GCCcore/6.4.0``
``2.2.5``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``2.2.6``|``GCCcore/8.2.0``
``2.2.7``|``GCCcore/8.3.0``
``2.2.9``|``FCC/4.5.0``, ``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/9.3.0``
``2.4.1``|``GCCcore/11.2.0``
``2.4.8``|``GCCcore/11.3.0``, ``GCCcore/12.1.0``
``2.4.9``|``GCCcore/12.2.0``

<a anchor="expect"/>
### *expect*

Expect is a tool for automating interactive applications    such as telnet, ftp, passwd, fsck, rlogin, tip, etc.    Expect really makes this stuff trivial.    Expect is also useful for testing these same applications.

*homepage*: https://core.tcl.tk/expect/index

version   |toolchain
----------|------------------------------------
``5.45.4``|``GCCcore/7.3.0``, ``GCCcore/9.3.0``

<a anchor="expecttest"/>
### *expecttest*

This library implements expect tests (also known as "golden" tests). Expect tests are a method of  writing tests where instead of hard-coding the expected output of a test, you run the test to get the output, and  the test framework automatically populates the expected output. If the output of the test changes, you can rerun  the test with the environment variable EXPECTTEST_ACCEPT=1 to automatically update the expected output.

*homepage*: https://github.com/ezyang/expecttest

version  |toolchain
---------|------------------------------------------------------------------------------
``0.1.3``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="express"/>
### *eXpress*

Streaming quantification for high-throughput sequencing

*homepage*: https://pachterlab.github.io/eXpress

version  |toolchain
---------|---------------
``1.5.1``|``intel/2017b``

<a anchor="extrae"/>
### *Extrae*

Extrae is the core instrumentation package developed by the Performance Tools group at BSC. Extrae is capable of instrumenting applications based on MPI, OpenMP, pthreads, CUDA1, OpenCL1, and StarSs1 using different instrumentation approaches. The information gathered by Extrae typically includes timestamped events of runtime calls, performance counters and source code references. Besides, Extrae provides its own API to allow the user to manually instrument his or her application.

*homepage*: https://www.bsc.es/computer-sciences/performance-tools

version  |toolchain
---------|---------------
``3.4.1``|``foss/2017a``
``3.7.1``|``intel/2019a``
``3.8.0``|``gompi/2020b``
``3.8.3``|``gompi/2021a``

<a anchor="f"/>
### *F*


<a href="#f90cache">f90cache</a> - <a href="#f90wrap">f90wrap</a> - <a href="#faber">Faber</a> - <a href="#fabio">FabIO</a> - <a href="#faceswap">faceswap</a> - <a href="#falcon">FALCON</a> - <a href="#fann">FANN</a> - <a href="#fast5">fast5</a> - <a href="#fasta">FASTA</a> - <a href="#fastahack">fastahack</a> - <a href="#fastaindex">FastaIndex</a> - <a href="#fastani">FastANI</a> - <a href="#fastaq">Fastaq</a> - <a href="#fastfold">FastFold</a> - <a href="#fastme">FastME</a> - <a href="#fastp">fastp</a> - <a href="#fastparquet">fastparquet</a> - <a href="#fastphase">fastPHASE</a> - <a href="#fastq-pair">fastq-pair</a> - <a href="#fastq-tools">fastq-tools</a> - <a href="#fastq_screen">FastQ_Screen</a> - <a href="#fastqc">FastQC</a> - <a href="#fastqsplitter">fastqsplitter</a> - <a href="#fastqtl">FastQTL</a> - <a href="#fastqz">fastqz</a> - <a href="#fastrfs">FastRFS</a> - <a href="#faststructure">fastStructure</a> - <a href="#fasttree">FastTree</a> - <a href="#fastviromeexplorer">FastViromeExplorer</a> - <a href="#fastx-toolkit">FASTX-Toolkit</a> - <a href="#fatslim">fatslim</a> - <a href="#fbm">fbm</a> - <a href="#fbpic">FBPIC</a> - <a href="#fcc">FCC</a> - <a href="#fcm">FCM</a> - <a href="#fdict">fdict</a> - <a href="#fds">FDS</a> - <a href="#fdstools">fdstools</a> - <a href="#fdtd_solutions">FDTD_Solutions</a> - <a href="#feh">feh</a> - <a href="#fenics">FEniCS</a> - <a href="#fermi-lite">fermi-lite</a> - <a href="#ferret">Ferret</a> - <a href="#festival">festival</a> - <a href="#fetchmg">fetchMG</a> - <a href="#ffc">FFC</a> - <a href="#fflas-ffpack">FFLAS-FFPACK</a> - <a href="#ffmpeg">FFmpeg</a> - <a href="#ffmpi">ffmpi</a> - <a href="#ffnet">ffnet</a> - <a href="#fftlib">fftlib</a> - <a href="#fftw">FFTW</a> - <a href="#fftw.mpi">FFTW.MPI</a> - <a href="#fgbio">fgbio</a> - <a href="#fgsl">FGSL</a> - <a href="#fhi-aims">FHI-aims</a> - <a href="#fiat">FIAT</a> - <a href="#figaro">FIGARO</a> - <a href="#figuregen">FigureGen</a> - <a href="#fiji">Fiji</a> - <a href="#file">file</a> - <a href="#filevercmp">filevercmp</a> - <a href="#filtlong">Filtlong</a> - <a href="#find_circ">find_circ</a> - <a href="#findhap">findhap</a> - <a href="#findutils">findutils</a> - <a href="#fineradstructure">fineRADstructure</a> - <a href="#finestructure">fineSTRUCTURE</a> - <a href="#fio">fio</a> - <a href="#fiona">Fiona</a> - <a href="#firefox">Firefox</a> - <a href="#firestarter">FIRESTARTER</a> - <a href="#fireworks">FireWorks</a> - <a href="#fit-sne">FIt-SNE</a> - <a href="#fix">FIX</a> - <a href="#fixesproto">fixesproto</a> - <a href="#flac">FLAC</a> - <a href="#flair">FLAIR</a> - <a href="#flair-nlp">flair-NLP</a> - <a href="#flann">FLANN</a> - <a href="#flash">FLASH</a> - <a href="#flask">Flask</a> - <a href="#flatbuffers">flatbuffers</a> - <a href="#flatbuffers-python">flatbuffers-python</a> - <a href="#fleur">FLEUR</a> - <a href="#flex">flex</a> - <a href="#flexbar">Flexbar</a> - <a href="#flexiblas">FlexiBLAS</a> - <a href="#flexidot">FlexiDot</a> - <a href="#flink">Flink</a> - <a href="#flint">FLINT</a> - <a href="#flook">flook</a> - <a href="#fltk">FLTK</a> - <a href="#fluent">FLUENT</a> - <a href="#flye">Flye</a> - <a href="#fmilibrary">FMILibrary</a> - <a href="#fmm3d">FMM3D</a> - <a href="#fmpy">FMPy</a> - <a href="#fmriprep">FMRIprep</a> - <a href="#fms">FMS</a> - <a href="#fmt">fmt</a> - <a href="#fobis">FoBiS</a> - <a href="#foldx">FoldX</a> - <a href="#fontconfig">fontconfig</a> - <a href="#fontsproto">fontsproto</a> - <a href="#ford">FORD</a> - <a href="#foss">foss</a> - <a href="#fosscuda">fosscuda</a> - <a href="#fox">FoX</a> - <a href="#fox-toolkit">FOX-Toolkit</a> - <a href="#fpm">FPM</a> - <a href="#fpocket">fpocket</a> - <a href="#fqtrim">fqtrim</a> - <a href="#fqzcomp">fqzcomp</a> - <a href="#fraggenescan">FragGeneScan</a> - <a href="#franz">FRANz</a> - <a href="#freebarcodes">FreeBarcodes</a> - <a href="#freebayes">freebayes</a> - <a href="#freefem">FreeFEM</a> - <a href="#freefem++">FreeFem++</a> - <a href="#freeglut">freeglut</a> - <a href="#freeimage">FreeImage</a> - <a href="#freesasa">FreeSASA</a> - <a href="#freesurfer">FreeSurfer</a> - <a href="#freetds">FreeTDS</a> - <a href="#freetype">freetype</a> - <a href="#freetype-py">freetype-py</a> - <a href="#freexl">FreeXL</a> - <a href="#freud-analysis">freud-analysis</a> - <a href="#fribidi">FriBidi</a> - <a href="#fruit">FRUIT</a> - <a href="#fruit_processor">FRUIT_processor</a> - <a href="#fsl">FSL</a> - <a href="#fsleyes">FSLeyes</a> - <a href="#fsom">fsom</a> - <a href="#fson">FSON</a> - <a href="#ftgl">FTGL</a> - <a href="#fujitsu">Fujitsu</a> - <a href="#fullrmc">fullrmc</a> - <a href="#fumi_tools">fumi_tools</a> - <a href="#funwave-tvd">FUNWAVE-TVD</a> - <a href="#fuse">FUSE</a> - <a href="#fuseq">FuSeq</a> - <a href="#fusioncatcher">FusionCatcher</a> - <a href="#futhark">futhark</a> - <a href="#futile">futile</a> - <a href="#future">future</a> - <a href="#fxtract">fxtract</a>


<a anchor="f90cache"/>
### *f90cache*

f90cache is a compiler cache. It acts as a caching pre-processor to Fortran compilers,  using the -E compiler switch and a hash to detect when a compilation can be satisfied from cache.  This often results in a great speedup in common compilations.

*homepage*: http://people.irisa.fr/Edouard.Canot/f90cache/

version |toolchain
--------|----------
``0.96``|``system``

<a anchor="f90wrap"/>
### *f90wrap*

f90wrap is a tool to automatically generate Python extension modules which interface to Fortran code that makes use of derived types. It builds on the capabilities of the popular f2py utility by generating a simpler Fortran 90 interface to the original Fortran code which is then suitable for wrapping with f2py, together with a higher-level Pythonic wrapper that makes the existance of an additional layer transparent to the final user.

*homepage*: https://github.com/jameskermode/f90wrap

version  |toolchain
---------|--------------
``0.2.8``|``foss/2021a``

<a anchor="faber"/>
### *Faber*

Faber started as a clone of Boost.Build, to experiment with a  new Python frontend. Meanwhile it has evolved into a new build system, which  retains most of the features found in Boost.Build, but with (hopefully !) much  simplified logic, in addition of course to using Python as scripting language,  rather than Jam. The original bjam engine is still in use as scheduler, though  at this point that is mostly an implementation detail.

*homepage*: https://stefanseefeld.github.io/faber

version|toolchain
-------|-----------------
``0.3``|``GCCcore/8.3.0``

<a anchor="fabio"/>
### *FabIO*

FabIO is an I/O library for images produced by 2D X-ray detectors and written in Python.  FabIO support images detectors from a dozen of companies (including Mar, Dectris, ADSC, Hamamatsu, Oxford, ...),  for a total of 20 different file formats (like CBF, EDF, TIFF, ...) and offers an unified interface to their headers  (as a python dictionary) and datasets (as a numpy ndarray of integers or floats).

*homepage*: http://www.silx.org

version   |versionsuffix    |toolchain
----------|-----------------|----------------------------------
``0.10.2``|``-Python-3.7.4``|``foss/2019b``, ``fosscuda/2019b``
``0.11.0``|                 |``foss/2020b``, ``fosscuda/2020b``

<a anchor="faceswap"/>
### *faceswap*

Faceswap is a tool that utilizes deep learning to recognize and swap faces in pictures and videos.

*homepage*: https://github.com/deepfakes/faceswap

version     |versionsuffix    |toolchain
------------|-----------------|--------------
``20180212``|``-Python-3.6.3``|``foss/2017b``

<a anchor="falcon"/>
### *FALCON*

Falcon: a set of tools for fast aligning long reads for consensus and assembly

*homepage*: https://github.com/PacificBiosciences/FALCON

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.8.8``|                  |``intel/2017b``
``1.8.8``|``-Python-2.7.16``|``intel/2019b``

<a anchor="fann"/>
### *FANN*

Fast Artificial Neural Network Library is a free open source neural network library,  which implements multilayer artificial neural networks in C with support for both fully connected  and sparsely connected networks.

*homepage*: http://leenissen.dk

version  |toolchain
---------|----------------------------------
``2.2.0``|``GCCcore/6.4.0``, ``intel/2018a``

<a anchor="fast5"/>
### *fast5*

A lightweight C++ library for accessing Oxford Nanopore  Technologies sequencing data.

*homepage*: http://simpsonlab.github.io/2017/02/27/packing_fast5/

version  |toolchain
---------|----------
``0.6.5``|``system``

<a anchor="fasta"/>
### *FASTA*

The FASTA programs find regions of local or global (new) similarity between  protein or DNA sequences, either by searching Protein or DNA databases, or by identifying  local duplications within a sequence.

*homepage*: http://fasta.bioch.virginia.edu

version    |toolchain
-----------|--------------
``36.3.5e``|``foss/2016b``

<a anchor="fastahack"/>
### *fastahack*

Utilities for indexing and sequence extraction from FASTA files.

*homepage*: https://github.com/ekg/fastahack

version  |toolchain
---------|-----------------------------------------------------------------------------
``1.0.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``

<a anchor="fastaindex"/>
### *FastaIndex*

FastA index (.fai) handler compatible with samtools faidx

*homepage*: https://github.com/lpryszcz/FastaIndex

version    |versionsuffix     |toolchain
-----------|------------------|---------------
``0.11rc7``|``-Python-2.7.14``|``intel/2017b``

<a anchor="fastani"/>
### *FastANI*

FastANI is developed for fast alignment-free computation of  whole-genome Average Nucleotide Identity (ANI). ANI is defined as mean  nucleotide identity of orthologous gene pairs shared between two microbial  genomes. FastANI supports pairwise comparison of both complete and draft  genome assemblies.

*homepage*: https://github.com/ParBLiSS/FastANI

version |toolchain
--------|-----------------------------------------------------------------------------------------------------
``1.1`` |``foss/2018b``, ``intel/2018b``
``1.2`` |``GCC/8.2.0-2.31.1``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``1.3`` |``iccifort/2019.5.281``
``1.31``|``iccifort/2020.1.217``
``1.33``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``iccifort/2020.4.304``, ``intel-compilers/2021.4.0``

<a anchor="fastaq"/>
### *Fastaq*

Python3 scripts to manipulate FASTA and FASTQ files.

*homepage*: https://github.com/sanger-pathogens/Fastaq

version   |toolchain
----------|--------------
``3.17.0``|``GCC/10.3.0``

<a anchor="fastfold"/>
### *FastFold*

Optimizing Protein Structure Prediction Model Training and Inference on GPU Clusters

*homepage*: https://github.com/hpcaitech/FastFold

version     |versionsuffix   |toolchain
------------|----------------|--------------
``20220729``|``-CUDA-11.3.1``|``foss/2021a``

<a anchor="fastme"/>
### *FastME*

FastME: a comprehensive, accurate and fast distance-based phylogeny inference program.

*homepage*: http://www.atgc-montpellier.fr/fastme/

version    |toolchain
-----------|----------------------------------------------------------------------------------------
``2.1.5``  |``foss/2016a``
``2.1.6.1``|``GCC/10.2.0``, ``GCC/8.3.0``, ``iccifort/2019.5.281``, ``intel/2018a``, ``intel/2018b``

<a anchor="fastp"/>
### *fastp*

A tool designed to provide fast all-in-one preprocessing for FastQ files.  This tool is developed in C++ with multithreading supported to afford high performance.

*homepage*: https://github.com/OpenGene/fastp

version   |toolchain
----------|------------------------------------------------------------
``0.19.7``|``foss/2018b``
``0.20.0``|``GCC/8.2.0-2.31.1``, ``GCC/8.3.0``, ``iccifort/2019.5.281``
``0.23.2``|``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/11.3.0``

<a anchor="fastparquet"/>
### *fastparquet*

fastparquet is a python implementation of the parquet format, aiming to integrate into python-based big data work-flows. It is used implicitly by the projects Dask, Pandas and intake-parquet.

*homepage*: https://fastparquet.readthedocs.io/

version  |toolchain
---------|--------------
``0.7.2``|``foss/2021a``
``0.8.0``|``foss/2021b``

<a anchor="fastphase"/>
### *fastPHASE*

fastPHASE: software for haplotype reconstruction,  and estimating missing genotypes from population data Documentation: http://scheet.org/code/fastphase_doc_1.4.pdf

*homepage*: http://stephenslab.uchicago.edu/software.html#fastphase

version  |toolchain
---------|----------
``1.4.8``|``system``

<a anchor="fastq-pair"/>
### *fastq-pair*

Match up paired end fastq files quickly and efficiently.

*homepage*: https://github.com/linsalrob/fastq-pair

version|toolchain
-------|-----------------
``1.0``|``GCCcore/8.2.0``

<a anchor="fastq-tools"/>
### *fastq-tools*

This package provides a number of small and efficient programs to perform  common tasks with high throughput sequencing data in the FASTQ format. All of the programs  work with typical FASTQ files as well as gzipped FASTQ files.

*homepage*: https://homes.cs.washington.edu/~dcjones/fastq-tools/

version  |toolchain
---------|------------------------------
``0.8``  |``foss/2016b``, ``foss/2018b``
``0.8.3``|``GCC/10.3.0``, ``GCC/11.2.0``

<a anchor="fastq_screen"/>
### *FastQ_Screen*

FastQ Screen allows you to screen a library of sequences in FastQ  format against a set of sequence databases so you can see if the composition of the  library matches with what you expect.

*homepage*: http://www.bioinformatics.babraham.ac.uk/projects/fastq_screen/

version   |versionsuffix   |toolchain
----------|----------------|---------------
``0.11.3``|``-Perl-5.24.0``|``foss/2016b``
``0.11.4``|``-Perl-5.24.0``|``foss/2016b``
``0.12.0``|``-Perl-5.26.1``|``intel/2018a``
``0.13.0``|``-Perl-5.28.0``|``foss/2018b``

<a anchor="fastqc"/>
### *FastQC*

FastQC is a quality control application for high throughput sequence data. It reads in sequence data in a variety of formats and can either provide an interactive application to review the results of several different QC checks, or create an HTML based report which can be integrated into a pipeline.

*homepage*: https://www.bioinformatics.babraham.ac.uk/projects/fastqc/

version   |versionsuffix      |toolchain
----------|-------------------|----------
``0.10.1``|``-Java-1.7.0_80`` |``system``
``0.11.2``|``-Java-1.7.0_60`` |``system``
``0.11.3``|``-Java-1.7.0_80`` |``system``
``0.11.4``|``-Java-1.8.0_66`` |``system``
``0.11.4``|``-Java-1.8.0_74`` |``system``
``0.11.5``|``-Java-1.7.0_80`` |``system``
``0.11.5``|``-Java-1.8.0_144``|``system``
``0.11.5``|``-Java-1.8.0_74`` |``system``
``0.11.7``|``-Java-1.8.0_162``|``system``
``0.11.8``|``-Java-1.8``      |``system``
``0.11.8``|``-Java-11``       |``system``
``0.11.9``|``-Java-1.8``      |``system``
``0.11.9``|``-Java-11``       |``system``

<a anchor="fastqsplitter"/>
### *fastqsplitter*

Splits fastq files evenly.

*homepage*: https://github.com/LUMC/fastqsplitter

version  |versionsuffix    |toolchain
---------|-----------------|-----------------
``1.2.0``|``-Python-3.7.2``|``GCCcore/8.2.0``

<a anchor="fastqtl"/>
### *FastQTL*

FastQTL is a QTL mapper

*homepage*: http://fastqtl.sourceforge.net/

version  |toolchain
---------|------------------------------
``2.184``|``GCC/11.2.0``, ``foss/2018b``

<a anchor="fastqz"/>
### *fastqz*

fastqz is a compressor for FASTQ files. FASTQ is the output of DNA sequencing machines.  It is one of the compressors described in the paper: Bonfield JK, Mahoney MV (2013) Compression of  FASTQ and SAM Format Sequencing Data. (mirror) PLoS ONE 8(3): e59190. doi:10.1371/journal.pone.0059190

*homepage*: http://mattmahoney.net/dc/fastqz/

version|toolchain
-------|-------------
``1.5``|``GCC/4.8.2``

<a anchor="fastrfs"/>
### *FastRFS*

Fast Robinson Foulds Supertrees

*homepage*: https://github.com/pranjalv123/FastRFS

version         |toolchain
----------------|---------------
``1.0-20190613``|``gompi/2019a``

<a anchor="faststructure"/>
### *fastStructure*

fastStructure is a fast algorithm for inferring population structure from large SNP genotype data. It is based on a variational Bayesian framework for posterior inference and is written in Python2.x. 

*homepage*: https://rajanil.github.io/fastStructure/

version|versionsuffix     |toolchain
-------|------------------|--------------
``1.0``|``-Python-2.7.11``|``foss/2016a``
``1.0``|``-Python-2.7.12``|``foss/2016b``
``1.0``|``-Python-2.7.13``|``foss/2017a``
``1.0``|``-Python-2.7.15``|``foss/2019a``

<a anchor="fasttree"/>
### *FastTree*

FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide  or protein sequences. FastTree can handle alignments with up to a million of sequences in a reasonable amount of  time and memory. 

*homepage*: http://www.microbesonline.org/fasttree/

version   |toolchain
----------|---------------------------------------------------------------------------------------------------------------------------------------
``2.1.10``|``foss/2018b``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``
``2.1.11``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

<a anchor="fastviromeexplorer"/>
### *FastViromeExplorer*

Identify the viruses/phages and their abundance in the viral metagenomics data.

*homepage*: https://code.vt.edu/saima5/FastViromeExplorer

version     |toolchain
------------|--------------
``20180422``|``foss/2019b``

<a anchor="fastx-toolkit"/>
### *FASTX-Toolkit*

The FASTX-Toolkit is a collection of command line tools for   Short-Reads FASTA/FASTQ files preprocessing.

*homepage*: http://hannonlab.cshl.edu/fastx_toolkit/

version   |toolchain
----------|-----------------------------------------------------------------------------------------------------------------
``0.0.14``|``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/9.3.0``, ``GCCcore/7.3.0``, ``foss/2016a``, ``foss/2016b``, ``intel/2018a``

<a anchor="fatslim"/>
### *fatslim*

FATSLiM stands for “Fast Analysis Toolbox for Simulations of Lipid Membranes” and its goal is to provide an efficient, yet robust, tool to extract physical parameters from MD trajectories.

*homepage*: https://github.com/FATSLiM/fatslim

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.2.1``|``-Python-3.6.4``|``foss/2018a``

<a anchor="fbm"/>
### *fbm*

Exact methods for simulating fractional Brownian motion and fractional Gaussian noise in Python

*homepage*: https://pypi.org/project/fbm

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.2.0``|``-Python-3.6.4``|``intel/2018a``

<a anchor="fbpic"/>
### *FBPIC*

FBPIC (Fourier-Bessel Particle-In-Cell) is a Particle-In-Cell (PIC) code for relativistic plasma  physics. It is especially well-suited for physical simulations of laser-wakefield acceleration and  plasma-wakefield acceleration.

*homepage*: https://fbpic.github.io/

version   |toolchain
----------|------------------
``0.20.3``|``fosscuda/2020b``

<a anchor="fcc"/>
### *FCC*

Fujitsu Compiler based compiler toolchain.

*homepage*: (none)

version  |toolchain
---------|----------
``4.5.0``|``system``

<a anchor="fcm"/>
### *FCM*

FCM is a set of tools for managing and building source code.

*homepage*: http://www.metoffice.gov.uk/research/collaboration/fcm

version  |toolchain
---------|----------
``2.3.1``|``system``

<a anchor="fdict"/>
### *fdict*

A variable and dictionary in pure fortran for retaining any data-type and a fast hash-table dictionary. 

*homepage*: https://github.com/zerothi/fdict

version  |toolchain
---------|-----------------------------------------------------------------------------------------------------------------------------------
``0.8.0``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``iccifort/2020.4.304``, ``intel-compilers/2021.2.0``, ``intel-compilers/2021.4.0``

<a anchor="fds"/>
### *FDS*

Fire Dynamics Simulator (FDS) is a large-eddy simulation (LES) code for low-speed flows,  with an emphasis on smoke and heat transport from fires.

*homepage*: https://pages.nist.gov/fds-smv

version  |versionsuffix|toolchain
---------|-------------|--------------------------------
``6.0.1``|``-no-OFED`` |``system``
``6.5.2``|             |``intel/2016b``
``6.5.3``|             |``intel/2017a``
``6.6.0``|             |``intel/2017b``, ``intel/2018a``
``6.7.0``|             |``intel/2018a``
``6.7.4``|             |``intel/2020a``
``6.7.5``|             |``intel/2020a``
``6.7.6``|             |``intel/2020b``
``6.7.7``|             |``intel/2021b``
``6.7.9``|             |``intel/2022a``

<a anchor="fdstools"/>
### *fdstools*

Forensic DNA Sequencing Tools Tools for characterisation and filtering of PCR stutter artefacts  and other systemic noise in Next Generation Sequencing data of forensic STR markers.

*homepage*: https://git.lumc.nl/jerryhoogenboom/fdstools

version     |versionsuffix     |toolchain
------------|------------------|--------------
``20160322``|``-Python-2.7.11``|``foss/2016a``

<a anchor="fdtd_solutions"/>
### *FDTD_Solutions*

High performance FDTD-method Maxwell solver for the design, analysis and optimization of nanophotonic devices, processes and materials.

*homepage*: http://www.lumerical.com/tcad-products/fdtd/

version      |toolchain
-------------|----------
``8.6.2``    |``system``
``8.11.337`` |``system``
``8.16.982`` |``system``
``8.20.1731``|``system``

<a anchor="feh"/>
### *feh*

feh is an X11 image viewer aimed mostly at console users. Unlike most other viewers, it does not have a fancy GUI, but simply displays images. It is controlled via commandline arguments and configurable key/mouse actions. 

*homepage*: https://feh.finalrewind.org/

version |toolchain
--------|-----------------
``2.26``|``GCCcore/6.4.0``

<a anchor="fenics"/>
### *FEniCS*

FEniCS is a computing platform for solving partial differential equations (PDEs).

*homepage*: https://fenicsproject.org/

version     |versionsuffix    |toolchain
------------|-----------------|--------------
``2019.1.0``|``-Python-3.7.4``|``foss/2019b``

<a anchor="fermi-lite"/>
### *fermi-lite*

Standalone C library for assembling Illumina short reads in small regions.

*homepage*: https://github.com/lh3/fermi-lite

version     |toolchain
------------|-----------------------------------------------------------------------------
``20190320``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``

<a anchor="ferret"/>
### *Ferret*

Ferret is an interactive computer visualization and analysis environment designed to meet the needs of oceanographers and meteorologists analyzing large and complex gridded data sets.

*homepage*: https://ferret.pmel.noaa.gov/Ferret/

version  |toolchain
---------|---------------
``7.3``  |``intel/2017b``
``7.5.0``|``foss/2019b``

<a anchor="festival"/>
### *festival*

University of Edinburgh's Festival Speech Synthesis Systems is a free software multi-lingual speech synthesis workbench that runs on multiple-platforms offering black box text to speech, as well as an open architecture for research in speech synthesis. It designed as a component of large speech technology systems. 

*homepage*: ['http://festvox.org/festival/']

version  |toolchain
---------|-----------------
``2.5.0``|``GCCcore/9.3.0``

<a anchor="fetchmg"/>
### *fetchMG*

The program “fetchMG” was written to extract the 40 MGs from genomes and metagenomes  in an easy and accurate manner.

*homepage*: http://vm-lux.embl.de/~mende/fetchMG/about.html

version|toolchain
-------|------------------------------------
``1.0``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``

<a anchor="ffc"/>
### *FFC*

The FEniCS Form Compiler (FFC) is a compiler for finite element variational forms.

*homepage*: https://bitbucket.org/fenics-project/ffc

version           |versionsuffix    |toolchain
------------------|-----------------|--------------
``2018.1.0``      |``-Python-3.6.4``|``foss/2018a``
``2019.1.0.post0``|``-Python-3.7.4``|``foss/2019b``

<a anchor="fflas-ffpack"/>
### *FFLAS-FFPACK*

Finite Field Linear Algebra Subroutines / Package

*homepage*: https://linbox-team.github.io/fflas-ffpack/

version  |toolchain
---------|--------------
``2.2.0``|``foss/2016a``

<a anchor="ffmpeg"/>
### *FFmpeg*

A complete, cross-platform solution to record, convert and stream audio and video.

*homepage*: https://www.ffmpeg.org/

version    |toolchain
-----------|---------------------------------------------------
``0.10.16``|``gimkl/2.11.5``, ``intel/2016a``
``2.8.6``  |``intel/2016a``
``2.8.7``  |``foss/2016a``, ``intel/2016a``
``3.0.2``  |``foss/2016a``, ``intel/2016a``
``3.1.3``  |``foss/2016b``, ``intel/2016b``
``3.2.4``  |``gimkl/2017a``
``3.3.1``  |``foss/2016b``
``3.3.4``  |``intel/2017a``
``3.4``    |``GCCcore/6.4.0``
``3.4.1``  |``foss/2017b``, ``intel/2017b``
``3.4.2``  |``foss/2018a``, ``intel/2018a``
``3.4.5``  |``foss/2018b``
``4.0``    |``foss/2018a``, ``intel/2018a``
``4.0.1``  |``intel/2018a``
``4.1``    |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``4.1.3``  |``GCCcore/8.2.0``
``4.2.1``  |``GCCcore/8.3.0``
``4.2.2``  |``GCCcore/9.3.0``
``4.3.1``  |``GCCcore/10.2.0``
``4.3.2``  |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``4.4.2``  |``GCCcore/11.3.0``
``5.0.1``  |``GCCcore/11.3.0``

<a anchor="ffmpi"/>
### *ffmpi*

Fujitsu Compiler based compiler toolchain, including Fujitsu MPI for MPI support.

*homepage*: (none)

version  |toolchain
---------|----------
``4.5.0``|``system``

<a anchor="ffnet"/>
### *ffnet*

Feed-forward neural network solution for python

*homepage*: http://ffnet.sourceforge.net/

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.8.3``|``-Python-2.7.11``|``intel/2016a``

<a anchor="fftlib"/>
### *fftlib*

A library that intercepts FFTW calls and adds features on top of it.  In particular, it enables FFT plan reuse when there are multiple calls for the same geometry.

*homepage*: https://github.com/flwende/fftlib

version     |toolchain
------------|---------------
``20170628``|``gompi/2020b``

<a anchor="fftw"/>
### *FFTW*

FFTW is a C subroutine library for computing the discrete Fourier transform (DFT)  in one or more dimensions, of arbitrary input size, and of both real and complex data.

*homepage*: https://www.fftw.org

version   |versionsuffix|toolchain
----------|-------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.0.0`` |``-fujitsu`` |``FCC/4.5.0``
``2.1.5`` |             |``intel/2016b``, ``intel/2017a``, ``intel/2018b``
``3.3.4`` |             |``gmpich/2016a``, ``gmvapich2/1.7.20``, ``gmvapich2/2016a``, ``gompi/2016.04``, ``gompi/2016.06``, ``gompi/2016.07``, ``gompi/2016a``, ``gompi/2016b``, ``intel/2016a``, ``intel/2016b``
``3.3.5`` |             |``gompi/2016.07``, ``gompi/2016.09``, ``gompi/2016b``, ``intel/2016b``
``3.3.6`` |             |``gimpi/2017b``, ``gimpic/2017b``, ``gompi/2017a``, ``gompi/2017b``, ``gompic/2017b``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intelcuda/2017b``
``3.3.7`` |             |``gimkl/2017a``, ``gimpi/2018a``, ``gmpich/2017.08``, ``gompi/2018a``, ``gompic/2018a``, ``intel/2017b``, ``intel/2018.00``, ``intel/2018.01``, ``intel/2018a``, ``iomkl/2018a``
``3.3.7`` |``-serial``  |``GCC/6.4.0-2.28``
``3.3.8`` |             |``gompi/2018.08``, ``gompi/2018b``, ``gompi/2019a``, ``gompi/2019b``, ``gompi/2020a``, ``gompi/2020b``, ``gompic/2018b``, ``gompic/2019a``, ``gompic/2019b``, ``gompic/2020a``, ``gompic/2020b``, ``iimpi/2020b``, ``intel/2018b``, ``intel/2019a``, ``intel/2019b``, ``intel/2020a``, ``intel/2020b``, ``iomkl/2018b``
``3.3.8`` |``-amd``     |``gompi/2020a``
``3.3.8`` |``-serial``  |``GCC/9.3.0``
``3.3.9`` |             |``gompi/2021a``, ``intel/2021a``
``3.3.10``|             |``GCC/11.3.0``, ``GCC/12.2.0``, ``gompi/2021b``, ``iimpi/2021b``, ``iimpi/2022a``

<a anchor="fftw.mpi"/>
### *FFTW.MPI*

FFTW is a C subroutine library for computing the discrete Fourier transform (DFT) in one or more dimensions, of arbitrary input size, and of both real and complex data.

*homepage*: https://www.fftw.org

version   |toolchain
----------|----------------------------------------------------------------------
``3.3.10``|``gompi/2022.05``, ``gompi/2022.10``, ``gompi/2022a``, ``gompi/2022b``

<a anchor="fgbio"/>
### *fgbio*

A set of tools to analyze genomic data with a focus on Next Generation Sequencing.

*homepage*: https://fulcrumgenomics.github.io/fgbio

version  |toolchain
---------|----------
``1.3.0``|``system``

<a anchor="fgsl"/>
### *FGSL*

FGSL: A Fortran interface to the GNU Scientific Library

*homepage*: https://www.lrz.de/services/software/mathematik/gsl/fortran/

version  |toolchain
---------|---------------
``1.1.0``|``intel/2016b``

<a anchor="fhi-aims"/>
### *FHI-aims*

FHI-aims is an efficient, accurate all-electron, full-potential electronic structure code package for computational molecular and materials science (non-periodic and periodic systems). The code supports DFT (semilocal and hybrid) and many-body perturbation theory. FHI-aims is particularly efficient for molecular systems and nanostructures, while maintaining high numerical accuracy for all production tasks. Production calculations handle up to several thousand atoms and can efficiently use (ten) thousands of cores.  

*homepage*: https://aimsclub.fhi-berlin.mpg.de/

version     |toolchain
------------|---------------
``200112_2``|``intel/2019b``

<a anchor="fiat"/>
### *FIAT*

The FInite element Automatic Tabulator (FIAT) supports generation of arbitrary order instances of the Lagrange elements on lines, triangles, and tetrahedra. It is also capable of generating arbitrary order instances of Jacobi-type quadrature rules on the same element shapes.

*homepage*: https://bitbucket.org/fenics-project/fiat

version     |versionsuffix     |toolchain
------------|------------------|-------------------------------
``1.6.0``   |``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``2018.1.0``|``-Python-3.6.4`` |``foss/2018a``
``2019.1.0``|``-Python-3.7.4`` |``foss/2019b``

<a anchor="figaro"/>
### *FIGARO*

FIGARO: An efficient and objective tool for optimizing microbiome rRNA gene trimming parameters.

*homepage*: https://github.com/Zymo-Research/figaro

version  |toolchain
---------|---------------
``1.1.2``|``intel/2020b``

<a anchor="figuregen"/>
### *FigureGen*

FigureGen is a Fortran program that creates images for ADCIRC files. It reads mesh files (fort.14, etc.), nodal attributes files (fort.13, etc.) and output  files (fort.63, fort.64, maxele.63, etc.). It plots contours, contour lines, and vectors.  Using FigureGen, you can go directly from the ADCIRC input and output files to a  presentation-quality figure, for one or multiple time snaps.  

*homepage*: https://ccht.ccee.ncsu.edu/category/software/figuregen/

version        |toolchain
---------------|--------------
``51-20190516``|``foss/2019a``

<a anchor="fiji"/>
### *Fiji*

Fiji is an image processing package—a 'batteries-included' distribution of  ImageJ, bundling a lot of plugins which facilitate scientific image analysis. This release is based on ImageJ-2.1.0 and Fiji-2.1.1

*homepage*: https://fiji.sc/

version          |versionsuffix|toolchain
-----------------|-------------|----------
``2.9.0``        |``-Java-1.8``|``system``
``2.9.0``        |``-Java-8``  |``system``
``20170530``     |             |``system``
``20191119-2057``|             |``system``
``20201104-1356``|             |``system``

<a anchor="file"/>
### *file*

The file command is 'a file type guesser', that is, a command-line tool  that tells you in words what kind of data a file contains.

*homepage*: https://www.darwinsys.com/file/

version |toolchain
--------|------------------------------------
``5.17``|``GCC/4.8.2``
``5.25``|``intel/2016a``
``5.28``|``foss/2016b``
``5.30``|``intel/2017a``
``5.33``|``GCCcore/6.4.0``
``5.35``|``GCCcore/7.3.0``
``5.38``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``5.39``|``GCCcore/10.2.0``
``5.40``|``GCCcore/10.3.0``
``5.41``|``GCCcore/11.2.0``
``5.43``|``GCCcore/11.3.0``

<a anchor="filevercmp"/>
### *filevercmp*

filevercmp function as in sort --version-sort.

*homepage*: https://github.com/ekg/filevercmp

version     |toolchain
------------|----------------------------------------------------------
``20141119``|``GCCcore/9.3.0``
``20191210``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``

<a anchor="filtlong"/>
### *Filtlong*

Filtlong is a tool for filtering long reads by quality. It can take a set  of long reads and produce a smaller, better subset. It uses both read length (longer is better)  and read identity (higher is better) when choosing which reads pass the filter

*homepage*: https://github.com/rrwick/Filtlong

version  |toolchain
---------|------------------------------
``0.2.0``|``GCC/10.2.0``, ``foss/2016b``
``0.2.1``|``GCC/10.3.0``

<a anchor="find_circ"/>
### *find_circ*

circRNA detection from RNA-seq reads

*homepage*: https://github.com/marvin-jens/find_circ

version         |versionsuffix     |toolchain
----------------|------------------|---------------
``1.2-20170228``|``-Python-2.7.14``|``intel/2017b``

<a anchor="findhap"/>
### *findhap*

Find haplotypes and impute genotypes using multiple chip sets and sequence data

*homepage*: http://aipl.arsusda.gov/software/findhap/

version|toolchain
-------|----------
``4``  |``system``

<a anchor="findutils"/>
### *findutils*

findutils: The GNU find, locate, updatedb, and xargs utilities

*homepage*: http://www.gnu.org/software/findutils/findutils.html

version  |toolchain
---------|-------------
``4.4.2``|``GCC/4.8.2``

<a anchor="fineradstructure"/>
### *fineRADstructure*

A package for population structure inference from RAD-seq data

*homepage*: http://cichlid.gurdon.cam.ac.uk/fineRADstructure.html

version     |toolchain
------------|---------------
``20180709``|``intel/2018a``

<a anchor="finestructure"/>
### *fineSTRUCTURE*

fineSTRUCTURE is a fast and powerful algorithm for identifying population structure using  dense sequencing data.

*homepage*: https://people.maths.bris.ac.uk/~madjl/finestructure/finestructure_info.html

version  |toolchain
---------|---------------
``2.1.3``|``intel/2017b``

<a anchor="fio"/>
### *fio*

Flexible I/O tester

*homepage*: https://github.com/axboe/fio

version |toolchain
--------|------------------
``3.30``|``GCCcore/10.3.0``
``3.32``|``GCCcore/11.3.0``

<a anchor="fiona"/>
### *Fiona*

Fiona is designed to be simple and dependable. It focuses on reading and writing data in standard Python IO style and relies upon familiar Python types and protocols such as files, dictionaries, mappings, and iterators instead of classes specific to OGR. Fiona can read and write real-world data using multi-layered GIS formats and zipped virtual file systems and integrates readily with other Python GIS packages such as pyproj, Rtree, and Shapely.

*homepage*: https://github.com/Toblerity/Fiona

version         |versionsuffix    |toolchain
----------------|-----------------|---------------------------------------------------
``1.8.13``      |``-Python-3.7.4``|``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``
``1.8.13.post1``|``-Python-3.7.2``|``foss/2019a``
``1.8.16``      |``-Python-3.8.2``|``foss/2020a``
``1.8.20``      |                 |``foss/2020b``, ``foss/2021a``, ``intel/2020b``
``1.8.21``      |                 |``foss/2021b``, ``foss/2022a``

<a anchor="firefox"/>
### *Firefox*

Firefox is a free, open source Web browser for Windows, Linux and Mac OS X. It is based on the  Mozilla code base and offers customization options and features such as its capability to block pop-up windows,  tabbed browsing, privacy and security measures, smart searching, and RSS live bookmarks.

*homepage*: https://www.mozilla.org/en-US/firefox/new/

version   |toolchain
----------|----------
``44.0.2``|``system``

<a anchor="firestarter"/>
### *FIRESTARTER*

FIRESTARTER: A Processor Stress Test Utility. FIRESTARTER maximizes the energy consumption of 64-Bit x86 processors by generating heavy load on the execution units as well as transferring data between the cores and multiple levels of the memory hierarchy.

*homepage*: https://github.com/tud-zih-energy/FIRESTARTER/

version|toolchain
-------|------------------------------------
``2.0``|``gcccuda/2020a``, ``gcccuda/2020b``

<a anchor="fireworks"/>
### *FireWorks*

FireWorks helps run calculation workflows, with a centralized workflow server  controlling many worker nodes.

*homepage*: https://pypi.python.org/pypi/FireWorks

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.4.2``|``-Python-2.7.13``|``intel/2017a``

<a anchor="fit-sne"/>
### *FIt-SNE*

t-distributed stochastic neighbor embedding (t-SNE) is widely used for  visualizing single-cell RNA-sequencing (scRNA-seq) data, but it scales poorly to large  datasets. We dramatically accelerate t-SNE, obviating the need for data downsampling, and  hence allowing visualization of rare cell populations. Furthermore, we implement a  heatmap-style visualization for scRNA-seq based on one-dimensional t-SNE for simultaneously  visualizing the expression patterns of thousands of genes.

*homepage*: https://github.com/KlugerLab/FIt-SNE

version  |toolchain
---------|---------------
``1.1.0``|``gompi/2018b``

<a anchor="fix"/>
### *FIX*

FIX attempts to auto-classify ICA components into "good" vs "bad" components, so that the bad components can be removed from the 4D FMRI data.

*homepage*: https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FIX

version    |versionsuffix           |toolchain
-----------|------------------------|--------------
``1.06.12``|``-Octave-Python-3.7.2``|``foss/2019a``

<a anchor="fixesproto"/>
### *fixesproto*

X.org FixesProto protocol headers.

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version|toolchain
-------|-------------------------------------------------
``5.0``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``

<a anchor="flac"/>
### *FLAC*

FLAC stands for Free Lossless Audio Codec, an audio format similar to MP3, but lossless, meaning that audio is compressed in FLAC without any loss in quality.

*homepage*: https://xiph.org/flac/

version  |toolchain
---------|----------------------------------------------------------
``1.3.3``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.3.4``|``GCCcore/11.3.0``

<a anchor="flair"/>
### *FLAIR*

FLAIR (Full-Length Alternative Isoform analysis of RNA)  for the correction, isoform definition, and alternative splicing analysis of noisy reads.  FLAIR has primarily been used for nanopore cDNA, native RNA, and PacBio sequencing reads.

*homepage*: https://github.com/BrooksLabUCSC/flair

version           |versionsuffix    |toolchain
------------------|-----------------|--------------
``1.5``           |``-Python-3.7.4``|``foss/2019b``
``1.5.1-20200630``|``-Python-3.7.4``|``foss/2019b``

<a anchor="flair-nlp"/>
### *flair-NLP*

A very simple framework for state-of-the-art NLP

*homepage*: https://github.com/flairNLP/flair

version   |versionsuffix   |toolchain
----------|----------------|--------------
``0.11.3``|                |``foss/2021a``
``0.11.3``|``-CUDA-11.3.1``|``foss/2021a``

<a anchor="flann"/>
### *FLANN*

FLANN is a library for performing fast approximate nearest neighbor searches in high dimensional spaces.

*homepage*: https://github.com/mariusmuja/flann/

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.8.4``|``-Python-2.7.11``|``intel/2016a``
``1.8.4``|``-Python-2.7.14``|``intel/2017b``
``1.8.4``|``-Python-3.8.2`` |``foss/2020a``
``1.9.1``|``-Python-3.8.2`` |``foss/2020a``

<a anchor="flash"/>
### *FLASH*

FLASH (Fast Length Adjustment of SHort reads) is a very fast and accurate software tool to merge paired-end reads from next-generation sequencing experiments. FLASH is designed to merge pairs of reads when the original DNA fragments are shorter than twice the length of reads. The resulting longer reads can significantly improve genome assemblies. They can also improve transcriptome assembly when FLASH is used to merge RNA-seq data. 

*homepage*: https://ccb.jhu.edu/software/FLASH/

version   |toolchain
----------|-------------------------------------------------------------
``1.2.11``|``GCC/8.3.0``, ``foss/2016a``, ``foss/2018a``, ``foss/2018b``
``2.2.00``|``GCC/11.2.0``, ``foss/2018b``, ``foss/2020b``

<a anchor="flask"/>
### *Flask*

Flask is a lightweight WSGI web application framework. It is designed to make getting started quick and easy, with the ability to scale up to complex applications. This module includes the Flask extensions: Flask-Cors

*homepage*: https://www.palletsprojects.com/p/flask/

version  |versionsuffix    |toolchain
---------|-----------------|------------------
``1.0.2``|``-Python-3.6.6``|``foss/2018b``
``1.1.2``|                 |``GCCcore/10.2.0``
``1.1.2``|``-Python-3.7.4``|``GCCcore/8.3.0``
``1.1.2``|``-Python-3.8.2``|``GCCcore/9.3.0``
``1.1.4``|                 |``GCCcore/10.3.0``
``2.0.2``|                 |``GCCcore/11.2.0``
``2.2.2``|                 |``GCCcore/11.3.0``

<a anchor="flatbuffers"/>
### *flatbuffers*

FlatBuffers: Memory Efficient Serialization Library

*homepage*: https://github.com/google/flatbuffers/

version   |toolchain
----------|----------------------------------------------------------
``1.12.0``|``GCCcore/10.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``2.0.0`` |``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="flatbuffers-python"/>
### *flatbuffers-python*

Python Flatbuffers runtime library.

*homepage*: https://github.com/google/flatbuffers/

version |versionsuffix    |toolchain
--------|-----------------|----------------------------------------------------------
``1.12``|                 |``GCCcore/10.2.0``
``1.12``|``-Python-3.7.4``|``GCCcore/8.3.0``
``2.0`` |                 |``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="fleur"/>
### *FLEUR*

FLEUR is a feature-full, freely available FLAPW (full potential linearized augmented planewave) code,  based on density-functional theory.

*homepage*: http://www.flapw.de/

version  |toolchain
---------|---------------
``0.26e``|``intel/2016a``

<a anchor="flex"/>
### *flex*

Flex (Fast Lexical Analyzer) is a tool for generating scanners. A scanner,  sometimes called a tokenizer, is a program which recognizes lexical patterns  in text. 

*homepage*: http://flex.sourceforge.net/

version   |toolchain
----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.5.35``|``system``
``2.5.38``|``GCC/4.8.2``
``2.5.39``|``GCC/4.9.2``, ``GCC/4.9.2-binutils-2.25``, ``GCC/4.9.3``, ``GCC/4.9.3-binutils-2.25``, ``GCC/5.1.0-binutils-2.25``, ``GCCcore/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GNU/4.9.3-2.25``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``system``
``2.6.0`` |``GCC/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/4.9.4``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.1.0``, ``GCCcore/6.2.0``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
``2.6.2`` |``intel/2016b``
``2.6.3`` |``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/7.1.0``, ``gimkl/2017a``, ``system``
``2.6.4`` |``FCC/4.5.0``, ``GCCcore/10.1.0``, ``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.1.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.1.0``, ``GCCcore/12.2.0``, ``GCCcore/5.5.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/7.4.0``, ``GCCcore/8.1.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/8.4.0``, ``GCCcore/9.1.0``, ``GCCcore/9.2.0``, ``GCCcore/9.3.0``, ``GCCcore/9.4.0``, ``GCCcore/9.5.0``, ``GCCcore/system``, ``system``

<a anchor="flexbar"/>
### *Flexbar*

The program Flexbar preprocesses high-throughput sequencing data efficiently

*homepage*: https://github.com/seqan/flexbar

version  |toolchain
---------|--------------
``3.5.0``|``foss/2018b``

<a anchor="flexiblas"/>
### *FlexiBLAS*

FlexiBLAS is a wrapper library that enables the exchange of the BLAS and LAPACK implementation used by a program without recompiling or relinking it.

*homepage*: https://gitlab.mpi-magdeburg.mpg.de/software/flexiblas-release

version  |toolchain
---------|------------------------------
``3.0.4``|``GCC/10.3.0``, ``GCC/11.2.0``
``3.1.3``|``GCC/11.2.0``
``3.2.0``|``GCC/11.3.0``
``3.2.1``|``GCC/12.2.0``

<a anchor="flexidot"/>
### *FlexiDot*

Highly customizable, ambiguity-aware dotplots for visual sequence analyses 

*homepage*: https://github.com/molbio-dresden/flexidot

version |versionsuffix     |toolchain
--------|------------------|--------------
``1.06``|``-Python-2.7.15``|``foss/2018b``

<a anchor="flink"/>
### *Flink*

Apache Flink is a framework and distributed processing engine for stateful computations over unbounded and bounded data streams. Flink has been designed to run in all common cluster environments, perform computations at in-memory speed and at any scale.

*homepage*: https://flink.apache.org/

version   |versionsuffix      |toolchain
----------|-------------------|----------
``1.11.2``|``-bin-scala_2.11``|``system``

<a anchor="flint"/>
### *FLINT*

FLINT (Fast Library for Number Theory) is a C library in support of computations  in number theory. Operations that can be performed include conversions, arithmetic, computing GCDs,  factoring, solving linear systems, and evaluating special functions. In addition, FLINT provides  various low-level routines for fast arithmetic. FLINT is extensively documented and tested.

*homepage*: https://www.flintlib.org/

version  |toolchain
---------|-----------------------------------------------------------------------------------------------
``2.5.2``|``GCC/7.3.0-2.30``, ``GCC/8.2.0-2.31.1``, ``GCC/8.3.0``, ``iccifort/2018.3.222-GCC-7.3.0-2.30``
``2.7.1``|``GCC/10.3.0``
``2.8.4``|``GCC/11.2.0``

<a anchor="flook"/>
### *flook*

The fortran-Lua-hook library.

*homepage*: https://github.com/ElectronicStructureLibrary/flook

version  |toolchain
---------|-----------------------------------------------------------------------------------------------------------------------------------
``0.8.1``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``iccifort/2020.4.304``, ``intel-compilers/2021.2.0``, ``intel-compilers/2021.4.0``

<a anchor="fltk"/>
### *FLTK*

FLTK is a cross-platform C++ GUI toolkit for UNIX/Linux (X11), Microsoft Windows,  and MacOS X. FLTK provides modern GUI functionality without the bloat and supports 3D graphics via OpenGL  and its built-in GLUT emulation.

*homepage*: https://www.fltk.org

version  |toolchain
---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.3.3``|``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``1.3.4``|``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``fosscuda/2017b``, ``fosscuda/2018a``, ``fosscuda/2018b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``, ``intelcuda/2017b``
``1.3.5``|``GCC/8.2.0-2.31.1``, ``GCC/8.3.0``, ``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``1.3.6``|``GCCcore/10.3.0``
``1.3.7``|``GCCcore/11.2.0``
``1.3.8``|``GCCcore/11.3.0``

<a anchor="fluent"/>
### *FLUENT*

ANSYS FLUENT software contains the broad physical modeling capabilities needed to model flow, turbulence, heat transfer, and reactions for industrial applications ranging from air flow over an aircraft wing to combustion in a furnace, from bubble columns to oil platforms, from blood flow to semiconductor manufacturing, and from clean room design to wastewater treatment plants.

*homepage*: https://www.ansys.com/products/fluids/ansys-fluent

version   |toolchain
----------|----------
``14.5``  |``system``
``15.0.7``|``system``
``16.0``  |``system``
``17.1``  |``system``
``18.0``  |``system``
``18.1``  |``system``
``18.2``  |``system``
``2021R1``|``system``
``2021R2``|``system``

<a anchor="flye"/>
### *Flye*

Flye is a de novo assembler for long and noisy reads, such as those produced by PacBio  and Oxford Nanopore Technologies.

*homepage*: https://github.com/fenderglass/Flye

version  |versionsuffix     |toolchain
---------|------------------|--------------------------------------------
``2.4``  |``-Python-2.7.15``|``intel/2018b``
``2.6``  |``-Python-3.7.2`` |``foss/2019a``
``2.6``  |``-Python-3.7.4`` |``intel/2019b``
``2.8.1``|``-Python-3.8.2`` |``intel/2020a``
``2.8.2``|``-Python-3.8.2`` |``foss/2020a``
``2.8.3``|                  |``GCC/10.2.0``, ``iccifort/2020.4.304``
``2.9``  |                  |``GCC/10.3.0``, ``intel-compilers/2021.2.0``
``2.9.1``|                  |``GCC/11.2.0``

<a anchor="fmilibrary"/>
### *FMILibrary*

FMI library is intended as a foundation for applications interfacing  FMUs (Functional Mockup Units) that follow FMI Standard. This version of the library supports  FMI 1.0 and FMI2.0. See http://www.fmi-standard.org/

*homepage*: https://jmodelica.org/

version  |toolchain
---------|---------------
``2.0.3``|``intel/2018b``

<a anchor="fmm3d"/>
### *FMM3D*

Flatiron Institute Fast Multipole Libraries: a set of libraries to compute N-body interactions governed by the Laplace and Helmholtz equations, to a specified precision, in three dimensions, on a multi-core shared-memory machine.

*homepage*: https://fmm3d.readthedocs.io

version     |toolchain
------------|--------------
``20211018``|``foss/2020b``

<a anchor="fmpy"/>
### *FMPy*

FMPy is a free Python library to simulate Functional Mock-up Units (FMUs).

*homepage*: https://github.com/CATIA-Systems/FMPy

version  |toolchain
---------|--------------
``0.3.2``|``foss/2021a``

<a anchor="fmriprep"/>
### *FMRIprep*

FMRIprep is a functional magnetic resonance imaging (fMRI) data preprocessing pipeline that is  designed to provide an easily accessible, state-of-the-art interface that is robust to variations in scan  acquisition protocols and that requires minimal user input, while providing easily interpretable and comprehensive  error and output reporting.

*homepage*: https://fmriprep.readthedocs.io

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.1.8``|``-Python-3.6.6``|``foss/2018b``
``1.4.1``|``-Python-3.6.6``|``foss/2018b``

<a anchor="fms"/>
### *FMS*

The Flexible Modeling System (FMS) is a software framework for   supporting the efficient development, construction, execution, and scientific  interpretation of atmospheric, oceanic, and climate system models.

*homepage*: https://github.com/NOAA-GFDL/FMS

version    |toolchain
-----------|--------------------------------
``2022.02``|``gompi/2022a``, ``iimpi/2022a``

<a anchor="fmt"/>
### *fmt*

fmt (formerly cppformat) is an open-source formatting library.

*homepage*: http://fmtlib.net/

version  |toolchain
---------|------------------------------------
``3.0.1``|``foss/2016b``, ``intel/2016b``
``3.0.2``|``GCCcore/6.4.0``, ``intel/2017a``
``5.3.0``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``6.2.1``|``GCCcore/9.3.0``
``7.0.3``|``GCCcore/9.3.0``
``7.1.1``|``GCCcore/11.2.0``
``9.1.0``|``GCCcore/11.3.0``

<a anchor="fobis"/>
### *FoBiS*

A Fortran Building System for automatic building modern Fortran projects

*homepage*: https://github.com/szaghi/FoBiS

version  |toolchain
---------|------------------
``3.0.5``|``GCCcore/10.3.0``

<a anchor="foldx"/>
### *FoldX*

FoldX is used to provide a fast and quantitative estimation of the importance of the interactions  contributing to the stability of proteins and protein complexes.

*homepage*: http://foldx.crg.es/

version        |toolchain
---------------|----------
``2.5.2``      |``system``
``3.0-beta5.1``|``system``
``3.0-beta6``  |``system``
``3.0-beta6.1``|``system``

<a anchor="fontconfig"/>
### *fontconfig*

Fontconfig is a library designed to provide system-wide font configuration,  customization and application access. 

*homepage*: https://www.freedesktop.org/wiki/Software/fontconfig/

version    |versionsuffix     |toolchain
-----------|------------------|--------------------------------------------------------------------------------------
``2.11.94``|                  |``foss/2016a``, ``intel/2016a``
``2.11.95``|                  |``foss/2016a``, ``intel/2016a``
``2.12.1`` |                  |``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``foss/2016b``, ``gimkl/2017a``, ``intel/2016b``
``2.12.1`` |``-libpng-1.6.29``|``GCCcore/6.3.0``
``2.12.4`` |                  |``GCCcore/6.4.0``
``2.12.6`` |                  |``GCCcore/6.4.0``
``2.13.0`` |                  |``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``2.13.1`` |                  |``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``2.13.92``|                  |``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``2.13.93``|                  |``GCCcore/10.3.0``
``2.13.94``|                  |``GCCcore/11.2.0``
``2.14.0`` |                  |``GCCcore/11.3.0``
``2.14.1`` |                  |``GCCcore/12.2.0``

<a anchor="fontsproto"/>
### *fontsproto*

X11 font extension wire protocol

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version  |toolchain
---------|-------------------------------------------------
``2.1.3``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``

<a anchor="ford"/>
### *FORD*

FORD is an automatic documentation generator for modern Fortran programs

*homepage*: https://github.com/Fortran-FOSS-Programmers/ford/wiki

version   |toolchain
----------|------------------
``6.1.1`` |``GCCcore/10.2.0``
``6.1.6`` |``GCCcore/10.3.0``
``6.1.15``|``GCCcore/11.3.0``

<a anchor="foss"/>
### *foss*

GNU Compiler Collection (GCC) based compiler toolchain, including  OpenMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.

*homepage*: https://easybuild.readthedocs.io/en/master/Common-toolchains.html#foss-toolchain

version    |toolchain
-----------|----------
``2016.04``|``system``
``2016.06``|``system``
``2016.07``|``system``
``2016.09``|``system``
``2016a``  |``system``
``2016b``  |``system``
``2017a``  |``system``
``2017b``  |``system``
``2018.08``|``system``
``2018a``  |``system``
``2018b``  |``system``
``2019a``  |``system``
``2019b``  |``system``
``2020a``  |``system``
``2020b``  |``system``
``2021a``  |``system``
``2021b``  |``system``
``2022.05``|``system``
``2022.10``|``system``
``2022a``  |``system``
``2022b``  |``system``

<a anchor="fosscuda"/>
### *fosscuda*

GCC based compiler toolchain __with CUDA support__, and including  OpenMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.

*homepage*: (none)

version  |toolchain
---------|----------
``2017b``|``system``
``2018a``|``system``
``2018b``|``system``
``2019a``|``system``
``2019b``|``system``
``2020a``|``system``
``2020b``|``system``

<a anchor="fox"/>
### *FoX*

FoX is an XML library written in Fortran 95. It allows software developers to read, write and modify XML documents from Fortran applications without the complications of dealing with multi-language development.

*homepage*: http://homepages.see.leeds.ac.uk/~earawa/FoX/

version  |toolchain
---------|---------------------------------------------------------------
``4.1.2``|``GCC/11.2.0``, ``GCC/9.3.0``, ``intel/2017b``, ``intel/2018a``

<a anchor="fox-toolkit"/>
### *FOX-Toolkit*

FOX is a C++ based Toolkit for developing Graphical User Interfaces easily and effectively. It offers a wide, and growing, collection of Controls, and provides state of the art facilities such as drag and drop, selection, as well as OpenGL widgets for 3D graphical manipulation. FOX also implements icons, images, and user-convenience features such as status line help, and tooltips.

*homepage*: https://www.fox-toolkit.org/

version   |toolchain
----------|-------------------------------------
``1.6.57``|``GCCcore/11.2.0``, ``GCCcore/9.3.0``

<a anchor="fpm"/>
### *FPM*

Effing package management! Build packages for multiple platforms (deb, rpm, etc) with great ease  and sanity.

*homepage*: https://github.com/jordansissel/fpm

version  |versionsuffix  |toolchain
---------|---------------|----------
``1.3.3``|``-Ruby-2.1.6``|``system``

<a anchor="fpocket"/>
### *fpocket*

The fpocket suite of programs is a very fast open source protein pocket detection algorithm based on Voronoi tessellation. The platform is suited for the scientific community willing to develop new scoring functions and  extract pocket descriptors on a large scale level.

*homepage*: https://github.com/Discngine/fpocket

version    |toolchain
-----------|---------------
``3.1.4.2``|``gompi/2020a``

<a anchor="fqtrim"/>
### *fqtrim*

fqtrim is a versatile stand-alone utility that can be used to trim adapters, poly-A tails,  terminal unknown bases (Ns) and low quality 3' regions in reads from high-throughput next-generation sequencing  machines.

*homepage*: http://ccb.jhu.edu/software/fqtrim/

version  |toolchain
---------|---------------
``0.9.4``|``intel/2016b``
``0.9.5``|``intel/2017a``

<a anchor="fqzcomp"/>
### *fqzcomp*

Fqzcomp is a basic fastq compressor, designed primarily for high performance.  Despite that it is comparable to bzip2 for compression levels.

*homepage*: http://sourceforge.net/projects/fqzcomp/

version|toolchain
-------|-------------
``4.6``|``GCC/4.8.2``

<a anchor="fraggenescan"/>
### *FragGeneScan*

FragGeneScan is an application for finding (fragmented) genes in short reads.

*homepage*: http://omics.informatics.indiana.edu/FragGeneScan/

version |toolchain
--------|---------------------------------------------------------------------------------------------
``1.31``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/8.2.0``, ``foss/2018b``

<a anchor="franz"/>
### *FRANz*

A fast and flexible parentage inference program for natural populations.

*homepage*: https://www.bioinf.uni-leipzig.de/Software/FRANz

version  |toolchain
---------|--------------
``2.0.0``|``foss/2018a``

<a anchor="freebarcodes"/>
### *FreeBarcodes*

A package for the generation and decoding of FREE divergence error-correcting DNA barcodes

*homepage*: https://github.com/hawkjo/freebarcodes

version   |toolchain
----------|--------------
``3.0.a5``|``foss/2021b``

<a anchor="freebayes"/>
### *freebayes*

Bayesian haplotype-based genetic polymorphism discovery and genotyping.

*homepage*: https://github.com/freebayes/freebayes

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.3.5``|                 |``GCC/10.2.0``
``1.3.5``|``-Python-3.8.2``|``GCC/9.3.0``
``1.3.6``|``-R-4.1.0``     |``foss/2021a``
``1.3.6``|``-R-4.1.2``     |``foss/2021b``

<a anchor="freefem"/>
### *FreeFEM*

FreeFEM offers a fast interpolation algorithm and a language for the manipulation  of data on multiple meshes.

*homepage*: https://freefem.org

version|versionsuffix    |toolchain
-------|-----------------|--------------
``4.5``|``-Python-3.7.4``|``foss/2019b``

<a anchor="freefem++"/>
### *FreeFem++*

FreeFem++ is a partial differential equation solver. It has its own language. freefem scripts can solve multiphysics non linear systems in 2D and 3D. 

*homepage*: http://www.freefem.org/

version   |versionsuffix       |toolchain
----------|--------------------|---------------
``3.58``  |``-downloaded-deps``|``foss/2017b``
``3.60``  |``-downloaded-deps``|``intel/2018a``
``3.61-1``|``-downloaded-deps``|``intel/2018a``

<a anchor="freeglut"/>
### *freeglut*

freeglut is a completely OpenSourced alternative to the OpenGL Utility Toolkit (GLUT) library.

*homepage*: http://freeglut.sourceforge.net/

version  |versionsuffix   |toolchain
---------|----------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``3.0.0``|                |``GCCcore/8.2.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``
``3.0.0``|``-Mesa-11.2.1``|``foss/2016a``, ``intel/2016a``
``3.2.1``|                |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``3.2.2``|                |``GCCcore/11.3.0``

<a anchor="freeimage"/>
### *FreeImage*

FreeImage is an Open Source library project for developers who would like to support popular graphics image formats like PNG, BMP, JPEG, TIFF and others as needed by today's multimedia applications. FreeImage is easy to use, fast, multithreading safe.

*homepage*: http://freeimage.sourceforge.net

version   |toolchain
----------|---------------------------------------------------------------------------------------------------------------------------------------
``3.18.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/7.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

<a anchor="freesasa"/>
### *FreeSASA*

FreeSASA is a command line tool, C-library and Python module for calculating solvent accessible surface areas (SASA).

*homepage*: https://freesasa.github.io

version  |toolchain
---------|-------------
``2.0.3``|``GCC/8.3.0``

<a anchor="freesurfer"/>
### *FreeSurfer*

FreeSurfer is a set of tools for analysis and visualization  of structural and functional brain imaging data.  FreeSurfer contains a fully  automatic structural imaging stream for processing cross sectional and  longitudinal data.

*homepage*: https://surfer.nmr.mgh.harvard.edu/

version  |versionsuffix      |toolchain
---------|-------------------|----------
``5.3.0``|``-centos4_x86_64``|``system``
``5.3.0``|``-centos6_x86_64``|``system``
``6.0.0``|``-centos6_x86_64``|``system``
``6.0.1``|``-centos6_x86_64``|``system``
``7.1.1``|``-centos6_x86_64``|``system``
``7.1.1``|``-centos7_x86_64``|``system``
``7.1.1``|``-centos8_x86_64``|``system``
``7.2.0``|``-centos7_x86_64``|``system``
``7.2.0``|``-centos8_x86_64``|``system``
``7.2.0``|``-ubuntu18_amd64``|``system``
``7.3.2``|``-centos7_x86_64``|``system``
``7.3.2``|``-centos8_x86_64``|``system``

<a anchor="freetds"/>
### *FreeTDS*

FreeTDS is a set of libraries for Unix and Linux that allows your programs to  natively talk to Microsoft SQL Server and Sybase databases.

*homepage*: https://www.freetds.org/

version  |toolchain
---------|------------------
``1.3.3``|``GCCcore/10.3.0``

<a anchor="freetype"/>
### *freetype*

FreeType 2 is a software font engine that is designed to be small, efficient,  highly customizable, and portable while capable of producing high-quality  output (glyph images). It can be used in graphics libraries, display servers,  font conversion tools, text image generation tools, and many other products  as well. 

*homepage*: https://freetype.org

version   |versionsuffix     |toolchain
----------|------------------|----------------------------------------------------------------------
``2.6.2`` |                  |``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``2.6.3`` |                  |``foss/2016a``, ``intel/2016a``
``2.6.5`` |                  |``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``foss/2016b``, ``intel/2016b``
``2.7``   |                  |``foss/2016b``, ``intel/2016b``
``2.7.1`` |                  |``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``gimkl/2017a``, ``intel/2016b``
``2.7.1`` |``-libpng-1.6.29``|``GCCcore/6.3.0``
``2.8``   |                  |``GCCcore/6.4.0``
``2.8.1`` |                  |``GCCcore/6.4.0``
``2.9``   |                  |``GCCcore/6.4.0``
``2.9.1`` |                  |``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``2.10.1``|                  |``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``2.10.3``|                  |``GCCcore/10.2.0``
``2.10.4``|                  |``GCCcore/10.3.0``
``2.11.0``|                  |``GCCcore/11.2.0``
``2.12.1``|                  |``GCCcore/11.3.0``, ``GCCcore/12.2.0``

<a anchor="freetype-py"/>
### *freetype-py*

Python binding for the freetype library

*homepage*: https://github.com/rougier/freetype-py

version  |versionsuffix    |toolchain
---------|-----------------|-----------------
``2.2.0``|``-Python-3.7.4``|``GCCcore/8.3.0``

<a anchor="freexl"/>
### *FreeXL*

FreeXL is an open source library to extract valid data from within an  Excel (.xls) spreadsheet. 

*homepage*: https://www.gaia-gis.it/fossil/freexl/index

version  |toolchain
---------|-------------------------------------------------------
``1.0.2``|``foss/2016b``, ``intel/2016b``
``1.0.3``|``GCCcore/6.4.0``
``1.0.5``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``

<a anchor="freud-analysis"/>
### *freud-analysis*

The freud Python library provides a simple, flexible, powerful set of tools for analyzing trajectories obtained from molecular dynamics or Monte Carlo simulations. High performance, parallelized C++ is used to compute standard tools such as radial distribution functions, correlation functions, order parameters, and clusters, as well as original analysis methods including potentials of mean force and torque (PMFTs) and local environment matching. The freud library supports many input formats and outputs NumPy arrays, enabling integration with the scientific Python ecosystem for many typical materials science workflows.

*homepage*: https://github.com/glotzerlab/freud

version  |toolchain
---------|--------------
``2.6.2``|``foss/2020b``

<a anchor="fribidi"/>
### *FriBidi*

The Free Implementation of the Unicode Bidirectional Algorithm. 

*homepage*: https://github.com/fribidi/fribidi

version   |toolchain
----------|----------------------------------------------------------
``1.0.1`` |``foss/2018a``, ``intel/2018a``
``1.0.2`` |``GCCcore/6.4.0``
``1.0.5`` |``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``1.0.9`` |``GCCcore/9.3.0``
``1.0.10``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.0.12``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``

<a anchor="fruit"/>
### *FRUIT*

FORTRAN Unit Test Framework (FRUIT)

*homepage*: https://fortranxunit.sourceforge.io

version  |versionsuffix  |toolchain
---------|---------------|-------------------------------
``3.4.3``|``-Ruby-2.5.1``|``foss/2018a``, ``intel/2018a``

<a anchor="fruit_processor"/>
### *FRUIT_processor*

FORTRAN Unit Test Framework (FRUIT)

*homepage*: https://fortranxunit.sourceforge.io

version  |versionsuffix  |toolchain
---------|---------------|-------------------------------
``3.4.3``|``-Ruby-2.5.1``|``foss/2018a``, ``intel/2018a``

<a anchor="fsl"/>
### *FSL*

FSL is a comprehensive library of analysis tools for FMRI, MRI and DTI brain imaging data.

*homepage*: https://www.fmrib.ox.ac.uk/fsl/

version    |versionsuffix                 |toolchain
-----------|------------------------------|------------------------------------------------
``5.0.9``  |                              |``intel/2016a``
``5.0.9``  |``-Mesa-11.2.1``              |``intel/2016a``
``5.0.9``  |``-centos6_64``               |``system``
``5.0.10`` |                              |``foss/2017b``, ``intel/2017a``, ``intel/2017b``
``5.0.11`` |                              |``foss/2018b``
``5.0.11`` |``-Python-3.6.6``             |``foss/2018b``
``6.0.1``  |``-Python-2.7.15``            |``foss/2019a``
``6.0.1``  |``-Python-3.7.2``             |``foss/2019a``
``6.0.2``  |``-Python-2.7.15``            |``foss/2018b``, ``foss/2019a``
``6.0.2``  |``-Python-2.7.15-CUDA-9.2.88``|``foss/2018b``
``6.0.2``  |``-Python-3.7.2``             |``foss/2019a``
``6.0.3``  |``-Python-3.7.4``             |``foss/2019b``
``6.0.4``  |``-Python-3.7.4``             |``foss/2019b``
``6.0.5.1``|                              |``foss/2021a``

<a anchor="fsleyes"/>
### *FSLeyes*

FSLeyes is the FSL image viewer.

*homepage*: https://git.fmrib.ox.ac.uk/fsl/fsleyes/fsleyes

version   |versionsuffix     |toolchain
----------|------------------|---------------
``0.15.0``|``-Python-2.7.13``|``intel/2017a``

<a anchor="fsom"/>
### *fsom*

A tiny C library for managing SOM (Self-Organizing Maps) neural networks.

*homepage*: https://github.com/ekg/fsom

version     |toolchain
------------|-----------------------------------------------------------------------------
``20141119``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``

<a anchor="fson"/>
### *FSON*

Fortran 95 JSON Parser

*homepage*: https://github.com/josephalevin/fson

version  |toolchain
---------|--------------
``1.0.5``|``GCC/10.2.0``

<a anchor="ftgl"/>
### *FTGL*

FTGL is a free open source library to enable developers to use arbitrary fonts in their OpenGL (www.opengl.org) applications. 

*homepage*: https://github.com/frankheckenbach/ftgl

version      |toolchain
-------------|----------------------------------------------------------------------------------------------------------
``2.1.3-rc5``|``GCCcore/10.2.0``, ``GCCcore/8.2.0``, ``foss/2017b``, ``foss/2018a``, ``fosscuda/2018b``, ``intel/2017b``
``2.4.0``    |``GCCcore/11.3.0``

<a anchor="fujitsu"/>
### *Fujitsu*

Toolchain using Fujitsu compilers and libraries.

*homepage*: (none)

version  |toolchain
---------|----------
``21.05``|``system``

<a anchor="fullrmc"/>
### *fullrmc*

Reverse Monte Carlo (RMC) is probably best known for its applications  in condensed matter physics and solid state chemistry. fullrmc which stands for  FUndamental Library Language for Reverse Monte Carlo is different than traditional  RMC but a stochastic modelling method to solve an inverse problem whereby  an atomic/molecular model is adjusted until its atoms position havei  the greatest consistency with a set of experimental data.

*homepage*: https://bachiraoun.github.io/fullrmc

version  |versionsuffix     |toolchain
---------|------------------|---------------
``3.2.0``|``-Python-2.7.14``|``intel/2018a``

<a anchor="fumi_tools"/>
### *fumi_tools*

This tool is intended to deduplicate UMIs from single-end and paired-end  sequencing data. Reads are considered identical when their UMIs have the same sequence,  they have the same length and map at the same position.

*homepage*: https://ccb-gitlab.cs.uni-saarland.de/tobias/fumi_tools

version   |versionsuffix    |toolchain
----------|-----------------|------------------------------
``0.18.2``|                 |``GCC/10.3.0``, ``GCC/11.2.0``
``0.18.2``|``-Python-3.6.6``|``foss/2018b``

<a anchor="funwave-tvd"/>
### *FUNWAVE-TVD*

FUNWAVE–TVD is the TVD version of the fully nonlinear Boussinesq wave model (FUNWAVE)  initially developed by Kirby et al.

*homepage*: https://fengyanshi.github.io

version         |versionsuffix|toolchain
----------------|-------------|---------------
``3.1-20170525``|             |``intel/2017a``
``3.1-20170525``|``-no-storm``|``intel/2017a``

<a anchor="fuse"/>
### *FUSE*

The reference implementation of the Linux FUSE (Filesystem in Userspace) interface

*homepage*: https://github.com/libfuse/libfuse

version  |toolchain
---------|---------------
``3.2.6``|``intel/2018a``
``3.4.1``|``foss/2018a``

<a anchor="fuseq"/>
### *FuSeq*

FuSeq is a novel method to discover fusion genes from paired-end RNA sequencing data.

*homepage*: https://github.com/nghiavtr/FuSeq

version  |toolchain
---------|---------------
``1.1.2``|``gompi/2019b``

<a anchor="fusioncatcher"/>
### *FusionCatcher*

FusionCatcher searches for novel/known somatic fusion genes, translocations, and chimeras in RNA-seq data (paired-end or single-end reads from Illumina NGS platforms like Solexa/HiSeq/NextSeq/MiSeq/MiniSeq) from diseased samples.

*homepage*: https://github.com/ndaniel/fusioncatcher

version |versionsuffix     |toolchain
--------|------------------|--------------
``1.20``|``-Python-2.7.16``|``foss/2019b``
``1.30``|``-Python-2.7.16``|``foss/2019b``

<a anchor="futhark"/>
### *futhark*

Futhark is a small programming language designed to be compiled to efficient parallel code. It     is a statically typed, data-parallel, and purely functional array language in the ML family, and comes     with a heavily optimising ahead-of-time compiler that presently generates GPU code via CUDA and OpenCL,     although the language itself is hardware-agnostic and can also run on multicore CPUs

*homepage*: https://futhark-lang.org/

version   |toolchain
----------|----------
``0.19.5``|``system``

<a anchor="futile"/>
### *futile*

The FUTILE project (Fortran Utilities for the Treatment of Innermost Level of Executables) is a set of modules   and wrapper that encapsulate the most common low-level operations of a Fortran code.  

*homepage*: https://launchpad.net/futile

version  |toolchain
---------|-------------------------------------------------------------------------------------------------
``1.8.3``|``foss/2020b``, ``foss/2021a``, ``foss/2021b``, ``intel/2020b``, ``intel/2021a``, ``intel/2021b``

<a anchor="future"/>
### *future*

python-future is the missing compatibility layer between Python 2 and Python 3.

*homepage*: https://python-future.org/

version   |versionsuffix     |toolchain
----------|------------------|------------------------------------------------
``0.16.0``|``-Python-2.7.14``|``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``0.16.0``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``0.16.0``|``-Python-3.6.3`` |``intel/2017b``
``0.16.0``|``-Python-3.6.6`` |``foss/2018b``

<a anchor="fxtract"/>
### *fxtract*

Extract sequences from a fastx (fasta or fastq) file given a subsequence.

*homepage*: https://github.com/ctSkennerton/fxtract

version|toolchain
-------|-----------------
``2.3``|``GCCcore/8.2.0``

<a anchor="g"/>
### *G*


<a href="#g-phocs">G-PhoCS</a> - <a href="#g2clib">g2clib</a> - <a href="#g2lib">g2lib</a> - <a href="#g2log">g2log</a> - <a href="#gaia">Gaia</a> - <a href="#gamess-us">GAMESS-US</a> - <a href="#gap">gap</a> - <a href="#gapcloser">GapCloser</a> - <a href="#gapfiller">GapFiller</a> - <a href="#gappa">gappa</a> - <a href="#garli">GARLI</a> - <a href="#garnett">garnett</a> - <a href="#gat">GAT</a> - <a href="#gate">GATE</a> - <a href="#gatk">GATK</a> - <a href="#gaussian">Gaussian</a> - <a href="#gawk">gawk</a> - <a href="#gblocks">Gblocks</a> - <a href="#gbprocess">GBprocesS</a> - <a href="#gbs2ploidy">gbs2ploidy</a> - <a href="#gc">gc</a> - <a href="#gc3pie">GC3Pie</a> - <a href="#gcc">GCC</a> - <a href="#gcccore">GCCcore</a> - <a href="#gcccuda">gcccuda</a> - <a href="#gcloud">gcloud</a> - <a href="#gconf">GConf</a> - <a href="#gcta">GCTA</a> - <a href="#gctf">Gctf</a> - <a href="#gd">GD</a> - <a href="#gdal">GDAL</a> - <a href="#gdb">GDB</a> - <a href="#gdbgui">gdbgui</a> - <a href="#gdbm">gdbm</a> - <a href="#gdc-client">gdc-client</a> - <a href="#gdchart">GDCHART</a> - <a href="#gdcm">GDCM</a> - <a href="#gdgraph">GDGraph</a> - <a href="#gdist">gdist</a> - <a href="#gdk-pixbuf">Gdk-Pixbuf</a> - <a href="#gdrcopy">GDRCopy</a> - <a href="#geant4">Geant4</a> - <a href="#geant4-data">Geant4-data</a> - <a href="#gearshifft">gearshifft</a> - <a href="#gegl">GEGL</a> - <a href="#gem-library">GEM-library</a> - <a href="#gemma">GEMMA</a> - <a href="#gemmi">gemmi</a> - <a href="#gencore_variant_detection">gencore_variant_detection</a> - <a href="#genemark-et">GeneMark-ET</a> - <a href="#gengetopt">gengetopt</a> - <a href="#genome_profiler">Genome_Profiler</a> - <a href="#genomemapper">GenomeMapper</a> - <a href="#genometester4">GenomeTester4</a> - <a href="#genomethreader">GenomeThreader</a> - <a href="#genometools">GenomeTools</a> - <a href="#genomeworks">GenomeWorks</a> - <a href="#genotypeharmonizer">GenotypeHarmonizer</a> - <a href="#genozip">genozip</a> - <a href="#gensim">gensim</a> - <a href="#geocube">geocube</a> - <a href="#geopandas">geopandas</a> - <a href="#geopy">geopy</a> - <a href="#georges">georges</a> - <a href="#geos">GEOS</a> - <a href="#gerris">Gerris</a> - <a href="#getorb">GETORB</a> - <a href="#getorganelle">GetOrganelle</a> - <a href="#gettext">gettext</a> - <a href="#gexiv2">gexiv2</a> - <a href="#gfbf">gfbf</a> - <a href="#gffcompare">GffCompare</a> - <a href="#gffread">gffread</a> - <a href="#gflags">gflags</a> - <a href="#gfold">GFOLD</a> - <a href="#gh">gh</a> - <a href="#ghc">GHC</a> - <a href="#ghostscript">Ghostscript</a> - <a href="#gibbs2">Gibbs2</a> - <a href="#giflib">giflib</a> - <a href="#gifsicle">gifsicle</a> - <a href="#gimic">GIMIC</a> - <a href="#gimkl">gimkl</a> - <a href="#gimp">GIMP</a> - <a href="#gimpi">gimpi</a> - <a href="#gimpic">gimpic</a> - <a href="#gimps">GIMPS</a> - <a href="#giolf">giolf</a> - <a href="#giolfc">giolfc</a> - <a href="#git">git</a> - <a href="#git-extras">git-extras</a> - <a href="#git-lfs">git-lfs</a> - <a href="#gitpython">GitPython</a> - <a href="#givaro">Givaro</a> - <a href="#giza">Giza</a> - <a href="#gl2ps">GL2PS</a> - <a href="#glade">Glade</a> - <a href="#glew">glew</a> - <a href="#glfw">GLFW</a> - <a href="#glib">GLib</a> - <a href="#glib-networking">glib-networking</a> - <a href="#glibc">glibc</a> - <a href="#glibmm">GLibmm</a> - <a href="#glimmer">GLIMMER</a> - <a href="#glimmerhmm">GlimmerHMM</a> - <a href="#glm">GLM</a> - <a href="#glm-aed">GLM-AED</a> - <a href="#globalarrays">GlobalArrays</a> - <a href="#globus-cli">Globus-CLI</a> - <a href="#globusconnectpersonal">GlobusConnectPersonal</a> - <a href="#glog">glog</a> - <a href="#glpk">GLPK</a> - <a href="#glproto">glproto</a> - <a href="#glucose">Glucose</a> - <a href="#gmap-gsnap">GMAP-GSNAP</a> - <a href="#gmp">GMP</a> - <a href="#gmpich">gmpich</a> - <a href="#gmpolf">gmpolf</a> - <a href="#gmpy2">gmpy2</a> - <a href="#gmsh">gmsh</a> - <a href="#gmt">GMT</a> - <a href="#gmvapich2">gmvapich2</a> - <a href="#gmvolf">gmvolf</a> - <a href="#gnu">GNU</a> - <a href="#gnuplot">gnuplot</a> - <a href="#gnutls">GnuTLS</a> - <a href="#go">Go</a> - <a href="#goalign">goalign</a> - <a href="#goatools">GOATOOLS</a> - <a href="#gobff">gobff</a> - <a href="#gobject-introspection">GObject-Introspection</a> - <a href="#goblf">goblf</a> - <a href="#godon">Godon</a> - <a href="#gofasta">gofasta</a> - <a href="#golf">golf</a> - <a href="#gomkl">gomkl</a> - <a href="#gompi">gompi</a> - <a href="#gompic">gompic</a> - <a href="#google-java-format">google-java-format</a> - <a href="#googletest">googletest</a> - <a href="#gotree">gotree</a> - <a href="#gp2c">GP2C</a> - <a href="#gpaw">GPAW</a> - <a href="#gpaw-setups">GPAW-setups</a> - <a href="#gperf">gperf</a> - <a href="#gperftools">gperftools</a> - <a href="#gpustat">gpustat</a> - <a href="#gpy">GPy</a> - <a href="#gpyopt">GPyOpt</a> - <a href="#gpytorch">GPyTorch</a> - <a href="#grace">Grace</a> - <a href="#gradle">Gradle</a> - <a href="#gradunwarp">gradunwarp</a> - <a href="#graph-tool">graph-tool</a> - <a href="#graphene">Graphene</a> - <a href="#graphicsmagick">GraphicsMagick</a> - <a href="#graphite2">graphite2</a> - <a href="#graphlan">GraPhlAn</a> - <a href="#graphmap">GraphMap</a> - <a href="#graphmap2">GraphMap2</a> - <a href="#graphviz">Graphviz</a> - <a href="#graphviz">graphviz</a> - <a href="#grasp">GRASP</a> - <a href="#grass">GRASS</a> - <a href="#greenlet">Greenlet</a> - <a href="#grep">Grep</a> - <a href="#gretl">gretl</a> - <a href="#grib_api">grib_api</a> - <a href="#grit">GRIT</a> - <a href="#grnboost">GRNBoost</a> - <a href="#groff">groff</a> - <a href="#groimp">GroIMP</a> - <a href="#gromacs">GROMACS</a> - <a href="#gromacswrapper">GromacsWrapper</a> - <a href="#groopm">GroopM</a> - <a href="#groovy">Groovy</a> - <a href="#grpc">gRPC</a> - <a href="#gsea">GSEA</a> - <a href="#gsettings-desktop-schemas">gsettings-desktop-schemas</a> - <a href="#gsl">GSL</a> - <a href="#gsoap">gSOAP</a> - <a href="#gsport">gsport</a> - <a href="#gst-plugins-bad">GST-plugins-bad</a> - <a href="#gst-plugins-base">GST-plugins-base</a> - <a href="#gstreamer">GStreamer</a> - <a href="#gsutil">gsutil</a> - <a href="#gsw">gsw</a> - <a href="#gtdb-tk">GTDB-Tk</a> - <a href="#gtk+">GTK+</a> - <a href="#gtk2">GTK2</a> - <a href="#gtk3">GTK3</a> - <a href="#gtk4">GTK4</a> - <a href="#gtksourceview">GtkSourceView</a> - <a href="#gtool">GTOOL</a> - <a href="#gts">GTS</a> - <a href="#gubbins">gubbins</a> - <a href="#guenomu">guenomu</a> - <a href="#guile">Guile</a> - <a href="#gulp">GULP</a> - <a href="#gurobi">Gurobi</a> - <a href="#gzip">gzip</a>


<a anchor="g-phocs"/>
### *G-PhoCS*

G-PhoCS is a software package for inferring ancestral population sizes, population divergence times, and migration rates from individual genome sequences. G-PhoCS accepts as input a set of multiple sequence alignments from separate neutrally evolving loci along the genome. Parameter inference is done in a Bayesian manner, using a Markov Chain Monte Carlo (MCMC) to jointly sample model parameters and genealogies at the input loci. 

*homepage*: http://compgen.cshl.edu/GPhoCS/

version  |toolchain
---------|----------------------------------------
``1.2.3``|``iccifort/2019.1.144-GCC-8.2.0-2.31.1``

<a anchor="g2clib"/>
### *g2clib*

Library contains GRIB2 encoder/decoder ('C' version).

*homepage*: https://github.com/NOAA-EMC/NCEPLIBS-g2c

version  |toolchain
---------|------------------------------------------------------------------------------------------------------------------------
``1.6.0``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``foss/2018b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``
``1.6.3``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.7.0``|``GCCcore/11.3.0``

<a anchor="g2lib"/>
### *g2lib*

Library contains GRIB2 encoder/decoder and search/indexing routines.

*homepage*: https://www.nco.ncep.noaa.gov/pmb/codes/GRIB2/

version  |toolchain
---------|---------------------------------------------------------------------
``1.4.0``|``intel/2017a``, ``intel/2017b``, ``intel/2018a``
``3.1.0``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``foss/2018b``, ``intel/2018b``
``3.2.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``

<a anchor="g2log"/>
### *g2log*

g2log, efficient asynchronous logger using C++11

*homepage*: https://sites.google.com/site/kjellhedstrom2//g2log-efficient-background-io-processign-with-c11

version|toolchain
-------|---------------------------------
``1.0``|``GCCcore/8.3.0``, ``foss/2016b``

<a anchor="gaia"/>
### *Gaia*

Gaia is a C++ library with python bindings which implements similarity measures and classiﬁcations  on the results of audio analysis, and generates classiﬁcation models that Essentia can use to compute high-level  description of music.

*homepage*: https://github.com/MTG/gaia

version  |versionsuffix     |toolchain
---------|------------------|-----------------
``2.4.5``|``-Python-2.7.15``|``GCCcore/8.2.0``

<a anchor="gamess-us"/>
### *GAMESS-US*

The General Atomic and Molecular Electronic Structure System (GAMESS)  is a general ab initio quantum chemistry package. 

*homepage*: http://www.msg.chem.iastate.edu/gamess/index.html

version        |versionsuffix|toolchain
---------------|-------------|---------------
``20141205-R1``|             |``intel/2016a``
``20170420-R1``|             |``intel/2016b``
``20170420-R1``|``-sockets`` |``intel/2016b``
``20180214-R1``|             |``foss/2016b``
``20180214-R1``|``-sockets`` |``foss/2016b``

<a anchor="gap"/>
### *gap*

GAP is a system for computational discrete algebra, with particular emphasis on Computational Group Theory.

*homepage*: https://www.gap-system.org

version   |toolchain
----------|---------------
``4.9.3`` |``intel/2018b``
``4.11.0``|``foss/2019a``

<a anchor="gapcloser"/>
### *GapCloser*

GapCloser is designed to close the gaps emerging during the scaffolding process  by SOAPdenovo or other assembler, using the abundant pair relationships of short reads.

*homepage*: https://sourceforge.net/projects/soapdenovo2/files/GapCloser/

version    |toolchain
-----------|-------------------------------
``1.12-r6``|``foss/2018a``, ``intel/2017b``

<a anchor="gapfiller"/>
### *GapFiller*

GapFiller is a seed-and-extend local assembler to fill the gap within paired reads.  It can be used for both DNA and RNA and it has been tested on Illumina data.

*homepage*: https://sourceforge.net/projects/gapfiller

version  |toolchain
---------|---------------
``2.1.1``|``intel/2017a``

<a anchor="gappa"/>
### *gappa*

gappa is a collection of commands for working with phylogenetic data. Its main focus are evolutionary placements of short environmental sequences on a reference phylogenetic tree. Such data is typically produced by tools like EPA-ng, RAxML-EPA or pplacer and usually stored in jplace files. 

*homepage*: https://github.com/lczech/gappa

version  |toolchain
---------|--------------
``0.7.1``|``GCC/10.3.0``

<a anchor="garli"/>
### *GARLI*

GARLI, Genetic Algorithm for Rapid Likelihood Inference is a   program for inferring phylogenetic trees. Using an approach similar to a   classical genetic algorithm, it rapidly searches the space of evolutionary   trees and model parameters to find the solution maximizing the likelihood   score. It implements nucleotide, amino acid and codon-based models of sequence   evolution, and runs on all platforms.

*homepage*: https://code.google.com/archive/p/garli/

version |toolchain
--------|---------------
``2.01``|``gompi/2019a``

<a anchor="garnett"/>
### *garnett*

Garnett is a software package that faciliates automated cell type classification from single-cell expression data.

*homepage*: https://cole-trapnell-lab.github.io/garnett

version   |versionsuffix|toolchain
----------|-------------|--------------
``0.1.20``|``-R-4.0.3`` |``foss/2020b``

<a anchor="gat"/>
### *GAT*

The Genomic Association Tester (GAT) is a tool for computing the significance of overlap between   multiple sets of genomic intervals. GAT estimates significance based on simulation.

*homepage*: https://gat.readthedocs.io/

version  |versionsuffix     |toolchain
---------|------------------|--------------
``1.2.2``|``-Python-2.7.11``|``foss/2016a``

<a anchor="gate"/>
### *GATE*

GATE is an advanced opensource software developed by the international OpenGATE collaboration and  dedicated to the numerical simulations in medical imaging. It currently supports simulations of Emission Tomography  (Positron Emission Tomography - PET and Single Photon Emission Computed Tomography - SPECT), and Computed Tomography

*homepage*: http://www.opengatecollaboration.org/

version    |versionsuffix                  |toolchain
-----------|-------------------------------|-------------------------------
``6.2``    |``-Python-2.7.11``             |``intel/2016a``
``7.1``    |``-Python-2.7.11``             |``intel/2016a``
``7.2``    |``-Python-2.7.11``             |``intel/2016a``
``8.0``    |``-Python-2.7.14``             |``foss/2017b``, ``intel/2017b``
``8.0``    |``-Python-2.7.14-Geant4-10.04``|``intel/2017b``
``8.1.p01``|``-Python-2.7.15``             |``foss/2018b``
``8.2``    |``-Python-2.7.14``             |``foss/2017b``, ``intel/2017b``
``8.2``    |``-Python-2.7.15``             |``foss/2018b``
``9.0``    |``-Python-3.7.4``              |``foss/2019b``
``9.1``    |                               |``foss/2021b``
``9.2``    |                               |``foss/2021b``

<a anchor="gatk"/>
### *GATK*

The Genome Analysis Toolkit or GATK is a software package developed at the Broad Institute  to analyse next-generation resequencing data. The toolkit offers a wide variety of tools,  with a primary focus on variant discovery and genotyping as well as strong emphasis on  data quality assurance. Its robust architecture, powerful processing engine and  high-performance computing features make it capable of taking on projects of any size.

*homepage*: https://www.broadinstitute.org/gatk/

version     |versionsuffix      |toolchain
------------|-------------------|------------------
``1.0.5083``|                   |``system``
``2.5-2``   |``-Java-1.7.0_10`` |``system``
``2.6-5``   |``-Java-1.7.0_10`` |``system``
``2.7-4``   |                   |``system``
``2.7-4``   |``-Java-1.7.0_10`` |``system``
``2.8-1``   |``-Java-1.7.0_10`` |``system``
``3.0-0``   |``-Java-1.7.0_10`` |``system``
``3.3-0``   |``-Java-1.7.0_21`` |``system``
``3.3-0``   |``-Java-1.7.0_80`` |``system``
``3.3-0``   |``-Java-1.8.0_66`` |``system``
``3.5``     |``-Java-1.8.0_66`` |``system``
``3.5``     |``-Java-1.8.0_74`` |``system``
``3.6``     |``-Java-1.8.0_92`` |``system``
``3.7``     |``-Java-1.8.0_112``|``system``
``3.8-0``   |``-Java-1.8.0_144``|``system``
``4.0.1.2`` |``-Java-1.8``      |``system``
``4.0.4.0`` |``-Python-2.7.14`` |``intel/2018a``
``4.0.4.0`` |``-Python-3.6.4``  |``intel/2018a``
``4.0.5.1`` |``-Python-3.6.4``  |``foss/2018a``
``4.0.7.0`` |``-Python-2.7.14`` |``intel/2018a``
``4.0.7.0`` |``-Python-3.6.4``  |``intel/2018a``
``4.0.8.1`` |``-Python-2.7.15`` |``foss/2018b``
``4.0.8.1`` |``-Python-3.6.6``  |``foss/2018b``
``4.0.10.0``|``-Python-3.6.6``  |``foss/2018b``
``4.0.12.0``|``-Python-3.6.6``  |``foss/2018b``
``4.1.0.0`` |``-Python-3.6.6``  |``foss/2018b``
``4.1.2.0`` |``-Java-1.8``      |``GCCcore/8.2.0``
``4.1.3.0`` |``-Java-1.8``      |``GCCcore/8.3.0``
``4.1.4.1`` |``-Java-1.8``      |``GCCcore/8.3.0``
``4.1.4.1`` |``-Java-11``       |``GCCcore/8.3.0``
``4.1.5.0`` |``-Java-1.8``      |``GCCcore/9.3.0``
``4.1.5.0`` |``-Java-11``       |``GCCcore/9.3.0``
``4.1.8.1`` |``-Java-1.8``      |``GCCcore/9.3.0``
``4.2.0.0`` |``-Java-1.8``      |``GCCcore/10.2.0``
``4.2.0.0`` |``-Java-11``       |``GCCcore/10.2.0``
``4.2.3.0`` |``-Java-11``       |``GCCcore/11.2.0``
``4.2.5.0`` |``-Java-11``       |``GCCcore/11.2.0``
``4.2.6.1`` |``-Java-11``       |``GCCcore/11.2.0``
``4.3.0.0`` |``-Java-11``       |``GCCcore/11.3.0``

<a anchor="gaussian"/>
### *Gaussian*

Gaussian provides state-of-the-art capabilities for electronic structure modeling. Gaussian 09 is licensed for a wide variety of computer systems. All versions of Gaussian 09 contain every scientific/modeling feature, and none imposes any artificial limitations on calculations other than your computing resources and patience.  This is the official gaussian AVX2 build. 

*homepage*: https://www.gaussian.com/

version    |versionsuffix|toolchain
-----------|-------------|----------
``09.e.01``|``-AVX``     |``system``
``16.A.03``|``-AVX2``    |``system``

<a anchor="gawk"/>
### *gawk*

The awk utility interprets a special-purpose programming language that makes it possible to handle simple data-reformatting jobs with just a few lines of code.

*homepage*: https://www.gnu.org/software/gawk

version  |toolchain
---------|----------------------------------------------
``5.1.0``|``GCC/10.2.0``
``5.1.1``|``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/11.3.0``

<a anchor="gblocks"/>
### *Gblocks*

Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis

*homepage*: http://molevol.cmima.csic.es/castresana/Gblocks.html

version  |toolchain
---------|----------
``0.91b``|``system``

<a anchor="gbprocess"/>
### *GBprocesS*

GBprocesS allows for the extraction of genomic inserts from NGS data for GBS experiments. Preprocessing is performed in different stages that are part of a linear pipeline where the steps are performed in order. GBprocesS provides a flexible way to adjust the functionality to your needs, as the operations required and the execution order vary depending on the GBS protocol used.

*homepage*: https://gbprocess.readthedocs.io/

version|versionsuffix    |toolchain
-------|-----------------|---------------
``2.3``|``-Python-3.8.2``|``intel/2020a``

<a anchor="gbs2ploidy"/>
### *gbs2ploidy*

Inference of Ploidy from (Genotyping-by-Sequencing) GBS Data

*homepage*: https://cran.r-project.org/web/packages/gbs2ploidy

version|versionsuffix|toolchain
-------|-------------|---------------
``1.0``|``-R-3.4.3`` |``intel/2017b``

<a anchor="gc"/>
### *gc*

The Boehm-Demers-Weiser conservative garbage collector can be used as a garbage collecting replacement for C malloc or C++ new. 

*homepage*: https://hboehm.info/gc/

version   |toolchain
----------|------------------------------------
``7.4.4`` |``GCC/4.9.3-2.25``, ``foss/2016a``
``7.6.0`` |``GCCcore/6.4.0``
``7.6.4`` |``GCCcore/7.3.0``
``7.6.10``|``GCCcore/8.2.0``
``7.6.12``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``8.0.4`` |``GCCcore/10.3.0``
``8.2.0`` |``GCCcore/11.2.0``
``8.2.2`` |``GCCcore/11.3.0``

<a anchor="gc3pie"/>
### *GC3Pie*

GC3Pie is a Python package for running large job campaigns on diverse batch-oriented execution  environments.

*homepage*: https://gc3pie.readthedocs.org

version  |toolchain
---------|----------
``2.4.2``|``system``
``2.5.0``|``system``
``2.5.2``|``system``

<a anchor="gcc"/>
### *GCC*

The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,  as well as libraries for these languages (libstdc++, libgcj,...).

*homepage*: http://gcc.gnu.org/

version   |versionsuffix      |toolchain
----------|-------------------|----------
``4.8.1`` |                   |``system``
``4.8.1`` |``-CLooG``         |``system``
``4.8.2`` |                   |``system``
``4.8.2`` |``-CLooG``         |``system``
``4.8.2`` |``-CLooG-multilib``|``system``
``4.8.2`` |``-multilib``      |``system``
``4.8.3`` |                   |``system``
``4.8.3`` |``-CLooG-multilib``|``system``
``4.8.4`` |                   |``system``
``4.8.4`` |``-CLooG``         |``system``
``4.8.4`` |``-CLooG-multilib``|``system``
``4.8.5`` |                   |``system``
``4.9.0`` |                   |``system``
``4.9.0`` |``-CLooG``         |``system``
``4.9.0`` |``-CLooG-multilib``|``system``
``4.9.1`` |                   |``system``
``4.9.1`` |``-CLooG``         |``system``
``4.9.1`` |``-CLooG-multilib``|``system``
``4.9.2`` |                   |``system``
``4.9.2`` |``-CLooG``         |``system``
``4.9.2`` |``-CLooG-multilib``|``system``
``4.9.2`` |``-binutils-2.25`` |``system``
``4.9.3`` |                   |``system``
``4.9.3`` |``-2.25``          |``system``
``4.9.3`` |``-binutils-2.25`` |``system``
``4.9.4`` |``-2.25``          |``system``
``5.1.0`` |                   |``system``
``5.1.0`` |``-binutils-2.25`` |``system``
``5.2.0`` |                   |``system``
``5.3.0`` |                   |``system``
``5.3.0`` |``-2.26``          |``system``
``5.4.0`` |``-2.26``          |``system``
``5.5.0`` |``-2.26``          |``system``
``6.1.0`` |``-2.27``          |``system``
``6.2.0`` |``-2.27``          |``system``
``6.3.0`` |``-2.27``          |``system``
``6.3.0`` |``-2.28``          |``system``
``6.4.0`` |``-2.28``          |``system``
``7.1.0`` |``-2.28``          |``system``
``7.2.0`` |``-2.29``          |``system``
``7.3.0`` |``-2.30``          |``system``
``7.4.0`` |``-2.31.1``        |``system``
``8.1.0`` |``-2.30``          |``system``
``8.2.0`` |``-2.31.1``        |``system``
``8.3.0`` |                   |``system``
``8.3.0`` |``-2.32``          |``system``
``8.4.0`` |                   |``system``
``9.1.0`` |``-2.32``          |``system``
``9.2.0`` |                   |``system``
``9.2.0`` |``-2.32``          |``system``
``9.3.0`` |                   |``system``
``9.4.0`` |                   |``system``
``9.5.0`` |                   |``system``
``10.1.0``|                   |``system``
``10.2.0``|                   |``system``
``10.3.0``|                   |``system``
``11.1.0``|                   |``system``
``11.2.0``|                   |``system``
``11.3.0``|                   |``system``
``12.1.0``|                   |``system``
``12.2.0``|                   |``system``
``system``|                   |``system``
``system``|``-2.29``          |``system``

<a anchor="gcccore"/>
### *GCCcore*

The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,  as well as libraries for these languages (libstdc++, libgcj,...).

*homepage*: http://gcc.gnu.org/

version   |toolchain
----------|----------
``4.9.2`` |``system``
``4.9.3`` |``system``
``4.9.4`` |``system``
``5.3.0`` |``system``
``5.4.0`` |``system``
``5.5.0`` |``system``
``6.1.0`` |``system``
``6.2.0`` |``system``
``6.3.0`` |``system``
``6.4.0`` |``system``
``6.5.0`` |``system``
``7.1.0`` |``system``
``7.2.0`` |``system``
``7.3.0`` |``system``
``7.4.0`` |``system``
``8.1.0`` |``system``
``8.2.0`` |``system``
``8.3.0`` |``system``
``8.4.0`` |``system``
``9.1.0`` |``system``
``9.2.0`` |``system``
``9.3.0`` |``system``
``9.4.0`` |``system``
``9.5.0`` |``system``
``10.1.0``|``system``
``10.2.0``|``system``
``10.3.0``|``system``
``11.1.0``|``system``
``11.2.0``|``system``
``11.3.0``|``system``
``12.1.0``|``system``
``12.2.0``|``system``
``system``|``system``

<a anchor="gcccuda"/>
### *gcccuda*

GNU Compiler Collection (GCC) based compiler toolchain, along with CUDA toolkit.

*homepage*: (none)

version    |toolchain
-----------|----------
``2016.08``|``system``
``2017b``  |``system``
``2018a``  |``system``
``2018b``  |``system``
``2019a``  |``system``
``2019b``  |``system``
``2020a``  |``system``
``2020b``  |``system``

<a anchor="gcloud"/>
### *gcloud*

Libraries and tools for interacting with Google Cloud products and services.

*homepage*: https://cloud.google.com/sdk

version    |toolchain
-----------|----------
``382.0.0``|``system``

<a anchor="gconf"/>
### *GConf*

GConf is a system for storing application preferences.  It is intended for user preferences; not configuration  of something like Apache, or arbitrary data storage.

*homepage*: https://projects.gnome.org/gconf/

version  |toolchain
---------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``3.2.6``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``foss/2016a``, ``foss/2018b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``

<a anchor="gcta"/>
### *GCTA*

GCTA (Genome-wide Complex Trait Analysis) is a software package, which was  initially developed to estimate the proportion of phenotypic variance explained  by all genome-wide SNPs for a complex trait but has been extensively extended  for many other analyses of data from genome-wide association studies (GWASs). 

*homepage*: https://yanglab.westlake.edu.cn/software/gcta/

version       |toolchain
--------------|------------------------------
``1.94.0beta``|``foss/2021b``, ``gfbf/2022a``

<a anchor="gctf"/>
### *Gctf*

Gctf: real-time CTF determination and correction, Kai Zhang, 2016

*homepage*: https://www.mrc-lmb.cam.ac.uk/kzhang/

version |toolchain
--------|----------
``1.06``|``system``

<a anchor="gd"/>
### *GD*

GD.pm - Interface to Gd Graphics Library

*homepage*: https://github.com/lstein/Perl-GD

version |versionsuffix   |toolchain
--------|----------------|------------------
``2.66``|``-Perl-5.24.0``|``foss/2016b``
``2.68``|``-Perl-5.26.1``|``GCCcore/6.4.0``
``2.69``|``-Perl-5.28.0``|``GCCcore/7.3.0``
``2.71``|                |``GCCcore/9.3.0``
``2.73``|                |``GCCcore/10.3.0``
``2.75``|                |``GCCcore/11.3.0``

<a anchor="gdal"/>
### *GDAL*

GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style  Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model  to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for  data translation and processing.

*homepage*: https://www.gdal.org

version  |versionsuffix                 |toolchain
---------|------------------------------|----------------------------------------------------------------
``2.0.2``|                              |``foss/2016a``, ``intel/2016a``
``2.1.0``|                              |``foss/2016a``, ``foss/2016b``, ``intel/2016b``
``2.1.1``|                              |``foss/2016a``
``2.1.1``|``-Python-2.7.12``            |``intel/2016b``
``2.1.2``|``-Python-2.7.12``            |``intel/2016b``
``2.1.3``|``-Python-2.7.12``            |``foss/2016b``, ``intel/2016b``
``2.1.3``|``-Python-2.7.13``            |``intel/2017a``
``2.1.3``|``-Python-3.6.1``             |``intel/2017a``
``2.2.0``|``-Python-2.7.13-HDF5-1.8.18``|``intel/2017a``
``2.2.0``|``-Python-2.7.13-HDF5-HDF``   |``intel/2017a``
``2.2.0``|``-Python-3.6.1``             |``intel/2017a``
``2.2.2``|``-Python-2.7.14``            |``intel/2017b``
``2.2.2``|``-Python-2.7.14-HDF5-1.8.19``|``intel/2017b``
``2.2.3``|``-Python-2.7.14``            |``foss/2017b``, ``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``2.2.3``|``-Python-2.7.15``            |``foss/2018b``
``2.2.3``|``-Python-3.6.3``             |``foss/2017b``
``2.2.3``|``-Python-3.6.4``             |``foss/2018a``, ``intel/2018a``, ``iomkl/2018a``
``2.2.3``|``-Python-3.6.6``             |``foss/2018b``, ``intel/2018b``
``3.0.0``|``-Python-2.7.15``            |``foss/2019a``, ``intel/2019a``
``3.0.0``|``-Python-3.7.2``             |``foss/2019a``, ``intel/2019a``
``3.0.2``|``-Python-3.7.4``             |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``
``3.0.4``|``-Python-3.8.2``             |``foss/2020a``, ``intel/2020a``
``3.2.1``|                              |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``
``3.3.0``|                              |``foss/2021a``
``3.3.2``|                              |``foss/2021b``
``3.5.0``|                              |``foss/2022a``

<a anchor="gdb"/>
### *GDB*

The GNU Project Debugger

*homepage*: https://www.gnu.org/software/gdb/gdb.html

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``7.8.2`` |                  |``GCC/4.9.2``
``7.9``   |                  |``GCC/4.9.2``
``7.10.1``|``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``7.11``  |``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``7.11.1``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``8.0.1`` |``-Python-2.7.14``|``foss/2017b``
``8.0.1`` |``-Python-3.6.3`` |``foss/2017b``
``8.1``   |``-Python-2.7.14``|``foss/2018a``
``8.1.1`` |``-Python-2.7.14``|``intel/2018a``
``8.3``   |``-Python-3.7.2`` |``GCCcore/8.2.0``
``9.1``   |``-Python-3.7.4`` |``GCCcore/8.3.0``
``10.1``  |                  |``GCCcore/10.2.0``
``10.2``  |                  |``GCCcore/10.3.0``
``10.2``  |``-Python-3.8.2`` |``GCCcore/9.3.0``
``11.1``  |                  |``GCCcore/11.2.0``
``12.1``  |                  |``GCCcore/11.3.0``

<a anchor="gdbgui"/>
### *gdbgui*

Browser-based frontend to gdb (gnu debugger). Add breakpoints, view the stack, visualize data structures, and more in C, C++, Go, Rust, and Fortran. Run gdbgui from the terminal and a new tab will open in your browser.

*homepage*: https://gdbgui.com

version     |versionsuffix    |toolchain
------------|-----------------|-----------------
``0.13.1.2``|``-Python-3.7.2``|``GCCcore/8.2.0``

<a anchor="gdbm"/>
### *gdbm*

GNU dbm (or GDBM, for short) is a library of database functions that use extensible hashing  and work similar to the standard UNIX dbm.  These routines are provided to a programmer needing to create and manipulate a hashed database.

*homepage*: https://www.gnu.org.ua/software/gdbm/gdbm.html

version   |toolchain
----------|------------------
``1.18.1``|``foss/2020a``
``1.21``  |``GCCcore/10.2.0``

<a anchor="gdc-client"/>
### *gdc-client*

The gdc-client provides several convenience functions over  the GDC API which provides general download/upload via HTTPS.

*homepage*: https://gdc.nci.nih.gov/access-data/gdc-data-transfer-tool

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``1.0.1``|``-Python-2.7.12``|``intel/2016b``
``1.3.0``|``-Python-2.7.14``|``foss/2018a``, ``intel/2017b``
``1.3.0``|``-Python-2.7.15``|``foss/2018b``
``1.6.0``|                  |``GCCcore/10.2.0``

<a anchor="gdchart"/>
### *GDCHART*

Easy to use C API, high performance library to create charts  and graphs in PNG, GIF and WBMP format.

*homepage*: http://users.fred.net/brv/chart

version      |toolchain
-------------|-----------------
``0.11.5dev``|``GCCcore/8.2.0``

<a anchor="gdcm"/>
### *GDCM*

Grassroots DICOM: Cross-platform DICOM implementation

*homepage*: https://sourceforge.net/projects/gdcm

version  |toolchain
---------|-------------------------------------
``2.8.8``|``GCCcore/6.4.0``
``2.8.9``|``GCCcore/7.3.0``
``3.0.4``|``GCCcore/8.2.0``
``3.0.5``|``GCCcore/8.3.0``
``3.0.8``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``

<a anchor="gdgraph"/>
### *GDGraph*

GDGraph is a Perl package to generate charts

*homepage*: https://metacpan.org/release/GDGraph

version |versionsuffix   |toolchain
--------|----------------|---------------
``1.54``|``-Perl-5.26.1``|``intel/2018a``
``1.54``|``-Perl-5.28.0``|``foss/2018b``

<a anchor="gdist"/>
### *gdist*

The gdist module is a Cython interface to a C++ library (http://code.google.com/p/geodesic/) for  computing geodesic distance which is the length of shortest line between two vertices on a triangulated mesh in three  dimensions, such that the line lies on the surface.

*homepage*: https://pypi.python.org/pypi/gdist

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.0.3``|``-Python-2.7.11``|``intel/2016a``

<a anchor="gdk-pixbuf"/>
### *Gdk-Pixbuf*

The Gdk Pixbuf is a toolkit for image loading and pixel buffer manipulation.  It is used by GTK+ 2 and GTK+ 3 to load and manipulate images. In the past it  was distributed as part of GTK+ 2 but it was split off into a separate package  in preparation for the change to GTK+ 3. 

*homepage*: https://docs.gtk.org/gdk-pixbuf/

version    |toolchain
-----------|------------------------------------------------------------------------------------
``2.32.3`` |``intel/2016a``
``2.35.1`` |``foss/2016a``, ``intel/2016a``
``2.36.0`` |``foss/2016b``, ``intel/2016b``
``2.36.8`` |``intel/2017a``
``2.36.10``|``intel/2017a``
``2.36.11``|``foss/2017b``, ``foss/2018a``, ``fosscuda/2018b``, ``intel/2017b``, ``intel/2018a``
``2.36.12``|``foss/2018b``, ``fosscuda/2018b``
``2.38.1`` |``GCCcore/8.2.0``
``2.38.2`` |``GCCcore/8.3.0``
``2.40.0`` |``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``2.42.6`` |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``2.42.8`` |``GCCcore/11.3.0``
``2.42.10``|``GCCcore/12.2.0``

<a anchor="gdrcopy"/>
### *GDRCopy*

A low-latency GPU memory copy library based on NVIDIA GPUDirect RDMA technology.

*homepage*: https://github.com/NVIDIA/gdrcopy

version|versionsuffix   |toolchain
-------|----------------|----------------------------------------------------------
``2.1``|``-CUDA-11.0.2``|``GCCcore/9.3.0``
``2.1``|``-CUDA-11.1.1``|``GCCcore/10.2.0``
``2.1``|``-CUDA-11.2.1``|``GCCcore/10.2.0``
``2.2``|                |``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``2.3``|                |``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``

<a anchor="geant4"/>
### *Geant4*

Geant4 is a toolkit for the simulation of the passage of particles through matter.  Its areas of application include high energy, nuclear and accelerator physics,   as well as studies in medical and space science.

*homepage*: https://geant4.cern.ch

version      |toolchain
-------------|----------------------------------------------------------------
``9.5.p02``  |``intel/2016a``
``9.6.p04``  |``intel/2016a``
``10.01.p03``|``intel/2016a``
``10.02.p01``|``intel/2016a``
``10.03.p03``|``foss/2017b``, ``intel/2017b``
``10.04``    |``intel/2017b``
``10.5``     |``foss/2017b``, ``foss/2018b``, ``intel/2017b``, ``intel/2018b``
``10.6``     |``foss/2019b``
``10.6.2``   |``foss/2020a``
``10.7.1``   |``GCC/10.2.0``, ``GCC/11.2.0``
``11.0.0``   |``GCC/11.2.0``
``11.0.1``   |``GCC/11.2.0``
``11.0.2``   |``GCC/11.2.0``

<a anchor="geant4-data"/>
### *Geant4-data*

Datasets for Geant4.

*homepage*: https://geant4.cern.ch/

version     |toolchain
------------|----------
``20201103``|``system``
``20210510``|``system``

<a anchor="gearshifft"/>
### *gearshifft*

Benchmark Suite for Heterogenuous FFT Implementations

*homepage*: https://github.com/mpicbg-scicomp/gearshifft

version  |toolchain
---------|--------------
``0.4.0``|``foss/2019a``

<a anchor="gegl"/>
### *GEGL*

GEGL (Generic Graphics Library) is a data flow based image processing framework,  providing floating point processing and non-destructive image processing capabilities to  GNU Image Manipulation Program (GIMP) and other projects.

*homepage*: https://www.gegl.org

version   |toolchain
----------|------------------
``0.4.30``|``GCCcore/10.3.0``

<a anchor="gem-library"/>
### *GEM-library*

Next-generation sequencing platforms (Illumina/Solexa, ABI/SOLiD, etc.)   call for powerful and very optimized tools to index/analyze huge genomes. The GEM library   strives to be a true "next-generation" tool for handling any kind of sequence data, offering   state-of-the-art algorithms and data structures specifically tailored to this demanding task.   At the moment, efficient indexing and searching algorithms based on the Burrows-Wheeler   transform (BWT) have been implemented. The library core is written in C for maximum speed,   with concise interfaces to higher-level programming languages like OCaml and Python.   Many high-performance standalone programs (mapper, splice mapper, etc.) are provided along   with the library; in general, new algorithms and tools can be easily implemented on the top of it. 

*homepage*: http://algorithms.cnag.cat/wiki/The_GEM_library

version            |versionsuffix                  |toolchain
-------------------|-------------------------------|----------
``20130406-045632``|``_pre-release-3_Linux-x86_64``|``system``

<a anchor="gemma"/>
### *GEMMA*

Genome-wide Efficient Mixed Model Association

*homepage*: https://github.com/genetics-statistics/GEMMA

version   |toolchain
----------|------------------------------
``0.97``  |``foss/2016b``, ``foss/2017a``
``0.98.1``|``foss/2018b``
``0.98.5``|``foss/2021b``

<a anchor="gemmi"/>
### *gemmi*

Gemmi is a library, accompanied by a set of programs, developed primarily for   use in macromolecular crystallography (MX). For working with:  macromolecular models (content of PDB, PDBx/mmCIF and mmJSON files), refinement restraints (CIF files), reflection data (MTZ and mmCIF formats), data on a 3D   grid (electron density maps, masks, MRC/CCP4 format) crystallographic symmetry. Parts of this library can be useful in structural bioinformatics (for symmetry- aware analysis of protein models), and in other molecular-structure sciences    that use CIF files (we have the fastest open-source CIF parser). 

*homepage*: https://gemmi.readthedocs.io/

version  |toolchain
---------|------------------
``0.4.5``|``GCCcore/10.2.0``

<a anchor="gencore_variant_detection"/>
### *gencore_variant_detection*

This is a bundled install of many software packages for doing variant detection analysis. 

*homepage*: https://nyuad-cgsb.github.io/variant_detection/public/index.html

version|toolchain
-------|----------
``1.0``|``system``

<a anchor="genemark-et"/>
### *GeneMark-ET*

Eukaryotic gene prediction suite with automatic training

*homepage*: http://exon.gatech.edu/GeneMark

version |toolchain
--------|------------------
``4.38``|``GCCcore/8.2.0``
``4.57``|``GCCcore/8.3.0``
``4.65``|``GCCcore/10.2.0``
``4.71``|``GCCcore/11.2.0``

<a anchor="gengetopt"/>
### *gengetopt*

Gengetopt is a tool to write command line option parsing code for C programs.

*homepage*: https://www.gnu.org/software/gengetopt/gengetopt.html

version |toolchain
--------|---------------------------------------------------------
``2.23``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/9.3.0``

<a anchor="genome_profiler"/>
### *Genome_Profiler*

Genome Profiler (GeP) is a program to perform whole-genome multilocus  sequence typing (wgMLST) analysis for bacterial isolates

*homepage*: https://sourceforge.net/projects/genomeprofiler/

version|versionsuffix   |toolchain
-------|----------------|--------------
``2.1``|``-Perl-5.24.0``|``foss/2016b``

<a anchor="genomemapper"/>
### *GenomeMapper*

GenomeMapper is a short read mapping tool designed for accurate read alignments.   It quickly aligns millions of reads either with ungapped or gapped alignments.   This version is used to align against a single reference.   If you are unsure which one is the appropriate GenomeMapper, you might want to use this one.

*homepage*: https://1001genomes.org/software/genomemapper_singleref.html

version  |toolchain
---------|--------------
``0.4.4``|``foss/2016a``

<a anchor="genometester4"/>
### *GenomeTester4*

A toolkit for performing set operations - union, intersection and complement - on k-mer lists.

*homepage*: https://github.com/bioinfo-ut/GenomeTester4

version|toolchain
-------|---------------
``4.0``|``intel/2018a``

<a anchor="genomethreader"/>
### *GenomeThreader*

GenomeThreader is a software tool to compute gene structure predictions.

*homepage*: http://genomethreader.org

version  |versionsuffix          |toolchain
---------|-----------------------|----------
``1.7.1``|``-Linux_x86_64-64bit``|``system``
``1.7.3``|``-Linux_x86_64-64bit``|``system``

<a anchor="genometools"/>
### *GenomeTools*

A comprehensive software library for efficient processing of structured genome annotations.

*homepage*: http://genometools.org

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------------------
``1.5.10``|                  |``foss/2018b``
``1.5.10``|``-Python-2.7.15``|``foss/2018b``
``1.6.1`` |                  |``GCC/10.2.0``, ``GCC/8.3.0``, ``GCC/9.3.0``
``1.6.1`` |``-Python-2.7.16``|``GCC/8.3.0``
``1.6.2`` |                  |``GCC/10.3.0``

<a anchor="genomeworks"/>
### *GenomeWorks*

SDK for GPU accelerated genome assembly and analysis

*homepage*: https://github.com/clara-parabricks/GenomeWorks

version      |toolchain
-------------|------------------
``2021.02.2``|``fosscuda/2020b``

<a anchor="genotypeharmonizer"/>
### *GenotypeHarmonizer*

The Genotype Harmonizer is an easy to use command-line tool that allows harmonization of genotype data  stored using different file formats with different and potentially unknown strands.

*homepage*: https://github.com/molgenis/systemsgenetics/wiki/Genotype-Harmonizer

version   |versionsuffix     |toolchain
----------|------------------|----------
``1.4.14``|``-Java-1.7.0_80``|``system``

<a anchor="genozip"/>
### *genozip*

Genozip is a compressor for genomic files - it compresses FASTQ, SAM/BAM/CRAM, VCF, FASTA and others.

*homepage*: https://genozip.com

version   |toolchain
----------|------------------
``13.0.5``|``GCCcore/11.2.0``

<a anchor="gensim"/>
### *gensim*

Gensim is a Python library for topic modelling, document indexing and similarity retrieval with  large corpora.

*homepage*: https://radimrehurek.com/gensim

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``0.13.2``|``-Python-2.7.11``|``foss/2016a``
``3.8.3`` |                  |``foss/2020b``, ``intel/2020b``
``4.2.0`` |                  |``foss/2021a``

<a anchor="geocube"/>
### *geocube*

Tool to convert geopandas vector data into rasterized xarray data.

*homepage*: https://corteva.github.io/geocube/latest/

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.0.14``|``-Python-3.8.2``|``foss/2020a``

<a anchor="geopandas"/>
### *geopandas*

GeoPandas is a project to add support for geographic data to pandas objects. It currently implements GeoSeries and GeoDataFrame types which are subclasses of pandas.Series and pandas.DataFrame respectively. GeoPandas objects can act on shapely geometry objects and perform geometric operations.

*homepage*: https://github.com/geopandas/geopandas

version   |versionsuffix    |toolchain
----------|-----------------|----------------------------------
``0.7.0`` |``-Python-3.7.4``|``foss/2019b``, ``fosscuda/2019b``
``0.8.0`` |``-Python-3.7.2``|``foss/2019a``
``0.8.1`` |``-Python-3.7.4``|``intel/2019b``
``0.8.1`` |``-Python-3.8.2``|``foss/2020a``
``0.10.2``|                 |``foss/2021a``
``0.11.0``|                 |``foss/2021b``

<a anchor="geopy"/>
### *geopy*

geopy is a Python client for several popular geocoding web services. geopy makes it easy for Python developers to locate the coordinates of addresses, cities, countries, and landmarks across the globe using third-party geocoders and other data sources. 

*homepage*: https://geopy.readthedocs.io/en/stable/

version   |versionsuffix    |toolchain
----------|-----------------|------------------
``1.11.0``|``-Python-3.6.1``|``intel/2017a``
``2.1.0`` |                 |``GCCcore/10.2.0``

<a anchor="georges"/>
### *georges*

Georges the lemur opinionated particle accelerator modeling Python package. Also a thin wrapper over MAD-X/PTC, BDSim and G4Beamline.

*homepage*: https://github.com/ULB-Metronu/georges

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``2019.2``|                 |``foss/2021a``
``2019.2``|``-Python-3.7.4``|``foss/2019b``

<a anchor="geos"/>
### *GEOS*

GEOS (Geometry Engine - Open Source) is a C++ port of the Java Topology Suite (JTS)

*homepage*: https://trac.osgeo.org/geos

version   |versionsuffix     |toolchain
----------|------------------|-----------------------------------------------------------------------
``3.5.0`` |``-Python-2.7.11``|``intel/2016a``
``3.5.0`` |``-Python-2.7.12``|``intel/2016b``
``3.6.1`` |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``3.6.1`` |``-Python-2.7.13``|``intel/2017a``
``3.6.1`` |``-Python-3.6.1`` |``intel/2017a``
``3.6.2`` |``-Python-2.7.14``|``foss/2017b``, ``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``3.6.2`` |``-Python-2.7.15``|``foss/2018b``
``3.6.2`` |``-Python-3.6.2`` |``foss/2017b``
``3.6.2`` |``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``, ``intel/2018.01``
``3.6.2`` |``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``, ``iomkl/2018a``
``3.6.2`` |``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``3.7.2`` |``-Python-2.7.15``|``foss/2019a``, ``intel/2019a``
``3.7.2`` |``-Python-3.7.2`` |``foss/2019a``, ``intel/2019a``
``3.8.0`` |                  |``GCC/8.3.0``
``3.8.0`` |``-Python-3.7.4`` |``GCC/8.3.0``, ``iccifort/2019.5.281``
``3.8.1`` |``-Python-3.8.2`` |``GCC/9.3.0``, ``iccifort/2020.1.217``
``3.9.1`` |                  |``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``iccifort/2020.4.304``
``3.10.3``|                  |``GCC/11.3.0``

<a anchor="gerris"/>
### *Gerris*

Gerris is a Free Software program for the solution of the partial                  differential equations describing fluid flow

*homepage*: http://gfs.sourceforge.net/wiki/index.php/Main_Page

version     |toolchain
------------|--------------
``20131206``|``foss/2017b``

<a anchor="getorb"/>
### *GETORB*

GETORB software package contains programs to handle the orbital data records (ODRs)

*homepage*: http://www.deos.tudelft.nl/ers/precorbs/tools/getorb_pack.shtml

version  |toolchain
---------|---------------
``2.3.2``|``intel/2017a``

<a anchor="getorganelle"/>
### *GetOrganelle*

This toolkit assemblies organelle genome from genomic skimming data.

*homepage*: https://github.com/Kinggerm/GetOrganelle

version       |versionsuffix    |toolchain
--------------|-----------------|--------------
``1.7.2``     |``-Python-3.8.2``|``foss/2020a``
``1.7.4-pre2``|                 |``foss/2020b``
``1.7.5.3``   |                 |``foss/2021b``
``1.7.6.1``   |                 |``foss/2021b``

<a anchor="gettext"/>
### *gettext*

GNU 'gettext' is an important step for the GNU Translation Project, as it is an asset on which we may build many other steps. This package offers to programmers, translators, and even users, a well integrated set of tools and documentation

*homepage*: https://www.gnu.org/software/gettext/

version     |versionsuffix     |toolchain
------------|------------------|---------------------------------------------------------------------------------------------------
``0.18.2``  |                  |``system``
``0.19.4``  |                  |``GCC/4.9.2``, ``system``
``0.19.6``  |                  |``GNU/4.9.3-2.25``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``, ``system``
``0.19.7``  |                  |``foss/2016a``, ``intel/2016a``, ``system``
``0.19.8``  |                  |``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``foss/2016.04``, ``foss/2016b``, ``intel/2016b``, ``system``
``0.19.8.1``|                  |``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``system``
``0.19.8.1``|``-libxml2-2.9.7``|``GCCcore/6.4.0``
``0.20.1``  |                  |``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``system``
``0.21``    |                  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``system``
``0.21.1``  |                  |``GCCcore/12.2.0``, ``system``

<a anchor="gexiv2"/>
### *gexiv2*

gexiv2 is a GObject wrapper around the Exiv2 photo metadata library.

*homepage*: https://wiki.gnome.org/Projects/gexiv2

version   |toolchain
----------|------------------
``0.12.2``|``GCCcore/10.3.0``

<a anchor="gfbf"/>
### *gfbf*

GNU Compiler Collection (GCC) based compiler toolchain, including  FlexiBLAS (BLAS and LAPACK support) and (serial) FFTW.

*homepage*: (none)

version  |toolchain
---------|----------
``2022a``|``system``
``2022b``|``system``

<a anchor="gffcompare"/>
### *GffCompare*

GffCompare provides classification and reference annotation mapping and  matching statistics for RNA-Seq assemblies (transfrags) or other generic GFF/GTF files.

*homepage*: https://ccb.jhu.edu/software/stringtie/gffcompare.shtml

version   |toolchain
----------|------------------------------------
``0.10.1``|``foss/2016b``
``0.10.6``|``GCCcore/7.3.0``
``0.11.6``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``0.12.2``|``GCC/10.3.0``
``0.12.6``|``GCC/11.2.0``

<a anchor="gffread"/>
### *gffread*

GFF/GTF parsing utility providing format conversions, region filtering, FASTA sequence extraction and  more.

*homepage*: https://github.com/gpertea/gffread

version   |toolchain
----------|--------------------------------------
``0.9.12``|``foss/2016b``
``0.10.6``|``GCCcore/7.3.0``
``0.11.6``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``0.12.7``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``

<a anchor="gflags"/>
### *gflags*

The gflags package contains a C++ library that implements commandline flags processing.  It includes built-in support for standard types such as string and the ability to define flags in the source file in which they are used. 

*homepage*: https://github.com/gflags/gflags

version  |toolchain
---------|-------------------------------------------------------------------------------------------------------------------
``2.1.2``|``foss/2016a``
``2.2.1``|``GCCcore/6.4.0``, ``intel/2017a``, ``intel/2017b``
``2.2.2``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

<a anchor="gfold"/>
### *GFOLD*

Generalized fold change for ranking differentially expressed genes from RNA-seq data

*homepage*: http://www.tongji.edu.cn/~zhanglab/GFOLD/index.html

version  |toolchain
---------|---------------
``1.1.4``|``intel/2016a``

<a anchor="gh"/>
### *gh*

gh is GitHub on the command line.

*homepage*: https://github.com/cli/cli

version   |toolchain
----------|----------
``1.3.1`` |``system``
``2.20.2``|``system``

<a anchor="ghc"/>
### *GHC*

The Glorious/Glasgow Haskell Compiler

*homepage*: https://haskell.org/ghc/

version   |versionsuffix|toolchain
----------|-------------|----------
``6.12.3``|             |``system``
``8.6.5`` |``-x86_64``  |``system``

<a anchor="ghostscript"/>
### *Ghostscript*

Ghostscript is a versatile processor for PostScript data with the ability to render PostScript to  different targets. It used to be part of the cups printing stack, but is no longer used for that.

*homepage*: https://ghostscript.com

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------------------------
``9.19``  |                  |``intel/2016a``, ``intel/2016b``
``9.20``  |                  |``foss/2016b``, ``intel/2016b``
``9.21``  |                  |``intel/2017a``
``9.22``  |                  |``GCCcore/6.4.0``, ``foss/2017b``, ``intel/2017b``
``9.22``  |``-cairo-1.14.12``|``GCCcore/6.4.0``
``9.23``  |                  |``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``9.23``  |``-cairo-1.14.12``|``GCCcore/6.4.0``
``9.27``  |                  |``GCCcore/8.2.0``
``9.50``  |                  |``GCCcore/8.3.0``
``9.52``  |                  |``GCCcore/9.3.0``
``9.53.3``|                  |``GCCcore/10.2.0``
``9.54.0``|                  |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``9.56.1``|                  |``GCCcore/11.3.0``
``10.0.0``|                  |``GCCcore/12.2.0``

<a anchor="gibbs2"/>
### *Gibbs2*

Gibbs2 is a program for the calculation of thermodynamic properties in periodic solids under arbitrary  conditions of temperature and pressure. Gibbs2 uses the results of periodic solid-state quantum-mechanical  calculations, specifically the energy-volume curve and possibly the harmonic phonon frequencies, to compute the  thermodynamic properties of the solid within the framework of the quasiharmonic approximation.

*homepage*: https://aoterodelaroza.github.io/gibbs2/

version|toolchain
-------|--------------
``1.0``|``GCC/10.3.0``

<a anchor="giflib"/>
### *giflib*

giflib is a library for reading and writing gif images. It is API and ABI compatible with libungif which was in wide use while the LZW compression algorithm was patented.

*homepage*: http://giflib.sourceforge.net/

version  |toolchain
---------|--------------------------------------------------------------------------------------------------------------------
``5.1.4``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``5.2.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

<a anchor="gifsicle"/>
### *gifsicle*

Gifsicle is a command-line tool for creating, editing,  and getting information about GIF images and animations.   Making a GIF animation with gifsicle is easy.

*homepage*: https://github.com/kohler/gifsicle

version |toolchain
--------|------------------
``1.92``|``GCCcore/8.2.0``
``1.93``|``GCCcore/11.3.0``

<a anchor="gimic"/>
### *GIMIC*

The GIMIC program calculates magnetically induced currents in molecules.  You need to provide this program with a density matrix in atomic-orbital (AO) basis  and three (effective) magnetically perturbed AO density matrices in the proper format. Currently ACES2, Turbomole, G09, QChem, FERMION++, and LSDalton can produce these matrices.

*homepage*: http://gimic.readthedocs.io

version       |versionsuffix     |toolchain
--------------|------------------|---------------
``2018.04.20``|``-Python-2.7.14``|``intel/2018a``

<a anchor="gimkl"/>
### *gimkl*

GNU Compiler Collection (GCC) based compiler toolchain with Intel MPI and MKL

*homepage*: (none)

version   |toolchain
----------|----------
``2.11.5``|``system``
``2017a`` |``system``
``2018b`` |``system``

<a anchor="gimp"/>
### *GIMP*

GIMP is a cross-platform image editor available for GNU/Linux, OS X, Windows and more operating systems.

*homepage*: https://www.gimp.org

version    |toolchain
-----------|--------------
``2.10.24``|``GCC/10.3.0``

<a anchor="gimpi"/>
### *gimpi*

GNU Compiler Collection (GCC) based compiler toolchain with Intel MPI.

*homepage*: (none)

version   |toolchain
----------|----------
``2.11.5``|``system``
``2017a`` |``system``
``2017b`` |``system``
``2018a`` |``system``
``2018b`` |``system``

<a anchor="gimpic"/>
### *gimpic*

GNU Compiler Collection (GCC) based compiler toolchain along with CUDA toolkit,  including IntelMPI for MPI support with CUDA features enabled.

*homepage*: (none)

version  |toolchain
---------|----------
``2017b``|``system``

<a anchor="gimps"/>
### *GIMPS*

GIMPS: Great Internet Mersenne Prime Search;  it can be useful for limited stress testing of nodes, in user space

*homepage*: http://www.mersenne.org/

version            |toolchain
-------------------|-------------
``p95v279``        |``GCC/4.8.2``
``p95v279.linux64``|``system``

<a anchor="giolf"/>
### *giolf*

GNU Compiler Collection (GCC) based compiler toolchain, including  IntelMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.

*homepage*: (none)

version  |toolchain
---------|----------
``2017b``|``system``
``2018a``|``system``

<a anchor="giolfc"/>
### *giolfc*

GCC based compiler toolchain __with CUDA support__, and including  IntelMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.

*homepage*: (none)

version  |toolchain
---------|----------
``2017b``|``system``

<a anchor="git"/>
### *git*

Git is a free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency.

*homepage*: http://git-scm.com/

version    |versionsuffix|toolchain
-----------|-------------|------------------------------------
``1.8.5.6``|             |``GCC/4.9.2``
``2.2.2``  |             |``GCC/4.9.2``
``2.4.1``  |             |``GCC/4.9.2``
``2.8.0``  |             |``foss/2016a``
``2.12.2`` |             |``foss/2016b``
``2.13.1`` |             |``foss/2016b``
``2.14.1`` |             |``GCCcore/6.4.0``
``2.16.1`` |             |``foss/2018a``
``2.18.0`` |             |``GCCcore/7.3.0``
``2.19.1`` |             |``GCCcore/7.3.0``
``2.21.0`` |             |``GCCcore/8.2.0``
``2.21.0`` |``-nodocs``  |``GCCcore/8.2.0``
``2.23.0`` |             |``GCCcore/8.3.0``
``2.23.0`` |``-nodocs``  |``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``2.28.0`` |``-nodocs``  |``GCCcore/10.2.0``
``2.32.0`` |``-nodocs``  |``GCCcore/10.3.0``
``2.33.1`` |``-nodocs``  |``GCCcore/11.2.0``
``2.36.0`` |``-nodocs``  |``GCCcore/11.3.0``
``2.38.1`` |``-nodocs``  |``GCCcore/12.2.0``

<a anchor="git-extras"/>
### *git-extras*

Extra useful scripts for git

*homepage*: https://github.com/tj/git-extras

version  |toolchain
---------|--------------
``5.1.0``|``foss/2016a``

<a anchor="git-lfs"/>
### *git-lfs*

Git Large File Storage (LFS) replaces large files such as audio  samples, videos, datasets, and graphics with text pointers inside Git, while  storing the file contents on a remote server like GitHub.com

*homepage*: https://git-lfs.github.com

version   |toolchain
----------|----------
``1.1.1`` |``system``
``2.7.1`` |``system``
``2.11.0``|``system``
``3.2.0`` |``system``

<a anchor="gitpython"/>
### *GitPython*

GitPython is a python library used to interact with Git repositories 

*homepage*: https://github.com/gitpython-developers/GitPython

version   |versionsuffix    |toolchain
----------|-----------------|-------------------------------
``2.1.11``|``-Python-3.6.6``|``foss/2018b``, ``intel/2018b``
``2.1.15``|                 |``system``
``3.0.3`` |``-Python-3.7.2``|``GCCcore/8.2.0``
``3.1.0`` |``-Python-3.7.4``|``GCCcore/8.3.0``
``3.1.9`` |``-Python-3.8.2``|``GCCcore/9.3.0``
``3.1.14``|                 |``GCCcore/10.2.0``
``3.1.18``|                 |``GCCcore/10.3.0``
``3.1.24``|                 |``GCCcore/11.2.0``
``3.1.27``|                 |``GCCcore/11.3.0``

<a anchor="givaro"/>
### *Givaro*

C++ library for arithmetic and algebraic computations

*homepage*: http://givaro.forge.imag.fr/

version  |toolchain
---------|--------------
``4.0.1``|``foss/2016a``

<a anchor="giza"/>
### *Giza*

Giza is an open, lightweight scientific plotting library built on  top of cairo that provides uniform output to multiple devices.

*homepage*: https://danieljprice.github.io/giza/

version  |toolchain
---------|--------------
``1.1.0``|``foss/2018b``

<a anchor="gl2ps"/>
### *GL2PS*

GL2PS: an OpenGL to PostScript printing library

*homepage*: https://www.geuz.org/gl2ps/

version  |versionsuffix   |toolchain
---------|----------------|------------------------------------------------------------------------------------------------------------------------------------
``1.3.9``|                |``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``1.3.9``|``-Mesa-11.2.1``|``foss/2016a``, ``intel/2016a``
``1.4.0``|                |``GCCcore/8.3.0``, ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``foss/2019a``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``
``1.4.2``|                |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``

<a anchor="glade"/>
### *Glade*

Glade is a RAD tool to enable quick & easy development of user interfaces for the GTK+ toolkit  and the GNOME desktop environment.

*homepage*: https://glade.gnome.org/

version  |versionsuffix     |toolchain
---------|------------------|---------------
``3.8.5``|``-Python-2.7.11``|``intel/2016a``
``3.8.5``|``-Python-2.7.14``|``intel/2017b``
``3.8.6``|``-Python-2.7.14``|``intel/2018a``
``3.8.6``|``-Python-2.7.15``|``foss/2018b``

<a anchor="glew"/>
### *glew*

The OpenGL Extension Wrangler Library (GLEW) is a cross-platform open-source C/C++ extension loading library. GLEW provides efficient run-time mechanisms for determining which OpenGL extensions are supported on the target platform.

*homepage*: https://github.com/nigels-com/glew

version  |versionsuffix|toolchain
---------|-------------|---------------------------------------------------------------------------------------------------------------
``2.1.0``|             |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``foss/2018b``
``2.2.0``|``-egl``     |``GCCcore/10.2.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``2.2.0``|``-glx``     |``GCCcore/10.2.0``, ``GCCcore/11.2.0``
``2.2.0``|``-osmesa``  |``GCCcore/10.2.0``, ``GCCcore/11.2.0``

<a anchor="glfw"/>
### *GLFW*

GLFW is an Open Source, multi-platform library for OpenGL, OpenGL ES and Vulkan development on the desktop

*homepage*: https://www.glfw.org

version  |toolchain
---------|--------------------------------------
``3.2.1``|``intel/2018a``
``3.3.2``|``GCCcore/9.3.0``
``3.3.3``|``GCCcore/10.2.0``
``3.3.4``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``3.3.8``|``GCCcore/11.3.0``

<a anchor="glib"/>
### *GLib*

GLib is one of the base libraries of the GTK+ project

*homepage*: https://www.gtk.org/

version   |toolchain
----------|-------------------------------------------------
``2.42.1``|``GCC/4.9.2``
``2.44.0``|``GCC/4.9.2``
``2.47.5``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``2.48.0``|``foss/2016a``, ``intel/2016a``
``2.49.5``|``foss/2016b``, ``intel/2016b``
``2.52.0``|``foss/2017a``, ``intel/2017a``
``2.53.5``|``GCCcore/6.3.0``, ``GCCcore/6.4.0``
``2.54.2``|``GCCcore/6.4.0``
``2.54.3``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``2.60.1``|``GCCcore/8.2.0``
``2.62.0``|``GCCcore/8.3.0``
``2.64.1``|``GCCcore/9.3.0``
``2.66.1``|``GCCcore/10.2.0``
``2.68.2``|``GCCcore/10.3.0``
``2.69.1``|``GCCcore/11.2.0``
``2.72.1``|``GCCcore/11.3.0``
``2.75.0``|``GCCcore/12.2.0``

<a anchor="glib-networking"/>
### *glib-networking*

Network extensions for GLib

*homepage*: https://gitlab.gnome.org/GNOME/glib-networking

version   |toolchain
----------|--------------------------------------
``2.68.1``|``GCCcore/10.3.0``
``2.72.1``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="glibc"/>
### *glibc*

The GNU C Library project provides the core libraries for the GNU system and GNU/Linux systems,  as well as many other systems that use Linux as the kernel.

*homepage*: https://www.gnu.org/software/libc/

version |toolchain
--------|-----------------
``2.17``|``GCCcore/6.4.0``
``2.26``|``GCCcore/6.4.0``
``2.30``|``GCCcore/8.3.0``

<a anchor="glibmm"/>
### *GLibmm*

C++ bindings for Glib

*homepage*: https://www.gtk.org/

version   |toolchain
----------|-------------------------------------------------------
``2.49.7``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``2.66.4``|``GCCcore/10.3.0``

<a anchor="glimmer"/>
### *GLIMMER*

Glimmer is a system for finding genes in microbial DNA, especially  the genomes of bacteria, archaea, and viruses.

*homepage*: http://ccb.jhu.edu/software/glimmer/index.shtml

version  |toolchain
---------|------------------------------
``3.02b``|``foss/2016b``, ``foss/2018b``

<a anchor="glimmerhmm"/>
### *GlimmerHMM*

GlimmerHMM is a new gene finder based on a Generalized Hidden Markov Model.  Although the gene finder conforms to the overall mathematical framework of a GHMM, additionally  it incorporates splice site models adapted from the GeneSplicer program and a decision tree adapted  from GlimmerM. It also utilizes Interpolated Markov Models for the coding and noncoding models.

*homepage*: https://ccb.jhu.edu/software/glimmerhmm

version   |toolchain
----------|------------------------------
``3.0.4`` |``foss/2016b``, ``foss/2018b``
``3.0.4c``|``GCC/10.2.0``, ``GCC/8.3.0``

<a anchor="glm"/>
### *GLM*

OpenGL Mathematics (GLM) is a header only C++ mathematics library for graphics software based on  the OpenGL Shading Language (GLSL) specifications.

*homepage*: https://github.com/g-truc/glm

version    |toolchain
-----------|-------------------------------------------------------------------------------------------------
``0.9.7.6``|``intel/2016a``
``0.9.8.3``|``GCCcore/5.4.0``, ``GCCcore/7.3.0``
``0.9.9.0``|``GCCcore/6.4.0``
``0.9.9.8``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/8.3.0``

<a anchor="glm-aed"/>
### *GLM-AED*

The General Lake Model (GLM) is a water balance and one-dimensional vertical stratification hydrodynamic model, which is dynamically coupled with the AED water quality modelling library. GLM-AED is suitable for simulating conditions in a wide range of natural and engineered lakes, including shallow (well-mixed) and deep (stratified) systems. The model has been successfully applied to systems from the scale of individual ponds and wetlands, to actively operated reservoirs, upto the scale of the Great Lakes.

*homepage*: https://github.com/AquaticEcoDynamics/glm-aed

version    |toolchain
-----------|---------------
``3.3.0a5``|``gompi/2021b``

<a anchor="globalarrays"/>
### *GlobalArrays*

Global Arrays (GA) is a Partitioned Global Address Space (PGAS) programming model

*homepage*: https://hpc.pnl.gov/globalarrays

version  |versionsuffix|toolchain
---------|-------------|--------------------------------
``5.7``  |             |``intel/2018b``
``5.7``  |``-peigs``   |``intel/2019a``
``5.7.2``|             |``intel/2019b``
``5.7.2``|``-peigs``   |``intel/2019b``
``5.8``  |             |``intel/2020a``, ``intel/2021a``
``5.8.1``|             |``intel/2022a``
``5.8.2``|             |``intel/2022a``

<a anchor="globus-cli"/>
### *Globus-CLI*

A Command Line Wrapper over the Globus SDK for Python, which provides an interface to Globus services  from the shell, and is suited to both interactive and simple scripting use cases.

*homepage*: https://docs.globus.org/cli/

version   |toolchain
----------|------------------
``1.11.0``|``GCCcore/8.3.0``
``3.1.1`` |``GCCcore/10.2.0``
``3.2.0`` |``GCCcore/10.3.0``
``3.6.0`` |``GCCcore/11.2.0``

<a anchor="globusconnectpersonal"/>
### *GlobusConnectPersonal*

Globus Connect Personal turns your laptop or other personal computer into a  Globus endpoint with a just a few clicks. With Globus Connect Personal you can  share and transfer files to/from a local machine—campus server, desktop computer or laptop—even if it's behind a firewall and you don't have administrator  privileges. 

*homepage*: https://www.globus.org/globus-connect-personal

version  |toolchain
---------|----------
``2.3.6``|``system``

<a anchor="glog"/>
### *glog*

A C++ implementation of the Google logging module.

*homepage*: https://github.com/google/glog

version  |toolchain
---------|-------------------------------------------------------
``0.3.4``|``foss/2016a``
``0.3.5``|``GCCcore/6.4.0``, ``intel/2017a``, ``intel/2017b``
``0.4.0``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``0.5.0``|``GCCcore/10.2.0``

<a anchor="glpk"/>
### *GLPK*

The GLPK (GNU Linear Programming Kit) package is intended for  solving large-scale linear programming (LP),  mixed integer programming (MIP), and other related problems.  It is a set of routines written in ANSI C   and organized in the form of a callable library.

*homepage*: https://www.gnu.org/software/glpk/

version |toolchain
--------|-----------------------------------------------------------------------------------------------------------------
``4.58``|``foss/2016a``, ``intel/2016a``
``4.60``|``GCCcore/5.4.0``, ``intel/2016b``
``4.61``|``intel/2017a``
``4.65``|``GCCcore/10.2.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``5.0`` |``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="glproto"/>
### *glproto*

X protocol and ancillary headers

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version   |toolchain
----------|-------------------------------------------------
``1.4.17``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``

<a anchor="glucose"/>
### *Glucose*

Glucose is based on a new scoring scheme (well, not so new now, it was introduced in 2009) for the clause learning mechanism of so called Modern SAT solvers (it is based on our IJCAI'09 paper). It is designed to be parallel, since v4.0.

*homepage*: https://www.labri.fr/perso/lsimon/glucose/

version|toolchain
-------|-------------
``4.1``|``GCC/9.3.0``

<a anchor="gmap-gsnap"/>
### *GMAP-GSNAP*

GMAP: A Genomic Mapping and Alignment Program for mRNA and EST Sequences  GSNAP: Genomic Short-read Nucleotide Alignment Program

*homepage*: http://research-pub.gene.com/gmap/

version       |toolchain
--------------|---------------
``2016-05-01``|``foss/2016a``
``2016-11-07``|``foss/2016b``
``2018-05-11``|``intel/2018a``
``2018-07-04``|``intel/2018a``
``2019-03-15``|``foss/2018b``
``2019-09-12``|``GCC/8.3.0``
``2020-12-17``|``GCC/9.3.0``
``2021-21-17``|``GCC/11.2.0``

<a anchor="gmp"/>
### *GMP*

GMP is a free library for arbitrary precision arithmetic, operating on signed  integers, rational numbers, and floating point numbers. 

*homepage*: https://gmplib.org/

version   |toolchain
----------|-----------------------------------------------------------------------------------------------------------------------------------------------------
``4.3.2`` |``system``
``5.1.3`` |``GCC/4.8.2``
``6.0.0`` |``GCC/4.9.2``
``6.0.0a``|``GCC/4.8.4``, ``GCC/4.9.2``, ``GNU/4.9.3-2.25``
``6.1.0`` |``GCC/4.9.3-2.25``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``
``6.1.1`` |``GCC/5.4.0-2.26``, ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``intel/2016b``
``6.1.2`` |``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``foss/2016b``
``6.2.0`` |``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``6.2.1`` |``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``

<a anchor="gmpich"/>
### *gmpich*

gcc and GFortran based compiler toolchain,  including MPICH for MPI support.

*homepage*: (none)

version    |toolchain
-----------|----------
``2016a``  |``system``
``2017.08``|``system``

<a anchor="gmpolf"/>
### *gmpolf*

gcc and GFortran based compiler toolchain,  MPICH for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.

*homepage*: (none)

version    |toolchain
-----------|----------
``2016a``  |``system``
``2017.10``|``system``

<a anchor="gmpy2"/>
### *gmpy2*

GMP/MPIR, MPFR, and MPC interface to Python 2.6+ and 3.x

*homepage*: https://github.com/aleaxit/gmpy

version    |versionsuffix     |toolchain
-----------|------------------|----------------------------------------------------------------------
``2.0.8``  |``-Python-2.7.13``|``intel/2017a``
``2.0.8``  |``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``2.0.8``  |``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``2.1.0b1``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``2.1.0b1``|``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``2.1.0b4``|                  |``GCC/8.3.0``
``2.1.0b5``|                  |``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/9.3.0``, ``iccifort/2020.4.304``
``2.1.2``  |                  |``GCC/11.2.0``, ``GCC/11.3.0``

<a anchor="gmsh"/>
### *gmsh*

Gmsh is a 3D finite element grid generator with a build-in CAD engine and post-processor.

*homepage*: https://gmsh.info/

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``3.0.6``|``-Python-2.7.14``|``foss/2017b``
``3.0.6``|``-Python-3.6.6`` |``foss/2018b``
``4.2.2``|``-Python-3.6.6`` |``foss/2018b``
``4.5.6``|``-Python-2.7.16``|``intel/2019b``
``4.5.6``|``-Python-3.7.4`` |``foss/2019b``
``4.7.1``|``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``4.8.4``|``-Python-3.6.4`` |``foss/2018a``
``4.9.0``|                  |``foss/2021a``

<a anchor="gmt"/>
### *GMT*

GMT is an open source collection of about 80 command-line tools for manipulating  geographic and Cartesian data sets (including filtering, trend fitting, gridding, projecting,  etc.) and producing PostScript illustrations ranging from simple x-y plots via contour maps  to artificially illuminated surfaces and 3D perspective views; the GMT supplements add another  40 more specialized and discipline-specific tools. 

*homepage*: https://www.generic-mapping-tools.org/

version   |toolchain
----------|-------------------------------
``4.5.17``|``foss/2018a``
``5.4.1`` |``intel/2017a``
``5.4.3`` |``foss/2018a``, ``intel/2017b``
``5.4.5`` |``foss/2019a``
``6.2.0`` |``foss/2019b``, ``foss/2020b``

<a anchor="gmvapich2"/>
### *gmvapich2*

GNU Compiler Collection (GCC) based compiler toolchain,  including MVAPICH2 for MPI support.

*homepage*: (none)

version   |toolchain
----------|----------
``1.7.20``|``system``
``2016a`` |``system``

<a anchor="gmvolf"/>
### *gmvolf*

GNU Compiler Collection (GCC) based compiler toolchain, including  MVAPICH2 for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.

*homepage*: (none)

version   |toolchain
----------|----------
``1.7.20``|``system``
``2016a`` |``system``

<a anchor="gnu"/>
### *GNU*

Compiler-only toolchain with GCC and binutils.

*homepage*: http://www.gnu.org/software/

version       |toolchain
--------------|----------
``4.9.2-2.25``|``system``
``4.9.3-2.25``|``system``
``5.1.0-2.25``|``system``

<a anchor="gnuplot"/>
### *gnuplot*

Portable interactive, function plotting utility

*homepage*: http://gnuplot.sourceforge.net

version  |toolchain
---------|----------------------------------------------------------------
``5.0.3``|``foss/2016a``, ``intel/2016a``
``5.0.5``|``foss/2016b``, ``intel/2016b``
``5.0.6``|``intel/2017a``
``5.2.2``|``foss/2017b``, ``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``5.2.5``|``foss/2018b``
``5.2.6``|``GCCcore/8.2.0``, ``foss/2018b``, ``fosscuda/2018b``
``5.2.8``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``5.4.1``|``GCCcore/10.2.0``
``5.4.2``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``5.4.4``|``GCCcore/11.3.0``

<a anchor="gnutls"/>
### *GnuTLS*

GnuTLS is a secure communications library implementing the SSL, TLS  and DTLS protocols and technologies around them. It provides a simple  C language application programming interface (API) to access the secure  communications protocols as well as APIs to parse and write X.509, PKCS #12,  OpenPGP and other required structures. It is aimed to be portable  and efficient with focus on security and interoperability.

*homepage*: https://www.gnutls.org

version   |toolchain
----------|------------------
``3.3.21``|``intel/2016a``
``3.4.7`` |``GNU/4.9.3-2.25``
``3.4.11``|``foss/2016a``
``3.7.2`` |``GCCcore/10.3.0``
``3.7.3`` |``GCCcore/11.2.0``
``3.7.8`` |``GCCcore/11.3.0``

<a anchor="go"/>
### *Go*

Go is an open source programming language that makes it easy to build  simple, reliable, and efficient software.

*homepage*: http://www.golang.org

version   |toolchain
----------|-----------------
``1.2.1`` |``GCC/4.8.2``
``1.4.2`` |``GCC/4.8.4``
``1.5``   |``GCC/4.8.4``
``1.8.1`` |``system``
``1.11.5``|``system``
``1.12``  |``system``
``1.12.1``|``GCCcore/7.3.0``
``1.13.1``|``system``
``1.14``  |``system``
``1.14.1``|``system``
``1.16.3``|``system``
``1.16.5``|``system``
``1.16.6``|``system``
``1.17.3``|``system``
``1.17.6``|``system``
``1.18.1``|``system``
``1.18.3``|``system``

<a anchor="goalign"/>
### *goalign*

Goalign is a set of command line tools to manipulate multiple alignments.

*homepage*: https://github.com/evolbioinfo/goalign

version  |toolchain
---------|----------
``0.3.2``|``system``

<a anchor="goatools"/>
### *GOATOOLS*

A Python library for Gene Ontology analyses

*homepage*: https://github.com/tanghaibao/goatools

version  |toolchain
---------|--------------
``1.1.6``|``foss/2020b``

<a anchor="gobff"/>
### *gobff*

GCC and GFortran based compiler toolchain with OpenMPI, BLIS, libFLAME, ScaLAPACK and FFTW.

*homepage*: (none)

version    |versionsuffix|toolchain
-----------|-------------|----------
``2020.06``|``-amd``     |``system``
``2020.11``|             |``system``
``2020b``  |             |``system``
``2021a``  |             |``system``

<a anchor="gobject-introspection"/>
### *GObject-Introspection*

GObject introspection is a middleware layer between C libraries  (using GObject) and language bindings. The C library can be scanned at  compile time and generate a metadata file, in addition to the actual  native C library. Then at runtime, language bindings can read this  metadata and automatically provide bindings to call into the C library.

*homepage*: https://gi.readthedocs.io/en/latest/

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------------
``1.47.1``|                  |``foss/2016a``, ``intel/2016a``
``1.48.0``|                  |``foss/2016a``, ``intel/2016a``
``1.49.1``|                  |``foss/2016b``, ``intel/2016b``
``1.52.0``|                  |``intel/2017a``
``1.53.5``|``-Python-2.7.13``|``intel/2017a``
``1.53.5``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``1.53.5``|``-Python-3.6.1`` |``intel/2017a``
``1.54.1``|``-Python-2.7.14``|``foss/2018a``, ``intel/2018a``
``1.54.1``|``-Python-2.7.15``|``foss/2018b``, ``fosscuda/2018b``
``1.54.1``|``-Python-3.6.6`` |``fosscuda/2018b``
``1.58.3``|``-Python-2.7.16``|``GCCcore/8.3.0``
``1.60.1``|``-Python-3.7.2`` |``GCCcore/8.2.0``
``1.63.1``|``-Python-3.7.4`` |``GCCcore/8.3.0``
``1.64.0``|``-Python-3.8.2`` |``GCCcore/9.3.0``
``1.66.1``|                  |``GCCcore/10.2.0``
``1.68.0``|                  |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.72.0``|                  |``GCCcore/11.3.0``
``1.74.0``|                  |``GCCcore/12.2.0``

<a anchor="goblf"/>
### *goblf*

GNU Compiler Collection (GCC) based compiler toolchain, including  OpenMPI for MPI support, BLIS (BLAS support), LAPACK, FFTW and ScaLAPACK.

*homepage*: (none)

version  |toolchain
---------|----------
``2018b``|``system``
``2020b``|``system``

<a anchor="godon"/>
### *Godon*

Godon is codon models software written in Go.

*homepage*: https://bitbucket.org/Davydov/godon/

version     |versionsuffix|toolchain
------------|-------------|----------
``20210913``|``-x86_64``  |``system``

<a anchor="gofasta"/>
### *gofasta*

Some functions for dealing with alignments,  developed to handle SARS-CoV-2 data as part of the COG-UK project.

*homepage*: https://github.com/cov-ert/gofasta

version  |toolchain
---------|----------
``0.0.5``|``system``

<a anchor="golf"/>
### *golf*

GNU Compiler Collection (GCC) based compiler toolchain, including  OpenBLAS (BLAS and LAPACK support) and FFTW.

*homepage*: (none)

version  |toolchain
---------|----------
``2018a``|``system``
``2020a``|``system``

<a anchor="gomkl"/>
### *gomkl*

GNU Compiler Collection (GCC) based compiler toolchain with OpenMPI and MKL

*homepage*: (none)

version  |toolchain
---------|----------
``2018b``|``system``
``2019a``|``system``
``2020a``|``system``
``2020b``|``system``
``2021a``|``system``

<a anchor="gompi"/>
### *gompi*

GNU Compiler Collection (GCC) based compiler toolchain,  including OpenMPI for MPI support.

*homepage*: (none)

version    |versionsuffix|toolchain
-----------|-------------|----------
``2016.04``|             |``system``
``2016.06``|             |``system``
``2016.07``|             |``system``
``2016.09``|             |``system``
``2016a``  |             |``system``
``2016b``  |             |``system``
``2017a``  |             |``system``
``2017b``  |             |``system``
``2018.08``|             |``system``
``2018a``  |             |``system``
``2018b``  |             |``system``
``2019a``  |             |``system``
``2019b``  |             |``system``
``2020a``  |             |``system``
``2020b``  |             |``system``
``2021a``  |             |``system``
``2021b``  |             |``system``
``2022.05``|             |``system``
``2022.10``|             |``system``
``2022a``  |             |``system``
``2022b``  |             |``system``
``system`` |``-2.29``    |``system``

<a anchor="gompic"/>
### *gompic*

GNU Compiler Collection (GCC) based compiler toolchain along with CUDA toolkit,  including OpenMPI for MPI support with CUDA features enabled.

*homepage*: (none)

version  |toolchain
---------|----------
``2017b``|``system``
``2018a``|``system``
``2018b``|``system``
``2019a``|``system``
``2019b``|``system``
``2020a``|``system``
``2020b``|``system``

<a anchor="google-java-format"/>
### *google-java-format*

Reformats Java source code to comply with Google Java Style.

*homepage*: https://github.com/google/google-java-format

version|versionsuffix|toolchain
-------|-------------|----------
``1.7``|``-Java-1.8``|``system``

<a anchor="googletest"/>
### *googletest*

Google's framework for writing C++ tests on a variety of platforms

*homepage*: https://github.com/google/googletest

version   |toolchain
----------|---------------------------------------------------------------------
``1.8.0`` |``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``foss/2016b``, ``intel/2016b``
``1.8.1`` |``GCCcore/8.2.0``
``1.10.0``|``GCCcore/10.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.11.0``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="gotree"/>
### *gotree*

GoTree is a set of command line tools to manipulate phylogenetic trees.

*homepage*: https://github.com/evolbioinfo/gotree

version  |toolchain
---------|----------
``0.4.0``|``system``

<a anchor="gp2c"/>
### *GP2C*

The gp2c compiler is a package for translating GP routines into the C programming language, so that  they can  be compiled and used with the PARI system or the GP calculator. 

*homepage*: http://pari.math.u-bordeaux.fr/pub/pari/manuals/gp2c/gp2c.html

version     |toolchain
------------|--------------
``0.0.9pl5``|``foss/2016a``

<a anchor="gpaw"/>
### *GPAW*

GPAW is a density-functional theory (DFT) Python code based on the projector-augmented wave (PAW)  method and the atomic simulation environment (ASE). It uses real-space uniform grids and multigrid methods or  atom-centered basis-functions.

*homepage*: https://wiki.fysik.dtu.dk/gpaw/

version    |versionsuffix               |toolchain
-----------|----------------------------|----------------------------------------------------------------
``1.4.0``  |``-Python-3.6.6``           |``foss/2018b``, ``intel/2018b``
``19.8.1`` |``-ASE-3.18.0-Python-3.6.6``|``foss/2018b``, ``intel/2018b``
``19.8.1`` |``-Python-3.7.2``           |``foss/2019a``, ``intel/2019a``
``20.1.0`` |``-Python-3.7.4``           |``foss/2019b``, ``intel/2019b``
``20.10.0``|                            |``foss/2020b``, ``intel/2020b``
``20.10.0``|``-ASE-3.20.1-Python-3.7.4``|``foss/2019b``, ``intel/2019b``
``21.1.0`` |``-ASE-3.21.1``             |``foss/2020b``, ``intel/2020b``
``21.6.0`` |                            |``foss/2021a``
``21.6.0`` |``-ASE-3.22.0``             |``foss/2020b``, ``intel/2020b``
``22.8.0`` |                            |``foss/2021b``, ``foss/2022a``, ``intel/2021b``, ``intel/2022a``

<a anchor="gpaw-setups"/>
### *GPAW-setups*

PAW setup for the GPAW Density Functional Theory package.   Users can install setups manually using 'gpaw install-data' or use setups from this package.   The versions of GPAW and GPAW-setups can be intermixed.

*homepage*: https://wiki.fysik.dtu.dk/gpaw/

version      |toolchain
-------------|----------
``0.8.7929`` |``system``
``0.9.9672`` |``system``
``0.9.11271``|``system``
``0.9.20000``|``system``

<a anchor="gperf"/>
### *gperf*

GNU gperf is a perfect hash function generator. For a given list of strings,  it produces a hash function and hash table, in form of C or C++ code, for  looking up a value depending on the input string. The hash function is  perfect, which means that the hash table has no collisions, and the hash  table lookup needs a single string comparison only. 

*homepage*: https://www.gnu.org/software/gperf/

version  |toolchain
---------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``3.0.4``|``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``
``3.1``  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

<a anchor="gperftools"/>
### *gperftools*

gperftools is a collection of a high-performance multi-threaded malloc() implementation, plus some pretty nifty performance analysis tools. Includes TCMalloc, heap-checker, heap-profiler and cpu-profiler. 

*homepage*: https://github.com/gperftools/gperftools

version   |toolchain
----------|----------------------------------------------------------
``2.5``   |``foss/2016a``, ``intel/2016b``
``2.6.3`` |``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``2.7.90``|``GCCcore/8.3.0``
``2.8``   |``GCCcore/9.3.0``
``2.9.1`` |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``2.10``  |``GCCcore/11.3.0``

<a anchor="gpustat"/>
### *gpustat*

A simple command-line utility for querying and monitoring GPU status

*homepage*: https://github.com/wookayin/gpustat

version    |versionsuffix     |toolchain
-----------|------------------|-------------------------------------
``0.5.0``  |``-Python-2.7.15``|``fosscuda/2018b``
``0.6.0``  |                  |``GCCcore/10.3.0``, ``gcccuda/2020b``
``0.6.0``  |``-Python-3.7.4`` |``fosscuda/2019b``
``1.0.0b1``|                  |``GCCcore/11.2.0``

<a anchor="gpy"/>
### *GPy*

GPy is a Gaussian Process (GP) framework written in Python

*homepage*: https://sheffieldml.github.io/GPy

version   |toolchain
----------|--------------
``1.10.0``|``foss/2021b``

<a anchor="gpyopt"/>
### *GPyOpt*

GPyOpt is a Python open-source library for Bayesian Optimization

*homepage*: https://sheffieldml.github.io/GPyOpt

version  |toolchain
---------|---------------
``1.2.6``|``intel/2020b``

<a anchor="gpytorch"/>
### *GPyTorch*

GPyTorch is a Gaussian process library implemented using PyTorch.

*homepage*: https://gpytorch.ai

version  |toolchain
---------|--------------
``1.3.0``|``foss/2020b``

<a anchor="grace"/>
### *Grace*

Grace is a WYSIWYG tool to make two-dimensional plots of numerical data.

*homepage*: https://plasma-gate.weizmann.ac.il/Grace/

version   |versionsuffix|toolchain
----------|-------------|-----------------------------------------------------------------------------------------------------------------
``5.1.25``|             |``foss/2016a``, ``foss/2021a``, ``foss/2021b``, ``intel/2016a``
``5.1.25``|``-5build1`` |``foss/2017b``, ``foss/2018a``, ``foss/2019a``, ``foss/2019b``, ``intel/2017b``, ``intel/2019a``, ``intel/2019b``

<a anchor="gradle"/>
### *Gradle*

Complete Gradle install. From mobile apps to microservices, from small startups to big enterprises, Gradle helps teams build, automate and deliver better software, faster. 

*homepage*: https://gradle.org

version  |toolchain
---------|----------
``4.5.1``|``system``
``6.1.1``|``system``
``6.9.1``|``system``

<a anchor="gradunwarp"/>
### *gradunwarp*

Gradient Unwarping. This is the Human Connectome Project fork of the no longer maintained original.

*homepage*: https://github.com/Washington-University/gradunwarp

version  |versionsuffix         |toolchain
---------|----------------------|--------------
``1.1.0``|``-HCP-Python-2.7.15``|``foss/2019a``
``1.2.0``|``-HCP-Python-2.7.15``|``foss/2019a``
``1.2.0``|``-HCP-Python-3.7.2`` |``foss/2019a``

<a anchor="graph-tool"/>
### *graph-tool*

Graph-tool is an efficient Python module for manipulation and  statistical analysis of graphs (a.k.a. networks). Contrary to  most other python modules with similar functionality, the core  data structures and algorithms are implemented in C++, making  extensive use of template metaprogramming, based heavily on  the Boost Graph Library. This confers it a level of  performance that is comparable (both in memory usage and  computation time) to that of a pure C/C++ library.

*homepage*: https://graph-tool.skewed.de/

version |versionsuffix    |toolchain
--------|-----------------|--------------
``2.26``|``-Python-3.6.3``|``foss/2017b``
``2.27``|``-Python-3.6.6``|``foss/2018b``

<a anchor="graphene"/>
### *Graphene*

Graphene is a a thin layer of types for graphic libraries

*homepage*: https://ebassi.github.io/graphene/

version   |toolchain
----------|------------------
``1.6.0`` |``intel/2017a``
``1.10.8``|``GCCcore/11.3.0``

<a anchor="graphicsmagick"/>
### *GraphicsMagick*

GraphicsMagick is the swiss army knife of image processing.

*homepage*: https://www.graphicsmagick.org/

version   |toolchain
----------|----------------------------------
``1.3.23``|``foss/2016a``, ``intel/2016a``
``1.3.25``|``intel/2016b``, ``intel/2017a``
``1.3.28``|``foss/2018a``
``1.3.31``|``foss/2018b``
``1.3.34``|``foss/2019a``, ``foss/2019b``
``1.3.36``|``GCCcore/11.2.0``, ``foss/2020b``

<a anchor="graphite2"/>
### *graphite2*

Graphite is a "smart font" system developed specifically to  handle the complexities of lesser-known languages of the world.

*homepage*: https://scripts.sil.org/cms/scripts/page.php?site_id=projects&item_id=graphite_home

version   |toolchain
----------|-------------------------------------------------------------------------------------------------
``1.3.14``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/8.3.0``

<a anchor="graphlan"/>
### *GraPhlAn*

GraPhlAn is a software tool for producing high-quality circular representations  of taxonomic and phylogenetic trees. It focuses on concise, integrative, informative,  and publication-ready representations of phylogenetically- and taxonomically-driven investigation. 

*homepage*: https://segatalab.github.io/tools/graphlan

version  |versionsuffix     |toolchain
---------|------------------|--------------
``1.1.3``|``-Python-2.7.16``|``foss/2019b``

<a anchor="graphmap"/>
### *GraphMap*

A highly sensitive and accurate mapper for long, error-prone reads

*homepage*: https://github.com/isovic/graphmap

version  |toolchain
---------|--------------
``0.5.2``|``foss/2019b``

<a anchor="graphmap2"/>
### *GraphMap2*

A highly sensitive and accurate mapper for long, error-prone reads

*homepage*: https://github.com/lbcb-sci/graphmap2

version  |toolchain
---------|--------------
``0.6.4``|``foss/2019b``

<a anchor="graphviz"/>
### *Graphviz*

Graphviz is open source graph visualization software. Graph visualization  is a way of representing structural information as diagrams of  abstract graphs and networks. It has important applications in networking,  bioinformatics,  software engineering, database and web design, machine learning,  and in visual interfaces for other technical domains.

*homepage*: https://www.graphviz.org/

version   |versionsuffix            |toolchain
----------|-------------------------|-------------------------------
``2.38.0``|                         |``foss/2016b``, ``intel/2016b``
``2.40.1``|                         |``foss/2018b``, ``intel/2018a``
``2.40.1``|``-Python-2.7.15``       |``foss/2018b``
``2.42.2``|                         |``foss/2019b``
``2.42.2``|``-Java-11``             |``GCCcore/8.3.0``
``2.42.2``|``-Python-3.7.4``        |``foss/2019b``
``2.44.1``|``-Java-11``             |``GCCcore/9.3.0``
``2.44.1``|``-Java-11-Python-3.8.2``|``GCCcore/9.3.0``
``2.47.0``|``-Java-11``             |``GCCcore/10.2.0``
``2.47.2``|                         |``GCCcore/10.3.0``
``2.50.0``|                         |``GCCcore/11.2.0``
``5.0.0`` |                         |``GCCcore/11.3.0``

<a anchor="graphviz"/>
### *graphviz*

Simple Python interface for Graphviz

*homepage*: https://pypi.python.org/pypi/graphviz

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``0.5.1``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``0.5.1``|``-Python-3.5.2`` |``intel/2016b``
``0.8.2``|``-Python-3.6.4`` |``intel/2018a``

<a anchor="grasp"/>
### *GRASP*

The General Relativistic Atomic Structure Package (GRASP) is a set of  Fortran 90 programs for performing fully-relativistic electron structure  calculations of atoms.

*homepage*: https://compas.github.io/grasp/

version |toolchain
--------|--------------
``2018``|``foss/2019b``

<a anchor="grass"/>
### *GRASS*

The Geographic Resources Analysis Support System - used  for geospatial data management and analysis, image processing,  graphics and maps production, spatial modeling, and visualization

*homepage*: https://grass.osgeo.org

version  |versionsuffix     |toolchain
---------|------------------|------------------
``7.6.0``|``-Python-2.7.15``|``foss/2018b``
``7.8.3``|``-Python-3.7.4`` |``fosscuda/2019b``

<a anchor="greenlet"/>
### *Greenlet*

The greenlet package is a spin-off of Stackless, a version of CPython that supports micro-threads called "tasklets". Tasklets run pseudo-concurrently (typically in a single or a few OS-level threads) and are synchronized with data exchanges on "channels". A "greenlet", on the other hand, is a still more primitive notion of micro-thread with no implicit scheduling; coroutines, in other words. This is useful when you want to control exactly when your code runs. 

*homepage*: https://github.com/python-greenlet/greenlet

version   |versionsuffix     |toolchain
----------|------------------|---------------
``0.4.9`` |``-Python-2.7.11``|``intel/2016a``
``0.4.9`` |``-Python-3.5.1`` |``intel/2016a``
``0.4.11``|``-Python-2.7.12``|``intel/2016b``
``0.4.12``|``-Python-2.7.14``|``intel/2017b``

<a anchor="grep"/>
### *Grep*

The grep command searches one or more input files for lines containing a match to a specified pattern.  By default, grep prints the matching lines.

*homepage*: http://www.gnu.org/software/grep/

version |toolchain
--------|-------------
``2.21``|``GCC/4.9.2``

<a anchor="gretl"/>
### *gretl*

A cross-platform software package for econometric analysis

*homepage*: http://gretl.sourceforge.net

version  |toolchain
---------|--------------
``2020a``|``foss/2019a``

<a anchor="grib_api"/>
### *grib_api*

The ECMWF GRIB API is an application program interface accessible from C, FORTRAN and Python  programs developed for encoding and decoding WMO FM-92 GRIB edition 1 and edition 2 messages. A useful set of  command line tools is also provided to give quick access to GRIB messages.

*homepage*: https://software.ecmwf.int/wiki/display/GRIB/Home

version   |toolchain
----------|------------------------------------------------
``1.16.0``|``intel/2016a``
``1.21.0``|``foss/2017a``
``1.24.0``|``foss/2017b``, ``intel/2017a``, ``intel/2017b``

<a anchor="grit"/>
### *GRIT*

GRIT - A tool for the integrative analysis of RNA-seq type assays

*homepage*: https://github.com/nboley/grit

version  |versionsuffix     |toolchain
---------|------------------|---------------
``2.0.5``|``-Python-2.7.12``|``intel/2016b``

<a anchor="grnboost"/>
### *GRNBoost*

XGBoost is an optimized distributed gradient boosting library designed to be highly efficient,  flexible and portable.

*homepage*: https://github.com/dmlc/xgboost

version     |versionsuffix      |toolchain
------------|-------------------|---------------
``20171009``|``-Java-1.8.0_152``|``intel/2017b``

<a anchor="groff"/>
### *groff*

Groff (GNU troff) is a typesetting system that reads plain text mixed with formatting commands  and produces formatted output.

*homepage*: https://www.gnu.org/software/groff

version   |toolchain
----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.22.4``|``FCC/4.5.0``, ``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.1.0``, ``GCCcore/12.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

<a anchor="groimp"/>
### *GroIMP*

GroIMP (Growth Grammar-related Interactive Modelling Platform) is a 3D-modelling platform.

*homepage*: http://www.grogra.de/software/groimp

version|toolchain
-------|----------
``1.5``|``system``

<a anchor="gromacs"/>
### *GROMACS*

GROMACS is a versatile package to perform molecular dynamics,  i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles.

*homepage*: http://www.gromacs.org

version   |versionsuffix                |toolchain
----------|-----------------------------|------------------------------------------------------------------------------------------------------------
``5.1.2`` |``-hybrid``                  |``foss/2016a``, ``intel/2016a``
``5.1.2`` |``-hybrid-dp``               |``intel/2016a``
``5.1.2`` |``-mt``                      |``foss/2016a``
``5.1.4`` |``-hybrid``                  |``foss/2016b``
``5.1.4`` |``-mt``                      |``foss/2016b``
``2016``  |``-hybrid``                  |``foss/2016b``
``2016``  |``-mt``                      |``foss/2016b``
``2016.1``|``-PLUMED``                  |``foss/2017a``
``2016.2``|                             |``foss/2017a``
``2016.3``|                             |``foss/2017a``, ``intel/2017a``
``2016.3``|``-GPU-enabled``             |``foss/2016b``
``2016.4``|                             |``foss/2017b``, ``fosscuda/2017b``, ``giolf/2017b``, ``intel/2017a``, ``intel/2017b``, ``intelcuda/2017b``
``2016.5``|                             |``intel/2018a``
``2018``  |                             |``foss/2018a``
``2018.1``|``-PLUMED``                  |``foss/2018b``
``2018.2``|                             |``foss/2017b``, ``foss/2018b``, ``fosscuda/2017b``, ``fosscuda/2018b``, ``intel/2017b``, ``intelcuda/2017b``
``2018.3``|                             |``foss/2018b``, ``fosscuda/2018b``
``2018.4``|``-PLUMED-2.5.0``            |``foss/2018b``, ``fosscuda/2018b``
``2019``  |                             |``foss/2018b``, ``fosscuda/2018b``
``2019.2``|                             |``fosscuda/2019a``
``2019.3``|                             |``foss/2019a``, ``foss/2019b``, ``fosscuda/2019a``, ``fosscuda/2019b``
``2019.4``|                             |``foss/2019b``
``2019.6``|                             |``fosscuda/2019b``
``2020``  |                             |``foss/2019b``, ``fosscuda/2019b``
``2020.1``|``-Python-3.8.2``            |``foss/2020a``
``2020.3``|                             |``fosscuda/2019b``
``2020.4``|``-Python-3.8.2``            |``foss/2020a``
``2020.5``|``-Python-3.8.2``            |``fosscuda/2020a``
``2021``  |                             |``foss/2020b``, ``fosscuda/2020b``
``2021.2``|                             |``fosscuda/2020b``
``2021.3``|                             |``foss/2021a``
``2021.3``|``-CUDA-11.3.1``             |``foss/2021a``
``2021.3``|``-CUDA-11.3.1-PLUMED-2.7.2``|``foss/2021a``
``2021.3``|``-PLUMED-2.7.2``            |``foss/2021a``
``2021.5``|                             |``foss/2021b``
``2021.5``|``-CUDA-11.4.1``             |``foss/2021b``
``2021.5``|``-CUDA-11.4.1-PLUMED-2.8.0``|``foss/2021b``
``2021.5``|``-PLUMED-2.8.0``            |``foss/2021b``

<a anchor="gromacswrapper"/>
### *GromacsWrapper*

GromacsWrapper is a python package that wraps system calls to Gromacs tools into thin classes.  This allows for fairly seamless integration of the gromacs tools into python scripts. 

*homepage*: https://gromacswrapper.readthedocs.org

version  |versionsuffix    |toolchain
---------|-----------------|------------------
``0.8.0``|``-Python-3.7.2``|``fosscuda/2019a``

<a anchor="groopm"/>
### *GroopM*

GroopM is a metagenomic binning toolset. It leverages spatio-temporal   dynamics (differential coverage) to accurately (and almost automatically)   extract population genomes from multi-sample metagenomic datasets.  

*homepage*: https://ecogenomics.github.io/GroopM/

version  |versionsuffix     |toolchain
---------|------------------|--------------
``0.3.4``|``-Python-2.7.12``|``foss/2016b``

<a anchor="groovy"/>
### *Groovy*

Groovy is a powerful, optionally typed and dynamic language, with static-typing and static compilation capabilities, for the Java platform aimed at improving developer productivity thanks to a concise, familiar and easy to learn syntax.

*homepage*: https://groovy-lang.org

version  |versionsuffix|toolchain
---------|-------------|----------
``2.5.9``|``-Java-11`` |``system``
``4.0.3``|``-Java-11`` |``system``

<a anchor="grpc"/>
### *gRPC*

gRPC is a modern, open source, high-performance remote procedure call (RPC) framework that can run anywhere. gRPC enables client and server applications to communicate transparently, and simplifies the building of connected systems.

*homepage*: https://grpc.io/

version   |toolchain
----------|------------------
``1.44.0``|``GCCcore/11.2.0``

<a anchor="gsea"/>
### *GSEA*

Gene Set Enrichment Analysis (GSEA) is a computational method that  determines whether an a priori defined set of genes shows statistically significant, concordant differences between two biological states (e.g. phenotypes). 

*homepage*: https://www.gsea-msigdb.org/gsea/index.jsp

version  |toolchain
---------|----------
``4.0.3``|``system``

<a anchor="gsettings-desktop-schemas"/>
### *gsettings-desktop-schemas*

gsettings-desktop-schemas contains a collection of GSettings schemas for  settings shared by various components of a desktop.

*homepage*: https://gitlab.gnome.org/GNOME/gsettings-desktop-schemas

version   |toolchain
----------|-----------------
``3.34.0``|``GCCcore/8.2.0``

<a anchor="gsl"/>
### *GSL*

The GNU Scientific Library (GSL) is a numerical library for C and C++ programmers.  The library provides a wide range of mathematical routines such as random number generators, special functions  and least-squares fitting.

*homepage*: https://www.gnu.org/software/gsl/

version  |toolchain
---------|----------------------------------------------------------------------------------------------------------------------------
``1.16`` |``foss/2016a``, ``intel/2016a``
``2.1``  |``GCC/5.4.0-2.26``, ``foss/2016a``, ``foss/2016b``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``, ``intel/2016a``, ``intel/2016b``
``2.2.1``|``intel/2016a``, ``intel/2016b``
``2.3``  |``foss/2016b``, ``foss/2017a``, ``intel/2016b``, ``intel/2017a``
``2.4``  |``GCCcore/6.4.0``
``2.5``  |``GCC/7.3.0-2.30``, ``GCC/8.2.0-2.31.1``, ``iccifort/2018.3.222-GCC-7.3.0-2.30``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``2.6``  |``GCC/10.2.0``, ``GCC/8.3.0``, ``GCC/9.3.0``, ``iccifort/2019.5.281``, ``iccifort/2020.1.217``, ``iccifort/2020.4.304``
``2.7``  |``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/11.3.0``, ``intel-compilers/2021.2.0``, ``intel-compilers/2021.4.0``

<a anchor="gsoap"/>
### *gSOAP*

The gSOAP toolkit is a C and C++ software development toolkit for  SOAP and REST XML Web services and generic C/C++ XML data bindings.  The toolkit analyzes WSDLs and XML schemas (separately or as a combined set) and maps the XML schema types  and the SOAP/REST XML messaging protocols to easy-to-use and efficient C and C++ code.  It also supports exposing (legacy) C and C++ applications as XML Web services  by auto-generating XML serialization code and WSDL specifications.  Or you can simply use it to automatically convert XML to/from C and C++ data.  The toolkit supports options to generate pure ANSI C or C++ with or without STL.

*homepage*: http://www.cs.fsu.edu/~engelen/soap.html

version    |toolchain
-----------|-----------------
``2.8.48`` |``GCCcore/6.3.0``
``2.8.100``|``GCCcore/8.3.0``

<a anchor="gsport"/>
### *gsport*

GSPORT command-line tool for accessing GenomeScan Customer Portal

*homepage*: https://github.com/genomescan/gsport

version  |versionsuffix    |toolchain
---------|-----------------|-----------------
``1.4.2``|``-Python-3.7.4``|``GCCcore/8.3.0``

<a anchor="gst-plugins-bad"/>
### *GST-plugins-bad*

GStreamer is a library for constructing graphs of media-handling  components. The applications it supports range from simple  Ogg/Vorbis playback, audio/video streaming to complex audio  (mixing) and video (non-linear editing) processing.

*homepage*: https://gstreamer.freedesktop.org/

version   |toolchain
----------|--------------
``1.20.2``|``GCC/11.3.0``

<a anchor="gst-plugins-base"/>
### *GST-plugins-base*

GStreamer is a library for constructing graphs of media-handling  components. The applications it supports range from simple  Ogg/Vorbis playback, audio/video streaming to complex audio  (mixing) and video (non-linear editing) processing.

*homepage*: https://gstreamer.freedesktop.org/

version    |toolchain
-----------|------------------------------------------------------------------------------------------------------------------
``0.10.36``|``foss/2016a``, ``foss/2017b``, ``foss/2018b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``
``1.6.4``  |``foss/2016a``
``1.8.3``  |``foss/2016a``
``1.16.0`` |``GCC/8.2.0-2.31.1``
``1.16.2`` |``GCC/8.3.0``
``1.18.4`` |``GCC/10.2.0``, ``GCC/10.3.0``
``1.18.5`` |``GCC/11.2.0``
``1.20.2`` |``GCC/11.3.0``

<a anchor="gstreamer"/>
### *GStreamer*

GStreamer is a library for constructing graphs of media-handling  components. The applications it supports range from simple  Ogg/Vorbis playback, audio/video streaming to complex audio  (mixing) and video (non-linear editing) processing.

*homepage*: https://gstreamer.freedesktop.org/

version    |toolchain
-----------|------------------------------------------------------------------------------------------------------------------
``0.10.36``|``foss/2016a``, ``foss/2017b``, ``foss/2018b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``
``1.6.4``  |``foss/2016a``
``1.8.3``  |``foss/2016a``
``1.15.1`` |``fosscuda/2018b``
``1.16.0`` |``GCC/8.2.0-2.31.1``
``1.16.2`` |``GCC/8.3.0``
``1.18.4`` |``GCC/10.2.0``, ``GCC/10.3.0``
``1.18.5`` |``GCC/11.2.0``
``1.20.2`` |``GCC/11.3.0``

<a anchor="gsutil"/>
### *gsutil*

gsutil is a Python application that lets you access Cloud Storage from the command line.

*homepage*: https://cloud.google.com/storage/docs/gsutil

version |toolchain
--------|------------------
``5.10``|``GCCcore/11.2.0``

<a anchor="gsw"/>
### *gsw*

This Python implementation of the Thermodynamic Equation of  Seawater 2010 (TEOS-10) is based primarily on numpy ufunc wrappers of the GSW-C  implementation. This library replaces the original python-gsw pure-python  implementation.. The primary reasons for this change are that by building on  the C implementation we reduce code duplication and we gain an immediate update  to the 75-term equation. Additional benefits include a major increase in speed, a reduction in memory  usage, and the inclusion of more functions. The penalty is that a C  (or MSVC C++ for Windows) compiler is required to build the package from source.

*homepage*: https://github.com/TEOS-10/GSW-python

version   |toolchain
----------|--------------
``3.6.16``|``foss/2022a``

<a anchor="gtdb-tk"/>
### *GTDB-Tk*

A toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes.

*homepage*: https://github.com/Ecogenomics/GTDBTk

version  |versionsuffix     |toolchain
---------|------------------|-----------------------------------------------
``0.2.2``|``-Python-2.7.15``|``intel/2018b``
``0.3.2``|``-Python-2.7.15``|``foss/2019a``, ``intel/2019a``
``1.0.2``|``-Python-3.7.4`` |``intel/2019b``
``1.3.0``|``-Python-3.8.2`` |``intel/2020a``
``1.5.0``|                  |``intel/2020b``
``1.7.0``|                  |``foss/2020b``, ``foss/2021a``, ``intel/2020b``
``2.0.0``|                  |``foss/2021a``, ``intel/2021b``

<a anchor="gtk+"/>
### *GTK+*

The GTK+ 2 package contains libraries used for creating graphical user interfaces for applications. 

*homepage*: https://developer.gnome.org/gtk+/stable/

version    |toolchain
-----------|--------------------------------------------------------------------------------
``2.24.28``|``intel/2016a``
``2.24.30``|``foss/2016a``, ``intel/2016a``
``2.24.31``|``foss/2016b``, ``intel/2016b``, ``intel/2017a``
``2.24.32``|``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``intel/2017b``, ``intel/2018a``
``2.24.33``|``GCCcore/10.3.0``
``3.22.30``|``fosscuda/2018b``
``3.24.8`` |``GCCcore/8.2.0``
``3.24.13``|``GCCcore/8.3.0``
``3.24.17``|``GCCcore/9.3.0``
``3.24.23``|``GCCcore/10.2.0``

<a anchor="gtk2"/>
### *GTK2*

The GTK+ 2 package contains libraries used for creating graphical user interfaces for applications. 

*homepage*: https://www.gtk.org

version    |toolchain
-----------|--------------------------------------
``2.24.33``|``GCCcore/10.3.0``, ``GCCcore/11.3.0``

<a anchor="gtk3"/>
### *GTK3*

GTK+ is the primary library used to construct user interfaces in GNOME. It  provides all the user interface controls, or widgets, used in a common  graphical application. Its object-oriented API allows you to construct  user interfaces without dealing with the low-level details of drawing and  device interaction. 

*homepage*: https://developer.gnome.org/gtk3/stable/

version    |toolchain
-----------|------------------
``3.24.29``|``GCCcore/10.3.0``
``3.24.31``|``GCCcore/11.2.0``
``3.24.33``|``GCCcore/11.3.0``
``3.24.35``|``GCCcore/12.2.0``

<a anchor="gtk4"/>
### *GTK4*

GTK+ is the primary library used to construct user interfaces in GNOME. It  provides all the user interface controls, or widgets, used in a common  graphical application. Its object-oriented API allows you to construct  user interfaces without dealing with the low-level details of drawing and  device interaction. 

*homepage*: https://docs.gtk.org/gtk4/

version  |toolchain
---------|--------------
``4.7.0``|``GCC/11.3.0``

<a anchor="gtksourceview"/>
### *GtkSourceView*

GtkSourceView is a GNOME library that extends GtkTextView, the standard GTK+  widget for multiline text editing. GtkSourceView adds support for syntax  highlighting, undo/redo, file loading and saving, search and replace, a  completion system, printing, displaying line numbers, and other features  typical of a source code editor. 

*homepage*: https://wiki.gnome.org/Projects/GtkSourceView

version    |toolchain
-----------|-------------------------------------
``3.24.11``|``GCCcore/10.2.0``, ``GCCcore/8.2.0``
``4.4.0``  |``GCCcore/8.2.0``

<a anchor="gtool"/>
### *GTOOL*

GTOOL is a program for transforming sets of genotype data for use with  the programs SNPTEST and IMPUTE.  

*homepage*: http://www.well.ox.ac.uk/~cfreeman/software/gwas/gtool.html

version  |toolchain
---------|----------
``0.7.5``|``system``

<a anchor="gts"/>
### *GTS*

GTS stands for the GNU Triangulated Surface Library.  It is an Open Source Free Software Library intended to provide a set of useful  functions to deal with 3D surfaces meshed with interconnected triangles.

*homepage*: http://gts.sourceforge.net/

version     |toolchain
------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``0.7.6``   |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2018b``, ``foss/2019b``, ``foss/2020a``, ``intel/2016a``, ``intel/2016b``, ``intel/2018a``
``20121130``|``foss/2017b``

<a anchor="gubbins"/>
### *gubbins*

Gubbins (Genealogies Unbiased By recomBinations In Nucleotide Sequences) is an algorithm that iteratively identifies loci containing elevated densities of base substitutions while concurrently constructing a phylogeny based on the putative point mutations outside of these regions. Simulations demonstrate the algorithm generates highly accurate reconstructions under realistic models of short-term bacterial evolution, and can be run in only a few hours on alignments of hundreds of bacterial genome sequences. 

*homepage*: https://sanger-pathogens.github.io/gubbins

version  |toolchain
---------|----------
``2.4.0``|``system``

<a anchor="guenomu"/>
### *guenomu*

guenomu is a software written in C that estimates the species tree for a given set of gene families.

*homepage*: https://bitbucket.org/leomrtns/guenomu

version       |versionsuffix|toolchain
--------------|-------------|---------------
``2019.07.05``|``-mpi``     |``iimpi/2019a``

<a anchor="guile"/>
### *Guile*

Guile is a programming language, designed to help programmers create flexible  applications that can be extended by users or other programmers with plug-ins,  modules, or scripts. 

*homepage*: https://www.gnu.org/software/guile/

version   |toolchain
----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.8.8`` |``GCCcore/5.4.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``GNU/4.9.3-2.25``, ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``intel/2016a``, ``intel/2016b``
``2.0.11``|``GCC/4.9.3-2.25``, ``foss/2016a``
``2.2.2`` |``GCCcore/6.4.0``
``2.2.4`` |``GCCcore/7.3.0``, ``GCCcore/9.3.0``
``2.2.7`` |``GCCcore/10.3.0``
``3.0.7`` |``GCCcore/11.2.0``
``3.0.8`` |``GCCcore/11.3.0``

<a anchor="gulp"/>
### *GULP*

GULP is a program for performing a variety of types of simulation on materials  using boundary conditions of 0-D (molecules and clusters), 1-D (polymers), 2-D (surfaces, slabs  and grain boundaries), or 3-D (periodic solids)Band Unfolding code for Plane-wave based calculations

*homepage*: https://gulp.curtin.edu.au/gulp/

version|toolchain
-------|---------------
``5.1``|``intel/2019a``
``6.1``|``foss/2021b``

<a anchor="gurobi"/>
### *Gurobi*

The Gurobi Optimizer is a state-of-the-art solver for mathematical programming. The solvers in the Gurobi Optimizer were designed from the ground up to exploit modern architectures and multi-core processors, using the most advanced implementations of the latest algorithms.

*homepage*: https://www.gurobi.com

version  |versionsuffix    |toolchain
---------|-----------------|--------------------------------------------------
``6.5.1``|                 |``system``
``6.5.2``|                 |``system``
``7.0.1``|                 |``system``
``7.5.2``|                 |``system``
``7.5.2``|``-Python-3.6.4``|``intel/2018a``
``8.1.1``|                 |``system``
``9.0.0``|                 |``system``
``9.0.0``|``-Python-3.6.6``|``foss/2018b``, ``intel/2018b``
``9.0.0``|``-Python-3.7.4``|``GCCcore/8.3.0``
``9.0.1``|                 |``system``
``9.0.1``|``-Python-3.8.2``|``GCCcore/9.3.0``
``9.0.3``|                 |``GCCcore/10.2.0``, ``system``
``9.1.0``|                 |``system``
``9.1.1``|                 |``GCCcore/10.2.0``
``9.1.2``|                 |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``system``
``9.5.0``|                 |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``9.5.2``|                 |``GCCcore/11.3.0``

<a anchor="gzip"/>
### *gzip*

gzip (GNU zip) is a popular data compression program as a replacement for compress

*homepage*: http://www.gnu.org/software/gzip/

version |toolchain
--------|-------------------------------------------------------------------------------------------------------------------
``1.8`` |``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``
``1.9`` |``GCCcore/7.3.0``
``1.10``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.12``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``

<a anchor="h"/>
### *H*


<a href="#h4toh5">h4toh5</a> - <a href="#h5hut">H5hut</a> - <a href="#h5py">h5py</a> - <a href="#hadoop">Hadoop</a> - <a href="#hal">HAL</a> - <a href="#hanythingondemand">hanythingondemand</a> - <a href="#hapgen2">HAPGEN2</a> - <a href="#harfbuzz">HarfBuzz</a> - <a href="#harminv">Harminv</a> - <a href="#harmony">harmony</a> - <a href="#hbase">HBase</a> - <a href="#hdbscan">HDBSCAN</a> - <a href="#hddm">HDDM</a> - <a href="#hdf">HDF</a> - <a href="#hdf-eos">HDF-EOS</a> - <a href="#hdf-eos5">HDF-EOS5</a> - <a href="#hdf5">HDF5</a> - <a href="#hdf5storage">hdf5storage</a> - <a href="#hdfview">HDFView</a> - <a href="#healpix">HEALPix</a> - <a href="#heaptrack">heaptrack</a> - <a href="#hector">hector</a> - <a href="#heffte">HeFFTe</a> - <a href="#hello">Hello</a> - <a href="#help2man">help2man</a> - <a href="#hh-suite">HH-suite</a> - <a href="#hic-pro">HiC-Pro</a> - <a href="#hicexplorer">HiCExplorer</a> - <a href="#hierfstat">hierfstat</a> - <a href="#hifiasm">hifiasm</a> - <a href="#highfive">HighFive</a> - <a href="#highway">Highway</a> - <a href="#hip">HIP</a> - <a href="#hipify-clang">hipify-clang</a> - <a href="#hips">HIPS</a> - <a href="#hipsycl">hipSYCL</a> - <a href="#hiredis">hiredis</a> - <a href="#hisat2">HISAT2</a> - <a href="#histolab">histolab</a> - <a href="#hivtrace">hivtrace</a> - <a href="#hl7apy">hl7apy</a> - <a href="#hlaminer">HLAminer</a> - <a href="#hmmer">HMMER</a> - <a href="#hmmer2">HMMER2</a> - <a href="#hmmlearn">hmmlearn</a> - <a href="#home">HOME</a> - <a href="#horovod">Horovod</a> - <a href="#horton">horton</a> - <a href="#how_are_we_stranded_here">how_are_we_stranded_here</a> - <a href="#hpcg">HPCG</a> - <a href="#hpcx">HPCX</a> - <a href="#hpdbscan">HPDBSCAN</a> - <a href="#hpl">HPL</a> - <a href="#htop">htop</a> - <a href="#htseq">HTSeq</a> - <a href="#htslib">HTSlib</a> - <a href="#hub">hub</a> - <a href="#humann">humann</a> - <a href="#hunspell">hunspell</a> - <a href="#hwloc">hwloc</a> - <a href="#hydra">Hydra</a> - <a href="#hyperopt">Hyperopt</a> - <a href="#hyperqueue">HyperQueue</a> - <a href="#hyperspy">hyperspy</a> - <a href="#hyphy">HyPhy</a> - <a href="#hypo">HyPo</a> - <a href="#hypothesis">hypothesis</a> - <a href="#hypre">Hypre</a>


<a anchor="h4toh5"/>
### *h4toh5*

The h4toh5 software consists of the h4toh5 and h5toh4 command-line utilities,  as well as a conversion library for converting between individual HDF4 and HDF5 objects.

*homepage*: http://www.hdfgroup.org/h4toh5/

version  |versionsuffix           |toolchain
---------|------------------------|------------------------------------------------
``2.2.2``|``-linux-x86_64-static``|``system``
``2.2.3``|                        |``foss/2018b``, ``gompi/2019b``, ``gompi/2020b``

<a anchor="h5hut"/>
### *H5hut*

HDF5 Utility Toolkit: High-Performance I/O Library for Particle-based Simulations

*homepage*: https://amas.psi.ch/H5hut/

version    |toolchain
-----------|---------------
``1.99.13``|``intel/2016b``

<a anchor="h5py"/>
### *h5py*

HDF5 for Python (h5py) is a general-purpose Python interface to the Hierarchical Data Format library,  version 5. HDF5 is a versatile, mature scientific software library designed for the fast, flexible storage of enormous  amounts of data.

*homepage*: https://www.h5py.org/

version   |versionsuffix                        |toolchain
----------|-------------------------------------|------------------------------------------------------------------------------------
``2.5.0`` |``-Python-2.7.11-HDF5-1.8.16``       |``foss/2016a``, ``intel/2016a``
``2.5.0`` |``-Python-2.7.11-HDF5-1.8.16-serial``|``foss/2016a``, ``intel/2016a``
``2.5.0`` |``-Python-3.5.1-HDF5-1.8.16``        |``foss/2016a``
``2.6.0`` |``-Python-2.7.11``                   |``intel/2016a``
``2.6.0`` |``-Python-2.7.12-HDF5-1.10.0-patch1``|``foss/2016b``, ``intel/2016b``
``2.6.0`` |``-Python-2.7.12-HDF5-1.8.17``       |``foss/2016b``, ``intel/2016b``
``2.6.0`` |``-Python-2.7.12-HDF5-1.8.18``       |``foss/2016b``, ``intel/2016b``
``2.6.0`` |``-Python-3.5.2-HDF5-1.10.0-patch1`` |``foss/2016b``, ``intel/2016b``
``2.6.0`` |``-Python-3.5.2-HDF5-1.8.17``        |``intel/2016b``
``2.6.0`` |``-Python-3.5.2-HDF5-1.8.18``        |``foss/2016b``, ``intel/2016b``
``2.7.0`` |``-Python-2.7.12``                   |``intel/2016b``
``2.7.0`` |``-Python-2.7.13``                   |``intel/2017a``
``2.7.0`` |``-Python-2.7.13-HDF5-1.10.1``       |``foss/2017a``
``2.7.0`` |``-Python-2.7.13-HDF5-1.8.19``       |``foss/2017a``
``2.7.0`` |``-Python-3.6.1``                    |``intel/2017a``
``2.7.0`` |``-Python-3.6.1-HDF5-1.10.0-patch1`` |``intel/2017a``
``2.7.0`` |``-Python-3.6.1-HDF5-1.10.1``        |``foss/2017a``
``2.7.0`` |``-Python-3.6.1-HDF5-1.8.19``        |``foss/2017a``
``2.7.1`` |``-Python-2.7.13``                   |``intel/2017a``
``2.7.1`` |``-Python-2.7.14``                   |``foss/2017b``, ``foss/2018a``, ``fosscuda/2017b``, ``intel/2017b``, ``intel/2018a``
``2.7.1`` |``-Python-2.7.14-serial``            |``intel/2018a``
``2.7.1`` |``-Python-3.6.1``                    |``intel/2017a``
``2.7.1`` |``-Python-3.6.3``                    |``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``
``2.7.1`` |``-Python-3.6.4``                    |``foss/2018a``, ``intel/2018a``
``2.8.0`` |``-Python-2.7.15``                   |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``2.8.0`` |``-Python-2.7.15-serial``            |``intel/2018b``
``2.8.0`` |``-Python-3.6.6``                    |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``2.9.0`` |                                     |``foss/2019a``, ``fosscuda/2019a``, ``intel/2019a``, ``intelcuda/2019a``
``2.10.0``|``-Python-2.7.18``                   |``intel/2020a``
``2.10.0``|``-Python-3.7.4``                    |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``, ``intelcuda/2019b``
``2.10.0``|``-Python-3.8.2``                    |``foss/2020a``, ``fosscuda/2020a``, ``intel/2020a``, ``intelcuda/2020a``
``2.10.0``|``-serial-Python-3.7.4``             |``foss/2019b``
``3.1.0`` |                                     |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``, ``intelcuda/2020b``
``3.2.1`` |                                     |``foss/2021a``, ``gomkl/2021a``
``3.6.0`` |                                     |``foss/2021b``, ``intel/2021b``
``3.7.0`` |                                     |``foss/2022a``

<a anchor="hadoop"/>
### *Hadoop*

Hadoop MapReduce by Cloudera

*homepage*: https://archive.cloudera.com/cdh5/cdh/5/

version            |versionsuffix             |toolchain
-------------------|--------------------------|------------------
``2.4.0``          |``-seagate-722af1-native``|``system``
``2.5.0-cdh5.3.1`` |``-native``               |``system``
``2.6.0-cdh5.4.5`` |``-native``               |``system``
``2.6.0-cdh5.7.0`` |``-native``               |``system``
``2.6.0-cdh5.8.0`` |``-native``               |``system``
``2.6.0-cdh5.12.0``|``-native``               |``system``
``2.9.2``          |``-native``               |``GCCcore/7.3.0``
``2.10.0``         |``-native``               |``GCCcore/8.3.0``
``2.10.0``         |``-native-Java-1.8``      |``GCCcore/10.2.0``

<a anchor="hal"/>
### *HAL*

HAL is a structure to efficiently store and index multiple genome alignments and ancestral reconstructions. HAL is a graph-based representation which provides several advantages over matrix/block-based formats such as MAF, such as improved scalability and the ability to perform queries with respect to an arbitrary reference or subtree.  This package includes the HAL API and several analysis and conversion tools which are described below. HAL files are presently stored in either HDF5 or mmap format, but we note that the tools and most of the API are format-independent, so other databases could be implemented in the future. 

*homepage*: https://github.com/ComparativeGenomicsToolkit/hal

version|toolchain
-------|--------------
``2.1``|``foss/2020b``

<a anchor="hanythingondemand"/>
### *hanythingondemand*

HanythingOnDemand (HOD) is a system for provisioning virtual Hadoop clusters over a large physical cluster. It uses the Torque resource manager to do node allocation.

*homepage*: https://github.com/hpcugent/hanythingondemand

version  |versionsuffix     |toolchain
---------|------------------|---------------
``3.0.0``|``-cli``          |``system``
``3.0.1``|``-cli``          |``system``
``3.0.2``|``-cli``          |``system``
``3.0.3``|``-cli``          |``system``
``3.0.4``|``-cli``          |``system``
``3.1.0``|``-Python-2.7.11``|``intel/2016a``
``3.1.0``|``-cli``          |``system``
``3.1.1``|``-Python-2.7.11``|``intel/2016a``
``3.1.1``|``-cli``          |``system``
``3.1.2``|``-Python-2.7.11``|``intel/2016a``
``3.1.2``|``-cli``          |``system``
``3.1.3``|``-Python-2.7.12``|``intel/2016b``
``3.1.3``|``-cli``          |``system``
``3.1.4``|``-Python-2.7.12``|``intel/2016b``
``3.1.4``|``-cli``          |``system``
``3.2.0``|``-Python-2.7.12``|``intel/2016b``
``3.2.0``|``-cli``          |``system``
``3.2.2``|``-Python-2.7.12``|``intel/2016b``
``3.2.2``|``-cli``          |``system``

<a anchor="hapgen2"/>
### *HAPGEN2*

'HAPGEN2' simulates case control datasets at SNP markers.

*homepage*: https://mathgen.stats.ox.ac.uk/genetics_software/hapgen/hapgen2.html

version  |toolchain
---------|----------
``2.2.0``|``system``

<a anchor="harfbuzz"/>
### *HarfBuzz*

HarfBuzz is an OpenType text shaping engine.

*homepage*: https://www.freedesktop.org/wiki/Software/HarfBuzz

version  |toolchain
---------|------------------------------------------------
``1.1.3``|``foss/2016a``, ``intel/2016a``
``1.2.7``|``foss/2016a``, ``intel/2016a``
``1.3.1``|``foss/2016b``, ``intel/2016b``, ``intel/2017a``
``1.5.1``|``intel/2017a``
``1.7.1``|``foss/2017b``, ``intel/2017b``
``1.7.5``|``foss/2018a``, ``intel/2018a``
``1.9.0``|``fosscuda/2018b``
``2.2.0``|``foss/2018b``, ``fosscuda/2018b``
``2.4.0``|``GCCcore/8.2.0``
``2.6.4``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``2.6.7``|``GCCcore/10.2.0``
``2.8.1``|``GCCcore/10.3.0``
``2.8.2``|``GCCcore/11.2.0``
``4.2.1``|``GCCcore/11.3.0``
``5.3.1``|``GCCcore/12.2.0``

<a anchor="harminv"/>
### *Harminv*

Harminv is a free program (and accompanying library) to solve the problem of harmonic inversion -  given a discrete-time, finite-length signal that consists of a sum of finitely-many sinusoids (possibly exponentially  decaying) in a given bandwidth, it determines the frequencies, decay constants, amplitudes, and phases of those  sinusoids.

*homepage*: https://github.com/stevengj/harminv

version  |toolchain
---------|----------------------------------------------------------------
``1.4``  |``foss/2016a``
``1.4.1``|``foss/2017b``, ``foss/2018a``, ``intel/2018a``, ``intel/2020a``

<a anchor="harmony"/>
### *harmony*

Harmony is a general-purpose R package with an efficient algorithm for integrating multiple data sets.

*homepage*: https://portals.broadinstitute.org/harmony

version           |versionsuffix|toolchain
------------------|-------------|--------------
``1.0.0-20200224``|``-R-4.0.0`` |``foss/2020a``

<a anchor="hbase"/>
### *HBase*

Apache HBase. is the Hadoop database, a distributed, scalable, big data store. 

*homepage*: http://hbase.apache.org/

version  |toolchain
---------|----------
``1.0.2``|``system``

<a anchor="hdbscan"/>
### *HDBSCAN*

The hdbscan library is a suite of tools to use unsupervised learning to find clusters, or dense  regions, of a dataset. The primary algorithm is HDBSCAN* as proposed by Campello, Moulavi, and Sander. The library  provides a high performance implementation of this algorithm, along with tools for analysing the resulting  clustering.

*homepage*: http://hdbscan.readthedocs.io/en/latest/

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.8.24``|``-Python-3.7.4``|``foss/2019b``
``0.8.27``|                 |``foss/2021a``

<a anchor="hddm"/>
### *HDDM*

HDDM is a Python toolbox for hierarchical Bayesian parameter estimation  of the Drift Diffusion Model (via PyMC).

*homepage*: http://ski.clps.brown.edu/hddm_docs

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.6.1``|``-Python-3.6.6`` |``intel/2018b``
``0.7.5``|``-Python-2.7.16``|``intel/2019b``
``0.7.5``|``-Python-3.7.4`` |``intel/2019b``

<a anchor="hdf"/>
### *HDF*

HDF (also known as HDF4) is a library and multi-object file format for  storing and managing data between machines. 

*homepage*: https://www.hdfgroup.org/products/hdf4/

version   |versionsuffix |toolchain
----------|--------------|-------------------------------------------------------------------------------------------------
``4.2.11``|              |``intel/2016a``
``4.2.12``|              |``intel/2017a``
``4.2.13``|              |``GCCcore/6.4.0``
``4.2.13``|``-no-netcdf``|``intel/2017a``
``4.2.14``|              |``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``4.2.15``|              |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/9.3.0``

<a anchor="hdf-eos"/>
### *HDF-EOS*

HDF-EOS libraries are software libraries built on HDF libraries.  It supports three data structures for remote sensing data: Grid, Point and Swath.

*homepage*: https://hdfeos.org/

version |toolchain
--------|--------------------------------------------------------
``2.20``|``GCCcore/10.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.3.0``

<a anchor="hdf-eos5"/>
### *HDF-EOS5*

HDF-EOS libraries are software libraries built on HDF libraries.  It supports three data structures for remote sensing data: Grid, Point and Swath.

*homepage*: https://hdfeos.org/

version |toolchain
--------|------------------------------------------------
``1.16``|``foss/2018b``, ``gompi/2019b``, ``gompi/2020b``

<a anchor="hdf5"/>
### *HDF5*

HDF5 is a data model, library, and file format for storing and managing data.  It supports an unlimited variety of datatypes, and is designed for flexible  and efficient I/O and for high volume and complex data.

*homepage*: https://portal.hdfgroup.org/display/support

version          |versionsuffix|toolchain
-----------------|-------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.8.10``       |``-serial``  |``GCC/4.8.1``
``1.8.11``       |``-serial``  |``GCC/4.8.1``
``1.8.12``       |             |``foss/2018b``, ``intel/2016b``
``1.8.13``       |             |``foss/2018b``
``1.8.16``       |             |``foss/2016a``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``
``1.8.16``       |``-serial``  |``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``1.8.17``       |             |``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``1.8.17``       |``-serial``  |``GCC/5.4.0-2.26``, ``foss/2016a``, ``intel/2016b``
``1.8.18``       |             |``foss/2016b``, ``foss/2017a``, ``gimkl/2017a``, ``intel/2016b``, ``intel/2017.01``, ``intel/2017a``
``1.8.18``       |``-serial``  |``intel/2017a``
``1.8.19``       |             |``foss/2017a``, ``foss/2017b``, ``intel/2017a``, ``intel/2017b``
``1.8.20``       |             |``foss/2018a``, ``gmpolf/2017.10``, ``intel/2017b``, ``intel/2018a``
``1.10.0-patch1``|             |``foss/2016b``, ``intel/2016b``, ``intel/2017.01``, ``intel/2017a``
``1.10.1``       |             |``foss/2017a``, ``foss/2017b``, ``foss/2018a``, ``fosscuda/2017b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018.00``, ``intel/2018.01``, ``intel/2018a``, ``intelcuda/2017b``, ``iomkl/2017b``, ``iomkl/2018a``
``1.10.2``       |             |``PGI/18.4-GCC-6.4.0-2.28``, ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``, ``iomkl/2018b``
``1.10.5``       |             |``gompi/2019a``, ``gompi/2019b``, ``gompic/2019a``, ``gompic/2019b``, ``iimpi/2019a``, ``iimpi/2019b``, ``iimpic/2019a``, ``iimpic/2019b``, ``iompi/2019b``
``1.10.5``       |``-serial``  |``GCCcore/8.3.0``
``1.10.6``       |             |``gompi/2020a``, ``gompic/2020a``, ``iimpi/2020a``, ``iimpic/2020a``
``1.10.7``       |             |``gompi/2020b``, ``gompi/2021a``, ``gompic/2020b``, ``iimpi/2020b``, ``iimpi/2021a``, ``iimpic/2020b``
``1.10.8``       |             |``gompi/2021b``, ``gompi/2022a``
``1.12.0``       |             |``gompi/2020a``, ``iimpi/2020a``
``1.12.1``       |             |``gompi/2021a``, ``gompi/2021b``, ``iimpi/2021b``
``1.12.2``       |             |``gompi/2022a``, ``iimpi/2022a``
``1.12.2``       |``-serial``  |``GCCcore/11.3.0``
``1.13.1``       |             |``gompi/2022a``, ``iimpi/2022a``
``1.13.1``       |``-serial``  |``GCCcore/11.3.0``

<a anchor="hdf5storage"/>
### *hdf5storage*

This Python package provides high level utilities to read/write a variety of Python types to/from  HDF5 (Heirarchal Data Format) formatted files. This package also provides support for MATLAB MAT v7.3 formatted  files, which are just HDF5 files with a different extension and some extra meta-data. All of this is done without  pickling data. Pickling is bad for security because it allows arbitrary code to be executed in the interpreter.  One wants to be able to read possibly HDF5 and MAT files from untrusted sources, so pickling is avoided in this  package.

*homepage*: https://pythonhosted.org/hdf5storage/

version   |toolchain
----------|----------------------------------
``0.1.15``|``foss/2019a``, ``fosscuda/2019a``

<a anchor="hdfview"/>
### *HDFView*

HDFView is a visual tool for browsing and editing HDF4 and HDF5 files.

*homepage*: https://support.hdfgroup.org/products/java/hdfview/

version |versionsuffix              |toolchain
--------|---------------------------|----------
``2.14``|``-Java-1.8.0_152-centos6``|``system``
``2.14``|``-Java-1.8.0_152-centos7``|``system``

<a anchor="healpix"/>
### *HEALPix*

Hierarchical Equal Area isoLatitude Pixelation of a sphere.

*homepage*: http://healpix.sourceforge.net/

version |toolchain
--------|------------------------------------
``3.50``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``

<a anchor="heaptrack"/>
### *heaptrack*

A heap memory profiler for Linux.

*homepage*: http://milianw.de/blog/heaptrack-a-heap-memory-profiler-for-linux

version  |toolchain
---------|--------------
``1.1.0``|``foss/2016b``

<a anchor="hector"/>
### *hector*

This is the repository for Hector, an open source, object-oriented,  simple global climate carbon-cycle model. It runs essentially instantaneously while  still representing the most critical global scale earth system processes, and is  one of a class of models heavily used for for emulating complex climate models  and uncertainty analyses.

*homepage*: https://github.com/JGCRI/hector

version  |toolchain
---------|--------------
``2.5.0``|``GCC/11.2.0``

<a anchor="heffte"/>
### *HeFFTe*

Highly Efficient FFT for Exascale (HeFFTe) library

*homepage*: https://icl.utk.edu/fft

version|toolchain
-------|--------------
``1.0``|``foss/2020a``

<a anchor="hello"/>
### *Hello*

The GNU Hello program produces a familiar, friendly greeting. Yes, this is another implementation of the classic program that prints "Hello, world!" when you run it.  However, unlike the minimal version often seen, GNU Hello processes its argument list to modify its behavior, supports greetings in many languages, and so on. 

*homepage*: https://www.gnu.org/software/hello/

version |toolchain
--------|-----------------
``2.10``|``GCCcore/8.2.0``

<a anchor="help2man"/>
### *help2man*

help2man produces simple manual pages from the '--help' and '--version' output of other commands.

*homepage*: https://www.gnu.org/software/help2man/

version    |toolchain
-----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.47.4`` |``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.1.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/system``, ``gimkl/2017a``, ``intel/2016b``, ``system``
``1.47.5`` |``GCCcore/5.5.0``
``1.47.6`` |``GCCcore/8.1.0``
``1.47.7`` |``GCCcore/8.2.0``
``1.47.8`` |``GCCcore/7.4.0``, ``GCCcore/8.3.0``, ``GCCcore/8.4.0``
``1.47.10``|``GCCcore/9.1.0``, ``GCCcore/9.2.0``
``1.47.12``|``GCCcore/9.3.0``
``1.47.15``|``GCCcore/10.1.0``
``1.47.16``|``GCCcore/10.2.0``
``1.48.3`` |``FCC/4.5.0``, ``GCCcore/10.3.0``, ``GCCcore/11.1.0``, ``GCCcore/11.2.0``, ``GCCcore/9.4.0``
``1.49.2`` |``GCCcore/11.3.0``, ``GCCcore/12.1.0``, ``GCCcore/12.2.0``, ``GCCcore/9.5.0``

<a anchor="hh-suite"/>
### *HH-suite*

The HH-suite is an open-source software package  for sensitive protein sequence searching based on the pairwise  alignment of hidden Markov models (HMMs).

*homepage*: https://github.com/soedinglab/hh-suite

version       |versionsuffix    |toolchain
--------------|-----------------|-----------------------------------------------------------------------------------
``3.0-beta.3``|                 |``intel/2018a``
``3.2.0``     |                 |``foss/2019b``, ``fosscuda/2019b``
``3.3.0``     |                 |``foss/2020a``, ``gompi/2020b``, ``gompi/2021a``, ``gompi/2021b``, ``gompic/2020b``
``3.3.0``     |``-Python-3.7.4``|``gompic/2019b``

<a anchor="hic-pro"/>
### *HiC-Pro*

HiC-Pro was designed to process Hi-C data, from raw fastq files (paired-end Illumina data)   to the normalized contact maps.

*homepage*: http://nservant.github.io/HiC-Pro

version  |versionsuffix     |toolchain
---------|------------------|--------------
``2.9.0``|``-Python-2.7.12``|``foss/2016b``

<a anchor="hicexplorer"/>
### *HiCExplorer*

HiCexplorer addresses the common tasks of Hi-C analysis from processing to visualization.

*homepage*: http://hicexplorer.readthedocs.org/

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``2.1.1``|``-Python-3.6.4``|``foss/2018a``

<a anchor="hierfstat"/>
### *hierfstat*

Estimates hierarchical F-statistics from haploid or diploid genetic data with any numbers of levels in the hierarchy.

*homepage*: https://cran.r-project.org/package=hierfstat

version  |versionsuffix            |toolchain
---------|-------------------------|--------------
``0.5-7``|``-R-4.0.0-Python-3.8.2``|``foss/2020a``

<a anchor="hifiasm"/>
### *hifiasm*

Hifiasm: a haplotype-resolved assembler for accurate Hifi reads.

*homepage*: https://github.com/chhylp123/hifiasm

version   |toolchain
----------|-------------------------------------
``0.15.2``|``GCCcore/10.3.0``, ``GCCcore/9.3.0``
``0.16.1``|``GCCcore/10.3.0``

<a anchor="highfive"/>
### *HighFive*

HighFive is a modern header-only C++11 friendly interface for libhdf5.

*homepage*: https://github.com/BlueBrain/HighFive

version  |toolchain
---------|---------------
``2.6.2``|``gompi/2021a``

<a anchor="highway"/>
### *Highway*

Highway is a C++ library for SIMD (Single Instruction, Multiple Data), i.e. applying the same operation to 'lanes'.

*homepage*: https://github.com/google/highway

version   |toolchain
----------|--------------------------------------
``0.12.2``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``

<a anchor="hip"/>
### *HIP*

HIP is a C++ Runtime API and Kernel Language that allows developers to create portable applications for AMD and NVIDIA GPUs from single source code.

*homepage*: https://github.com/ROCm-Developer-Tools/HIP

version  |versionsuffix|toolchain
---------|-------------|------------------
``4.5.0``|``-amd``     |``GCCcore/11.2.0``

<a anchor="hipify-clang"/>
### *hipify-clang*

Hipify-clang is a clang-based tool for translating CUDA   sources into HIP sources. It translates CUDA source into an abstract syntax   tree, which is traversed by transformation matchers. After applying all the   matchers, the output HIP source is produced.

*homepage*: https://github.com/ROCm-Developer-Tools/HIPIFY

version  |toolchain
---------|-----------------
``4.2.0``|``gcccuda/2020b``

<a anchor="hips"/>
### *HIPS*

HIPS (Hierarchical Iterative Parallel Solver) is a scientific library that provides  an efficient parallel iterative solver for very large sparse linear systems. 

*homepage*: http://hips.gforge.inria.fr/

version     |toolchain
------------|--------------
``1.2b-rc5``|``foss/2017b``

<a anchor="hipsycl"/>
### *hipSYCL*

hipSYCL is a modern SYCL implementation targeting CPUs and GPUs, with a focus on leveraging existing toolchains such as CUDA or HIP

*homepage*: https://github.com/illuhad/hipSYCL

version  |toolchain
---------|---------------------------------
``0.9.1``|``GCC/10.2.0``, ``gcccuda/2020b``

<a anchor="hiredis"/>
### *hiredis*

Hiredis is a minimalistic C client library for the Redis database.  It is minimalistic because it just adds minimal support for the protocol, but at the same time it uses a high level printf-alike API in order to make it much higher level than otherwise suggested by its minimal code base and the lack of explicit bindings for every Redis command.

*homepage*: https://github.com/redis/hiredis

version  |toolchain
---------|--------------------------------------
``1.0.2``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="hisat2"/>
### *HISAT2*

HISAT2 is a fast and sensitive alignment program for mapping next-generation sequencing reads  (both DNA and RNA) against the general human population (as well as against a single reference genome).

*homepage*: https://daehwankimlab.github.io/hisat2

version       |toolchain
--------------|---------------------------------------------------------------------------------
``2.0.3-beta``|``intel/2016a``
``2.0.4``     |``foss/2016b``
``2.0.5``     |``intel/2017a``
``2.1.0``     |``foss/2017b``, ``foss/2018b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``
``2.2.0``     |``foss/2018b``
``2.2.1``     |``foss/2019b``, ``foss/2020a``, ``gompi/2020b``, ``gompi/2021b``

<a anchor="histolab"/>
### *histolab*

Library for Digital Pathology Image Processing

*homepage*: https://github.com/histolab/histolab

version  |toolchain
---------|------------------------------
``0.4.1``|``foss/2021a``, ``foss/2021b``

<a anchor="hivtrace"/>
### *hivtrace*

HIV-TRACE is an application that identifies potential transmission clusters within a  supplied FASTA file with an option to find potential links against the Los Alamos HIV Sequence Database.

*homepage*: https://github.com/veg/hivtrace

version  |toolchain
---------|--------------
``0.6.2``|``foss/2021a``

<a anchor="hl7apy"/>
### *hl7apy*

Python library to parse, create and handle HL7 v2 messages.

*homepage*: https://github.com/crs4/hl7apy

version  |versionsuffix    |toolchain
---------|-----------------|-----------------
``1.3.3``|``-Python-3.7.4``|``GCCcore/8.3.0``

<a anchor="hlaminer"/>
### *HLAminer*

HLAminer is a software for HLA predictions from next-generation shotgun (NGS) sequence read data and supports direct  read alignment and targeted de novo assembly of sequence reads. 

*homepage*: http://www.bcgsc.ca/platform/bioinfo/software/hlaminer

version|versionsuffix   |toolchain
-------|----------------|--------------
``1.4``|``-Perl-5.28.0``|``foss/2018b``

<a anchor="hmmer"/>
### *HMMER*

HMMER is used for searching sequence databases for homologs  of protein sequences, and for making protein sequence alignments. It  implements methods using probabilistic models called profile hidden Markov  models (profile HMMs).  Compared to BLAST, FASTA, and other sequence  alignment and database search tools based on older scoring methodology,  HMMER aims to be significantly more accurate and more able to detect remote  homologs because of the strength of its underlying mathematical models. In the  past, this strength came at significant computational expense, but in the new  HMMER3 project, HMMER is now essentially as fast as BLAST.

*homepage*: http://hmmer.org/

version  |toolchain
---------|--------------------------------------------------------------------------------------------------------------------------------------------
``3.1b2``|``GCC/6.4.0-2.28``, ``foss/2016a``, ``foss/2016b``, ``foss/2018a``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``, ``intel/2017a``, ``intel/2018a``
``3.2.1``|``GCC/8.2.0-2.31.1``, ``foss/2018b``, ``gompi/2019b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``iimpi/2019b``, ``intel/2018b``
``3.3.1``|``gompi/2020a``, ``iimpi/2020a``
``3.3.2``|``gompi/2019b``, ``gompi/2020a``, ``gompi/2020b``, ``gompi/2021a``, ``gompi/2021b``, ``gompic/2020b``, ``iimpi/2020b``, ``iimpi/2021b``

<a anchor="hmmer2"/>
### *HMMER2*

HMMER is used for searching sequence databases for sequence homologs,  and for making sequence alignments.

*homepage*: http://hmmer.org

version  |toolchain
---------|---------------------------------------------
``2.3.2``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/8.3.0``

<a anchor="hmmlearn"/>
### *hmmlearn*

hmmlearn is a set of algorithms for unsupervised learning and inference of Hidden Markov Models

*homepage*: https://github.com/hmmlearn/hmmlearn

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.2.0``|``-Python-2.7.14``|``intel/2017b``
``0.2.0``|``-Python-3.6.4`` |``intel/2018a``

<a anchor="home"/>
### *HOME*

HOME (histogram of methylation) is a python package for differential methylation region (DMR) identification. The method uses histogram of methylation features and the linear Support Vector Machine (SVM) to identify DMRs  from whole genome bisulfite sequencing (WGBS) data.

*homepage*: https://github.com/ListerLab/HOME

version|versionsuffix     |toolchain
-------|------------------|--------------
``0.9``|``-Python-2.7.13``|``foss/2017a``

<a anchor="horovod"/>
### *Horovod*

Horovod is a distributed training framework for TensorFlow.

*homepage*: https://github.com/uber/horovod

version   |versionsuffix                      |toolchain
----------|-----------------------------------|----------------------------------
``0.9.10``|``-Python-3.6.3``                  |``intel/2017b``
``0.18.1``|``-Python-3.7.2``                  |``foss/2019a``
``0.18.2``|``-TensorFlow-1.15.0-Python-3.7.4``|``fosscuda/2019b``
``0.18.2``|``-TensorFlow-1.15.2-Python-3.7.4``|``fosscuda/2019b``
``0.18.2``|``-TensorFlow-2.0.0-Python-3.7.4`` |``fosscuda/2019b``
``0.19.1``|``-TensorFlow-2.1.0-Python-3.7.4`` |``fosscuda/2019b``
``0.19.5``|``-TensorFlow-2.2.0-Python-3.7.4`` |``fosscuda/2019b``
``0.20.3``|``-TensorFlow-2.3.1-Python-3.7.4`` |``fosscuda/2019b``
``0.21.1``|``-PyTorch-1.7.1-Python-3.7.4``    |``fosscuda/2019b``
``0.21.1``|``-TensorFlow-2.4.1``              |``fosscuda/2020b``
``0.21.1``|``-TensorFlow-2.4.1-Python-3.7.4`` |``fosscuda/2019b``
``0.21.3``|``-PyTorch-1.7.1``                 |``fosscuda/2020b``
``0.21.3``|``-TensorFlow-2.3.1-Python-3.8.2`` |``foss/2020a``, ``fosscuda/2020a``
``0.22.0``|``-PyTorch-1.8.1``                 |``fosscuda/2020b``
``0.22.1``|``-CUDA-11.3.1-TensorFlow-2.6.0``  |``foss/2021a``
``0.22.1``|``-TensorFlow-2.5.0``              |``fosscuda/2020b``
``0.22.1``|``-TensorFlow-2.5.0-Python-3.7.4`` |``fosscuda/2019b``
``0.23.0``|``-CUDA-11.3.1-PyTorch-1.10.0``    |``foss/2021a``
``0.23.0``|``-TensorFlow-2.5.0``              |``fosscuda/2020b``

<a anchor="horton"/>
### *horton*

HORTON is a Helpful Open-source Research TOol for N-fermion systems, written  primarily in the Python programming language. (HORTON is named after the helpful  pachyderm, not the Canadian caffeine supply store.) The ultimate goal of HORTON  is to provide a platform for testing new ideas on the quantum many-body problem  at a reasonable computational cost. Although HORTON is primarily designed to be  a quantum-chemistry program, it can perform computations involving model  Hamiltonians, and could be extended for computations in nuclear physics.

*homepage*: https://theochem.github.io/horton

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``2.1.1``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``2.1.1``|``-Python-2.7.18``|``intel/2020a``

<a anchor="how_are_we_stranded_here"/>
### *how_are_we_stranded_here*

Python package for testing strandedness of RNA-Seq fastq files

*homepage*: https://github.com/signalbash/how_are_we_stranded_here

version  |toolchain
---------|--------------
``1.0.1``|``foss/2021b``

<a anchor="hpcg"/>
### *HPCG*

The HPCG Benchmark project is an effort to create a more relevant metric for ranking HPC systems than  the High Performance LINPACK (HPL) benchmark, that is currently used by the TOP500 benchmark.

*homepage*: https://software.sandia.gov/hpcg

version|toolchain
-------|-------------------------------------------------------------------------------------------------
``3.0``|``foss/2016b``, ``foss/2018b``, ``intel/2018b``
``3.1``|``foss/2018b``, ``foss/2021a``, ``foss/2021b``, ``intel/2018b``, ``intel/2021a``, ``intel/2021b``

<a anchor="hpcx"/>
### *HPCX*

The Mellanox HPC-X Toolkit is a comprehensive MPI and SHMEM/PGAS software suite for high performance computing environments

*homepage*: http://www.mellanox.com/page/products_dyn?product_family=189&mtag=hpc-x

version  |toolchain
---------|--------------------
``2.3.0``|``GCC/8.2.0-2.31.1``

<a anchor="hpdbscan"/>
### *HPDBSCAN*

Highly parallel density based spatial clustering for application with noise

*homepage*: https://github.com/Markus-Goetz/hpdbscan

version     |toolchain
------------|--------------
``20171110``|``foss/2017b``
``20210826``|``foss/2020b``

<a anchor="hpl"/>
### *HPL*

HPL is a software package that solves a (random) dense linear system in double precision (64 bits)  arithmetic on distributed-memory computers. It can thus be regarded as a portable as well as freely available  implementation of the High Performance Computing Linpack Benchmark.

*homepage*: https://www.netlib.org/benchmark/hpl/

version|toolchain
-------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.1``|``foss/2016.04``, ``foss/2016.06``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``gmpolf/2016a``, ``gmvolf/1.7.20``, ``gmvolf/2016a``, ``intel/2016.00``, ``intel/2016.01``, ``intel/2016.02-GCC-4.9``, ``intel/2016.02-GCC-5.3``, ``intel/2016.03-GCC-4.9``, ``intel/2016.03-GCC-5.3``, ``intel/2016.03-GCC-5.4``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``pomkl/2016.03``, ``pomkl/2016.04``, ``pomkl/2016.09``
``2.2``|``foss/2016.07``, ``foss/2016.09``, ``foss/2017a``, ``foss/2017b``, ``foss/2018.08``, ``foss/2018a``, ``foss/2018b``, ``fosscuda/2017b``, ``fosscuda/2018a``, ``fosscuda/2018b``, ``gimkl/2018b``, ``giolf/2017b``, ``giolf/2018a``, ``giolfc/2017b``, ``gmpolf/2017.10``, ``goblf/2018b``, ``gomkl/2018b``, ``intel/2017.00``, ``intel/2017.01``, ``intel/2017.02``, ``intel/2017.09``, ``intel/2017a``, ``intel/2017b``, ``intel/2018.00``, ``intel/2018.01``, ``intel/2018.02``, ``intel/2018.04``, ``intel/2018a``, ``intel/2018b``, ``intel/2019.00``, ``intel/2019.01``, ``intelcuda/2016.10``, ``intelcuda/2017b``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``iomkl/2016.09-GCC-5.4.0-2.26``, ``iomkl/2017.01``, ``iomkl/2017a``, ``iomkl/2017b``, ``iomkl/2018.02``, ``iomkl/2018a``, ``iomkl/2018b``, ``pomkl/2016.09``
``2.3``|``CrayCCE/19.06``, ``CrayGNU/19.06``, ``CrayIntel/19.06``, ``Fujitsu/21.05``, ``foss/2019a``, ``foss/2019b``, ``foss/2020a``, ``foss/2020b``, ``foss/2021a``, ``foss/2021b``, ``foss/2022.05``, ``foss/2022.10``, ``foss/2022a``, ``foss/2022b``, ``fosscuda/2019b``, ``fosscuda/2020a``, ``gobff/2020.06-amd``, ``gobff/2020.11``, ``gobff/2020b``, ``gobff/2021a``, ``goblf/2020b``, ``gomkl/2019a``, ``gomkl/2020b``, ``gomkl/2021a``, ``iibff/2020b``, ``intel/2019.02``, ``intel/2019.03``, ``intel/2019a``, ``intel/2019b``, ``intel/2020.00``, ``intel/2020.06-impi-18.5``, ``intel/2020.12``, ``intel/2020a``, ``intel/2020b``, ``intel/2021a``, ``intel/2021b``, ``intel/2022.00``, ``intel/2022.09``, ``intel/2022.11``, ``intel/2022b``, ``intelcuda/2019b``, ``intelcuda/2020a``, ``iomkl/2019.01``, ``iomkl/2021a``

<a anchor="htop"/>
### *htop*

An interactive process viewer for Unix

*homepage*: https://htop.dev

version  |toolchain
---------|----------
``2.0.0``|``system``
``2.0.1``|``system``
``3.2.1``|``system``

<a anchor="htseq"/>
### *HTSeq*

HTSeq is a Python library to facilitate processing and analysis  of data from high-throughput sequencing (HTS) experiments.

*homepage*: https://github.com/simon-anders/htseq

version    |versionsuffix     |toolchain
-----------|------------------|------------------------------------------------
``0.6.1p1``|``-Python-2.7.11``|``foss/2016a``
``0.6.1p1``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``0.9.1``  |``-Python-2.7.12``|``foss/2016b``
``0.9.1``  |``-Python-2.7.13``|``intel/2017a``
``0.9.1``  |``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``, ``intel/2018a``
``0.9.1``  |``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``0.10.0`` |``-Python-2.7.14``|``foss/2018a``
``0.11.0`` |``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``0.11.2`` |``-Python-3.6.6`` |``foss/2018b``
``0.11.2`` |``-Python-3.7.2`` |``foss/2019a``
``0.11.2`` |``-Python-3.7.4`` |``foss/2019b``
``0.11.3`` |                  |``foss/2020b``, ``foss/2021b``
``2.0.2``  |                  |``foss/2022a``

<a anchor="htslib"/>
### *HTSlib*

PacBio fork of C library for reading/writing high-throughput sequencing data.  This package includes the utilities bgzip and tabix

*homepage*: https://github.com/PacificBiosciences/htslib

version     |versionsuffix|toolchain
------------|-------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------
``1.2.1``   |             |``foss/2016b``
``1.3``     |             |``foss/2016a``, ``intel/2016a``
``1.3.1``   |             |``foss/2016a``, ``foss/2016b``, ``intel/2016b``
``1.3.2``   |             |``intel/2016b``
``1.4``     |             |``foss/2016b``, ``intel/2016b``
``1.4.1``   |             |``foss/2016a``, ``intel/2017a``
``1.6``     |             |``foss/2016b``, ``foss/2017b``, ``intel/2017b``
``1.7``     |             |``intel/2018a``
``1.8``     |             |``GCC/6.4.0-2.28``, ``foss/2018a``, ``intel/2018a``
``1.9``     |             |``GCC/6.4.0-2.28``, ``GCC/8.2.0-2.31.1``, ``foss/2018b``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2018b``
``1.10.2``  |             |``GCC/8.3.0``, ``GCC/9.3.0``, ``iccifort/2019.5.281``
``1.11``    |             |``GCC/10.2.0``, ``iccifort/2020.4.304``
``1.12``    |             |``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/9.3.0``
``1.14``    |             |``GCC/11.2.0``
``1.15.1``  |             |``GCC/11.3.0``
``20160107``|``-PacBio``  |``intel/2017a``

<a anchor="hub"/>
### *hub*

hub is a command-line wrapper for git that makes you better at GitHub.

*homepage*: https://hub.github.com/

version  |versionsuffix   |toolchain
---------|----------------|----------
``2.2.2``|``-linux-amd64``|``system``

<a anchor="humann"/>
### *humann*

HUMAnN v3 is a pipeline for efficiently  and accurately determining the coverage and abundance  of microbial pathways in a community from metagenomic data. Sequencing a metagenome typically produces millions  of short DNA/RNA reads. This process, referred  to as functional profiling, aims to describe the metabolic potential  of a microbial community and its members. More generally,  functional profiling answers the question:  What are the microbes in my community-of-interest doing (or capable of doing)?

*homepage*: http://huttenhower.sph.harvard.edu/humann

version|toolchain
-------|--------------
``3.6``|``foss/2022a``

<a anchor="hunspell"/>
### *hunspell*

Hunspell is a spell checker and morphological analyzer library and program designed for languageswith rich morphology and complex word compounding or character encoding.

*homepage*: https://hunspell.github.io/

version  |toolchain
---------|--------------------------------------
``1.6.1``|``intel/2017a``
``1.7.0``|``GCCcore/8.2.0``
``1.7.1``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="hwloc"/>
### *hwloc*

The Portable Hardware Locality (hwloc) software package provides a portable  abstraction (across OS, versions, architectures, ...) of the hierarchical  topology of modern architectures, including NUMA memory nodes, sockets, shared  caches, cores and simultaneous multithreading. It also gathers various system  attributes such as cache and memory information as well as the locality of I/O  devices such as network interfaces, InfiniBand HCAs or GPUs. It primarily  aims at helping applications with gathering information about modern computing  hardware so as to exploit it accordingly and efficiently. 

*homepage*: https://www.open-mpi.org/projects/hwloc/

version    |toolchain
-----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.7.2``  |``GCC/4.8.2``
``1.8.1``  |``GCC/4.8.2``, ``GCC/4.8.3``
``1.9``    |``GCC/4.8.3``
``1.10.0`` |``GCC/4.9.2``
``1.10.1`` |``GCC/4.8.4``, ``GNU/4.9.2-2.25``
``1.11.0`` |``GNU/4.9.3-2.25``
``1.11.1`` |``GCC/4.9.3``
``1.11.2`` |``GCC/4.9.3-2.25``, ``GNU/4.9.3-2.25``
``1.11.3`` |``GCC/5.2.0``, ``GCC/5.3.0-2.26``, ``GCC/5.4.0-2.26``, ``GCC/6.1.0-2.27``, ``PGI/16.3-GCC-4.9.3-2.25``, ``PGI/16.4-GCC-5.3.0-2.26``, ``iccifort/2016.3.210-GCC-4.9.3-2.25``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``, ``intel/2016a``, ``intel/2016b``
``1.11.4`` |``GCC/6.2.0-2.27``, ``PGI/16.7-GCC-5.4.0-2.26``, ``iccifort/2017.1.132-GCC-5.4.0-2.26``
``1.11.5`` |``GCC/5.4.0-2.26``, ``GCC/6.3.0-2.27``, ``iccifort/2017.1.132-GCC-6.3.0-2.27``
``1.11.6`` |``GCC/6.3.0-2.28``
``1.11.7`` |``GCCcore/5.4.0``, ``GCCcore/6.4.0``
``1.11.8`` |``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``intel/2017a``
``1.11.10``|``GCCcore/7.3.0``
``1.11.11``|``GCCcore/8.2.0``
``1.11.12``|``GCCcore/8.3.0``
``2.0.2``  |``GCCcore/8.2.0``
``2.0.3``  |``GCCcore/8.3.0``
``2.1.0``  |``GCCcore/9.2.0``
``2.2.0``  |``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``2.4.1``  |``GCCcore/10.3.0``
``2.5.0``  |``GCCcore/11.2.0``
``2.7.1``  |``GCCcore/11.3.0``
``2.8.0``  |``GCCcore/12.2.0``

<a anchor="hydra"/>
### *Hydra*

Hydra is an open-source Python framework that simplifies the development of research and other complex applications. The key feature is the ability to dynamically create a hierarchical configuration by composition and override it through config files and the command line. The name Hydra comes from its ability to run multiple similar jobs - much like a Hydra with multiple heads.

*homepage*: https://hydra.cc/

version  |toolchain
---------|------------------
``1.1.1``|``GCCcore/10.3.0``

<a anchor="hyperopt"/>
### *Hyperopt*

hyperopt is a Python library for optimizing over awkward search spaces with real-valued,  discrete, and conditional dimensions.

*homepage*: http://hyperopt.github.io/hyperopt/

version  |versionsuffix             |toolchain
---------|--------------------------|------------------
``0.1``  |``-Python-2.7.14``        |``intel/2017b``
``0.1.1``|``-Python-3.6.6``         |``intel/2018b``
``0.2.4``|``-Python-3.7.4-Java-1.8``|``intel/2019b``
``0.2.5``|                          |``fosscuda/2020b``
``0.2.7``|                          |``foss/2021a``

<a anchor="hyperqueue"/>
### *HyperQueue*

HyperQueue lets you build a computation plan consisting of a large amount of tasks and then execute it transparently over a system like SLURM/PBS. It dynamically groups jobs into SLURM/PBS jobs and distributes them to fully utilize allocated nodes. You thus do not have to manually aggregate your tasks into SLURM/PBS jobs.

*homepage*: https://it4innovations.github.io/hyperqueue/

version   |toolchain
----------|----------
``0.13.0``|``system``

<a anchor="hyperspy"/>
### *hyperspy*

HyperSpy is an open source Python library which provides tools to facilitate the interactive  data analysis of multi-dimensional datasets that can be described as multi-dimensional arrays of a given signal  (e.g. a 2D array of spectra a.k.a spectrum image)

*homepage*: https://hyperspy.org/

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``1.1.1``|``-Python-3.5.2``|``intel/2016b``

<a anchor="hyphy"/>
### *HyPhy*

HyPhy (Hypothesis Testing using Phylogenies) is an open-source software package   for the analysis of genetic sequences (in particular the inference of natural selection)   using techniques in phylogenetics, molecular evolution, and machine learning

*homepage*: https://veg.github.io/hyphy-site/

version   |toolchain
----------|---------------
``2.3.13``|``foss/2016b``
``2.5.1`` |``gompi/2019a``
``2.5.33``|``gompi/2021a``

<a anchor="hypo"/>
### *HyPo*

HyPo: Super Fast & Accurate Polisher for Long Read Genome Assemblies

*homepage*: https://github.com/kensung-lab/hypo

version  |versionsuffix    |toolchain
---------|-----------------|-------------
``1.0.3``|``-Python-3.7.4``|``GCC/8.3.0``

<a anchor="hypothesis"/>
### *hypothesis*

Hypothesis is an advanced testing library for Python. It lets you write tests which are parametrized  by a source of examples, and then generates simple and comprehensible examples that make your tests fail. This lets  you find more bugs in your code with less work.

*homepage*: https://github.com/HypothesisWorks/hypothesis

version   |versionsuffix     |toolchain
----------|------------------|----------------------------------
``4.5.0`` |``-Python-3.6.6`` |``foss/2018b``, ``fosscuda/2018b``
``4.23.4``|                  |``GCCcore/8.2.0``
``4.39.3``|``-Python-3.6.4`` |``intel/2018a``
``4.44.2``|``-Python-3.7.4`` |``GCCcore/8.3.0``
``4.53.1``|                  |``GCCcore/10.2.0``
``4.57.1``|``-Python-2.7.18``|``GCCcore/11.2.0``
``5.6.0`` |``-Python-3.8.2`` |``GCCcore/9.3.0``
``5.41.2``|                  |``GCCcore/10.2.0``
``5.41.5``|                  |``GCCcore/10.2.0``
``6.7.0`` |                  |``GCCcore/10.2.0``
``6.13.1``|                  |``GCCcore/10.3.0``
``6.14.6``|                  |``GCCcore/11.2.0``
``6.46.7``|                  |``GCCcore/11.3.0``

<a anchor="hypre"/>
### *Hypre*

Hypre is a library for solving large, sparse linear systems of equations on massively  parallel computers. The problems of interest arise in the simulation codes being developed at LLNL  and elsewhere to study physical phenomena in the defense, environmental, energy, and biological sciences.

*homepage*: https://computation.llnl.gov/projects/hypre-scalable-linear-solvers-multigrid-methods

version   |toolchain
----------|----------------------------------------------------------------
``2.11.1``|``foss/2016a``, ``intel/2016a``
``2.14.0``|``foss/2018a``, ``intel/2018a``
``2.15.1``|``foss/2019a``, ``intel/2019a``
``2.18.2``|``foss/2019b``, ``foss/2020a``, ``intel/2019b``, ``intel/2020a``
``2.20.0``|``foss/2020b``, ``intel/2020b``
``2.21.0``|``foss/2021a``, ``fosscuda/2020b``
``2.24.0``|``intel/2021b``
``2.25.0``|``foss/2022a``

<a anchor="i"/>
### *I*


<a href="#i-cistarget">i-cisTarget</a> - <a href="#i-pi">i-PI</a> - <a href="#i-tasser">I-TASSER</a> - <a href="#i7z">i7z</a> - <a href="#ica-aroma">ICA-AROMA</a> - <a href="#icc">icc</a> - <a href="#iccifort">iccifort</a> - <a href="#iccifortcuda">iccifortcuda</a> - <a href="#ichorcna">ichorCNA</a> - <a href="#icmake">icmake</a> - <a href="#icount">iCount</a> - <a href="#icu">ICU</a> - <a href="#idba-ud">IDBA-UD</a> - <a href="#idemux">idemux</a> - <a href="#ieeg-cli">ieeg-cli</a> - <a href="#ifort">ifort</a> - <a href="#igblast">IgBLAST</a> - <a href="#igmplot">IGMPlot</a> - <a href="#igraph">igraph</a> - <a href="#igv">IGV</a> - <a href="#igv-reports">igv-reports</a> - <a href="#igvtools">IGVTools</a> - <a href="#iibff">iibff</a> - <a href="#iimkl">iimkl</a> - <a href="#iimpi">iimpi</a> - <a href="#iimpic">iimpic</a> - <a href="#ijulia">IJulia</a> - <a href="#ilamb">ILAMB</a> - <a href="#ima2">IMa2</a> - <a href="#ima2p">IMa2p</a> - <a href="#imagecodecs">imagecodecs</a> - <a href="#imageio">imageio</a> - <a href="#imagej">ImageJ</a> - <a href="#imagemagick">ImageMagick</a> - <a href="#imake">imake</a> - <a href="#imath">Imath</a> - <a href="#imb">IMB</a> - <a href="#imbalanced-learn">imbalanced-learn</a> - <a href="#imgaug">imgaug</a> - <a href="#imkl">imkl</a> - <a href="#imkl-fftw">imkl-FFTW</a> - <a href="#imlib2">Imlib2</a> - <a href="#immunedeconv">immunedeconv</a> - <a href="#imod">IMOD</a> - <a href="#impi">impi</a> - <a href="#impute2">IMPUTE2</a> - <a href="#inchi">InChI</a> - <a href="#indicators">indicators</a> - <a href="#inelastica">Inelastica</a> - <a href="#infercnv">inferCNV</a> - <a href="#infercnvpy">infercnvpy</a> - <a href="#infernal">Infernal</a> - <a href="#infomap">Infomap</a> - <a href="#inline">inline</a> - <a href="#inparanoid">InParanoid</a> - <a href="#inputproto">inputproto</a> - <a href="#inspector">Inspector</a> - <a href="#intarna">IntaRNA</a> - <a href="#integrate">INTEGRATE</a> - <a href="#integrate-neo">INTEGRATE-Neo</a> - <a href="#intel">intel</a> - <a href="#intel-compilers">intel-compilers</a> - <a href="#intelclusterchecker">IntelClusterChecker</a> - <a href="#intelcuda">intelcuda</a> - <a href="#inteldaal">IntelDAAL</a> - <a href="#intelpython">IntelPython</a> - <a href="#interproscan">InterProScan</a> - <a href="#interproscan_data">InterProScan_data</a> - <a href="#intervaltree">intervaltree</a> - <a href="#intervaltree-python">intervaltree-python</a> - <a href="#intltool">intltool</a> - <a href="#io_lib">io_lib</a> - <a href="#ioapi">ioapi</a> - <a href="#iomkl">iomkl</a> - <a href="#iompi">iompi</a> - <a href="#ior">IOR</a> - <a href="#iozone">IOzone</a> - <a href="#ipm">IPM</a> - <a href="#ipopt">Ipopt</a> - <a href="#ipp">ipp</a> - <a href="#ipy">IPy</a> - <a href="#ipyparallel">ipyparallel</a> - <a href="#ipyrad">ipyrad</a> - <a href="#ipython">IPython</a> - <a href="#iq-tree">IQ-TREE</a> - <a href="#iris">Iris</a> - <a href="#irkernel">IRkernel</a> - <a href="#ironpython">IronPython</a> - <a href="#isa-l">ISA-L</a> - <a href="#isl">ISL</a> - <a href="#isocirc">isoCirc</a> - <a href="#isonet">IsoNet</a> - <a href="#ispc">ispc</a> - <a href="#itac">itac</a> - <a href="#itk">ITK</a> - <a href="#itpp">itpp</a> - <a href="#itstool">ITSTool</a> - <a href="#itsx">ITSx</a> - <a href="#ivar">iVar</a>


<a anchor="i-cistarget"/>
### *i-cisTarget*

An integrative genomics method for the prediction of regulatory features and cis-regulatory modules  in Human, Mouse, and Fly

*homepage*: https://gbiomed.kuleuven.be/apps/lcb/i-cisTarget

version     |versionsuffix     |toolchain
------------|------------------|---------------
``20160602``|``-Python-2.7.11``|``intel/2016a``

<a anchor="i-pi"/>
### *i-PI*

A Python wrapper for (ab initio) (path integrals) molecular dynamics

*homepage*: https://github.com/i-pi/i-pi

version         |versionsuffix     |toolchain
----------------|------------------|---------------
``1.0-20160213``|``-Python-2.7.11``|``intel/2016a``

<a anchor="i-tasser"/>
### *I-TASSER*

I-TASSER is a set of pre-compiled binaries and scripts for protein structure and function modelling and comparison.

*homepage*: http://zhanglab.ccmb.med.umich.edu/I-TASSER/

version|toolchain
-------|----------
``4.0``|``system``
``4.2``|``system``
``5.1``|``system``

<a anchor="i7z"/>
### *i7z*

A better i7 (and now i3, i5) reporting tool for Linux 

*homepage*: https://github.com/ajaiantilal/i7z

version     |toolchain
------------|------------------
``20131012``|``GCCcore/11.2.0``

<a anchor="ica-aroma"/>
### *ICA-AROMA*

ICA-AROMA (i.e. 'ICA-based Automatic Removal Of Motion Artifacts') concerns a data-driven method  to identify and remove motion-related independent components from fMRI data.

*homepage*: https://github.com/maartenmennes/ICA-AROMA

version       |versionsuffix    |toolchain
--------------|-----------------|--------------
``0.4.4-beta``|``-Python-3.6.6``|``foss/2018b``

<a anchor="icc"/>
### *icc*

C and C++ compiler from Intel

*homepage*: http://software.intel.com/en-us/intel-compilers/

version       |versionsuffix        |toolchain
--------------|---------------------|----------
``2016.0.109``|                     |``system``
``2016.0.109``|``-GCC-4.9.3-2.25``  |``system``
``2016.1.150``|``-GCC-4.9.3-2.25``  |``system``
``2016.2.181``|``-GCC-4.9.3-2.25``  |``system``
``2016.2.181``|``-GCC-5.3.0-2.26``  |``system``
``2016.3.210``|``-GCC-4.9.3-2.25``  |``system``
``2016.3.210``|``-GCC-5.3.0-2.26``  |``system``
``2016.3.210``|``-GCC-5.4.0-2.26``  |``system``
``2017.0.098``|``-GCC-5.4.0-2.26``  |``system``
``2017.1.132``|``-GCC-5.4.0-2.26``  |``system``
``2017.1.132``|``-GCC-6.3.0-2.27``  |``system``
``2017.2.174``|``-GCC-6.3.0-2.27``  |``system``
``2017.4.196``|``-GCC-6.4.0-2.28``  |``system``
``2017.5.239``|``-GCC-6.4.0-2.28``  |``system``
``2017.6.256``|``-GCC-6.4.0-2.28``  |``system``
``2017.7.259``|``-GCC-6.4.0-2.28``  |``system``
``2018.0.128``|``-GCC-6.4.0-2.28``  |``system``
``2018.1.163``|``-GCC-6.4.0-2.28``  |``system``
``2018.2.199``|``-GCC-6.4.0-2.28``  |``system``
``2018.3.222``|``-GCC-7.3.0-2.30``  |``system``
``2018.5.274``|``-GCC-7.3.0-2.30``  |``system``
``2019.0.117``|``-GCC-8.2.0-2.31.1``|``system``
``2019.1.144``|``-GCC-8.2.0-2.31.1``|``system``
``2019.2.187``|``-GCC-8.2.0-2.31.1``|``system``
``2019.3.199``|``-GCC-8.3.0-2.32``  |``system``
``system``    |``-GCC-system-2.29`` |``system``

<a anchor="iccifort"/>
### *iccifort*

Intel C, C++ & Fortran compilers

*homepage*: http://software.intel.com/en-us/intel-cluster-toolkit-compiler/

version       |versionsuffix        |toolchain
--------------|---------------------|----------
``2016.0.109``|                     |``system``
``2016.0.109``|``-GCC-4.9.3-2.25``  |``system``
``2016.1.150``|``-GCC-4.9.3-2.25``  |``system``
``2016.2.181``|``-GCC-4.9.3-2.25``  |``system``
``2016.2.181``|``-GCC-5.3.0-2.26``  |``system``
``2016.3.210``|``-GCC-4.9.3-2.25``  |``system``
``2016.3.210``|``-GCC-5.3.0-2.26``  |``system``
``2016.3.210``|``-GCC-5.4.0-2.26``  |``system``
``2017.0.098``|``-GCC-5.4.0-2.26``  |``system``
``2017.1.132``|``-GCC-5.4.0-2.26``  |``system``
``2017.1.132``|``-GCC-6.3.0-2.27``  |``system``
``2017.2.174``|``-GCC-6.3.0-2.27``  |``system``
``2017.4.196``|``-GCC-6.4.0-2.28``  |``system``
``2017.5.239``|``-GCC-6.4.0-2.28``  |``system``
``2018.0.128``|``-GCC-6.4.0-2.28``  |``system``
``2018.1.163``|``-GCC-6.4.0-2.28``  |``system``
``2018.2.199``|``-GCC-6.4.0-2.28``  |``system``
``2018.3.222``|``-GCC-7.3.0-2.30``  |``system``
``2018.5.274``|``-GCC-7.3.0-2.30``  |``system``
``2019.0.117``|``-GCC-8.2.0-2.31.1``|``system``
``2019.1.144``|``-GCC-8.2.0-2.31.1``|``system``
``2019.2.187``|``-GCC-8.2.0-2.31.1``|``system``
``2019.3.199``|``-GCC-8.3.0-2.32``  |``system``
``2019.4.243``|                     |``system``
``2019.5.281``|                     |``system``
``2020.0.166``|                     |``system``
``2020.0.166``|``-GCC-9.2.0``       |``system``
``2020.1.217``|                     |``system``
``2020.4.304``|                     |``system``
``system``    |``-GCC-system-2.29`` |``system``

<a anchor="iccifortcuda"/>
### *iccifortcuda*

Intel C, C++ & Fortran compilers with CUDA toolkit

*homepage*: (none)

version       |versionsuffix      |toolchain
--------------|-------------------|----------
``2016.10``   |                   |``system``
``2017.4.196``|``-GCC-6.4.0-2.28``|``system``
``2019a``     |                   |``system``
``2019b``     |                   |``system``
``2020a``     |                   |``system``
``2020b``     |                   |``system``

<a anchor="ichorcna"/>
### *ichorCNA*

ichorCNA is a tool for estimating the fraction of tumor in cell-free DNA from ultra-low-pass whole genome sequencing

*homepage*: https://github.com/broadinstitute/ichorCNA/wiki

version           |toolchain
------------------|--------------
``0.2.0``         |``foss/2019b``
``0.3.2-20191219``|``foss/2020a``

<a anchor="icmake"/>
### *icmake*

Icmake is a hybrid between a 'make' utility and a 'shell script' language. Originally, it was written  to provide a useful tool for automatic program maintenance and system administrative tasks on old MS-DOS platforms.

*homepage*: http://icmake.sourceforge.net/

version    |toolchain
-----------|--------------
``7.23.02``|``foss/2016a``

<a anchor="icount"/>
### *iCount*

iCount: protein-RNA interaction analysis  is a Python module and associated command-line interface (CLI),  which provides all the commands needed to process iCLIP data on protein-RNA interactions.

*homepage*: https://github.com/tomazc/iCount

version     |versionsuffix    |toolchain
------------|-----------------|--------------
``20180820``|``-Python-3.6.6``|``foss/2018b``

<a anchor="icu"/>
### *ICU*

ICU is a mature, widely used set of C/C++ and Java libraries providing Unicode and Globalization  support for software applications.

*homepage*: https://icu.unicode.org

version |toolchain
--------|--------------------------------------
``61.1``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``64.2``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``65.1``|``GCCcore/8.3.0``
``66.1``|``GCCcore/9.3.0``
``67.1``|``GCCcore/10.2.0``
``69.1``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``71.1``|``GCCcore/11.3.0``
``72.1``|``GCCcore/12.2.0``

<a anchor="idba-ud"/>
### *IDBA-UD*

IDBA-UD is a iterative De Bruijn Graph De Novo Assembler for Short Reads  Sequencing data with Highly Uneven Sequencing Depth. It is an extension of IDBA algorithm.  IDBA-UD also iterates from small k to a large k. In each iteration, short and low-depth  contigs are removed iteratively with cutoff threshold from low to high to reduce the errors  in low-depth and high-depth regions. Paired-end reads are aligned to contigs and assembled  locally to generate some missing k-mers in low-depth regions. With these technologies, IDBA-UD  can iterate k value of de Bruijn graph to a very large value with less gaps and less branches  to form long contigs in both low-depth and high-depth regions.

*homepage*: http://i.cs.hku.hk/~alse/hkubrg/projects/idba_ud/

version  |toolchain
---------|----------------------------------------------------------------------------------------------------------------------------------
``1.1.3``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/8.2.0-2.31.1``, ``GCC/8.3.0``, ``GCC/9.3.0``, ``foss/2018a``, ``foss/2018b``

<a anchor="idemux"/>
### *idemux*

idemux - inline barcode demultiplexing Idemux is a command line tool designed to demultiplex paired-end FASTQ files from QuantSeq-Pool.

*homepage*: https://github.com/Lexogen-Tools/idemux

version  |toolchain
---------|------------------
``0.1.6``|``GCCcore/10.2.0``

<a anchor="ieeg-cli"/>
### *ieeg-cli*

IEEG.ORG is a collaborative initiative funded by the National Institutes  of Neurological Disorders and Stroke. This initiative seeks to advance research towards  the understanding of epilepsy by providing a platform for sharing data,  tools and expertise between researchers.

*homepage*: https://www.ieeg.org/

version    |toolchain
-----------|----------
``1.14.56``|``system``

<a anchor="ifort"/>
### *ifort*

Fortran compiler from Intel

*homepage*: http://software.intel.com/en-us/intel-compilers/

version       |versionsuffix        |toolchain
--------------|---------------------|----------
``2016.0.109``|                     |``system``
``2016.0.109``|``-GCC-4.9.3-2.25``  |``system``
``2016.1.150``|``-GCC-4.9.3-2.25``  |``system``
``2016.2.181``|``-GCC-4.9.3-2.25``  |``system``
``2016.2.181``|``-GCC-5.3.0-2.26``  |``system``
``2016.3.210``|``-GCC-4.9.3-2.25``  |``system``
``2016.3.210``|``-GCC-5.3.0-2.26``  |``system``
``2016.3.210``|``-GCC-5.4.0-2.26``  |``system``
``2017.0.098``|``-GCC-5.4.0-2.26``  |``system``
``2017.1.132``|``-GCC-5.4.0-2.26``  |``system``
``2017.1.132``|``-GCC-6.3.0-2.27``  |``system``
``2017.2.174``|``-GCC-6.3.0-2.27``  |``system``
``2017.4.196``|``-GCC-6.4.0-2.28``  |``system``
``2017.5.239``|``-GCC-6.4.0-2.28``  |``system``
``2017.6.256``|``-GCC-6.4.0-2.28``  |``system``
``2017.7.259``|``-GCC-6.4.0-2.28``  |``system``
``2018.0.128``|``-GCC-6.4.0-2.28``  |``system``
``2018.1.163``|``-GCC-6.4.0-2.28``  |``system``
``2018.2.199``|``-GCC-6.4.0-2.28``  |``system``
``2018.3.222``|``-GCC-7.3.0-2.30``  |``system``
``2018.5.274``|``-GCC-7.3.0-2.30``  |``system``
``2019.0.117``|``-GCC-8.2.0-2.31.1``|``system``
``2019.1.144``|``-GCC-8.2.0-2.31.1``|``system``
``2019.2.187``|``-GCC-8.2.0-2.31.1``|``system``
``2019.3.199``|``-GCC-8.3.0-2.32``  |``system``
``system``    |``-GCC-system-2.29`` |``system``

<a anchor="igblast"/>
### *IgBLAST*

IgBLAST faclilitates the analysis of immunoglobulin and T cell receptor variable domain sequences.

*homepage*: https://ncbi.github.io/igblast

version   |versionsuffix |toolchain
----------|--------------|----------
``1.15.0``|``-x64-linux``|``system``
``1.18.0``|``-x64-linux``|``system``

<a anchor="igmplot"/>
### *IGMPlot*

IGMPlot is a free open-source program developed to identify molecular interactions and  prepare data to build 2D and 3D representations of them in the molecular environment.

*homepage*: http://igmplot.univ-reims.fr

version   |toolchain
----------|--------------------------------------
``2.4.2`` |``GCC/8.3.0``, ``iccifort/2019.5.281``
``2.6.9b``|``GCC/11.2.0``

<a anchor="igraph"/>
### *igraph*

igraph is a collection of network analysis tools with the emphasis on efficiency, portability and ease of use. igraph is open source and free. igraph can be programmed in R, Python and C/C++.

*homepage*: https://igraph.org

version  |toolchain
---------|------------------------------------------------
``0.7.1``|``foss/2018b``, ``intel/2016b``, ``intel/2017b``
``0.8.0``|``foss/2019b``
``0.8.2``|``foss/2020a``
``0.8.5``|``foss/2020b``
``0.9.1``|``foss/2020b``, ``fosscuda/2020b``
``0.9.4``|``foss/2021a``
``0.9.5``|``foss/2021b``

<a anchor="igv"/>
### *IGV*

This package contains command line utilities for  preprocessing, computing feature count density (coverage),  sorting, and  indexing data files.

*homepage*: https://www.broadinstitute.org/software/igv/

version   |versionsuffix     |toolchain
----------|------------------|----------
``2.3.68``|``-Java-1.7.0_80``|``system``
``2.3.80``|``-Java-1.7.0_80``|``system``
``2.5.0`` |``-Java-11``      |``system``
``2.8.0`` |``-Java-11``      |``system``
``2.9.4`` |``-Java-11``      |``system``
``2.12.3``|``-Java-11``      |``system``

<a anchor="igv-reports"/>
### *igv-reports*

Python application to generate self-contained igv.js pages that can be opened within a browser with "file" protocol.

*homepage*: https://github.com/igvteam/igv-reports

version  |versionsuffix    |toolchain
---------|-----------------|-------------
``0.9.8``|``-Python-3.7.4``|``GCC/8.3.0``

<a anchor="igvtools"/>
### *IGVTools*

This package contains command line utilities for preprocessing,   computing feature count density (coverage),  sorting, and indexing data files.  See also http://www.broadinstitute.org/software/igv/igvtools_commandline. 

*homepage*: https://software.broadinstitute.org/software/igv/igvtools

version   |versionsuffix     |toolchain
----------|------------------|----------
``2.3.68``|``-Java-1.7.0_80``|``system``
``2.3.72``|``-Java-1.7.0_80``|``system``
``2.3.75``|``-Java-1.7.0_80``|``system``
``2.4.18``|``-Java-1.8``     |``system``

<a anchor="iibff"/>
### *iibff*

GCC and GFortran based compiler toolchain with OpenMPI, BLIS, libFLAME, ScaLAPACK and FFTW.

*homepage*: (none)

version  |toolchain
---------|----------
``2020b``|``system``

<a anchor="iimkl"/>
### *iimkl*

Intel C/C++ and Fortran compilers, alongside Intel Math Kernel Library (MKL).

*homepage*: https://software.intel.com/en-us/intel-cluster-toolkit-compiler/

version  |toolchain
---------|----------
``2018a``|``system``
``2022b``|``system``

<a anchor="iimpi"/>
### *iimpi*

Intel C/C++ and Fortran compilers, alongside Intel MPI.

*homepage*: http://software.intel.com/en-us/intel-cluster-toolkit-compiler/

version              |versionsuffix       |toolchain
---------------------|--------------------|----------
``8.1.5``            |``-GCC-4.9.3-2.25`` |``system``
``2016.00``          |``-GCC-4.9.3-2.25`` |``system``
``2016.01``          |``-GCC-4.9.3-2.25`` |``system``
``2016.02``          |``-GCC-4.9.3-2.25`` |``system``
``2016.02``          |``-GCC-5.3.0-2.26`` |``system``
``2016.03``          |``-GCC-4.9.3-2.25`` |``system``
``2016.03``          |``-GCC-5.3.0-2.26`` |``system``
``2016.03``          |``-GCC-5.4.0-2.26`` |``system``
``2016b``            |                    |``system``
``2017.00``          |``-GCC-5.4.0-2.26`` |``system``
``2017.01``          |``-GCC-5.4.0-2.26`` |``system``
``2017.02``          |``-GCC-6.3.0-2.27`` |``system``
``2017.09``          |                    |``system``
``2017a``            |                    |``system``
``2017b``            |                    |``system``
``2018.00``          |                    |``system``
``2018.01``          |                    |``system``
``2018.02``          |                    |``system``
``2018.04``          |                    |``system``
``2018a``            |                    |``system``
``2018b``            |                    |``system``
``2019.00``          |                    |``system``
``2019.01``          |                    |``system``
``2019.02``          |                    |``system``
``2019.03``          |                    |``system``
``2019a``            |                    |``system``
``2019b``            |                    |``system``
``2020.00``          |                    |``system``
``2020.06-impi-18.5``|                    |``system``
``2020.12``          |                    |``system``
``2020a``            |                    |``system``
``2020b``            |                    |``system``
``2021a``            |                    |``system``
``2021b``            |                    |``system``
``2022.00``          |                    |``system``
``2022.05``          |                    |``system``
``2022.09``          |                    |``system``
``2022.11``          |                    |``system``
``2022a``            |                    |``system``
``2022b``            |                    |``system``
``system``           |``-GCC-system-2.29``|``system``

<a anchor="iimpic"/>
### *iimpic*

Intel C/C++ and Fortran compilers, alongside Intel MPI and CUDA.

*homepage*: (none)

version    |toolchain
-----------|----------
``2016.10``|``system``
``2017b``  |``system``
``2019a``  |``system``
``2019b``  |``system``
``2020a``  |``system``
``2020b``  |``system``

<a anchor="ijulia"/>
### *IJulia*

Julia kernel for Jupyter

*homepage*: https://github.com/JuliaLang/IJulia.jl

version   |versionsuffix   |toolchain
----------|----------------|----------
``1.23.3``|``-Julia-1.6.7``|``system``

<a anchor="ilamb"/>
### *ILAMB*

The International Land Model Benchmarking (ILAMB) project is a model-data intercomparison and integration project designed to improve the performance of land models and, in parallel, improve the design of new measurement campaigns to reduce uncertainties associated with key land surface processes. 

*homepage*: https://www.ilamb.org/

version|versionsuffix    |toolchain
-------|-----------------|--------------
``2.5``|``-Python-3.8.2``|``foss/2020a``

<a anchor="ima2"/>
### *IMa2*

IMa2 is a progam for population genetic analysis that can handle two or more populations.

*homepage*: https://bio.cst.temple.edu/~hey/software/software.htm#IMa2

version    |toolchain
-----------|--------------
``8.27.12``|``foss/2016a``

<a anchor="ima2p"/>
### *IMa2p*

IMa2p is a parallel implementation of IMa2, using OpenMPI-C++ - a Bayesian MCMC  based method for inferring population demography under the IM (Isolation with  Migration) model. http://dx.doi.org/10.1111/1755-0998.12437 

*homepage*: https://github.com/arunsethuraman/ima2p

version     |toolchain
------------|---------------
``20151123``|``foss/2016a``
``20160804``|``intel/2016b``

<a anchor="imagecodecs"/>
### *imagecodecs*

Imagecodecs is a Python library that provides block-oriented, in-memory buffer transformation, compression, and decompression functions for use in the tifffile, czifile, zarr, and other scientific image input/output modules.

*homepage*: https://github.com/cgohlke/imagecodecs

version      |toolchain
-------------|--------------
``2021.8.26``|``foss/2020b``
``2022.9.26``|``foss/2021a``

<a anchor="imageio"/>
### *imageio*

Imageio is a Python library that provides an easy interface to read and write a wide range of  image data, including animated images, video, volumetric data, and scientific formats.

*homepage*: https://imageio.github.io

version   |versionsuffix    |toolchain
----------|-----------------|----------------------------------
``2.3.0`` |``-Python-3.6.4``|``intel/2018a``
``2.5.0`` |                 |``foss/2019a``
``2.9.0`` |                 |``foss/2020b``, ``fosscuda/2020b``
``2.9.0`` |``-Python-3.7.4``|``foss/2019b``, ``intel/2019b``
``2.10.5``|                 |``foss/2021a``
``2.13.5``|                 |``foss/2021b``
``2.22.2``|                 |``foss/2022a``

<a anchor="imagej"/>
### *ImageJ*

Image Processing and Analysis in Java

*homepage*: https://imagej.nih.gov/ij

version  |versionsuffix|toolchain
---------|-------------|----------
``1.51a``|             |``system``
``1.51i``|             |``system``
``1.51k``|             |``system``
``1.52q``|``-Java-1.8``|``system``

<a anchor="imagemagick"/>
### *ImageMagick*

ImageMagick is a software suite to create, edit, compose, or convert bitmap images

*homepage*: https://www.imagemagick.org/

version      |versionsuffix                      |toolchain
-------------|-----------------------------------|-------------------------------
``6.9.4-8``  |                                   |``intel/2016a``
``7.0.1-6``  |                                   |``intel/2016a``
``7.0.1-9``  |                                   |``intel/2016a``
``7.0.2-9``  |                                   |``intel/2016a``
``7.0.3-1``  |                                   |``intel/2016b``
``7.0.5-4``  |                                   |``intel/2017a``
``7.0.5-10`` |                                   |``foss/2016b``
``7.0.7-8``  |``-JasPer-1.900.1``                |``intel/2017a``
``7.0.7-14`` |                                   |``foss/2017b``, ``intel/2017b``
``7.0.7-15`` |                                   |``GCCcore/6.4.0``
``7.0.7-26`` |                                   |``foss/2018a``
``7.0.7-30`` |                                   |``GCCcore/6.4.0``
``7.0.7-30`` |``-Ghostscript-9.22-cairo-1.14.12``|``GCCcore/6.4.0``
``7.0.7-39`` |``-Ghostscript-9.23-cairo-1.14.12``|``GCCcore/6.4.0``
``7.0.8-11`` |                                   |``GCCcore/7.3.0``
``7.0.8-46`` |                                   |``GCCcore/8.2.0``
``7.0.9-5``  |                                   |``GCCcore/8.3.0``
``7.0.10-1`` |                                   |``GCCcore/9.3.0``
``7.0.10-35``|                                   |``GCCcore/10.2.0``
``7.0.11-14``|                                   |``GCCcore/10.3.0``
``7.1.0-4``  |                                   |``GCCcore/11.2.0``
``7.1.0-37`` |                                   |``GCCcore/11.3.0``
``7.1.0-53`` |                                   |``GCCcore/12.2.0``

<a anchor="imake"/>
### *imake*

imake is a Makefile-generator that is intended to make it easier to develop software portably for multiple systems.

*homepage*: https://www.x.org/

version  |toolchain
---------|----------------------------------------------------------
``1.0.7``|``intel/2016a``
``1.0.8``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``

<a anchor="imath"/>
### *Imath*

Imath is a C++ and python library of 2D and 3D vector, matrix, and math operations for computer graphics 

*homepage*: https://imath.readthedocs.io/en/latest/

version  |toolchain
---------|------------------
``3.1.5``|``GCCcore/11.3.0``

<a anchor="imb"/>
### *IMB*

The Intel MPI Benchmarks perform a set of MPI performance measurements for point-to-point and  global communication operations for a range of message sizes

*homepage*: https://software.intel.com/en-us/articles/intel-mpi-benchmarks

version   |toolchain
----------|------------------------------------------------------------------
``4.1``   |``foss/2016a``, ``foss/2017a``, ``intel/2017.02``, ``intel/2017a``
``2018.1``|``intel/2017a``
``2019.3``|``gompi/2019a``, ``iimpi/2019a``
``2021.3``|``gompi/2021b``

<a anchor="imbalanced-learn"/>
### *imbalanced-learn*

imbalanced-learn is a Python package offering a number of re-sampling techniques commonly used in  datasets showing strong between-class imbalance.

*homepage*: https://github.com/scikit-learn-contrib/imbalanced-learn

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.2.1``|``-Python-2.7.12``|``intel/2016b``
``0.2.1``|``-Python-3.5.2`` |``intel/2016b``
``0.3.3``|``-Python-3.6.4`` |``foss/2018a``
``0.4.3``|``-Python-3.6.6`` |``foss/2018b``
``0.7.0``|                  |``foss/2020b``
``0.9.0``|                  |``foss/2021b``

<a anchor="imgaug"/>
### *imgaug*

This python library helps you with augmenting images for your machine learning projects.  It converts a set of input images into a new, much larger set of slightly altered images. 

*homepage*: https://imgaug.readthedocs.io/en/latest/

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------------------
``0.2.8``|``-Python-3.6.6``|``foss/2018b``
``0.4.0``|                 |``foss/2021a``, ``foss/2021b``, ``foss/2022a``
``0.4.0``|``-CUDA-11.3.1`` |``foss/2021a``
``0.4.0``|``-CUDA-11.4.1`` |``foss/2021b``
``0.4.0``|``-CUDA-11.7.0`` |``foss/2022a``
``0.4.0``|``-Python-3.7.4``|``foss/2019b``

<a anchor="imkl"/>
### *imkl*

Intel oneAPI Math Kernel Library

*homepage*: https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onemkl.html

version       |versionsuffix|toolchain
--------------|-------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``11.2.3.187``|             |``gimpi/2.11.5``
``11.3.0.109``|             |``iimpi/2016.00-GCC-4.9.3-2.25``
``11.3.1.150``|             |``iimpi/2016.01-GCC-4.9.3-2.25``, ``iimpi/8.1.5-GCC-4.9.3-2.25``
``11.3.2.181``|             |``iimpi/2016.02-GCC-4.9.3-2.25``, ``iimpi/2016.02-GCC-5.3.0-2.26``, ``pompi/2016.03``
``11.3.3.210``|             |``iimpi/2016.03-GCC-4.9.3-2.25``, ``iimpi/2016.03-GCC-5.3.0-2.26``, ``iimpi/2016.03-GCC-5.4.0-2.26``, ``iimpi/2016b``, ``iimpic/2016.10``, ``iompi/2016.07``, ``iompi/2016.09-GCC-4.9.3-2.25``, ``iompi/2016.09-GCC-5.4.0-2.26``, ``pompi/2016.04``, ``pompi/2016.09``
``2017.0.098``|             |``iimpi/2017.00-GCC-5.4.0-2.26``
``2017.1.132``|             |``gimpi/2017a``, ``iimpi/2017.01-GCC-5.4.0-2.26``, ``iimpi/2017a``, ``iompi/2017.01``, ``iompi/2017a``
``2017.2.174``|             |``iimpi/2017.02-GCC-6.3.0-2.27``
``2017.3.196``|             |``gompi/2017b``, ``iimpi/2017b``, ``iimpic/2017b``, ``iompi/2017b``
``2017.4.239``|             |``iimpi/2017.09``
``2018.0.128``|             |``iimpi/2018.00``
``2018.1.163``|             |``iimpi/2018.01``, ``iimpi/2018a``, ``iompi/2018a``
``2018.1.163``|``-serial``  |``iccifort/2018.1.163-GCC-6.4.0-2.28``
``2018.2.199``|             |``iimpi/2018.02``, ``iompi/2018.02``
``2018.3.222``|             |``gimpi/2018b``, ``gompi/2018b``, ``iimpi/2018b``, ``iompi/2018b``
``2018.4.274``|             |``iimpi/2018.04``
``2019.0.117``|             |``iimpi/2019.00``
``2019.1.144``|             |``gompi/2019a``, ``iimpi/2019.01``, ``iimpi/2019a``, ``iimpic/2019a``, ``iompi/2019.01``
``2019.2.187``|             |``iimpi/2019.02``
``2019.3.199``|             |``iimpi/2019.03``
``2019.5.281``|             |``gompi/2019b``, ``gompic/2019b``, ``iimpi/2019b``, ``iimpic/2019b``, ``iompi/2019b``
``2020.0.166``|             |``iimpi/2020.00``
``2020.1.217``|             |``gompi/2020a``, ``iimpi/2020.06-impi-18.5``, ``iimpi/2020a``, ``iimpic/2020a``, ``iompi/2020a``
``2020.4.304``|             |``NVHPC/21.2``, ``gompi/2020b``, ``gompic/2020b``, ``iimpi/2020b``, ``iimpic/2020b``, ``iompi/2020b``
``2021.1.1``  |             |``iimpi/2020.12``
``2021.2.0``  |             |``gompi/2021a``, ``iimpi/2021a``, ``iompi/2021a``
``2021.3.0``  |             |``gompi/2021a``
``2021.4.0``  |             |``system``
``2022.0.1``  |             |``system``
``2022.1.0``  |             |``system``
``2022.2.0``  |             |``system``
``2022.2.1``  |             |``system``

<a anchor="imkl-fftw"/>
### *imkl-FFTW*

FFTW interfaces using Intel oneAPI Math Kernel Library

*homepage*: https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onemkl.html

version     |toolchain
------------|----------------------------------
``2021.4.0``|``gompi/2021b``, ``iimpi/2021b``
``2022.0.1``|``iimpi/2022.00``
``2022.1.0``|``iimpi/2022.05``, ``iimpi/2022a``
``2022.2.0``|``iimpi/2022.09``
``2022.2.1``|``iimpi/2022.11``, ``iimpi/2022b``

<a anchor="imlib2"/>
### *Imlib2*

This is the Imlib 2 library - a library that does image file loading and saving as well as rendering, manipulation, arbitrary polygon support, etc. It does ALL of these operations FAST. Imlib2 also tries to be highly intelligent about doing them, so writing naive programs can be done easily, without sacrificing speed. 

*homepage*: https://docs.enlightenment.org/api/imlib2/html/

version  |toolchain
---------|-----------------
``1.5.1``|``GCCcore/6.4.0``

<a anchor="immunedeconv"/>
### *immunedeconv*

immunedeconv is an R package for unified access to computational methods for estimating immune cell fractions from bulk RNA sequencing data.

*homepage*: https://github.com/icbi-lab/immunedeconv

version  |versionsuffix|toolchain
---------|-------------|--------------
``2.0.2``|``-R-4.0.0`` |``foss/2020a``

<a anchor="imod"/>
### *IMOD*

IMOD is a set of image processing, modeling and display programs used for tomographic reconstruction and for 3D reconstruction of EM serial sections and optical sections. The package contains tools for assembling and aligning data within multiple types and sizes of image stacks, viewing 3-D data from any orientation, and modeling and display of the image files. IMOD was developed primarily by David Mastronarde, Rick Gaudette, Sue Held, Jim Kremer, Quanren Xiong, and John Heumann at the University of Colorado.

*homepage*: http://bio3d.colorado.edu/imod/

version   |versionsuffix        |toolchain
----------|---------------------|----------------------------------
``4.7.15``|``_RHEL6-64_CUDA6.0``|``system``
``4.11.5``|                     |``foss/2020b``, ``fosscuda/2020b``

<a anchor="impi"/>
### *impi*

The Intel(R) MPI Library for Linux* OS is a multi-fabric message  passing library based on ANL MPICH2 and OSU MVAPICH2. The Intel MPI Library for  Linux OS implements the Message Passing Interface, version 2 (MPI-2) specification.

*homepage*: http://software.intel.com/en-us/intel-mpi-library/

version        |versionsuffix|toolchain
---------------|-------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``3.2.2.006``  |             |``system``
``4.0.0.028``  |             |``system``
``4.0.0.028``  |``-32bit``   |``system``
``4.0.2.003``  |             |``system``
``4.1.0.027``  |             |``system``
``4.1.0.030``  |             |``system``
``4.1.1.036``  |             |``system``
``4.1.2.040``  |             |``system``
``4.1.3.045``  |             |``system``
``4.1.3.049``  |             |``GCC/4.8.3``, ``system``
``5.0.3.048``  |             |``GCC/4.9.3``
``5.1.1.109``  |             |``iccifort/2016.0.109-GCC-4.9.3-2.25``
``5.1.2.150``  |             |``iccifort/2016.1.150-GCC-4.9.3-2.25``
``5.1.3.181``  |             |``iccifort/2016.2.181-GCC-4.9.3-2.25``, ``iccifort/2016.2.181-GCC-5.3.0-2.26``, ``iccifort/2016.3.210-GCC-4.9.3-2.25``, ``iccifort/2016.3.210-GCC-5.3.0-2.26``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``, ``iccifortcuda/2016.10``
``2017.0.098`` |             |``iccifort/2017.0.098-GCC-5.4.0-2.26``
``2017.1.132`` |             |``GCC/5.4.0-2.26``, ``iccifort/2017.1.132-GCC-5.4.0-2.26``, ``iccifort/2017.1.132-GCC-6.3.0-2.27``
``2017.2.174`` |             |``iccifort/2017.2.174-GCC-6.3.0-2.27``
``2017.3.196`` |             |``GCC/6.4.0-2.28``, ``gcccuda/2017b``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``, ``iccifortcuda/2017.4.196-GCC-6.4.0-2.28``
``2017.4.239`` |             |``iccifort/2017.5.239-GCC-6.4.0-2.28``
``2018.0.128`` |             |``iccifort/2018.0.128-GCC-6.4.0-2.28``
``2018.1.163`` |             |``GCC/6.4.0-2.28``, ``iccifort/2018.1.163-GCC-6.4.0-2.28``
``2018.2.199`` |             |``iccifort/2018.2.199-GCC-6.4.0-2.28``
``2018.3.222`` |             |``GCC/7.3.0-2.30``, ``iccifort/2018.3.222-GCC-7.3.0-2.30``
``2018.4.274`` |             |``iccifort/2018.5.274-GCC-7.3.0-2.30``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``iccifortcuda/2019a``
``2018.5.288`` |             |``iccifort/2019.5.281``, ``iccifort/2020.1.217``, ``iccifortcuda/2019b``
``2019.0.117`` |             |``iccifort/2019.0.117-GCC-8.2.0-2.31.1``
``2019.1.144`` |             |``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``2019.2.187`` |             |``iccifort/2019.2.187-GCC-8.2.0-2.31.1``
``2019.3.199`` |             |``iccifort/2019.3.199-GCC-8.3.0-2.32``
``2019.6.166`` |             |``iccifort/2020.0.166-GCC-9.2.0``
``2019.7.217`` |             |``iccifort/2020.1.217``, ``iccifortcuda/2020a``
``2019.9.304`` |             |``iccifort/2020.4.304``, ``iccifortcuda/2020b``
``2019.12.320``|             |``iccifort/2020.4.304``
``2021.1.1``   |             |``intel-compilers/2021.1.2``
``2021.2.0``   |             |``intel-compilers/2021.2.0``
``2021.3.0``   |             |``intel-compilers/2021.3.0``
``2021.4.0``   |             |``intel-compilers/2021.4.0``
``2021.5.0``   |             |``intel-compilers/2022.0.1``
``2021.6.0``   |             |``intel-compilers/2022.1.0``
``2021.7.0``   |             |``intel-compilers/2022.2.0``
``2021.7.1``   |             |``intel-compilers/2022.2.1``
``system``     |             |``iccifort/system-GCC-system-2.29``

<a anchor="impute2"/>
### *IMPUTE2*

IMPUTE version 2 (also known as IMPUTE2) is a genotype imputation   and haplotype phasing program based on ideas from Howie et al. 2009 

*homepage*: http://mathgen.stats.ox.ac.uk/impute/impute_v2.html

version  |versionsuffix      |toolchain
---------|-------------------|----------
``2.3.0``|``_x86_64_dynamic``|``system``
``2.3.0``|``_x86_64_static`` |``system``
``2.3.2``|``_x86_64_dynamic``|``system``
``2.3.2``|``_x86_64_static`` |``system``

<a anchor="inchi"/>
### *InChI*

The IUPAC International Chemical Identifier (InChI TM) is a non-proprietary identifier for chemical substances that can be used in printed and electronic data sources thus enabling easier linking of diverse data compilations.

*homepage*: https://www.inchi-trust.org/

version |toolchain
--------|--------------
``1.06``|``GCC/10.3.0``

<a anchor="indicators"/>
### *indicators*

- Thread-safe progress bars and spinners - Header-only library. Grab a copy of include/indicators. - Single-header version in single_include/indicators. 

*homepage*: https://github.com/p-ranav/indicators

version|toolchain
-------|------------------
``2.2``|``GCCcore/11.3.0``

<a anchor="inelastica"/>
### *Inelastica*

Python package for eigenchannels, vibrations and inelastic electron transport based on SIESTA/TranSIESTA DFT.

*homepage*: https://github.com/tfrederiksen/inelastica

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.3.5``|``-Python-2.7.15``|``intel/2018b``

<a anchor="infercnv"/>
### *inferCNV*

InferCNV is used to explore tumor single cell RNA-Seq data to identify evidence  for somatic large-scale chromosomal copy number alterations, such as gains or  deletions of entire chromosomes or large segments of chromosomes.

*homepage*: https://github.com/broadinstitute/inferCNV/wiki

version   |versionsuffix            |toolchain
----------|-------------------------|------------------------------
``1.0.4`` |``-Python-3.7.2-R-3.6.0``|``foss/2019a``
``1.2.1`` |``-Python-3.7.4``        |``foss/2019b``
``1.3.3`` |                         |``foss/2020b``, ``foss/2021a``
``1.3.3`` |``-Python-3.8.2``        |``foss/2020a``
``1.10.1``|``-R-4.1.2``             |``foss/2021b``

<a anchor="infercnvpy"/>
### *infercnvpy*

Infer copy number variation (CNV) from scRNA-seq data. Plays nicely with Scanpy.

*homepage*: https://github.com/icbi-lab/infercnvpy

version  |toolchain
---------|--------------
``0.4.0``|``foss/2021b``

<a anchor="infernal"/>
### *Infernal*

Infernal ("INFERence of RNA ALignment") is for searching DNA sequence databases  for RNA structure and sequence similarities.

*homepage*: http://eddylab.org/infernal/

version  |toolchain
---------|----------------------------------------------------------------
``1.1.2``|``foss/2016b``, ``foss/2018b``, ``intel/2017a``, ``intel/2018b``
``1.1.4``|``foss/2020b``, ``foss/2021a``, ``foss/2021b``

<a anchor="infomap"/>
### *Infomap*

Multi-level network clustering based on the Map equation.

*homepage*: https://www.mapequation.org/code.html#Linux

version     |toolchain
------------|--------------------
``20190308``|``GCC/8.2.0-2.31.1``

<a anchor="inline"/>
### *inline*

Functionality to dynamically define R functions and S4 methods  with 'inlined' C, C++ or Fortran code supporting the .C and .Call calling  conventions.

*homepage*: https://cran.r-project.org/web/packages/inline

version   |versionsuffix|toolchain
----------|-------------|--------------
``0.3.19``|``-R-4.0.4`` |``foss/2020b``

<a anchor="inparanoid"/>
### *InParanoid*

InParanoid: ortholog groups with inparalogs.

*homepage*: https://inparanoid.sbc.su.se

version         |toolchain
----------------|--------------
``5.0-20220607``|``GCC/10.3.0``

<a anchor="inputproto"/>
### *inputproto*

X.org InputProto protocol headers.

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version  |toolchain
---------|-------------------------------------------------
``2.3.1``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``2.3.2``|``intel/2016a``

<a anchor="inspector"/>
### *Inspector*

Intel Inspector is a dynamic memory and threading error  checking tool for users developing serial and parallel applications

*homepage*: https://software.intel.com/en-us/inspector

version         |toolchain
----------------|----------
``2013_update6``|``system``
``2013_update7``|``system``
``2016_update3``|``system``
``2017_update1``|``system``
``2017_update2``|``system``
``2018_update1``|``system``
``2018_update2``|``system``
``2018_update3``|``system``
``2019_update2``|``system``
``2019_update5``|``system``
``2021.4.0``    |``system``
``2022.0.0``    |``system``
``2022.1.0``    |``system``

<a anchor="intarna"/>
### *IntaRNA*

Efficient RNA-RNA interaction prediction incorporating accessibility and seeding of interaction sites

*homepage*: https://github.com/BackofenLab/IntaRNA

version  |versionsuffix     |toolchain
---------|------------------|--------------
``2.3.1``|``-Python-2.7.15``|``foss/2018b``

<a anchor="integrate"/>
### *INTEGRATE*

INTEGRATE is a tool calling gene fusions with exact fusion junctions and genomic breakpoints  by combining RNA-Seq and WGS data. It is highly sensitive and accurate by applying a fast split-read  mapping algorithm based on Burrow-Wheeler transform. 

*homepage*: https://sourceforge.net/p/integrate-fusion/wiki/Home/

version  |toolchain
---------|-----------------
``0.2.6``|``GCCcore/8.2.0``

<a anchor="integrate-neo"/>
### *INTEGRATE-Neo*

INTEGRATE-Neo is a gene fusion neoantigen discovering tool using next-generation sequencing data.  It is written in C++ and Python. 

*homepage*: 

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.2.1``|``-Python-3.6.6``|``foss/2018b``

<a anchor="intel"/>
### *intel*

Compiler toolchain including Intel compilers, Intel MPI and Intel Math Kernel Library (MKL).

*homepage*: https://easybuild.readthedocs.io/en/master/Common-toolchains.html#intel-toolchain

version              |versionsuffix|toolchain
---------------------|-------------|----------
``2016.00``          |             |``system``
``2016.01``          |             |``system``
``2016.02``          |``-GCC-4.9`` |``system``
``2016.02``          |``-GCC-5.3`` |``system``
``2016.03``          |``-GCC-4.9`` |``system``
``2016.03``          |``-GCC-5.3`` |``system``
``2016.03``          |``-GCC-5.4`` |``system``
``2016a``            |             |``system``
``2016b``            |             |``system``
``2017.00``          |             |``system``
``2017.01``          |             |``system``
``2017.02``          |             |``system``
``2017.09``          |             |``system``
``2017a``            |             |``system``
``2017b``            |             |``system``
``2018.00``          |             |``system``
``2018.01``          |             |``system``
``2018.02``          |             |``system``
``2018.04``          |             |``system``
``2018a``            |             |``system``
``2018b``            |             |``system``
``2019.00``          |             |``system``
``2019.01``          |             |``system``
``2019.02``          |             |``system``
``2019.03``          |             |``system``
``2019a``            |             |``system``
``2019b``            |             |``system``
``2020.00``          |             |``system``
``2020.06-impi-18.5``|             |``system``
``2020.12``          |             |``system``
``2020a``            |             |``system``
``2020b``            |             |``system``
``2021a``            |             |``system``
``2021b``            |             |``system``
``2022.00``          |             |``system``
``2022.05``          |             |``system``
``2022.09``          |             |``system``
``2022.11``          |             |``system``
``2022a``            |             |``system``
``2022b``            |             |``system``

<a anchor="intel-compilers"/>
### *intel-compilers*

Intel C, C++ & Fortran compilers (classic and oneAPI)

*homepage*: https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit.html

version     |toolchain
------------|----------
``2021.1.2``|``system``
``2021.2.0``|``system``
``2021.3.0``|``system``
``2021.4.0``|``system``
``2022.0.1``|``system``
``2022.0.2``|``system``
``2022.1.0``|``system``
``2022.2.0``|``system``
``2022.2.1``|``system``

<a anchor="intelclusterchecker"/>
### *IntelClusterChecker*

Intel Cluster Checker verifies the configuration and  performance of Linux OS-based clusters. Anomalies and performance differences  can be identified and practical resolutions provided. 

*homepage*: https://software.intel.com/en-us/cluster-checker

version       |toolchain
--------------|----------
``2017.1.016``|``system``
``2021.5.0``  |``system``

<a anchor="intelcuda"/>
### *intelcuda*

Intel Cluster Toolkit Compiler Edition provides Intel C/C++ and Fortran compilers,  Intel MPI & Intel MKL, with CUDA toolkit

*homepage*: (none)

version    |toolchain
-----------|----------
``2016.10``|``system``
``2017b``  |``system``
``2019a``  |``system``
``2019b``  |``system``
``2020a``  |``system``
``2020b``  |``system``

<a anchor="inteldaal"/>
### *IntelDAAL*

Intel® Data Analytics Acceleration Library (Intel® DAAL) is the library of  Intel® architecture optimized building blocks covering all stages of data analytics: data acquisition from a data source, preprocessing, transformation, data mining, modeling, validation, and decision making.  

*homepage*: https://software.intel.com/en-us/daal

version       |toolchain
--------------|----------
``2019.4.007``|``system``

<a anchor="intelpython"/>
### *IntelPython*

Intel® Distribution for Python. Powered by Anaconda.  Accelerating Python* performance on modern architectures from Intel. 

*homepage*: https://software.intel.com/en-us/intel-distribution-for-python

version   |versionsuffix  |toolchain
----------|---------------|----------
``2.7.15``|``-2019.2.066``|``system``
``3.6.8`` |``-2019.2.066``|``system``

<a anchor="interproscan"/>
### *InterProScan*

InterProScan is a sequence analysis application (nucleotide and protein sequences) that combines  different protein signature recognition methods into one resource [code only: libraries and external binaries  but no data].

*homepage*: https://www.ebi.ac.uk/interpro/

version      |toolchain
-------------|------------------
``5.26-65.0``|``intel/2017b``
``5.27-66.0``|``intel/2017b``
``5.28-67.0``|``intel/2018a``
``5.52-86.0``|``GCCcore/10.3.0``
``5.55-88.0``|``foss/2021a``

<a anchor="interproscan_data"/>
### *InterProScan_data*

InterProScan is a sequence analysis application (nucleotide and protein sequences) that combines  different protein signature recognition methods into one resource [data only].

*homepage*: https://www.ebi.ac.uk/interpro/

version      |toolchain
-------------|--------------
``5.55-88.0``|``foss/2021a``

<a anchor="intervaltree"/>
### *intervaltree*

An interval tree can be used to efficiently find a set of numeric intervals  overlapping or containing another interval. This library provides a basic implementation of an  interval tree using C++ templates, allowing the insertion of arbitrary types into the tree. 

*homepage*: https://github.com/ekg/intervaltree

version|toolchain
-------|-----------------------------------------------------------------------------
``0.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``

<a anchor="intervaltree-python"/>
### *intervaltree-python*

A mutable, self-balancing interval tree. Queries may be by  point, by range overlap, or by range containment. 

*homepage*: https://github.com/chaimleib/intervaltree

version  |versionsuffix    |toolchain
---------|-----------------|--------------------------------------
``3.0.2``|``-Python-3.6.6``|``foss/2018b``
``3.1.0``|                 |``GCCcore/10.3.0``, ``GCCcore/11.2.0``

<a anchor="intltool"/>
### *intltool*

intltool is a set of tools to centralize translation of  many different file formats using GNU gettext-compatible PO files.

*homepage*: http://freedesktop.org/wiki/Software/intltool/

version   |versionsuffix   |toolchain
----------|----------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------
``0.51.0``|                |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``0.51.0``|``-Perl-5.20.3``|``intel/2016a``
``0.51.0``|``-Perl-5.22.1``|``foss/2016a``, ``intel/2016a``
``0.51.0``|``-Perl-5.24.0``|``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``foss/2016b``, ``gimkl/2017a``, ``intel/2016b``
``0.51.0``|``-Perl-5.24.1``|``GCCcore/6.3.0``, ``intel/2017a``
``0.51.0``|``-Perl-5.26.0``|``GCCcore/6.4.0``
``0.51.0``|``-Perl-5.26.1``|``GCCcore/6.4.0``
``0.51.0``|``-Perl-5.28.0``|``GCCcore/7.3.0``

<a anchor="io_lib"/>
### *io_lib*

Io_lib is a library of file reading and writing code to provide a general purpose trace file (and  Experiment File) reading interface. The programmer simply calls the (eg) read_reading to create a "Read" C structure  with the data loaded into memory. It has been compiled and tested on a variety of unix systems, MacOS X and MS   Windows.

*homepage*: http://sourceforge.net/projects/staden/files/io_lib/

version   |toolchain
----------|--------------
``1.14.8``|``foss/2016a``

<a anchor="ioapi"/>
### *ioapi*

The Models-3/EDSS Input/Output Applications Programming Interface (I/O API) provides the  environmental model developer with an easy-to-learn, easy-to-use programming library for data storage and  access, available from both Fortran and C. The same routines can be used for both file storage (using netCDF  files) and model coupling (using PVM mailboxes). It is the standard data access library for both the  NCSC/CMAS's EDSS project and EPA's Models-3, CMAQ, and SMOKE, as well as various other atmospheric and  hydrological modeling systems.

*homepage*: https://www.cmascenter.org/ioapi/

version        |versionsuffix|toolchain
---------------|-------------|---------------
``3.2-2020111``|``-nocpl``   |``gompi/2019b``

<a anchor="iomkl"/>
### *iomkl*

Compiler toolchain including Intel compilers, Open MPI and Intel Math Kernel Library (MKL).

*homepage*: https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit.html

version                   |toolchain
--------------------------|----------
``2016.07``               |``system``
``2016.09-GCC-4.9.3-2.25``|``system``
``2016.09-GCC-5.4.0-2.26``|``system``
``2017.01``               |``system``
``2017a``                 |``system``
``2017b``                 |``system``
``2018.02``               |``system``
``2018a``                 |``system``
``2018b``                 |``system``
``2019.01``               |``system``
``2019b``                 |``system``
``2020a``                 |``system``
``2020b``                 |``system``
``2021a``                 |``system``

<a anchor="iompi"/>
### *iompi*

Intel C/C++ and Fortran compilers, alongside Open MPI.

*homepage*: https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit.html

version                   |toolchain
--------------------------|----------
``2016.07``               |``system``
``2016.09-GCC-4.9.3-2.25``|``system``
``2016.09-GCC-5.4.0-2.26``|``system``
``2017.01``               |``system``
``2017a``                 |``system``
``2017b``                 |``system``
``2018.02``               |``system``
``2018a``                 |``system``
``2018b``                 |``system``
``2019.01``               |``system``
``2019b``                 |``system``
``2020a``                 |``system``
``2020b``                 |``system``
``2021a``                 |``system``

<a anchor="ior"/>
### *IOR*

The IOR software is used for benchmarking parallel file systems using POSIX, MPIIO,  or HDF5 interfaces. 

*homepage*: https://github.com/hpc/ior

version  |versionsuffix|toolchain
---------|-------------|-------------------------------------------------
``3.0.1``|``-mpiio``   |``foss/2016a``
``3.2.1``|             |``gompi/2019b``
``3.3.0``|             |``gompi/2020b``, ``gompi/2021a``, ``gompi/2022a``

<a anchor="iozone"/>
### *IOzone*

IOzone is a filesystem benchmark tool. The benchmark generates and measures a variety of file  operations. Iozone has been ported to many machines and runs under many operating systems.

*homepage*: http://www.iozone.org/

version  |toolchain
---------|--------------
``3.434``|``foss/2016a``

<a anchor="ipm"/>
### *IPM*

IPM is a portable profiling infrastructure for parallel codes. It provides  a low-overhead profile of application performance and resource utilization  in a parallel program. Communication, computation, and IO are the primary  focus. 

*homepage*: https://github.com/nerscadmin/IPM

version  |toolchain
---------|-----------------------------------------------------------------------------------
``2.0.6``|``gompi/2019b``, ``gompi/2020a``, ``iimpi/2019b``, ``iimpi/2020a``, ``iompi/2020a``

<a anchor="ipopt"/>
### *Ipopt*

IPOPT (Interior Point Optimizer, pronounced Eye-Pea-Opt)  is an open source software package for large-scale nonlinear optimization. 

*homepage*: https://projects.coin-or.org/Ipopt

version    |toolchain
-----------|---------------
``3.12.9`` |``foss/2017b``
``3.12.13``|``intel/2019a``

<a anchor="ipp"/>
### *ipp*

Intel Integrated Performance Primitives (Intel IPP) is an extensive library  of multicore-ready, highly optimized software functions for multimedia, data processing,  and communications applications. Intel IPP offers thousands of optimized functions  covering frequently used fundamental algorithms.

*homepage*: https://software.intel.com/en-us/articles/intel-ipp/

version       |toolchain
--------------|----------
``7.0.5.233`` |``system``
``8.1.0.144`` |``system``
``9.0.1.150`` |``system``
``2017.1.132``|``system``

<a anchor="ipy"/>
### *IPy*

Class and tools for handling of IPv4 and IPv6 addresses and networks

*homepage*: https://pypi.python.org/pypi/IPy

version |toolchain
--------|----------
``0.83``|``system``

<a anchor="ipyparallel"/>
### *ipyparallel*

ipyparallel is a Python package and collection  of CLI scripts for controlling clusters for Jupyter

*homepage*: https://ipyparallel.readthedocs.io

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``6.2.2``|``-Python-3.6.4``|``foss/2018a``

<a anchor="ipyrad"/>
### *ipyrad*

ipyrad is an interactive toolkit for assembly and analysis of restriction-site associated genomic  data sets (e.g., RAD, ddRAD, GBS) for population genetic and phylogenetic studies.

*homepage*: https://ipyrad.readthedocs.io

version   |versionsuffix     |toolchain
----------|------------------|---------------
``0.6.15``|``-Python-2.7.13``|``intel/2017a``

<a anchor="ipython"/>
### *IPython*

IPython provides a rich architecture for interactive computing with:  Powerful interactive shells (terminal and Qt-based).  A browser-based notebook with support for code, text, mathematical expressions, inline plots and other rich media.  Support for interactive data visualization and use of GUI toolkits.  Flexible, embeddable interpreters to load into your own projects.  Easy to use, high performance tools for parallel computing.

*homepage*: https://ipython.org/index.html

version   |versionsuffix     |toolchain
----------|------------------|---------------------------------------------------------------------------------------
``3.2.3`` |``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``4.2.0`` |``-Python-2.7.11``|``intel/2016a``
``5.0.0`` |``-Python-2.7.11``|``foss/2016a``
``5.0.0`` |``-Python-3.5.1`` |``foss/2016a``
``5.1.0`` |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``5.1.0`` |``-Python-3.5.2`` |``intel/2016b``
``5.2.2`` |``-Python-2.7.12``|``intel/2016b``
``5.3.0`` |``-Python-2.7.13``|``intel/2017a``
``5.7.0`` |``-Python-2.7.14``|``foss/2018a``, ``intel/2018a``
``5.8.0`` |``-Python-2.7.14``|``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``5.8.0`` |``-Python-2.7.15``|``foss/2018b``, ``foss/2019a``, ``fosscuda/2018b``, ``fosscuda/2019a``, ``intel/2018b``
``5.8.0`` |``-Python-3.6.3`` |``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``5.10.0``|``-Python-2.7.18``|``foss/2021b``
``6.2.1`` |``-Python-3.6.4`` |``foss/2017a``
``6.3.1`` |``-Python-3.6.4`` |``intel/2018a``
``6.4.0`` |``-Python-3.6.4`` |``foss/2018a``
``7.2.0`` |``-Python-3.6.6`` |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``7.7.0`` |``-Python-3.7.2`` |``foss/2019a``, ``fosscuda/2019a``, ``intel/2019a``
``7.9.0`` |``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``
``7.13.0``|``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``7.15.0``|``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``7.18.1``|                  |``GCCcore/10.2.0``
``7.25.0``|                  |``GCCcore/10.3.0``
``7.26.0``|                  |``GCCcore/11.2.0``
``8.5.0`` |                  |``GCCcore/11.3.0``

<a anchor="iq-tree"/>
### *IQ-TREE*

Efficient phylogenomic software by maximum likelihood

*homepage*: http://www.iqtree.org/

version   |versionsuffix|toolchain
----------|-------------|-----------------------------------------------
``1.5.5`` |``-omp-mpi`` |``foss/2016a``
``1.6.6`` |             |``intel/2018a``
``1.6.12``|             |``foss/2018b``, ``foss/2020a``, ``intel/2019b``
``2.1.2`` |             |``foss/2020a``, ``gompi/2020b``
``2.1.3`` |             |``gompi/2021a``
``2.2.1`` |             |``gompi/2021b``

<a anchor="iris"/>
### *Iris*

A module for improving the insertion sequences of structural variant calls

*homepage*: https://github.com/mkirsche/Iris

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.0.5``|``-Java-15`` |``GCC/11.2.0``

<a anchor="irkernel"/>
### *IRkernel*

The R kernel for the 'Jupyter' environment executes R code   which the front-end (Jupyter Notebook or other front-ends) submits to the   kernel via the network.

*homepage*: https://irkernel.github.io

version   |versionsuffix             |toolchain
----------|--------------------------|----------------------------------
``0.8.15``|``-R-3.4.3-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``1.1``   |``-R-3.6.2-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``
``1.1``   |``-R-3.6.3-Python-3.8.2`` |``foss/2020a``
``1.2``   |``-R-4.0.0-Python-3.8.2`` |``foss/2020a``
``1.2``   |``-R-4.1.0``              |``foss/2021a``
``1.3``   |``-R-4.2.0``              |``foss/2021b``

<a anchor="ironpython"/>
### *IronPython*

IronPython is an open-source implementation of the Python programming language  which is tightly integrated with the .NET Framework. IronPython can use the .NET Framework and Python libraries, and other .NET languages can use Python code just as easily.

*homepage*: http://ironpython.net/

version|toolchain
-------|---------------
``2.7``|``intel/2016b``

<a anchor="isa-l"/>
### *ISA-L*

Intelligent Storage Acceleration Library

*homepage*: https://github.com/intel/isa-l

version   |toolchain
----------|------------------------------------------------------------------------------
``2.30.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="isl"/>
### *ISL*

isl is a library for manipulating sets and relations of integer points bounded by linear constraints.

*homepage*: http://isl.gforge.inria.fr/

version |toolchain
--------|---------------------------------------------------------
``0.14``|``GCC/4.9.2``
``0.15``|``GCC/4.9.3-2.25``, ``GNU/4.9.3-2.25``, ``foss/2016a``
``0.16``|``GCC/4.9.3-2.25``
``0.17``|``foss/2016a``
``0.23``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/9.3.0``
``0.24``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="isocirc"/>
### *isoCirc*

isoCirc: computational pipeline to identify high-confidence BSJs and full-length circRNA isoforms from isoCirc long-read data 

*homepage*: https://github.com/Xinglab/isoCirc

version  |toolchain
---------|--------------
``1.0.4``|``foss/2020b``

<a anchor="isonet"/>
### *IsoNet*

IsoNet stands for for ISOtropic reconstructioN of Electron Tomography. It trains deep convolutional neural networks to reconstruct meaningful contents in the mis sing wedge for electron tomography, and to increase signal-to-noise ratio,       using the information learned from the original tomogram. The software requires  tomograms as input. Observing at about 30A resolution, the IsoNet generated      tomograms are largely isotropic.

*homepage*: https://github.com/Heng-Z/IsoNet

version                    |toolchain
---------------------------|------------------
``0.1_20210822_04_674f67f``|``fosscuda/2020b``

<a anchor="ispc"/>
### *ispc*

Intel SPMD Program Compilers; An open-source compiler for high-performance  SIMD programming on the CPU. ispc is a compiler for a variant of the C programming language,  with extensions for 'single program, multiple data' (SPMD) programming.  Under the SPMD model, the programmer writes a program that generally appears  to be a regular serial program, though the execution model is actually that  a number of program instances execute in parallel on the hardware.

*homepage*: http://ispc.github.io/ , https://github.com/ispc/ispc/

version   |toolchain
----------|----------
``1.6.0`` |``system``
``1.10.0``|``system``
``1.12.0``|``system``
``1.16.0``|``system``

<a anchor="itac"/>
### *itac*

The Intel Trace Collector is a low-overhead tracing library that performs  event-based tracing in applications. The Intel Trace Analyzer provides a convenient way to monitor application  activities gathered by the Intel Trace Collector through graphical displays. 

*homepage*: https://software.intel.com/en-us/intel-trace-analyzer/

version       |toolchain
--------------|----------
``8.0.0.011`` |``system``
``8.1.4.045`` |``system``
``9.0.3.051`` |``system``
``2017.1.024``|``system``
``2018.1.017``|``system``
``2018.3.022``|``system``
``2019.2.026``|``system``
``2019.4.036``|``system``
``2021.2.0``  |``system``
``2021.5.0``  |``system``
``2021.6.0``  |``system``

<a anchor="itk"/>
### *ITK*

Insight Segmentation and Registration Toolkit (ITK) provides  an extensive suite of software tools for registering and segmenting  multidimensional imaging data.

*homepage*: https://itk.org

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------------------------------------
``4.12.2``|``-Python-2.7.12``|``foss/2016b``
``4.13.0``|``-Python-2.7.14``|``foss/2018a``
``4.13.0``|``-Python-3.6.4`` |``foss/2018a``
``4.13.1``|``-Python-2.7.14``|``foss/2018a``
``4.13.1``|``-Python-2.7.15``|``foss/2018b``
``4.13.1``|``-Python-3.6.4`` |``foss/2018a``
``4.13.1``|``-Python-3.6.6`` |``foss/2018b``
``4.13.1``|``-Python-3.7.4`` |``foss/2019b``
``5.0.1`` |``-Python-3.7.2`` |``foss/2019a``
``5.0.1`` |``-Python-3.7.4`` |``foss/2019b``
``5.0b01``|``-Python-3.6.6`` |``foss/2018b``
``5.1.2`` |                  |``foss/2020a``, ``fosscuda/2020a``
``5.1.2`` |``-Python-3.8.2`` |``foss/2020a``, ``fosscuda/2020a``
``5.2.1`` |                  |``foss/2020b``, ``foss/2021a``, ``foss/2021b``, ``foss/2022a``

<a anchor="itpp"/>
### *itpp*

IT++ is a C++ library of mathematical, signal processing and communication  classes and functions. Its main use is in simulation of communication systems and for  performing research in the area of communications.

*homepage*: https://sourceforge.net/projects/itpp/

version  |toolchain
---------|--------------
``4.3.1``|``foss/2019b``

<a anchor="itstool"/>
### *ITSTool*

ITS Tool allows you to translate your XML documents with PO files

*homepage*: http://itstool.org/

version  |versionsuffix     |toolchain
---------|------------------|-----------------
``2.0.5``|``-Python-2.7.14``|``intel/2018a``
``2.0.5``|``-Python-2.7.15``|``foss/2018b``
``2.0.6``|``-Python-3.7.2`` |``GCCcore/8.2.0``

<a anchor="itsx"/>
### *ITSx*

ITSx: Improved software detection and extraction of ITS1 and ITS2 from ribosomal ITS sequences of fungi and other eukaryotes for use in environmental sequencing.

*homepage*: https://microbiology.se/software/itsx/

version  |toolchain
---------|------------------
``1.1.2``|``GCCcore/9.3.0``
``1.1.3``|``GCCcore/10.3.0``

<a anchor="ivar"/>
### *iVar*

iVar is a computational package that contains functions broadly useful for viral amplicon-based sequencing. 

*homepage*: https://github.com/andersen-lab/ivar

version  |toolchain
---------|-----------------------------
``1.0.1``|``foss/2018b``
``1.3.1``|``GCC/10.2.0``, ``GCC/9.3.0``

<a anchor="j"/>
### *J*


<a href="#jags">JAGS</a> - <a href="#jansson">Jansson</a> - <a href="#jasmine">Jasmine</a> - <a href="#jasper">JasPer</a> - <a href="#java">Java</a> - <a href="#javafx">JavaFX</a> - <a href="#jax">jax</a> - <a href="#jaxfrontce">JAXFrontCE</a> - <a href="#jbigkit">jbigkit</a> - <a href="#jellyfish">Jellyfish</a> - <a href="#jemalloc">jemalloc</a> - <a href="#jhbuild">jhbuild</a> - <a href="#jitcode">JiTCODE</a> - <a href="#jmodeltest">jModelTest</a> - <a href="#jorg">Jorg</a> - <a href="#joypy">joypy</a> - <a href="#jq">jq</a> - <a href="#json-c">json-c</a> - <a href="#json-fortran">json-fortran</a> - <a href="#json-glib">JSON-GLib</a> - <a href="#jsoncpp">JsonCpp</a> - <a href="#jube">JUBE</a> - <a href="#judy">Judy</a> - <a href="#julia">Julia</a> - <a href="#junit">JUnit</a> - <a href="#jupyter-matlab-proxy">jupyter-matlab-proxy</a> - <a href="#jupyter-resource-usage">jupyter-resource-usage</a> - <a href="#jupyter-server">jupyter-server</a> - <a href="#jupyter-server-proxy">jupyter-server-proxy</a> - <a href="#jupyterhub">JupyterHub</a> - <a href="#jupyterlab">JupyterLab</a> - <a href="#jwm">JWM</a> - <a href="#jxrlib">jxrlib</a>


<a anchor="jags"/>
### *JAGS*

JAGS is Just Another Gibbs Sampler.  It is a program for analysis   of Bayesian hierarchical models using Markov Chain Monte Carlo (MCMC) simulation  

*homepage*: http://mcmc-jags.sourceforge.net/

version  |toolchain
---------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------
``4.2.0``|``foss/2016a``, ``intel/2016a``, ``intel/2017a``
``4.3.0``|``foss/2017b``, ``foss/2018b``, ``foss/2019a``, ``foss/2019b``, ``foss/2020a``, ``foss/2020b``, ``foss/2021a``, ``foss/2021b``, ``fosscuda/2020b``, ``intel/2017b``

<a anchor="jansson"/>
### *Jansson*

Jansson is a C library for encoding, decoding and manipulating JSON data.  Its main features and design principles are:  * Simple and intuitive API and data model  * Comprehensive documentation  * No dependencies on other libraries  * Full Unicode support (UTF-8)  * Extensive test suite

*homepage*: http://www.digip.org/jansson/

version   |toolchain
----------|------------------------------
``2.6``   |``GCC/4.8.3``
``2.13.1``|``GCC/10.2.0``, ``GCC/11.2.0``
``2.14``  |``GCC/11.3.0``

<a anchor="jasmine"/>
### *Jasmine*

SV Merging Across Samples

*homepage*: https://github.com/mkirsche/Jasmine

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.1.4``|``-Java-15`` |``GCC/11.2.0``

<a anchor="jasper"/>
### *JasPer*

The JasPer Project is an open-source initiative to provide a free  software-based reference implementation of the codec specified in  the JPEG-2000 Part-1 standard. 

*homepage*: https://www.ece.uvic.ca/~frodo/jasper/

version    |toolchain
-----------|----------------------------------------------------------------------------------------------------------------------------------------------------------
``1.900.1``|``GCCcore/5.4.0``, ``GCCcore/6.4.0``, ``GCCcore/8.2.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``
``2.0.10`` |``intel/2016b``
``2.0.12`` |``GCCcore/6.4.0``, ``foss/2016b``, ``intel/2017a``
``2.0.14`` |``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``2.0.16`` |``GCCcore/9.3.0``
``2.0.24`` |``GCCcore/10.2.0``
``2.0.28`` |``GCCcore/10.3.0``
``2.0.33`` |``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``4.0.0``  |``GCCcore/12.2.0``

<a anchor="java"/>
### *Java*

Java Platform, Standard Edition (Java SE) lets you develop and deploy Java applications on desktops and servers.

*homepage*: https://java.com/

version      |versionsuffix           |toolchain
-------------|------------------------|----------
``1.6.0_24`` |                        |``system``
``1.7.0_10`` |                        |``system``
``1.7.0_15`` |                        |``system``
``1.7.0_21`` |                        |``system``
``1.7.0_40`` |                        |``system``
``1.7.0_45`` |                        |``system``
``1.7.0_60`` |                        |``system``
``1.7.0_75`` |                        |``system``
``1.7.0_76`` |                        |``system``
``1.7.0_79`` |                        |``system``
``1.7.0_80`` |                        |``system``
``1.8``      |                        |``system``
``1.8.0_20`` |                        |``system``
``1.8.0_25`` |                        |``system``
``1.8.0_31`` |                        |``system``
``1.8.0_40`` |                        |``system``
``1.8.0_45`` |                        |``system``
``1.8.0_60`` |                        |``system``
``1.8.0_65`` |                        |``system``
``1.8.0_66`` |                        |``system``
``1.8.0_72`` |                        |``system``
``1.8.0_74`` |                        |``system``
``1.8.0_77`` |                        |``system``
``1.8.0_92`` |                        |``system``
``1.8.0_112``|                        |``system``
``1.8.0_121``|                        |``system``
``1.8.0_131``|                        |``system``
``1.8.0_141``|                        |``system``
``1.8.0_144``|                        |``system``
``1.8.0_152``|                        |``system``
``1.8.0_162``|                        |``system``
``1.8.0_172``|                        |``system``
``1.8.0_181``|                        |``system``
``1.8.0_192``|                        |``system``
``1.8.0_202``|                        |``system``
``1.8.0_212``|                        |``system``
``1.8.0_221``|                        |``system``
``1.8.0_231``|                        |``system``
``1.8.0_241``|                        |``system``
``1.8.0_271``|                        |``system``
``1.8.0_281``|                        |``system``
``1.8.0_292``|``-OpenJDK``            |``system``
``1.8.0_311``|                        |``system``
``1.8_191``  |``-b26-OpenJDK``        |``system``
``1.8_265``  |``-b01-OpenJDK-aarch64``|``system``
``1.9.0.4``  |                        |``system``
``8``        |                        |``system``
``8.345``    |                        |``system``
``11``       |                        |``system``
``11.0.2``   |                        |``system``
``11.0.6``   |``-ppc64le``            |``system``
``11.0.8``   |``-aarch64``            |``system``
``11.0.16``  |                        |``system``
``13``       |                        |``system``
``13.0.2``   |                        |``system``
``15``       |                        |``system``
``15.0.1``   |                        |``system``
``16``       |                        |``system``
``16.0.1``   |                        |``system``
``17``       |                        |``system``
``17.0.1``   |                        |``system``
``17.0.2``   |                        |``system``
``17.0.4``   |                        |``system``

<a anchor="javafx"/>
### *JavaFX*

OpenJFX is an open source, next generation client application platform for desktop,  mobile and embedded systems built on Java 

*homepage*: https://openjfx.io/

version   |versionsuffix         |toolchain
----------|----------------------|----------
``11.0.2``|``_linux-x64_bin-sdk``|``system``

<a anchor="jax"/>
### *jax*

Composable transformations of Python+NumPy programs: differentiate, vectorize, JIT to GPU/TPU, and more

*homepage*: https://pypi.python.org/pypi/jax

version   |versionsuffix   |toolchain
----------|----------------|----------------------------------
``0.2.19``|                |``foss/2020b``, ``fosscuda/2020b``
``0.2.20``|                |``foss/2021a``
``0.2.24``|                |``foss/2021a``
``0.2.24``|``-CUDA-11.3.1``|``foss/2021a``
``0.3.9`` |                |``foss/2021a``
``0.3.9`` |``-CUDA-11.3.1``|``foss/2021a``
``0.3.14``|``-CUDA-11.7.0``|``foss/2022a``
``0.3.23``|                |``foss/2022a``
``0.3.23``|``-CUDA-11.4.1``|``foss/2021b``

<a anchor="jaxfrontce"/>
### *JAXFrontCE*

JAXFront is a technology to generate graphical user interfaces  on multiple channels (Java Swing, HTML, PDF) on the basis of an XML schema.

*homepage*: http://www.jaxfront.org/pages/free_community_edition.html

version |toolchain
--------|----------
``2.75``|``system``

<a anchor="jbigkit"/>
### *jbigkit*

JBIG-KIT is a software implementation of the JBIG1 data  compression standard (ITU-T T.82), which was designed for bi-level image  data, such as scanned documents.

*homepage*: https://www.cl.cam.ac.uk/~mgk25/jbigkit/

version|toolchain
-------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

<a anchor="jellyfish"/>
### *Jellyfish*

Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA.

*homepage*: http://www.genome.umd.edu/jellyfish.html

version   |toolchain
----------|-------------------------------------------------------------------
``1.1.11``|``foss/2016a``, ``foss/2016b``
``1.1.12``|``foss/2018b``, ``intel/2018a``
``2.2.6`` |``foss/2016b``, ``intel/2017a``
``2.2.10``|``foss/2018b``, ``intel/2018a``
``2.3.0`` |``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/8.2.0-2.31.1``, ``GCC/8.3.0``

<a anchor="jemalloc"/>
### *jemalloc*

jemalloc is a general purpose malloc(3) implementation that emphasizes fragmentation avoidance and  scalable concurrency support.

*homepage*: http://jemalloc.net

version  |toolchain
---------|------------------------------------------------------------------------------------------------
``4.1.0``|``intel/2016a``
``4.2.0``|``foss/2016a``, ``intel/2016a``
``4.2.1``|``intel/2016b``
``4.5.0``|``intel/2017a``
``5.0.1``|``GCCcore/6.4.0``
``5.1.0``|``GCCcore/7.3.0``
``5.2.0``|``GCCcore/8.2.0``
``5.2.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``5.3.0``|``GCCcore/11.3.0``

<a anchor="jhbuild"/>
### *jhbuild*

JHBuild allows you to automatically download and compile “modules” (i.e. source code packages).  Modules are listed in “module set” files, which also include dependency information so that JHBuild can discover  what modules need to be built and in what order.

*homepage*: https://wiki.gnome.org/action/show/Projects/Jhbuild

version    |toolchain
-----------|-----------------
``3.15.92``|``GCCcore/4.9.3``

<a anchor="jitcode"/>
### *JiTCODE*

Just-in-time compilation for ordinary/delay/stochastic differential equations (DDEs)

*homepage*: https://jitcde-common.readthedocs.io

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``1.3.2``|``-Python-3.6.4``|``intel/2018a``
``1.4.0``|``-Python-3.7.2``|``foss/2019a``

<a anchor="jmodeltest"/>
### *jModelTest*

jModelTest is a tool to carry out statistical selection of best-fit models of nucleotide substitution.

*homepage*: https://github.com/ddarriba/jmodeltest2

version            |versionsuffix     |toolchain
-------------------|------------------|----------
``2.1.10r20160303``|``-Java-1.8.0_92``|``system``

<a anchor="jorg"/>
### *Jorg*

A MAG Circularization Method By Lauren Lui, Torben Nielsen, and Adam Arkin

*homepage*: https://github.com/lmlui/Jorg

version  |toolchain
---------|--------------
``1.0.1``|``foss/2020b``

<a anchor="joypy"/>
### *joypy*

Joyplots in Python with matplotlib & pandas

*homepage*: https://github.com/sbebo/joypy

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.2.2``|``-Python-3.7.4``|``intel/2019b``
``0.2.4``|                 |``intel/2020b``

<a anchor="jq"/>
### *jq*

jq is a lightweight and flexible command-line JSON processor.

*homepage*: https://stedolan.github.io/jq/

version|toolchain
-------|----------------------------------------------------------
``1.5``|``GCCcore/6.4.0``
``1.6``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``

<a anchor="json-c"/>
### *json-c*

JSON-C implements a reference counting object model that allows you to easily construct JSON objects  in C, output them as JSON formatted strings and parse JSON formatted strings back into the C representation of JSON objects.

*homepage*: https://github.com/json-c/json-c

version |toolchain
--------|----------------------------------------------------------
``0.15``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``0.16``|``GCCcore/11.3.0``

<a anchor="json-fortran"/>
### *json-fortran*

JSON-Fortran: A Modern Fortran JSON API

*homepage*: https://github.com/jacobwilliams/json-fortran

version  |toolchain
---------|------------------
``8.3.0``|``GCCcore/11.2.0``

<a anchor="json-glib"/>
### *JSON-GLib*

JSON-GLib implements a full JSON parser and generator using GLib and GObject,  and integrates JSON with GLib data types.

*homepage*: https://wiki.gnome.org/Projects/JsonGlib

version  |toolchain
---------|------------------
``1.6.2``|``GCCcore/10.3.0``

<a anchor="jsoncpp"/>
### *JsonCpp*

JsonCpp is a C++ library that allows manipulating JSON values,  including serialization and deserialization to and from strings. It can also preserve existing comment in  unserialization/serialization steps, making it a convenient format to store user input files. 

*homepage*: https://open-source-parsers.github.io/jsoncpp-docs/doxygen/index.html

version   |toolchain
----------|-----------------------------------------------------------------------------
``0.10.7``|``GCCcore/8.2.0``
``1.9.3`` |``GCCcore/8.3.0``
``1.9.4`` |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``
``1.9.5`` |``GCCcore/11.3.0``

<a anchor="jube"/>
### *JUBE*

The JUBE benchmarking environment provides a script based framework to easily create benchmark sets, run those sets on different computer systems and evaluate the results. 

*homepage*: https://www.fz-juelich.de/jsc/jube

version  |toolchain
---------|----------
``2.0.3``|``system``
``2.0.4``|``system``
``2.0.5``|``system``
``2.4.0``|``system``
``2.4.1``|``system``
``2.4.2``|``system``

<a anchor="judy"/>
### *Judy*

A C library that implements a dynamic array.

*homepage*: http://judy.sourceforge.net/

version  |toolchain
---------|--------------------------------------------------------------------------------------------------------------------
``1.0.5``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``

<a anchor="julia"/>
### *Julia*

Julia is a high-level, high-performance dynamic programming language for numerical computing

*homepage*: https://julialang.org

version  |versionsuffix    |toolchain
---------|-----------------|----------
``1.1.1``|``-linux-x86_64``|``system``
``1.2.0``|``-linux-x86_64``|``system``
``1.3.1``|``-linux-x86_64``|``system``
``1.4.0``|``-linux-x86_64``|``system``
``1.4.1``|``-linux-x86_64``|``system``
``1.4.2``|``-linux-x86_64``|``system``
``1.5.1``|``-linux-x86_64``|``system``
``1.5.3``|``-linux-x86_64``|``system``
``1.6.1``|``-linux-x86_64``|``system``
``1.6.2``|``-linux-x86_64``|``system``
``1.6.4``|``-linux-x86_64``|``system``
``1.6.5``|``-linux-x86_64``|``system``
``1.6.6``|``-linux-x86_64``|``system``
``1.6.7``|``-linux-x86_64``|``system``
``1.7.0``|``-linux-x86_64``|``system``
``1.7.1``|``-linux-x86_64``|``system``
``1.7.2``|``-linux-x86_64``|``system``
``1.7.3``|``-linux-x86_64``|``system``
``1.8.0``|``-linux-x86_64``|``system``
``1.8.2``|``-linux-x86_64``|``system``

<a anchor="junit"/>
### *JUnit*

A programmer-oriented testing framework for Java.

*homepage*: http://sourceforge.net/projects/junit

version |versionsuffix      |toolchain
--------|-------------------|----------
``4.10``|``-Java-1.7.0_10`` |``system``
``4.10``|``-Java-1.7.0_21`` |``system``
``4.11``|``-Java-1.7.0_15`` |``system``
``4.11``|``-Java-1.7.0_21`` |``system``
``4.11``|``-Java-1.7.0_60`` |``system``
``4.11``|``-Java-1.7.0_75`` |``system``
``4.11``|``-Java-1.7.0_79`` |``system``
``4.12``|``-Java-1.7.0_80`` |``system``
``4.12``|``-Java-1.8``      |``system``
``4.12``|``-Java-1.8.0_112``|``system``
``4.12``|``-Java-1.8.0_121``|``system``
``4.12``|``-Java-1.8.0_144``|``system``
``4.12``|``-Java-1.8.0_152``|``system``
``4.12``|``-Java-1.8.0_162``|``system``
``4.12``|``-Java-1.8.0_66`` |``system``
``4.12``|``-Java-1.8.0_72`` |``system``
``4.12``|``-Java-1.8.0_77`` |``system``
``4.12``|``-Java-1.8.0_92`` |``system``

<a anchor="jupyter-matlab-proxy"/>
### *jupyter-matlab-proxy*

MATLAB Integration for Jupyter

*homepage*: https://github.com/mathworks/jupyter-matlab-proxy

version  |toolchain
---------|------------------
``0.3.4``|``GCCcore/10.3.0``

<a anchor="jupyter-resource-usage"/>
### *jupyter-resource-usage*

Jupyter Notebook Extension for monitoring your own Resource Usage (memory and/or CPU)

*homepage*: https://github.com/jupyter-server/jupyter-resource-usage

version  |toolchain
---------|------------------
``0.6.1``|``GCCcore/10.3.0``

<a anchor="jupyter-server"/>
### *jupyter-server*

The Jupyter Server provides the backend (i.e. the core services, APIs, and REST endpoints) for Jupyter web applications like Jupyter notebook, JupyterLab, and Voila.

*homepage*: https://jupyter.org/

version   |toolchain
----------|------------------
``1.21.0``|``GCCcore/11.3.0``

<a anchor="jupyter-server-proxy"/>
### *jupyter-server-proxy*

Jupyter Server Proxy lets you run arbitrary external processes (such as RStudio, Shiny Server, Syncthing, PostgreSQL, Code Server, etc) alongside your notebook server and provide authenticated web access to them     using a path like /rstudio next to others like /lab. Alongside the python package that provides the main functionality, the JupyterLab extension (@jupyterlab/server-proxy) provides buttons in the JupyterLab launcher window to get to RStudio for example.

*homepage*: https://github.com/jupyterhub/jupyter-server-proxy

version  |toolchain
---------|------------------
``3.2.1``|``GCCcore/10.3.0``

<a anchor="jupyterhub"/>
### *JupyterHub*

JupyterHub is a multiuser version of the Jupyter (IPython) notebook designed  for centralized deployments in companies, university classrooms and research labs.

*homepage*: https://jupyter.org

version  |versionsuffix    |toolchain
---------|-----------------|------------------
``0.6.1``|``-Python-3.5.1``|``foss/2016a``
``0.8.1``|``-Python-3.6.4``|``foss/2017a``
``1.1.0``|                 |``GCCcore/10.2.0``
``1.4.1``|                 |``GCCcore/10.3.0``

<a anchor="jupyterlab"/>
### *JupyterLab*

JupyterLab is the next-generation user interface for Project Jupyter offering all the familiar  building blocks of the classic Jupyter Notebook (notebook, terminal, text editor, file browser, rich outputs,  etc.) in a flexible and powerful user interface. JupyterLab will eventually replace the classic Jupyter  Notebook.

*homepage*: https://jupyter.org/

version   |versionsuffix    |toolchain
----------|-----------------|----------------------------------
``1.2.5`` |``-Python-3.7.4``|``foss/2019b``, ``fosscuda/2019b``
``2.2.8`` |                 |``GCCcore/10.2.0``
``3.0.16``|                 |``GCCcore/10.3.0``
``3.1.6`` |                 |``GCCcore/11.2.0``
``3.2.8`` |                 |``GCCcore/10.3.0``
``3.5.0`` |                 |``GCCcore/11.3.0``

<a anchor="jwm"/>
### *JWM*

JWM is a light-weight window manager for the X11 Window System.

*homepage*: https://joewing.net/projects/jwm/

version  |toolchain
---------|---------------
``2.3.5``|``intel/2016a``

<a anchor="jxrlib"/>
### *jxrlib*

Open source implementation of jpegxr

*homepage*: https://github.com/4creators/jxrlib

version|toolchain
-------|--------------------------------------
``1.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``

<a anchor="k"/>
### *K*


<a href="#kahip">KaHIP</a> - <a href="#kaiju">Kaiju</a> - <a href="#kaleido">Kaleido</a> - <a href="#kalign">Kalign</a> - <a href="#kallisto">kallisto</a> - <a href="#kat">KAT</a> - <a href="#kbproto">kbproto</a> - <a href="#kedro">kedro</a> - <a href="#kent_tools">Kent_tools</a> - <a href="#keras">Keras</a> - <a href="#khmer">khmer</a> - <a href="#kim-api">kim-api</a> - <a href="#king">king</a> - <a href="#kite">KITE</a> - <a href="#kma">kma</a> - <a href="#kmc">KMC</a> - <a href="#kmergenie">KmerGenie</a> - <a href="#knime">KNIME</a> - <a href="#kpcalg">kpcalg</a> - <a href="#kraken">Kraken</a> - <a href="#kraken2">Kraken2</a> - <a href="#kratos">Kratos</a> - <a href="#kronatools">KronaTools</a> - <a href="#kwant">kwant</a> - <a href="#kwiml">KWIML</a> - <a href="#kwip">kWIP</a> - <a href="#kyotocabinet">KyotoCabinet</a>


<a anchor="kahip"/>
### *KaHIP*

The graph partitioning framework KaHIP -- Karlsruhe High Quality Partitioning.

*homepage*: https://kahip.github.io/

version |toolchain
--------|---------------
``3.14``|``gompi/2022a``

<a anchor="kaiju"/>
### *Kaiju*

Kaiju is a program for sensitive taxonomic classification of high-throughput sequencing reads from metagenomic whole genome sequencing experiments

*homepage*: http://kaiju.binf.ku.dk/

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``1.5.0``|                 |``intel/2016b``
``1.7.2``|``-Python-3.7.2``|``iimpi/2019a``
``1.7.3``|``-Python-3.7.4``|``gompi/2019b``

<a anchor="kaleido"/>
### *Kaleido*

Fast static image export for web-based visualization libraries with zero dependencies

*homepage*: https://github.com/plotly/Kaleido

version  |toolchain
---------|------------------
``0.1.0``|``GCCcore/10.2.0``
``0.2.1``|``GCCcore/10.3.0``

<a anchor="kalign"/>
### *Kalign*

Kalign is a fast multiple sequence alignment program for biological sequences.

*homepage*: https://github.com/TimoLassmann/kalign

version  |toolchain
---------|--------------------------------------
``2.0.4``|``GCCcore/10.2.0``
``3.3.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``3.3.2``|``GCCcore/11.2.0``

<a anchor="kallisto"/>
### *kallisto*

kallisto is a program for quantifying abundances of transcripts from RNA-Seq data, or more generally  of target sequences using high-throughput sequencing reads.

*homepage*: https://pachterlab.github.io/kallisto/

version   |toolchain
----------|------------------------------------------------
``0.42.5``|``foss/2016a``
``0.43.0``|``intel/2016b``
``0.43.1``|``foss/2016b``, ``intel/2017a``, ``intel/2017b``
``0.44.0``|``foss/2016b``, ``intel/2018a``
``0.45.0``|``foss/2018b``
``0.45.1``|``foss/2019a``
``0.46.0``|``intel/2019a``
``0.46.1``|``foss/2019b``, ``iimpi/2020a``, ``iimpi/2020b``
``0.46.2``|``foss/2020b``
``0.48.0``|``gompi/2021b``

<a anchor="kat"/>
### *KAT*

The K-mer Analysis Toolkit (KAT) contains a number of tools that analyse and compare K-mer spectra.

*homepage*: https://www.earlham.ac.uk/kat-tools

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``2.4.2``|``-Python-3.6.4``|``foss/2018a``
``2.4.2``|``-Python-3.7.2``|``foss/2019a``

<a anchor="kbproto"/>
### *kbproto*

X.org KBProto protocol headers.

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version  |toolchain
---------|------------------------------------------------------------------
``1.0.7``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2017b``

<a anchor="kedro"/>
### *kedro*

Kedro is an open-source Python framework that applies software engineering best-practice to data and machine-learning pipelines. 

*homepage*: https://github.com/quantumblacklabs/kedro

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.16.5``|``-Python-3.8.2``|``foss/2020a``

<a anchor="kent_tools"/>
### *Kent_tools*

Kent utilities: collection of tools used by the UCSC genome browser.

*homepage*: https://genome.cse.ucsc.edu/

version     |versionsuffix    |toolchain
------------|-----------------|---------------
``401``     |                 |``gompi/2019b``
``411``     |                 |``GCC/10.2.0``
``418``     |                 |``GCC/10.3.0``
``422``     |                 |``GCC/11.2.0``
``20130806``|``-linux.x86_64``|``system``
``20171107``|``-linux.x86_64``|``system``
``20180716``|``-linux.x86_64``|``system``
``20190326``|``-linux.x86_64``|``system``

<a anchor="keras"/>
### *Keras*

Keras is a deep learning API written in Python, running on top of the machine learning platform TensorFlow. 

*homepage*: https://keras.io/

version  |versionsuffix     |toolchain
---------|------------------|----------------------------------
``1.0.8``|``-Python-3.5.2`` |``intel/2016b``
``1.1.0``|``-Python-3.5.2`` |``intel/2016b``
``2.0.4``|``-Python-2.7.13``|``intel/2017a``
``2.0.4``|``-Python-3.6.1`` |``intel/2017a``
``2.0.5``|``-Python-3.6.1`` |``intel/2017a``
``2.0.8``|``-Python-3.6.1`` |``intel/2017a``
``2.1.1``|``-Python-2.7.14``|``intel/2017b``
``2.1.1``|``-Python-3.6.3`` |``intel/2017b``
``2.1.2``|``-Python-2.7.14``|``intel/2017b``
``2.1.2``|``-Python-3.6.3`` |``foss/2017b``
``2.1.3``|``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``2.2.0``|``-Python-2.7.14``|``fosscuda/2017b``
``2.2.0``|``-Python-3.6.3`` |``fosscuda/2017b``
``2.2.0``|``-Python-3.6.4`` |``foss/2018a``
``2.2.2``|``-Python-2.7.15``|``fosscuda/2018b``
``2.2.4``|``-Python-3.6.4`` |``intel/2018a``
``2.2.4``|``-Python-3.6.6`` |``foss/2018b``, ``fosscuda/2018b``
``2.2.4``|``-Python-3.7.2`` |``foss/2019a``, ``fosscuda/2019a``
``2.3.1``|``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``
``2.3.1``|``-Python-3.8.2`` |``foss/2020a``
``2.4.3``|                  |``foss/2020b``, ``fosscuda/2020b``

<a anchor="khmer"/>
### *khmer*

In-memory nucleotide sequence k-mer counting, filtering, graph traversal and more 

*homepage*: https://github.com/ged-lab/khmer/

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.4.1``|``-Python-2.7.12``|``foss/2016b``
``2.1.1``|``-Python-2.7.13``|``intel/2017a``

<a anchor="kim-api"/>
### *kim-api*

Open Knowledgebase of Interatomic Models.  KIM is an API and OpenKIM is a collection of interatomic models (potentials) for atomistic simulations.  This is a library that can be used by simulation programs to get access to the models in the OpenKIM database.  This EasyBuild only installs the API, the models can be installed with the package openkim-models, or the user can install them manually by running     kim-api-collections-management install user MODELNAME or     kim-api-collections-management install user OpenKIM to install them all.  

*homepage*: https://openkim.org/

version  |toolchain
---------|----------------------------------------------------------------
``2.1.2``|``foss/2019a``, ``intel/2019a``
``2.1.3``|``foss/2019b``, ``foss/2020a``, ``intel/2019b``, ``intel/2020a``
``2.2.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``2.3.0``|``GCCcore/11.2.0``

<a anchor="king"/>
### *king*

KING is a toolset that makes use of high-throughput  SNP data typically seen in a genome-wide association study (GWAS) or  a sequencing project. Applications of KING include family relationship  inference and pedigree error checking, quality control, population  substructure identification, forensics, gene mapping, etc.

*homepage*: https://kingrelatedness.com/

version  |toolchain
---------|----------
``2.2.4``|``system``
``2.2.7``|``system``

<a anchor="kite"/>
### *KITE*

KITE is an open-source Python/C++ software suite for efficient real-space tight-binding (TB)  simulations of electronic structure and bulk quantum transport properties of disordered systems scalable to  multi billions of atomic orbitals.

*homepage*: https://github.com/quantum-kite/kite

version|toolchain
-------|---------------
``1.1``|``gompi/2022a``

<a anchor="kma"/>
### *kma*

KMA is a mapping method designed to map raw reads directly against redundant databases,  in an ultra-fast manner using seed and extend.

*homepage*: https://bitbucket.org/genomicepidemiology/kma

version   |toolchain
----------|---------------
``1.2.22``|``intel/2019b``

<a anchor="kmc"/>
### *KMC*

KMC is a disk-based programm for counting k-mers from (possibly gzipped) FASTQ/FASTA files.

*homepage*: http://sun.aei.polsl.pl/kmc

version     |versionsuffix    |toolchain
------------|-----------------|------------------------------
``3.1.0``   |                 |``foss/2018a``, ``foss/2018b``
``3.1.1``   |``-Python-3.7.2``|``GCC/8.2.0-2.31.1``
``3.1.2rc1``|``-Python-3.7.4``|``GCC/8.3.0``
``3.2.1``   |                 |``GCC/11.2.0``

<a anchor="kmergenie"/>
### *KmerGenie*

KmerGenie estimates the best k-mer length for genome de novo assembly.

*homepage*: http://kmergenie.bx.psu.edu/

version   |toolchain
----------|---------------
``1.7044``|``intel/2017a``
``1.7048``|``intel/2018a``

<a anchor="knime"/>
### *KNIME*

KNIME Analytics Platform is the open source software for creating data science applications and services. KNIME stands for KoNstanz Information MinEr.  

*homepage*: https://www.knime.com/

version  |toolchain
---------|----------
``3.6.2``|``system``

<a anchor="kpcalg"/>
### *kpcalg*

Kernel PC (kPC) algorithm for causal structure learning and  causal inference using graphical models. kPC is a version of PC algorithm that  uses kernel based independence criteria in order to be able to deal with  non-linear relationships and non-Gaussian noise.  Includes pcalg: Functions for causal structure learning and causal inference  using graphical models.

*homepage*: https://cran.r-project.org/package=kpcalg

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.0.1``|``-R-3.5.1`` |``foss/2018b``

<a anchor="kraken"/>
### *Kraken*

Kraken is a system for assigning taxonomic labels to short DNA sequences,  usually obtained through metagenomic studies. Previous attempts by other  bioinformatics software to accomplish this task have often used sequence  alignment or machine learning techniques that were quite slow, leading to  the development of less sensitive but much faster abundance estimation  programs. Kraken aims to achieve high sensitivity and high speed by  utilizing exact alignments of k-mers and a novel classification algorithm.

*homepage*: https://ccb.jhu.edu/software/kraken/

version        |versionsuffix   |toolchain
---------------|----------------|---------------------------------------------------------
``0.10.5-beta``|``-Perl-5.22.1``|``foss/2016a``
``0.10.5-beta``|``-Perl-5.24.0``|``foss/2016b``
``1.0``        |``-Perl-5.26.1``|``intel/2018a``
``1.1``        |``-Perl-5.28.0``|``foss/2018b``
``1.1.1``      |                |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/9.3.0``
``1.1.1``      |``-Perl-5.28.1``|``GCCcore/8.2.0``

<a anchor="kraken2"/>
### *Kraken2*

Kraken is a system for assigning taxonomic labels to short DNA sequences,  usually obtained through metagenomic studies. Previous attempts by other  bioinformatics software to accomplish this task have often used sequence  alignment or machine learning techniques that were quite slow, leading to  the development of less sensitive but much faster abundance estimation  programs. Kraken aims to achieve high sensitivity and high speed by  utilizing exact alignments of k-mers and a novel classification algorithm.

*homepage*: https://github.com/DerrickWood/kraken2/wiki

version       |versionsuffix   |toolchain
--------------|----------------|--------------------------------
``2.0.6-beta``|``-Perl-5.26.1``|``foss/2018a``
``2.0.7-beta``|``-Perl-5.28.0``|``foss/2018b``
``2.0.8-beta``|``-Perl-5.30.0``|``gompi/2019b``
``2.0.9-beta``|``-Perl-5.28.0``|``foss/2018b``
``2.0.9-beta``|``-Perl-5.30.2``|``gompi/2020a``
``2.1.1``     |                |``gompi/2020b``
``2.1.2``     |                |``gompi/2021a``, ``gompi/2021b``

<a anchor="kratos"/>
### *Kratos*

Kratos Multiphysics (A.K.A Kratos) is a framework for building parallel multi-disciplinary  simulation software.

*homepage*: https://www.cimne.com/kratos

version|versionsuffix    |toolchain
-------|-----------------|-------------------------------
``6.0``|``-Python-3.6.4``|``foss/2018a``, ``intel/2018a``

<a anchor="kronatools"/>
### *KronaTools*

Krona Tools is a set of scripts to create Krona charts from  several Bioinformatics tools as well as from text and XML files.

*homepage*: https://github.com/marbl/Krona/wiki/KronaTools

version  |toolchain
---------|----------------------------------
``2.7``  |``GCCcore/7.3.0``
``2.7.1``|``GCCcore/8.2.0``
``2.8``  |``GCC/10.3.0``, ``GCCcore/10.2.0``
``2.8.1``|``GCCcore/11.2.0``

<a anchor="kwant"/>
### *kwant*

Kwant is a free (open source), powerful, and easy to use Python package for   numerical calculations on tight-binding models with a strong focus on quantum transport.

*homepage*: https://kwant-project.org/

version  |versionsuffix    |toolchain
---------|-----------------|-------------------------------
``1.4.1``|``-Python-3.7.2``|``foss/2019a``, ``intel/2019a``

<a anchor="kwiml"/>
### *KWIML*

The Kitware Information Macro Library

*homepage*: https://gitlab.kitware.com/utils/kwiml

version     |toolchain
------------|-----------------
``20180201``|``GCCcore/6.4.0``

<a anchor="kwip"/>
### *kWIP*

This software implements a de novo, alignment free measure of sample  genetic dissimilarity which operates upon raw sequencing reads.  It is able to calculate the genetic dissimilarity between samples without any reference genome,  and without assembling one.

*homepage*: https://github.com/kdmurray91/kWIP

version  |toolchain
---------|-----------------
``0.2.0``|``GCCcore/6.4.0``

<a anchor="kyotocabinet"/>
### *KyotoCabinet*

Kyoto Cabinet is a library of routines for managing a database.

*homepage*: https://fallabs.com/kyotocabinet

version   |toolchain
----------|------------------------------------
``1.2.77``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``

<a anchor="l"/>
### *L*


<a href="#l_rna_scaffolder">L_RNA_scaffolder</a> - <a href="#ladr">LADR</a> - <a href="#lagrangian-filtering">lagrangian-filtering</a> - <a href="#lame">LAME</a> - <a href="#lammps">LAMMPS</a> - <a href="#lancet">lancet</a> - <a href="#lapack">LAPACK</a> - <a href="#last">LAST</a> - <a href="#lastz">LASTZ</a> - <a href="#lavaan">lavaan</a> - <a href="#lbfgs++">LBFGS++</a> - <a href="#lcov">LCov</a> - <a href="#ldc">LDC</a> - <a href="#lddt">lDDT</a> - <a href="#leadit">LeadIT</a> - <a href="#leidenalg">leidenalg</a> - <a href="#lemon">LEMON</a> - <a href="#leptonica">Leptonica</a> - <a href="#lerc">LERC</a> - <a href="#less">less</a> - <a href="#leveldb">LevelDB</a> - <a href="#lftp">lftp</a> - <a href="#lhapdf">LHAPDF</a> - <a href="#libaec">libaec</a> - <a href="#libaio">libaio</a> - <a href="#libarchive">libarchive</a> - <a href="#libav">libav</a> - <a href="#libavif">libavif</a> - <a href="#libbigwig">libBigWig</a> - <a href="#libbitmask">libbitmask</a> - <a href="#libcdms">libcdms</a> - <a href="#libcerf">libcerf</a> - <a href="#libcint">libcint</a> - <a href="#libcircle">libcircle</a> - <a href="#libcmaes">libcmaes</a> - <a href="#libconfig">libconfig</a> - <a href="#libcpuset">libcpuset</a> - <a href="#libcroco">libcroco</a> - <a href="#libctl">libctl</a> - <a href="#libdap">libdap</a> - <a href="#libde265">libde265</a> - <a href="#libdeflate">libdeflate</a> - <a href="#libdivsufsort">libdivsufsort</a> - <a href="#libdrm">libdrm</a> - <a href="#libdrs">libdrs</a> - <a href="#libdwarf">libdwarf</a> - <a href="#libedit">libedit</a> - <a href="#libelf">libelf</a> - <a href="#libepoxy">libepoxy</a> - <a href="#libev">libev</a> - <a href="#libevent">libevent</a> - <a href="#libexif">libexif</a> - <a href="#libfabric">libfabric</a> - <a href="#libfdf">libfdf</a> - <a href="#libffcall">libffcall</a> - <a href="#libffi">libffi</a> - <a href="#libflame">libFLAME</a> - <a href="#libfontenc">libfontenc</a> - <a href="#libgcrypt">libgcrypt</a> - <a href="#libgd">libgd</a> - <a href="#libgdiplus">libgdiplus</a> - <a href="#libgdsii">libGDSII</a> - <a href="#libgeotiff">libgeotiff</a> - <a href="#libgit2">libgit2</a> - <a href="#libglade">libglade</a> - <a href="#libglu">libGLU</a> - <a href="#libglvnd">libglvnd</a> - <a href="#libgpg-error">libgpg-error</a> - <a href="#libgpuarray">libgpuarray</a> - <a href="#libgridxc">libGridXC</a> - <a href="#libgtextutils">libgtextutils</a> - <a href="#libharu">libharu</a> - <a href="#libheif">libheif</a> - <a href="#libibmad">libibmad</a> - <a href="#libibumad">libibumad</a> - <a href="#libice">libICE</a> - <a href="#libiconv">libiconv</a> - <a href="#libidn">libidn</a> - <a href="#libidn2">libidn2</a> - <a href="#libint">Libint</a> - <a href="#libis">LiBis</a> - <a href="#libjpeg-turbo">libjpeg-turbo</a> - <a href="#libjxl">libjxl</a> - <a href="#liblzf">LibLZF</a> - <a href="#libmad">libmad</a> - <a href="#libmatheval">libmatheval</a> - <a href="#libmaus2">libmaus2</a> - <a href="#libmbd">libmbd</a> - <a href="#libmemcached">libMemcached</a> - <a href="#libmicrohttpd">libmicrohttpd</a> - <a href="#libmo_unpack">libmo_unpack</a> - <a href="#libmypaint">libmypaint</a> - <a href="#libnsl">libnsl</a> - <a href="#libobjcryst">libobjcryst</a> - <a href="#libogg">libogg</a> - <a href="#libopus">libopus</a> - <a href="#libosmium">libosmium</a> - <a href="#libpci">libpci</a> - <a href="#libpciaccess">libpciaccess</a> - <a href="#libplinkio">libplinkio</a> - <a href="#libpng">libpng</a> - <a href="#libpsl">libpsl</a> - <a href="#libpsml">libPSML</a> - <a href="#libpsml">libpsml</a> - <a href="#libpsortb">libpsortb</a> - <a href="#libpspio">libpspio</a> - <a href="#libpthread-stubs">libpthread-stubs</a> - <a href="#libqglviewer">libQGLViewer</a> - <a href="#libreadline">libreadline</a> - <a href="#librmath">libRmath</a> - <a href="#librosa">librosa</a> - <a href="#librsb">librsb</a> - <a href="#librsvg">librsvg</a> - <a href="#libsamplerate">libsamplerate</a> - <a href="#libsbml">libSBML</a> - <a href="#libsigc++">libsigc++</a> - <a href="#libsigsegv">libsigsegv</a> - <a href="#libsm">libSM</a> - <a href="#libsndfile">libsndfile</a> - <a href="#libsodium">libsodium</a> - <a href="#libsoup">LibSoup</a> - <a href="#libspatialindex">libspatialindex</a> - <a href="#libspatialite">libspatialite</a> - <a href="#libssh">libssh</a> - <a href="#libstatgen">libStatGen</a> - <a href="#libsvm">LIBSVM</a> - <a href="#libtar">libtar</a> - <a href="#libtasn1">libtasn1</a> - <a href="#libtecla">libtecla</a> - <a href="#libtiff">LibTIFF</a> - <a href="#libtirpc">libtirpc</a> - <a href="#libtool">libtool</a> - <a href="#libtree">libtree</a> - <a href="#libunistring">libunistring</a> - <a href="#libunwind">libunwind</a> - <a href="#libutempter">libutempter</a> - <a href="#libuuid">LibUUID</a> - <a href="#libuv">libuv</a> - <a href="#libvdwxc">libvdwxc</a> - <a href="#libvorbis">libvorbis</a> - <a href="#libwebp">libwebp</a> - <a href="#libwpe">libwpe</a> - <a href="#libx11">libX11</a> - <a href="#libxau">libXau</a> - <a href="#libxc">libxc</a> - <a href="#libxcb">libxcb</a> - <a href="#libxcursor">libXcursor</a> - <a href="#libxdamage">libXdamage</a> - <a href="#libxdmcp">libXdmcp</a> - <a href="#libxext">libXext</a> - <a href="#libxfixes">libXfixes</a> - <a href="#libxfont">libXfont</a> - <a href="#libxft">libXft</a> - <a href="#libxi">libXi</a> - <a href="#libxinerama">libXinerama</a> - <a href="#libxkbcommon">libxkbcommon</a> - <a href="#libxml++">libxml++</a> - <a href="#libxml2">libxml2</a> - <a href="#libxml2-python">libxml2-python</a> - <a href="#libxmu">libXmu</a> - <a href="#libxp">libXp</a> - <a href="#libxpm">libXpm</a> - <a href="#libxrandr">libXrandr</a> - <a href="#libxrender">libXrender</a> - <a href="#libxslt">libxslt</a> - <a href="#libxsmm">libxsmm</a> - <a href="#libxt">libXt</a> - <a href="#libxxf86vm">libXxf86vm</a> - <a href="#libyaml">libyaml</a> - <a href="#libzeep">libzeep</a> - <a href="#libzip">libzip</a> - <a href="#lie_learn">lie_learn</a> - <a href="#lifelines">lifelines</a> - <a href="#lighter">Lighter</a> - <a href="#liknorm">liknorm</a> - <a href="#likwid">likwid</a> - <a href="#limix">limix</a> - <a href="#linbox">LinBox</a> - <a href="#line_profiler">line_profiler</a> - <a href="#lingeling">Lingeling</a> - <a href="#littlecms">LittleCMS</a> - <a href="#lldb">LLDB</a> - <a href="#llvm">LLVM</a> - <a href="#lmdb">LMDB</a> - <a href="#lmfit">LMfit</a> - <a href="#lmod">Lmod</a> - <a href="#lncloom">LncLOOM</a> - <a href="#locarna">LocARNA</a> - <a href="#lofreq">LoFreq</a> - <a href="#log-log4perl">Log-Log4perl</a> - <a href="#logaddexp">logaddexp</a> - <a href="#lohhla">LOHHLA</a> - <a href="#loki">Loki</a> - <a href="#longestrunsubsequence">longestrunsubsequence</a> - <a href="#longread_umi">longread_umi</a> - <a href="#longshot">Longshot</a> - <a href="#loompy">loompy</a> - <a href="#lpeg">LPeg</a> - <a href="#lpjml">LPJmL</a> - <a href="#lpsolve">lpsolve</a> - <a href="#lrslib">lrslib</a> - <a href="#ls-prepost">LS-PrePost</a> - <a href="#lsd2">LSD2</a> - <a href="#lsms">LSMS</a> - <a href="#ltr_retriever">LTR_retriever</a> - <a href="#ltrdetector">LtrDetector</a> - <a href="#lua">Lua</a> - <a href="#luajit">LuaJIT</a> - <a href="#luajit2-openresty">LuaJIT2-OpenResty</a> - <a href="#lucene-geo-gazetteer">Lucene-Geo-Gazetteer</a> - <a href="#lumpy">LUMPY</a> - <a href="#luscus">LUSCUS</a> - <a href="#lwgrp">lwgrp</a> - <a href="#lxml">lxml</a> - <a href="#lynx">lynx</a> - <a href="#lz4">lz4</a> - <a href="#lzo">LZO</a>


<a anchor="l_rna_scaffolder"/>
### *L_RNA_scaffolder*

L_RNA_scaffolder is a novel scaffolding tool using long trancriptome reads to scaffold genome fragments

*homepage*: http://www.fishbrowser.org/software/L_RNA_scaffolder/

version     |versionsuffix   |toolchain
------------|----------------|---------------
``20141124``|``-Perl-5.24.0``|``intel/2016b``

<a anchor="ladr"/>
### *LADR*

Prover9, Mace4, and several related programs come packaged in  a system called LADR (Library for Automated Deduction Research). 

*homepage*: https://www.cs.unm.edu/~mccune/mace4/download

version     |toolchain
------------|------------------
``2009-11A``|``GCCcore/10.2.0``

<a anchor="lagrangian-filtering"/>
### *lagrangian-filtering*

Temporal filtering of data in a Lagrangian frame of reference.

*homepage*: https://github.com/angus-g/lagrangian-filtering

version  |toolchain
---------|--------------
``0.8.3``|``foss/2022a``

<a anchor="lame"/>
### *LAME*

LAME is a high quality MPEG Audio Layer III (MP3) encoder licensed under the LGPL.

*homepage*: http://lame.sourceforge.net/

version   |toolchain
----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``3.99.5``|``foss/2016b``, ``intel/2017a``
``3.100`` |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``intel/2017b``

<a anchor="lammps"/>
### *LAMMPS*

LAMMPS is a classical molecular dynamics code, and an acronym for Large-scale Atomic/Molecular Massively Parallel Simulator. LAMMPS has potentials for solid-state materials (metals, semiconductors) and soft matter (biomolecules, polymers) and coarse-grained or mesoscopic systems. It can be used to model atoms or, more generically, as a parallel particle simulator at the atomic, meso, or continuum scale. LAMMPS runs on single processors or in parallel using message-passing techniques and a spatial-decomposition of the simulation domain. The code is designed to be easy to modify or extend with new functionality. 

*homepage*: https://lammps.sandia.gov/

version      |versionsuffix                |toolchain
-------------|-----------------------------|-------------------------------
``3Mar2020`` |``-Python-3.7.4-kokkos``     |``foss/2019b``, ``intel/2019b``
``3Mar2020`` |``-Python-3.8.2-kokkos``     |``foss/2020a``, ``intel/2020a``
``7Aug2019`` |``-Python-3.7.4-kokkos``     |``foss/2019b``, ``intel/2019b``
``7Aug2019`` |``-Python-3.7.4-kokkos-OCTP``|``intel/2019b``
``23Jun2022``|``-kokkos``                  |``foss/2021a``, ``foss/2021b``
``23Jun2022``|``-kokkos-CUDA-11.3.1``      |``foss/2021a``
``23Jun2022``|``-kokkos-CUDA-11.4.1``      |``foss/2021b``

<a anchor="lancet"/>
### *lancet*

Lancet is a somatic variant caller (SNVs and indels) for short read data.

*homepage*: https://github.com/nygenome/lancet

version  |toolchain
---------|-----------------------
``1.1.0``|``iccifort/2019.5.281``

<a anchor="lapack"/>
### *LAPACK*

LAPACK is written in Fortran90 and provides routines for solving systems of  simultaneous linear equations, least-squares solutions of linear systems of equations, eigenvalue  problems, and singular value problems.

*homepage*: https://www.netlib.org/lapack/

version   |toolchain
----------|------------------------------
``3.8.0`` |``GCC/7.3.0-2.30``
``3.9.1`` |``GCC/10.2.0``
``3.10.1``|``GCC/11.2.0``, ``GCC/11.3.0``

<a anchor="last"/>
### *LAST*

LAST finds similar regions between sequences.

*homepage*: http://last.cbrc.jp/

version |toolchain
--------|---------------
``869`` |``intel/2017a``
``914`` |``intel/2017b``
``1045``|``intel/2019b``
``1179``|``GCC/10.2.0``

<a anchor="lastz"/>
### *LASTZ*

LASTZ is a program for aligning DNA sequences, a pairwise aligner. Originally designed to handle  sequences the size of human chromosomes and from different species, it is also useful for sequences produced by NGS   sequencing technologies such as Roche 454. 

*homepage*: https://www.bx.psu.edu/~rsharris/lastz/

version    |toolchain
-----------|--------------
``1.02.00``|``foss/2016a``
``1.04.03``|``foss/2019b``

<a anchor="lavaan"/>
### *lavaan*

Fit a variety of latent variable models, including confirmatory factor analysis, structural  equation modeling and latent growth curve models.

*homepage*: https://cran.r-project.org/web/packages/lavaan

version       |versionsuffix|toolchain
--------------|-------------|---------------
``0.6-2``     |``-R-3.4.4`` |``intel/2018a``
``0.6-4.1433``|``-R-3.6.0`` |``foss/2019a``
``0.6-9``     |``-R-4.1.0`` |``foss/2021a``

<a anchor="lbfgs++"/>
### *LBFGS++*

A header-only C++ library for L-BFGS and L-BFGS-B algorithms

*homepage*: https://lbfgspp.statr.me

version  |toolchain
---------|----------
``0.1.0``|``system``

<a anchor="lcov"/>
### *LCov*

LCOV - the LTP GCOV extension

*homepage*: http://ltp.sourceforge.net/coverage/lcov.php

version |toolchain
--------|-----------------
``1.13``|``GCCcore/7.2.0``

<a anchor="ldc"/>
### *LDC*

The LLVM-based D Compiler

*homepage*: https://wiki.dlang.org/LDC

version   |versionsuffix|toolchain
----------|-------------|------------------
``0.17.6``|``-x86_64``  |``system``
``1.24.0``|``-x86_64``  |``system``
``1.25.1``|             |``GCCcore/10.2.0``
``1.26.0``|             |``GCCcore/10.3.0``
``1.30.0``|             |``GCCcore/11.3.0``

<a anchor="lddt"/>
### *lDDT*

The local Distance Difference Test (lDDT) is a superposition-free score which evaluates local distance differences in a model compared to a reference structure.

*homepage*: https://openstructure.org

version|toolchain
-------|----------
``1.2``|``system``

<a anchor="leadit"/>
### *LeadIT*

Visually Informed LeadOpt

*homepage*: http://www.biosolveit.de/LeadIT/index.html

version  |toolchain
---------|----------
``2.1.9``|``system``

<a anchor="leidenalg"/>
### *leidenalg*

Implementation of the Leiden algorithm for various quality functions to be used with igraph in Python.

*homepage*: https://github.com/vtraag/leidenalg

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------
``0.8.2``|``-Python-3.8.2``|``foss/2020a``
``0.8.3``|                 |``foss/2020b``, ``fosscuda/2020b``
``0.8.7``|                 |``foss/2021a``
``0.8.8``|                 |``foss/2021b``

<a anchor="lemon"/>
### *LEMON*

LEMON stands for Library for Efficient Modeling and Optimization in Networks.  It is a C++ template library providing efficient implementations of common data structures and algorithms  with focus on combinatorial optimization tasks connected mainly with graphs and networks. 

*homepage*: https://lemon.cs.elte.hu

version  |toolchain
---------|--------------------
``1.3.1``|``GCC/8.2.0-2.31.1``

<a anchor="leptonica"/>
### *Leptonica*

Leptonica is a collection of pedagogically-oriented open source software  that is broadly useful for image processing and image analysis applications.

*homepage*: http://www.leptonica.org

version   |toolchain
----------|------------------
``1.77.0``|``GCCcore/7.3.0``
``1.78.0``|``GCCcore/8.2.0``
``1.82.0``|``GCCcore/10.3.0``
``1.83.0``|``GCCcore/11.3.0``

<a anchor="lerc"/>
### *LERC*

LERC is an open-source image or raster format which supports rapid encoding and decoding for any pixel type (not just RGB or Byte). Users set the maximum compression error per pixel while encoding, so the precision of the original input image is preserved (within user defined error bounds).

*homepage*: https://github.com/Esri/lerc

version|toolchain
-------|--------------------------------------
``3.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``

<a anchor="less"/>
### *less*

Less is a free, open-source file pager. It can be found on most versions of Linux,  Unix and Mac OS, as well as on many other operating systems.

*homepage*: http://www.greenwoodsoftware.com/less/

version|toolchain
-------|-------------
``458``|``GCC/4.8.2``

<a anchor="leveldb"/>
### *LevelDB*

LevelDB is a fast key-value storage library written at Google that provides an  ordered mapping from string keys to string values.

*homepage*: https://github.com/google/leveldb

version |toolchain
--------|------------------------------------------------
``1.18``|``foss/2016a``, ``intel/2017a``, ``intel/2017b``
``1.20``|``GCCcore/7.3.0``
``1.22``|``GCCcore/8.2.0``, ``GCCcore/9.3.0``

<a anchor="lftp"/>
### *lftp*

LFTP is a sophisticated ftp/http client, and a file transfer program supporting a number of network protocols. Like BASH, it has job control and uses the readline library for input. It has bookmarks, a built-in mirror command, and can transfer several files in parallel. It was designed with reliability in mind.

*homepage*: https://lftp.yar.ru

version  |toolchain
---------|------------------
``4.6.4``|``GNU/4.9.3-2.25``
``4.8.4``|``GCCcore/6.4.0``
``4.9.2``|``GCCcore/11.2.0``

<a anchor="lhapdf"/>
### *LHAPDF*

Les Houches Parton Density Function  LHAPDF is the standard tool for evaluating parton distribution functions (PDFs) in high-energy physics. 

*homepage*: http://lhapdf.hepforge.org/

version  |toolchain
---------|--------------
``6.5.3``|``GCC/11.3.0``

<a anchor="libaec"/>
### *libaec*

Libaec provides fast lossless compression of 1 up to 32 bit wide signed or unsigned integers (samples). The library achieves best results for low entropy data as often encountered in space imaging instrument data or numerical model output from weather or climate simulations. While floating point representations are not directly supported, they can also be efficiently coded by grouping exponents and mantissa.

*homepage*: https://gitlab.dkrz.de/k202009/libaec

version  |toolchain
---------|------------------------------------------------------------------------------
``1.0.6``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="libaio"/>
### *libaio*

Asynchronous input/output library that uses the kernels native interface.

*homepage*: https://pagure.io/libaio

version    |toolchain
-----------|------------------------------------------------------------------------------
``0.3.111``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``0.3.112``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="libarchive"/>
### *libarchive*

Multi-format archive and compression library 

*homepage*: https://www.libarchive.org/

version  |toolchain
---------|---------------------------------------------------------
``3.4.0``|``GCCcore/8.2.0``
``3.4.2``|``GCCcore/9.3.0``
``3.4.3``|``GCCcore/10.2.0``
``3.5.1``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/8.3.0``
``3.6.1``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``

<a anchor="libav"/>
### *libav*

Libav is a friendly and community-driven effort to provide its users with  a set of portable, functional and high-performance libraries for dealing  with multimedia formats of all sorts. 

*homepage*: https://libav.org/

version  |toolchain
---------|-----------------
``11.10``|``GCCcore/6.4.0``

<a anchor="libavif"/>
### *libavif*

This library aims to be a friendly, portable C implementation of the AV1 Image File Format, as described here: https://aomediacodec.github.io/av1-avif/ 

*homepage*: https://github.com/AOMediaCodec/libavif

version   |toolchain
----------|--------------
``0.9.0`` |``foss/2020b``
``0.11.1``|``foss/2021a``

<a anchor="libbigwig"/>
### *libBigWig*

A C library for handling bigWig files

*homepage*: https://github.com/dpryan79/libBigWig

version  |toolchain
---------|------------------
``0.4.4``|``GCCcore/8.3.0``
``0.4.6``|``GCCcore/11.2.0``

<a anchor="libbitmask"/>
### *libbitmask*

libbitmask provides a convenient, powerful bitmask data type

*homepage*: http://oss.sgi.com/projects/cpusets/

version|toolchain
-------|----------
``2.0``|``system``

<a anchor="libcdms"/>
### *libcdms*

Climate Data Management System Library.

*homepage*: https://github.com/CDAT/libcdms/

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``3.1.2``|``-Python-3.8.2``|``foss/2020a``

<a anchor="libcerf"/>
### *libcerf*

libcerf is a self-contained numeric library that provides an efficient and  accurate implementation of complex error functions, along with Dawson,  Faddeeva, and Voigt functions. 

*homepage*: https://jugit.fz-juelich.de/mlz/libcerf

version |toolchain
--------|---------------------------------------------------------------------------------------------------------
``1.4`` |``foss/2016a``, ``foss/2016b``, ``intel/2016a``
``1.5`` |``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``foss/2016b``, ``intel/2016b``, ``intel/2017a``
``1.7`` |``GCCcore/7.3.0``
``1.11``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``1.13``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.14``|``GCCcore/10.2.0``
``1.15``|``GCCcore/10.3.0``
``1.17``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``2.1`` |``GCCcore/11.3.0``

<a anchor="libcint"/>
### *libcint*

libcint is an open source library for analytical Gaussian integrals.

*homepage*: https://github.com/sunqm/libcint

version  |toolchain
---------|-----------------------------------------------
``4.4.0``|``foss/2020b``, ``foss/2021a``, ``gomkl/2021a``
``5.1.6``|``foss/2022a``

<a anchor="libcircle"/>
### *libcircle*

An API to provide an efficient distributed queue on a cluster. libcircle is an  API for distributing embarrassingly parallel workloads using self-stabilization. 

*homepage*: https://github.com/hpc/libcircle/

version       |toolchain
--------------|-------------------------------------------------
``0.2.1-rc.1``|``gompi/2019a``, ``iimpi/2019a``
``0.3``       |``gompi/2020a``, ``gompi/2020b``, ``iimpi/2020a``

<a anchor="libcmaes"/>
### *libcmaes*

libcmaes is a multithreaded C++11 library for high performance blackbox stochastic optimization using the CMA-ES algorithm for Covariance Matrix Adaptation Evolution Strategy.

*homepage*: http://beniz.github.io/libcmaes/

version  |toolchain
---------|--------------
``0.9.5``|``foss/2016a``

<a anchor="libconfig"/>
### *libconfig*

Libconfig is a simple library for processing structured configuration files

*homepage*: https://hyperrealm.github.io/libconfig

version  |toolchain
---------|--------------------------------------
``1.5``  |``intel/2016b``
``1.7.1``|``GCCcore/6.4.0``
``1.7.2``|``GCCcore/7.3.0``
``1.7.3``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``

<a anchor="libcpuset"/>
### *libcpuset*

libcpuset provides full access to cpuset capabilities

*homepage*: http://oss.sgi.com/projects/cpusets/

version|toolchain
-------|----------
``1.0``|``system``

<a anchor="libcroco"/>
### *libcroco*

Libcroco is a standalone css2 parsing and manipulation library.

*homepage*: https://github.com/GNOME/libcroco

version   |toolchain
----------|------------------------------
``0.6.11``|``intel/2016a``
``0.6.13``|``GCC/10.2.0``, ``foss/2019a``

<a anchor="libctl"/>
### *libctl*

libctl is a free Guile-based library implementing flexible control files for scientific simulations.

*homepage*: https://libctl.readthedocs.io/en/latest/

version  |toolchain
---------|-----------------
``3.2.2``|``foss/2016a``
``4.0.0``|``intel/2020a``
``4.1.3``|``GCCcore/6.4.0``

<a anchor="libdap"/>
### *libdap*

A C++ SDK which contains an implementation of DAP 2.0 and  DAP4.0. This includes both Client- and Server-side support classes.

*homepage*: https://www.opendap.org/software/libdap

version    |versionsuffix     |toolchain
-----------|------------------|---------------------------------------------------------
``3.18.1`` |                  |``intel/2017a``
``3.18.1`` |``-Python-2.7.11``|``foss/2016a``
``3.19.1`` |                  |``GCCcore/6.4.0``, ``foss/2017b``, ``intel/2017b``
``3.20.3`` |                  |``GCCcore/7.3.0``
``3.20.4`` |                  |``GCCcore/8.2.0``
``3.20.6`` |                  |``GCCcore/8.3.0``
``3.20.7`` |                  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/9.3.0``
``3.20.8`` |                  |``GCCcore/11.2.0``
``3.20.11``|                  |``GCCcore/11.3.0``

<a anchor="libde265"/>
### *libde265*

libde265 is an open source implementation of the h.265 video codec

*homepage*: https://github.com/strukturag/libde265

version  |toolchain
---------|------------------------------
``1.0.8``|``GCC/10.3.0``, ``GCC/11.2.0``

<a anchor="libdeflate"/>
### *libdeflate*

Heavily optimized library for DEFLATE/zlib/gzip compression and decompression.

*homepage*: https://github.com/ebiggers/libdeflate

version |toolchain
--------|---------------------------------------------------------
``1.5`` |``GCCcore/7.3.0``
``1.7`` |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/9.3.0``
``1.8`` |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.10``|``GCCcore/11.3.0``
``1.15``|``GCCcore/12.2.0``

<a anchor="libdivsufsort"/>
### *libdivsufsort*

libdivsufsort is a software library that implements a lightweight suffix  array construction algorithm.

*homepage*: https://github.com/y-256/libdivsufsort

version  |toolchain
---------|--------------
``2.0.1``|``foss/2018b``

<a anchor="libdrm"/>
### *libdrm*

Direct Rendering Manager runtime library.

*homepage*: https://dri.freedesktop.org

version    |toolchain
-----------|--------------------------------------------------
``2.4.67`` |``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``2.4.68`` |``foss/2016a``, ``intel/2016a``
``2.4.70`` |``GCCcore/5.4.0``, ``foss/2016b``, ``intel/2016b``
``2.4.76`` |``GCCcore/6.3.0``, ``intel/2017a``
``2.4.88`` |``GCCcore/6.4.0``
``2.4.91`` |``GCCcore/6.4.0``
``2.4.92`` |``GCCcore/7.3.0``
``2.4.97`` |``GCCcore/8.2.0``
``2.4.99`` |``GCCcore/8.3.0``
``2.4.100``|``GCCcore/9.3.0``
``2.4.102``|``GCCcore/10.2.0``
``2.4.106``|``GCCcore/10.3.0``
``2.4.107``|``GCCcore/11.2.0``
``2.4.110``|``GCCcore/11.3.0``
``2.4.114``|``GCCcore/12.2.0``

<a anchor="libdrs"/>
### *libdrs*

PCMDI's old DRS format implementation

*homepage*: https://github.com/CDAT/libdrs/

version  |toolchain
---------|--------------
``3.1.2``|``foss/2020a``

<a anchor="libdwarf"/>
### *libdwarf*

The DWARF Debugging Information Format is of interest to programmers working on compilers and debuggers (and anyone interested in reading or writing DWARF information))

*homepage*: https://www.prevanders.net/dwarf.html

version     |toolchain
------------|---------------------------------------------------
``0.4.1``   |``GCCcore/11.3.0``
``20140805``|``GCC/4.9.2``
``20150310``|``GCC/4.9.2``, ``GCCcore/5.4.0``, ``GCCcore/6.3.0``
``20190529``|``GCCcore/8.2.0``
``20201201``|``GCCcore/10.2.0``
``20210305``|``GCCcore/10.3.0``
``20210528``|``GCCcore/11.2.0``

<a anchor="libedit"/>
### *libedit*

This BSD-style licensed command line editor library provides generic line editing, history, and tokenization functions, similar to those found in GNU Readline. 

*homepage*: https://thrysoee.dk/editline/

version     |toolchain
------------|------------------
``20150325``|``GNU/4.9.3-2.25``
``20180525``|``GCCcore/6.4.0``
``20191231``|``GCCcore/9.3.0``
``20210910``|``GCCcore/11.2.0``

<a anchor="libelf"/>
### *libelf*

libelf is a free ELF object file access library

*homepage*: https://directory.fsf.org/wiki/Libelf

version   |toolchain
----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``0.8.13``|``GCC/4.8.3``, ``GCC/4.9.2``, ``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/8.2.0``, ``GCCcore/9.3.0``

<a anchor="libepoxy"/>
### *libepoxy*

Epoxy is a library for handling OpenGL function pointer management for you

*homepage*: https://github.com/anholt/libepoxy

version   |toolchain
----------|--------------------------------------------------------
``1.5.2`` |``foss/2018a``
``1.5.3`` |``GCCcore/8.2.0``, ``fosscuda/2018b``
``1.5.4`` |``GCCcore/10.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.5.8`` |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.5.10``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``

<a anchor="libev"/>
### *libev*

A full-featured and high-performance (see benchmark) event loop that is loosely modelled after libevent, but without its limitations and bugs. It is used in GNU Virtual Private Ethernet, rxvt-unicode, auditd, the Deliantra MORPG Server and Client, and many other programs.

*homepage*: http://software.schmorp.de/pkg/libev.html

version |toolchain
--------|------------------------------
``4.33``|``GCC/11.2.0``, ``GCC/11.3.0``

<a anchor="libevent"/>
### *libevent*

The libevent API provides a mechanism to execute a callback function when a specific  event occurs on a file descriptor or after a timeout has been reached.  Furthermore, libevent also support callbacks due to signals or regular timeouts.

*homepage*: http://libevent.org/

version   |toolchain
----------|--------------------------------------------------------------------------------------------------------------
``2.0.22``|``GCC/4.9.2``, ``GCC/5.4.0-2.26``, ``GCCcore/4.9.3``, ``GNU/4.9.3-2.25``
``2.1.8`` |``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``system``
``2.1.11``|``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``2.1.12``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``system``

<a anchor="libexif"/>
### *libexif*

A library for parsing, editing, and saving EXIF data.

*homepage*: https://libexif.github.io/

version   |toolchain
----------|------------------
``0.6.24``|``GCCcore/11.2.0``

<a anchor="libfabric"/>
### *libfabric*

Libfabric is a core component of OFI. It is the library that defines and exports the user-space API of OFI, and is typically the only software that applications deal with directly. It works in conjunction with provider libraries, which are often integrated directly into libfabric. 

*homepage*: https://ofiwg.github.io/libfabric/

version   |toolchain
----------|-------------------------------------
``1.9.1`` |``GCCcore/9.3.0``
``1.10.1``|``GCCcore/9.3.0``
``1.11.0``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``1.12.1``|``GCCcore/10.3.0``
``1.13.0``|``GCCcore/11.2.0``
``1.13.1``|``GCCcore/11.2.0``
``1.13.2``|``GCCcore/11.2.0``
``1.15.1``|``GCCcore/11.3.0``
``1.16.1``|``GCCcore/12.2.0``

<a anchor="libfdf"/>
### *libfdf*

LibFDF provides a Fortran API to parse files in the  Flexible Data Format (FDF).

*homepage*: https://launchpad.net/libfdf

version  |versionsuffix|toolchain
---------|-------------|-----------------------------------------------------------------------------------------------------------------------------------
``0.2.2``|``-serial``  |``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``iccifort/2020.4.304``, ``intel-compilers/2021.2.0``, ``intel-compilers/2021.4.0``

<a anchor="libffcall"/>
### *libffcall*

GNU Libffcall is a collection of four libraries which can be used to build  foreign function call interfaces in embedded interpreters 

*homepage*: https://www.gnu.org/software/libffcall/

version |toolchain
--------|------------------------------------
``1.13``|``GCCcore/6.4.0``
``2.2`` |``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``2.4`` |``GCCcore/10.2.0``

<a anchor="libffi"/>
### *libffi*

The libffi library provides a portable, high level programming interface to  various calling conventions. This allows a programmer to call any function  specified by a call interface description at run-time.

*homepage*: https://sourceware.org/libffi/

version  |toolchain
---------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``3.2.1``|``GCC/4.9.2``, ``GCC/4.9.3-2.25``, ``GCC/5.4.0-2.26``, ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GNU/4.9.3-2.25``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``, ``system``
``3.3``  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/9.3.0``
``3.4.2``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``3.4.4``|``GCCcore/12.2.0``

<a anchor="libflame"/>
### *libFLAME*

libFLAME is a portable library for dense matrix computations, providing much of the functionality present in LAPACK.

*homepage*: https://developer.amd.com/amd-cpu-libraries/blas-library/#libflame

version  |versionsuffix|toolchain
---------|-------------|-------------------------------------------------------------------------
``1.0``  |``-amd``     |``GCC/7.3.0-2.30``
``2.2``  |``-amd``     |``GCCcore/9.3.0``
``5.2.0``|             |``GCC/10.3.0``, ``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/9.3.0``

<a anchor="libfontenc"/>
### *libfontenc*

X11 font encoding library

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs/

version  |toolchain
---------|-------------------------------------------------
``1.1.3``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``

<a anchor="libgcrypt"/>
### *libgcrypt*

Libgcrypt is a general purpose cryptographic library originally based on code from GnuPG

*homepage*: https://gnupg.org/related_software/libgcrypt/index.html

version  |toolchain
---------|--------------------------------------
``1.6.5``|``intel/2016a``
``1.8.4``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``1.8.5``|``GCCcore/8.3.0``
``1.9.2``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``1.9.3``|``GCCcore/11.2.0``

<a anchor="libgd"/>
### *libgd*

GD is an open source code library for the dynamic creation of images by programmers.

*homepage*: https://libgd.github.io

version  |toolchain
---------|------------------------------------------------------------------------------------------------------------
``2.1.1``|``foss/2016a``, ``intel/2016a``
``2.2.3``|``foss/2016b``, ``intel/2016b``
``2.2.4``|``GCCcore/6.4.0``, ``foss/2016b``, ``intel/2017a``
``2.2.5``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``intel/2017b``, ``intel/2018a``
``2.3.0``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``2.3.1``|``GCCcore/10.3.0``
``2.3.3``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="libgdiplus"/>
### *libgdiplus*

Libgdiplus is the Mono library that provides a GDI+-compatible  API on non-Windows operating systems.

*homepage*: https://www.mono-project.com/docs/gui/libgdiplus

version|toolchain
-------|------------------
``6.1``|``GCCcore/11.2.0``

<a anchor="libgdsii"/>
### *libGDSII*

libGDSII is a C++ library for working with GDSII binary data files,  intended primarily for use with the computational electromagnetism codes  scuff-em and meep but sufficiently general-purpose to allow other uses as well.

*homepage*: https://github.com/HomerReid/libGDSII

version |toolchain
--------|-----------------
``0.21``|``GCCcore/6.4.0``

<a anchor="libgeotiff"/>
### *libgeotiff*

Library for reading and writing coordinate system information from/to GeoTIFF files

*homepage*: https://directory.fsf.org/wiki/Libgeotiff

version  |toolchain
---------|-------------------------------------------------------
``1.4.2``|``foss/2018a``, ``foss/2018b``, ``intel/2018b``
``1.5.1``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.6.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``1.7.0``|``GCCcore/11.2.0``
``1.7.1``|``GCCcore/11.3.0``

<a anchor="libgit2"/>
### *libgit2*

libgit2 is a portable, pure C implementation of the Git core methods provided as a re-entrant linkable library with a solid API, allowing you to write native speed custom Git applications in any language which supports C bindings.

*homepage*: https://libgit2.org/

version  |toolchain
---------|---------------------------------------------------------
``1.0.0``|``GCCcore/8.3.0``
``1.1.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/9.3.0``
``1.1.1``|``GCCcore/11.2.0``
``1.4.3``|``GCCcore/11.3.0``

<a anchor="libglade"/>
### *libglade*

Libglade is a library for constructing user interfaces dynamically from XML descriptions.

*homepage*: https://developer.gnome.org/libglade/

version  |toolchain
---------|-----------------------------------------------------------------
``2.6.4``|``foss/2018b``, ``intel/2016a``, ``intel/2017b``, ``intel/2018a``

<a anchor="libglu"/>
### *libGLU*

The OpenGL Utility Library (GLU) is a computer graphics library for OpenGL. 

*homepage*: https://mesa.freedesktop.org/archive/glu/

version  |versionsuffix   |toolchain
---------|----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``9.0.0``|                |``GCCcore/8.2.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``fosscuda/2017b``, ``fosscuda/2018a``, ``fosscuda/2018b``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``, ``intelcuda/2017b``, ``iomkl/2018a``
``9.0.0``|``-Mesa-11.2.1``|``foss/2016a``, ``intel/2016a``
``9.0.1``|                |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``9.0.2``|                |``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``

<a anchor="libglvnd"/>
### *libglvnd*

libglvnd is a vendor-neutral dispatch layer for arbitrating OpenGL API calls between multiple vendors.

*homepage*: https://gitlab.freedesktop.org/glvnd/libglvnd

version  |toolchain
---------|-------------------------------------------------------
``1.2.0``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.3.2``|``GCCcore/10.2.0``
``1.3.3``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.4.0``|``GCCcore/11.3.0``
``1.6.0``|``GCCcore/12.2.0``

<a anchor="libgpg-error"/>
### *libgpg-error*

Libgpg-error is a small library that defines common error values for all GnuPG components.

*homepage*: https://gnupg.org/related_software/libgpg-error/index.html

version |toolchain
--------|--------------------------------------
``1.21``|``intel/2016a``
``1.35``|``GCCcore/7.3.0``
``1.36``|``GCCcore/8.2.0``
``1.38``|``GCCcore/8.3.0``
``1.41``|``GCCcore/10.2.0``
``1.42``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.46``|``GCCcore/11.3.0``

<a anchor="libgpuarray"/>
### *libgpuarray*

Library to manipulate tensors on the GPU.

*homepage*: http://deeplearning.net/software/libgpuarray/

version  |versionsuffix     |toolchain
---------|------------------|--------------------------------------------------------
``0.7.5``|``-Python-2.7.14``|``fosscuda/2017b``, ``intelcuda/2017b``
``0.7.5``|``-Python-3.6.3`` |``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``0.7.6``|                  |``fosscuda/2019a``, ``fosscuda/2020b``
``0.7.6``|``-Python-2.7.15``|``fosscuda/2018b``
``0.7.6``|``-Python-3.6.6`` |``fosscuda/2018b``
``0.7.6``|``-Python-3.7.4`` |``fosscuda/2019b``

<a anchor="libgridxc"/>
### *libGridXC*

A library to compute the exchange and correlation energy  and potential in spherical (i.e. atoms) or periodic systems.

*homepage*: https://gitlab.com/siesta-project/libraries/libgridxc

version  |toolchain
---------|----------------------------------------------------------------------------------------------------
``0.8.5``|``iimpi/2019b``
``0.9.6``|``gompi/2020b``, ``gompi/2021a``, ``gompi/2021b``, ``iimpi/2020b``, ``iimpi/2021a``, ``iimpi/2021b``

<a anchor="libgtextutils"/>
### *libgtextutils*

ligtextutils is a dependency of fastx-toolkit and is provided via the same upstream

*homepage*: http://hannonlab.cshl.edu/fastx_toolkit/

version|toolchain
-------|------------------------------------------------------------------
``0.7``|``GCCcore/7.3.0``, ``foss/2016a``, ``foss/2016b``, ``intel/2018a``

<a anchor="libharu"/>
### *libharu*

libHaru is a free, cross platform, open source library for generating PDF files.

*homepage*: http://libharu.org/

version  |toolchain
---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.3.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2021a``, ``foss/2021b``, ``intel/2017a``, ``intel/2018b``

<a anchor="libheif"/>
### *libheif*

libheif is an HEIF and AVIF file format decoder and encoder

*homepage*: https://github.com/strukturag/libheif

version   |toolchain
----------|------------------------------
``1.12.0``|``GCC/10.3.0``, ``GCC/11.2.0``

<a anchor="libibmad"/>
### *libibmad*

libibmad is a convenience library to encode, decode, and dump IB MAD packets. It  is implemented on top of and in conjunction with libibumad (the umad kernel  interface library.)

*homepage*: http://www.openfabrics.org

version   |toolchain
----------|------------------
``1.3.12``|``GCC/4.9.3-2.25``

<a anchor="libibumad"/>
### *libibumad*

libibumad is the umad kernel interface library.

*homepage*: http://www.openfabrics.org

version     |toolchain
------------|------------------
``1.3.10.2``|``GCC/4.9.3-2.25``

<a anchor="libice"/>
### *libICE*

X Inter-Client Exchange library for freedesktop.org

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version  |toolchain
---------|-------------------------------------------------
``1.0.9``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``

<a anchor="libiconv"/>
### *libiconv*

Libiconv converts from one character encoding to another through Unicode conversion

*homepage*: https://www.gnu.org/software/libiconv

version |toolchain
--------|-------------------------------------------------------------------------------------------------------------------
``1.15``|``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``
``1.16``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.17``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``

<a anchor="libidn"/>
### *libidn*

GNU Libidn is a fully documented implementation of the Stringprep, Punycode and IDNA specifications. Libidn's purpose is to encode and decode internationalized domain names.

*homepage*: http://www.gnu.org/software/libidn

version |toolchain
--------|--------------------------------------------------------------------------------------
``1.32``|``GCCcore/5.4.0``, ``GNU/4.9.3-2.25``, ``foss/2016a``, ``foss/2016b``, ``intel/2016a``
``1.34``|``GCCcore/6.4.0``
``1.35``|``GCCcore/7.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.36``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``1.38``|``GCCcore/11.2.0``
``1.41``|``GCCcore/11.3.0``

<a anchor="libidn2"/>
### *libidn2*

Libidn2 implements the revised algorithm for internationalized domain names called IDNA2008/TR46.

*homepage*: http://www.gnu.org/software/libidn2

version  |toolchain
---------|-------------------------------------------------------------------------------------------------------------------------------------
``2.3.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``2.3.2``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="libint"/>
### *Libint*

Libint library is used to evaluate the traditional (electron repulsion) and certain novel two-body  matrix elements (integrals) over Cartesian Gaussian functions used in modern atomic and molecular theory.

*homepage*: https://github.com/evaleev/libint

version  |versionsuffix   |toolchain
---------|----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.1.4``|                |``intel/2016a``
``1.1.6``|                |``GCC/8.2.0-2.31.1``, ``foss/2016b``, ``foss/2018a``, ``foss/2020a``, ``foss/2020b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2016b``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``, ``intel/2020a``
``2.0.3``|                |``foss/2018b``, ``gompi/2019a``, ``intel/2018b``
``2.1.0``|                |``intel/2016b``
``2.4.2``|                |``intel/2018a``
``2.5.0``|                |``gompi/2019a``, ``iimpi/2019a``
``2.6.0``|``-lmax-6-cp2k``|``GCC/10.2.0``, ``GCC/10.3.0``, ``gompi/2020a``, ``iccifort/2020.4.304``, ``iimpi/2020a``, ``iimpi/2021a``
``2.7.2``|``-lmax-6-cp2k``|``GCC/11.3.0``

<a anchor="libis"/>
### *LiBis*

An ultrasensitive alignment method for low input bisulfite sequencing

*homepage*: https://github.com/Dangertrip/LiBis

version     |versionsuffix    |toolchain
------------|-----------------|--------------
``20200428``|``-Python-3.7.4``|``foss/2019b``

<a anchor="libjpeg-turbo"/>
### *libjpeg-turbo*

libjpeg-turbo is a fork of the original IJG libjpeg which uses SIMD to  accelerate baseline JPEG compression and decompression. libjpeg is a library  that implements JPEG image encoding, decoding and transcoding. 

*homepage*: https://sourceforge.net/projects/libjpeg-turbo/

version  |versionsuffix    |toolchain
---------|-----------------|------------------------------------------------------------------
``1.4.2``|                 |``foss/2016a``, ``foss/2016b``, ``intel/2016a``
``1.4.2``|``-NASM-2.12.01``|``foss/2016a``, ``intel/2016a``
``1.5.0``|                 |``GCCcore/5.4.0``, ``foss/2016a``, ``foss/2016b``, ``intel/2016b``
``1.5.1``|                 |``foss/2016b``, ``intel/2016b``, ``intel/2017a``
``1.5.2``|                 |``GCCcore/6.3.0``, ``GCCcore/6.4.0``
``1.5.3``|                 |``GCCcore/6.4.0``
``2.0.0``|                 |``GCCcore/7.3.0``
``2.0.2``|                 |``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``2.0.3``|                 |``GCCcore/8.3.0``
``2.0.4``|                 |``GCCcore/9.3.0``
``2.0.5``|                 |``GCCcore/10.2.0``
``2.0.6``|                 |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``2.1.3``|                 |``GCCcore/11.3.0``
``2.1.4``|                 |``GCCcore/12.2.0``

<a anchor="libjxl"/>
### *libjxl*

JPEG XL image format reference implementation

*homepage*: https://github.com/libjxl/libjxl

version  |toolchain
---------|------------------
``0.5``  |``GCCcore/10.3.0``
``0.6.1``|``GCCcore/10.2.0``

<a anchor="liblzf"/>
### *LibLZF*

LibLZF is a very small data compression library. It consists of only two .c and two .h files and is very easy to incorporate into your own programs. The compression algorithm is very, very fast, yet still written in portable C.

*homepage*: http://oldhome.schmorp.de/marc/liblzf.html

version|toolchain
-------|------------------
``3.4``|``GCCcore/10.2.0``
``3.6``|``GCCcore/10.3.0``

<a anchor="libmad"/>
### *libmad*

MAD is a high-quality MPEG audio decoder.

*homepage*: https://www.underbit.com/products/mad/

version    |toolchain
-----------|------------------
``0.15.1b``|``GCCcore/11.3.0``

<a anchor="libmatheval"/>
### *libmatheval*

GNU libmatheval is a library (callable from C and Fortran) to parse  and evaluate symbolic expressions input as text.

*homepage*: http://www.gnu.org/software/libmatheval/

version   |toolchain
----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.1.11``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``foss/2016b``, ``foss/2017a``, ``intel/2016a``, ``intel/2016b``

<a anchor="libmaus2"/>
### *libmaus2*

libmaus2 is a collection of data structures and algorithms.

*homepage*: https://github.com/gt1/libmaus2

version    |toolchain
-----------|---------------
``2.0.453``|``intel/2018a``

<a anchor="libmbd"/>
### *libmbd*

Libmbd implements the many-body dispersion (MBD) method in several programming languages and frameworks:   - The Fortran implementation is the reference, most advanced implementation, with support for analytical     gradients and distributed parallelism, and additional functionality beyond the MBD method itself.     It provides a low-level and a high-level Fortran API, as well as a C API. Furthermore, Python bindings     to the C API are provided.  - The Python/Numpy implementation is intended for prototyping, and as a high-level language reference.  - The Python/Tensorflow implementation is an experiment that should enable rapid prototyping of machine     learning applications with MBD.  The Python-based implementations as well as Python bindings to the Libmbd C API are accessible from the  Python package called Pymbd. 

*homepage*: https://libmbd.github.io/index.html

version   |toolchain
----------|----------------------------------------------------------------
``0.10.4``|``foss/2020b``, ``foss/2021a``, ``intel/2020b``, ``intel/2021a``

<a anchor="libmemcached"/>
### *libMemcached*

libMemcached is an open source C/C++ client library and tools for  the memcached server (http://danga.com/memcached). It has been designed to be light  on memory usage, thread safe, and provide full access to server side methods.

*homepage*: https://memcached.org

version   |toolchain
----------|------------------------------------
``1.0.18``|``GCCcore/6.4.0``, ``GCCcore/9.3.0``

<a anchor="libmicrohttpd"/>
### *libmicrohttpd*

GNU libmicrohttpd is a small C library that is supposed to make it easy to run  an HTTP server as part of another application. 

*homepage*: https://www.gnu.org/software/libmicrohttpd/

version   |toolchain
----------|--------------------------------------------------------
``0.9.71``|``GCCcore/9.3.0``
``0.9.73``|``GCCcore/10.2.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``

<a anchor="libmo_unpack"/>
### *libmo_unpack*

A library for handling the WGDOS and RLE compression schemes used in UM files.

*homepage*: https://github.com/SciTools/libmo_unpack

version  |toolchain
---------|-----------------
``3.1.2``|``GCCcore/6.4.0``

<a anchor="libmypaint"/>
### *libmypaint*

libmypaint, a.k.a. "brushlib", is a library for making brushstrokes which is used by MyPaint  and other projects

*homepage*: https://github.com/mypaint/libmypaint

version  |toolchain
---------|------------------
``1.6.1``|``GCCcore/10.3.0``

<a anchor="libnsl"/>
### *libnsl*

The libnsl package contains the public client interface for NIS(YP).

*homepage*: https://github.com/thkukuk/libnsl

version  |toolchain
---------|------------------
``1.3.0``|``GCCcore/10.3.0``

<a anchor="libobjcryst"/>
### *libobjcryst*

ObjCryst++ is Object-Oriented Crystallographic Library for C++

*homepage*: https://github.com/diffpy/libobjcryst

version     |toolchain
------------|---------------
``2017.2.3``|``intel/2020a``
``2021.1.2``|``foss/2021b``

<a anchor="libogg"/>
### *libogg*

Ogg is a multimedia container format, and the native file and stream format for the Xiph.org multimedia codecs.

*homepage*: https://xiph.org/ogg/

version  |toolchain
---------|--------------------------------------
``1.3.4``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``1.3.5``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="libopus"/>
### *libopus*

Opus is a totally open, royalty-free, highly versatile audio codec. Opus is unmatched for interactive  speech and music transmission over the Internet, but is also intended for storage and streaming applications. It is  standardized by the Internet Engineering Task Force (IETF) as RFC 6716 which incorporated technology from Skype’s  SILK codec and Xiph.Org’s CELT codec.

*homepage*: https://www.opus-codec.org/

version  |toolchain
---------|------------------
``1.3.1``|``GCCcore/11.3.0``

<a anchor="libosmium"/>
### *libosmium*

A fast and flexible C++ library for working with OpenStreetMap data.  The Osmium Library has extensive support for all types of OSM entities:  nodes, ways, relations, and changesets.  It allows reading from and writing to OSM files in XML and PBF formats, including change files and full history files.  Osmium can store OSM data in memory and on disk in various formats and using various indexes.  Its easy to use handler interface allows you to quickly write data filtering and conversion functions.  Osmium can create WKT, WKB, OGR, GEOS and GeoJSON geometries for easy conversion into many GIS formats  and it can assemble multipolygons from ways and relations.

*homepage*: https://osmcode.org/libosmium/

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``2.15.4``|``-Python-3.6.6``|``foss/2018b``
``2.15.6``|``-Python-3.7.4``|``foss/2019b``

<a anchor="libpci"/>
### *libpci*

Library for portable access to PCI bus configuration registers from PCI Utils.

*homepage*: https://github.com/pciutils/pciutils

version  |toolchain
---------|-----------------------------------------------------------------------------
``3.7.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/6.4.0``

<a anchor="libpciaccess"/>
### *libpciaccess*

Generic PCI access library.

*homepage*: https://cgit.freedesktop.org/xorg/lib/libpciaccess/

version   |toolchain
----------|--------------------------------------------------------------------------------------------------------------------
``0.13.4``|``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``
``0.14``  |``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``0.16``  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/9.2.0``, ``GCCcore/9.3.0``
``0.17``  |``GCCcore/12.2.0``

<a anchor="libplinkio"/>
### *libplinkio*

A small C and Python library for reading PLINK genotype files.

*homepage*: https://github.com/mfranberg/libplinkio

version  |toolchain
---------|-----------------
``0.9.8``|``GCCcore/9.3.0``

<a anchor="libpng"/>
### *libpng*

libpng is the official PNG reference library

*homepage*: http://www.libpng.org/pub/png/libpng.html

version   |toolchain
----------|--------------------------------------------------------------------------------------------------------------------
``1.2.58``|``system``
``1.2.59``|``system``
``1.5.30``|``system``
``1.6.21``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``1.6.23``|``foss/2016a``, ``foss/2016b``, ``intel/2016b``
``1.6.24``|``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``foss/2016b``, ``intel/2016b``
``1.6.26``|``foss/2016b``, ``intel/2016b``
``1.6.27``|``intel/2016b``
``1.6.28``|``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``gimkl/2017a``
``1.6.29``|``GCCcore/6.3.0``
``1.6.32``|``GCCcore/6.4.0``
``1.6.34``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``1.6.36``|``GCCcore/8.2.0``
``1.6.37``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.6.38``|``GCCcore/12.2.0``

<a anchor="libpsl"/>
### *libpsl*

C library for the Public Suffix List

*homepage*: https://rockdaboot.github.io/libpsl

version   |toolchain
----------|------------------------------------------------------------------------------
``0.20.2``|``GCCcore/7.3.0``
``0.21.0``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``0.21.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="libpsml"/>
### *libPSML*

A library to handle PSML, the pseudopotential markup language.

*homepage*: https://launchpad.net/libpsml

version  |toolchain
---------|-----------------------
``1.1.8``|``iccifort/2019.5.281``

<a anchor="libpsml"/>
### *libpsml*

LibPSML provides a Fortran API to parse files in the  PSeudopotential Markup Language (PSML) format.

*homepage*: https://launchpad.net/libpsml

version   |toolchain
----------|-----------------------------------------------------------------------------------------------------------------------------------
``1.1.7`` |``foss/2016b``, ``foss/2017a``
``1.1.10``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``iccifort/2020.4.304``, ``intel-compilers/2021.2.0``, ``intel-compilers/2021.4.0``

<a anchor="libpsortb"/>
### *libpsortb*

PSORT family of programs for subcellular localization prediction as well as other datasets and  resources relevant to localization prediction.

*homepage*: http://psort.org/

version|toolchain
-------|--------------
``1.0``|``foss/2016a``

<a anchor="libpspio"/>
### *libpspio*

libpspio is a library to read and write pseudopotentials in multiple formats.

*homepage*: https://gitlab.e-cam2020.eu/esl/pspio/

version  |toolchain
---------|-------------------------------------------------------
``0.2.4``|``GCC/10.2.0``, ``GCC/10.3.0``, ``iccifort/2020.4.304``

<a anchor="libpthread-stubs"/>
### *libpthread-stubs*

The X protocol C-language Binding (XCB) is a replacement for Xlib featuring  a small footprint, latency hiding, direct access to the protocol, improved  threading support, and extensibility. 

*homepage*: http://xcb.freedesktop.org/

version|toolchain
-------|-----------------------------------------------------------------------------------------------------
``0.3``|``GCCcore/6.4.0``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``
``0.4``|``GCCcore/6.4.0``

<a anchor="libqglviewer"/>
### *libQGLViewer*

libQGLViewer is a C++ library based on Qt that eases the creation of OpenGL 3D viewers.

*homepage*: http://libqglviewer.com/

version  |versionsuffix   |toolchain
---------|----------------|-----------------------------------------------
``2.6.3``|                |``foss/2016a``, ``foss/2016b``, ``intel/2016b``
``2.6.3``|``-Mesa-11.2.1``|``foss/2016a``, ``intel/2016a``
``2.6.4``|                |``intel/2016b``
``2.7.1``|                |``intel/2018a``
``2.8.0``|                |``GCCcore/10.3.0``

<a anchor="libreadline"/>
### *libreadline*

The GNU Readline library provides a set of functions for use by applications  that allow users to edit command lines as they are typed in. Both Emacs and  vi editing modes are available. The Readline library includes additional  functions to maintain a list of previously-entered command lines, to recall  and perhaps reedit those lines, and perform csh-like history expansion on  previous commands. 

*homepage*: https://tiswww.case.edu/php/chet/readline/rltop.html

version  |toolchain
---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``6.3``  |``GCC/4.8.2``, ``GCC/4.8.4``, ``GCC/4.9.2``, ``GCC/4.9.3-2.25``, ``GCC/5.4.0-2.26``, ``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GNU/4.9.3-2.25``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``gimkl/2017a``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
``7.0``  |``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``
``8.0``  |``GCCcore/10.2.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``system``
``8.1``  |``FCC/4.5.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``8.1.2``|``GCCcore/11.3.0``, ``GCCcore/12.1.0``
``8.2``  |``GCCcore/12.2.0``

<a anchor="librmath"/>
### *libRmath*

The routines supporting the distribution and special functions in R and a few others are declared  in C header file Rmath.h. These can be compiled into a standalone library for linking to other applications.

*homepage*: https://cran.r-project.org/doc/manuals/r-release/R-admin.html#The-standalone-Rmath-library

version  |toolchain
---------|------------------
``3.6.0``|``foss/2018b``
``4.0.0``|``GCCcore/9.3.0``
``4.1.0``|``GCCcore/10.2.0``
``4.1.2``|``GCCcore/11.2.0``
``4.2.0``|``GCCcore/10.3.0``

<a anchor="librosa"/>
### *librosa*

Python module for audio and music processing

*homepage*: https://librosa.github.io

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------
``0.7.2``|``-Python-3.7.4``|``foss/2019b``, ``fosscuda/2019b``

<a anchor="librsb"/>
### *librsb*

A shared memory parallel sparse matrix computations library for the Recursive Sparse Blocks format

*homepage*: https://sourceforge.net/projects/librsb/

version     |toolchain
------------|--------------
``1.2.0.9`` |``GCC/10.2.0``
``1.2.0.10``|``GCC/10.2.0``
``1.2.0.11``|``GCC/10.2.0``
``1.3.0.0`` |``GCC/10.2.0``
``1.3.0.1`` |``GCC/10.2.0``

<a anchor="librsvg"/>
### *librsvg*

Librsvg is a library to render SVG files using cairo.

*homepage*: https://wiki.gnome.org/Projects/LibRsvg

version    |toolchain
-----------|------------------
``2.40.15``|``intel/2016a``
``2.48.4`` |``foss/2019a``
``2.51.2`` |``GCCcore/10.3.0``
``2.52.8`` |``GCCcore/11.2.0``
``2.55.1`` |``GCCcore/11.3.0``

<a anchor="libsamplerate"/>
### *libsamplerate*

Secret Rabbit Code (aka libsamplerate) is a Sample Rate Converter for audio.

*homepage*: http://www.mega-nerd.com/libsamplerate

version  |toolchain
---------|-----------------
``0.1.9``|``GCCcore/8.2.0``

<a anchor="libsbml"/>
### *libSBML*

libSBML (Systems Biology Markup Language library) is a free, open-source programming library to help you read, write, manipulate, translate, and validate SBML files and data streams. It's not an application itself (though it does come with example programs), but rather a library you embed in your own applications.

*homepage*: http://sbml.org/Software/libSBML

version   |toolchain
----------|--------------
``5.19.0``|``GCC/10.3.0``

<a anchor="libsigc++"/>
### *libsigc++*

The libsigc++ package implements a typesafe callback system for standard C++.

*homepage*: https://libsigcplusplus.github.io/libsigcplusplus/

version   |toolchain
----------|------------------------------------
``2.10.0``|``GCCcore/6.4.0``
``2.10.1``|``GCCcore/7.3.0``
``2.10.2``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``2.10.8``|``GCCcore/10.3.0``

<a anchor="libsigsegv"/>
### *libsigsegv*

GNU libsigsegv is a library for handling page faults in user mode.

*homepage*: https://www.gnu.org/software/libsigsegv/

version |toolchain
--------|------------------
``2.11``|``GCCcore/6.4.0``
``2.12``|``GCCcore/9.3.0``
``2.13``|``GCCcore/10.2.0``

<a anchor="libsm"/>
### *libSM*

X11 Session Management library, which allows for applications to both manage sessions,  and make use of session managers to save and restore their state for later use.

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version  |toolchain
---------|-------------------------------------------------
``1.2.2``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``

<a anchor="libsndfile"/>
### *libsndfile*

Libsndfile is a C library for reading and writing files containing sampled sound  (such as MS Windows WAV and the Apple/SGI AIFF format) through one standard library interface.

*homepage*: http://www.mega-nerd.com/libsndfile

version   |toolchain
----------|----------------------------------------------------------------------------------------------------------------------------------
``1.0.28``|``GCCcore/10.2.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``intel/2017a``
``1.0.31``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.1.0`` |``GCCcore/11.3.0``

<a anchor="libsodium"/>
### *libsodium*

Sodium is a modern, easy-to-use software library for encryption, decryption, signatures,  password hashing and more.

*homepage*: https://doc.libsodium.org/

version   |toolchain
----------|--------------------------------------------------------------------------------------------------------------------
``1.0.6`` |``intel/2016a``
``1.0.8`` |``foss/2016a``
``1.0.11``|``foss/2016b``, ``intel/2016b``
``1.0.12``|``GCCcore/6.4.0``, ``intel/2017a``
``1.0.13``|``GCCcore/6.4.0``, ``foss/2017a``
``1.0.16``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``1.0.17``|``GCCcore/8.2.0``
``1.0.18``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

<a anchor="libsoup"/>
### *LibSoup*

libsoup is an HTTP client/server library for GNOME. It uses GObjects and the glib main loop, to integrate well with GNOME applications, and also has a synchronous API, for use in threaded applications.

*homepage*: https://wiki.gnome.org/Projects/libsoup

version   |toolchain
----------|------------------
``2.66.1``|``GCCcore/8.2.0``
``2.70.0``|``GCCcore/8.3.0``
``2.72.0``|``GCCcore/10.2.0``
``2.74.0``|``GCCcore/10.3.0``
``3.0.7`` |``GCC/11.2.0``

<a anchor="libspatialindex"/>
### *libspatialindex*

C++ implementation of R*-tree, an MVR-tree and a TPR-tree with C API

*homepage*: https://libspatialindex.org

version  |toolchain
---------|--------------------------------------------------------------------------------------
``1.8.5``|``GCCcore/6.4.0``, ``GCCcore/8.2.0``, ``foss/2016b``, ``intel/2016b``, ``intel/2018a``
``1.9.3``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="libspatialite"/>
### *libspatialite*

SpatiaLite is an open source library intended to extend the SQLite core to support  fully fledged Spatial SQL capabilities.

*homepage*: https://www.gaia-gis.it/fossil/libspatialite/home

version   |versionsuffix    |toolchain
----------|-----------------|-----------------------------------------------
``4.3.0a``|                 |``foss/2016b``, ``foss/2018b``, ``intel/2016b``
``4.3.0a``|``-Python-3.7.2``|``foss/2019a``
``4.3.0a``|``-Python-3.7.4``|``GCC/8.3.0``

<a anchor="libssh"/>
### *libssh*

Multiplatform C library implementing the SSHv2 protocol on client and server side

*homepage*: https://www.libssh.org

version  |toolchain
---------|-----------------
``0.9.0``|``GCCcore/6.4.0``

<a anchor="libstatgen"/>
### *libStatGen*

Useful set of classes for creating statistical genetic programs.

*homepage*: https://genome.sph.umich.edu/wiki/C++_Library:_libStatGen

version     |toolchain
------------|------------------
``1.0.15``  |``GCCcore/10.2.0``
``20190330``|``GCCcore/6.4.0``

<a anchor="libsvm"/>
### *LIBSVM*

LIBSVM is an integrated software for support vector classification, (C-SVC, nu-SVC), regression  (epsilon-SVR, nu-SVR) and distribution estimation (one-class SVM). It supports multi-class classification.

*homepage*: https://www.csie.ntu.edu.tw/~cjlin/libsvm/

version |toolchain
--------|----------------------------------------------------------
``3.22``|``intel/2016b``, ``intel/2017b``
``3.23``|``foss/2018b``, ``intel/2018b``
``3.24``|``GCCcore/9.3.0``
``3.25``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``3.30``|``GCCcore/11.3.0``

<a anchor="libtar"/>
### *libtar*

C library for manipulating POSIX tar files

*homepage*: https://repo.or.cz/libtar.git

version   |toolchain
----------|------------------------------------
``1.2.20``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``

<a anchor="libtasn1"/>
### *libtasn1*

Libtasn1 is the ASN.1 library used by GnuTLS, GNU Shishi and some other packages.  It was written by Fabio Fiorina, and has been shipped as part of GnuTLS  for some time but is now a proper GNU package.

*homepage*: https://www.gnu.org/software/libtasn1/

version   |toolchain
----------|---------------------------------------------------
``4.7``   |``GNU/4.9.3-2.25``, ``foss/2016a``, ``intel/2016a``
``4.12``  |``GCCcore/5.4.0``
``4.13``  |``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``4.16.0``|``GCCcore/10.2.0``, ``GCCcore/8.3.0``
``4.17.0``|``GCCcore/10.3.0``
``4.18.0``|``GCCcore/11.2.0``
``4.19.0``|``GCCcore/11.3.0``

<a anchor="libtecla"/>
### *libtecla*

The tecla library provides UNIX and LINUX programs with interactive command line editing facilities,  similar to those of the UNIX tcsh shell. In addition to simple command-line editing, it supports recall of previously  entered command lines, TAB completion of file names or other tokens, and in-line wild-card expansion of filenames.  The internal functions which perform file-name completion and wild-card expansion are also available externally for  optional use by programs. 

*homepage*: https://sites.astro.caltech.edu/~mcs/tecla/

version  |toolchain
---------|------------------
``1.6.3``|``GCCcore/10.2.0``

<a anchor="libtiff"/>
### *LibTIFF*

tiff: Library and tools for reading and writing TIFF data files

*homepage*: https://libtiff.gitlab.io/libtiff/

version   |toolchain
----------|---------------------------------------------------------------------------------------------------------
``4.0.6`` |``GCCcore/5.4.0``, ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``4.0.7`` |``foss/2016b``, ``intel/2017a``
``4.0.8`` |``intel/2017a``, ``intel/2017b``
``4.0.9`` |``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``foss/2017b``, ``intel/2017b``, ``intel/2018.01``, ``intel/2018b``
``4.0.10``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``4.1.0`` |``GCCcore/10.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``4.2.0`` |``GCCcore/10.3.0``
``4.3.0`` |``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``4.4.0`` |``GCCcore/12.2.0``

<a anchor="libtirpc"/>
### *libtirpc*

Libtirpc is a port of Suns Transport-Independent RPC library to Linux.

*homepage*: https://sourceforge.net/projects/libtirpc/

version  |toolchain
---------|----------------------------------------------------------
``1.1.4``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``1.2.6``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.3.1``|``GCCcore/10.2.0``
``1.3.2``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="libtool"/>
### *libtool*

GNU libtool is a generic library support script. Libtool hides the complexity  of using shared libraries behind a consistent, portable interface. 

*homepage*: https://www.gnu.org/software/libtool

version  |toolchain
---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.4.2``|``GCC/4.8.2``, ``GCC/4.9.2``
``2.4.5``|``GCC/4.8.4``, ``GCC/4.9.2``
``2.4.6``|``FCC/4.5.0``, ``GCC/4.8.4``, ``GCC/4.9.2``, ``GCC/4.9.3``, ``GCC/4.9.3-2.25``, ``GCC/5.2.0``, ``GCC/5.4.0-2.26``, ``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.1.0``, ``GCCcore/6.2.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.2.0``, ``GCCcore/9.3.0``, ``GNU/4.9.2-2.25``, ``GNU/4.9.3-2.25``, ``GNU/5.1.0-2.25``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
``2.4.7``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``system``

<a anchor="libtree"/>
### *libtree*

libtree is a tool that turns ldd into a tree, explains why  shared libraries are found and why not and optionally deploys executables and  dependencies into a single directory

*homepage*: https://github.com/haampie/libtree

version  |toolchain
---------|------------------
``2.0.0``|``GCCcore/10.3.0``
``3.0.3``|``system``

<a anchor="libunistring"/>
### *libunistring*

This library provides functions for manipulating Unicode strings and for  manipulating C strings according to the Unicode standard.

*homepage*: https://www.gnu.org/software/libunistring/

version   |toolchain
----------|----------------------------------------------------------------------------------------------
``0.9.3`` |``GCC/4.9.3-2.25``, ``GNU/4.9.3-2.25``, ``foss/2016a``, ``intel/2016a``
``0.9.6`` |``GCCcore/5.4.0``, ``foss/2016b``, ``foss/2017a``, ``intel/2016b``
``0.9.7`` |``GCCcore/6.4.0``
``0.9.10``|``GCCcore/10.3.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.0``   |``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="libunwind"/>
### *libunwind*

The primary goal of libunwind is to define a portable and efficient C programming interface  (API) to determine the call-chain of a program. The API additionally provides the means to manipulate the  preserved (callee-saved) state of each call-frame and to resume execution at any point in the call-chain  (non-local goto). The API supports both local (same-process) and remote (across-process) operation.  As such, the API is useful in a number of applications

*homepage*: https://www.nongnu.org/libunwind/

version  |toolchain
---------|-----------------------------------------------------------------------
``1.1``  |``GCC/4.9.2``, ``foss/2016a``, ``intel/2016b``
``1.2.1``|``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``foss/2016b``
``1.3.1``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.4.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``1.5.0``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.6.2``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``

<a anchor="libutempter"/>
### *libutempter*

libutempter is library that provides an interface for terminal  emulators such as screen and xterm to record user sessions to utmp and wtmp files.

*homepage*: http://git.altlinux.org/people/ldv/packages/?p=libutempter.git

version    |toolchain
-----------|------------------
``1.1.6.2``|``GCC/6.4.0-2.28``

<a anchor="libuuid"/>
### *LibUUID*

Portable uuid C library

*homepage*: http://sourceforge.net/projects/libuuid/

version  |toolchain
---------|----------------------------------------------------------------------------------------------------------------------------
``1.0.3``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``foss/2016a``, ``intel/2017a``, ``intel/2017b``

<a anchor="libuv"/>
### *libuv*

libuv is a multi-platform support library with a focus on asynchronous I/O.

*homepage*: https://libuv.org

version   |toolchain
----------|-----------------
``1.37.0``|``GCCcore/8.3.0``

<a anchor="libvdwxc"/>
### *libvdwxc*

libvdwxc is a general library for evaluating energy and potential for exchange-correlation (XC) functionals from the vdW-DF family that can be used with various of density functional theory (DFT) codes.

*homepage*: https://libvdwxc.org

version  |toolchain
---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------
``0.3.2``|``foss/2018b``
``0.4.0``|``foss/2019a``, ``foss/2019b``, ``foss/2020a``, ``foss/2020b``, ``foss/2021a``, ``foss/2021b``, ``foss/2022a``, ``intel/2020b``, ``intel/2021a``, ``intel/2021b``

<a anchor="libvorbis"/>
### *libvorbis*

Ogg Vorbis is a fully open, non-proprietary, patent-and-royalty-free, general-purpose compressed audio format

*homepage*: https://xiph.org/vorbis/

version  |toolchain
---------|------------------------------------------------------------------------------
``1.3.7``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="libwebp"/>
### *libwebp*

WebP is a modern image format that provides superior lossless and lossy compression for images on the web. Using WebP, webmasters and web developers can create smaller, richer images that make the web faster.

*homepage*: https://developers.google.com/speed/webp/

version  |toolchain
---------|--------------------------------------------------------
``1.0.0``|``foss/2018b``
``1.0.2``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``1.1.0``|``GCCcore/10.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.2.0``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.2.4``|``GCCcore/11.3.0``

<a anchor="libwpe"/>
### *libwpe*

WPE is the reference WebKit port for embedded and low-consumption computer devices. It has been designed from the ground-up with performance, small footprint, accelerated content rendering, and simplicity of deployment in mind, bringing the excellence of the WebKit engine to countless platforms and target devices.

*homepage*: https://webkit.org/wpe

version   |toolchain
----------|------------------
``1.13.3``|``GCCcore/11.2.0``

<a anchor="libx11"/>
### *libX11*

X11 client-side library

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version  |toolchain
---------|-------------------------------------------------
``1.6.3``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``

<a anchor="libxau"/>
### *libXau*

The libXau package contains a library implementing the X11 Authorization Protocol. This is useful for restricting client access to the display.

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version  |toolchain
---------|--------------------------------------------------------------------
``1.0.8``|``GCCcore/6.4.0``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``

<a anchor="libxc"/>
### *libxc*

Libxc is a library of exchange-correlation functionals for density-functional theory.  The aim is to provide a portable, well tested and reliable set of exchange and correlation functionals.

*homepage*: https://www.tddft.org/programs/libxc

version  |toolchain
---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.2.2``|``GCC/8.2.0-2.31.1``, ``foss/2018b``, ``intel/2018b``
``2.2.3``|``foss/2016b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017b``, ``intel/2018a``
``3.0.0``|``GCC/5.4.0-2.26``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``
``3.0.1``|``GCC/8.2.0-2.31.1``, ``foss/2016b``, ``foss/2017a``, ``foss/2018a``, ``foss/2018b``, ``foss/2020a``, ``foss/2020b``, ``gimkl/2017a``, ``intel/2018a``, ``intel/2018b``, ``intel/2020a``
``4.0.1``|``foss/2017b``, ``intel/2017b``
``4.0.3``|``foss/2016b``, ``foss/2017a``
``4.2.3``|``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``gimkl/2017a``, ``intel/2018a``, ``intel/2018b``
``4.3.4``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/8.2.0-2.31.1``, ``GCC/8.3.0``, ``GCC/9.3.0``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``iccifort/2019.5.281``, ``iccifort/2020.1.217``, ``iccifort/2020.4.304``, ``intel-compilers/2021.2.0``, ``intel-compilers/2021.4.0``
``5.1.2``|``GCC/10.2.0``
``5.1.3``|``GCC/10.2.0``
``5.1.5``|``GCC/10.3.0``, ``intel-compilers/2021.2.0``
``5.1.6``|``GCC/11.2.0``, ``intel-compilers/2021.4.0``
``5.2.3``|``GCC/11.3.0``, ``intel-compilers/2022.1.0``

<a anchor="libxcb"/>
### *libxcb*

The X protocol C-language Binding (XCB) is a replacement for Xlib featuring a small footprint, latency hiding, direct access to the protocol, improved threading support, and extensibility.

*homepage*: https://xcb.freedesktop.org/

version   |toolchain
----------|-------------------------------------------------
``1.11.1``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``1.13``  |``GCCcore/6.4.0``

<a anchor="libxcursor"/>
### *libXcursor*

X Cursor management library

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version   |toolchain
----------|-------------------------------
``1.1.14``|``foss/2016a``, ``intel/2016a``

<a anchor="libxdamage"/>
### *libXdamage*

X Damage extension library

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version  |toolchain
---------|-------------------------------------------------
``1.1.4``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``

<a anchor="libxdmcp"/>
### *libXdmcp*

The libXdmcp package contains a library implementing the X Display Manager Control Protocol. This is useful for allowing clients to interact with the X Display Manager. 

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version  |toolchain
---------|--------------------------------------------------------------------
``1.1.2``|``GCCcore/6.4.0``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``

<a anchor="libxext"/>
### *libXext*

Common X Extensions library

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version  |toolchain
---------|-------------------------------------------------
``1.3.3``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``

<a anchor="libxfixes"/>
### *libXfixes*

X Fixes extension library

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version  |toolchain
---------|-------------------------------------------------
``5.0.1``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``5.0.2``|``intel/2016a``

<a anchor="libxfont"/>
### *libXfont*

X font libary

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version  |versionsuffix      |toolchain
---------|-------------------|-------------------------------------------------
``1.5.1``|                   |``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``1.5.1``|``-freetype-2.6.3``|``foss/2016a``, ``intel/2016a``

<a anchor="libxft"/>
### *libXft*

X11 client-side library

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version  |versionsuffix          |toolchain
---------|-----------------------|-------------------------------
``2.3.2``|                       |``foss/2016a``, ``intel/2016a``
``2.3.2``|``-fontconfig-2.11.95``|``intel/2016a``
``2.3.2``|``-freetype-2.6.3``    |``foss/2016a``

<a anchor="libxi"/>
### *libXi*

LibXi provides an X Window System client interface to the XINPUT extension to the X protocol.

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version  |toolchain
---------|-------------------------------
``1.7.6``|``foss/2016a``, ``intel/2016a``

<a anchor="libxinerama"/>
### *libXinerama*

Xinerama multiple monitor library

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version  |toolchain
---------|-------------------------------
``1.1.3``|``foss/2016a``, ``intel/2016a``

<a anchor="libxkbcommon"/>
### *libxkbcommon*

xkbcommon is a library to handle keyboard descriptions,  including loading them from disk, parsing them and handling their state.  It's mainly meant for client toolkits, window systems,  and other system applications.

*homepage*: http://xkbcommon.org/

version  |toolchain
---------|-------------------------------
``0.6.1``|``foss/2016a``, ``intel/2016a``

<a anchor="libxml++"/>
### *libxml++*

libxml++ is a C++ wrapper for the libxml XML parser library.

*homepage*: http://libxmlplusplus.sourceforge.net

version   |toolchain
----------|-------------------------------------------------------
``2.40.1``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``2.42.1``|``GCC/10.3.0``

<a anchor="libxml2"/>
### *libxml2*

Libxml2 is the XML C parser and toolchain developed for the Gnome project  (but usable outside of the Gnome platform). 

*homepage*: http://xmlsoft.org/

version   |versionsuffix     |toolchain
----------|------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.9.2`` |                  |``GCC/4.8.3``, ``GCC/4.8.4``, ``GCC/4.9.2``, ``GCC/4.9.3-2.25``, ``GNU/4.9.3-2.25``
``2.9.3`` |                  |``GCC/4.9.3-2.25``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``2.9.3`` |``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``2.9.4`` |                  |``GCC/5.4.0-2.26``, ``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2017a``, ``intel/2016b``
``2.9.4`` |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``2.9.4`` |``-Python-2.7.13``|``intel/2017a``
``2.9.5`` |                  |``GCCcore/6.3.0``
``2.9.6`` |                  |``GCCcore/6.4.0``
``2.9.7`` |                  |``GCCcore/6.4.0``
``2.9.8`` |                  |``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``2.9.9`` |                  |``GCCcore/8.3.0``
``2.9.10``|                  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.2.0``, ``GCCcore/9.3.0``
``2.9.13``|                  |``GCCcore/11.3.0``
``2.10.3``|                  |``GCCcore/12.2.0``

<a anchor="libxml2-python"/>
### *libxml2-python*

Libxml2 is the XML C parser and toolchain developed for the Gnome project  (but usable outside of the Gnome platform). This is the Python binding.

*homepage*: http://xmlsoft.org/

version   |versionsuffix     |toolchain
----------|------------------|------------------
``2.9.7`` |``-Python-2.7.14``|``intel/2018a``
``2.9.8`` |``-Python-2.7.15``|``foss/2018b``
``2.9.8`` |``-Python-3.7.2`` |``GCCcore/8.2.0``
``2.9.13``|                  |``GCCcore/11.3.0``

<a anchor="libxmu"/>
### *libXmu*

libXmu provides a set of miscellaneous utility convenience functions for X libraries to use.  libXmuu is a lighter-weight version that does not depend on libXt or libXext

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version  |toolchain
---------|-------------------------------
``1.1.2``|``foss/2016a``, ``intel/2016a``

<a anchor="libxp"/>
### *libXp*

libXp provides the X print library.

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version  |toolchain
---------|---------------
``1.0.3``|``intel/2016a``

<a anchor="libxpm"/>
### *libXpm*

libXp provides the X print library.

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version   |toolchain
----------|-------------------------------
``3.5.11``|``foss/2016a``, ``intel/2016a``

<a anchor="libxrandr"/>
### *libXrandr*

X Resize, Rotate and Reflection extension library

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version  |toolchain
---------|-------------------------------
``1.5.0``|``foss/2016a``, ``intel/2016a``

<a anchor="libxrender"/>
### *libXrender*

X11 client-side library

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version  |toolchain
---------|-------------------------------------------------
``0.9.9``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``

<a anchor="libxslt"/>
### *libxslt*

Libxslt is the XSLT C library developed for the GNOME project  (but usable outside of the Gnome platform).

*homepage*: http://xmlsoft.org/

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------------------------------------------------------------------------------------------
``1.1.28``|                  |``foss/2016a``, ``intel/2016a``
``1.1.28``|``-Python-2.7.11``|``intel/2016a``
``1.1.29``|                  |``foss/2016b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``
``1.1.30``|                  |``GCCcore/6.3.0``
``1.1.32``|                  |``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``1.1.33``|                  |``GCCcore/8.2.0``
``1.1.34``|                  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

<a anchor="libxsmm"/>
### *libxsmm*

LIBXSMM is a library for small dense and small sparse matrix-matrix multiplications targeting Intel Architecture (x86).

*homepage*: https://github.com/hfp/libxsmm

version   |toolchain
----------|-----------------------------------------------------------------------------------------------
``1.4``   |``intel/2016a``
``1.4.4`` |``foss/2016b``, ``intel/2016b``
``1.6.4`` |``foss/2016b``, ``intel/2016b``
``1.8.2`` |``intel/2017b``
``1.8.3`` |``foss/2018a``, ``intel/2018a``
``1.10``  |``GCC/8.2.0-2.31.1``, ``foss/2018b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2018b``
``1.16.1``|``GCC/10.2.0``, ``GCC/9.3.0``, ``iccifort/2020.1.217``, ``iccifort/2020.4.304``
``1.16.2``|``GCC/10.3.0``, ``intel-compilers/2021.2.0``
``1.17``  |``GCC/11.2.0``

<a anchor="libxt"/>
### *libXt*

libXt provides the X Toolkit Intrinsics, an abstract widget library upon which other toolkits are  based. Xt is the basis for many toolkits, including the Athena widgets (Xaw), and LessTif (a Motif implementation).

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version  |toolchain
---------|-------------------------------------------------
``1.1.5``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``

<a anchor="libxxf86vm"/>
### *libXxf86vm*

X11 XFree86 video mode extension library

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version  |toolchain
---------|-------------------------------
``1.1.4``|``foss/2016a``, ``intel/2016a``

<a anchor="libyaml"/>
### *libyaml*

LibYAML is a YAML parser and emitter written in C.

*homepage*: https://pyyaml.org/wiki/LibYAML

version  |toolchain
---------|--------------------------------------------------------------------------------------------------
``0.1.6``|``GCCcore/6.4.0``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``0.1.7``|``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``system``
``0.2.1``|``GCCcore/7.3.0``, ``system``
``0.2.2``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``0.2.5``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``

<a anchor="libzeep"/>
### *libzeep*

Libzeep was originally developed to make it easy to create SOAP servers.

*homepage*: https://github.com/mhekkel/libzeep

version  |toolchain
---------|---------------
``5.0.1``|``gompi/2019b``

<a anchor="libzip"/>
### *libzip*

libzip is a C library for reading, creating, and modifying zip archives.

*homepage*: https://libzip.org/

version  |toolchain
---------|----------------------------------------------------------
``1.5.2``|``GCCcore/8.2.0``
``1.7.3``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.9.2``|``GCCcore/11.3.0``

<a anchor="lie_learn"/>
### *lie_learn*

lie_learn is a python package that knows how to do various tricky computations related to Lie groups and manifolds (mainly the sphere S2 and rotation group SO3).

*homepage*: https://github.com/AMLab-Amsterdam/lie_learn

version        |versionsuffix    |toolchain
---------------|-----------------|------------------
``0.0.1.post1``|``-Python-3.7.4``|``fosscuda/2019b``

<a anchor="lifelines"/>
### *lifelines*

lifelines is a complete survival analysis library, written in pure Python.

*homepage*: https://lifelines.readthedocs.io/en/latest/

version   |versionsuffix    |toolchain
----------|-----------------|------------------
``0.22.8``|``-Python-3.7.2``|``fosscuda/2019a``
``0.26.3``|                 |``fosscuda/2020b``

<a anchor="lighter"/>
### *Lighter*

Fast and memory-efficient sequencing error corrector

*homepage*: https://github.com/mourisl/Lighter

version  |toolchain
---------|------------------------------
``1.1.1``|``foss/2018a``
``1.1.2``|``GCC/11.2.0``, ``foss/2018b``

<a anchor="liknorm"/>
### *liknorm*

Moments of the product of an exponential-family likelihood with a Normal distribution.

*homepage*: https://github.com/limix/liknorm

version  |toolchain
---------|-----------------
``1.5.2``|``GCCcore/7.3.0``

<a anchor="likwid"/>
### *likwid*

Likwid stands for Like I knew what I am doing. This project contributes easy to use command line tools for Linux to support programmers in developing high performance multi threaded programs. 

*homepage*: https://github.com/RRZE-HPC/likwid

version  |toolchain
---------|-----------------------------------------------------------------------------------------------------
``4.0.1``|``GNU/4.9.3-2.25``
``4.1.0``|``GCCcore/4.9.3``
``4.2.0``|``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``foss/2017a``, ``intel/2017a``
``4.3.2``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``5.0.1``|``GCCcore/8.3.0``
``5.1.0``|``GCCcore/9.3.0``
``5.2.0``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``iccifort/2020.4.304``, ``intel-compilers/2021.2.0``
``5.2.1``|``GCC/11.2.0``
``5.2.2``|``GCC/11.3.0``

<a anchor="limix"/>
### *limix*

Limix is a flexible and efficient linear mixed model library with interfaces to Python.  Genomic analyses require flexible models that can be adapted to the needs of the user.  Limix is smart about how particular models are fitted to save computational cost. 

*homepage*: https://github.com/limix/limix

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``2.0.4``|``-Python-3.6.6``|``foss/2018b``

<a anchor="linbox"/>
### *LinBox*

C++ library for exact, high-performance linear algebra

*homepage*: http://linalg.org/

version  |toolchain
---------|--------------
``1.4.0``|``foss/2016a``

<a anchor="line_profiler"/>
### *line_profiler*

line_profiler is a module for doing line-by-line profiling of functions. kernprof is a convenient script for running either line_profiler or the Python standard library's cProfile or profile modules, depending on what is available.

*homepage*: https://github.com/pyutils/line_profiler

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``3.1.0``|``-Python-3.8.2``|``foss/2020a``
``3.5.1``|                 |``foss/2021b``
``4.0.0``|                 |``foss/2022a``

<a anchor="lingeling"/>
### *Lingeling*

One of the design principles of the state-of-the-art SAT solver Lingeling is to use as compact data structures as possible. These reduce memory usage, increase cache efficiency and thus improve runtime, particularly, when using multiple solver instances on multi-core machines, as in our parallel portfolio solver Plingeling and our cube and conquer solver Treengeling.

*homepage*: http://fmv.jku.at/lingeling/

version|toolchain
-------|-------------
``bcp``|``GCC/9.3.0``

<a anchor="littlecms"/>
### *LittleCMS*

Little CMS intends to be an OPEN SOURCE small-footprint color management engine,  with special focus on accuracy and performance. 

*homepage*: http://www.littlecms.com/

version   |toolchain
----------|----------------------------------------------------------------------------------------------------------------------------------------------
``2.7``   |``intel/2016a``
``2.8``   |``GCCcore/6.4.0``, ``foss/2016b``, ``intel/2016b``, ``intel/2017a``
``2.9``   |``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``foss/2017b``, ``foss/2018a``, ``intel/2017b``
``2.11``  |``GCCcore/10.2.0``
``2.12``  |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``2.13.1``|``GCCcore/11.3.0``
``2.14``  |``GCCcore/12.2.0``

<a anchor="lldb"/>
### *LLDB*

The debugger component of the LLVM project

*homepage*: https://lldb.llvm.org/

version   |versionsuffix    |toolchain
----------|-----------------|-----------------
``11.0.0``|``-Python-3.8.2``|``GCCcore/9.3.0``

<a anchor="llvm"/>
### *LLVM*

The LLVM Core libraries provide a modern source- and target-independent  optimizer, along with code generation support for many popular CPUs  (as well as some less common ones!) These libraries are built around a well  specified code representation known as the LLVM intermediate representation  ("LLVM IR"). The LLVM Core libraries are well documented, and it is  particularly easy to invent your own language (or port an existing compiler)  to use LLVM as an optimizer and code generator.

*homepage*: https://llvm.org/

version   |toolchain
----------|------------------------------------------------------------------------
``3.7.1`` |``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``3.8.0`` |``foss/2016a``, ``intel/2016a``
``3.8.1`` |``GCCcore/4.9.3``, ``foss/2016b``, ``intel/2016b``
``3.9.0`` |``foss/2016b``, ``intel/2016b``
``3.9.1`` |``foss/2017a``
``4.0.0`` |``foss/2017a``, ``intel/2017a``
``4.0.1`` |``intel/2017a``, ``intel/2017b``
``5.0.0`` |``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``5.0.1`` |``GCCcore/6.4.0``
``6.0.0`` |``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``7.0.0`` |``GCCcore/7.2.0``, ``GCCcore/7.3.0``
``7.0.1`` |``GCCcore/8.2.0``
``8.0.1`` |``GCCcore/8.3.0``
``9.0.0`` |``GCCcore/8.3.0``
``9.0.1`` |``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``10.0.0``|``GCCcore/8.3.0``
``10.0.1``|``GCCcore/10.2.0``
``11.0.0``|``GCCcore/10.2.0``
``11.1.0``|``GCCcore/10.3.0``
``12.0.1``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``14.0.3``|``GCCcore/11.3.0``
``15.0.5``|``GCCcore/12.2.0``

<a anchor="lmdb"/>
### *LMDB*

LMDB is a fast, memory-efficient database. With memory-mapped files, it has the read performance  of a pure in-memory database while retaining the persistence of standard disk-based databases.

*homepage*: https://symas.com/lmdb

version   |toolchain
----------|--------------------------------------------------------
``0.9.18``|``foss/2016a``
``0.9.21``|``GCCcore/6.4.0``, ``intel/2017a``
``0.9.22``|``GCCcore/7.3.0``
``0.9.23``|``GCCcore/8.2.0``
``0.9.24``|``GCCcore/10.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``0.9.28``|``GCCcore/10.3.0``
``0.9.29``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="lmfit"/>
### *LMfit*

Lmfit provides a high-level interface to non-linear optimization and curve fitting problems for Python

*homepage*: https://lmfit.github.io/lmfit-py

version   |versionsuffix     |toolchain
----------|------------------|-----------------------------------------------
``0.9.9`` |``-Python-3.6.4`` |``intel/2018a``
``0.9.14``|``-Python-2.7.15``|``intel/2018b``
``1.0.0`` |``-Python-3.7.4`` |``foss/2019b``, ``intel/2019b``
``1.0.1`` |``-Python-3.8.2`` |``foss/2020a``
``1.0.2`` |                  |``foss/2020b``, ``intel/2020b``
``1.0.3`` |                  |``foss/2021a``, ``foss/2022a``, ``intel/2022a``

<a anchor="lmod"/>
### *Lmod*

Lmod is a Lua based module system. Modules allow for dynamic modification  of a user's environment under Unix systems. See www.tacc.utexas.edu/tacc-projects/lmod  for a complete description. Lmod is a new implementation that easily handles the MODULEPATH  Hierarchical problem. It is drop-in replacement for TCL/C modules and reads TCL modulefiles directly.

*homepage*: http://sourceforge.net/projects/lmod/

version  |toolchain
---------|----------------------------
``5.2``  |``GCC/4.8.2``
``5.2.5``|``GCC/4.8.2``
``5.3``  |``GCC/4.8.2``
``5.4``  |``GCC/4.8.2``
``5.4.2``|``GCC/4.8.2``
``5.5``  |``GCC/4.8.2``
``5.5.1``|``GCC/4.8.2``
``5.6``  |``GCC/4.8.2``
``5.7``  |``GCC/4.8.2``
``5.8``  |``GCC/4.8.2``
``5.8.5``|``GCC/4.8.2``
``5.9``  |``GCC/4.8.2``, ``GCC/4.8.4``
``6.4.2``|``system``
``7.3``  |``system``

<a anchor="lncloom"/>
### *LncLOOM*

LncLOOM is a graph-based framework that uses integer programming to identify combinations of short motifs that are deeply conserved in rapidly evolving sequences.

*homepage*: https://github.com/lncLOOM/lncLOOMv2/

version|toolchain
-------|--------------
``2.0``|``foss/2020b``

<a anchor="locarna"/>
### *LocARNA*

LocARNA is a collection of alignment tools for the structural analysis of RNA.  Given a set of RNA sequences, LocARNA simultaneously aligns and predicts common structures for  your RNAs. In this way, LocARNA performs Sankoff-like alignment and is in particular suited for  analyzing sets of related RNAs without known common structure.

*homepage*: http://www.bioinf.uni-freiburg.de/Software/LocARNA/

version    |versionsuffix    |toolchain
-----------|-----------------|--------------
``1.9.2``  |                 |``foss/2016b``
``1.9.2.2``|``-Python-3.6.6``|``foss/2018b``
``1.9.2.3``|                 |``foss/2021b``

<a anchor="lofreq"/>
### *LoFreq*

Fast and sensitive variant calling from next-gen sequencing data

*homepage*: http://csb5.github.io/lofreq

version    |versionsuffix     |toolchain
-----------|------------------|------------------------------------------------
``2.1.2``  |``-Python-2.7.12``|``intel/2016b``
``2.1.3.1``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``, ``intel/2018a``

<a anchor="log-log4perl"/>
### *Log-Log4perl*

Log4perl

*homepage*: https://metacpan.org/pod/Log::Log4perl

version |toolchain
--------|--------------
``1.47``|``foss/2016a``

<a anchor="logaddexp"/>
### *logaddexp*

C library that implements the logarithm of the sum of exponentiations. Inspired by NumPy's logaddexp function.

*homepage*: https://github.com/horta/logaddexp

version  |toolchain
---------|-----------------
``1.0.3``|``GCCcore/7.3.0``

<a anchor="lohhla"/>
### *LOHHLA*

LOHHLA, Loss Of Heterozygosity in Human Leukocyte Antigen, a computational tool to evaluate HLA loss using next-generation sequencing data.

*homepage*: https://bitbucket.org/mcgranahanlab/lohhla

version       |versionsuffix|toolchain
--------------|-------------|--------------
``2018.11.05``|``-R-3.5.1`` |``foss/2018b``

<a anchor="loki"/>
### *Loki*

Loki is a C++ library of designs, containing flexible implementations of common design patterns and   idioms. 

*homepage*: http://loki-lib.sourceforge.net/

version  |toolchain
---------|---------------------------------------------------------------------------------
``0.1.7``|``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``, ``intel/2018a``

<a anchor="longestrunsubsequence"/>
### *longestrunsubsequence*

Implementation of a solver for the Longest Run Subsequence Problem. Given a  sequence as input, compute a longest subsequence such that there is at most one run for every  character. 

*homepage*: https://pypi.org/project/longestrunsubsequence/

version  |toolchain
---------|------------------
``1.0.1``|``GCCcore/10.3.0``

<a anchor="longread_umi"/>
### *longread_umi*

A collection of scripts for processing longread UMI data.

*homepage*: https://github.com/SorenKarst/longread_umi

version  |toolchain
---------|--------------
``0.3.2``|``foss/2020b``

<a anchor="longshot"/>
### *Longshot*

Longshot is a variant calling tool for diploid genomes using long error prone reads such as Pacific  Biosciences (PacBio) SMRT and Oxford Nanopore Technologies (ONT). It takes as input an aligned BAM file and outputs  a phased VCF file with variants and haplotype information. It can also output haplotype-separated BAM files that can  be used for downstream analysis. Currently, it only calls single nucleotide variants (SNVs).

*homepage*: https://github.com/pjedge/longshot

version  |toolchain
---------|-------------------------------------
``0.3.4``|``GCCcore/8.2.0``
``0.4.1``|``GCC/7.3.0-2.30``, ``GCCcore/8.3.0``
``0.4.3``|``GCCcore/10.2.0``

<a anchor="loompy"/>
### *loompy*

Python implementation of the Loom file format, an efficient file format for large omics datasets

*homepage*: https://loompy.org/

version  |toolchain
---------|-------------------------------
``3.0.6``|``intel/2020b``
``3.0.7``|``foss/2021b``, ``intel/2021b``

<a anchor="lpeg"/>
### *LPeg*

LPeg is a new pattern-matching library for Lua, based on Parsing Expression Grammars (PEGs).

*homepage*: http://www.inf.puc-rio.br/~roberto/lpeg/

version  |toolchain
---------|------------------
``1.0.2``|``GCCcore/10.2.0``

<a anchor="lpjml"/>
### *LPJmL*

Dynamic global vegetation model with managed land and river routing

*homepage*: https://github.com/PIK-LPJmL/LPJmL

version    |toolchain
-----------|---------------
``4.0.003``|``iimpi/2020b``

<a anchor="lpsolve"/>
### *lpsolve*

Mixed Integer Linear Programming (MILP) solver

*homepage*: https://sourceforge.net/projects/lpsolve/

version     |toolchain
------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``5.5.2.5`` |``GCC/6.4.0-2.28``, ``GCC/8.3.0``, ``foss/2018a``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``iccifort/2019.5.281``, ``intel/2017a``, ``intel/2018b``
``5.5.2.11``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/9.3.0``

<a anchor="lrslib"/>
### *lrslib*

lrslib is a self-contained ANSI C implementation of the reverse search algorithm for vertex enumeration/convex hull problems

*homepage*: http://cgm.cs.mcgill.ca/~avis/C/lrs.html

version |toolchain
--------|---------------
``6.2`` |``intel/2018b``
``7.0a``|``gompi/2019a``

<a anchor="ls-prepost"/>
### *LS-PrePost*

LS-PrePost is an advanced pre and post-processor that is delivered free with LS-DYNA.

*homepage*: http://lstc.com/products/ls-prepost

version   |versionsuffix|toolchain
----------|-------------|----------
``4.6``   |``-centos6`` |``system``
``4.6``   |``-centos7`` |``system``
``4.6.24``|             |``system``
``4.7.8`` |             |``system``
``4.7.15``|             |``system``

<a anchor="lsd2"/>
### *LSD2*

Least-squares methods to estimate rates and dates from phylogenies

*homepage*: https://github.com/tothuhien/lsd2

version  |toolchain
---------|--------------------------------------
``1.9.7``|``GCCcore/9.3.0``
``2.2``  |``GCCcore/10.2.0``
``2.3``  |``GCCcore/10.3.0``, ``GCCcore/11.2.0``

<a anchor="lsms"/>
### *LSMS*

LSMS benchmark, part of CORAL suite

*homepage*: https://asc.llnl.gov/CORAL-benchmarks/#lsms

version     |toolchain
------------|--------------
``3_rev237``|``foss/2016a``

<a anchor="ltr_retriever"/>
### *LTR_retriever*

LTR_retriever is a highly accurate and sensitive program for  identification of LTR retrotransposons; The LTR Assembly Index (LAI) is also  included in this package. 

*homepage*: https://github.com/oushujun/LTR_retriever

version  |toolchain
---------|--------------
``2.9.0``|``foss/2020b``

<a anchor="ltrdetector"/>
### *LtrDetector*

A modern tool-suite for detectinglong terminal repeat retrotransposons de-novo onthe genomic scale

*homepage*: https://github.com/TulsaBioinformaticsToolsmith/LtrDetector

version|versionsuffix    |toolchain
-------|-----------------|---------------
``1.0``|``-Python-3.7.4``|``intel/2019b``

<a anchor="lua"/>
### *Lua*

Lua is a powerful, fast, lightweight, embeddable scripting language.  Lua combines simple procedural syntax with powerful data description constructs based  on associative arrays and extensible semantics. Lua is dynamically typed,  runs by interpreting bytecode for a register-based virtual machine,  and has automatic memory management with incremental garbage collection,  making it ideal for configuration, scripting, and rapid prototyping.

*homepage*: https://www.lua.org/

version    |toolchain
-----------|---------------------------------------------------------------------------------------
``5.1.4-5``|``GCC/4.8.2``
``5.1.4-8``|``GCC/4.8.2``, ``GCC/4.8.4``, ``system``
``5.1.5``  |``GCCcore/7.3.0``, ``GCCcore/8.3.0``
``5.2.4``  |``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``5.3.4``  |``GCCcore/7.2.0``, ``system``
``5.3.5``  |``GCCcore/10.2.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``system``
``5.4.2``  |``GCCcore/10.2.0``
``5.4.3``  |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``5.4.4``  |``GCCcore/11.3.0``

<a anchor="luajit"/>
### *LuaJIT*

LuaJIT is a Just-In-Time Compiler (JIT) for the Lua programming language. Lua is a powerful, dynamic and light-weight programming language. It may be embedded or used as a general-purpose, stand-alone language. 

*homepage*: http://luajit.org/

version  |toolchain
---------|-------------
``2.0.2``|``GCC/4.9.2``

<a anchor="luajit2-openresty"/>
### *LuaJIT2-OpenResty*

openresty/luajit2 - OpenResty's maintained branch of LuaJIT. LuaJIT is a Just-In-Time Compiler (JIT) for the Lua programming language. Lua is a powerful, dynamic and light-weight programming language. It may be embedded or used as a general-purpose, stand-alone language. 

*homepage*: https://github.com/openresty/luajit2

version         |toolchain
----------------|-------------
``2.1-20220411``|``GCC/9.3.0``

<a anchor="lucene-geo-gazetteer"/>
### *Lucene-Geo-Gazetteer*

A command line gazetteer built around the Geonames.org dataset, that uses the Apache Lucene library  to create a searchable gazetteer.

*homepage*: https://github.com/chrismattmann/lucene-geo-gazetteer

version     |toolchain
------------|----------
``20170718``|``system``

<a anchor="lumpy"/>
### *LUMPY*

A probabilistic framework for structural variant discovery. 

*homepage*: https://github.com/arq5x/lumpy-sv

version   |toolchain
----------|--------------
``0.2.13``|``foss/2016b``
``0.3.1`` |``foss/2020b``

<a anchor="luscus"/>
### *LUSCUS*

Luscus is the program for graphical display and editing of molecular systems.

*homepage*: https://sourceforge.net/projects/luscus/

version  |toolchain
---------|-------------------------------
``0.8.6``|``foss/2018b``, ``intel/2018a``

<a anchor="lwgrp"/>
### *lwgrp*

The light-weight group library defines data structures and collective operations to group MPI processes as an ordered set.  Such groups are useful as substitutes for MPI communicators when the overhead of communicator creation is too costly.  For example, certain sorting algorithms recursively divide processes into subgroups as the sort algorithm progresses.  These groups may be different with each invocation, so that it is inefficient to create and destroy communicators during the sort routine.

*homepage*: https://github.com/LLNL/lwgrp

version  |toolchain
---------|------------------------------------------------------------------
``1.0.2``|``gompi/2019a``, ``gompi/2020a``, ``iimpi/2019a``, ``iimpi/2020a``
``1.0.3``|``gompi/2020b``

<a anchor="lxml"/>
### *lxml*

The lxml XML toolkit is a Pythonic binding for the C libraries libxml2 and libxslt.

*homepage*: https://lxml.de/

version  |versionsuffix     |toolchain
---------|------------------|--------------------------------------
``3.5.0``|``-Python-2.7.11``|``intel/2016a``
``3.6.0``|``-Python-2.7.11``|``intel/2016a``
``3.6.4``|``-Python-2.7.12``|``intel/2016b``
``4.0.0``|``-Python-2.7.13``|``intel/2017a``
``4.1.1``|``-Python-2.7.14``|``intel/2017b``
``4.2.0``|``-Python-2.7.14``|``foss/2018a``, ``intel/2018a``
``4.2.0``|``-Python-3.6.4`` |``intel/2018a``
``4.2.5``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``4.2.5``|``-Python-3.6.6`` |``foss/2018b``
``4.3.3``|                  |``GCCcore/8.2.0``
``4.4.2``|                  |``GCCcore/8.3.0``
``4.5.2``|                  |``GCCcore/9.3.0``
``4.6.2``|                  |``GCCcore/10.2.0``
``4.6.3``|                  |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``4.9.1``|                  |``GCCcore/11.3.0``

<a anchor="lynx"/>
### *lynx*

lynx is an alphanumeric display oriented World-Wide Web Client

*homepage*: http://lynx.browser.org/

version  |versionsuffix|toolchain
---------|-------------|--------------
``2.8.9``|``-develop`` |``foss/2016b``

<a anchor="lz4"/>
### *lz4*

LZ4 is lossless compression algorithm, providing compression speed at 400 MB/s per core.  It features an extremely fast decoder, with speed in multiple GB/s per core.

*homepage*: https://lz4.github.io/lz4/

version  |toolchain
---------|----------------------------------------------------------
``1.8.2``|``GCCcore/5.4.0``, ``GCCcore/6.4.0``
``1.9.0``|``GCCcore/7.3.0``
``1.9.1``|``GCCcore/8.2.0``
``1.9.2``|``GCCcore/10.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.9.3``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``1.9.4``|``GCCcore/12.2.0``

<a anchor="lzo"/>
### *LZO*

Portable lossless data compression library

*homepage*: http://www.oberhumer.com/opensource/lzo/

version |toolchain
--------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.09``|``intel/2016b``, ``intel/2017b``
``2.10``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``foss/2018a``, ``foss/2018b``, ``fosscuda/2018b``, ``intel/2017a``

<a anchor="m"/>
### *M*


<a href="#m1qn3">M1QN3</a> - <a href="#m4">M4</a> - <a href="#mach">MACH</a> - <a href="#macs2">MACS2</a> - <a href="#macse">MACSE</a> - <a href="#maeparser">maeparser</a> - <a href="#mafft">MAFFT</a> - <a href="#mageck">MAGeCK</a> - <a href="#magick">magick</a> - <a href="#magics">Magics</a> - <a href="#magma">MAGMA</a> - <a href="#magma">magma</a> - <a href="#magrespython">MagresPython</a> - <a href="#mahotas">mahotas</a> - <a href="#majiq">MAJIQ</a> - <a href="#make">make</a> - <a href="#makedepend">makedepend</a> - <a href="#makedepf90">makedepf90</a> - <a href="#makeinfo">makeinfo</a> - <a href="#mako">Mako</a> - <a href="#mamba">Mamba</a> - <a href="#manta">manta</a> - <a href="#mapdamage">mapDamage</a> - <a href="#maple">Maple</a> - <a href="#mapsplice">MapSplice</a> - <a href="#maq">Maq</a> - <a href="#mariadb">MariaDB</a> - <a href="#mariadb-connector-c">MariaDB-connector-c</a> - <a href="#mars">MARS</a> - <a href="#mash">Mash</a> - <a href="#masurca">MaSuRCA</a> - <a href="#mathematica">Mathematica</a> - <a href="#mathgl">MathGL</a> - <a href="#matio">MATIO</a> - <a href="#matlab">MATLAB</a> - <a href="#matlab-engine">MATLAB-Engine</a> - <a href="#matlab-proxy">matlab-proxy</a> - <a href="#matplotlib">matplotlib</a> - <a href="#matplotlib-inline">matplotlib-inline</a> - <a href="#matsim">MATSim</a> - <a href="#maude">Maude</a> - <a href="#mauvealigner">mauveAligner</a> - <a href="#maven">Maven</a> - <a href="#mawk">mawk</a> - <a href="#maxbin">MaxBin</a> - <a href="#maxquant">MaxQuant</a> - <a href="#mayavi">mayavi</a> - <a href="#maze">maze</a> - <a href="#mbedtls">MbedTLS</a> - <a href="#mbrola">MBROLA</a> - <a href="#mbuffer">mbuffer</a> - <a href="#mc">mc</a> - <a href="#mcl">MCL</a> - <a href="#mcr">MCR</a> - <a href="#mcu">mcu</a> - <a href="#mdanalysis">MDAnalysis</a> - <a href="#mdbm">MDBM</a> - <a href="#mdsplus">MDSplus</a> - <a href="#mdsplus-java">MDSplus-Java</a> - <a href="#mdsplus-python">MDSplus-Python</a> - <a href="#mdtest">mdtest</a> - <a href="#mdtraj">MDTraj</a> - <a href="#medaka">medaka</a> - <a href="#medimgproc">medImgProc</a> - <a href="#medpy">MedPy</a> - <a href="#meep">Meep</a> - <a href="#mega">MEGA</a> - <a href="#megacc">MEGACC</a> - <a href="#megahit">MEGAHIT</a> - <a href="#megalodon">Megalodon</a> - <a href="#megan">MEGAN</a> - <a href="#meld">Meld</a> - <a href="#mem">MEM</a> - <a href="#meme">MEME</a> - <a href="#memkind">memkind</a> - <a href="#memory-profiler">memory-profiler</a> - <a href="#memote">MEMOTE</a> - <a href="#memtester">memtester</a> - <a href="#merantk">meRanTK</a> - <a href="#merckx">MERCKX</a> - <a href="#mercurial">Mercurial</a> - <a href="#mesa">Mesa</a> - <a href="#mesa-demos">Mesa-demos</a> - <a href="#meshalyzer">meshalyzer</a> - <a href="#meshio">meshio</a> - <a href="#meshtool">meshtool</a> - <a href="#meson">Meson</a> - <a href="#meson-python">meson-python</a> - <a href="#mesquite">Mesquite</a> - <a href="#mess">MESS</a> - <a href="#metabat">MetaBAT</a> - <a href="#metaboanalystr">MetaboAnalystR</a> - <a href="#metaerg">metaerg</a> - <a href="#metaeuk">MetaEuk</a> - <a href="#metageneannotator">MetaGeneAnnotator</a> - <a href="#metagenome-atlas">Metagenome-Atlas</a> - <a href="#metal">Metal</a> - <a href="#metamorpheus">MetaMorpheus</a> - <a href="#metaphlan">MetaPhlAn</a> - <a href="#metaphlan2">MetaPhlAn2</a> - <a href="#metawrap">metaWRAP</a> - <a href="#metaxa2">Metaxa2</a> - <a href="#methyldackel">MethylDackel</a> - <a href="#methylpy">methylpy</a> - <a href="#metis">METIS</a> - <a href="#mgltools">mgltools</a> - <a href="#mhcflurry">mhcflurry</a> - <a href="#mhcnuggets">mhcnuggets</a> - <a href="#microbeannotator">MicrobeAnnotator</a> - <a href="#microctools">microctools</a> - <a href="#migec">MiGEC</a> - <a href="#migrate-n">MIGRATE-N</a> - <a href="#miller">Miller</a> - <a href="#mimalloc">mimalloc</a> - <a href="#minc">MINC</a> - <a href="#minced">MinCED</a> - <a href="#mini-xml">Mini-XML</a> - <a href="#miniasm">miniasm</a> - <a href="#minibar">minibar</a> - <a href="#minicard">MiniCARD</a> - <a href="#miniconda2">Miniconda2</a> - <a href="#miniconda3">Miniconda3</a> - <a href="#minieigen">minieigen</a> - <a href="#minimac4">Minimac4</a> - <a href="#minimap2">minimap2</a> - <a href="#minipolish">Minipolish</a> - <a href="#minisat">MiniSat</a> - <a href="#minpack">MINPACK</a> - <a href="#minpath">MinPath</a> - <a href="#mira">MIRA</a> - <a href="#mirdeep2">miRDeep2</a> - <a href="#mish-cuda">Mish-Cuda</a> - <a href="#misha">misha</a> - <a href="#mitgcmutils">MITgcmutils</a> - <a href="#mitobim">MITObim</a> - <a href="#mitoz">MitoZ</a> - <a href="#mixcr">MiXCR</a> - <a href="#mixmhc2pred">MixMHC2pred</a> - <a href="#mkl-dnn">mkl-dnn</a> - <a href="#mkl-service">mkl-service</a> - <a href="#mkl_fft">mkl_fft</a> - <a href="#mlc">MLC</a> - <a href="#mlxtend">MLxtend</a> - <a href="#mm-common">mm-common</a> - <a href="#mmg">Mmg</a> - <a href="#mmseq">MMSEQ</a> - <a href="#mmseqs2">MMseqs2</a> - <a href="#mmtf-cpp">mmtf-cpp</a> - <a href="#mne-python">MNE-Python</a> - <a href="#moabs">MOABS</a> - <a href="#modeltest-ng">ModelTest-NG</a> - <a href="#modred">modred</a> - <a href="#molcas">Molcas</a> - <a href="#mold">mold</a> - <a href="#molden">Molden</a> - <a href="#molekel">Molekel</a> - <a href="#molmod">molmod</a> - <a href="#molpro">Molpro</a> - <a href="#mona">MONA</a> - <a href="#monai">MONAI</a> - <a href="#mongolite">mongolite</a> - <a href="#mono">Mono</a> - <a href="#monocle3">Monocle3</a> - <a href="#moonjit">moonjit</a> - <a href="#moose">MOOSE</a> - <a href="#mordecai">mordecai</a> - <a href="#moreronn">MoreRONN</a> - <a href="#mosdepth">mosdepth</a> - <a href="#mothur">Mothur</a> - <a href="#motif">motif</a> - <a href="#motioncor2">MotionCor2</a> - <a href="#motionsegmentation">motionSegmentation</a> - <a href="#moviepy">MoviePy</a> - <a href="#mpb">MPB</a> - <a href="#mpc">MPC</a> - <a href="#mpfr">MPFR</a> - <a href="#mpi4py">mpi4py</a> - <a href="#mpich">MPICH</a> - <a href="#mpich2">MPICH2</a> - <a href="#mpifileutils">mpifileutils</a> - <a href="#mpip">mpiP</a> - <a href="#mpj-express">MPJ-Express</a> - <a href="#mpmath">mpmath</a> - <a href="#mrbayes">MrBayes</a> - <a href="#mrcfile">mrcfile</a> - <a href="#mrchem">MRChem</a> - <a href="#mrcpp">MRCPP</a> - <a href="#mricron">MRIcron</a> - <a href="#mrtrix">MRtrix</a> - <a href="#msgpack-c">msgpack-c</a> - <a href="#msm">MSM</a> - <a href="#mspc">MSPC</a> - <a href="#msprime">msprime</a> - <a href="#mstore">mstore</a> - <a href="#mtl4">MTL4</a> - <a href="#mujoco">MuJoCo</a> - <a href="#mujoco-py">mujoco-py</a> - <a href="#multichoose">multichoose</a> - <a href="#multilevelestimators">MultilevelEstimators</a> - <a href="#multinest">MultiNest</a> - <a href="#multiqc">MultiQC</a> - <a href="#multiwfn">Multiwfn</a> - <a href="#mumerge">muMerge</a> - <a href="#mummer">MUMmer</a> - <a href="#mumps">MUMPS</a> - <a href="#muparser">muParser</a> - <a href="#muparserx">muparserx</a> - <a href="#mupexi">MuPeXI</a> - <a href="#muscle">MUSCLE</a> - <a href="#music">MuSiC</a> - <a href="#must">MUST</a> - <a href="#mutect">MuTect</a> - <a href="#mutil">mutil</a> - <a href="#mvapich2">MVAPICH2</a> - <a href="#mxml">mxml</a> - <a href="#mxmlplus">mxmlplus</a> - <a href="#mxnet">MXNet</a> - <a href="#mycc">MyCC</a> - <a href="#mygene">mygene</a> - <a href="#mymedialite">MyMediaLite</a> - <a href="#mympingpong">mympingpong</a> - <a href="#myokit">Myokit</a> - <a href="#mypy">mypy</a> - <a href="#mysql">MySQL</a> - <a href="#mysql-python">MySQL-python</a> - <a href="#mysqlclient">mysqlclient</a>


<a anchor="m1qn3"/>
### *M1QN3*

A solver of large-scale unconstrained minimization problems

*homepage*: https://who.rocq.inria.fr/Jean-Charles.Gilbert/modulopt/optimization-routines/m1qn3/m1qn3.html

version|toolchain
-------|--------------
``3.3``|``GCC/10.3.0``

<a anchor="m4"/>
### *M4*

GNU M4 is an implementation of the traditional Unix macro processor. It is  mostly SVR4 compatible although it has some extensions (for example, handling  more than 9 positional parameters to macros). GNU M4 also has built-in  functions for including files, running shell commands, doing arithmetic, etc. 

*homepage*: https://www.gnu.org/software/m4/m4.html

version   |toolchain
----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.4.16``|``GCC/4.8.1``, ``GCC/4.8.2``, ``system``
``1.4.17``|``GCC/4.8.2``, ``GCC/4.8.4``, ``GCC/4.9.2``, ``GCC/4.9.2-binutils-2.25``, ``GCC/4.9.3``, ``GCC/4.9.3-2.25``, ``GCC/4.9.3-binutils-2.25``, ``GCC/5.1.0-binutils-2.25``, ``GCC/5.2.0``, ``GCC/5.4.0-2.26``, ``GCCcore/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/4.9.4``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.1.0``, ``GCCcore/6.2.0``, ``GNU/4.9.2-2.25``, ``GNU/4.9.3-2.25``, ``GNU/5.1.0-2.25``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
``1.4.18``|``FCC/4.5.0``, ``GCCcore/10.1.0``, ``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.1.0``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/5.5.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.1.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/7.4.0``, ``GCCcore/8.1.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/8.4.0``, ``GCCcore/9.1.0``, ``GCCcore/9.2.0``, ``GCCcore/9.3.0``, ``GCCcore/system``, ``system``
``1.4.19``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.1.0``, ``GCCcore/12.2.0``, ``GCCcore/9.4.0``, ``GCCcore/9.5.0``, ``system``

<a anchor="mach"/>
### *MACH*

MACH 1.0 is a Markov Chain based haplotyper  that can resolve long haplotypes or infer missing genotypes  in samples of unrelated individuals.

*homepage*: http://csg.sph.umich.edu/abecasis/MACH/

version   |toolchain
----------|----------
``1.0.18``|``system``

<a anchor="macs2"/>
### *MACS2*

Model Based Analysis for ChIP-Seq data

*homepage*: https://github.com/taoliu/MACS

version           |versionsuffix     |toolchain
------------------|------------------|-------------------------------
``2.1.1.20160309``|``-Python-2.7.14``|``intel/2017b``
``2.1.2.1``       |``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``2.1.2.1``       |``-Python-2.7.15``|``intel/2019a``
``2.2.5``         |``-Python-3.6.6`` |``foss/2018b``
``2.2.7.1``       |                  |``foss/2021b``

<a anchor="macse"/>
### *MACSE*

MACSE aligns coding NT sequences with respect to their AA translation while allowing NT sequences to contain multiple frameshifts and/or stop codons. MACSE is hence the first automatic solution to align protein-coding gene datasets containing non-functional sequences (pseudogenes) without disrupting the underlying codon structure.

*homepage*: https://bioweb.supagro.inra.fr/macse/index.php

version |versionsuffix|toolchain
--------|-------------|----------
``2.06``|``-Java-15`` |``system``

<a anchor="maeparser"/>
### *maeparser*

maeparser is a parser for Schrodinger Maestro files.

*homepage*: https://github.com/schrodinger/maeparser

version  |toolchain
---------|------------------------------------------------------------------
``1.2.2``|``gompi/2019a``, ``iimpi/2019a``
``1.3.0``|``gompi/2019b``, ``gompi/2021a``, ``gompi/2022a``, ``iimpi/2020a``

<a anchor="mafft"/>
### *MAFFT*

MAFFT is a multiple sequence alignment program for unix-like operating systems. It offers a range of multiple alignment methods, L-INS-i (accurate; for alignment of <~200 sequences), FFT-NS-2 (fast; for alignment of <~30,000 sequences), etc.

*homepage*: https://mafft.cbrc.jp/alignment/software/

version  |versionsuffix       |toolchain
---------|--------------------|---------------------------------------------------------------------------------------
``7.305``|``-with-extensions``|``foss/2016b``
``7.397``|``-with-extensions``|``intel/2018a``
``7.427``|``-with-extensions``|``foss/2018b``, ``intel/2018b``
``7.429``|``-with-extensions``|``GCC/8.2.0-2.31.1``
``7.453``|``-with-extensions``|``GCC/8.3.0``, ``GCC/9.3.0``, ``gompi/2020a``, ``iccifort/2019.5.281``, ``iimpi/2020a``
``7.470``|``-with-extensions``|``GCC/9.3.0``, ``gompi/2020a``
``7.471``|``-with-extensions``|``iimpi/2020a``
``7.475``|``-with-extensions``|``GCC/10.2.0``, ``gompi/2020b``
``7.487``|``-with-extensions``|``gompi/2021a``
``7.490``|``-with-extensions``|``GCC/10.3.0``, ``GCC/11.2.0``, ``gompi/2021b``

<a anchor="mageck"/>
### *MAGeCK*

Model-based Analysis of Genome-wide CRISPR-Cas9 Knockout (MAGeCK) is a computational tool to identify  important genes from the recent genome-scale CRISPR-Cas9 knockout screens (or GeCKO) technology. MAGeCK is developed  by Wei Li and Han Xu from Dr. Xiaole Shirley Liu's lab at Dana-Farber Cancer Institute, and is being actively updated  by Wei Li lab from Children's National Medical Center.

*homepage*: https://sourceforge.net/p/mageck/wiki/Home/

version    |toolchain
-----------|--------------
``0.5.9.4``|``foss/2021a``

<a anchor="magick"/>
### *magick*

R bindings to the open-source image processing library ImageMagick

*homepage*: https://cran.r-project.org/web/packages/magick/

version|versionsuffix|toolchain
-------|-------------|--------------
``2.0``|``-R-3.5.1`` |``foss/2018b``

<a anchor="magics"/>
### *Magics*

Magics is ECMWF's meteorological plotting software and can be either accessed directly through its Python or Fortran interfaces or by using Metview.

*homepage*: https://confluence.ecmwf.int/display/MAGP/

version   |toolchain
----------|---------------
``4.13.0``|``gompi/2022a``

<a anchor="magma"/>
### *MAGMA*

MAGMA is a tool for gene analysis and generalized gene-set analysis of GWAS data. It can be used to analyse both raw genotype data as well as summary SNP p-values from a previous  GWAS or meta-analysis.

*homepage*: https://ctg.cncr.nl/software/magma

version   |toolchain
----------|--------------
``1.07b`` |``foss/2018b``
``1.07bb``|``GCC/8.3.0``
``1.09b`` |``GCC/11.2.0``

<a anchor="magma"/>
### *magma*

The MAGMA project aims to develop a dense linear algebra library similar to  LAPACK but for heterogeneous/hybrid architectures, starting with current Multicore+GPU systems.

*homepage*: https://icl.cs.utk.edu/magma/

version  |versionsuffix   |toolchain
---------|----------------|------------------------------------------------------------------------------------------------------------------------
``2.3.0``|                |``fosscuda/2017b``, ``intelcuda/2017b``
``2.3.0``|``-CUDA-9.1.85``|``foss/2018a``
``2.4.0``|                |``fosscuda/2018b``
``2.5.0``|                |``fosscuda/2018b``
``2.5.1``|                |``fosscuda/2019a``, ``fosscuda/2019b``
``2.5.4``|                |``fosscuda/2019a``, ``fosscuda/2019b``, ``fosscuda/2020a``, ``fosscuda/2020b``, ``intelcuda/2019b``, ``intelcuda/2020b``
``2.6.1``|``-CUDA-11.3.1``|``foss/2021a``
``2.6.2``|``-CUDA-11.4.1``|``foss/2021b``
``2.6.2``|``-CUDA-11.7.0``|``foss/2022a``

<a anchor="magrespython"/>
### *MagresPython*

MagresPython is a Python library for parsing the CCP-NC ab-initio magnetic resonance file format.  This is used in the latest version of the CASTEP and Quantum ESPRESSO (PWSCF) codes. 

*homepage*: http://tfgg.me/magres-python

version     |versionsuffix     |toolchain
------------|------------------|--------------
``20160329``|``-Python-2.7.15``|``foss/2018b``

<a anchor="mahotas"/>
### *mahotas*

Mahotas is a computer vision and image processing library for Python

*homepage*: https://mahotas.readthedocs.io/

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.4.3``|``-Python-2.7.12``|``intel/2016b``

<a anchor="majiq"/>
### *MAJIQ*

MAJIQ and Voila are two software packages that together detect, quantify, and visualize  local splicing variations (LSV) from RNA-Seq data.

*homepage*: https://majiq.biociphers.org

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``1.1.1``|``-Python-3.6.4``|``intel/2018a``

<a anchor="make"/>
### *make*

GNU version of make utility

*homepage*: https://www.gnu.org/software/make/make.html

version  |toolchain
---------|-------------------------------------------------------------------------------------------------
``3.82`` |``GCC/4.8.2``
``4.1``  |``GCC/4.9.2``
``4.2.1``|``GCC/7.3.0-2.30``, ``GCCcore/8.3.0``
``4.3``  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/9.3.0``

<a anchor="makedepend"/>
### *makedepend*

The makedepend package contains a C-preprocessor like utility to determine build-time dependencies.

*homepage*: https://linux.die.net/man/1/makedepend

version  |toolchain
---------|-----------------------------------------------------------------------------------------------
``1.0.5``|``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``1.0.6``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/7.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.0.7``|``GCCcore/11.3.0``

<a anchor="makedepf90"/>
### *makedepf90*

Makedepf90 is a program for automatic creation of  Makefile-style dependency lists for Fortran source code. 

*homepage*: https://linux.die.net/man/1/makedepf90

version  |toolchain
---------|--------------
``2.8.8``|``foss/2017a``

<a anchor="makeinfo"/>
### *makeinfo*

makeinfo is part of the Texinfo project, the official documentation format of the GNU project.

*homepage*: https://www.gnu.org/software/texinfo/

version|versionsuffix|toolchain
-------|-------------|-------------------------------------------------------------------------------------------
``6.7``|             |``FCC/4.5.0``, ``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``6.7``|``-minimal`` |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``6.8``|             |``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="mako"/>
### *Mako*

A super-fast templating language that borrows the best ideas from the existing templating languages

*homepage*: https://www.makotemplates.org

version  |versionsuffix     |toolchain
---------|------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.0.4``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``1.0.6``|``-Python-2.7.13``|``foss/2017a``, ``intel/2017a``
``1.0.7``|``-Python-2.7.14``|``foss/2017b``, ``foss/2018a``, ``fosscuda/2017b``, ``fosscuda/2018a``, ``intel/2017b``, ``intel/2018.01``, ``intel/2018a``, ``intelcuda/2017b``, ``iomkl/2018a``
``1.0.7``|``-Python-2.7.15``|``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``1.0.7``|``-Python-3.6.4`` |``intel/2018a``
``1.0.7``|``-Python-3.6.6`` |``fosscuda/2018b``
``1.0.8``|                  |``GCCcore/8.2.0``
``1.1.0``|                  |``GCCcore/8.3.0``
``1.1.2``|                  |``GCCcore/9.3.0``
``1.1.3``|                  |``GCCcore/10.2.0``
``1.1.4``|                  |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.2.0``|                  |``GCCcore/11.3.0``
``1.2.4``|                  |``GCCcore/12.2.0``

<a anchor="mamba"/>
### *Mamba*

Mamba is a fast, robust, and cross-platform package manager. It runs on Windows, OS X and Linux  (ARM64 and PPC64LE included) and is fully compatible with conda packages and supports most of conda's commands. 

*homepage*: https://mamba.readthedocs.io/

version     |toolchain
------------|----------
``4.14.0-0``|``system``

<a anchor="manta"/>
### *manta*

Manta calls structural variants (SVs) and indels from mapped paired-end sequencing reads. It is optimized for analysis of germline variation in small sets of individuals and  somatic variation in tumor/normal sample pairs. Manta discovers, assembles and  scores large-scale SVs, medium-sized indels and large insertions within a  single efficient workflow.  

*homepage*: https://github.com/Illumina/manta

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``1.1.1``|                  |``system``
``1.6.0``|                  |``system``
``1.6.0``|``-Python-2.7.16``|``gompi/2019b``
``1.6.0``|``-Python-2.7.18``|``GCC/10.2.0``, ``gompi/2020a``

<a anchor="mapdamage"/>
### *mapDamage*

mapDamage2 is a computational framework written in Python and     R, which tracks and quantifies DNA damage patterns among ancient DNA     sequencing reads generated by Next-Generation Sequencing platforms.

*homepage*: https://ginolhac.github.io/mapDamage/

version  |versionsuffix|toolchain
---------|-------------|--------------
``2.2.1``|             |``foss/2021b``
``2.2.1``|``-R-4.0.4`` |``foss/2020b``
``2.2.1``|``-R-4.1.0`` |``foss/2021a``

<a anchor="maple"/>
### *Maple*

Maple combines the world's most powerful mathematical computation engine with an intuitive,    'clickable' user interface.

*homepage*: http://www.maplesoft.com/products/maple/

version   |toolchain
----------|----------
``15``    |``system``
``2017.2``|``system``

<a anchor="mapsplice"/>
### *MapSplice*

MapSplice is a software for mapping RNA-seq data to reference genome for splice  junction discovery that depends only on reference genome, and not on any further annotations.

*homepage*: http://www.netlab.uky.edu/p/bioinfo/MapSplice2

version  |versionsuffix     |toolchain
---------|------------------|---------------
``2.2.1``|``-Python-2.7.12``|``foss/2016b``
``2.2.1``|``-Python-2.7.15``|``intel/2018b``

<a anchor="maq"/>
### *Maq*

Maq is a software that builds mapping assemblies from short reads generated by the  next-generation sequencing  machines.

*homepage*: http://maq.sourceforge.net/maq-man.shtml

version  |toolchain
---------|----------
``0.7.0``|``system``

<a anchor="mariadb"/>
### *MariaDB*

MariaDB is an enhanced, drop-in replacement for MySQL. Included engines: myISAM, Aria, InnoDB, RocksDB, TokuDB, OQGraph, Mroonga.

*homepage*: https://mariadb.org/

version    |toolchain
-----------|-------------------------------
``10.0.21``|``GNU/4.9.3-2.25``
``10.1.13``|``intel/2016a``
``10.1.14``|``foss/2016a``, ``intel/2016a``
``10.1.17``|``intel/2016b``
``10.1.24``|``intel/2017a``
``10.2.11``|``foss/2017b``, ``intel/2017b``
``10.3.7`` |``intel/2018a``
``10.3.10``|``foss/2018b``
``10.3.14``|``foss/2019a``
``10.4.13``|``gompi/2019b``
``10.5.8`` |``GCC/10.2.0``
``10.6.4`` |``GCC/10.3.0``, ``GCC/11.2.0``
``10.9.3`` |``GCC/11.3.0``

<a anchor="mariadb-connector-c"/>
### *MariaDB-connector-c*

MariaDB Connector/C is used to connect applications developed in C/C++ to MariaDB and MySQL databases.

*homepage*: https://downloads.mariadb.org/connector-c/

version   |toolchain
----------|----------------------------------------------------
``2.3.4`` |``foss/2017b``, ``intel/2017b``
``2.3.5`` |``intel/2018a``, ``iomkl/2018a``
``2.3.7`` |``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``foss/2018b``
``3.1.7`` |``GCCcore/9.3.0``
``3.1.11``|``GCCcore/10.2.0``
``3.2.2`` |``GCCcore/10.3.0``

<a anchor="mars"/>
### *MARS*

improving Multiple circular sequence Alignment using Refined Sequences

*homepage*: https://github.com/lorrainea/MARS

version     |toolchain
------------|-----------------
``20191101``|``GCCcore/8.3.0``

<a anchor="mash"/>
### *Mash*

Fast genome and metagenome distance estimation using MinHash

*homepage*: http://mash.readthedocs.org

version|toolchain
-------|------------------------------------------------------------
``2.0``|``foss/2018a``
``2.1``|``foss/2018b``
``2.2``|``GCC/9.3.0``
``2.3``|``GCC/10.3.0``, ``GCC/11.2.0``, ``intel-compilers/2021.4.0``

<a anchor="masurca"/>
### *MaSuRCA*

MaSuRCA is whole genome assembly software. It combines the efficiency of the de Bruijn graph  and Overlap-Layout-Consensus (OLC) approaches. MaSuRCA can assemble data sets containing  only short reads from Illumina sequencing or a mixture of short reads and long reads  (Sanger, 454, Pacbio and Nanopore).

*homepage*: https://www.genome.umd.edu/masurca.html

version  |versionsuffix   |toolchain
---------|----------------|--------------
``3.2.2``|                |``foss/2016a``
``3.2.2``|``-Perl-5.26.0``|``foss/2017b``
``3.2.4``|                |``foss/2018a``
``3.2.5``|``-Perl-5.26.0``|``foss/2017b``
``3.3.1``|``-Perl-5.28.0``|``foss/2018b``
``4.0.1``|``-Perl-5.30.2``|``foss/2020a``
``4.0.9``|``-Perl-5.32.1``|``foss/2021a``

<a anchor="mathematica"/>
### *Mathematica*

Mathematica is a computational software program used in many scientific, engineering, mathematical and computing fields.

*homepage*: http://www.wolfram.com/mathematica

version   |toolchain
----------|----------
``9.0.1`` |``system``
``10.0.2``|``system``
``10.1.0``|``system``
``10.4.1``|``system``
``11.0.1``|``system``
``11.1.1``|``system``
``11.3.0``|``system``
``12.0.0``|``system``
``12.1.1``|``system``
``13.0.0``|``system``

<a anchor="mathgl"/>
### *MathGL*

MathGL is ...  a library for making high-quality scientific graphics under Linux and Windows;  a library for the fast data plotting and data processing of large data arrays;  a library for working in window and console modes and for easy embedding into other programs;  a library with large and growing set of graphics. 

*homepage*: http://mathgl.sourceforge.net/doc_en/Main.html

version  |toolchain
---------|--------------
``2.4.1``|``foss/2018a``

<a anchor="matio"/>
### *MATIO*

matio is an C library for reading and writing Matlab MAT files.

*homepage*: http://sourceforge.net/projects/matio/

version   |toolchain
----------|------------------
``1.5.9`` |``GCCcore/5.4.0``
``1.5.11``|``foss/2017b``
``1.5.12``|``GCCcore/6.4.0``
``1.5.17``|``GCCcore/8.3.0``
``1.5.19``|``GCCcore/9.3.0``
``1.5.21``|``GCCcore/10.2.0``

<a anchor="matlab"/>
### *MATLAB*

MATLAB is a high-level language and interactive environment  that enables you to perform computationally intensive tasks faster than with  traditional programming languages such as C, C++, and Fortran.

*homepage*: https://www.mathworks.com/products/matlab

version  |toolchain
---------|----------
``2012b``|``system``
``2013b``|``system``
``2015a``|``system``
``2016a``|``system``
``2017a``|``system``
``2018b``|``system``
``2019b``|``system``
``2020a``|``system``
``2020b``|``system``
``2021a``|``system``
``2021b``|``system``

<a anchor="matlab-engine"/>
### *MATLAB-Engine*

The MATLAB Engine API for Python provides a package for Python  to call MATLAB as a computational engine.

*homepage*: https://www.mathworks.com/help/matlab/matlab-engine-for-python.html

version          |versionsuffix     |toolchain
-----------------|------------------|-------------------------------
``2018b``        |``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``2018b``        |``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``2019b``        |                  |``GCCcore/8.3.0``
``2021a-9.10.1`` |                  |``GCCcore/10.2.0``
``2021b-9.11.19``|                  |``GCCcore/11.2.0``

<a anchor="matlab-proxy"/>
### *matlab-proxy*

A Python package which enables you to launch MATLAB and access it from a web browser.

*homepage*: https://github.com/mathworks/matlab-proxy

version  |toolchain
---------|------------------
``0.5.4``|``GCCcore/11.3.0``

<a anchor="matplotlib"/>
### *matplotlib*

matplotlib is a python 2D plotting library which produces publication quality figures in a variety of  hardcopy formats and interactive environments across platforms. matplotlib can be used in python scripts, the python  and ipython shell, web application servers, and six graphical user interface toolkits.

*homepage*: https://matplotlib.org

version  |versionsuffix                    |toolchain
---------|---------------------------------|-----------------------------------------------------------------------------------
``1.4.3``|``-Python-2.7.12``               |``intel/2016b``
``1.5.1``|``-Python-2.7.11``               |``foss/2016a``, ``intel/2016a``
``1.5.1``|``-Python-2.7.11-freetype-2.6.3``|``foss/2016a``, ``intel/2016a``
``1.5.1``|``-Python-2.7.12``               |``intel/2016b``
``1.5.1``|``-Python-3.5.1``                |``foss/2016a``, ``intel/2016a``
``1.5.1``|``-Python-3.5.2``                |``intel/2016b``
``1.5.2``|``-Python-2.7.12``               |``foss/2016b``, ``intel/2016b``
``1.5.2``|``-Python-3.5.2``                |``intel/2016b``
``1.5.3``|``-Python-2.7.12``               |``foss/2016b``, ``intel/2016b``
``1.5.3``|``-Python-3.5.2``                |``foss/2016b``, ``intel/2016b``
``2.0.0``|``-Python-2.7.12``               |``foss/2016b``, ``intel/2016b``
``2.0.0``|``-Python-2.7.13``               |``intel/2017a``
``2.0.1``|``-Python-3.6.1``                |``intel/2017a``
``2.0.2``|``-Python-2.7.13``               |``foss/2017a``, ``intel/2017a``
``2.0.2``|``-Python-2.7.13-Qt-4.8.7``      |``intel/2017a``
``2.0.2``|``-Python-2.7.13-libpng-1.6.29`` |``intel/2017a``
``2.0.2``|``-Python-3.6.1``                |``foss/2017a``
``2.0.2``|``-Python-3.6.1-libpng-1.6.29``  |``intel/2017a``
``2.1.0``|``-Python-2.7.14``               |``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``2.1.0``|``-Python-3.6.2``                |``foss/2017b``
``2.1.0``|``-Python-3.6.3``                |``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``2.1.1``|``-Python-2.7.14``               |``intel/2017b``
``2.1.1``|``-Python-3.6.3``                |``foss/2017b``, ``intel/2017b``
``2.1.2``|``-Python-2.7.14``               |``foss/2018a``, ``intel/2018a``
``2.1.2``|``-Python-3.6.4``                |``foss/2018a``, ``intel/2018a``, ``iomkl/2018.02``, ``iomkl/2018a``
``2.2.3``|``-Python-2.7.15``               |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``2.2.3``|``-Python-3.6.6``                |``intel/2018b``
``2.2.4``|``-Python-2.7.15``               |``foss/2019a``, ``fosscuda/2019a``, ``intel/2019a``, ``intelcuda/2019a``
``2.2.4``|``-Python-2.7.16``               |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``
``2.2.5``|``-Python-2.7.16``               |``foss/2019b``
``2.2.5``|``-Python-2.7.18``               |``foss/2020a``, ``foss/2020b``, ``foss/2021b``, ``fosscuda/2020a``, ``intel/2020a``
``3.0.0``|``-Python-3.6.6``                |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``, ``iomkl/2018b``
``3.0.2``|``-Python-3.6.6``                |``foss/2018b``
``3.0.3``|``-Python-3.7.2``                |``foss/2019a``, ``fosscuda/2019a``, ``intel/2019a``, ``intelcuda/2019a``
``3.1.1``|``-Python-3.7.4``                |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``, ``intelcuda/2019b``
``3.2.1``|``-Python-3.8.2``                |``foss/2020a``, ``fosscuda/2020a``, ``intel/2020a``, ``intelcuda/2020a``
``3.3.3``|                                 |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``, ``intelcuda/2020b``
``3.4.2``|                                 |``foss/2021a``, ``gomkl/2021a``, ``intel/2021a``
``3.4.3``|                                 |``foss/2021b``, ``intel/2021b``
``3.5.1``|                                 |``foss/2020b``, ``intel/2020b``
``3.5.2``|                                 |``foss/2021b``, ``foss/2022a``, ``intel/2022a``

<a anchor="matplotlib-inline"/>
### *matplotlib-inline*

Matplotlib Inline Back-end for IPython and Jupyter.

*homepage*: https://github.com/ipython/matplotlib-inline

version  |toolchain
---------|------------------
``0.1.3``|``GCCcore/10.2.0``

<a anchor="matsim"/>
### *MATSim*

MATSim is an open-source framework to implement large-scale agent-based transport simulations.

*homepage*: https://www.matsim.org/

version  |versionsuffix      |toolchain
---------|-------------------|------------------------------
``0.8.1``|``-Java-1.8.0_112``|``intel/2016b``
``14.0`` |``-Java-11``       |``GCCcore/11.2.0``, ``system``

<a anchor="maude"/>
### *Maude*

Maude is a high-performance reflective language and system supporting both equational and rewriting  logic specification and programming for a wide range of applications. Maude has been influenced in important ways by  the OBJ3 language, which can be regarded as an equational logic sublanguage. Besides supporting equational  specification and programming, Maude also supports rewriting logic computation. 

*homepage*: https://maude.cs.illinois.edu/w/index.php/The_Maude_System

version|toolchain
-------|------------------
``3.1``|``GCCcore/10.2.0``

<a anchor="mauvealigner"/>
### *mauveAligner*

Mauve is a system for constructing multiple genome alignments in the presence of large-scale evolutionary events such as rearrangement and inversion. Multiple genome alignments provide a basis for research into comparative genomics and the study of genome-wide evolutionary dynamics. This version was built without Graphical User Interface. 

*homepage*: https://darlinglab.org/mauve/mauve.html

version |toolchain
--------|---------------
``4736``|``gompi/2020a``

<a anchor="maven"/>
### *Maven*

Binary maven install, Apache Maven is a software project management and comprehension tool. Based on the concept of a project object model (POM), Maven can manage a project's build, reporting and documentation from a central piece of information. 

*homepage*: https://maven.apache.org/index.html

version  |toolchain
---------|----------
``3.2.3``|``system``
``3.3.3``|``system``
``3.3.9``|``system``
``3.5.0``|``system``
``3.5.2``|``system``
``3.6.0``|``system``
``3.6.3``|``system``

<a anchor="mawk"/>
### *mawk*

mawk is an interpreter for the AWK Programming Language.

*homepage*: https://invisible-island.net/mawk/

version           |toolchain
------------------|-------------------------------------------------------------------------------------------------
``1.3.4-20141206``|``GCC/4.9.2``
``1.3.4-20171017``|``foss/2018a``, ``foss/2018b``, ``foss/2019a``, ``intel/2018a``, ``intel/2018b``, ``intel/2019a``

<a anchor="maxbin"/>
### *MaxBin*

MaxBin is software for binning assembled metagenomic sequences based on an Expectation-Maximization algorithm.

*homepage*: https://sourceforge.net/projects/maxbin/

version  |versionsuffix   |toolchain
---------|----------------|-------------------------------------------------
``2.2.6``|``-Perl-5.28.0``|``foss/2018b``
``2.2.7``|                |``gompi/2020b``, ``gompi/2021a``, ``gompi/2021b``
``2.2.7``|``-Perl-5.28.1``|``GCC/8.2.0-2.31.1``

<a anchor="maxquant"/>
### *MaxQuant*

MaxQuant is a quantitative proteomics software package designed for analyzing large  mass-spectrometric data sets. It is specifically aimed at high-resolution MS data. Several labeling  techniques as well as label-free quantification are supported.

*homepage*: https://maxquant.org/maxquant

version      |toolchain
-------------|------------------
``1.6.10.43``|``foss/2018b``
``2.0.3.0``  |``GCCcore/11.2.0``

<a anchor="mayavi"/>
### *mayavi*

The Mayavi scientific data 3-dimensional visualizer

*homepage*: http://code.enthought.com/projects/mayavi/

version  |versionsuffix     |toolchain
---------|------------------|---------------
``4.4.4``|``-Python-2.7.11``|``intel/2016a``
``4.6.2``|``-Python-3.6.6`` |``foss/2018b``
``4.7.1``|``-Python-2.7.15``|``foss/2019a``
``4.7.1``|``-Python-3.7.2`` |``foss/2019a``
``4.7.4``|                  |``foss/2021b``

<a anchor="maze"/>
### *maze*

In a similar manner to dot plots, maze highlights local sequence similarity between two DNA sequences. In particular, maximal exact substring matches are computed with MUMmer3 and visualised.

*homepage*: https://github.com/dellytools/maze

version     |toolchain
------------|--------------
``20170124``|``foss/2020b``

<a anchor="mbedtls"/>
### *MbedTLS*

An open source, portable, easy to use, readable and flexible SSL library.

*homepage*: https://tls.mbed.org/

version   |toolchain
----------|------------------
``2.26.0``|``GCCcore/10.2.0``

<a anchor="mbrola"/>
### *MBROLA*

MBROLA is a speech synthesizer based on the concatenation of diphones. It takes a list of phonemes as input, together with prosodic information (duration of phonemes and a piecewise linear description of pitch), and produces speech samples on 16 bits (linear), at the sampling frequency of the diphone database.  MBROLA voices project provides list of MBROLA speech synthesizer voices. It is intended to provide easier collaboration and automatic updates for individual users and packagers. 

*homepage*: ['https://github.com/numediart/MBROLA', 'https://github.com/numediart/MBROLA-voices']

version|versionsuffix       |toolchain
-------|--------------------|-----------------
``3.3``|``-voices-20200330``|``GCCcore/9.3.0``

<a anchor="mbuffer"/>
### *mbuffer*

mbuffer is a tool for buffering data streams with a large set of unique features. 

*homepage*: https://www.maier-komor.de/mbuffer.html

version     |toolchain
------------|-----------------
``20191016``|``GCCcore/9.3.0``

<a anchor="mc"/>
### *mc*

mc-4.6.1: User-friendly file manager and visual shell

*homepage*: https://www.midnight-commander.org/

version   |toolchain
----------|-------------
``4.8.13``|``GCC/4.9.2``

<a anchor="mcl"/>
### *MCL*

The MCL algorithm is short for the Markov Cluster Algorithm, a fast and scalable unsupervised cluster algorithm for graphs (also known as networks) based on simulation of (stochastic) flow in graphs. 

*homepage*: http://micans.org/mcl/

version   |versionsuffix   |toolchain
----------|----------------|-------------------------------------------------------------------------------------------------------------
``02.063``|                |``intel/2016b``
``14.137``|                |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``foss/2016a``, ``intel/2016b``
``14.137``|``-Perl-5.26.1``|``GCCcore/6.4.0``
``14.137``|``-Perl-5.28.0``|``GCCcore/7.3.0``

<a anchor="mcr"/>
### *MCR*

The MATLAB Runtime is a standalone set of shared libraries  that enables the execution of compiled MATLAB applications  or components on computers that do not have MATLAB installed.

*homepage*: https://www.mathworks.com/products/compiler/mcr/

version   |versionsuffix|toolchain
----------|-------------|----------
``R2013a``|             |``system``
``R2013b``|             |``system``
``R2014a``|             |``system``
``R2014b``|             |``system``
``R2015a``|             |``system``
``R2015b``|             |``system``
``R2016a``|             |``system``
``R2016b``|             |``system``
``R2018a``|             |``system``
``R2018b``|             |``system``
``R2019a``|             |``system``
``R2019b``|``.8``       |``system``
``R2020a``|``.6``       |``system``
``R2020b``|``.5``       |``system``
``R2021a``|``.0``       |``system``
``R2021a``|``.3``       |``system``
``R2021b``|             |``system``
``R2021b``|``.1``       |``system``
``R2021b``|``.2``       |``system``
``R2022a``|             |``system``
``R2022a``|``.1``       |``system``

<a anchor="mcu"/>
### *mcu*

A package for periodic wavefunction and crystallography analysis.  mcu is designed to support large scale analysis and topological descriptions for periodic wavefunction.

*homepage*: https://hungqpham.com/mcu/

version       |toolchain
--------------|---------------
``2021-04-06``|``gomkl/2021a``

<a anchor="mdanalysis"/>
### *MDAnalysis*

MDAnalysis is an object-oriented Python library to analyze trajectories from molecular dynamics (MD) simulations in many popular formats.

*homepage*: https://www.mdanalysis.org/

version   |versionsuffix    |toolchain
----------|-----------------|-----------------------------------------------
``0.20.1``|``-Python-3.7.4``|``foss/2019b``, ``intel/2019b``
``1.1.1`` |                 |``foss/2020b``
``2.0.0`` |                 |``foss/2021a``, ``foss/2021b``, ``intel/2021b``
``2.2.0`` |                 |``foss/2022a``

<a anchor="mdbm"/>
### *MDBM*

MDBM is a super-fast memory-mapped key/value store

*homepage*: https://github.com/yahoo/mdbm

version   |toolchain
----------|------------------------------------
``4.13.0``|``GCCcore/6.4.0``, ``GCCcore/9.3.0``

<a anchor="mdsplus"/>
### *MDSplus*

MDSplus is a set of software tools for data acquisition and storage and a methodology  for management of complex scientific data.

*homepage*: https://mdsplus.org/

version    |versionsuffix                   |toolchain
-----------|--------------------------------|-----------------
``7.0.67`` |``-Java-1.7.0_79-Python-2.7.11``|``foss/2016a``
``7.46.1`` |                                |``foss/2018a``
``7.96.8`` |                                |``GCCcore/9.3.0``
``7.96.12``|                                |``GCCcore/9.3.0``

<a anchor="mdsplus-java"/>
### *MDSplus-Java*

MDSplus is a set of software tools for data acquisition and storage and a methodology  for management of complex scientific data.

*homepage*: https://mdsplus.org

version    |versionsuffix|toolchain
-----------|-------------|-----------------
``7.96.12``|``-Java-13`` |``GCCcore/9.3.0``

<a anchor="mdsplus-python"/>
### *MDSplus-Python*

MDSplus is a set of software tools for data acquisition and storage and a methodology  for management of complex scientific data.

*homepage*: https://mdsplus.org

version    |versionsuffix    |toolchain
-----------|-----------------|--------------
``7.96.12``|``-Python-3.8.2``|``foss/2020a``

<a anchor="mdtest"/>
### *mdtest*

mdtest is an MPI-coordinated metadata benchmark test that performs   open/stat/close operations on files and directories and then reports the performance.

*homepage*: http://sourceforge.net/projects/mdtest/

version  |toolchain
---------|---------------
``1.9.3``|``intel/2017a``

<a anchor="mdtraj"/>
### *MDTraj*

Read, write and analyze MD trajectories with only a few lines of Python code.

*homepage*: https://mdtraj.org

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------------------------------------
``1.9.1``|``-Python-3.6.3``|``intel/2017b``
``1.9.2``|``-Python-3.6.6``|``intel/2018b``
``1.9.3``|``-Python-3.7.4``|``foss/2019b``, ``intel/2019b``
``1.9.4``|``-Python-3.8.2``|``foss/2020a``, ``intel/2020a``
``1.9.5``|                 |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``
``1.9.7``|                 |``foss/2021a``, ``foss/2022a``, ``intel/2021b``, ``intel/2022a``

<a anchor="medaka"/>
### *medaka*

medaka is a tool to create a consensus sequence from nanopore sequencing data.

*homepage*: https://github.com/nanoporetech/medaka

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.4.3`` |``-Python-3.6.6``|``foss/2018b``
``0.11.4``|``-Python-3.6.6``|``foss/2018b``
``0.12.0``|``-Python-3.6.6``|``foss/2018b``
``1.1.1`` |``-Python-3.7.4``|``foss/2019b``
``1.1.3`` |``-Python-3.7.4``|``foss/2019b``
``1.2.0`` |``-Python-3.7.4``|``foss/2019b``
``1.4.3`` |                 |``foss/2020b``
``1.4.3`` |``-Python-3.7.4``|``foss/2019b``
``1.5.0`` |                 |``foss/2021a``
``1.6.0`` |                 |``foss/2021b``

<a anchor="medimgproc"/>
### *medImgProc*

Motion correction, explicit spatio-temporal regularization of motion tracking, random speckles enhancement, and segmentation.

*homepage*: https://github.com/WeiXuanChan/motionSegmentation

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``2.5.7``|``-Python-3.7.4``|``foss/2019b``

<a anchor="medpy"/>
### *MedPy*

MedPy is a library and script collection for medical image processing in Python, providing basic  functionalities for reading, writing and manipulating large images of arbitrary dimensionality. Its main  contributions are n-dimensional versions of popular image filters, a collection of image feature extractors, ready  to be used with scikit-learn, and an exhaustive n-dimensional graph-cut package.

*homepage*: https://pypi.org/project/MedPy/

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.4.0``|                 |``foss/2020b``
``0.4.0``|``-Python-3.7.4``|``foss/2019b``

<a anchor="meep"/>
### *Meep*

Meep (or MEEP) is a free finite-difference time-domain (FDTD) simulation software package  developed at MIT to model electromagnetic systems.

*homepage*: https://meep.readthedocs.io/en/latest/

version  |versionsuffix     |toolchain
---------|------------------|-----------------------------------------------
``1.3``  |                  |``foss/2016a``
``1.4.3``|                  |``intel/2020a``
``1.6.0``|``-Python-2.7.14``|``foss/2017b``, ``foss/2018a``, ``intel/2018a``

<a anchor="mega"/>
### *MEGA*

MEGA-CC (Molecular Evolutionary Genetics Analysis Computational Core) is an integrated suite of tools  for statistics-based comparative analysis of molecular sequence data based on evolutionary principles.

*homepage*: http://www.megasoftware.net/

version     |toolchain
------------|----------
``7.0.20-1``|``system``
``10.0.5``  |``system``
``11.0.10`` |``system``

<a anchor="megacc"/>
### *MEGACC*

MEGA-Computing Core - Sophisticated and user-friendly software suite for analyzing DNA and   protein sequence data from species and populations.

*homepage*: http://www.megasoftware.net

version     |toolchain
------------|----------
``7.0.18-1``|``system``

<a anchor="megahit"/>
### *MEGAHIT*

An ultra-fast single-node solution for large and complex  metagenomics assembly via succinct de Bruijn graph

*homepage*: https://github.com/voutcn/megahit

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------------------------------------------------
``1.1.2``|``-Python-2.7.14``|``foss/2018a``
``1.1.3``|``-Python-2.7.14``|``foss/2017b``, ``foss/2018a``
``1.1.3``|``-Python-3.6.3`` |``foss/2017b``
``1.1.4``|``-Python-2.7.15``|``foss/2018b``
``1.1.4``|``-Python-3.6.6`` |``foss/2018b``
``1.2.8``|                  |``GCCcore/8.2.0``
``1.2.9``|                  |``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``, ``foss/2018b``
``1.2.9``|``-Python-2.7.18``|``GCCcore/10.2.0``

<a anchor="megalodon"/>
### *Megalodon*

Megalodon is a research command line tool to extract high accuracy modified base and sequence variant calls from raw nanopore reads by anchoring the information rich basecalling neural network output to a reference genome/transriptome.

*homepage*: https://github.com/nanoporetech/megalodon

version  |versionsuffix   |toolchain
---------|----------------|----------------------------------
``2.3.5``|                |``foss/2020b``, ``fosscuda/2020b``
``2.5.0``|                |``foss/2021a``
``2.5.0``|``-CUDA-11.3.1``|``foss/2021a``

<a anchor="megan"/>
### *MEGAN*

MEGAN is a comprehensive toolbox for interactively analyzing microbiome data

*homepage*: http://megan.informatik.uni-tuebingen.de/

version   |versionsuffix|toolchain
----------|-------------|----------
``6.22.0``|``-Java-17`` |``system``

<a anchor="meld"/>
### *Meld*

Meld is a visual diff and merge tool targeted at developers.   Meld helps you compare files, directories, and version controlled projects.   It provides two- and three-way comparison of both files and directories, and has support   for many popular version control systems.   

*homepage*: https://meldmerge.org/

version   |versionsuffix    |toolchain
----------|-----------------|-----------------
``3.20.1``|``-Python-3.7.2``|``GCCcore/8.2.0``

<a anchor="mem"/>
### *MEM*

Marker Enrichment Modeling (MEM) is a tool designed to calculate enrichment scores.

*homepage*: https://github.com/cytolab/mem

version     |versionsuffix|toolchain
------------|-------------|--------------
``20191023``|             |``foss/2019b``
``20191023``|``-R-4.0.0`` |``foss/2020a``

<a anchor="meme"/>
### *MEME*

The MEME Suite allows you to: * discover motifs using MEME, DREME (DNA only) or  GLAM2 on groups of related DNA or protein sequences, * search sequence databases with motifs using  MAST, FIMO, MCAST or GLAM2SCAN, * compare a motif to all motifs in a database of motifs, * associate  motifs with Gene Ontology terms via their putative target genes, and * analyse motif enrichment  using SpaMo or CentriMo.

*homepage*: https://meme-suite.org/meme/index.html

version  |versionsuffix                 |toolchain
---------|------------------------------|-------------------------------
``5.0.4``|``-Perl-5.26.0-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``5.0.4``|``-Perl-5.26.0-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``5.1.1``|``-Python-3.6.6``             |``foss/2018b``
``5.1.1``|``-Python-3.7.4``             |``foss/2019b``, ``intel/2019b``
``5.4.1``|                              |``GCC/10.3.0``, ``gompi/2021b``
``5.4.1``|``-Python-2.7.18``            |``gompi/2021b``

<a anchor="memkind"/>
### *memkind*

User Extensible Heap Manager built on top of jemalloc which enables control of memory characteristics  and a partitioning of the heap between kinds of memory. 

*homepage*: http://memkind.github.io

version  |toolchain
---------|-----------------
``1.5.0``|``GCCcore/5.4.0``

<a anchor="memory-profiler"/>
### *memory-profiler*

memory-profiler is a Python module for monitoring memory consumption of a process  as well as line-by-line analysis of memory consumption for python programs.

*homepage*: https://pypi.org/project/memory-profiler

version   |toolchain
----------|-------------------------------
``0.55.0``|``foss/2019a``, ``intel/2019a``

<a anchor="memote"/>
### *MEMOTE*

The genome-scale metabolic model test suite

*homepage*: https://github.com/opencobra/memote

version   |toolchain
----------|--------------
``0.13.0``|``foss/2021a``

<a anchor="memtester"/>
### *memtester*

A userspace utility for testing the memory subsystem for faults

*homepage*: https://pyropus.ca./software/memtester/

version  |toolchain
---------|------------------
``4.5.1``|``GCCcore/11.3.0``

<a anchor="merantk"/>
### *meRanTK*

meRanTK is a versatile high performance toolkit for complete analysis of methylated RNA data.

*homepage*: http://www.icbi.at/index.html

version   |toolchain
----------|----------
``1.1.1b``|``system``

<a anchor="merckx"/>
### *MERCKX*

Multilingual Entity/Resource Combiner & Knowledge eXtractor

*homepage*: https://github.com/madewild/MERCKX

version     |versionsuffix     |toolchain
------------|------------------|---------------
``20170330``|``-Python-2.7.13``|``intel/2017a``

<a anchor="mercurial"/>
### *Mercurial*

Mercurial is a free, distributed source control management tool. It efficiently handles projects of any size and offers an easy and intuitive interface. 

*homepage*: https://www.mercurial-scm.org

version  |versionsuffix     |toolchain
---------|------------------|------------------
``3.8.3``|``-Python-2.7.11``|``foss/2016a``
``5.7.1``|                  |``GCCcore/10.2.0``
``5.7.1``|``-Python-3.8.2`` |``GCCcore/9.3.0``
``5.8``  |                  |``GCCcore/10.3.0``
``6.2``  |                  |``GCCcore/11.3.0``

<a anchor="mesa"/>
### *Mesa*

Mesa is an open-source implementation of the OpenGL specification -  a system for rendering interactive 3D graphics.

*homepage*: https://www.mesa3d.org/

version   |toolchain
----------|--------------------------------------------------------------------
``11.1.2``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``11.2.1``|``foss/2016a``, ``intel/2016a``
``12.0.2``|``foss/2016b``, ``intel/2016b``
``17.0.2``|``foss/2017a``, ``intel/2017a``
``17.2.4``|``intel/2017b``, ``intelcuda/2017b``
``17.2.5``|``foss/2017b``, ``fosscuda/2017b``
``17.3.6``|``foss/2018a``, ``fosscuda/2018a``, ``intel/2018a``, ``iomkl/2018a``
``18.1.1``|``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``19.0.1``|``GCCcore/8.2.0``
``19.1.7``|``GCCcore/8.3.0``
``19.2.1``|``GCCcore/8.3.0``
``20.0.2``|``GCCcore/9.3.0``
``20.2.1``|``GCCcore/10.2.0``
``21.1.1``|``GCCcore/10.3.0``
``21.1.7``|``GCCcore/11.2.0``
``22.0.3``|``GCCcore/11.3.0``
``22.2.4``|``GCCcore/12.2.0``

<a anchor="mesa-demos"/>
### *Mesa-demos*

Mesa utility and demo programs, including glxinfo and eglinfo.

*homepage*: https://www.mesa3d.org/

version  |toolchain
---------|------------------
``8.4.0``|``GCCcore/10.3.0``

<a anchor="meshalyzer"/>
### *meshalyzer*

Graphical program for display time dependent data on 3D finite element meshes

*homepage*: https://git.opencarp.org/openCARP/meshalyzer

version     |versionsuffix    |toolchain
------------|-----------------|--------------
``2.0``     |``-Python-3.8.2``|``foss/2020a``
``2.2``     |                 |``foss/2020b``
``20200308``|``-Python-3.8.2``|``foss/2020a``

<a anchor="meshio"/>
### *meshio*

meshio is a tool for reading/writing various mesh formats representing unstructured meshes

*homepage*: https://github.com/nschloe/meshio

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.7.1``|``-Python-2.7.12``|``intel/2016b``
``2.0.2``|``-Python-2.7.14``|``intel/2018a``
``2.0.2``|``-Python-3.6.4`` |``intel/2018a``

<a anchor="meshtool"/>
### *meshtool*

Meshtool is a comand-line tool written in C++. It is designed to apply various manipulations to volumetric meshes.

*homepage*: https://bitbucket.org/aneic/meshtool

version|toolchain
-------|-----------------------------
``16`` |``GCC/10.2.0``, ``GCC/9.3.0``

<a anchor="meson"/>
### *Meson*

Meson is a cross-platform build system designed to be both as fast and as user friendly as possible.

*homepage*: https://mesonbuild.com

version   |versionsuffix    |toolchain
----------|-----------------|----------------------------------
``0.43.0``|``-Python-3.6.3``|``intel/2017b``
``0.46.1``|``-Python-3.6.4``|``foss/2018a``
``0.48.1``|``-Python-3.6.4``|``foss/2018a``, ``intel/2018a``
``0.48.1``|``-Python-3.6.6``|``foss/2018b``, ``fosscuda/2018b``
``0.50.0``|``-Python-3.7.2``|``GCCcore/8.2.0``
``0.51.2``|``-Python-3.7.4``|``GCCcore/8.3.0``
``0.53.1``|``-Python-3.6.3``|``intel/2017b``
``0.53.2``|``-Python-3.8.2``|``GCCcore/9.3.0``
``0.55.1``|``-Python-3.8.2``|``GCCcore/9.3.0``
``0.55.3``|                 |``GCCcore/10.2.0``
``0.58.0``|                 |``GCCcore/10.3.0``
``0.58.2``|                 |``GCCcore/11.2.0``
``0.59.1``|``-Python-3.7.4``|``GCCcore/8.3.0``
``0.62.1``|                 |``GCCcore/11.3.0``
``0.64.0``|                 |``GCCcore/12.2.0``

<a anchor="meson-python"/>
### *meson-python*

Python build backend (PEP 517) for Meson projects

*homepage*: https://github.com/mesonbuild/meson-python

version   |toolchain
----------|------------------
``0.11.0``|``GCCcore/12.2.0``

<a anchor="mesquite"/>
### *Mesquite*

Mesh-Quality Improvement Library

*homepage*: https://software.sandia.gov/mesquite/

version  |toolchain
---------|----------------------------------------------------------------------------------------
``2.3.0``|``GCCcore/6.4.0``, ``GCCcore/8.3.0``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2017a``

<a anchor="mess"/>
### *MESS*

Master Equation System Solver (MESS)

*homepage*: https://github.com/PACChem/MESS

version  |toolchain
---------|--------------
``0.1.6``|``foss/2019b``

<a anchor="metabat"/>
### *MetaBAT*

An efficient tool for accurately reconstructing single genomes from complex microbial communities

*homepage*: https://bitbucket.org/berkeleylab/metabat

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------
``2.12.1``|``-Python-2.7.15``|``foss/2018b``
``2.14``  |                  |``gompi/2019a``
``2.15``  |                  |``gompi/2021a``, ``gompi/2021b``
``2.15``  |``-Python-2.7.18``|``gompi/2020b``

<a anchor="metaboanalystr"/>
### *MetaboAnalystR*

MetaboAnalystR contains the R functions and libraries underlying the popular MetaboAnalyst web  server, including > 500 functions for metabolomic data analysis, visualization, and functional interpretation.

*homepage*: https://www.metaboanalyst.ca

version           |versionsuffix|toolchain
------------------|-------------|--------------
``2.0.1-20190827``|``-R-3.6.0`` |``foss/2019a``

<a anchor="metaerg"/>
### *metaerg*

MetaErg is a stand-alone and fully automated metagenomic and metaproteomic data annotation pipeline.

*homepage*: https://github.com/xiaoli-dong/metaerg

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.2.3``|``-Python-2.7.16``|``intel/2019b``

<a anchor="metaeuk"/>
### *MetaEuk*

MetaEuk is a modular toolkit designed for large-scale gene discovery and annotation in eukaryotic  metagenomic contigs.

*homepage*: https://metaeuk.soedinglab.org

version|toolchain
-------|--------------
``4``  |``GCC/10.2.0``
``5``  |``GCC/10.3.0``
``6``  |``GCC/11.2.0``

<a anchor="metageneannotator"/>
### *MetaGeneAnnotator*

MetaGeneAnnotator is a gene-finding program for prokaryote and phage.

*homepage*: http://metagene.nig.ac.jp/

version     |versionsuffix|toolchain
------------|-------------|----------
``20080819``|``-x86-64``  |``system``

<a anchor="metagenome-atlas"/>
### *Metagenome-Atlas*

Metagenome-atlas is a easy-to-use metagenomic pipeline based on snakemake.  It handles all steps from QC, Assembly, Binning, to Annotation.

*homepage*: https://github.com/metagenome-atlas/atlas

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``2.4.3``|``-Python-3.8.2``|``intel/2020a``

<a anchor="metal"/>
### *Metal*

Metal - Meta Analysis Helper. The METAL software is designed to facilitate meta-analysis of large  datasets (such as several whole genome scans) in a convenient, rapid and memory efficient manner.

*homepage*: http://csg.sph.umich.edu/abecasis/Metal/

version       |toolchain
--------------|--------------
``2011-03-25``|``foss/2016a``
``2020-05-05``|``GCC/10.2.0``

<a anchor="metamorpheus"/>
### *MetaMorpheus*

MetaMorpheus is a bottom-up proteomics database search software  with integrated post-translational modification (PTM) discovery capability.  This program combines features of Morpheus and G-PTM-D in a single tool. 

*homepage*: https://github.com/smith-chem-wisc/MetaMorpheus

version    |toolchain
-----------|------------------
``0.0.320``|``GCCcore/10.3.0``

<a anchor="metaphlan"/>
### *MetaPhlAn*

MetaPhlAn is a computational tool for profiling the composition  of microbial communities from metagenomic shotgun sequencing data 

*homepage*: https://github.com/biobakery/MetaPhlAn

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``3.0.9``|``-Python-3.8.2``|``foss/2020a``

<a anchor="metaphlan2"/>
### *MetaPhlAn2*

MetaPhlAn is a computational tool for profiling the composition of microbial  communities (Bacteria, Archaea, Eukaryotes and Viruses) from metagenomic shotgun sequencing  data (i.e. not 16S) with species-level. With the newly added StrainPhlAn module, it is now  possible to perform accurate strain-level microbial profiling.

*homepage*: https://bitbucket.org/biobakery/metaphlan2/

version  |versionsuffix     |toolchain
---------|------------------|--------------
``2.7.8``|``-Python-2.7.12``|``foss/2016b``
``2.7.8``|``-Python-3.6.6`` |``foss/2018b``
``2.7.8``|``-Python-3.8.2`` |``foss/2020a``

<a anchor="metawrap"/>
### *metaWRAP*

MetaWRAP aims to be an easy-to-use metagenomic wrapper suite that accomplishes the core tasks of metagenomic analysis from start to finish: read quality control, assembly, visualization, taxonomic profiling, extracting draft genomes (binning), and functional annotation.

*homepage*: https://github.com/bxlab/metaWRAP

version  |versionsuffix     |toolchain
---------|------------------|--------------
``1.2``  |``-Python-2.7.15``|``foss/2018b``
``1.2.2``|``-Python-2.7.15``|``foss/2019a``
``1.3``  |``-Python-2.7.18``|``foss/2020b``

<a anchor="metaxa2"/>
### *Metaxa2*

Metaxa2 -- Identifies Small Subunit (SSU) rRNAs and classifies them taxonomically

*homepage*: https://microbiology.se/software/metaxa2/

version|toolchain
-------|---------------
``2.2``|``gompi/2019a``

<a anchor="methyldackel"/>
### *MethylDackel*

A (mostly) universal methylation extractor for BS-seq experiments.

*homepage*: https://github.com/dpryan79/MethylDackel

version  |toolchain
---------|----------------------------------------
``0.4.0``|``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``0.5.0``|``iccifort/2019.5.281``
``0.6.1``|``GCC/11.2.0``

<a anchor="methylpy"/>
### *methylpy*

Bisulfite sequencing data processing and differential methylation analysis.

*homepage*: https://pypi.python.org/pypi/methylpy

version  |versionsuffix     |toolchain
---------|------------------|--------------
``1.1.9``|``-Python-2.7.13``|``foss/2017a``
``1.2.9``|                  |``foss/2021b``

<a anchor="metis"/>
### *METIS*

METIS is a set of serial programs for partitioning graphs, partitioning finite element meshes, and producing fill reducing orderings for sparse matrices. The algorithms implemented in METIS are based on the multilevel recursive-bisection, multilevel k-way, and multi-constraint partitioning schemes.

*homepage*: http://glaros.dtc.umn.edu/gkhome/metis/metis/overview

version  |versionsuffix|toolchain
---------|-------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``5.0.2``|             |``gimkl/2.11.5``
``5.1.0``|             |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``foss/2018b``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``
``5.1.0``|``-32bitIDX``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``

<a anchor="mgltools"/>
### *mgltools*

The MGLTools software suite can be used for visualization     and analysis of molecular structures and comprises     the Python Molecular Viewer (PMV, a general purpose molecular viewer),     AutoDockTools (ADT, a set of PMV commands specifically developed to support AutoDock users)     and Vision (a visual programming environment). 

*homepage*: https://ccsb.scripps.edu/mgltools/

version  |toolchain
---------|----------
``1.5.7``|``system``

<a anchor="mhcflurry"/>
### *mhcflurry*

MHCflurry implements class I peptide/MHC binding affinity  prediction. By default it supports 112 MHC alleles using ensembles of  allele-specific models. Pan-allele predictors supporting virtually any MHC  allele of known sequence are available for testing (see below). MHCflurry  runs on Python 2.7 and 3.4+ using the keras neural network library. It  exposes command-line and Python library interfaces.

*homepage*: https://github.com/openvax/mhcflurry

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------
``1.2.4``|``-Python-3.7.2``|``foss/2019a``, ``fosscuda/2019a``

<a anchor="mhcnuggets"/>
### *mhcnuggets*

MHCnuggets: Neoantigen peptide MHC binding prediction for class I and II.

*homepage*: https://github.com/KarchinLab/mhcnuggets

version|versionsuffix    |toolchain
-------|-----------------|----------------------------------
``2.3``|                 |``foss/2020b``, ``fosscuda/2020b``
``2.3``|``-Python-3.7.2``|``foss/2019a``, ``fosscuda/2019a``

<a anchor="microbeannotator"/>
### *MicrobeAnnotator*

Easy-to-use pipeline for the comprehensive metabolic annotation of microbial genomes.

*homepage*: https://github.com/cruizperez/MicrobeAnnotator

version  |toolchain
---------|--------------
``2.0.5``|``foss/2021a``

<a anchor="microctools"/>
### *microctools*

Various worker functions for microclimc package

*homepage*: https://github.com/ilyamaclean/microctools

version           |versionsuffix|toolchain
------------------|-------------|--------------
``0.1.0-20201209``|``-R-4.0.4`` |``foss/2020b``

<a anchor="migec"/>
### *MiGEC*

MIGEC is a software pipeline that facilitates processing and analysis of   immune repertoire sequencing data generated using molecular barcoding technique

*homepage*: https://milaboratory.com/software/migec/

version  |versionsuffix      |toolchain
---------|-------------------|----------
``1.2.8``|``-Java-1.8.0_162``|``system``
``1.2.9``|``-Java-1.8``      |``system``

<a anchor="migrate-n"/>
### *MIGRATE-N*

Migrate estimates population parameters, effective population sizes and migration rates of n populations, using genetic data.  It  uses a coalescent theory approach taking into account history of  mutations and uncertainty of the genealogy.  

*homepage*: https://peterbeerli.com/migrate-html5/index.html

version   |toolchain
----------|--------------
``4.2.8`` |``foss/2016a``
``4.2.14``|``foss/2018a``
``5.0.4`` |``foss/2021b``

<a anchor="miller"/>
### *Miller*

Miller is like awk, sed, cut, join, and sort for name-indexed data such as CSV, TSV, and tabular JSON

*homepage*: https://miller.readthedocs.io/

version  |toolchain
---------|----------
``6.4.0``|``system``

<a anchor="mimalloc"/>
### *mimalloc*

mimalloc is a general purpose allocator with excellent performance characteristics.

*homepage*: https://microsoft.github.io/mimalloc/

version  |toolchain
---------|--------------------------------------
``1.7.2``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``

<a anchor="minc"/>
### *MINC*

Medical Image NetCDF or MINC isn't netCDF.

*homepage*: https://github.com/BIC-MNI/libminc

version   |toolchain
----------|-----------------------------------------------
``2.4.03``|``foss/2017b``, ``foss/2018a``, ``intel/2017b``

<a anchor="minced"/>
### *MinCED*

Mining CRISPRs in Environmental Datasets

*homepage*: https://github.com/ctSkennerton/minced

version  |versionsuffix|toolchain
---------|-------------|-----------------
``0.4.2``|``-Java-11`` |``GCCcore/8.3.0``

<a anchor="mini-xml"/>
### *Mini-XML*

Mini-XML is a small XML parsing library that you can use to read XML data files or strings in your  application without requiring large non-standard libraries

*homepage*: https://www.msweet.org/mxml

version  |toolchain
---------|------------------
``2.9``  |``GCCcore/8.2.0``
``2.12`` |``GCCcore/9.3.0``
``3.2``  |``GCCcore/10.3.0``
``3.3.1``|``GCCcore/11.2.0``

<a anchor="miniasm"/>
### *miniasm*

Miniasm is a very fast OLC-based de novo assembler for noisy long reads. It takes all-vs-all read self-mappings (typically by minimap) as input and outputs an assembly graph in the GFA format. Different from mainstream assemblers, miniasm does not have a consensus step. It simply concatenates pieces of read sequences to generate the final unitig sequences. Thus the per-base error rate is similar to the raw input reads.

*homepage*: https://github.com/lh3/minimap2

version         |toolchain
----------------|--------------------------------------
``0.3-20191007``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``

<a anchor="minibar"/>
### *minibar*

Dual barcode and primer demultiplexing for MinION sequenced reads

*homepage*: https://github.com/calacademy-research/minibar

version     |versionsuffix    |toolchain
------------|-----------------|-----------------------
``20200326``|``-Python-3.7.4``|``iccifort/2019.5.281``
``20200326``|``-Python-3.8.2``|``iccifort/2020.1.217``

<a anchor="minicard"/>
### *MiniCARD*

MiniCARD is a *cardinality solver* based on MiniSAT [www.minisat.se].  MiniCARD handles cardinality constraints natively, using the same efficient data structures and techniques MiniSAT uses for clauses, giving it much better performance on cardinality constraints than CNF encodings of those constraints passed to a typical SAT solver.  It can read the standard DIMACS CNF format, the OPB pseudo-boolean format (with linear cardinality constraints only), and CNF+, a format that extends CNF to include cardinality constraints.

*homepage*: https://github.com/liffiton/minicard

version|toolchain
-------|-------------
``1.2``|``GCC/9.3.0``

<a anchor="miniconda2"/>
### *Miniconda2*

Miniconda is a free minimal installer for conda. It is a small,  bootstrap version of Anaconda that includes only conda, Python, the packages they  depend on, and a small number of other useful packages.

*homepage*: https://docs.conda.io/en/latest/miniconda.html

version   |toolchain
----------|----------
``4.3.21``|``system``
``4.6.14``|``system``
``4.7.10``|``system``

<a anchor="miniconda3"/>
### *Miniconda3*

Miniconda is a free minimal installer for conda. It is a small,  bootstrap version of Anaconda that includes only conda, Python, the packages they  depend on, and a small number of other useful packages.

*homepage*: https://docs.conda.io/en/latest/miniconda.html

version   |toolchain
----------|----------
``4.4.10``|``system``
``4.5.12``|``system``
``4.6.14``|``system``
``4.7.10``|``system``
``4.8.3`` |``system``
``4.9.2`` |``system``
``4.12.0``|``system``

<a anchor="minieigen"/>
### *minieigen*

A small wrapper for core parts of EIgen, c++ library for linear algebra.

*homepage*: https://launchpad.net/minieigen/

version  |versionsuffix                  |toolchain
---------|-------------------------------|-------------------------------
``0.5.3``|``-Python-2.7.11``             |``foss/2016a``, ``intel/2016a``
``0.5.3``|``-Python-2.7.12``             |``foss/2016b``, ``intel/2016b``
``0.5.3``|``-Python-2.7.12-Boost-1.63.0``|``intel/2016b``
``0.5.4``|``-Python-2.7.14``             |``intel/2018a``

<a anchor="minimac4"/>
### *Minimac4*

Minimac4 is a latest version in the series of genotype imputation software  - preceded by Minimac3 (2015), Minimac2 (2014), minimac (2012) and MaCH (2010).  Minimac4 is a lower memory and more computationally efficient implementation  of the original algorithms with comparable imputation quality.

*homepage*: https://genome.sph.umich.edu/wiki/Minimac4

version  |toolchain
---------|--------------
``1.0.0``|``foss/2018a``

<a anchor="minimap2"/>
### *minimap2*

Minimap2 is a fast sequence mapping and alignment program that can find overlaps between long noisy reads, or map long reads or their assemblies to a reference genome optionally with detailed alignment (i.e. CIGAR). At present, it works efficiently with query sequences from a few kilobases to ~100 megabases in length at an error rate ~15%. Minimap2 outputs in the PAF or the SAM format. On limited test data sets, minimap2 is over 20 times faster than most other long-read aligners. It will replace BWA-MEM for long reads and contig alignment.

*homepage*: https://github.com/lh3/minimap2

version   |toolchain
----------|------------------------------------------------------
``2.0rc1``|``foss/2016b``
``2.10``  |``foss/2018a``
``2.11``  |``intel/2018a``
``2.12``  |``foss/2018a``
``2.13``  |``foss/2018b``
``2.17``  |``GCC/8.2.0-2.31.1``, ``GCC/8.3.0``, ``GCCcore/9.3.0``
``2.18``  |``GCCcore/10.2.0``
``2.20``  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``2.22``  |``GCCcore/11.2.0``
``2.24``  |``GCCcore/11.2.0``

<a anchor="minipolish"/>
### *Minipolish*

A tool for Racon polishing of miniasm assemblies

*homepage*: https://github.com/rrwick/Minipolish

version  |toolchain
---------|--------------
``0.1.3``|``GCC/11.2.0``

<a anchor="minisat"/>
### *MiniSat*

MiniSat is a minimalistic, open-source SAT solver, developed to help researchers and developers alike to get started on SAT.

*homepage*: http://minisat.se/

version     |toolchain
------------|-------------
``2.2.0``   |``GCC/9.3.0``
``20130925``|``GCC/9.3.0``

<a anchor="minpack"/>
### *MINPACK*

Minpack includes software for solving nonlinear equations and  nonlinear least squares problems.  Five algorithmic paths each include  a core subroutine and an easy-to-use driver.  The algorithms proceed  either from an analytic specification of the Jacobian matrix or  directly from the problem functions.  The paths include facilities for  systems of equations with a banded Jacobian matrix, for least squares  problems with a large amount of data, and for checking the consistency  of the Jacobian matrix with the functions. 

*homepage*: https://www.netlib.org/minpack/

version     |toolchain
------------|--------------
``19961126``|``GCC/10.3.0``

<a anchor="minpath"/>
### *MinPath*

MinPath (Minimal set of Pathways) is a parsimony approach for biological pathway reconstructions  using protein family predictions, achieving a more conservative, yet more faithful, estimation of the biological  pathways for a query dataset.

*homepage*: https://omics.informatics.indiana.edu/MinPath

version|versionsuffix     |toolchain
-------|------------------|-------------------------------
``1.4``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``1.4``|``-Python-2.7.16``|``intel/2019b``
``1.6``|                  |``GCCcore/11.2.0``

<a anchor="mira"/>
### *MIRA*

MIRA is a whole genome shotgun and EST sequence assembler for Sanger, 454, Solexa (Illumina),  IonTorrent data and PacBio (the latter at the moment only CCS and error-corrected CLR reads).

*homepage*: https://sourceforge.net/p/mira-assembler/wiki/Home/

version   |versionsuffix     |toolchain
----------|------------------|------------------------------------------------
``4.0.2`` |                  |``foss/2018b``, ``gompi/2019b``, ``intel/2017b``
``4.0.2`` |``-Python-2.7.11``|``foss/2016a``
``4.9.6`` |                  |``intel/2017b``
``5.0rc2``|                  |``foss/2020b``

<a anchor="mirdeep2"/>
### *miRDeep2*

miRDeep2 is a completely overhauled tool which discovers microRNA genes by analyzing sequenced RNAs

*homepage*: https://www.mdc-berlin.de/8551903/en/

version    |versionsuffix    |toolchain
-----------|-----------------|---------------
``0.1.1``  |``-Python-3.6.6``|``foss/2018b``
``2.0.0.8``|                 |``intel/2016b``

<a anchor="mish-cuda"/>
### *Mish-Cuda*

Mish-Cuda: Self Regularized Non-Monotonic Activation Function

*homepage*: https://github.com/JunnYu/mish-cuda

version     |versionsuffix     |toolchain
------------|------------------|------------------
``20210309``|``-PyTorch-1.9.0``|``fosscuda/2020b``

<a anchor="misha"/>
### *misha*

The misha package is intended to help users to efficiently analyze genomic data achieved  from various experiments.

*homepage*: https://github.com/tanaylab/misha

version   |versionsuffix|toolchain
----------|-------------|--------------
``4.0.10``|``-R-4.0.0`` |``foss/2020a``

<a anchor="mitgcmutils"/>
### *MITgcmutils*

A numerical model designed for study of the atmosphere, ocean,  and climate, MITgcm’s flexible non-hydrostatic formulation enables it to  efficiently simulate fluid phenomena over a wide range of scales; its adjoint  capabilities enable it to be applied to sensitivity questions and to parameter  and state estimation problems. By employing fluid equation isomorphisms, a  single dynamical kernel can be used to simulate flow of both the atmosphere  and ocean. The model is developed to perform efficiently on a wide variety of  computational platforms.

*homepage*: https://mitgcm.org/

version  |toolchain
---------|--------------
``0.1.2``|``foss/2022a``

<a anchor="mitobim"/>
### *MITObim*

The MITObim procedure (mitochondrial baiting and iterative mapping) represents  a highly efficient approach to assembling novel mitochondrial genomes of non-model organisms  directly from total genomic DNA derived NGS reads.

*homepage*: https://github.com/chrishah/MITObim

version  |toolchain
---------|-------------------------------
``1.9.1``|``foss/2018b``, ``gompi/2019b``

<a anchor="mitoz"/>
### *MitoZ*

MitoZ is a Python3-based toolkit which aims to automatically filter pair-end raw data (fastq files),  assemble genome, search for mitogenome sequences from the genome assembly result, annotate mitogenome  (genbank file as result), and mitogenome visualization.

*homepage*: https://github.com/linzhi2013/MitoZ

version|versionsuffix    |toolchain
-------|-----------------|--------------
``2.3``|``-Python-3.6.6``|``foss/2018b``

<a anchor="mixcr"/>
### *MiXCR*

MiXCR processes big immunome data from raw sequences  to quantitated clonotypes 

*homepage*: https://milaboratory.com/software/mixcr/

version   |versionsuffix      |toolchain
----------|-------------------|----------
``2.1.9`` |``-Java-1.8.0_162``|``system``
``3.0.3`` |``-Java-1.8``      |``system``
``3.0.13``|``-Java-1.8``      |``system``
``3.0.13``|``-Java-11``       |``system``

<a anchor="mixmhc2pred"/>
### *MixMHC2pred*

MixMHC2pred is a predictor of HLA class II ligands and epitopes.  It is described in publication Racle, J., et al. Robust prediction of HLA class II  epitopes by deep motif deconvolution of immunopeptidomes

*homepage*: https://github.com/GfellerLab/MixMHC2pred

version|toolchain
-------|----------
``1.2``|``system``

<a anchor="mkl-dnn"/>
### *mkl-dnn*

Intel(R) Math Kernel Library for Deep Neural Networks (Intel(R) MKL-DNN)

*homepage*: https://01.org/mkl-dnn

version   |toolchain
----------|-------------------------------
``0.11``  |``intel/2017b``
``0.13``  |``intel/2018a``
``0.16``  |``foss/2018b``, ``intel/2018b``
``0.17.2``|``foss/2018a``

<a anchor="mkl-service"/>
### *mkl-service*

Python hooks for Intel(R) Math Kernel Library runtime control settings.

*homepage*: https://github.com/IntelPython/mkl-service

version  |versionsuffix    |toolchain
---------|-----------------|--------------------------------
``2.0.2``|                 |``intel/2019a``
``2.3.0``|                 |``intel/2020b``, ``intel/2021b``
``2.3.0``|``-Python-3.7.4``|``intel/2019b``

<a anchor="mkl_fft"/>
### *mkl_fft*

NumPy-based Python interface to Intel(R) MKL FFT functionality

*homepage*: https://github.com/IntelPython/mkl_fft

version   |versionsuffix    |toolchain
----------|-----------------|---------------
``1.0.14``|``-Python-3.6.6``|``intel/2018b``

<a anchor="mlc"/>
### *MLC*

Intel Memory Latency Checker (Intel MLC) is a tool used to measure memory latencies and b/w,  and how they change with increasing load on the system.

*homepage*: https://software.intel.com/en-us/articles/intelr-memory-latency-checker

version|toolchain
-------|----------
``3.0``|``system``

<a anchor="mlxtend"/>
### *MLxtend*

Mlxtend (machine learning extensions) is a Python library of useful tools for the day-to-day data  science tasks.

*homepage*: https://rasbt.github.io/mlxtend/

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.17.3``|``-Python-3.8.2``|``foss/2020a``

<a anchor="mm-common"/>
### *mm-common*

The mm-common module provides the build infrastructure and utilities shared among the GNOME C++ binding  libraries.

*homepage*: https://gitlab.gnome.org/GNOME/mm-common

version  |toolchain
---------|------------------
``1.0.4``|``GCCcore/10.3.0``

<a anchor="mmg"/>
### *Mmg*

Mmg is an open source software for simplicial remeshing. It provides 3 applications and 4 libraries: the mmg2d application and the libmmg2d library: adaptation and optimization of a two-dimensional triangulation and generation of a triangulation from a set of points or from given boundary edges the mmgs application and the libmmgs library: adaptation and optimization of a surface triangulation and isovalue discretization the mmg3d application and the libmmg3d library: adaptation and optimization of a tetrahedral mesh and implicit domain meshing the libmmg library gathering the libmmg2d, libmmgs and libmmg3d libraries.

*homepage*: https://www.mmgtools.org/

version  |toolchain
---------|--------------------------------
``5.3.9``|``foss/2017b``
``5.6.0``|``gompi/2021a``, ``gompi/2021b``

<a anchor="mmseq"/>
### *MMSEQ*

The MMSEQ package contains a collection of statistical tools  for analysing RNA-seq expression data. 

*homepage*: https://github.com/eturro/mmseq

version  |versionsuffix      |toolchain
---------|-------------------|----------
``1.0.8``|``-linux64-static``|``system``

<a anchor="mmseqs2"/>
### *MMseqs2*

MMseqs2: ultra fast and sensitive search and clustering suite

*homepage*: https://mmseqs.com

version              |toolchain
---------------------|-------------------------------------------------
``1-c7a89``          |``foss/2016b``
``5-9375b``          |``intel/2018a``
``8-fac81``          |``intel/2018b``
``10-6d92c``         |``gompi/2019b``, ``iimpi/2019b``
``11-e1a1c``         |``iimpi/2019b``
``13-45111``         |``gompi/2020b``, ``gompi/2021a``, ``gompi/2021b``
``13-45111-20211006``|``gompi/2020b``
``13-45111-20211019``|``gompi/2020b``

<a anchor="mmtf-cpp"/>
### *mmtf-cpp*

The Macromolecular Transmission Format (MMTF) is a new compact binary format to  transmit and store biomolecular structures for fast 3D visualization and        analysis. 

*homepage*: https://mmtf.rcsb.org/

version  |toolchain
---------|------------------
``1.0.0``|``GCCcore/10.2.0``

<a anchor="mne-python"/>
### *MNE-Python*

MNE-Python software is an open-source Python package for exploring, visualizing, and analyzing human neurophysiological data such as MEG, EEG, sEEG, ECoG, and more. It includes modules for data input/output, preprocessing, visualization, source estimation, time-frequency analysis, connectivity analysis, machine learning, and statistics.

*homepage*: https://mne.tools/stable/index.html

version   |toolchain
----------|--------------
``0.24.1``|``foss/2021a``

<a anchor="moabs"/>
### *MOABS*

MOABS: MOdel based Analysis of Bisulfite Sequencing data

*homepage*: https://github.com/sunnyisgalaxy/moabs

version    |toolchain
-----------|---------------
``1.3.9.6``|``gompi/2019b``

<a anchor="modeltest-ng"/>
### *ModelTest-NG*

ModelTest-NG is a tool for selecting the best-fit model of evolution for DNA and protein alignments. ModelTest-NG supersedes jModelTest and ProtTest in one single tool, with graphical and command console interfaces. 

*homepage*: https://github.com/ddarriba/modeltest

version  |toolchain
---------|---------------
``0.1.7``|``gompi/2021b``

<a anchor="modred"/>
### *modred*

Compute modal decompositions and reduced-order models, easily, efficiently, and in parallel.

*homepage*: https://pypi.python.org/pypi/modred/

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``2.0.2``|``-Python-3.5.2``|``foss/2016b``

<a anchor="molcas"/>
### *Molcas*

Molcas is an ab initio quantum chemistry software package developed by scientists to be used by scientists. The basic philosophy is is to be able to treat general electronic structures for molecules consisting of atoms from most of the periodic table. As such, the primary focus of the package is on multiconfigurational methods with applications typically connected to the treatment of highly degenerate states.

*homepage*: http://www.molcas.org

version         |versionsuffix         |toolchain
----------------|----------------------|----------
``8.0-15.06.18``|``_CentOS_6.6_x86_64``|``system``
``8.2``         |``-centos-mkl-par``   |``system``
``8.2``         |``-centos-par``       |``system``

<a anchor="mold"/>
### *mold*

mold is a high-performance drop-in replacement for existing Unix linkers.

*homepage*: https://github.com/rui314/mold

version  |toolchain
---------|------------------
``0.9.6``|``GCCcore/11.2.0``
``1.0.0``|``GCCcore/11.2.0``
``1.2.1``|``GCCcore/11.3.0``
``1.3.0``|``GCCcore/11.3.0``
``1.7.1``|``GCCcore/12.2.0``

<a anchor="molden"/>
### *Molden*

Molden is a package for displaying Molecular Density from the  Ab Initio packages GAMESS-UK, GAMESS-US and GAUSSIAN and the Semi-Empirical  packages Mopac/Ampac

*homepage*: https://www.theochem.ru.nl/molden/

version|toolchain
-------|-------------------------------------
``5.6``|``foss/2016a``
``5.7``|``intel/2016b``
``5.8``|``foss/2018a``
``6.1``|``GCCcore/8.2.0``
``6.8``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``7.1``|``GCCcore/11.3.0``

<a anchor="molekel"/>
### *Molekel*

Molekel is an open-source multi-platform molecular visualization program. 

*homepage*: http://ugovaretto.github.io/molekel/wiki/pmwiki.php/Main/HomePage.html

version  |versionsuffix    |toolchain
---------|-----------------|----------
``5.4.0``|``-Linux_x86_64``|``system``

<a anchor="molmod"/>
### *molmod*

MolMod is a Python library with many compoments that are useful to write molecular modeling programs.

*homepage*: https://molmod.github.io/molmod/

version  |versionsuffix     |toolchain
---------|------------------|----------------------------------------------
``1.1``  |``-Python-2.7.11``|``intel/2016a``
``1.1``  |``-Python-2.7.12``|``intel/2016b``
``1.1``  |``-Python-2.7.13``|``intel/2017a``
``1.4.3``|``-Python-2.7.14``|``intel/2017b``
``1.4.3``|``-Python-3.6.3`` |``intel/2017b``
``1.4.4``|``-Python-2.7.14``|``foss/2018a``, ``intel/2018a``
``1.4.4``|``-Python-2.7.15``|``intel/2018b``
``1.4.4``|``-Python-3.7.2`` |``intel/2019a``
``1.4.5``|``-Python-3.7.4`` |``foss/2019b``, ``intel/2019b``
``1.4.5``|``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``1.4.8``|                  |``foss/2020b``, ``foss/2021a``, ``foss/2021b``

<a anchor="molpro"/>
### *Molpro*

Molpro is a complete system of ab initio programs for molecular electronic structure calculations.

*homepage*: https://www.molpro.net

version      |versionsuffix           |toolchain
-------------|------------------------|----------
``2010.1.23``|``.Linux_x86_64``       |``system``
``2015.1.0`` |``.linux_x86_64_intel`` |``system``
``2015.1.3`` |``.linux_x86_64_openmp``|``system``
``2015.1.10``|``.linux_x86_64_openmp``|``system``

<a anchor="mona"/>
### *MONA*

MONA is a tool that translates formulas to finite-state automata. The formulas may express search patterns, temporal properties of reactive systems, parse tree constraints, etc. MONA analyses the automaton resulting from the compilation and prints out "valid" or a counter-example.

*homepage*: https://www.brics.dk/mona/index.html

version   |toolchain
----------|--------------
``1.4-18``|``GCC/11.2.0``

<a anchor="monai"/>
### *MONAI*

MONAI is a PyTorch-based, open-source framework for deep learning in healthcare imaging, part of PyTorch Ecosystem. 

*homepage*: https://monai.io/

version  |versionsuffix   |toolchain
---------|----------------|--------------
``0.8.0``|                |``foss/2021a``
``0.8.0``|``-CUDA-11.3.1``|``foss/2021a``
``1.0.1``|                |``foss/2022a``
``1.0.1``|``-CUDA-11.7.0``|``foss/2022a``

<a anchor="mongolite"/>
### *mongolite*

High-performance MongoDB client based on 'mongo-c-driver' and 'jsonlite'.  Includes support for aggregation, indexing, map-reduce, streaming, encryption,  enterprise authentication, and GridFS. The online user manual provides an overview  of the available methods in the package: <https://jeroen.github.io/mongolite/>.

*homepage*: https://cran.r-project.org/web/packages/mongolite

version  |versionsuffix|toolchain
---------|-------------|--------------
``2.3.0``|``-R-4.0.0`` |``foss/2020a``
``2.3.0``|``-R-4.0.3`` |``foss/2020b``
``2.3.0``|``-R-4.0.4`` |``foss/2020b``

<a anchor="mono"/>
### *Mono*

An open source, cross-platform, implementation of C# and the CLR that is  binary compatible with Microsoft.NET.

*homepage*: https://www.mono-project.com/

version       |toolchain
--------------|-------------------------------
``2.10.6``    |``intel/2016b``
``4.6.2.7``   |``intel/2016b``, ``system``
``4.8.0.495`` |``intel/2017a``
``5.4.1.6``   |``foss/2017b``, ``intel/2017b``
``5.10.0.160``|``foss/2018a``
``5.18.1.0``  |``foss/2018a``
``6.4.0.198`` |``foss/2018b``
``6.8.0.105`` |``GCCcore/8.3.0``
``6.12.0.122``|``GCCcore/11.2.0``

<a anchor="monocle3"/>
### *Monocle3*

An analysis toolkit for single-cell RNA-seq. 

*homepage*: https://cole-trapnell-lab.github.io/monocle3/

version  |versionsuffix            |toolchain
---------|-------------------------|--------------
``0.2.0``|``-Python-3.7.2-R-3.6.0``|``foss/2019a``
``0.2.3``|``-R-4.0.3``             |``foss/2020b``

<a anchor="moonjit"/>
### *moonjit*

Moonjit is a Just-In-Time Compiler (JIT) for the Lua programming  language. Lua is a powerful, dynamic and light-weight programming language. It may be embedded or used as a general-purpose, stand-alone language.

*homepage*: https://github.com/moonjit/moonjit

version  |toolchain
---------|-----------------
``2.2.0``|``GCCcore/9.3.0``

<a anchor="moose"/>
### *MOOSE*

The Multiphysics Object-Oriented Simulation Environment (MOOSE) is a finite-element, multiphysics framework primarily developed by Idaho National Laboratory

*homepage*: https://mooseframework.inl.gov

version       |versionsuffix    |toolchain
--------------|-----------------|--------------
``2021-05-18``|``-Python-3.7.4``|``foss/2019b``

<a anchor="mordecai"/>
### *mordecai*

mordecai is a full text geoparsing as a Python library. Extract the place names from a piece of text, resolve them to the correct place, and return their coordinates and structured geographic information.

*homepage*: https://github.com/openeventdata/mordecai

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``2.0.1``|``-Python-3.6.4``|``foss/2018a``

<a anchor="moreronn"/>
### *MoreRONN*

MoreRONN is the spiritual successor of RONN and is useful for surveying  disorder in proteins as well as designing expressible constructs for X-ray crystallography.

*homepage*: https://github.com/varun-ramraj/MoreRONN

version|toolchain
-------|-------------
``4.9``|``GCC/8.3.0``

<a anchor="mosdepth"/>
### *mosdepth*

Fast BAM/CRAM depth calculation for WGS, exome, or targeted sequencing

*homepage*: https://github.com/brentp/mosdepth

version  |toolchain
---------|---------------
``0.2.2``|``intel/2018a``
``0.2.3``|``intel/2018a``
``0.2.4``|``foss/2018b``
``0.3.3``|``GCC/11.2.0``

<a anchor="mothur"/>
### *Mothur*

Mothur is a single piece of open-source, expandable software  to fill the bioinformatics needs of the microbial ecology community.

*homepage*: https://www.mothur.org/

version   |versionsuffix     |toolchain
----------|------------------|---------------
``1.39.5``|``-Python-2.7.13``|``intel/2017a``
``1.41.0``|``-Python-2.7.15``|``foss/2018b``
``1.43.0``|``-Python-3.7.2`` |``foss/2019a``

<a anchor="motif"/>
### *motif*

Motif refers to both a graphical user interface (GUI) specification and the widget toolkit for building  applications that follow that specification under the X Window System on Unix and other POSIX-compliant systems.  It was the standard toolkit for the Common Desktop Environment and thus for Unix.

*homepage*: https://motif.ics.com/

version  |toolchain
---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.3.5``|``foss/2016a``, ``intel/2016a``
``2.3.7``|``intel/2017a``
``2.3.8``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``foss/2019a``, ``intel/2017b``, ``intel/2018a``, ``intel/2019a``

<a anchor="motioncor2"/>
### *MotionCor2*

MotionCor2 correct anisotropic image motion at the single pixel level across the whole frame, suitable for both single particle and tomographic images. Iterative, patch-based motion detection is combined with spatial and temporal constraints and dose weighting.  Cite publication: Shawn Q. Zheng, Eugene Palovcak, Jean-Paul Armache, Yifan Cheng and David A. Agard (2016) Anisotropic Correction of Beam-induced Motion for Improved Single-particle Electron Cryo-microscopy, Nature Methods, submitted. BioArxiv: https://biorxiv.org/content/early/2016/07/04/061960 

*homepage*: https://msg.ucsf.edu/

version  |toolchain
---------|--------------------------------------
``1.2.6``|``GCCcore/8.2.0``
``1.3.1``|``GCCcore/8.3.0``
``1.3.2``|``GCCcore/8.3.0``
``1.4.2``|``GCCcore/10.2.0``
``1.4.4``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``1.5.0``|``GCCcore/11.3.0``

<a anchor="motionsegmentation"/>
### *motionSegmentation*

Motion correction, explicit spatio-temporal regularization of motion tracking, random speckles enhancement, and segmentation.

*homepage*: https://github.com/WeiXuanChan/motionSegmentation

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``2.7.9``|``-Python-3.7.4``|``foss/2019b``

<a anchor="moviepy"/>
### *MoviePy*

MoviePy (full documentation) is a Python library for video editing: cutting, concatenations,  title insertions, video compositing (a.k.a. non-linear editing), video processing, and creation of custom effects.

*homepage*: https://zulko.github.io/moviepy/

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.0.1``|``-Python-3.7.2``|``foss/2019a``
``1.0.3``|                 |``foss/2021a``

<a anchor="mpb"/>
### *MPB*

MPB is a free and open-source software package for computing  the band structures, or dispersion relations, and electromagnetic  modes of periodic dielectric structures, on both serial  and parallel computers. MPB is an acronym for MIT Photonic Bands.

*homepage*: https://mpb.readthedocs.io/en/latest/

version  |versionsuffix     |toolchain
---------|------------------|-----------------------------------------------
``1.6.2``|``-Python-2.7.14``|``foss/2017b``, ``foss/2018a``, ``intel/2018a``

<a anchor="mpc"/>
### *MPC*

Gnu Mpc is a C library for the arithmetic of  complex numbers with arbitrarily high precision and correct  rounding of the result. It extends the principles of the IEEE-754  standard for fixed precision real floating point numbers to  complex numbers, providing well-defined semantics for every  operation. At the same time, speed of operation at high precision  is a major design goal.

*homepage*: http://www.multiprecision.org/

version  |versionsuffix  |toolchain
---------|---------------|------------------------------------------------------------------------------
``1.0.3``|               |``intel/2017a``
``1.0.3``|``-MPFR-3.1.6``|``foss/2017b``, ``intel/2017b``
``1.1.0``|               |``GCC/8.3.0``, ``GCC/9.3.0``, ``GCCcore/9.3.0``
``1.2.1``|               |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="mpfr"/>
### *MPFR*

The MPFR library is a C library for multiple-precision floating-point  computations with correct rounding. 

*homepage*: https://www.mpfr.org

version  |toolchain
---------|------------------------------------------------------------------------------
``2.4.2``|``system``
``3.1.4``|``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``3.1.5``|``GCCcore/6.4.0``, ``intel/2017a``
``3.1.6``|``GCCcore/6.4.0``
``4.0.1``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``4.0.2``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``4.1.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="mpi4py"/>
### *mpi4py*

MPI for Python (mpi4py) provides bindings of the Message Passing Interface (MPI) standard for  the Python programming language, allowing any Python program to exploit multiple processors.

*homepage*: https://bitbucket.org/mpi4py/mpi4py

version  |versionsuffix                    |toolchain
---------|---------------------------------|--------------------------------
``1.3.1``|``-Python-2.7.11-timed-pingpong``|``intel/2016a``
``1.3.1``|``-Python-2.7.12-timed-pingpong``|``intel/2016b``
``2.0.0``|``-Python-2.7.12``               |``intel/2016b``
``2.0.0``|``-Python-2.7.13-timed-pingpong``|``intel/2017a``
``3.0.0``|``-Python-2.7.14-timed-pingpong``|``intel/2018a``
``3.0.0``|``-Python-3.6.3``                |``intel/2017b``
``3.0.1``|``-Python-3.6.6``                |``intel/2018b``
``3.0.2``|``-timed-pingpong``              |``gompi/2019a``, ``iimpi/2019a``

<a anchor="mpich"/>
### *MPICH*

MPICH is a high-performance and widely portable implementation  of the Message Passing Interface (MPI) standard (MPI-1, MPI-2 and MPI-3).

*homepage*: https://www.mpich.org/

version  |toolchain
---------|--------------------------------------
``3.0.4``|``GCC/4.8.1``
``3.2``  |``GCC/4.9.3-2.25``, ``GCC/7.2.0-2.29``
``3.2.1``|``GCC/7.2.0-2.29``
``3.3.2``|``GCC/10.2.0``, ``GCC/9.3.0``
``3.4.2``|``GCC/10.3.0``

<a anchor="mpich2"/>
### *MPICH2*

MPICH v3.x is an open source high-performance MPI 3.0 implementation. It does not support InfiniBand (use MVAPICH2 with InfiniBand devices).

*homepage*: http://www.mpich.org/

version|toolchain
-------|-------------
``1.1``|``GCC/4.8.1``

<a anchor="mpifileutils"/>
### *mpifileutils*

MPI-Based File Utilities For Distributed Systems 

*homepage*: https://hpc.github.io/mpifileutils/

version   |toolchain
----------|--------------------------------
``0.9.1`` |``gompi/2019a``, ``iimpi/2019a``
``0.10``  |``gompi/2020a``, ``iimpi/2020a``
``0.10.1``|``gompi/2020a``

<a anchor="mpip"/>
### *mpiP*

mpiP is a lightweight profiling library for MPI applications.  Because it  only collects statistical information about MPI functions, mpiP generates  considerably less overhead and much less data than tracing tools.  All the  information captured by mpiP is task-local. It only uses communication during  report generation, typically at the end of the experiment, to merge results  from all of the tasks into one output file. 

*homepage*: http://mpip.sourceforge.net/

version  |toolchain
---------|---------------------------------------------------
``3.4.1``|``gompi/2019a``, ``iimpi/2019a``, ``iompi/2019.01``

<a anchor="mpj-express"/>
### *MPJ-Express*

MPJ Express is an open source Java message passing library that allows application   developers to write and execute parallel applications for multicore processors and compute clusters/clouds.

*homepage*: http://mpj-express.org/

version |versionsuffix     |toolchain
--------|------------------|--------------
``0.44``|``-Java-1.8.0_92``|``foss/2016a``

<a anchor="mpmath"/>
### *mpmath*

mpmath can be used as an arbitrary-precision substitute for Python's float/complex  types and math/cmath modules, but also does much more advanced mathematics. Almost any calculation  can be performed just as well at 10-digit or 1000-digit precision, with either real or complex  numbers, and in many cases mpmath implements efficient algorithms that scale well for extremely  high precision work.

*homepage*: https://mpmath.org/

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``0.19`` |``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``1.0.0``|``-Python-2.7.14``|``intel/2018a``
``1.0.0``|``-Python-2.7.15``|``foss/2018b``
``1.1.0``|``-Python-3.8.2`` |``GCCcore/9.3.0``

<a anchor="mrbayes"/>
### *MrBayes*

MrBayes is a program for Bayesian inference and model choice across  a wide range of phylogenetic and evolutionary models.

*homepage*: https://nbisweden.github.io/MrBayes/

version   |toolchain
----------|-----------------------------------------------
``3.2.6`` |``foss/2016a``, ``foss/2017a``, ``gompi/2020b``
``3.2.7`` |``gompi/2020b``, ``gompic/2019b``
``3.2.7a``|``foss/2020a``, ``iimpi/2019a``

<a anchor="mrcfile"/>
### *mrcfile*

mrcfile is a Python implementation of the MRC2014 file format, which is used in  structural biology to store image and volume data. It allows MRC files to be created and opened easily using a very simple API,    which exposes the file’s header and data as numpy arrays. The code runs in      Python 2 and 3 and is fully unit-tested. This library aims to allow users and developers to read and write standard- compliant MRC files in Python as easily as possible, and with no dependencies on any compiled libraries except numpy. You can use it interactively to inspect  files, correct headers and so on, or in scripts and larger software packages to  provide basic MRC file I/O functions. 

*homepage*: https://github.com/ccpem/mrcfile

version  |toolchain
---------|--------------------------------------------------
``1.3.0``|``foss/2020b``, ``foss/2021a``, ``fosscuda/2020b``

<a anchor="mrchem"/>
### *MRChem*

MRChem is a numerical real-space code for molecular electronic structure calculations within the self-consistent field (SCF) approximations of quantum chemistry: Hartree-Fock and Density Functional Theory.

*homepage*: https://mrchem.readthedocs.io

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.0.0``|``-Python-3.8.2``|``foss/2020a``
``1.1.1``|                 |``foss/2022a``

<a anchor="mrcpp"/>
### *MRCPP*

MultiResolution Computation Program Package

*homepage*: https://mrcpp.readthedocs.io

version  |toolchain
---------|--------------
``1.3.6``|``foss/2020a``
``1.4.1``|``foss/2022a``

<a anchor="mricron"/>
### *MRIcron*

MRIcron allows viewing of medical images. It  includes tools to complement SPM and FSL. Native format is NIFTI  but includes a conversion program (see dcm2nii) for converting DICOM images.  Features layers, ROIs, and volume rendering.  

*homepage*: http://www.mccauslandcenter.sc.edu/mricro/mricron/

version         |toolchain
----------------|----------
``1.0.20180614``|``system``
``20150601``    |``system``

<a anchor="mrtrix"/>
### *MRtrix*

MRtrix provides a set of tools to perform diffusion-weighted MR white-matter tractography in a manner  robust to crossing fibres, using constrained spherical deconvolution (CSD) and probabilistic streamlines.

*homepage*: http://www.brain.org.au/software/index.html#mrtrix

version            |versionsuffix     |toolchain
-------------------|------------------|---------------
``0.3.14``         |``-Python-2.7.11``|``intel/2016a``
``0.3.15``         |``-Python-2.7.12``|``intel/2016b``
``3.0-rc-20191217``|``-Python-2.7.16``|``foss/2019b``
``3.0-rc-20191217``|``-Python-3.7.4`` |``foss/2019b``
``3.0.0``          |``-Python-3.8.2`` |``foss/2020a``
``3.0.3``          |                  |``foss/2021a``
``3.0_RC2``        |``-Python-2.7.13``|``foss/2017a``
``3.0_RC3``        |``-Python-2.7.14``|``intel/2018a``

<a anchor="msgpack-c"/>
### *msgpack-c*

MessagePack is an efficient binary serialization format, which lets you exchange data among multiple languages like JSON, except that it's faster and smaller.    Small integers are encoded into a single byte while typical short strings        require only one extra byte in addition to the strings themselves.

*homepage*: http://msgpack.org/

version  |toolchain
---------|------------------
``3.3.0``|``GCCcore/10.2.0``

<a anchor="msm"/>
### *MSM*

Multimodal Surface Matching with Higher order Clique Reduction

*homepage*: https://github.com/ecr05/MSM_HOCR

version|toolchain
-------|-------------------------------
``1.0``|``foss/2017b``, ``intel/2017b``

<a anchor="mspc"/>
### *MSPC*

Using combined evidence from replicates to evaluate ChIP-seq peaks

*homepage*: https://genometric.github.io/MSPC/

version  |toolchain
---------|----------
``3.3.1``|``system``

<a anchor="msprime"/>
### *msprime*

msprime is a coalescent simulator and library for processing tree-based genetic data.

*homepage*: https://msprime.readthedocs.io

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.7.0``|``-Python-3.7.2``|``intel/2019a``

<a anchor="mstore"/>
### *mstore*

Molecular structure store for testing

*homepage*: https://github.com/grimme-lab/mstore

version  |toolchain
---------|--------------
``0.2.0``|``GCC/11.2.0``

<a anchor="mtl4"/>
### *MTL4*

The Matrix Template Library 4 incorporates the most modern programming techniques  to provide an easy and intuitive interface to users while enabling optimal performance. The natural  mathematical notation in MTL4 empowers all engineers and scientists to implement their algorithms and  models in minimal time. All technical aspects are encapsulated in the library.

*homepage*: http://www.simunova.com/mtl4

version     |toolchain
------------|----------
``4.0.8878``|``system``
``4.0.9555``|``system``

<a anchor="mujoco"/>
### *MuJoCo*

MuJoCo stands for Multi-Joint dynamics with Contact. It is a general purpose physics engine that aims to facilitate research and development in robotics, biomechanics, graphics and animation, machine learning, and other areas which demand fast and accurate simulation of articulated structures interacting with their environment.

*homepage*: https://mujoco.org/

version  |toolchain
---------|------------------
``2.1.1``|``GCCcore/11.2.0``

<a anchor="mujoco-py"/>
### *mujoco-py*

MuJoCo is a physics engine for detailed, efficient rigid body simulations with contacts. mujoco-py allows using MuJoCo from Python 3.

*homepage*: https://github.com/openai/mujoco-py

version     |toolchain
------------|--------------
``2.1.2.14``|``foss/2021b``

<a anchor="multichoose"/>
### *multichoose*

generate multiset combinations (n multichoose k).

*homepage*: https://github.com/ekg/multichoose

version  |toolchain
---------|-----------------------------------------------------------------------------
``1.0.3``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``

<a anchor="multilevelestimators"/>
### *MultilevelEstimators*

The Julia module for Multilevel Monte Carlo methods

*homepage*: https://github.com/PieterjanRobbe/MultilevelEstimators.jl

version  |versionsuffix   |toolchain
---------|----------------|--------------
``0.1.0``|``-Julia-1.7.2``|``GCC/11.2.0``

<a anchor="multinest"/>
### *MultiNest*

MultiNest is a Bayesian inference tool which calculates the evidence and explores the parameter space  which may contain multiple posterior modes and pronounced (curving) degeneracies in moderately high dimensions.

*homepage*: https://ccpforge.cse.rl.ac.uk/gf/project/multinest/

version |toolchain
--------|---------------
``3.10``|``intel/2016a``

<a anchor="multiqc"/>
### *MultiQC*

Aggregate results from bioinformatics analyses across many samples into a single   report.   MultiQC searches a given directory for analysis logs and compiles a HTML report. It's a general  use tool, perfect for summarising the output from numerous bioinformatics tools.

*homepage*: https://multiqc.info

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``0.9``   |``-Python-2.7.12``|``foss/2016b``
``1.2``   |``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``1.2``   |``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``1.6``   |``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``1.6``   |``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``1.7``   |``-Python-2.7.15``|``intel/2018b``
``1.7``   |``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``1.8``   |``-Python-2.7.16``|``intel/2019b``
``1.8``   |``-Python-3.7.4`` |``foss/2019b``, ``intel/2019b``
``1.9``   |``-Python-3.7.4`` |``foss/2019b``
``1.9``   |``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``1.10.1``|                  |``foss/2020b``
``1.11``  |                  |``foss/2021a``
``1.12``  |                  |``foss/2021b``

<a anchor="multiwfn"/>
### *Multiwfn*

Multiwfn is an extremely powerful program for realizingi  electronic wavefunction analysis, which is a key ingredient of quantum  chemistry. Multiwfn is free, open-source, high-efficient, very user-friendly  and flexible, it supports almost all of the most important wavefunction  analysis methods.

*homepage*: http://sobereva.com/multiwfn/

version  |toolchain
---------|--------------------------------
``3.4.1``|``intel/2017b``
``3.6``  |``intel/2019a``, ``intel/2019b``

<a anchor="mumerge"/>
### *muMerge*

muMerge is a tool for combining bed regions from multiple bed files that overlap.

*homepage*: https://github.com/Dowell-Lab/mumerge

version  |toolchain
---------|--------------
``1.1.0``|``foss/2022a``

<a anchor="mummer"/>
### *MUMmer*

MUMmer is a system for rapidly aligning entire genomes,  whether in complete or draft form. AMOS makes use of it. 

*homepage*: http://mummer.sourceforge.net/

version       |toolchain
--------------|-------------------------------------------------------------------------
``3.23``      |``GCCcore/10.3.0``, ``GCCcore/9.3.0``, ``foss/2016b``
``4.0.0beta2``|``GCCcore/10.2.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``, ``foss/2018b``
``4.0.0rc1``  |``GCCcore/11.2.0``

<a anchor="mumps"/>
### *MUMPS*

A parallel sparse direct solver

*homepage*: https://graal.ens-lyon.fr/MUMPS/

version  |versionsuffix |toolchain
---------|--------------|-----------------------------------------------------------------------------------------------------------------
``5.1.2``|``-metis``    |``foss/2017b``
``5.2.1``|``-metis``    |``foss/2018b``, ``foss/2019a``, ``foss/2019b``, ``foss/2020a``, ``intel/2019a``, ``intel/2019b``, ``intel/2020a``
``5.2.1``|``-metis-seq``|``foss/2019a``, ``intel/2019a``
``5.3.5``|``-metis``    |``foss/2020b``, ``intel/2020b``
``5.4.0``|``-metis``    |``foss/2021a``
``5.4.1``|``-metis``    |``foss/2021b``, ``intel/2021b``
``5.5.0``|``-metis``    |``foss/2021a``
``5.5.1``|``-metis``    |``foss/2022a``

<a anchor="muparser"/>
### *muParser*

muParser is an extensible high performance math expression parser library  written in C++. It works by transforming a mathematical expression into  bytecode and precalculating constant parts of the expression. 

*homepage*: https://beltoforion.de/article.php?a=muparser

version  |toolchain
---------|-------------------------------------
``2.2.5``|``GCCcore/6.4.0``
``2.3.2``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``2.3.3``|``GCCcore/10.3.0``

<a anchor="muparserx"/>
### *muparserx*

A C++ Library for Parsing Expressions with Strings, Complex Numbers, Vectors, Matrices and more.

*homepage*: https://beltoforion.de/en/muparserx/

version  |toolchain
---------|--------------------------------------
``4.0.8``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``

<a anchor="mupexi"/>
### *MuPeXI*

MuPeXI: Mutant Peptide eXtractor and Informer. Given a list of somatic mutations (VCF file) as input,  MuPeXI returns a table containing all mutated peptides (neo-peptides) of user-defined lengths,  along with several pieces of information relevant for identifying  which of these neo-peptides are likely to serve as neo-epitopes.

*homepage*: https://github.com/ambj/MuPeXI

version  |versionsuffix                 |toolchain
---------|------------------------------|--------------
``1.2.0``|``-Perl-5.28.0-Python-2.7.15``|``foss/2018b``

<a anchor="muscle"/>
### *MUSCLE*

MUSCLE is one of the best-performing multiple alignment programs  according to published benchmark tests, with accuracy and speed that are consistently  better than CLUSTALW. MUSCLE can align hundreds of sequences in seconds. Most users  learn everything they need to know about MUSCLE in a few minutes-only a handful of  command-line options are needed to perform common alignment tasks.

*homepage*: https://drive5.com/muscle/

version     |versionsuffix  |toolchain
------------|---------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``3.8.31``  |               |``GCC/7.3.0-2.30``, ``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``, ``foss/2016a``, ``foss/2017b``, ``foss/2018a``, ``intel/2016a``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``
``3.8.31``  |``-i86linux64``|``system``
``3.8.1551``|               |``GCC/10.2.0``, ``GCC/8.2.0-2.31.1``, ``GCC/8.3.0``
``5.0.1428``|               |``GCCcore/10.3.0``
``5.1``     |               |``GCCcore/11.2.0``

<a anchor="music"/>
### *MuSiC*

Multi-subject Single Cell deconvolution (MuSiC) is a deconvolution method that utilizes  cross-subject scRNA-seq to estimate cell type proportions in bulk RNA-seq data.

*homepage*: https://github.com/xuranw/MuSiC

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.6.2``|``-R-3.5.1`` |``foss/2018b``

<a anchor="must"/>
### *MUST*

MUST detects usage errors of the Message Passing Interface (MPI) and reports them to the user.  As MPI calls are complex and usage errors common, this functionality is extremely helpful for application developers  that want to develop correct MPI applications.  This includes errors that already manifest – segmentation faults or incorrect results – as well as many errors  that are not visible to the application developer or do not manifest on a certain system or MPI implementation.

*homepage*: https://hpc.rwth-aachen.de/must/

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.6``  |``-Python-3.6.6``|``foss/2018b``
``1.6``  |``-Python-3.7.4``|``foss/2019b``
``1.7.2``|                 |``foss/2021a``

<a anchor="mutect"/>
### *MuTect*

MuTect is a method developed at the Broad Institute for the reliable  and accurate identification of somatic point mutations in next generation sequencing  data of cancer genomes. 

*homepage*: http://www.broadinstitute.org/cancer/cga/mutect

version  |versionsuffix     |toolchain
---------|------------------|----------
``1.1.4``|``-Java-1.7.0_76``|``system``
``1.1.4``|``-Java-1.7.0_80``|``system``
``1.1.7``|``-Java-1.7.0_80``|``system``

<a anchor="mutil"/>
### *mutil*

Mutil is a set of standard utilities that have been parallelized to maximize performance on  modern file systems. These currently include multi-threaded drop-in replacements for cp and md5sum from GNU coreutils, which have achieved 10/30x rates on one/many nodes. 

*homepage*: http://people.nas.nasa.gov/~kolano/projects/mutil.html

version    |toolchain
-----------|---------------
``1.822.3``|``intel/2016a``

<a anchor="mvapich2"/>
### *MVAPICH2*

This is an MPI 3.0 implementation.  It is based on MPICH2 and MVICH.

*homepage*: http://mvapich.cse.ohio-state.edu/overview/mvapich2/

version  |toolchain
---------|------------------
``2.0.1``|``GCC/4.8.4``
``2.1``  |``GCC/4.9.3-2.25``
``2.2b`` |``GCC/4.9.3-2.25``

<a anchor="mxml"/>
### *mxml*

Mini-XML is a tiny XML library that you can use to read and write XML and  XML-like data files in your application without requiring large non-standard  libraries. 

*homepage*: https://github.com/michaelrsweet/mxml

version|toolchain
-------|-----------------
``3.2``|``GCCcore/8.3.0``

<a anchor="mxmlplus"/>
### *mxmlplus*

Mxml is a pure C library (yet having an object oriented layout) that is meant to help developers  implementing XML file interpretation in their projects.

*homepage*: http://mxml.sourceforge.net

version  |toolchain
---------|-------------
``0.9.2``|``GCC/9.3.0``

<a anchor="mxnet"/>
### *MXNet*

Flexible and Efficient Library for Deep Learning

*homepage*: http://mxnet.io/

version  |versionsuffix             |toolchain
---------|--------------------------|--------------
``0.9.3``|``-Python-2.7.12-R-3.3.3``|``foss/2016b``

<a anchor="mycc"/>
### *MyCC*

MyCC is built and delivered as a tailored solution for metagenomics sequencesclassfication.

*homepage*: https://sourceforge.net/projects/sb2nhri/files/MyCC/

version       |versionsuffix     |toolchain
--------------|------------------|---------------
``2017-03-01``|``-Python-2.7.16``|``intel/2019b``

<a anchor="mygene"/>
### *mygene*

Python Client for MyGene.Info services.

*homepage*: https://github.com/biothings/mygene.py

version  |toolchain
---------|---------------
``3.1.0``|``intel/2019a``

<a anchor="mymedialite"/>
### *MyMediaLite*

MyMediaLite is a lightweight, multi-purpose library of recommender system algorithms.

*homepage*: http://www.ismll.uni-hildesheim.de/mymedialite/

version |toolchain
--------|---------------
``3.10``|``intel/2016b``
``3.11``|``intel/2016b``
``3.12``|``intel/2017a``

<a anchor="mympingpong"/>
### *mympingpong*

A mpi4py based random pair pingpong network stress test.

*homepage*: https://github.com/hpcugent/mympingpong

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``0.7.0``|``-Python-2.7.11``|``intel/2016a``
``0.7.1``|``-Python-2.7.12``|``intel/2016b``
``0.8.0``|``-Python-2.7.13``|``intel/2017a``
``0.8.0``|``-Python-2.7.14``|``intel/2018a``
``0.8.0``|``-Python-2.7.15``|``foss/2019a``, ``intel/2019a``

<a anchor="myokit"/>
### *Myokit*

Myokit is an open-source Python-based toolkit that facilitates modeling and simulation of cardiac cellular electrophysiology.

*homepage*: http://myokit.org

version   |toolchain
----------|----------------------------------
``1.32.0``|``foss/2020b``, ``fosscuda/2020b``

<a anchor="mypy"/>
### *mypy*

Optional static typing for Python

*homepage*: http://www.mypy-lang.org/

version  |toolchain
---------|---------------
``0.4.5``|``intel/2016b``

<a anchor="mysql"/>
### *MySQL*

MySQL is one of the world's most widely used open-source  relational database management system (RDBMS).

*homepage*: http://www.mysql.com/

version   |versionsuffix  |toolchain
----------|---------------|------------------
``5.6.26``|``-clientonly``|``GNU/4.9.3-2.25``
``5.7.21``|``-clientonly``|``GCCcore/6.4.0``

<a anchor="mysql-python"/>
### *MySQL-python*

MySQL database connector for Python

*homepage*: https://github.com/farcepest/MySQLdb1

version  |versionsuffix                     |toolchain
---------|----------------------------------|---------------
``1.2.5``|``-Python-2.7.11``                |``intel/2016a``
``1.2.5``|``-Python-2.7.11-MariaDB-10.1.14``|``intel/2016a``

<a anchor="mysqlclient"/>
### *mysqlclient*

Python interface to MySQL

*homepage*: https://github.com/PyMySQL/mysqlclient-python

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``1.3.7``|``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``

<a anchor="n"/>
### *N*


<a href="#n2v">n2v</a> - <a href="#nag">NAG</a> - <a href="#nagfor">NAGfor</a> - <a href="#namd">NAMD</a> - <a href="#namedlist">namedlist</a> - <a href="#nano">nano</a> - <a href="#nanocomp">NanoComp</a> - <a href="#nanocompore">nanocompore</a> - <a href="#nanofilt">NanoFilt</a> - <a href="#nanofilt">nanofilt</a> - <a href="#nanoget">nanoget</a> - <a href="#nanolyse">NanoLyse</a> - <a href="#nanomath">nanomath</a> - <a href="#nanomax-analysis-utils">nanomax-analysis-utils</a> - <a href="#nanonet">nanonet</a> - <a href="#nanoplot">NanoPlot</a> - <a href="#nanopolish">nanopolish</a> - <a href="#nanopolishcomp">NanopolishComp</a> - <a href="#napari">napari</a> - <a href="#nasm">NASM</a> - <a href="#nauty">nauty</a> - <a href="#nbo">NBO</a> - <a href="#ncbi-toolkit">NCBI-Toolkit</a> - <a href="#ncbi-vdb">ncbi-vdb</a> - <a href="#nccl">NCCL</a> - <a href="#nccl-tests">NCCL-tests</a> - <a href="#ncdf4">ncdf4</a> - <a href="#ncdu">ncdu</a> - <a href="#nciplot">NCIPLOT</a> - <a href="#ncl">NCL</a> - <a href="#ncl">ncl</a> - <a href="#nco">NCO</a> - <a href="#ncompress">ncompress</a> - <a href="#ncurses">ncurses</a> - <a href="#ncview">ncview</a> - <a href="#nd2reader">nd2reader</a> - <a href="#ne">ne</a> - <a href="#neci">NECI</a> - <a href="#nedit">NEdit</a> - <a href="#nek5000">Nek5000</a> - <a href="#nektar++">Nektar++</a> - <a href="#neon">neon</a> - <a href="#neptune-client">neptune-client</a> - <a href="#net-core">Net-core</a> - <a href="#netcdf">netCDF</a> - <a href="#netcdf-c++">netCDF-C++</a> - <a href="#netcdf-c++4">netCDF-C++4</a> - <a href="#netcdf-fortran">netCDF-Fortran</a> - <a href="#netcdf4-python">netcdf4-python</a> - <a href="#netloc">netloc</a> - <a href="#netlogo">NetLogo</a> - <a href="#netmhc">netMHC</a> - <a href="#netmhciipan">netMHCIIpan</a> - <a href="#netmhcpan">netMHCpan</a> - <a href="#netpipe">NetPIPE</a> - <a href="#netpyne">NetPyNE</a> - <a href="#nettle">nettle</a> - <a href="#networktools">networkTools</a> - <a href="#networkx">networkx</a> - <a href="#neurokit">NeuroKit</a> - <a href="#neuron">NEURON</a> - <a href="#newhybrids">NewHybrids</a> - <a href="#nextflow">Nextflow</a> - <a href="#nextgenmap">NextGenMap</a> - <a href="#nf-core-mag">nf-core-mag</a> - <a href="#nfft">NFFT</a> - <a href="#nghttp2">nghttp2</a> - <a href="#nghttp3">nghttp3</a> - <a href="#ngless">NGLess</a> - <a href="#nglview">nglview</a> - <a href="#ngs">NGS</a> - <a href="#ngs-python">NGS-Python</a> - <a href="#ngsadmix">NGSadmix</a> - <a href="#ngspeciesid">NGSpeciesID</a> - <a href="#ngspice">ngspice</a> - <a href="#ngtcp2">ngtcp2</a> - <a href="#nibabel">NiBabel</a> - <a href="#nifti">NIfTI</a> - <a href="#nifti2dicom">nifti2dicom</a> - <a href="#nilearn">Nilearn</a> - <a href="#nim">Nim</a> - <a href="#nimble">NIMBLE</a> - <a href="#ninja">Ninja</a> - <a href="#nipype">Nipype</a> - <a href="#nlmpy">NLMpy</a> - <a href="#nlohmann_json">nlohmann_json</a> - <a href="#nlopt">NLopt</a> - <a href="#nltk">NLTK</a> - <a href="#nnu-net">nnU-Net</a> - <a href="#node-red">Node-RED</a> - <a href="#nodejs">nodejs</a> - <a href="#normaliz">Normaliz</a> - <a href="#nose-parameterized">nose-parameterized</a> - <a href="#novoalign">novoalign</a> - <a href="#novoplasty">NOVOPlasty</a> - <a href="#npstat">npstat</a> - <a href="#nrgljubljana">NRGLjubljana</a> - <a href="#nsight-compute">Nsight-Compute</a> - <a href="#nsight-systems">Nsight-Systems</a> - <a href="#nspr">NSPR</a> - <a href="#nss">NSS</a> - <a href="#nsync">nsync</a> - <a href="#ntcard">ntCard</a> - <a href="#ntedit">ntEdit</a> - <a href="#nthits">ntHits</a> - <a href="#ntl">NTL</a> - <a href="#ntpoly">NTPoly</a> - <a href="#num2words">num2words</a> - <a href="#numactl">numactl</a> - <a href="#numba">numba</a> - <a href="#numdiff">numdiff</a> - <a href="#numexpr">numexpr</a> - <a href="#numpy">numpy</a> - <a href="#nvhpc">NVHPC</a> - <a href="#nvshmem">NVSHMEM</a> - <a href="#nvtop">nvtop</a> - <a href="#nwchem">NWChem</a> - <a href="#nxtrim">NxTrim</a>


<a anchor="n2v"/>
### *n2v*

Noise2Void - Learning Denoising from Single Noisy Images The field of image denoising is currently dominated by discriminative deep       learning methods that are trained on pairs of noisy input and clean target       images. Recently it has been shown that such methods can also be trained without clean targets. Instead, independent pairs of noisy images can be used, in an     approach known as NOISE2NOISE (N2N). Here, we introduce NOISE2VOID (N2V), a      training scheme that takes this idea one step further. It does not require noisy image pairs, nor clean target images. Consequently, N2V allows us to train       directly on the body of data to be denoised and can therefore be applied when    other methods cannot. Especially interesting is the application to biomedical    image data, where the acquisition of training targets, clean or noisy, is        frequently not possible. We compare the performance of N2V to approaches that    have either clean target images and/or noisy image pairs available. Intuitively, N2V cannot be expected to outperform methods that have more information          available during training. Still, we observe that the denoising performance of   NOISE2VOID drops in moderation and compares favorably to training-free denoising methods.

*homepage*: https://github.com/juglab/n2v

version  |versionsuffix   |toolchain
---------|----------------|--------------
``0.3.2``|``-CUDA-11.3.1``|``foss/2021a``

<a anchor="nag"/>
### *NAG*

The worlds largest collection of robust, documented, tested and maintained numerical algorithms.

*homepage*: http://www.nag.co.uk

version|toolchain
-------|----------------------------------
``24`` |``PGI/17.4-GCC-6.4.0-2.28``
``26`` |``GCCcore/6.4.0``, ``intel/2018a``

<a anchor="nagfor"/>
### *NAGfor*

The checking compiler for improved code portability and detailed error reporting.

*homepage*: http://www.nag.co.uk

version   |toolchain
----------|----------
``6.2.14``|``system``

<a anchor="namd"/>
### *NAMD*

NAMD is a parallel molecular dynamics code designed for high-performance simulation of  large biomolecular systems.

*homepage*: https://www.ks.uiuc.edu/Research/namd/

version |versionsuffix   |toolchain
--------|----------------|----------------------------------------------------------------
``2.11``|``-mpi``        |``intel/2016a``
``2.12``|``-CUDA-8.0.61``|``foss/2016b``
``2.12``|``-mpi``        |``foss/2017a``, ``foss/2017b``, ``intel/2017a``, ``intel/2017b``
``2.13``|                |``fosscuda/2018b``
``2.13``|``-mpi``        |``foss/2018b``, ``foss/2019b``, ``intel/2018b``
``2.14``|                |``fosscuda/2019b``
``2.14``|``-CUDA-11.3.1``|``foss/2021a``
``2.14``|``-mpi``        |``foss/2019b``, ``foss/2020a``, ``intel/2020a``

<a anchor="namedlist"/>
### *namedlist*

A Python object, similar to namedtuple, but for lists.

*homepage*: https://gitlab.com/ericvsmith/namedlist

version|toolchain
-------|------------------
``1.8``|``GCCcore/11.2.0``

<a anchor="nano"/>
### *nano*

a simple editor, inspired by Pico

*homepage*: https://www.nano-editor.org/

version|toolchain
-------|------------------
``6.4``|``GCCcore/11.3.0``
``7.0``|``GCCcore/11.3.0``

<a anchor="nanocomp"/>
### *NanoComp*

Comparing runs of Oxford Nanopore sequencing data and alignments

*homepage*: https://github.com/wdecoster/NanoComp

version   |versionsuffix    |toolchain
----------|-----------------|---------------
``1.10.1``|``-Python-3.7.4``|``intel/2019b``
``1.13.1``|                 |``intel/2020b``

<a anchor="nanocompore"/>
### *nanocompore*

Nanocompore identifies differences in ONT nanopore sequencing raw signal corresponding to RNA modifications by comparing 2 samples

*homepage*: https://nanocompore.rna.rocks/

version       |versionsuffix    |toolchain
--------------|-----------------|---------------
``1.0.0rc3-2``|``-Python-3.8.2``|``intel/2020a``

<a anchor="nanofilt"/>
### *NanoFilt*

Filtering and trimming of Oxford Nanopore Sequencing data

*homepage*: https://github.com/wdecoster/nanofilt

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``2.6.0``|``-Python-3.7.4``|``intel/2019b``

<a anchor="nanofilt"/>
### *nanofilt*

Filtering and trimming of long read sequencing data.

*homepage*: https://github.com/wdecoster/nanofilt

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``2.5.0``|``-Python-3.6.6``|``foss/2018b``
``2.6.0``|``-Python-3.7.4``|``foss/2019b``
``2.6.0``|``-Python-3.8.2``|``intel/2020a``

<a anchor="nanoget"/>
### *nanoget*

Functions to extract information from Oxford Nanopore sequencing data and alignments

*homepage*: https://github.com/wdecoster/nanoget

version   |versionsuffix    |toolchain
----------|-----------------|---------------
``1.12.1``|``-Python-3.7.4``|``intel/2019b``
``1.15.0``|                 |``intel/2020b``
``1.18.1``|                 |``foss/2021a``

<a anchor="nanolyse"/>
### *NanoLyse*

Remove reads mapping to the lambda phage genome from a fastq file.

*homepage*: https://github.com/wdecoster/nanolyse

version  |toolchain
---------|--------------
``1.2.1``|``foss/2021a``

<a anchor="nanomath"/>
### *nanomath*

A few simple math function for other Oxford Nanopore processing scripts

*homepage*: https://github.com/wdecoster/nanomath

version   |versionsuffix    |toolchain
----------|-----------------|---------------
``0.23.1``|``-Python-3.7.4``|``intel/2019b``
``1.2.0`` |                 |``intel/2020b``
``1.2.1`` |                 |``foss/2021a``

<a anchor="nanomax-analysis-utils"/>
### *nanomax-analysis-utils*

A set of tools for handling and analysing data at the NanoMAX beamline.

*homepage*: https://github.com/maxiv-science/nanomax-analysis-utils

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------
``0.4``  |``-Python-3.7.4``|``foss/2019b``, ``fosscuda/2019b``
``0.4.3``|                 |``foss/2020b``, ``fosscuda/2020b``

<a anchor="nanonet"/>
### *nanonet*

Nanonet provides recurrent neural network basecalling for Oxford Nanopore MinION data.

*homepage*: https://github.com/nanoporetech/nanonet

version  |versionsuffix     |toolchain
---------|------------------|---------------
``2.0.0``|``-Python-2.7.13``|``intel/2017a``

<a anchor="nanoplot"/>
### *NanoPlot*

Plotting suite for long read sequencing data and alignments

*homepage*: https://github.com/wdecoster/NanoPlot

version   |versionsuffix    |toolchain
----------|-----------------|-------------------------------
``1.28.4``|``-Python-3.7.4``|``intel/2019b``
``1.33.0``|                 |``foss/2021a``, ``intel/2020b``

<a anchor="nanopolish"/>
### *nanopolish*

Software package for signal-level analysis of Oxford Nanopore sequencing data.

*homepage*: https://github.com/jts/nanopolish

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``0.9.2`` |                  |``intel/2018a``
``0.10.2``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``0.13.1``|``-Python-3.6.6`` |``foss/2018b``
``0.13.2``|``-Python-3.8.2`` |``foss/2020a``
``0.13.3``|                  |``foss/2020b``

<a anchor="nanopolishcomp"/>
### *NanopolishComp*

NanopolishComp is a Python3 package for downstream analyses of Nanopolish output files

*homepage*: https://adrienleger.com/NanopolishComp/

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.6.11``|``-Python-3.8.2``|``foss/2020a``

<a anchor="napari"/>
### *napari*

napari is a fast, interactive, multi-dimensional image viewer for Python. It's   designed for browsing, annotating, and analyzing large multi-dimensional images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based           rendering), and the scientific Python stack (numpy, scipy).

*homepage*: https://napari.org/

version   |toolchain
----------|--------------
``0.4.15``|``foss/2021b``

<a anchor="nasm"/>
### *NASM*

NASM: General-purpose x86 assembler

*homepage*: https://www.nasm.us/

version    |toolchain
-----------|--------------------------------------------------------------------------------------------------
``2.11.08``|``GCCcore/5.4.0``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016a``
``2.12.01``|``foss/2016a``, ``intel/2016a``
``2.12.02``|``foss/2016a``, ``foss/2016b``, ``intel/2016b``, ``intel/2017a``, ``system``
``2.13.01``|``GCCcore/6.3.0``, ``GCCcore/6.4.0``
``2.13.03``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``2.14.02``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``2.15.05``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``

<a anchor="nauty"/>
### *nauty*

nauty and Traces are programs for computing automorphism groups of graphs and digraphs. They can also produce a canonical label.

*homepage*: https://pallini.di.uniroma1.it/

version   |toolchain
----------|--------------------
``2.6r12``|``GCC/8.2.0-2.31.1``
``2.7rc2``|``GCC/8.2.0-2.31.1``
``2.7rc5``|``GCC/8.3.0``

<a anchor="nbo"/>
### *NBO*

The Natural Bond Orbital (NBO) program is a discovery tool for chemical insights from complex wavefunctions. 

*homepage*: http://nbo.chem.wisc.edu/

version|toolchain
-------|---------------
``1.1``|``intel/2016a``
``7.0``|``intel/2017b``

<a anchor="ncbi-toolkit"/>
### *NCBI-Toolkit*

The NCBI Toolkit is a collection of utilities developed for the   production and distribution of GenBank, Entrez, BLAST, and related services   by the National Center for Biotechnology Information.

*homepage*: http://ncbi.github.io/cxx-toolkit/

version   |toolchain
----------|---------------
``18.0.0``|``intel/2017a``

<a anchor="ncbi-vdb"/>
### *ncbi-vdb*

The SRA Toolkit and SDK from NCBI is a collection of tools and libraries for  using data in the INSDC Sequence Read Archives.

*homepage*: https://github.com/ncbi/ncbi-vdb

version    |toolchain
-----------|------------------------------------------------
``2.5.8-1``|``foss/2016b``, ``intel/2016a``
``2.7.0``  |``foss/2016b``
``2.8.2``  |``foss/2017b``, ``intel/2017a``, ``intel/2017b``
``2.9.1-1``|``foss/2018a``, ``intel/2018a``
``2.9.3``  |``foss/2018b``
``2.10.4`` |``gompi/2019b``
``2.10.7`` |``gompi/2020a``
``2.10.9`` |``gompi/2020b``, ``gompi/2021a``
``2.11.2`` |``gompi/2021b``
``3.0.0``  |``gompi/2022a``

<a anchor="nccl"/>
### *NCCL*

The NVIDIA Collective Communications Library (NCCL) implements multi-GPU and multi-node collective communication primitives that are performance optimized for NVIDIA GPUs.

*homepage*: https://developer.nvidia.com/nccl

version    |versionsuffix      |toolchain
-----------|-------------------|------------------------------
``2.1.4``  |``-CUDA-9.0.176``  |``system``
``2.2.13`` |``-CUDA-9.2.148.1``|``system``
``2.3.7``  |                   |``fosscuda/2018b``
``2.4.2``  |                   |``gcccuda/2019a``
``2.4.8``  |                   |``gcccuda/2019b``
``2.8.3``  |``-CUDA-11.0.2``   |``GCCcore/9.3.0``, ``system``
``2.8.3``  |``-CUDA-11.1.1``   |``GCCcore/10.2.0``, ``system``
``2.9.9``  |``-CUDA-11.3.1``   |``system``
``2.10.3`` |``-CUDA-11.3.1``   |``GCCcore/10.3.0``
``2.10.3`` |``-CUDA-11.4.1``   |``GCCcore/11.2.0``
``2.11.4`` |                   |``gcccuda/2019b``
``2.12.12``|``-CUDA-11.7.0``   |``GCCcore/11.3.0``
``2.16.2`` |``-CUDA-12.0.0``   |``GCCcore/12.2.0``

<a anchor="nccl-tests"/>
### *NCCL-tests*

Tests check both the performance and the correctness of NCCL operations.

*homepage*: https://github.com/NVIDIA/nccl-tests

version   |versionsuffix   |toolchain
----------|----------------|----------------
``2.0.0`` |                |``gompic/2019b``
``2.13.6``|``-CUDA-11.7.0``|``gompi/2022a``

<a anchor="ncdf4"/>
### *ncdf4*

ncdf4: Interface to Unidata netCDF (version 4 or earlier) format data files

*homepage*: https://cran.r-project.org/web/packages/ncdf4

version   |versionsuffix|toolchain
----------|-------------|-------------------------------
``1.16``  |``-R-3.4.0`` |``intel/2017a``
``1.16``  |``-R-3.4.3`` |``intel/2017b``
``1.16``  |``-R-3.4.4`` |``intel/2018a``
``1.16.1``|``-R-3.5.1`` |``foss/2018b``, ``intel/2018b``
``1.16.1``|``-R-3.6.0`` |``foss/2019a``
``1.17``  |             |``foss/2019b``
``1.17``  |``-R-4.0.0`` |``foss/2020a``
``1.17``  |``-R-4.0.3`` |``foss/2020b``
``1.17``  |``-R-4.1.0`` |``foss/2021a``

<a anchor="ncdu"/>
### *ncdu*

Ncdu is a disk usage analyzer with an ncurses interface. It is designed to find space hogs on a  remote server where you don't have an entire graphical setup available, but it is a useful tool even on regular  desktop systems. Ncdu aims to be fast, simple and easy to use, and should be able to run in any minimal POSIX-like  environment with ncurses installed.

*homepage*: https://dev.yorhel.nl/ncdu

version   |toolchain
----------|------------------------------
``1.13``  |``GCCcore/7.3.0``
``1.14``  |``GCCcore/7.3.0``
``1.15.1``|``GCCcore/9.3.0``
``1.16``  |``GCC/10.3.0``, ``GCC/11.2.0``
``1.17``  |``GCC/11.3.0``

<a anchor="nciplot"/>
### *NCIPLOT*

NCIPLOT is a program for revealing non covalent interactions  based on the reduced density gradient. 

*homepage*: https://www.lct.jussieu.fr/pagesperso/contrera/index-nci.html

version         |toolchain
----------------|-----------------------
``4.0-20190718``|``iccifort/2019.5.281``
``4.0-20200106``|``iccifort/2019.5.281``
``4.0-20200624``|``GCC/11.2.0``

<a anchor="ncl"/>
### *NCL*

NCL is an interpreted language designed specifically for scientific data analysis and visualization.

*homepage*: https://www.ncl.ucar.edu

version  |toolchain
---------|-----------------------------------------------------------------------------------------------------------------
``6.4.0``|``intel/2017a``, ``intel/2017b``, ``intel/2018a``
``6.5.0``|``intel/2018a``
``6.6.2``|``foss/2018b``, ``foss/2020a``, ``foss/2020b``, ``foss/2021a``, ``intel/2018b``, ``intel/2019b``, ``intel/2020a``

<a anchor="ncl"/>
### *ncl*

The NEXUS Class Library is a C++ library for parsing NEXUS files.

*homepage*: http://ncl.sourceforge.net/

version   |toolchain
----------|--------------------
``2.1.18``|``GCC/8.2.0-2.31.1``

<a anchor="nco"/>
### *NCO*

The NCO toolkit manipulates and analyzes data stored in netCDF-accessible formats,  including DAP, HDF4, and HDF5.

*homepage*: https://github.com/nco/nco

version  |toolchain
---------|-------------------------------
``4.6.0``|``foss/2016a``
``4.6.6``|``intel/2017a``
``4.7.0``|``intel/2017b``
``4.7.1``|``intel/2017b``
``4.7.4``|``foss/2017b``
``4.7.6``|``intel/2018a``
``4.7.9``|``foss/2018b``, ``intel/2018b``
``4.8.1``|``foss/2019a``
``4.9.3``|``foss/2019b``
``4.9.7``|``foss/2020a``, ``foss/2020b``
``5.0.1``|``foss/2021a``
``5.0.3``|``foss/2021b``, ``intel/2021b``
``5.1.0``|``foss/2022a``
``5.1.3``|``foss/2021a``

<a anchor="ncompress"/>
### *ncompress*

Compress is a fast, simple LZW file compressor. Compress does not have the  highest compression rate, but it is one of the fastest programs to compress  data. Compress is the defacto standard in the UNIX community for compressing  files. 

*homepage*: http://ncompress.sourceforge.net/

version    |toolchain
-----------|-----------------
``4.2.4.4``|``GCCcore/6.4.0``

<a anchor="ncurses"/>
### *ncurses*

The Ncurses (new curses) library is a free software emulation of curses in  System V Release 4.0, and more. It uses Terminfo format, supports pads and  color and multiple highlights and forms characters and function-key mapping,  and has all the other SYSV-curses enhancements over BSD Curses. 

*homepage*: https://www.gnu.org/software/ncurses/

version|toolchain
-------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``5.9``|``GCC/4.8.1``, ``GCC/4.8.2``, ``GCC/4.8.3``, ``GCC/4.8.4``, ``GCC/4.9.2``, ``GNU/4.9.3-2.25``, ``gimkl/2.11.5``, ``system``
``6.0``|``GCC/4.9.3-2.25``, ``GCC/5.4.0-2.26``, ``GCCcore/4.9.3``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.2.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GNU/4.9.3-2.25``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2017a``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
``6.1``|``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``system``
``6.2``|``FCC/4.5.0``, ``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``, ``system``
``6.3``|``GCCcore/11.3.0``, ``GCCcore/12.1.0``, ``GCCcore/12.2.0``, ``system``

<a anchor="ncview"/>
### *ncview*

Ncview is a visual browser for netCDF format files.  Typically you would use ncview to get a quick and easy, push-button  look at your netCDF files. You can view simple movies of the data,  view along various dimensions, take a look at the actual data values,  change color maps, invert the data, etc.

*homepage*: http://meteora.ucsd.edu/~pierce/ncview_home_page.html

version  |toolchain
---------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.1.7``|``foss/2017b``, ``foss/2018b``, ``foss/2019b``, ``gompi/2019a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``, ``intel/2019b``, ``iomkl/2018b``
``2.1.8``|``gompi/2020a``, ``gompi/2021a``, ``gompi/2021b``, ``gompi/2022a``

<a anchor="nd2reader"/>
### *nd2reader*

nd2reader is a pure-Python package that reads images produced by NIS Elements 4.0+.

*homepage*: https://www.lighthacking.nl/nd2reader/

version  |versionsuffix     |toolchain
---------|------------------|---------------
``3.0.6``|``-Python-2.7.14``|``intel/2017b``

<a anchor="ne"/>
### *ne*

ne is a free (GPL'd) text editor based on the POSIX standard that runs (we hope) on almost any UN*X machine. ne is easy to use for the beginner, but powerful and fully configurable for the wizard, and most sparing in its resource usage.

*homepage*: http://ne.di.unimi.it/

version  |toolchain
---------|---------------
``3.0.1``|``gimkl/2017a``

<a anchor="neci"/>
### *NECI*

Standalone NECI codebase designed for FCIQMC and other stochastic quantum chemistry methods.

*homepage*: https://github.com/ghb24/NECI_STABLE

version     |toolchain
------------|--------------
``20220711``|``foss/2022a``

<a anchor="nedit"/>
### *NEdit*

NEdit is a multi-purpose text editor for the X Window System,  which combines a standard, easy to use, graphical user interface with the thorough  functionality and stability required by users who edit text eight hours a day.

*homepage*: http://www.nedit.org

version|versionsuffix |toolchain
-------|--------------|----------
``5.5``|``-Linux-x86``|``system``

<a anchor="nek5000"/>
### *Nek5000*

a fast and scalable high-order solver for computational fluid dynamics

*homepage*: https://nek5000.mcs.anl.gov/

version |toolchain
--------|---------------
``17.0``|``intel/2018a``

<a anchor="nektar++"/>
### *Nektar++*

Nektar++ is a tensor product based finite element package designed to allow one to construct efficient classical low polynomial order h-type solvers (where h is the size of the finite element) as well as higher p-order piecewise polynomial order solvers.

*homepage*: https://www.nektar.info

version  |toolchain
---------|--------------
``5.0.1``|``foss/2020b``

<a anchor="neon"/>
### *neon*

neon is an HTTP/1.1 and WebDAV client library, with a C interface. 

*homepage*: https://github.com/notroj/neon

version   |toolchain
----------|-----------------
``0.31.2``|``GCCcore/8.3.0``

<a anchor="neptune-client"/>
### *neptune-client*

Neptune is an experiment tracking hub that brings organization and collaboration to your data science team. 

*homepage*: https://docs.neptune.ai

version    |versionsuffix    |toolchain
-----------|-----------------|------------------------------
``0.4.129``|``-Python-3.8.2``|``foss/2020a``
``0.10.5`` |                 |``foss/2020b``
``0.16.2`` |                 |``foss/2021a``, ``foss/2022a``

<a anchor="net-core"/>
### *Net-core*

.NET Core is a free and open-source managed computer software framework for the Windows,  Linux, and macOS operating systems .NET Core fully supports C# and F# and partially supports Visual Basic

*homepage*: https://dotnet.microsoft.com/

version  |toolchain
---------|----------
``2.1.8``|``system``
``2.2.5``|``system``
``3.0.0``|``system``

<a anchor="netcdf"/>
### *netCDF*

NetCDF (network Common Data Form) is a set of software libraries  and machine-independent data formats that support the creation, access, and sharing of array-oriented  scientific data.

*homepage*: https://www.unidata.ucar.edu/software/netcdf/

version    |versionsuffix   |toolchain
-----------|----------------|--------------------------------------------------------------------------------------------------------------------------
``4.3.3.1``|                |``foss/2016a``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``
``4.4.0``  |                |``foss/2016a``, ``intel/2016a``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``
``4.4.1``  |                |``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``4.4.1.1``|                |``foss/2016b``, ``foss/2017b``, ``fosscuda/2017b``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intelcuda/2017b``
``4.4.1.1``|``-HDF5-1.10.1``|``foss/2017a``, ``intel/2017a``
``4.4.1.1``|``-HDF5-1.8.18``|``intel/2017a``
``4.4.1.1``|``-HDF5-1.8.19``|``foss/2017a``, ``foss/2017b``, ``intel/2017b``
``4.5.0``  |                |``foss/2017b``, ``intel/2017b``, ``intel/2018.00``, ``intel/2018.01``
``4.6.0``  |                |``foss/2018a``, ``intel/2018a``, ``iomkl/2018a``
``4.6.1``  |                |``PGI/18.4-GCC-6.4.0-2.28``, ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``, ``iomkl/2018b``
``4.6.2``  |                |``gompi/2019a``, ``iimpi/2019a``
``4.7.1``  |                |``gompi/2019b``, ``gompic/2019b``, ``iimpi/2019b``, ``iimpic/2019b``
``4.7.4``  |                |``gompi/2020a``, ``gompi/2020b``, ``gompic/2020a``, ``gompic/2020b``, ``iimpi/2020a``, ``iimpi/2020b``, ``iimpic/2020b``
``4.8.0``  |                |``gompi/2021a``, ``iimpi/2021a``
``4.8.1``  |                |``gompi/2021b``, ``iimpi/2021b``
``4.9.0``  |                |``gompi/2022a``, ``iimpi/2022a``

<a anchor="netcdf-c++"/>
### *netCDF-C++*

NetCDF (network Common Data Form) is a set of software libraries   and machine-independent data formats that support the creation, access, and sharing of array-oriented   scientific data.

*homepage*: http://www.unidata.ucar.edu/software/netcdf/

version|toolchain
-------|--------------
``4.2``|``foss/2016a``

<a anchor="netcdf-c++4"/>
### *netCDF-C++4*

NetCDF (network Common Data Form) is a set of software libraries  and machine-independent data formats that support the creation, access, and sharing of array-oriented  scientific data.

*homepage*: https://www.unidata.ucar.edu/software/netcdf/

version  |versionsuffix   |toolchain
---------|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``4.3.0``|                |``foss/2018b``, ``gompi/2019a``, ``iimpi/2019a``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2018a``, ``intel/2018b``, ``iomkl/2018b``
``4.3.0``|``-HDF5-1.8.18``|``intel/2017a``
``4.3.0``|``-HDF5-1.8.19``|``intel/2017b``
``4.3.1``|                |``gompi/2019b``, ``gompi/2020a``, ``gompi/2020b``, ``gompi/2021a``, ``gompi/2021b``, ``gompi/2022a``, ``iimpi/2019b``, ``iimpi/2020a``, ``iimpi/2020b``, ``iimpi/2021a``, ``iimpi/2021b``, ``iimpi/2022a``

<a anchor="netcdf-fortran"/>
### *netCDF-Fortran*

NetCDF (network Common Data Form) is a set of software libraries  and machine-independent data formats that support the creation, access, and sharing of array-oriented  scientific data.

*homepage*: https://www.unidata.ucar.edu/software/netcdf/

version  |versionsuffix   |toolchain
---------|----------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``4.4.1``|                |``foss/2016b``, ``intel/2016b``
``4.4.2``|                |``intel/2016.02-GCC-4.9``
``4.4.3``|                |``foss/2016a``, ``intel/2016a``
``4.4.4``|                |``PGI/18.4-GCC-6.4.0-2.28``, ``foss/2016b``, ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``fosscuda/2017b``, ``fosscuda/2018b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``, ``intelcuda/2017b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``iomkl/2018b``
``4.4.4``|``-HDF5-1.8.18``|``intel/2017a``
``4.4.4``|``-HDF5-1.8.19``|``foss/2017b``, ``intel/2017b``
``4.4.5``|                |``gompi/2019a``, ``iimpi/2019a``
``4.5.2``|                |``gompi/2019b``, ``gompi/2020a``, ``gompic/2019b``, ``gompic/2020a``, ``iimpi/2019b``, ``iimpi/2020a``, ``iimpic/2019b``
``4.5.3``|                |``gompi/2020b``, ``gompi/2021a``, ``gompi/2021b``, ``gompic/2020b``, ``iimpi/2020b``, ``iimpi/2021a``, ``iimpi/2021b``
``4.6.0``|                |``gompi/2022a``, ``iimpi/2022a``

<a anchor="netcdf4-python"/>
### *netcdf4-python*

Python/numpy interface to netCDF.

*homepage*: https://unidata.github.io/netcdf4-python/

version    |versionsuffix                |toolchain
-----------|-----------------------------|---------------------------------------------------
``1.2.9``  |``-Python-2.7.13``           |``intel/2017a``
``1.3.1``  |``-Python-3.6.3``            |``intel/2017b``
``1.3.1``  |``-Python-3.6.3-HDF5-1.8.19``|``intel/2017b``
``1.4.0``  |``-Python-3.6.2-HDF5-1.8.19``|``foss/2017b``
``1.4.0``  |``-Python-3.6.4``            |``intel/2018a``
``1.4.1``  |``-Python-2.7.15``           |``intel/2018b``
``1.4.1``  |``-Python-3.6.6``            |``foss/2018b``, ``intel/2018b``
``1.4.2``  |``-Python-3.6.4``            |``foss/2018a``
``1.4.3``  |``-Python-3.6.6``            |``foss/2018b``
``1.5.2``  |                             |``intel/2019a``
``1.5.3``  |``-Python-2.7.16``           |``foss/2019b``
``1.5.3``  |``-Python-3.7.4``            |``foss/2019b``, ``intel/2019b``
``1.5.3``  |``-Python-3.8.2``            |``foss/2020a``, ``intel/2020a``
``1.5.5.1``|                             |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``
``1.5.7``  |                             |``foss/2021a``, ``foss/2021b``, ``intel/2021b``
``1.6.1``  |                             |``foss/2022a``, ``intel/2022a``

<a anchor="netloc"/>
### *netloc*

The Portable Network Locality (netloc) software package provides  network topology discovery tools, and an abstract representation of those networks  topologies for a range of network types and configurations.  It is provided as a companion to the Portable Hardware Locality (hwloc) package.

*homepage*: http://www.open-mpi.org/software/netloc

version|toolchain
-------|-------------
``0.5``|``GCC/4.8.3``

<a anchor="netlogo"/>
### *NetLogo*

NetLogo is a multi-agent programmable modeling environment. It is used by tens of thousands of students, teachers and researchers worldwide. It also powers HubNet participatory simulations. It is authored by Uri Wilensky and developed at the CCL.

*homepage*: https://ccl.northwestern.edu/netlogo/

version  |versionsuffix|toolchain
---------|-------------|----------
``5.3.1``|``-64``      |``system``
``6.0.4``|``-64``      |``system``
``6.2.0``|``-64``      |``system``

<a anchor="netmhc"/>
### *netMHC*

NetMHC 4.0 software  predicts binding of peptides to a number of different  HLA alleles  using artificial neural networks (ANN). 

*homepage*: http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?netMHC

version |toolchain
--------|----------
``4.0a``|``system``

<a anchor="netmhciipan"/>
### *netMHCIIpan*

NetMHCIIpan 3.2 server predicts binding of peptides to MHC  class II molecules. The predictions are available for the three human MHC  class II isotypes HLA-DR, HLA-DP and HLA-DQ, as well as mouse molecules (H-2).

*homepage*: https://www.cbs.dtu.dk/services/NetMHCIIpan/

version|toolchain
-------|------------------------------------
``3.2``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``

<a anchor="netmhcpan"/>
### *netMHCpan*

The NetMHCpan software predicts  binding of peptides  to any known MHC  molecule using artificial neural networks (ANNs). 

*homepage*: http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?netMHCpan

version |toolchain
--------|----------
``4.0a``|``system``

<a anchor="netpipe"/>
### *NetPIPE*

NetPIPE is a protocol independent communication performance benchmark that visually represents  the network performance under a variety of conditions.

*homepage*: https://netpipe.cs.ksu.edu/

version  |toolchain
---------|--------------------------------
``5.1``  |``intel/2018a``
``5.1.4``|``gompi/2020b``, ``iimpi/2020b``

<a anchor="netpyne"/>
### *NetPyNE*

NetPyNE is an open-source Python package to facilitate the development, parallel simulation, analysis, and optimization of biological neuronal networks using the NEURON simulator.

*homepage*: http://www.netpyne.org

version    |toolchain
-----------|--------------
``1.0.2.1``|``foss/2021b``

<a anchor="nettle"/>
### *nettle*

Nettle is a cryptographic library that is designed to fit easily  in more or less any context: In crypto toolkits for object-oriented  languages (C++, Python, Pike, ...), in applications like LSH or GNUPG,  or even in kernel space.

*homepage*: https://www.lysator.liu.se/~nisse/nettle/

version  |toolchain
---------|-------------------------------------------------------------------------------------------------------------------------------------------
``3.1.1``|``GNU/4.9.3-2.25``, ``foss/2016a``, ``intel/2016a``
``3.2``  |``GCCcore/5.4.0``, ``foss/2016b``, ``intel/2016b``
``3.3``  |``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``intel/2017a``
``3.4``  |``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``foss/2018a``, ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018a``, ``intel/2018b``, ``iomkl/2018a``
``3.4.1``|``GCCcore/8.2.0``
``3.5.1``|``GCCcore/8.3.0``
``3.6``  |``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``3.7.2``|``GCCcore/10.3.0``
``3.7.3``|``GCCcore/11.2.0``
``3.8``  |``GCCcore/11.3.0``

<a anchor="networktools"/>
### *networkTools*

Dynamical Network Analysis is a method of characterizing allosteric signalling through biomolecular complexes.

*homepage*: http://faculty.scs.illinois.edu/schulten/software/networkTools/

version|toolchain
-------|-------------
``2``  |``GCC/9.3.0``

<a anchor="networkx"/>
### *networkx*

NetworkX is a Python package for the creation, manipulation, and study of the structure, dynamics, and functions of complex networks.

*homepage*: https://pypi.python.org/pypi/networkx

version  |versionsuffix     |toolchain
---------|------------------|------------------------------------------------------------------------
``1.1``  |``-Python-2.7.12``|``intel/2016b``
``1.11`` |``-Python-2.7.11``|``foss/2016a``
``1.11`` |``-Python-2.7.14``|``intel/2017b``
``1.11`` |``-Python-3.5.1`` |``foss/2016a``
``2.0``  |``-Python-2.7.14``|``intel/2017b``
``2.1``  |``-Python-2.7.14``|``intel/2017b``
``2.2``  |``-Python-2.7.15``|``foss/2018b``, ``foss/2019a``, ``intel/2018b``
``2.2``  |``-Python-2.7.16``|``foss/2019b``
``2.2``  |``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``2.3``  |``-Python-3.7.2`` |``foss/2019a``, ``intel/2019a``
``2.4``  |``-Python-3.7.2`` |``foss/2019a``, ``intel/2019a``
``2.4``  |``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``, ``intelcuda/2019b``
``2.4``  |``-Python-3.8.2`` |``foss/2020a``, ``fosscuda/2020a``, ``intel/2020a``
``2.5``  |                  |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``, ``intelcuda/2020b``
``2.5.1``|                  |``foss/2021a``
``2.6.2``|                  |``foss/2020b``
``2.6.3``|                  |``foss/2021a``, ``foss/2021b``, ``intel/2021b``
``2.8.4``|                  |``foss/2022a``

<a anchor="neurokit"/>
### *NeuroKit*

NeuroKit is a Python module that provides high-level integrative functions with good and  flexible defaults, allowing users to focus on what’s important.

*homepage*: https://neurokit.readthedocs.io

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.2.7``|``-Python-3.6.4``|``intel/2018a``

<a anchor="neuron"/>
### *NEURON*

Empirically-based simulations of neurons and networks of neurons.

*homepage*: https://www.neuron.yale.edu/neuron

version  |versionsuffix     |toolchain
---------|------------------|---------------
``7.4``  |                  |``intel/2016b``
``7.6.5``|``-Python-2.7.15``|``intel/2018b``
``7.8.2``|                  |``foss/2021b``

<a anchor="newhybrids"/>
### *NewHybrids*

This implements a Gibbs sampler to estimate the posterior probability that genetically sampled individuals fall into each of a set of user-defined hybrid categories.

*homepage*: https://ib.berkeley.edu/labs/slatkin/eriq/software/software.htm#NewHybs

version      |toolchain
-------------|--------------
``1.1_Beta3``|``GCC/10.2.0``

<a anchor="nextflow"/>
### *Nextflow*

Nextflow is a reactive workflow framework and a programming DSL   that eases writing computational pipelines with complex data

*homepage*: https://www.nextflow.io/

version    |toolchain
-----------|----------
``19.04.0``|``system``
``19.07.0``|``system``
``19.12.0``|``system``
``20.01.0``|``system``
``20.04.1``|``system``
``20.10.0``|``system``
``21.03.0``|``system``
``21.08.0``|``system``
``21.10.6``|``system``
``22.04.0``|``system``
``22.10.0``|``system``
``22.10.1``|``system``

<a anchor="nextgenmap"/>
### *NextGenMap*

NextGenMap is a flexible highly sensitive short read mapping tool that  handles much higher mismatch rates than comparable algorithms while still outperforming  them in terms of runtime.

*homepage*: http://cibiv.github.io/NextGenMap/

version  |toolchain
---------|------------------------------
``0.5.5``|``GCC/11.2.0``, ``foss/2016b``

<a anchor="nf-core-mag"/>
### *nf-core-mag*

The Nextflow pipeline 'mag' ported to EasyBuild/EESSI.

*homepage*: https://nf-co.re/mag

version     |toolchain
------------|--------------
``20221110``|``foss/2021a``

<a anchor="nfft"/>
### *NFFT*

The NFFT (nonequispaced fast Fourier transform or nonuniform fast Fourier transform) is a C subroutine  library for computing the nonequispaced discrete Fourier transform (NDFT) and its generalisations in one or more  dimensions, of arbitrary input size, and of complex data.

*homepage*: https://www-user.tu-chemnitz.de/~potts/nfft/

version  |toolchain
---------|----------------------------------------------
``3.1.3``|``foss/2020b``, ``fosscuda/2020b``
``3.5.1``|``foss/2018b``, ``foss/2019a``
``3.5.2``|``foss/2020a``, ``foss/2021a``, ``foss/2021b``
``3.5.3``|``foss/2022a``

<a anchor="nghttp2"/>
### *nghttp2*

This is an implementation of the Hypertext Transfer Protocol version 2 in C.  The framing layer of HTTP/2 is implemented as a reusable C library.  On top of that, we have implemented an HTTP/2 client, server and proxy.  We have also developed load test and benchmarking tools for HTTP/2.  An HPACK encoder and decoder are available as a public API.

*homepage*: https://github.com/nghttp2/nghttp2

version   |toolchain
----------|--------------
``1.48.0``|``GCC/11.2.0``

<a anchor="nghttp3"/>
### *nghttp3*

nghttp3 is an implementation of RFC 9114 HTTP/3  mapping over QUIC and RFC 9204 QPACK in C. It does not depend on any particular QUIC transport implementation.

*homepage*: https://github.com/ngtcp2/nghttp3

version  |toolchain
---------|--------------------------------------
``0.6.0``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="ngless"/>
### *NGLess*

NGLess is a domain-specific language for NGS (next-generation sequencing data) processing.

*homepage*: https://ngless.embl.de/

version  |versionsuffix      |toolchain
---------|-------------------|----------
``1.1.1``|``-static-Linux64``|``system``

<a anchor="nglview"/>
### *nglview*

IPython widget to interactively view molecular structures and trajectories.

*homepage*: https://github.com/arose/nglview

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``2.7.0``|``-Python-3.7.2``|``intel/2019a``
``2.7.7``|``-Python-3.8.2``|``intel/2020a``
``3.0.3``|                 |``foss/2021a``

<a anchor="ngs"/>
### *NGS*

NGS is a new, domain-specific API for accessing reads, alignments and pileups produced from  Next Generation Sequencing.

*homepage*: https://github.com/ncbi/ngs

version   |versionsuffix      |toolchain
----------|-------------------|--------------------------------------
``1.2.3`` |                   |``foss/2016b``, ``intel/2016a``
``1.2.5`` |                   |``foss/2016b``
``1.3.0`` |                   |``GCCcore/6.4.0``, ``intel/2017a``
``2.9.1`` |``-Java-1.8.0_162``|``foss/2018a``, ``intel/2018a``
``2.9.3`` |``-Java-1.8``      |``foss/2018b``
``2.10.0``|``-Java-1.8``      |``GCCcore/8.2.0``
``2.10.0``|``-Java-11``       |``GCCcore/8.2.0``
``2.10.4``|``-Java-11``       |``GCCcore/8.3.0``
``2.10.5``|                   |``GCCcore/9.3.0``
``2.10.9``|                   |``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``2.11.2``|                   |``GCCcore/11.2.0``

<a anchor="ngs-python"/>
### *NGS-Python*

NGS is a new, domain-specific API for accessing reads, alignments and pileups produced from  Next Generation Sequencing.

*homepage*: https://github.com/ncbi/ngs

version   |versionsuffix     |toolchain
----------|------------------|---------------
``1.2.3`` |``-Python-2.7.11``|``intel/2016a``
``2.9.3`` |``-Python-3.6.6`` |``intel/2018b``
``2.10.4``|``-Python-2.7.16``|``gompi/2019b``

<a anchor="ngsadmix"/>
### *NGSadmix*

NGSadmix is a tool for finding admixture proportions  from NGS data, based on genotype likelihoods.

*homepage*: http://www.popgen.dk/software/index.php/NgsAdmix

version|toolchain
-------|------------------
``32`` |``GCC/7.3.0-2.30``

<a anchor="ngspeciesid"/>
### *NGSpeciesID*

NGSpeciesID is a tool for clustering and consensus forming of targeted ONT reads.

*homepage*: https://github.com/ksahlin/NGSpeciesID

version    |versionsuffix    |toolchain
-----------|-----------------|--------------
``0.1.1.1``|``-Python-3.7.4``|``foss/2019b``
``0.1.2.1``|                 |``foss/2021b``

<a anchor="ngspice"/>
### *ngspice*

Ngspice is a mixed-level/mixed-signal circuit simulator. Its code  is based on three open source software packages: Spice3f5, Cider1b1  and Xspice. 

*homepage*: https://ngspice.sourceforge.net

version|toolchain
-------|--------------
``31`` |``foss/2019b``

<a anchor="ngtcp2"/>
### *ngtcp2*

'Call it TCP/2. One More Time.'  ngtcp2 project is an effort to implement RFC9000 QUIC protocol.

*homepage*: https://github.com/ngtcp2/ngtcp2

version  |toolchain
---------|------------------------------
``0.7.0``|``GCC/11.2.0``, ``GCC/11.3.0``

<a anchor="nibabel"/>
### *NiBabel*

NiBabel provides read/write access to some common medical and neuroimaging file formats,  including: ANALYZE (plain, SPM99, SPM2 and later), GIFTI, NIfTI1, NIfTI2, MINC1, MINC2, MGH and ECAT  as well as Philips PAR/REC. We can read and write Freesurfer geometry, and read Freesurfer morphometry and  annotation files. There is some very limited support for DICOM. NiBabel is the successor of PyNIfTI.

*homepage*: https://nipy.github.io/nibabel

version  |versionsuffix                    |toolchain
---------|---------------------------------|--------------------------------------------------
``2.0.2``|``-Python-2.7.11``               |``intel/2016a``
``2.0.2``|``-Python-2.7.11-freetype-2.6.3``|``intel/2016a``
``2.1.0``|``-Python-2.7.13``               |``intel/2017a``
``2.2.1``|``-Python-3.6.4``                |``intel/2018a``
``2.3.0``|``-Python-2.7.14``               |``foss/2017b``, ``intel/2017b``
``2.3.0``|``-Python-3.6.3``                |``foss/2017b``, ``intel/2017b``
``2.3.0``|``-Python-3.6.6``                |``foss/2018b``
``2.4.0``|                                 |``foss/2019a``, ``intel/2019a``
``2.5.1``|``-Python-3.6.6``                |``foss/2018b``
``3.1.0``|``-Python-3.7.4``                |``foss/2019b``, ``intel/2019b``
``3.2.0``|``-Python-3.8.2``                |``foss/2020a``
``3.2.1``|                                 |``foss/2020b``, ``foss/2021a``, ``fosscuda/2020b``
``3.2.2``|                                 |``foss/2021b``
``4.0.2``|                                 |``foss/2022a``

<a anchor="nifti"/>
### *NIfTI*

Niftilib is a set of i/o libraries for reading and writing files in the nifti-1 data format.

*homepage*: http://niftilib.sourceforge.net/

version  |toolchain
---------|-----------------
``2.0.0``|``GCCcore/6.4.0``

<a anchor="nifti2dicom"/>
### *nifti2dicom*

Nifti2Dicom is a conversion tool that converts 3D NIfTI files (and other formats supported by ITK, including Analyze, MetaImage Nrrd and VTK) to DICOM. Unlike other conversion tools, it can import a DICOM file that is used to import the patient and study DICOM tags, and allows you to edit the accession number and other DICOM tags, in order to create a valid DICOM that can be imported in a PACS.

*homepage*: https://github.com/biolab-unige/nifti2dicom

version   |toolchain
----------|--------------
``0.4.11``|``foss/2019b``

<a anchor="nilearn"/>
### *Nilearn*

Nilearn is a Python module for fast and easy statistical learning on NeuroImaging data.

*homepage*: https://nilearn.github.io/

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``0.5.2``|                  |``foss/2019a``, ``intel/2019a``
``0.5.2``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``0.5.2``|``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``0.7.0``|``-Python-3.8.2`` |``foss/2020a``
``0.7.1``|                  |``foss/2020b``

<a anchor="nim"/>
### *Nim*

Nim is a systems and applications programming language.

*homepage*: https://nim-lang.org/

version   |toolchain
----------|------------------
``0.18.0``|``GCCcore/6.4.0``
``0.19.2``|``GCCcore/7.3.0``
``1.0.0`` |``GCCcore/8.3.0``
``1.4.6`` |``GCCcore/10.2.0``
``1.4.8`` |``GCCcore/10.3.0``
``1.6.6`` |``GCCcore/11.2.0``

<a anchor="nimble"/>
### *NIMBLE*

NIMBLE is a system for building and sharing analysis methods for statistical models,  especially for hierarchical models and computationally-intensive methods.

*homepage*: https://r-nimble.org

version  |versionsuffix|toolchain
---------|-------------|--------------
``0.7.0``|``-R-3.5.1`` |``foss/2018b``

<a anchor="ninja"/>
### *Ninja*

Ninja is a small build system with a focus on speed.

*homepage*: https://ninja-build.org/

version   |toolchain
----------|------------------------------------------------------------------------------------
``1.8.2`` |``foss/2018a``, ``foss/2018b``, ``fosscuda/2018b``, ``intel/2017b``, ``intel/2018a``
``1.9.0`` |``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``foss/2018b``
``1.10.0``|``GCCcore/9.3.0``
``1.10.1``|``GCCcore/10.2.0``
``1.10.2``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``1.11.1``|``GCCcore/12.2.0``

<a anchor="nipype"/>
### *Nipype*

Nipype is a Python project that provides a uniform interface to existing neuroimaging software and  facilitates interaction between these packages within a single workflow.

*homepage*: https://nipype.readthedocs.io

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``1.0.2``|``-Python-3.6.4``|``intel/2018a``
``1.1.3``|``-Python-3.6.6``|``foss/2018b``
``1.4.2``|``-Python-3.7.4``|``intel/2019b``
``1.8.5``|                 |``foss/2021a``

<a anchor="nlmpy"/>
### *NLMpy*

NLMpy is a Python package for the creation of neutral landscape models that are widely used  in the modelling of ecological patterns and processes across landscapes.

*homepage*: https://pypi.org/project/nlmpy

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.1.5``|``-Python-3.7.4``|``intel/2019b``

<a anchor="nlohmann_json"/>
### *nlohmann_json*

JSON for Modern C++

*homepage*: https://github.com/nlohmann/json

version   |toolchain
----------|------------------
``3.10.0``|``GCCcore/10.3.0``
``3.10.4``|``GCCcore/11.2.0``
``3.10.5``|``GCCcore/11.3.0``

<a anchor="nlopt"/>
### *NLopt*

NLopt is a free/open-source library for nonlinear optimization,  providing a common interface for a number of different free optimization routines  available online as well as original implementations of various other algorithms. 

*homepage*: http://ab-initio.mit.edu/wiki/index.php/NLopt

version  |toolchain
---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.4.2``|``GCC/5.4.0-2.26``, ``GCCcore/7.3.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017b``, ``foss/2018a``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``, ``iomkl/2018a``
``2.6.1``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``2.6.2``|``GCCcore/10.2.0``
``2.7.0``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``2.7.1``|``GCCcore/11.3.0``

<a anchor="nltk"/>
### *NLTK*

NLTK is a leading platform for building Python programs to work with human language data.

*homepage*: https://www.nltk.org/

version  |versionsuffix     |toolchain
---------|------------------|----------------------------------
``3.2.2``|``-Python-2.7.13``|``intel/2017a``
``3.2.4``|``-Python-2.7.13``|``intel/2017a``
``3.5``  |``-Python-3.7.4`` |``foss/2019b``
``3.5``  |``-Python-3.8.2`` |``foss/2020a``, ``fosscuda/2020a``
``3.6.7``|                  |``foss/2021a``
``3.7``  |                  |``foss/2021b``

<a anchor="nnu-net"/>
### *nnU-Net*

nnU-Net is the first segmentation method that is designed to deal with the dataset diversity found  in the domain It condenses and automates the keys decisions for designing a successful segmentation pipeline for  any given dataset.

*homepage*: https://github.com/MIC-DKFZ/nnUNet

version  |toolchain
---------|--------------
``1.7.0``|``foss/2020b``

<a anchor="node-red"/>
### *Node-RED*

Node-RED is a programming tool for wiring together hardware devices, APIs and online services  in new and interesting ways.

*homepage*: http://nodered.org

version   |toolchain
----------|--------------
``0.16.2``|``foss/2017a``

<a anchor="nodejs"/>
### *nodejs*

Node.js is a platform built on Chrome's JavaScript runtime   for easily building fast, scalable network applications. Node.js uses an   event-driven, non-blocking I/O model that makes it lightweight and efficient,   perfect for data-intensive real-time applications that run across distributed devices.

*homepage*: http://nodejs.org

version    |toolchain
-----------|-------------------------------------------------------
``4.4.7``  |``foss/2016a``
``6.10.3`` |``foss/2017a``
``8.9.4``  |``foss/2017a``
``10.15.1``|``foss/2018b``
``10.15.3``|``GCCcore/8.2.0``
``12.16.1``|``GCCcore/7.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``12.19.0``|``GCCcore/10.2.0``
``14.17.0``|``GCCcore/10.3.0``
``14.17.2``|``GCCcore/10.3.0``
``14.17.6``|``GCCcore/11.2.0``
``16.15.1``|``GCCcore/11.3.0``
``18.12.1``|``GCCcore/12.2.0``

<a anchor="normaliz"/>
### *Normaliz*

Normaliz is a open source tool for computations in affine monoids, vector configurations, rational polyhedra and rational cones. Normaliz now computes rational and algebraic polyhedra, i.e., polyhedra defined over real algebraic extensions of QQ.

*homepage*: https://www.normaliz.uni-osnabrueck.de/

version  |toolchain
---------|---------------
``3.6.3``|``intel/2018b``
``3.7.4``|``gompi/2019a``
``3.8.4``|``GCC/8.3.0``

<a anchor="nose-parameterized"/>
### *nose-parameterized*

Parameterized testing with any Python test framework.

*homepage*: hmat://github.com/wolever/nose-parameterized

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.5.0``|``-Python-2.7.11``|``foss/2016a``
``0.5.0``|``-Python-3.5.1`` |``foss/2016a``
``0.5.0``|``-Python-3.5.2`` |``intel/2016b``
``0.6.0``|``-Python-3.6.1`` |``intel/2017a``

<a anchor="novoalign"/>
### *novoalign*

NovoCraft is a software bundle.  NovoAlign:  Market’s leading aligner with fully packed features designed for mapping  of short reads onto a reference genome from Illumina, Ion Torrent, 454,  and Color Spance NGS platforms.  NovoAlignCS:  Leading aligner packed with features designed to fully support reads from  ABI SOLiD Color Space.  NovoSort:  Custom designed multi-threaded sort/merge tools for BAM files.  NovoMethyl:  It can analyse a set of alignments to identify methylated cytosine’s.  WARNING!  You can only use the sofware without a license (with some features disabled)  if you are a non-profit organisation and use is for your own research or for  use by students as part of their course. A license is required for use where  these programs are part of a service where a third party is billed.

*homepage*: http://www.novocraft.com/

version    |versionsuffix|toolchain
-----------|-------------|--------------
``3.09.00``|             |``system``
``3.09.01``|``-R-3.5.1`` |``foss/2018b``

<a anchor="novoplasty"/>
### *NOVOPlasty*

NOVOPlasty is a de novo assembler and heteroplasmy/variance caller for short circular genomes.

*homepage*: https://github.com/ndierckx/NOVOPlasty

version|toolchain
-------|-----------------
``3.7``|``GCCcore/8.3.0``

<a anchor="npstat"/>
### *npstat*

npstat implements some population genetics tests and estimators that can be applied  to pooled sequences from Next Generation Sequencing experiments.

*homepage*: https://code.google.com/archive/p/npstat/

version |toolchain
--------|-------------------------------
``0.99``|``foss/2016a``, ``intel/2016a``

<a anchor="nrgljubljana"/>
### *NRGLjubljana*

NRG Ljubljana is an efficient implementation of the numerical renormalization group (NRG)   technique for solving quantum impurity problems that arise as simplified models of   magnetic impurities and as effective models in the dynamical mean field theory (DMFT)   approach to bulk correlated materials. 

*homepage*: http://nrgljubljana.ijs.si/

version     |toolchain
------------|--------------
``2.4.3.23``|``foss/2016b``

<a anchor="nsight-compute"/>
### *Nsight-Compute*

NVIDIA® Nsight™ Compute is an interactive kernel profiler for CUDA applications. It provides detailed performance metrics and API debugging via a user interface and a command line tool. In addition, its baseline feature allows users to compare results within the tool. Nsight Compute provides a customizable and data-driven user interface and fast metric collection, which can be extended with rules-based analysis scripts for post-processing results. The rules-based guided analysis helps isolate and fix memory throughput, compute, and occupancy inefficiencies in the target application.

*homepage*: https://developer.nvidia.com/nsight-compute

version     |toolchain
------------|----------
``2020.3.0``|``system``
``2021.2.0``|``system``

<a anchor="nsight-systems"/>
### *Nsight-Systems*

NVIDIA® Nsight™ Systems is a system-wide performance analysis tool designed to visualize an application’s algorithm, help you select the largest opportunities to optimize, and tune to scale efficiently across any quantity of CPUs and GPUs in your computer; from laptops to DGX servers.

*homepage*: https://developer.nvidia.com/nsight-compute

version     |toolchain
------------|----------
``2020.5.1``|``system``

<a anchor="nspr"/>
### *NSPR*

Netscape Portable Runtime (NSPR) provides a platform-neutral API for system level  and libc-like functions.

*homepage*: https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSPR

version |toolchain
--------|------------------------------------
``4.20``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``4.21``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``4.25``|``GCCcore/9.3.0``
``4.29``|``GCCcore/10.2.0``
``4.30``|``GCCcore/10.3.0``
``4.32``|``GCCcore/11.2.0``
``4.34``|``GCCcore/11.3.0``
``4.35``|``GCCcore/12.2.0``

<a anchor="nss"/>
### *NSS*

Network Security Services (NSS) is a set of libraries designed to support cross-platform development  of security-enabled client and server applications.

*homepage*: https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS

version   |toolchain
----------|------------------------------------
``3.39``  |``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``3.42.1``|``GCCcore/8.2.0``
``3.45``  |``GCCcore/8.3.0``
``3.51``  |``GCCcore/9.3.0``
``3.57``  |``GCCcore/10.2.0``
``3.65``  |``GCCcore/10.3.0``
``3.69``  |``GCCcore/11.2.0``
``3.79``  |``GCCcore/11.3.0``
``3.85``  |``GCCcore/12.2.0``

<a anchor="nsync"/>
### *nsync*

nsync is a C library that exports various synchronization primitives, such as mutexes

*homepage*: https://github.com/google/nsync

version   |toolchain
----------|------------------------------------------------------------------------------------------------
``1.24.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.25.0``|``GCCcore/11.3.0``

<a anchor="ntcard"/>
### *ntCard*

ntCard is a streaming algorithm for estimating the frequencies of k-mers in genomics datasets.

*homepage*: https://www.bcgsc.ca/resources/software/ntcard

version  |toolchain
---------|-----------------------------
``1.2.1``|``GCC/11.2.0``, ``GCC/8.3.0``

<a anchor="ntedit"/>
### *ntEdit*

ntEdit is a fast and scalable genomics application for polishing genome assembly drafts.

*homepage*: https://github.com/bcgsc/ntEdit

version  |toolchain
---------|--------------------------------------
``1.3.1``|``iccifort/2018.3.222-GCC-7.3.0-2.30``

<a anchor="nthits"/>
### *ntHits*

ntHits is a method for identifying repeats in high-throughput DNA sequencing data.

*homepage*: https://github.com/bcgsc/ntHits

version  |toolchain
---------|--------------------------------------
``0.0.1``|``iccifort/2018.3.222-GCC-7.3.0-2.30``

<a anchor="ntl"/>
### *NTL*

NTL is a high-performance, portable C++ library providing data structures and algorithms for manipulating signed, arbitrary length integers, and for vectors, matrices, and polynomials over the integers and over finite fields.

*homepage*: https://shoup.net/ntl/

version   |toolchain
----------|--------------------
``11.3.4``|``GCC/8.2.0-2.31.1``

<a anchor="ntpoly"/>
### *NTPoly*

is a massively parallel library for computing the functions of sparse, symmetric matrices based on  polynomial expansions. For sufficiently sparse matrices, most of the matrix functions  in NTPoly can be computed in linear time.

*homepage*: https://github.com/william-dawson/NTPoly

version  |toolchain
---------|-------------------------------
``2.5.1``|``foss/2020b``, ``intel/2020b``
``2.7.0``|``foss/2021a``, ``intel/2021a``
``2.7.1``|``intel/2020b``

<a anchor="num2words"/>
### *num2words*

Modules to convert numbers to words. 42 --> forty-two

*homepage*: https://github.com/savoirfairelinux/num2words

version   |toolchain
----------|------------------
``0.5.10``|``GCCcore/10.3.0``

<a anchor="numactl"/>
### *numactl*

The numactl program allows you to run your application program on specific cpu's and memory nodes. It does this by supplying a NUMA memory policy to the operating system before running your program. The libnuma library provides convenient ways for you to add NUMA memory policies into your own program.

*homepage*: http://oss.sgi.com/projects/libnuma/

version   |toolchain
----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.0.9`` |``GCC/4.8.3``
``2.0.10``|``GCC/4.8.4``, ``GCC/4.9.2``, ``GNU/4.9.2-2.25``, ``GNU/4.9.3-2.25``
``2.0.11``|``GCC/4.9.3``, ``GCC/4.9.3-2.25``, ``GCC/5.2.0``, ``GCC/5.3.0-2.26``, ``GCC/5.4.0-2.26``, ``GCC/6.1.0-2.27``, ``GCC/6.2.0-2.27``, ``GCC/6.3.0-2.27``, ``GCCcore/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``foss/2016a``, ``iccifort/2016.3.210-GCC-4.9.3-2.25``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``, ``intel/2016a``, ``intel/2016b``
``2.0.12``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``2.0.13``|``GCCcore/10.2.0``, ``GCCcore/9.2.0``, ``GCCcore/9.3.0``
``2.0.14``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``2.0.16``|``GCCcore/12.2.0``

<a anchor="numba"/>
### *numba*

Numba is an Open Source NumPy-aware optimizing compiler for Python sponsored by Continuum Analytics, Inc. It uses the remarkable LLVM compiler infrastructure to compile Python syntax to machine code.

*homepage*: https://numba.pydata.org/

version   |versionsuffix     |toolchain
----------|------------------|---------------------------------------------------
``0.24.0``|``-Python-2.7.11``|``intel/2016a``
``0.24.0``|``-Python-3.5.1`` |``intel/2016a``
``0.26.0``|``-Python-2.7.11``|``intel/2016a``
``0.32.0``|``-Python-2.7.13``|``intel/2017a``
``0.37.0``|``-Python-2.7.14``|``foss/2018a``, ``intel/2017b``
``0.37.0``|``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``0.39.0``|``-Python-3.6.6`` |``foss/2018b``
``0.43.1``|                  |``intel/2019a``
``0.46.0``|                  |``foss/2019a``
``0.47.0``|``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``
``0.50.0``|``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``0.50.1``|``-Python-3.8.2`` |``foss/2020a``
``0.52.0``|                  |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``
``0.52.0``|``-Python-3.8.2`` |``foss/2020a``
``0.53.1``|                  |``foss/2020b``, ``foss/2021a``, ``fosscuda/2020b``
``0.54.1``|                  |``foss/2021b``, ``intel/2021b``
``0.54.1``|``-CUDA-11.4.1``  |``foss/2021b``

<a anchor="numdiff"/>
### *numdiff*

Numdiff (which I will also write numdiff) is a little program that can be used to compare putatively  similar files line by line and field by field, ignoring small numeric differences or/and different numeric formats.  Equivalently, Numdiff is a program with the capability to appropriately compare files containing numerical fields  (and not only).

*homepage*: https://www.nongnu.org/numdiff/

version  |toolchain
---------|------------------
``5.9.0``|``GCCcore/10.2.0``

<a anchor="numexpr"/>
### *numexpr*

The numexpr package evaluates multiple-operator array expressions many times faster than NumPy can.  It accepts the expression as a string, analyzes it, rewrites it more efficiently, and compiles it on the fly into  code for its internal virtual machine (VM). Due to its integrated just-in-time (JIT) compiler, it does not require a  compiler at runtime.

*homepage*: https://numexpr.readthedocs.io/en/latest/

version  |versionsuffix     |toolchain
---------|------------------|---------------------------------------------------
``2.5.2``|``-Python-2.7.11``|``intel/2016a``
``2.6.1``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``2.6.1``|``-Python-3.5.2`` |``intel/2016b``
``2.6.4``|``-Python-2.7.13``|``foss/2017a``
``2.6.4``|``-Python-2.7.14``|``intel/2018a``
``2.6.4``|``-Python-3.5.1`` |``foss/2016a``
``2.6.4``|``-Python-3.6.1`` |``intel/2017a``
``2.6.4``|``-Python-3.6.3`` |``intel/2017b``
``2.6.4``|``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``2.6.5``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``2.6.5``|``-Python-3.6.6`` |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``2.7.0``|                  |``intel/2019a``
``2.7.1``|``-Python-2.7.16``|``intel/2019b``
``2.7.1``|``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``
``2.7.1``|``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``2.8.1``|                  |``foss/2021a``, ``intel/2021a``

<a anchor="numpy"/>
### *numpy*

NumPy is the fundamental package for scientific computing with Python. It contains among other things:  a powerful N-dimensional array object, sophisticated (broadcasting) functions, tools for integrating C/C++ and Fortran  code, useful linear algebra, Fourier transform, and random number capabilities. Besides its obvious scientific uses,  NumPy can also be used as an efficient multi-dimensional container of generic data. Arbitrary data-types can be   defined. This allows NumPy to seamlessly and speedily integrate with a wide variety of databases.

*homepage*: http://www.numpy.org

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``1.8.2`` |``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``1.9.2`` |``-Python-2.7.12``|``intel/2016b``
``1.10.4``|``-Python-2.7.11``|``intel/2016a``
``1.11.0``|``-Python-2.7.11``|``intel/2016a``
``1.12.1``|``-Python-3.5.2`` |``intel/2016b``
``1.13.0``|``-Python-2.7.13``|``intel/2017a``
``1.13.1``|``-Python-3.6.1`` |``intel/2017a``

<a anchor="nvhpc"/>
### *NVHPC*

C, C++ and Fortran compilers included with the NVIDIA HPC SDK (previously: PGI)

*homepage*: https://developer.nvidia.com/hpc-sdk/

version  |toolchain
---------|----------
``20.7`` |``system``
``20.9`` |``system``
``20.11``|``system``
``21.2`` |``system``
``21.3`` |``system``
``21.5`` |``system``
``21.7`` |``system``
``21.9`` |``system``
``21.11``|``system``

<a anchor="nvshmem"/>
### *NVSHMEM*

NVSHMEM is a parallel programming interface based on OpenSHMEM that provides  efficient and scalable communication for NVIDIA GPU clusters. NVSHMEM creates a  global address space for data that spans the memory of multiple GPUs and can be  accessed with fine-grained GPU-initiated operations, CPU-initiated operations,  and operations on CUDA streams. 

*homepage*: https://developer.nvidia.com/nvshmem

version  |versionsuffix   |toolchain
---------|----------------|---------------
``2.4.1``|``-CUDA-11.4.1``|``gompi/2021b``
``2.5.0``|``-CUDA-11.7.0``|``gompi/2022a``
``2.7.0``|``-CUDA-11.7.0``|``gompi/2022a``

<a anchor="nvtop"/>
### *nvtop*

htop-like GPU usage monitor

*homepage*: https://github.com/Syllo/nvtop

version  |toolchain
---------|--------------------------------------
``1.0.0``|``fosscuda/2018b``
``1.1.0``|``fosscuda/2019b``
``1.2.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``1.2.2``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``2.0.2``|``GCCcore/11.3.0``

<a anchor="nwchem"/>
### *NWChem*

NWChem aims to provide its users with computational chemistry tools that are scalable both in  their ability to treat large scientific computational chemistry problems efficiently, and in their use of available  parallel computing resources from high-performance parallel supercomputers to conventional workstation clusters.  NWChem software can handle: biomolecules, nanostructures, and solid-state; from quantum to classical, and all  combinations; Gaussian basis functions or plane-waves; scaling from one to thousands of processors; properties  and relativity.

*homepage*: https://www.nwchem-sw.org

version              |versionsuffix                                 |toolchain
---------------------|----------------------------------------------|--------------------------------
``6.6.revision27746``|``-2015-10-20-Python-2.7.12``                 |``iomkl/2017a``
``6.6.revision27746``|``-2015-10-20-patches-20170814-Python-2.7.13``|``intel/2017a``
``6.8.revision47``   |``-2017-12-14-Python-2.7.14``                 |``intel/2017b``, ``intel/2018a``
``7.0.0``            |``-Python-3.7.4``                             |``intel/2019b``
``7.0.2``            |                                              |``intel/2021a``, ``intel/2022a``
``7.0.2``            |``-Python-3.7.4``                             |``intel/2019b``

<a anchor="nxtrim"/>
### *NxTrim*

NxTrim is a software to remove Nextera Mate Pair junction adapters and categorise reads according to the orientation implied by the adapter location.

*homepage*: https://github.com/sequencing/NxTrim

version  |toolchain
---------|--------------
``0.4.3``|``foss/2018a``

<a anchor="o"/>
### *O*


<a href="#oases">Oases</a> - <a href="#obitools">OBITools</a> - <a href="#obitools3">OBITools3</a> - <a href="#ocaml">OCaml</a> - <a href="#occt">occt</a> - <a href="#oceanspy">oceanspy</a> - <a href="#ocnet">OCNet</a> - <a href="#octave">Octave</a> - <a href="#octopus-vcf">Octopus-vcf</a> - <a href="#ogdf">OGDF</a> - <a href="#olaflow">olaFlow</a> - <a href="#olego">olego</a> - <a href="#oma">OMA</a> - <a href="#omegafold">OmegaFold</a> - <a href="#onedrive">onedrive</a> - <a href="#onnx">ONNX</a> - <a href="#onnx-runtime">ONNX-Runtime</a> - <a href="#ont-fast5-api">ont-fast5-api</a> - <a href="#ont-remora">ont-remora</a> - <a href="#oompa">OOMPA</a> - <a href="#opari2">OPARI2</a> - <a href="#open-data-cube-core">Open-Data-Cube-Core</a> - <a href="#openai-gym">OpenAI-Gym</a> - <a href="#openbabel">OpenBabel</a> - <a href="#openblas">OpenBLAS</a> - <a href="#opencarp">openCARP</a> - <a href="#opencensus-python">OpenCensus-python</a> - <a href="#opencoarrays">OpenCoarrays</a> - <a href="#opencolorio">OpenColorIO</a> - <a href="#opencv">OpenCV</a> - <a href="#openexr">OpenEXR</a> - <a href="#openface">OpenFace</a> - <a href="#openfast">OpenFAST</a> - <a href="#openfoam">OpenFOAM</a> - <a href="#openfoam-extend">OpenFOAM-Extend</a> - <a href="#openfold">OpenFold</a> - <a href="#openforcefield">OpenForceField</a> - <a href="#openimageio">OpenImageIO</a> - <a href="#openjpeg">OpenJPEG</a> - <a href="#openkim-api">OpenKIM-API</a> - <a href="#openkim-models">openkim-models</a> - <a href="#openmm">OpenMM</a> - <a href="#openmm-plumed">OpenMM-PLUMED</a> - <a href="#openmmtools">OpenMMTools</a> - <a href="#openmolcas">OpenMolcas</a> - <a href="#openmpi">OpenMPI</a> - <a href="#openms">OpenMS</a> - <a href="#opennlp">OpenNLP</a> - <a href="#openpgm">OpenPGM</a> - <a href="#openpiv">OpenPIV</a> - <a href="#openpyxl">openpyxl</a> - <a href="#openrefine">OpenRefine</a> - <a href="#openscenegraph">OpenSceneGraph</a> - <a href="#opensees">OpenSees</a> - <a href="#openslide">OpenSlide</a> - <a href="#openslide-python">openslide-python</a> - <a href="#openssl">OpenSSL</a> - <a href="#openstackclient">OpenStackClient</a> - <a href="#opera">OPERA</a> - <a href="#opera-ms">OPERA-MS</a> - <a href="#optitype">OptiType</a> - <a href="#optix">OptiX</a> - <a href="#optuna">Optuna</a> - <a href="#or-tools">OR-Tools</a> - <a href="#orca">ORCA</a> - <a href="#orffinder">ORFfinder</a> - <a href="#orfm">OrfM</a> - <a href="#orthofinder">OrthoFinder</a> - <a href="#orthomcl">OrthoMCL</a> - <a href="#osi">Osi</a> - <a href="#ospray">OSPRay</a> - <a href="#osu-micro-benchmarks">OSU-Micro-Benchmarks</a> - <a href="#otf2">OTF2</a> - <a href="#ovito">OVITO</a> - <a href="#owncloud">ownCloud</a> - <a href="#oxford_asl">oxford_asl</a>


<a anchor="oases"/>
### *Oases*

Oases is a de novo transcriptome assembler designed to produce transcripts from  short read sequencing technologies, such as Illumina, SOLiD, or 454 in the absence of any genomic assembly.

*homepage*: http://www.ebi.ac.uk/~zerbino/oases/

version   |versionsuffix|toolchain
----------|-------------|---------------
``0.2.08``|             |``foss/2016b``
``0.2.08``|``-kmer_101``|``intel/2017b``

<a anchor="obitools"/>
### *OBITools*

The OBITools programs aims to help you to manipulate various data and sequence files in a  convenient way using the Unix command line interface. They follow the standard Unix interface for command line  program, allowing to chain a set of commands using the pipe mecanism.

*homepage*: http://www.grenoble.prabi.fr/trac/OBITools

version   |versionsuffix     |toolchain
----------|------------------|--------------
``1.2.9`` |``-Python-2.7.11``|``foss/2016a``
``1.2.13``|``-Python-2.7.15``|``foss/2019a``

<a anchor="obitools3"/>
### *OBITools3*

A package for the management of analyses and data in DNA metabarcoding.

*homepage*: https://metabarcoding.org/obitools3

version    |toolchain
-----------|------------------
``3.0.1b8``|``GCCcore/10.2.0``

<a anchor="ocaml"/>
### *OCaml*

OCaml is a general purpose industrial-strength programming language  with an emphasis on expressiveness and safety. Developed for more than 20 years at Inria  it benefits from one of the most advanced type systems and supports functional,  imperative and object-oriented styles of programming.

*homepage*: http://ocaml.org/

version   |toolchain
----------|--------------
``4.02.3``|``foss/2016a``
``4.07.1``|``foss/2018b``

<a anchor="occt"/>
### *occt*

Open CASCADE Technology (OCCT) is an object-oriented C++ class library designed for rapid production of sophisticated domain-specific CAD/CAM/CAE applications.

*homepage*: https://www.opencascade.com/

version    |toolchain
-----------|--------------
``7.3.0p4``|``foss/2019b``
``7.5.0p1``|``foss/2021a``

<a anchor="oceanspy"/>
### *oceanspy*

OceanSpy - A Python package to facilitate ocean model data analysis  and visualization.

*homepage*: https://github.com/hainegroup/oceanspy

version  |toolchain
---------|--------------
``0.2.0``|``foss/2022a``

<a anchor="ocnet"/>
### *OCNet*

Generate and analyze Optimal Channel Networks (OCNs): oriented spanning trees reproducing all scaling features characteristic of real, natural river networks. As such, they can be used in a variety of numerical experiments in the fields of hydrology, ecology and epidemiology.

*homepage*: https://cran.r-project.org/web/packages/OCNet

version  |versionsuffix|toolchain
---------|-------------|--------------
``0.2.0``|``-R-3.6.0`` |``foss/2019a``

<a anchor="octave"/>
### *Octave*

GNU Octave is a high-level interpreted language, primarily intended for numerical computations.

*homepage*: https://www.gnu.org/software/octave/

version  |versionsuffix|toolchain
---------|-------------|------------------------------------------------
``4.0.0``|             |``foss/2016a``
``4.0.3``|             |``intel/2016b``
``4.2.1``|             |``foss/2018a``, ``intel/2016b``, ``intel/2017a``
``4.2.1``|``-mt``      |``intel/2017a``
``4.2.2``|             |``foss/2018a``
``4.4.1``|             |``foss/2018b``
``5.1.0``|             |``foss/2019a``, ``foss/2019b``
``6.2.0``|             |``foss/2020b``
``7.1.0``|             |``foss/2021b``

<a anchor="octopus-vcf"/>
### *Octopus-vcf*

Octopus is a mapping-based variant caller that implements several calling models within  a unified haplotype-aware framework. Octopus takes inspiration from particle filtering by constructing a  tree of haplotypes and dynamically pruning and extending the tree based on haplotype posterior probabilities  in a sequential manner. This allows octopus to implicitly consider all possible haplotypes at a given loci in  reasonable time.

*homepage*: https://github.com/luntergroup/octopus

version  |toolchain
---------|--------------
``0.7.1``|``foss/2020b``
``0.7.2``|``foss/2020b``

<a anchor="ogdf"/>
### *OGDF*

OGDF is a self-contained C++ library for graph algorithms, in particular for (but not restricted to) automatic graph drawing. It offers sophisticated algorithms and data structures to use within your own applications or scientific projects.

*homepage*: https://ogdf.uos.de/

version           |toolchain
------------------|--------------
``dogwood-202202``|``GCC/11.2.0``

<a anchor="olaflow"/>
### *olaFlow*

olaFlow CFD Suite is a free and open source project committed to bringing the latest advances for the simulation of wave dynamics to the OpenFOAM® and FOAM-extend communities.

*homepage*: https://olaflow.github.io/

version     |toolchain
------------|--------------
``20210820``|``foss/2021b``

<a anchor="olego"/>
### *olego*

OLego is a program specifically designed for de novo spliced mapping of mRNA-seq reads. OLego adopts a seed-and-extend scheme, and does not rely on a separate external mapper.

*homepage*: https://github.com/chaolinzhanglab/olego

version  |toolchain
---------|--------------
``1.1.9``|``GCC/11.2.0``

<a anchor="oma"/>
### *OMA*

The OMA ('Orthologous MAtrix') project is a method and database for the inference   of orthologs among complete genomes

*homepage*: https://omabrowser.org/standalone/

version  |toolchain
---------|----------
``2.1.1``|``system``

<a anchor="omegafold"/>
### *OmegaFold*

OmegaFold: High-resolution de novo Structure Prediction from Primary Sequence

*homepage*: https://github.com/HeliXonProtein/OmegaFold

version  |versionsuffix   |toolchain
---------|----------------|--------------
``1.1.0``|``-CUDA-11.7.0``|``foss/2022a``

<a anchor="onedrive"/>
### *onedrive*

A free Microsoft OneDrive Client which supports OneDrive Personal, OneDrive for Business, OneDrive for Office365 and SharePoint.

*homepage*: https://abraunegg.github.io/

version   |toolchain
----------|------------------
``2.4.11``|``GCCcore/10.2.0``
``2.4.21``|``GCCcore/11.3.0``

<a anchor="onnx"/>
### *ONNX*

Open Neural Network Exchange (ONNX) is an open ecosystem that empowers AI developers to choose the right tools as their project evolves. ONNX provides an open source format for AI models, both deep learning and traditional ML. It defines an extensible computation graph model, as well as definitions of built-in operators and standard data types. Currently we focus on the capabilities needed for inferencing (scoring).

*homepage*: https://onnx.ai/

version   |toolchain
----------|--------------
``1.11.0``|``foss/2021a``

<a anchor="onnx-runtime"/>
### *ONNX-Runtime*

ONNX Runtime inference can enable faster customer experiences and lower costs, supporting models from deep learning frameworks such as PyTorch and TensorFlow/Keras as well as classical machine learning libraries such as scikit-learn, LightGBM, XGBoost, etc. ONNX Runtime is compatible with different hardware, drivers, and operating systems, and provides optimal performance by leveraging hardware accelerators where applicable alongside graph optimizations and transforms.

*homepage*: https://onnxruntime.ai/

version   |versionsuffix   |toolchain
----------|----------------|--------------
``1.10.0``|                |``foss/2021a``
``1.10.0``|``-CUDA-11.3.1``|``foss/2021a``

<a anchor="ont-fast5-api"/>
### *ont-fast5-api*

ont_fast5_api is a simple interface to HDF5 files of the Oxford Nanopore .fast5 file format.

*homepage*: https://github.com/nanoporetech/ont_fast5_api

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------
``3.3.0``|                 |``foss/2020b``, ``fosscuda/2020b``
``3.3.0``|``-Python-3.7.4``|``foss/2019b``
``4.0.0``|                 |``foss/2021a``
``4.0.2``|                 |``foss/2021a``, ``foss/2021b``

<a anchor="ont-remora"/>
### *ont-remora*

Methylation/modified base calling separated from basecalling. Remora primarily provides an API to call modified bases for basecaller programs such as Bonito. Remora also provides the tools to prepare datasets, train modified base models and run simple inference.

*homepage*: https://github.com/nanoporetech/remora

version  |versionsuffix   |toolchain
---------|----------------|--------------
``0.1.2``|                |``foss/2021a``
``0.1.2``|``-CUDA-11.3.1``|``foss/2021a``
``1.0.0``|                |``foss/2021a``
``1.0.0``|``-CUDA-11.3.1``|``foss/2021a``

<a anchor="oompa"/>
### *OOMPA*

OOMPA is a suite of R packages for the analysis of gene expression (RNA), proteomics profiling,  and other high throughput molecular biology data.

*homepage*: https://r-forge.r-project.org/projects/oompa

version  |versionsuffix|toolchain
---------|-------------|---------------
``3.1.2``|``-R-3.3.1`` |``intel/2016b``

<a anchor="opari2"/>
### *OPARI2*

OPARI2, the successor of Forschungszentrum Juelich's OPARI, is a  source-to-source instrumentation tool for OpenMP and hybrid codes.  It surrounds OpenMP directives and runtime library calls with calls  to the POMP2 measurement interface. 

*homepage*: https://www.score-p.org

version  |toolchain
---------|-------------------------------------------------------
``2.0``  |``foss/2016a``
``2.0.5``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``2.0.6``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``

<a anchor="open-data-cube-core"/>
### *Open-Data-Cube-Core*

The Open Data Cube Core provides an integrated gridded data analysis environment for decades of analysis ready earth observation satellite and related data from multiple satellite and other acquisition systems.

*homepage*: https://github.com/opendatacube/datacube-core

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.8.3``|``-Python-3.8.2``|``foss/2020a``

<a anchor="openai-gym"/>
### *OpenAI-Gym*

A toolkit for developing and comparing reinforcement learning algorithms.

*homepage*: https://gym.openai.com

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.17.1``|``-Python-3.7.4``|``foss/2019b``
``0.21.0``|                 |``foss/2021b``

<a anchor="openbabel"/>
### *OpenBabel*

Open Babel is a chemical toolbox designed to speak the many  languages of chemical data. It's an open, collaborative project allowing anyone  to search, convert, analyze, or store data from molecular modeling, chemistry,  solid-state materials, biochemistry, or related areas.

*homepage*: https://openbabel.org

version  |versionsuffix     |toolchain
---------|------------------|--------------------------------
``2.3.2``|``-Python-2.7.11``|``foss/2016a``
``2.4.1``|``-Python-2.7.14``|``intel/2017b``
``2.4.1``|``-Python-2.7.15``|``intel/2018b``
``2.4.1``|``-Python-3.6.6`` |``intel/2018b``
``2.4.1``|``-Python-3.7.2`` |``intel/2019a``
``3.0.0``|``-Python-3.7.4`` |``gompi/2019b``
``3.1.1``|                  |``gompi/2021a``, ``gompi/2022a``
``3.1.1``|``-Python-3.7.4`` |``gompi/2019b``
``3.1.1``|``-Python-3.8.2`` |``iimpi/2020a``

<a anchor="openblas"/>
### *OpenBLAS*

OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD version.

*homepage*: https://xianyi.github.com/OpenBLAS/

version   |versionsuffix    |toolchain
----------|-----------------|----------------------------------------------------------
``0.2.9`` |``-LAPACK-3.5.0``|``GCC/4.8.3``
``0.2.12``|``-LAPACK-3.5.0``|``GCC/4.9.2``
``0.2.13``|``-LAPACK-3.5.0``|``GCC/4.8.4``, ``GCC/4.9.2``
``0.2.14``|``-LAPACK-3.5.0``|``GNU/4.9.2-2.25``, ``GNU/4.9.3-2.25``
``0.2.15``|``-LAPACK-3.6.0``|``GCC/4.9.3-2.25``
``0.2.18``|``-LAPACK-3.6.0``|``GCC/4.9.4-2.25``, ``GCC/5.3.0-2.26``, ``GCC/5.4.0-2.26``
``0.2.18``|``-LAPACK-3.6.1``|``GCC/5.4.0-2.26``, ``gompi/2016.07``
``0.2.19``|``-LAPACK-3.6.1``|``gompi/2016.09``
``0.2.19``|``-LAPACK-3.7.0``|``GCC/5.4.0-2.26``, ``GCC/6.3.0-2.27``
``0.2.20``|                 |``GCC/5.4.0-2.26``, ``GCC/6.4.0-2.28``, ``GCC/7.2.0-2.29``
``0.3.0`` |                 |``GCC/6.4.0-2.28``, ``GCC/7.3.0-2.30``
``0.3.1`` |                 |``GCC/7.3.0-2.30``
``0.3.3`` |                 |``GCC/8.2.0-2.31.1``
``0.3.4`` |                 |``GCC/8.2.0-2.31.1``
``0.3.5`` |                 |``GCC/8.2.0-2.31.1``
``0.3.6`` |                 |``GCC/8.3.0-2.32``
``0.3.7`` |                 |``GCC/8.3.0``
``0.3.8`` |                 |``GCC/9.2.0``
``0.3.9`` |                 |``GCC/9.3.0``
``0.3.12``|                 |``GCC/10.2.0``
``0.3.15``|                 |``GCC/10.3.0``
``0.3.17``|                 |``GCC/10.3.0``, ``GCC/11.2.0``
``0.3.18``|                 |``GCC/11.2.0``
``0.3.20``|                 |``GCC/11.2.0``, ``GCC/11.3.0``
``0.3.21``|                 |``GCC/12.2.0``

<a anchor="opencarp"/>
### *openCARP*

openCARP is an open cardiac electrophysiology simulator for in-silico experiments.

*homepage*: https://opencarp.org

version|versionsuffix    |toolchain
-------|-----------------|--------------
``3.2``|``-Python-3.8.2``|``foss/2020a``
``6.0``|                 |``foss/2020b``
``8.2``|                 |``foss/2021a``

<a anchor="opencensus-python"/>
### *OpenCensus-python*

OpenCensus for Python. OpenCensus provides a framework to measure a server's resource usage and collect performance stats. This repository contains Python related utilities and supporting software needed by OpenCensus.

*homepage*: https://opencensus.io/

version  |toolchain
---------|------------------
``0.8.0``|``GCCcore/11.2.0``

<a anchor="opencoarrays"/>
### *OpenCoarrays*

OpenCoarrays is an open-source software project that supports the coarray Fortran (CAF) parallel programming features of the Fortran 2008 standard and several features proposed for Fortran 2015 in the draft Technical Specification TS 18508 Additional Parallel Features in Fortran.

*homepage*: https://github.com/sourceryinstitute/opencoarrays

version  |toolchain
---------|---------------
``1.9.0``|``gompi/2017a``
``2.2.0``|``gompi/2018b``
``2.8.0``|``gompi/2019b``
``2.9.2``|``gompi/2020a``

<a anchor="opencolorio"/>
### *OpenColorIO*

OpenColorIO (OCIO) is a complete color management solution geared towards motion picture production  with an emphasis on visual effects and computer animation.

*homepage*: http://opencolorio.org/

version  |toolchain
---------|--------------
``1.1.0``|``foss/2018b``

<a anchor="opencv"/>
### *OpenCV*

OpenCV (Open Source Computer Vision Library) is an open source computer vision  and machine learning software library. OpenCV was built to provide  a common infrastructure for computer vision applications and to accelerate  the use of machine perception in the commercial products.  Includes extra modules for OpenCV from the contrib repository.

*homepage*: https://opencv.org/

version   |versionsuffix            |toolchain
----------|-------------------------|----------------------------------------------------------------
``2.4.12``|                         |``intel/2016a``
``3.1.0`` |                         |``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``3.3.0`` |``-Python-2.7.13``       |``intel/2017a``
``3.3.0`` |``-Python-2.7.14``       |``foss/2017b``, ``intel/2017b``
``3.3.0`` |``-Python-3.6.1``        |``intel/2017a``
``3.3.0`` |``-Python-3.6.3``        |``foss/2017b``, ``intel/2017b``
``3.4.1`` |``-Python-2.7.14``       |``foss/2018a``, ``intel/2018a``
``3.4.1`` |``-Python-3.6.4``        |``foss/2018a``, ``intel/2018a``
``3.4.1`` |``-Python-3.6.4-contrib``|``foss/2018a``
``3.4.5`` |``-Python-2.7.15``       |``foss/2018b``
``3.4.7`` |``-Python-2.7.15``       |``foss/2019a``, ``fosscuda/2019a``
``3.4.7`` |``-Python-3.7.2``        |``foss/2019a``, ``fosscuda/2019a``
``4.0.1`` |``-Python-2.7.15``       |``foss/2018b``
``4.0.1`` |``-Python-3.6.6``        |``foss/2018b``
``4.2.0`` |``-Python-3.7.4``        |``foss/2019b``, ``fosscuda/2019b``
``4.2.0`` |``-Python-3.8.2``        |``foss/2020a``
``4.2.0`` |``-Python-3.8.2-contrib``|``foss/2020a``
``4.5.1`` |``-contrib``             |``foss/2020b``, ``fosscuda/2020b``
``4.5.3`` |``-CUDA-11.3.1-contrib`` |``foss/2021a``
``4.5.3`` |``-contrib``             |``foss/2021a``
``4.5.5`` |``-CUDA-11.4.1-contrib`` |``foss/2021b``
``4.5.5`` |``-contrib``             |``foss/2021b``
``4.6.0`` |``-CUDA-11.7.0-contrib`` |``foss/2022a``
``4.6.0`` |``-contrib``             |``foss/2022a``

<a anchor="openexr"/>
### *OpenEXR*

OpenEXR is a high dynamic-range (HDR) image file format developed by Industrial Light & Magic  for use in computer imaging applications

*homepage*: https://www.openexr.com/

version  |toolchain
---------|--------------------------------
``2.2.0``|``intel/2016b``, ``intel/2017a``
``2.3.0``|``foss/2018b``, ``intel/2018b``
``2.4.0``|``GCCcore/8.3.0``
``2.4.1``|``GCCcore/9.3.0``
``2.5.5``|``GCCcore/10.2.0``
``3.0.1``|``GCCcore/10.3.0``
``3.1.1``|``GCCcore/11.2.0``
``3.1.5``|``GCCcore/11.3.0``

<a anchor="openface"/>
### *OpenFace*

OpenFace – a state-of-the art tool intended for facial landmark detection, head pose estimation, facial action unit recognition, and eye-gaze estimation.

*homepage*: https://github.com/TadasBaltrusaitis/OpenFace

version  |versionsuffix   |toolchain
---------|----------------|--------------
``2.2.0``|                |``foss/2021a``
``2.2.0``|``-CUDA-11.3.1``|``foss/2021a``

<a anchor="openfast"/>
### *OpenFAST*

OpenFAST is a wind turbine simulation tool which builds on FAST v8. FAST.Farm extends the capability of OpenFAST to simulate multi-turbine wind farms

*homepage*: http://openfast.readthedocs.io/

version  |toolchain
---------|--------------
``3.0.0``|``foss/2021b``

<a anchor="openfoam"/>
### *OpenFOAM*

OpenFOAM is a free, open source CFD software package.  OpenFOAM has an extensive range of features to solve anything from complex fluid flows  involving chemical reactions, turbulence and heat transfer,  to solid dynamics and electromagnetics.

*homepage*: https://www.openfoam.com/

version         |versionsuffix|toolchain
----------------|-------------|-------------------------------------------------
``2.2.2``       |             |``intel/2016a``, ``intel/2017a``, ``intel/2018a``
``2.2.x``       |             |``intel/2019a``
``2.3.1``       |             |``intel/2017a``, ``intel/2019b``
``2.4.0``       |             |``intel/2017a``, ``intel/2019a``
``3.0.0``       |             |``foss/2016a``
``3.0.1``       |             |``intel/2016b``, ``intel/2018a``
``4.0``         |             |``foss/2016b``, ``intel/2016b``
``4.1``         |             |``foss/2016b``, ``intel/2017a``
``4.x-20170904``|             |``intel/2016b``
``5.0``         |             |``foss/2017b``, ``intel/2017a``, ``intel/2017b``
``5.0-20180108``|             |``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``6``           |             |``foss/2018b``, ``foss/2019b``, ``intel/2018a``
``7``           |             |``foss/2019b``
``7``           |``-20200508``|``foss/2019b``
``8``           |             |``foss/2020a``, ``foss/2020b``
``8``           |``-20210316``|``foss/2020b``
``9``           |             |``foss/2021a``, ``intel/2021a``
``10``          |             |``foss/2022a``
``v1606+``      |             |``foss/2018b``
``v1612+``      |             |``foss/2018b``
``v1712``       |             |``foss/2017b``, ``intel/2017b``
``v1806``       |             |``foss/2018b``
``v1812``       |             |``foss/2018b``
``v1906``       |             |``foss/2019b``
``v1912``       |             |``foss/2019b``, ``intel/2019b``
``v2006``       |             |``foss/2020a``, ``intel/2020a``
``v2012``       |             |``foss/2020a``
``v2106``       |             |``foss/2021a``
``v2112``       |             |``foss/2021b``
``v2206``       |             |``foss/2022a``

<a anchor="openfoam-extend"/>
### *OpenFOAM-Extend*

OpenFOAM is a free, open source CFD software package. OpenFOAM has an extensive range of features to solve anything from complex fluid flows involving chemical reactions, turbulence and heat transfer, to solid dynamics and electromagnetics.

*homepage*: http://www.extend-project.de/

version         |versionsuffix     |toolchain
----------------|------------------|---------------------------------
``3.1``         |                  |``gimkl/2.11.5``
``3.2``         |                  |``gimkl/2.11.5``, ``intel/2016a``
``4.0``         |                  |``intel/2017a``
``4.0``         |``-Python-2.7.16``|``intel/2019b``
``4.1-20191120``|``-Python-2.7.16``|``intel/2019b``
``4.1-20200408``|``-Python-2.7.16``|``foss/2019b``

<a anchor="openfold"/>
### *OpenFold*

A faithful PyTorch reproduction of DeepMind's AlphaFold 2

*homepage*: https://github.com/aqlaboratory/openfold

version  |versionsuffix   |toolchain
---------|----------------|--------------
``1.0.0``|``-CUDA-11.3.1``|``foss/2021a``
``1.0.1``|``-CUDA-11.3.1``|``foss/2021a``

<a anchor="openforcefield"/>
### *OpenForceField*

Simulation and Parameter Estimation in Geophysics  - A python package for simulation and gradient based parameter estimation in the context of geophysical applications.

*homepage*: https://github.com/openforcefield/openforcefield/releases

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.7.0``|``-Python-3.8.2``|``intel/2020a``

<a anchor="openimageio"/>
### *OpenImageIO*

OpenImageIO is a library for reading and writing images, and a bunch of related classes, utilities,  and applications.

*homepage*: https://openimageio.org/

version     |toolchain
------------|--------------------------------
``1.6.17``  |``intel/2016b``
``1.7.17``  |``intel/2017a``
``1.8.16``  |``foss/2018b``, ``intel/2018b``
``2.0.12``  |``gompi/2019b``, ``iimpi/2019b``
``2.1.12.0``|``gompi/2020a``, ``iimpi/2020a``
``2.3.17.0``|``GCC/11.3.0``

<a anchor="openjpeg"/>
### *OpenJPEG*

OpenJPEG is an open-source JPEG 2000 codec written in  C language. It has been developed in order to promote the use of JPEG 2000,  a still-image compression standard from the Joint Photographic Experts Group  (JPEG). Since may 2015, it is officially recognized by ISO/IEC and ITU-T as  a JPEG 2000 Reference Software.

*homepage*: https://www.openjpeg.org/

version  |toolchain
---------|-----------------------------------------------------------------------------
``2.1``  |``GCCcore/6.4.0``
``2.3.0``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``2.3.1``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``2.4.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``
``2.5.0``|``GCCcore/11.3.0``

<a anchor="openkim-api"/>
### *OpenKIM-API*

Open Knowledgebase of Interatomic Models.  OpenKIM is an API and a collection of interatomic models (potentials) for atomistic simulations.  It is a library that can be used by simulation programs to get access to the models in the OpenKIM database.  This EasyBuild only installs the API, the models have to be installed by the user by running     kim-api-collections-management install user MODELNAME or     kim-api-collections-management install user OpenKIM to install them all.  

*homepage*: https://openkim.org/

version  |toolchain
---------|------------------------------------------------
``1.9.2``|``foss/2016b``, ``foss/2017b``
``1.9.7``|``foss/2018b``, ``intel/2018b``, ``iomkl/2018b``

<a anchor="openkim-models"/>
### *openkim-models*

Open Knowledgebase of Interatomic Models.  OpenKIM is an API and a collection of interatomic models (potentials) for atomistic simulations.  It is a library that can be used by simulation programs to get access to the models in the OpenKIM database.  This EasyBuild installs the models.  The API itself is in the kim-api package.  

*homepage*: https://openkim.org/

version     |toolchain
------------|------------------------------------------------------------------------------------
``20190725``|``GCCcore/10.2.0``, ``foss/2019a``, ``foss/2019b``, ``intel/2019a``, ``intel/2019b``
``20210128``|``GCCcore/10.2.0``

<a anchor="openmm"/>
### *OpenMM*

OpenMM is a toolkit for molecular simulation.

*homepage*: https://openmm.org

version  |versionsuffix      |toolchain
---------|-------------------|---------------------------------------------------
``7.1.1``|``-Python-3.6.3``  |``intel/2017b``
``7.4.1``|``-Python-3.7.4``  |``foss/2019b``, ``intel/2019b``
``7.4.2``|``-Python-3.8.2``  |``intel/2020a``
``7.5.0``|                   |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``
``7.5.0``|``-Python-3.8.2``  |``fosscuda/2020a``, ``intel/2020a``
``7.5.1``|                   |``foss/2020b``, ``fosscuda/2020b``
``7.5.1``|``-DeepMind-patch``|``foss/2021a``
``7.7.0``|                   |``foss/2021a``, ``foss/2022a``
``7.7.0``|``-CUDA-11.3.1``   |``foss/2021a``

<a anchor="openmm-plumed"/>
### *OpenMM-PLUMED*

This project provides a connection between OpenMM and PLUMED. It allows you to bias or analyze an OpenMM simulation based on collective variables.

*homepage*: https://github.com/openmm/openmm-plumed/

version|versionsuffix    |toolchain
-------|-----------------|---------------
``1.0``|``-Python-3.8.2``|``intel/2020a``

<a anchor="openmmtools"/>
### *OpenMMTools*

A batteries-included toolkit for the GPU-accelerated OpenMM molecular simulation engine. openmmtools is a Python library layer that sits on top of OpenMM to provide access to a variety of useful tools  for building full-featured molecular simulation packages. 

*homepage*: https://github.com/choderalab/openmmtools

version   |versionsuffix    |toolchain
----------|-----------------|---------------
``0.20.0``|``-Python-3.8.2``|``intel/2020a``

<a anchor="openmolcas"/>
### *OpenMolcas*

OpenMolcas is a quantum chemistry software package.

*homepage*: https://gitlab.com/Molcas/OpenMolcas

version  |versionsuffix         |toolchain
---------|----------------------|---------------
``18.09``|``-Python-3.6.6``     |``intel/2018b``
``20.10``|``-Python-3.8.2``     |``intel/2020a``
``20.10``|``-Python-3.8.2-noGA``|``intel/2020a``
``21.06``|                      |``intel/2021a``
``22.06``|                      |``intel/2022a``
``22.10``|                      |``intel/2022a``
``22.10``|``-noGA``             |``intel/2022a``

<a anchor="openmpi"/>
### *OpenMPI*

The Open MPI Project is an open source MPI implementation.

*homepage*: http://www.open-mpi.org/

version   |versionsuffix   |toolchain
----------|----------------|--------------------------------------------------------------------------------------------------------------------
``1.6.5`` |                |``GCC/4.8.1``, ``GCC/4.8.2``, ``GCC/4.8.3``
``1.6.5`` |``-no-OFED``    |``GCC/4.8.1``, ``GCC/4.8.2``
``1.7.3`` |                |``GCC/4.8.2``
``1.8.1`` |                |``GCC/4.8.3``
``1.8.3`` |                |``GCC/4.9.2``
``1.8.4`` |                |``GCC/4.8.4``, ``GCC/4.9.2``
``1.8.5`` |                |``GNU/4.9.2-2.25``
``1.8.6`` |                |``GNU/4.9.3-2.25``
``1.8.8`` |                |``GNU/4.9.3-2.25``
``1.10.1``|                |``GCC/4.9.3-2.25``
``1.10.2``|                |``GCC/4.9.3-2.25``, ``GCC/5.3.0-2.26``, ``GCC/6.1.0-2.27``, ``PGI/16.3-GCC-4.9.3-2.25``, ``PGI/16.4-GCC-5.3.0-2.26``
``1.10.3``|                |``GCC/5.4.0-2.26``, ``GCC/6.1.0-2.27``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``
``1.10.4``|                |``PGI/16.7-GCC-5.4.0-2.26``, ``iccifort/2016.3.210-GCC-4.9.3-2.25``
``2.0.0`` |                |``GCC/5.2.0``
``2.0.1`` |                |``GCC/6.2.0-2.27``, ``iccifort/2017.1.132-GCC-5.4.0-2.26``
``2.0.2`` |                |``GCC/6.3.0-2.27``, ``iccifort/2017.1.132-GCC-6.3.0-2.27``
``2.0.2`` |``-opa``        |``GCC/6.3.0-2.27``
``2.1.0`` |                |``GCC/6.3.0-2.28``
``2.1.1`` |                |``GCC/6.4.0-2.28``, ``gcccuda/2017b``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``
``2.1.2`` |                |``GCC/6.4.0-2.28``, ``gcccuda/2018a``, ``iccifort/2018.1.163-GCC-6.4.0-2.28``
``2.1.3`` |                |``iccifort/2018.2.199-GCC-6.4.0-2.28``
``3.0.0`` |                |``GCC/7.2.0-2.29``
``3.1.0`` |                |``GCC/7.3.0-2.30``
``3.1.1`` |                |``GCC/7.3.0-2.30``, ``gcccuda/2018b``, ``iccifort/2018.3.222-GCC-7.3.0-2.30``
``3.1.2`` |                |``GCC/8.2.0-2.31.1``
``3.1.3`` |                |``GCC/8.2.0-2.31.1``, ``gcccuda/2019a``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``3.1.4`` |                |``GCC/8.3.0``, ``GCC/8.3.0-2.32``, ``gcccuda/2019b``, ``iccifort/2019.5.281``
``4.0.0`` |                |``GCC/8.2.0-2.31.1``
``4.0.0`` |``-hpcx``       |``GCC/8.2.0-2.31.1``
``4.0.1`` |                |``GCC/8.3.0-2.32``
``4.0.2`` |                |``GCC/9.2.0-2.32``
``4.0.3`` |                |``GCC/9.3.0``, ``gcccuda/2020a``, ``iccifort/2020.1.217``, ``iccifortcuda/2020a``
``4.0.5`` |                |``GCC/10.2.0``, ``GCC/9.3.0``, ``gcccuda/2020b``, ``iccifort/2020.4.304``
``4.0.5`` |``-CUDA-11.2.1``|``NVHPC/21.2``
``4.0.6`` |                |``GCC/10.3.0``
``4.0.7`` |                |``GCC/10.3.0``
``4.1.0`` |                |``GCC/10.2.0``
``4.1.1`` |                |``GCC/10.3.0``, ``GCC/11.2.0``, ``intel-compilers/2021.2.0``
``4.1.2`` |                |``GCC/10.2.0``, ``GCC/11.2.0``
``4.1.4`` |                |``GCC/11.3.0``, ``GCC/12.2.0``
``system``|                |``GCC/system-2.29``

<a anchor="openms"/>
### *OpenMS*

As part of the deNBI Center for integrative Bioinformatics, OpenMS offers an open-source software  C++ library (+ python bindings) for LC/MS data management and analyses. It provides an infrastructure  for the rapid development of mass spectrometry related software as well as a rich toolset built on top of it.

*homepage*: https://www.openms.de/

version  |toolchain
---------|--------------
``2.4.0``|``foss/2018b``

<a anchor="opennlp"/>
### *OpenNLP*

The Apache OpenNLP library is a machine learning based toolkit for the processing of  natural language text.

*homepage*: https://opennlp.apache.org/

version  |toolchain
---------|----------
``1.8.1``|``system``

<a anchor="openpgm"/>
### *OpenPGM*

OpenPGM is an open source implementation of the Pragmatic General Multicast (PGM)  specification in RFC 3208 available at www.ietf.org. PGM is a reliable and scalable multicast protocol  that enables receivers to detect loss, request retransmission of lost data, or notify an application  of unrecoverable loss. PGM is a receiver-reliable protocol, which means the receiver is responsible  for ensuring all data is received, absolving the sender of reception responsibility.

*homepage*: http://code.google.com/p/openpgm/

version    |toolchain
-----------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``5.2.122``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``

<a anchor="openpiv"/>
### *OpenPIV*

OpenPIV is an open source Particle Image Velocimetry analysis software

*homepage*: http://www.openpiv.net

version   |versionsuffix    |toolchain
----------|-----------------|---------------
``0.21.8``|``-Python-3.7.4``|``intel/2019b``

<a anchor="openpyxl"/>
### *openpyxl*

A Python library to read/write Excel 2010 xlsx/xlsm files

*homepage*: https://openpyxl.readthedocs.io

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------------
``2.6.2`` |                  |``GCCcore/8.2.0``
``2.6.4`` |``-Python-2.7.16``|``GCCcore/8.3.0``
``3.0.3`` |``-Python-3.7.4`` |``GCCcore/8.3.0``
``3.0.7`` |                  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``3.0.9`` |                  |``GCCcore/11.2.0``
``3.0.10``|                  |``GCCcore/11.3.0``

<a anchor="openrefine"/>
### *OpenRefine*

OpenRefine is a power tool that allows you to load data, understand it, clean it up, reconcile it,  and augment it with data coming from the web.

*homepage*: https://github.com/OpenRefine/OpenRefine

version  |versionsuffix      |toolchain
---------|-------------------|----------
``2.7``  |``-Java-1.8.0_144``|``system``
``3.4.1``|``-Java-11``       |``system``

<a anchor="openscenegraph"/>
### *OpenSceneGraph*

The OpenSceneGraph is an open source high performance 3D graphics toolkit,  used by application developers in fields such as visual simulation, games, virtual reality,  scientific visualization and modelling. Written entirely in Standard C++ and OpenGL it runs  on all Windows platforms, OSX, GNU/Linux, IRIX, Solaris, HP-Ux, AIX and FreeBSD operating  systems. The OpenSceneGraph is now well established as the world leading scene graph  technology, used widely in the vis-sim, space, scientific, oil-gas, games and virtual  reality industries.

*homepage*: http://www.openscenegraph.org/

version  |toolchain
---------|----------------------------------------------
``3.6.5``|``foss/2021a``, ``foss/2021b``, ``foss/2022a``

<a anchor="opensees"/>
### *OpenSees*

Open System for Earthquake Engineering Simulation

*homepage*: https://opensees.berkeley.edu/index.php

version  |versionsuffix             |toolchain
---------|--------------------------|---------------
``3.2.0``|``-Python-3.8.2``         |``intel/2020a``
``3.2.0``|``-Python-3.8.2-parallel``|``intel/2020a``

<a anchor="openslide"/>
### *OpenSlide*

OpenSlide is a C library that provides a simple interface to read whole-slide images (also known as virtual slides).

*homepage*: https://openslide.org/

version  |versionsuffix  |toolchain
---------|---------------|---------------------------------------------------------
``3.4.1``|               |``GCCcore/11.2.0``, ``GCCcore/8.2.0``
``3.4.1``|``-largefiles``|``GCCcore/10.3.0``, ``GCCcore/11.3.0``, ``GCCcore/8.2.0``

<a anchor="openslide-python"/>
### *openslide-python*

OpenSlide Python is a Python interface to the OpenSlide library.

*homepage*: https://github.com/openslide/openslide-python

version  |toolchain
---------|--------------------------------------
``1.1.1``|``GCCcore/8.2.0``
``1.1.2``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``

<a anchor="openssl"/>
### *OpenSSL*

The OpenSSL Project is a collaborative effort to develop a robust, commercial-grade, full-featured,  and Open Source toolchain implementing the Secure Sockets Layer (SSL v2/v3) and Transport Layer Security (TLS v1)   protocols as well as a full-strength general purpose cryptography library. 

*homepage*: https://www.openssl.org/

version   |toolchain
----------|---------------------------------------------------------------------
``1.0``   |``system``
``1.0.1f``|``GCC/4.8.2``
``1.0.1k``|``GCC/4.9.2``
``1.0.1s``|``foss/2016a``, ``intel/2016a``
``1.0.2g``|``GCCcore/4.9.3``
``1.0.2h``|``foss/2016.04``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``
``1.1``   |``system``
``1.1.0c``|``GCC/5.4.0-2.26``
``1.1.0e``|``intel/2016b``
``1.1.0h``|``GCCcore/7.3.0``
``1.1.1b``|``GCCcore/8.2.0``
``1.1.1d``|``GCCcore/8.3.0``
``1.1.1e``|``GCCcore/9.3.0``
``1.1.1h``|``GCCcore/10.2.0``
``1.1.1k``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.1.1n``|``GCCcore/11.3.0``
``1.1.1q``|``GCCcore/10.3.0``

<a anchor="openstackclient"/>
### *OpenStackClient*

OpenStackClient (aka OSC) is a command-line client for OpenStack that brings the command set for Compute, Identity, Image, Network, Object Store and Block        Storage APIs together in a single shell with a uniform command structure.

*homepage*: https://docs.openstack.org/python-openstackclient/latest/

version  |toolchain
---------|------------------
``5.5.0``|``GCCcore/10.2.0``
``5.8.0``|``GCCcore/11.2.0``
``6.0.0``|``GCCcore/12.2.0``

<a anchor="opera"/>
### *OPERA*

An optimal genome scaffolding program

*homepage*: https://sourceforge.net/projects/operasf

version  |versionsuffix   |toolchain
---------|----------------|-------------------------------
``2.0.6``|``-Perl-5.28.0``|``foss/2018b``, ``intel/2018b``

<a anchor="opera-ms"/>
### *OPERA-MS*

OPERA-MS is a hybrid metagenomic assembler which combines the  advantages of short and long-read technologies to provide high quality  assemblies, addressing issues of low contiguity for short-read only assemblies,  and low base-pair quality for long-read only assemblies.

*homepage*: https://github.com/CSB5/OPERA-MS

version           |versionsuffix    |toolchain
------------------|-----------------|--------------
``0.9.0-20200802``|``-Python-3.8.2``|``foss/2020a``

<a anchor="optitype"/>
### *OptiType*

OptiType is a novel HLA genotyping algorithm based on integer linear programming,  capable of producing accurate 4-digit HLA genotyping predictions from NGS data  by simultaneously selecting all major and minor HLA Class I alleles. 

*homepage*: https://github.com/FRED-2/OptiType

version  |versionsuffix     |toolchain
---------|------------------|--------------
``1.3.2``|``-Python-2.7.15``|``foss/2018b``
``1.3.2``|``-Python-3.6.6`` |``foss/2018b``

<a anchor="optix"/>
### *OptiX*

OptiX is NVIDIA SDK for easy ray tracing performance.  It provides a simple framework for accessing the GPU’s massive ray tracing  power using state-of-the-art GPU algorithms.

*homepage*: https://developer.nvidia.com/optix

version  |toolchain
---------|------------------
``3.8.0``|``GNU/4.9.3-2.25``
``3.9.0``|``GNU/4.9.3-2.25``
``6.5.0``|``system``
``7.2.0``|``system``

<a anchor="optuna"/>
### *Optuna*

Optuna is an automatic hyperparameter optimization software framework, particularly designed for machine learning. It features an imperative, define-by-run style user API. Thanks to our define-by-run API, the code written with Optuna enjoys high modularity, and the user of Optuna can dynamically construct the search spaces for the hyperparameters.

*homepage*: https://optuna.org/

version   |toolchain
----------|--------------
``2.9.1`` |``foss/2021a``
``2.10.0``|``foss/2021b``

<a anchor="or-tools"/>
### *OR-Tools*

Google Optimization Tools (a.k.a., OR-Tools) is an open-source, fast and portable software suite for solving combinatorial optimization problems.

*homepage*: https://developers.google.com/optimization/

version|versionsuffix    |toolchain
-------|-----------------|--------------
``7.1``|``-Python-3.7.2``|``foss/2019a``

<a anchor="orca"/>
### *ORCA*

ORCA is a flexible, efficient and easy-to-use general purpose tool for quantum chemistry with specific emphasis on spectroscopic properties of open-shell molecules. It features a wide variety of standard quantum chemical methods ranging from semiempirical methods to DFT to single- and multireference correlated ab initio methods. It can also treat environmental and relativistic effects.

*homepage*: https://orcaforum.kofo.mpg.de

version               |versionsuffix     |toolchain
----------------------|------------------|--------------------------------
``3_0_2-linux_x86-64``|``-OpenMPI-1.8.1``|``system``
``4.0.0.2``           |``-OpenMPI-2.0.2``|``system``
``4.0.1``             |``-OpenMPI-2.0.2``|``system``
``4.1.0``             |``-OpenMPI-3.1.3``|``system``
``4.2.0``             |                  |``gompi/2019b``
``4.2.1``             |                  |``gompi/2019b``
``5.0.0``             |                  |``gompi/2021a``
``5.0.0``             |``-static``       |``gompi/2021a``
``5.0.1``             |                  |``gompi/2021a``
``5.0.1``             |``-static``       |``gompi/2021a``
``5.0.2``             |                  |``gompi/2021a``, ``gompi/2021b``
``5.0.2``             |``-static``       |``gompi/2021a``, ``gompi/2021b``
``5.0.3``             |                  |``gompi/2021b``

<a anchor="orffinder"/>
### *ORFfinder*

ORF finder searches for open reading frames (ORFs) in the  DNA sequence you enter. The program returns the range of each ORF,  along with its protein translation. Use ORF finder to search newly  sequenced DNA for potential protein encoding segments, verify  predicted protein using newly developed SMART BLAST or regular BLASTP.

*homepage*: https://www.ncbi.nlm.nih.gov/orffinder/

version  |toolchain
---------|----------
``0.4.3``|``system``

<a anchor="orfm"/>
### *OrfM*

A simple and not slow open reading frame (ORF) caller.

*homepage*: https://github.com/wwood/OrfM

version  |toolchain
---------|----------------------------------------
``0.6.1``|``foss/2016b``
``0.7.1``|``iccifort/2019.1.144-GCC-8.2.0-2.31.1``

<a anchor="orthofinder"/>
### *OrthoFinder*

OrthoFinder is a fast, accurate and comprehensive platform for comparative genomics

*homepage*: https://github.com/davidemms/OrthoFinder

version   |versionsuffix     |toolchain
----------|------------------|---------------
``2.2.7`` |``-Python-2.7.14``|``intel/2018a``
``2.3.3`` |``-Python-2.7.15``|``intel/2018b``
``2.3.8`` |``-Python-2.7.16``|``foss/2019b``
``2.3.11``|``-Python-3.7.4`` |``intel/2019b``
``2.5.2`` |                  |``foss/2020b``
``2.5.4`` |                  |``foss/2020b``

<a anchor="orthomcl"/>
### *OrthoMCL*

OrthoMCL is a genome-scale algorithm for grouping orthologous protein sequences.

*homepage*: http://orthomcl.org/

version  |versionsuffix   |toolchain
---------|----------------|---------------
``1.4``  |``-Perl-5.24.0``|``intel/2016b``
``2.0.9``|``-Perl-5.24.0``|``intel/2016b``

<a anchor="osi"/>
### *Osi*

Osi (Open Solver Interface) provides an abstract base class to a generic linear programming (LP) solver, along with derived classes for specific solvers. Many applications may be able to use the Osi to insulate themselves from a specific LP solver. That is, programs written to the OSI standard may be linked to any solver with an OSI interface and should produce correct results. The OSI has been significantly extended compared to its first incarnation. Currently, the OSI supports linear programming solvers and has rudimentary support for integer programming.

*homepage*: https://github.com/coin-or/Osi

version    |toolchain
-----------|----------------------------------
``0.108.5``|``GCCcore/7.3.0``, ``foss/2018b``
``0.108.6``|``GCC/10.3.0``, ``GCCcore/10.2.0``
``0.108.7``|``GCC/11.2.0``

<a anchor="ospray"/>
### *OSPRay*

Open, Scalable, and Portable Ray Tracing Engine

*homepage*: https://www.ospray.org

version  |toolchain
---------|----------
``2.5.0``|``system``

<a anchor="osu-micro-benchmarks"/>
### *OSU-Micro-Benchmarks*

OSU Micro-Benchmarks

*homepage*: https://mvapich.cse.ohio-state.edu/benchmarks/

version  |versionsuffix   |toolchain
---------|----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``5.3.2``|                |``foss/2016a``, ``foss/2017a``
``5.6.2``|                |``gompi/2019a``
``5.6.3``|                |``gompi/2019b``, ``gompi/2020a``, ``gompi/2020b``, ``gompic/2019b``, ``gompic/2020a``, ``iimpi/2019a``, ``iimpi/2019b``, ``iimpi/2020a``, ``iimpi/2020b``, ``iimpic/2019b``, ``iimpic/2020a``
``5.7``  |                |``gompi/2020b``, ``gompic/2020b``, ``iimpi/2020b``
``5.7.1``|                |``ffmpi/4.5.0``, ``gompi/2021a``, ``gompi/2021b``, ``iimpi/2021a``, ``iompi/2021a``
``5.7.1``|``-CUDA-11.3.1``|``gompi/2021a``
``5.8``  |                |``iimpi/2021b``
``5.9``  |                |``gompi/2022.05``, ``gompi/2022a``
``5.9``  |``-CUDA-11.3.1``|``gompi/2021a``
``5.9``  |``-CUDA-11.4.1``|``gompi/2021b``
``5.9``  |``-CUDA-11.7.0``|``gompi/2022a``
``6.2``  |                |``gompi/2022.10``, ``gompi/2022b``

<a anchor="otf2"/>
### *OTF2*

The Open Trace Format 2 is a highly scalable, memory efficient event trace  data format plus support library. It is the new standard trace format for  Scalasca, Vampir, and TAU and is open for other tools.  

*homepage*: https://www.score-p.org

version|toolchain
-------|-------------------------------------------------------
``2.0``|``foss/2016a``, ``foss/2017a``
``2.2``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``2.3``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``3.0``|``GCCcore/11.3.0``

<a anchor="ovito"/>
### *OVITO*

OVITO is a scientific visualization and data analysis solution for atomistic and other particle-based models. It helps scientists gain meaningful and quick insights from numerical simulation results.

*homepage*: https://www.ovito.org

version   |versionsuffix|toolchain
----------|-------------|---------------
``3.7.11``|``-basic``   |``gompi/2022a``

<a anchor="owncloud"/>
### *ownCloud*

The ownCloud Desktop Client is a tool to synchronize files from ownCloud Server with your computer.

*homepage*: https://github.com/owncloud/client

version  |toolchain
---------|-----------------
``2.4.3``|``foss/2018b``
``2.5.4``|``GCCcore/8.2.0``

<a anchor="oxford_asl"/>
### *oxford_asl*

A command line tool for quantification of perfusion from ASL data

*homepage*: https://github.com/ibme-qubic/oxford_asl

version  |versionsuffix             |toolchain
---------|--------------------------|---------------
``3.9.6``|``-centos7-Python-2.7.13``|``intel/2017a``

<a anchor="p"/>
### *P*


<a href="#p11-kit">p11-kit</a> - <a href="#p4-phylogenetics">p4-phylogenetics</a> - <a href="#p4est">p4est</a> - <a href="#p4vasp">p4vasp</a> - <a href="#p7zip">p7zip</a> - <a href="#packmol">packmol</a> - <a href="#pagmo">pagmo</a> - <a href="#pairsnp">pairsnp</a> - <a href="#pal2nal">PAL2NAL</a> - <a href="#paladin">paladin</a> - <a href="#paml">PAML</a> - <a href="#panaroo">panaroo</a> - <a href="#pandapower">pandapower</a> - <a href="#pandas">pandas</a> - <a href="#pandas-datareader">pandas-datareader</a> - <a href="#pandaseq">PANDAseq</a> - <a href="#pandoc">Pandoc</a> - <a href="#panedr">Panedr</a> - <a href="#pango">Pango</a> - <a href="#pangolin">pangolin</a> - <a href="#panito">panito</a> - <a href="#papi">PAPI</a> - <a href="#parallel">parallel</a> - <a href="#parallel-fastq-dump">parallel-fastq-dump</a> - <a href="#parallelio">ParallelIO</a> - <a href="#parameterized">parameterized</a> - <a href="#parasail">parasail</a> - <a href="#paraver">Paraver</a> - <a href="#paraview">ParaView</a> - <a href="#parcels">Parcels</a> - <a href="#pari-gp">PARI-GP</a> - <a href="#parmed">ParmEd</a> - <a href="#parmetis">ParMETIS</a> - <a href="#parmgridgen">ParMGridGen</a> - <a href="#partitionfinder">PartitionFinder</a> - <a href="#pasta">pasta</a> - <a href="#pastix">PaStiX</a> - <a href="#patch">patch</a> - <a href="#patchelf">patchelf</a> - <a href="#path.py">path.py</a> - <a href="#paup">PAUP</a> - <a href="#pauvre">pauvre</a> - <a href="#pbbam">pbbam</a> - <a href="#pbcopper">pbcopper</a> - <a href="#pbdagcon">pbdagcon</a> - <a href="#pbmm2">pbmm2</a> - <a href="#pbs_python">pbs_python</a> - <a href="#pbsuite">PBSuite</a> - <a href="#pcangsd">PCAngsd</a> - <a href="#pcc">PCC</a> - <a href="#pcl">PCL</a> - <a href="#pcmsolver">PCMSolver</a> - <a href="#pcraster">PCRaster</a> - <a href="#pcre">PCRE</a> - <a href="#pcre2">PCRE2</a> - <a href="#pdt">PDT</a> - <a href="#pear">PEAR</a> - <a href="#penncnv">PennCNV</a> - <a href="#pept">PEPT</a> - <a href="#percolator">Percolator</a> - <a href="#perl">Perl</a> - <a href="#perl-app-cpanminus">perl-app-cpanminus</a> - <a href="#perl4-corelibs">Perl4-CoreLibs</a> - <a href="#pest++">PEST++</a> - <a href="#petsc">PETSc</a> - <a href="#petsc4py">petsc4py</a> - <a href="#pfind">pfind</a> - <a href="#pftoolsv3">pftoolsV3</a> - <a href="#pfunit">pFUnit</a> - <a href="#pgdspider">PGDSpider</a> - <a href="#pgi">PGI</a> - <a href="#phanotate">PHANOTATE</a> - <a href="#phantompeakqualtools">Phantompeakqualtools</a> - <a href="#phase">PHASE</a> - <a href="#phast">PHAST</a> - <a href="#phewas">PheWAS</a> - <a href="#phipack">PhiPack</a> - <a href="#phlat">PHLAT</a> - <a href="#phonemizer">phonemizer</a> - <a href="#phono3py">phono3py</a> - <a href="#phonopy">phonopy</a> - <a href="#photontorch">photontorch</a> - <a href="#phototonic">phototonic</a> - <a href="#phylip">PHYLIP</a> - <a href="#phylobayes-mpi">PhyloBayes-MPI</a> - <a href="#phylokit">phylokit</a> - <a href="#phylonaut">phylonaut</a> - <a href="#phylophlan">PhyloPhlAn</a> - <a href="#phyml">PhyML</a> - <a href="#phyx">phyx</a> - <a href="#picard">picard</a> - <a href="#pigz">pigz</a> - <a href="#pil">PIL</a> - <a href="#pileometh">PileOMeth</a> - <a href="#pillow">Pillow</a> - <a href="#pillow-simd">Pillow-SIMD</a> - <a href="#pilon">Pilon</a> - <a href="#pims">PIMS</a> - <a href="#pindel">Pindel</a> - <a href="#pingouin">Pingouin</a> - <a href="#pint">Pint</a> - <a href="#pip">pip</a> - <a href="#pipits">PIPITS</a> - <a href="#pirs">pIRS</a> - <a href="#pisces">Pisces</a> - <a href="#pisvm">piSvM</a> - <a href="#pisvm-jsc">piSvM-JSC</a> - <a href="#pixman">pixman</a> - <a href="#pizzly">pizzly</a> - <a href="#pkg-config">pkg-config</a> - <a href="#pkgconf">pkgconf</a> - <a href="#pkgconfig">pkgconfig</a> - <a href="#plantcv">plantcv</a> - <a href="#plascope">PlaScope</a> - <a href="#plasmapy">PlasmaPy</a> - <a href="#plast">PLAST</a> - <a href="#platanus">Platanus</a> - <a href="#platypus">Platypus</a> - <a href="#plc">plc</a> - <a href="#plink">PLINK</a> - <a href="#plinkqc">plinkQC</a> - <a href="#plinkseq">PLINKSEQ</a> - <a href="#ploticus">Ploticus</a> - <a href="#plotly">plotly</a> - <a href="#plotly-orca">plotly-orca</a> - <a href="#plotly.py">plotly.py</a> - <a href="#plplot">PLplot</a> - <a href="#plumed">PLUMED</a> - <a href="#ply">PLY</a> - <a href="#pmix">PMIx</a> - <a href="#pmx">pmx</a> - <a href="#pnetcdf">PnetCDF</a> - <a href="#pocl">pocl</a> - <a href="#poetry">poetry</a> - <a href="#polars">polars</a> - <a href="#polymake">polymake</a> - <a href="#pomkl">pomkl</a> - <a href="#pompi">pompi</a> - <a href="#poppler">poppler</a> - <a href="#popscle">popscle</a> - <a href="#popt">popt</a> - <a href="#porechop">Porechop</a> - <a href="#porefoam">porefoam</a> - <a href="#poretools">poretools</a> - <a href="#portcullis">Portcullis</a> - <a href="#postgres-xl">Postgres-XL</a> - <a href="#postgresql">PostgreSQL</a> - <a href="#pot">POT</a> - <a href="#pov-ray">POV-Ray</a> - <a href="#ppanggolin">PPanGGOLiN</a> - <a href="#ppfold">PPfold</a> - <a href="#ppl">ppl</a> - <a href="#pplacer">pplacer</a> - <a href="#pplpy">pplpy</a> - <a href="#prank">PRANK</a> - <a href="#prc">PRC</a> - <a href="#precice">preCICE</a> - <a href="#prequal">PREQUAL</a> - <a href="#preseq">preseq</a> - <a href="#presto">presto</a> - <a href="#pretty-yaml">pretty-yaml</a> - <a href="#primer3">Primer3</a> - <a href="#prinseq">PRINSEQ</a> - <a href="#printproto">printproto</a> - <a href="#prisms-pf">PRISMS-PF</a> - <a href="#probabel">ProbABEL</a> - <a href="#prodigal">prodigal</a> - <a href="#profit">ProFit</a> - <a href="#proj">PROJ</a> - <a href="#projectq">ProjectQ</a> - <a href="#prokka">prokka</a> - <a href="#prompt-toolkit">prompt-toolkit</a> - <a href="#proovread">proovread</a> - <a href="#propy">propy</a> - <a href="#proteinortho">Proteinortho</a> - <a href="#prothint">ProtHint</a> - <a href="#protobuf">protobuf</a> - <a href="#protobuf-python">protobuf-python</a> - <a href="#protozero">protozero</a> - <a href="#prsice">PRSice</a> - <a href="#pscom">pscom</a> - <a href="#psi">PSI</a> - <a href="#psi4">PSI4</a> - <a href="#psiclass">PsiCLASS</a> - <a href="#psipred">PSIPRED</a> - <a href="#psmc">psmc</a> - <a href="#psmpi">psmpi</a> - <a href="#psmpi2">psmpi2</a> - <a href="#psolver">PSolver</a> - <a href="#psortb">PSORTb</a> - <a href="#psrecord">psrecord</a> - <a href="#pstoedit">pstoedit</a> - <a href="#psutil">psutil</a> - <a href="#psycopg2">psycopg2</a> - <a href="#ptemcee">ptemcee</a> - <a href="#ptesfinder">PTESFinder</a> - <a href="#pubtcrs">pubtcrs</a> - <a href="#pugixml">pugixml</a> - <a href="#pullseq">pullseq</a> - <a href="#pulp">PuLP</a> - <a href="#purge_dups">purge_dups</a> - <a href="#py">py</a> - <a href="#py-aiger">py-aiger</a> - <a href="#py-aiger-bdd">py-aiger-bdd</a> - <a href="#py-cpuinfo">py-cpuinfo</a> - <a href="#pyabc">pyABC</a> - <a href="#pyamg">PyAMG</a> - <a href="#pyaps3">PyAPS3</a> - <a href="#pybedtools">pybedtools</a> - <a href="#pyberny">PyBerny</a> - <a href="#pybigwig">pyBigWig</a> - <a href="#pybind11">pybind11</a> - <a href="#pycairo">PyCairo</a> - <a href="#pycalib">PyCalib</a> - <a href="#pyccel">pyccel</a> - <a href="#pycharm">PyCharm</a> - <a href="#pychemps2">PyCheMPS2</a> - <a href="#pycifrw">PyCifRW</a> - <a href="#pyclone">PyClone</a> - <a href="#pycma">pycma</a> - <a href="#pycocotools">pycocotools</a> - <a href="#pycodestyle">pycodestyle</a> - <a href="#pycogent">PyCogent</a> - <a href="#pycoqc">pycoQC</a> - <a href="#pycuda">PyCUDA</a> - <a href="#pycurl">PycURL</a> - <a href="#pydamage">PyDamage</a> - <a href="#pydantic">pydantic</a> - <a href="#pydatastream">PyDatastream</a> - <a href="#pydicom">pydicom</a> - <a href="#pydlpoly">pydlpoly</a> - <a href="#pydot">pydot</a> - <a href="#pyega3">pyEGA3</a> - <a href="#pyenchant">pyenchant</a> - <a href="#pyevtk">PyEVTK</a> - <a href="#pyfai">pyFAI</a> - <a href="#pyfaidx">pyfaidx</a> - <a href="#pyfasta">pyfasta</a> - <a href="#pyffmpeg">PyFFmpeg</a> - <a href="#pyfftw">pyFFTW</a> - <a href="#pyfits">pyfits</a> - <a href="#pyfmi">PyFMI</a> - <a href="#pyfoam">PyFoam</a> - <a href="#pyfr">PyFR</a> - <a href="#pyfrag">PyFrag</a> - <a href="#pygccxml">pygccxml</a> - <a href="#pygenometracks">pyGenomeTracks</a> - <a href="#pygeos">PyGEOS</a> - <a href="#pygimli">pyGIMLi</a> - <a href="#pygments">Pygments</a> - <a href="#pygmo">pygmo</a> - <a href="#pygobject">PyGObject</a> - <a href="#pygraphviz">pygraphviz</a> - <a href="#pygrib">pygrib</a> - <a href="#pygtk">PyGTK</a> - <a href="#pygts">PyGTS</a> - <a href="#pygwas">PyGWAS</a> - <a href="#pyhdf">pyhdf</a> - <a href="#pyimagej">PyImageJ</a> - <a href="#pyiron">pyiron</a> - <a href="#pyke3">Pyke3</a> - <a href="#pylift">pylift</a> - <a href="#pylint">Pylint</a> - <a href="#pymatgen">pymatgen</a> - <a href="#pymatgen-db">pymatgen-db</a> - <a href="#pymbar">pymbar</a> - <a href="#pymc3">PyMC3</a> - <a href="#pymca">pymca</a> - <a href="#pymemcache">pymemcache</a> - <a href="#pymol">PyMOL</a> - <a href="#pynast">PyNAST</a> - <a href="#pyobjcryst">pyobjcryst</a> - <a href="#pyod">PyOD</a> - <a href="#pyomo">Pyomo</a> - <a href="#pyopencl">PyOpenCL</a> - <a href="#pyopengl">PyOpenGL</a> - <a href="#pyparsing">pyparsing</a> - <a href="#pyplusplus">pyplusplus</a> - <a href="#pypower">PYPOWER</a> - <a href="#pyproj">pyproj</a> - <a href="#pypsa">PyPSA</a> - <a href="#pyqstem">pyqstem</a> - <a href="#pyqt">PyQt</a> - <a href="#pyqt5">PyQt5</a> - <a href="#pyqtgraph">PyQtGraph</a> - <a href="#pyradiomics">pyradiomics</a> - <a href="#pyre">PyRe</a> - <a href="#pyretis">PyRETIS</a> - <a href="#pyringe">pyringe</a> - <a href="#pyro-api">pyro-api</a> - <a href="#pyro-ppl">pyro-ppl</a> - <a href="#pyro4">Pyro4</a> - <a href="#pyrosetta">PyRosetta</a> - <a href="#pysam">Pysam</a> - <a href="#pysamstats">pysamstats</a> - <a href="#pysat">PySAT</a> - <a href="#pyscaf">pyScaf</a> - <a href="#pyscenic">pySCENIC</a> - <a href="#pyscf">PySCF</a> - <a href="#pysheds">pysheds</a> - <a href="#pyshp">pyshp</a> - <a href="#pyside2">PySide2</a> - <a href="#pysndfx">pysndfx</a> - <a href="#pysolar">Pysolar</a> - <a href="#pyspoa">pyspoa</a> - <a href="#pysqlite">pysqlite</a> - <a href="#pystan">PyStan</a> - <a href="#pysteps">pysteps</a> - <a href="#pystran">pystran</a> - <a href="#pytables">PyTables</a> - <a href="#pytest">pytest</a> - <a href="#pytest-benchmark">pytest-benchmark</a> - <a href="#pytest-xdist">pytest-xdist</a> - <a href="#pythia">PYTHIA</a> - <a href="#python">Python</a> - <a href="#python-docx">python-docx</a> - <a href="#python-hl7">python-hl7</a> - <a href="#python-igraph">python-igraph</a> - <a href="#python-irodsclient">python-irodsclient</a> - <a href="#python-isal">python-isal</a> - <a href="#python-levenshtein">python-Levenshtein</a> - <a href="#python-libsbml">python-libsbml</a> - <a href="#python-parasail">python-parasail</a> - <a href="#python-telegram-bot">python-telegram-bot</a> - <a href="#python-weka-wrapper3">python-weka-wrapper3</a> - <a href="#pythran">pythran</a> - <a href="#pytorch">PyTorch</a> - <a href="#pytorch-geometric">PyTorch-Geometric</a> - <a href="#pytorch-lightning">PyTorch-Lightning</a> - <a href="#pytorch3d">PyTorch3D</a> - <a href="#pyvcf">PyVCF</a> - <a href="#pywannier90">pyWannier90</a> - <a href="#pywavelets">PyWavelets</a> - <a href="#pyyaml">PyYAML</a> - <a href="#pyzmq">PyZMQ</a>


<a anchor="p11-kit"/>
### *p11-kit*

Provides a way to load and enumerate PKCS#11 modules.  Provides a standard configuration setup for installing  PKCS#11 modules in such a way that they're discoverable.  Also solves problems with coordinating the use of PKCS#11  by different components or libraries living in the same process.

*homepage*: https://p11-glue.freedesktop.org/p11-kit.html

version   |toolchain
----------|----------------------------------------------------------------------
``0.23.2``|``GCCcore/5.4.0``, ``GNU/4.9.3-2.25``, ``foss/2016a``, ``intel/2016a``
``0.24.0``|``GCCcore/10.3.0``
``0.24.1``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="p4-phylogenetics"/>
### *p4-phylogenetics*

A Python phyloinformatic toolkit, and an implementation of tree-heterogeneous models of evolution.

*homepage*: http://p4.nhm.ac.uk/

version         |versionsuffix    |toolchain
----------------|-----------------|--------------
``1.4-20210322``|``-Python-3.7.4``|``foss/2019b``

<a anchor="p4est"/>
### *p4est*

p4est is a C library to manage a collection (a forest) of multiple connected adaptive quadtrees or octrees in parallel.

*homepage*: https://www.p4est.org

version|toolchain
-------|-------------------------------
``2.2``|``foss/2019a``, ``intel/2019a``
``2.8``|``foss/2021a``

<a anchor="p4vasp"/>
### *p4vasp*

Visualization suite for VASP

*homepage*: http://www.p4vasp.at/

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------
``0.3.29``|``-Python-2.7.11``|``intel/2016a``
``0.3.30``|``-Python-2.7.14``|``intel/2017b``, ``intel/2018a``
``0.3.30``|``-Python-2.7.15``|``foss/2018b``

<a anchor="p7zip"/>
### *p7zip*

p7zip is a quick port of 7z.exe and 7za.exe (command line version of 7zip) for Unix.  7-Zip is a file archiver with highest compression ratio.

*homepage*: http://p7zip.sourceforge.net/

version   |toolchain
----------|---------------------------------------------------------------------------
``9.38.1``|``GCC/4.9.2``, ``system``
``16.02`` |``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``, ``intel/2018a``
``17.03`` |``GCCcore/10.2.0``
``17.04`` |``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="packmol"/>
### *packmol*

Packing Optimization for Molecular Dynamics Simulations

*homepage*: http://m3g.iqm.unicamp.br/packmol

version    |toolchain
-----------|-------------------------------
``16.103`` |``intel/2016a``
``18.013`` |``foss/2018a``, ``intel/2018a``
``20.2.2`` |``GCC/10.2.0``
``v20.2.2``|``iccifort/2020.1.217``

<a anchor="pagmo"/>
### *pagmo*

pagmo is a C++ scientific library for massively parallel optimization.

*homepage*: https://esa.github.io/pagmo2

version   |toolchain
----------|------------------------------
``2.17.0``|``foss/2020b``
``2.18.0``|``foss/2021a``, ``foss/2021b``

<a anchor="pairsnp"/>
### *pairsnp*

A set of scripts for very quickly obtaining pairwise SNP distance matrices from multiple sequence alignments  using sparse matrix libraries to improve performance. 

*homepage*: https://github.com/gtonkinhill/pairsnp

version  |toolchain
---------|--------------
``0.0.7``|``foss/2021a``

<a anchor="pal2nal"/>
### *PAL2NAL*

PAL2NAL is a program that converts a multiple sequence alignment of proteins and  the corresponding DNA (or mRNA) sequences into a codon alignment. The program automatically  assigns the corresponding codon sequence even if the input DNA sequence has mismatches with the  input protein sequence, or contains UTRs, polyA tails. It can also deal with frame shifts in the  input alignment, which is suitable for the analysis of pseudogenes. The resulting codon alignment  can further be subjected to the calculation of synonymous (d_S) and non-synonymous (d_N) subs-  titution rates. 

*homepage*: http://www.bork.embl.de/pal2nal/

version|toolchain
-------|--------------------------------------
``14`` |``GCCcore/10.2.0``, ``GCCcore/10.3.0``

<a anchor="paladin"/>
### *paladin*

Protein ALignment And Detection INterface  PALADIN is a protein sequence alignment tool designed for the accurate functional characterization of metagenomes. 

*homepage*: https://github.com/ToniWestbrook/paladin

version  |toolchain
---------|--------------------------------------
``1.4.6``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``

<a anchor="paml"/>
### *PAML*

PAML is a package of programs for phylogenetic   analyses of DNA or protein sequences using maximum likelihood.

*homepage*: http://abacus.gene.ucl.ac.uk/software/paml.html

version |toolchain
--------|-----------------------------------------------------------------------------
``4.9i``|``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``
``4.9j``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``

<a anchor="panaroo"/>
### *panaroo*

A pangenome analysis pipeline.

*homepage*: https://gtonkinhill.github.io/panaroo/

version  |toolchain
---------|--------------
``1.2.8``|``foss/2020b``
``1.2.9``|``foss/2021a``

<a anchor="pandapower"/>
### *pandapower*

An easy to use open source tool for power system modeling, analysis and optimization with a high degree of automation

*homepage*: http://www.pandapower.org/

version  |toolchain
---------|--------------
``2.7.0``|``foss/2020b``

<a anchor="pandas"/>
### *pandas*

pandas is an open source, BSD-licensed library providing high-performance, easy-to-use data structures  and data analysis tools for the Python programming language.

*homepage*: https://pypi.python.org/pypi/pandas/

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``0.18.0``|``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``0.18.0``|``-Python-3.5.1`` |``foss/2016a``, ``intel/2016a``
``0.18.1``|``-Python-2.7.12``|``intel/2016b``
``0.18.1``|``-Python-3.5.2`` |``intel/2016b``
``0.19.0``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``0.19.0``|``-Python-3.5.2`` |``foss/2016b``, ``intel/2016b``
``0.19.1``|``-Python-2.7.12``|``intel/2016b``
``0.19.1``|``-Python-3.5.2`` |``intel/2016b``
``0.20.1``|``-Python-3.6.1`` |``intel/2017a``
``0.21.0``|``-Python-2.7.13``|``intel/2017a``
``0.21.0``|``-Python-3.6.3`` |``intel/2017b``
``1.1.2`` |``-Python-3.8.2`` |``foss/2020a``

<a anchor="pandas-datareader"/>
### *pandas-datareader*

Up to date remote data access for pandas, works for multiple versions of pandas.

*homepage*: https://pypi.org/project/pandas-datareader

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.7.0``|``-Python-3.6.4``|``intel/2018a``

<a anchor="pandaseq"/>
### *PANDAseq*

PANDASEQ is a program to align Illumina reads, optionally with PCR primers embedded     in the sequence, and reconstruct an overlapping sequence.

*homepage*: https://github.com/neufeld/pandaseq

version |toolchain
--------|----------------------------------------------------------
``2.10``|``GCC/5.4.0-2.26``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``
``2.11``|``foss/2017b``, ``intel/2017b``, ``intel/2018a``

<a anchor="pandoc"/>
### *Pandoc*

If you need to convert files from one markup format into another, pandoc is your swiss-army knife

*homepage*: http://pandoc.org

version  |toolchain
---------|----------
``2.1.3``|``system``
``2.5``  |``system``
``2.10`` |``system``
``2.13`` |``system``

<a anchor="panedr"/>
### *Panedr*

Panedr uses the Pyedr library to read a Gromacs EDR binary energy XDR file and returns its contents as a pandas dataframe.

*homepage*: https://github.com/MDAnalysis/panedr

version  |toolchain
---------|--------------
``0.7.0``|``foss/2021a``

<a anchor="pango"/>
### *Pango*

Pango is a library for laying out and rendering of text, with an emphasis on internationalization. Pango can be used anywhere that text layout is needed, though most of the work on Pango so far has been done in the context of the GTK+ widget toolkit. Pango forms the core of text and font handling for GTK+-2.x.

*homepage*: https://www.pango.org/

version    |toolchain
-----------|------------------------------------
``1.39.0`` |``foss/2016a``, ``intel/2016a``
``1.40.1`` |``foss/2016a``, ``intel/2016a``
``1.40.3`` |``foss/2016b``, ``intel/2016b``
``1.40.5`` |``intel/2017a``
``1.40.12``|``intel/2017a``
``1.40.14``|``foss/2017b``, ``intel/2017b``
``1.41.0`` |``foss/2017b``, ``intel/2017b``
``1.41.1`` |``foss/2018a``, ``intel/2018a``
``1.42.4`` |``foss/2018b``, ``fosscuda/2018b``
``1.43.0`` |``GCCcore/8.2.0``
``1.44.7`` |``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.47.0`` |``GCCcore/10.2.0``
``1.48.5`` |``GCCcore/10.3.0``
``1.48.8`` |``GCCcore/11.2.0``
``1.50.7`` |``GCCcore/11.3.0``
``1.50.12``|``GCCcore/12.2.0``

<a anchor="pangolin"/>
### *pangolin*

Software package for assigning SARS-CoV-2 genome sequences to global lineages. This module also contains the faToVcf tool

*homepage*: https://cov-lineages.org/pangolin.html

version   |versionsuffix             |toolchain
----------|--------------------------|--------------
``3.1.11``|                          |``foss/2020b``
``3.1.16``|``-pangoLEARN-2021-10-18``|``foss/2021b``
``3.1.16``|``-pangoLEARN-2021-11-25``|``foss/2021b``

<a anchor="panito"/>
### *panito*

Calculate genome wide average nucleotide identity (gwANI) for a multiFASTA alignment.

*homepage*: https://github.com/sanger-pathogens/panito

version  |toolchain
---------|--------------
``0.0.1``|``GCC/10.3.0``

<a anchor="papi"/>
### *PAPI*

PAPI provides the tool designer and application engineer with a consistent  interface and methodology for use of the performance counter hardware found  in most major microprocessors. PAPI enables software engineers to see, in near  real time, the relation between software performance and processor events.  In addition Component PAPI provides access to a collection of components  that expose performance measurement opportunites across the hardware and  software stack. 

*homepage*: https://icl.cs.utk.edu/projects/papi/

version    |toolchain
-----------|--------------------------------------------------------
``5.4.3``  |``foss/2016a``
``5.5.1``  |``GCCcore/6.3.0``, ``GCCcore/6.4.0``
``5.6.0``  |``GCCcore/6.4.0``
``5.7.0``  |``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``6.0.0``  |``GCCcore/10.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``6.0.0.1``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``

<a anchor="parallel"/>
### *parallel*

parallel: Build and execute shell commands in parallel

*homepage*: https://savannah.gnu.org/projects/parallel/

version     |toolchain
------------|-------------------------------
``20141122``|``GCC/4.9.2``
``20150322``|``GCC/4.9.2``
``20150822``|``GCC/4.9.2``
``20160622``|``foss/2016a``
``20170822``|``intel/2017a``
``20171022``|``intel/2017b``
``20171122``|``foss/2017b``, ``intel/2017b``
``20180422``|``intel/2018a``
``20180822``|``foss/2018b``
``20181222``|``intel/2018b``
``20190222``|``GCCcore/7.3.0``
``20190622``|``GCCcore/8.2.0``
``20190922``|``GCCcore/8.3.0``
``20200422``|``GCCcore/9.3.0``
``20200522``|``GCCcore/9.3.0``
``20210322``|``GCCcore/10.2.0``
``20210622``|``GCCcore/10.3.0``
``20210722``|``GCCcore/11.2.0``
``20220722``|``GCCcore/11.3.0``

<a anchor="parallel-fastq-dump"/>
### *parallel-fastq-dump*

parallel fastq-dump wrapper

*homepage*: https://github.com/rvalieris/parallel-fastq-dump

version  |versionsuffix    |toolchain
---------|-----------------|-----------------
``0.6.5``|``-Python-3.7.2``|``GCCcore/8.2.0``
``0.6.6``|``-Python-3.8.2``|``GCCcore/9.3.0``
``0.6.7``|                 |``gompi/2020b``

<a anchor="parallelio"/>
### *ParallelIO*

A high-level Parallel I/O Library for structured grid applications

*homepage*: https://github.com/NCAR/ParallelIO

version   |toolchain
----------|---------------
``2.2.2a``|``intel/2017a``

<a anchor="parameterized"/>
### *parameterized*

Parameterized testing with any Python test framework 

*homepage*: https://github.com/wolever/parameterized

version  |toolchain
---------|------------------
``0.8.1``|``GCCcore/10.3.0``

<a anchor="parasail"/>
### *parasail*

parasail is a SIMD C (C99) library containing implementations  of the Smith-Waterman (local), Needleman-Wunsch (global), and semi-global  pairwise sequence alignment algorithms. 

*homepage*: https://github.com/jeffdaily/parasail

version  |toolchain
---------|--------------------------------------
``2.2``  |``intel/2018a``
``2.4``  |``foss/2018b``
``2.4.1``|``GCC/8.3.0``, ``intel/2019b``
``2.4.2``|``GCC/9.3.0``, ``iccifort/2020.1.217``
``2.4.3``|``GCC/10.2.0``, ``GCC/10.3.0``
``2.5``  |``GCC/11.2.0``

<a anchor="paraver"/>
### *Paraver*

A very powerful performance visualization and analysis tool based on  traces that can be used to analyse any information that is expressed on its input trace format.  Traces for parallel MPI, OpenMP and other programs can be genereated with Extrae.

*homepage*: https://tools.bsc.es/paraver

version  |toolchain
---------|--------------
``4.8.1``|``foss/2019a``
``4.9.2``|``foss/2021a``

<a anchor="paraview"/>
### *ParaView*

ParaView is a scientific parallel visualizer.

*homepage*: https://www.paraview.org

version   |versionsuffix         |toolchain
----------|----------------------|-------------------------------------------------------------------------------------------------
``4.4.0`` |                      |``foss/2016a``, ``intel/2016a``
``4.4.0`` |``-mpi``              |``gimkl/2.11.5``
``5.1.2`` |``-mpi``              |``foss/2016b``, ``intel/2016b``
``5.2.0`` |``-mpi``              |``foss/2016b``, ``intel/2017a``
``5.3.0`` |``-mpi``              |``foss/2016b``
``5.4.1`` |``-mpi``              |``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``
``5.5.2`` |``-Python-2.7.15-mpi``|``foss/2018b``
``5.6.2`` |``-Python-3.7.4-mpi`` |``foss/2019b``, ``intel/2019b``
``5.8.0`` |``-Python-3.8.2-mpi`` |``foss/2020a``, ``intel/2020a``
``5.8.1`` |``-mpi``              |``foss/2020b``
``5.9.1`` |``-mpi``              |``foss/2021a``, ``foss/2021b``, ``intel/2021a``
``5.10.1``|``-mpi``              |``foss/2022a``

<a anchor="parcels"/>
### *Parcels*

Parcels (Probably A Really Computationally Efficient Lagrangian Simulator) is a set of Python classes and methods to create customisable particle tracking simulations using output from Ocean Circulation models. Parcels can be used to track passive and active particulates such as water, plankton, plastic and fish.

*homepage*: http://www.oceanparcels.org/

version  |toolchain
---------|--------------
``2.4.0``|``foss/2022a``

<a anchor="pari-gp"/>
### *PARI-GP*

PARI/GP is a widely used computer algebra system designed for fast computations in number theory  (factorizations, algebraic  number theory, elliptic curves...), but also contains a large number of other useful  functions to compute with mathematical entities such as matrices, polynomials, power series, algebraic numbers etc.,  and a lot of transcendental functions. PARI is  also available as a C library to allow for faster computations. 

*homepage*: http://pari.math.u-bordeaux.fr

version  |toolchain
---------|--------------
``2.7.6``|``foss/2016a``

<a anchor="parmed"/>
### *ParmEd*

ParmEd is a general tool for aiding in investigations of biomolecular systems using popular  molecular simulation packages, like Amber, CHARMM, and OpenMM written in Python.

*homepage*: https://parmed.github.io/ParmEd

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``2.7.3``|``-Python-3.6.3``|``intel/2017b``
``3.2.0``|``-Python-3.7.4``|``intel/2019b``
``3.2.0``|``-Python-3.8.2``|``intel/2020a``

<a anchor="parmetis"/>
### *ParMETIS*

ParMETIS is an MPI-based parallel library that implements a variety of algorithms for partitioning  unstructured graphs, meshes, and for computing fill-reducing orderings of sparse matrices. ParMETIS extends the  functionality provided by METIS and includes routines that are especially suited for parallel AMR computations and  large scale numerical simulations. The algorithms implemented in ParMETIS are based on the parallel multilevel k-way  graph-partitioning, adaptive repartitioning, and parallel multi-constrained partitioning schemes.

*homepage*: http://glaros.dtc.umn.edu/gkhome/metis/parmetis/overview

version  |toolchain
---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``4.0.3``|``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``gimkl/2.11.5``, ``gompi/2019a``, ``gompi/2019b``, ``gompi/2020a``, ``gompi/2020b``, ``gompi/2021a``, ``gompi/2021b``, ``gompi/2022a``, ``iimpi/2019a``, ``iimpi/2019b``, ``iimpi/2020a``, ``iimpi/2020b``, ``iimpi/2021a``, ``iimpi/2021b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``

<a anchor="parmgridgen"/>
### *ParMGridGen*

ParMGridGen is an MPI-based parallel library that is based on the serial package MGridGen,  that implements (serial) algorithms for obtaining a sequence of successive coarse grids that are well-suited  for geometric multigrid methods.

*homepage*: http://www-users.cs.umn.edu/~moulitsa/software.html

version|toolchain
-------|----------------------------------------------------------------------------------------------------------------------
``1.0``|``gimkl/2.11.5``, ``gompi/2019b``, ``gompi/2020a``, ``iimpi/2019b``, ``iimpi/2020a``, ``intel/2016a``, ``intel/2017a``

<a anchor="partitionfinder"/>
### *PartitionFinder*

PartitionFinder 2 is a Python program for simultaneously choosing partitioning schemes and models of molecular evolution for phylogenetic analyses of DNA, protein, and morphological data. You can PartitionFinder 2 before running a phylogenetic analysis, in order to decide how to divide up your sequence data into separate blocks before analysis, and to simultaneously perform model selection on each of those blocks.

*homepage*: https://www.robertlanfear.com/partitionfinder

version  |versionsuffix     |toolchain
---------|------------------|---------------
``2.1.1``|``-Python-2.7.16``|``intel/2019b``

<a anchor="pasta"/>
### *pasta*

PASTA (Practical Alignment using SATe and Transitivity)

*homepage*: https://github.com/smirarab/pasta

version  |versionsuffix    |toolchain
---------|-----------------|--------------------
``1.8.5``|``-Python-3.7.2``|``GCC/8.2.0-2.31.1``

<a anchor="pastix"/>
### *PaStiX*

PaStiX (Parallel Sparse matriX package) is a scientific library that provides a high performance  parallel solver for very large sparse linear systems based on direct methods.

*homepage*: http://pastix.gforge.inria.fr/

version  |toolchain
---------|--------------
``5.2.3``|``foss/2017b``

<a anchor="patch"/>
### *patch*

Patch takes a patch file containing a difference listing produced by the diff program  and applies those differences to one or more original files, producing patched versions.

*homepage*: https://savannah.gnu.org/projects/patch

version  |toolchain
---------|----------
``2.7.6``|``system``

<a anchor="patchelf"/>
### *patchelf*

PatchELF is a small utility to modify the dynamic linker and RPATH of ELF executables.

*homepage*: http://nixos.org/patchelf.html

version   |toolchain
----------|---------------------------------------------------------
``0.8``   |``GNU/4.9.3-2.25``
``0.9``   |``GCCcore/6.4.0``, ``foss/2016a``
``0.10``  |``GCCcore/7.2.0``, ``GCCcore/8.3.0``
``0.12``  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/9.3.0``
``0.13``  |``GCCcore/11.2.0``
``0.15.0``|``GCCcore/11.3.0``

<a anchor="path.py"/>
### *path.py*

path.py is a Python library implementing path objects as first-class entities,  allowing common operations on files to be invoked on those path objects directly.

*homepage*: https://github.com/jaraco/path.py

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``8.2.1``|``-Python-2.7.11``|``foss/2016a``
``8.2.1``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``8.2.1``|``-Python-3.5.1`` |``foss/2016a``
``8.2.1``|``-Python-3.5.2`` |``intel/2016b``
``10.1`` |``-Python-2.7.12``|``intel/2016b``

<a anchor="paup"/>
### *PAUP*

PAUP* (Phylogenetic Analysis Using Parsimony *and other methods) is a computational phylogenetics program for inferring evolutionary trees.

*homepage*: https://paup.phylosolutions.com/

version    |versionsuffix|toolchain
-----------|-------------|----------
``4.0a166``|``-centos64``|``system``
``4.0a168``|``-centos64``|``system``

<a anchor="pauvre"/>
### *pauvre*

Tools for plotting Oxford Nanopore and other long-read data

*homepage*: https://github.com/conchoecia/pauvre

version   |versionsuffix    |toolchain
----------|-----------------|---------------
``0.2.3`` |                 |``foss/2021a``
``0.1923``|``-Python-3.7.4``|``intel/2019b``
``0.1924``|                 |``intel/2020b``

<a anchor="pbbam"/>
### *pbbam*

The pbbam software package provides components to create, query, & edit PacBio BAM files  and associated indices.

*homepage*: http://pbbam.readthedocs.io/

version     |toolchain
------------|---------------
``1.0.6``   |``gompi/2019a``
``20170508``|``intel/2017a``

<a anchor="pbcopper"/>
### *pbcopper*

The pbcopper library provides a suite of data structures, algorithms,  and utilities for C++ applications.

*homepage*: https://github.com/PacificBiosciences/pbcopper

version  |toolchain
---------|---------------
``1.3.0``|``gompi/2019a``

<a anchor="pbdagcon"/>
### *pbdagcon*

pbdagcon is a tool that implements DAGCon (Directed Acyclic Graph Consensus) which is a sequence  consensus algorithm based on using directed acyclic graphs to encode multiple sequence alignment.

*homepage*: https://github.com/PacificBiosciences/pbdagcon

version     |toolchain
------------|---------------
``20170330``|``intel/2017a``

<a anchor="pbmm2"/>
### *pbmm2*

A minimap2 frontend for PacBio native data formats

*homepage*: https://github.com/PacificBiosciences/pbmm2

version  |toolchain
---------|---------------
``1.1.0``|``gompi/2019a``

<a anchor="pbs_python"/>
### *pbs_python*

The pbs_python package is a wrapper class for the Torque C library. With this package you now can  write utilities/extensions in Python instead of C. We developed this package because we want to replace xpbsmon by  an ascii version named pbsmon. PBSQuery is also included in this package. This is a python module build on top of  the pbs python module to simplify querying the batch server, eg: how many jobs, how many nodes, ...

*homepage*: https://oss.trac.surfsara.nl/pbs_python

version  |versionsuffix     |toolchain
---------|------------------|---------------
``4.6.0``|                  |``system``
``4.6.0``|``-Python-2.7.11``|``intel/2016a``
``4.6.0``|``-Python-2.7.12``|``intel/2016b``
``4.6.0``|``-Python-2.7.13``|``intel/2017a``

<a anchor="pbsuite"/>
### *PBSuite*

PBJelly is a highly automated pipeline that aligns long sequencing reads (such as PacBio RS reads or  long 454 reads in fasta format) to high-confidence draft assembles.

*homepage*: https://sourceforge.net/p/pb-jelly/wiki/Home/

version    |versionsuffix     |toolchain
-----------|------------------|---------------
``15.8.24``|``-Python-2.7.12``|``intel/2016b``

<a anchor="pcangsd"/>
### *PCAngsd*

PCAngsd, which estimates the covariance matrix for low depth  NGS data in an iterative procedure based on genotype likelihoods and is able  to perform multiple population genetic analyses in heterogeneous populations.

*homepage*: http://www.popgen.dk/software/index.php/PCAngsd

version |versionsuffix     |toolchain
--------|------------------|--------------
``0.97``|``-Python-2.7.14``|``foss/2018a``

<a anchor="pcc"/>
### *PCC*

The compiler is based on the original Portable C Compiler by S. C. Johnson,  written in the late 70's. About 50% of the frontend code and 80% of the backend code has been modified.

*homepage*: http://pcc.ludd.ltu.se/

version     |toolchain
------------|----------
``20131024``|``system``

<a anchor="pcl"/>
### *PCL*

The Point Cloud Library (PCL) is a standalone, large scale, open project for 2D/3D image and  point cloud processing.

*homepage*: http://pointclouds.org/

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.7.2``|``-Python-2.7.11``|``intel/2016a``
``1.8.1``|``-Python-2.7.14``|``intel/2017b``

<a anchor="pcmsolver"/>
### *PCMSolver*

An API for the Polarizable Continuum Model.

*homepage*: https://pcmsolver.readthedocs.org

version     |versionsuffix     |toolchain
------------|------------------|---------------
``1.1.4``   |``-Python-2.7.11``|``intel/2016a``
``1.2.3``   |``-Python-3.6.6`` |``foss/2018b``
``1.2.3``   |``-Python-3.7.2`` |``gompi/2019a``
``1.2.3``   |``-Python-3.7.4`` |``iimpi/2019b``
``20160205``|``-Python-2.7.11``|``intel/2016a``

<a anchor="pcraster"/>
### *PCRaster*

PCRaster Is a collection of software targeted at the development  and deployment of spatio-temporal environmental models.

*homepage*: http://pcraster.geo.uu.nl/

version  |versionsuffix     |toolchain
---------|------------------|---------------
``4.1.0``|``-Python-2.7.14``|``intel/2017b``

<a anchor="pcre"/>
### *PCRE*

The PCRE library is a set of functions that implement regular expression  pattern matching using the same syntax and semantics as Perl 5. 

*homepage*: https://www.pcre.org/

version |toolchain
--------|----------------------------------------------------------------------------------
``8.38``|``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``
``8.39``|``GCCcore/5.4.0``, ``foss/2016b``, ``intel/2016b``
``8.40``|``GCCcore/6.3.0``, ``gimkl/2017a``, ``intel/2016b``, ``intel/2017a``
``8.41``|``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``8.42``|``GCCcore/6.4.0``
``8.43``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``8.44``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/9.3.0``
``8.45``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="pcre2"/>
### *PCRE2*

The PCRE library is a set of functions that implement regular expression pattern matching using the same syntax  and semantics as Perl 5. 

*homepage*: https://www.pcre.org/

version  |toolchain
---------|--------------------------------------
``10.21``|``foss/2016a``
``10.31``|``foss/2018b``
``10.33``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``10.34``|``GCCcore/9.3.0``
``10.35``|``GCCcore/10.2.0``
``10.36``|``GCCcore/10.3.0``
``10.37``|``GCCcore/11.2.0``
``10.40``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``

<a anchor="pdt"/>
### *PDT*

Program Database Toolkit (PDT) is a framework for analyzing source code  written in several programming languages and for making rich program  knowledge accessible to developers of static and dynamic analysis tools.  PDT implements a standard program representation, the program database  (PDB), that can be accessed in a uniform way through a class library  supporting common PDB operations. 

*homepage*: https://www.cs.uoregon.edu/research/pdt/

version   |toolchain
----------|--------------------------------------------------------
``3.22``  |``foss/2016a``
``3.25``  |``GCCcore/10.2.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``3.25.1``|``GCCcore/10.3.0``, ``GCCcore/9.3.0``

<a anchor="pear"/>
### *PEAR*

PEAR is an ultrafast, memory-efficient and highly accurate pair-end read merger.   It is fully parallelized and can run with as low as just a few kilobytes of memory.

*homepage*: http://sco.h-its.org/exelixis/web/software/pear/

version   |toolchain
----------|------------------------------------------------------------------------------------
``0.9.8`` |``foss/2016b``, ``intel/2016b``
``0.9.10``|``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``
``0.9.11``|``GCC/10.3.0``, ``GCC/11.2.0``, ``GCCcore/7.3.0``, ``GCCcore/9.3.0``, ``foss/2018a``

<a anchor="penncnv"/>
### *PennCNV*

A free software tool for Copy Number Variation (CNV) detection from SNP genotyping arrays. Currently  it can handle signal intensity data from Illumina and Affymetrix arrays. With appropriate preparation of file format,  it can also handle other types of SNP arrays and oligonucleotide arrays.

*homepage*: https://penncnv.openbioinformatics.org/

version  |toolchain
---------|-----------------
``1.0.5``|``GCCcore/8.3.0``

<a anchor="pept"/>
### *PEPT*

A Python library that integrates all the tools necessary to perform research using Positron Emission  Particle Tracking (PEPT). The library includes algorithms for the location, identification and tracking of particles,  in addition to tools for visualisation and analysis, and utilities allowing the realistic simulation of PEPT data.

*homepage*: https://pept.readthedocs.io/en/latest/

version  |toolchain
---------|--------------
``0.4.1``|``foss/2021a``

<a anchor="percolator"/>
### *Percolator*

Semi-supervised learning for peptide identification from shotgun proteomics datasets

*homepage*: https://github.com/percolator/percolator

version|toolchain
-------|---------------
``3.4``|``gompi/2019a``

<a anchor="perl"/>
### *Perl*

Larry Wall's Practical Extraction and Report Language

*homepage*: https://www.perl.org/

version   |versionsuffix|toolchain
----------|-------------|-----------------------------------------------------------------------------------------
``5.20.1``|``-bare``    |``GCC/4.8.2``, ``GCC/4.9.2``
``5.20.2``|``-bare``    |``GCC/4.9.2``
``5.20.3``|             |``foss/2016a``, ``intel/2016a``
``5.22.0``|``-bare``    |``GCC/4.9.2``
``5.22.1``|             |``foss/2016a``, ``foss/2016b``, ``intel/2016a``
``5.22.1``|``-bare``    |``foss/2016a``, ``intel/2016a``
``5.22.2``|             |``intel/2016a``
``5.24.0``|             |``GCC/5.4.0-2.26``, ``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``foss/2016b``, ``intel/2016b``
``5.24.0``|``-bare``    |``foss/2016b``
``5.24.1``|             |``GCCcore/6.3.0``, ``foss/2017a``, ``intel/2017a``
``5.26.0``|             |``GCCcore/6.4.0``, ``foss/2017b``, ``intel/2017b``, ``intel/2018.00``, ``intel/2018.01``
``5.26.1``|             |``GCCcore/6.4.0``, ``foss/2018a``
``5.26.1``|``-bare``    |``foss/2018a``
``5.28.0``|             |``GCCcore/7.3.0``
``5.28.1``|             |``GCCcore/8.2.0``
``5.30.0``|             |``GCCcore/8.3.0``
``5.30.0``|``-minimal`` |``GCCcore/8.3.0``
``5.30.2``|             |``GCCcore/9.3.0``
``5.30.2``|``-minimal`` |``GCCcore/9.3.0``
``5.32.0``|             |``GCCcore/10.2.0``
``5.32.0``|``-minimal`` |``GCCcore/10.2.0``
``5.32.1``|             |``FCC/4.5.0``, ``GCCcore/10.3.0``
``5.32.1``|``-minimal`` |``GCCcore/10.3.0``
``5.34.0``|             |``GCCcore/11.2.0``
``5.34.0``|``-minimal`` |``GCCcore/11.2.0``
``5.34.1``|             |``GCCcore/11.3.0``
``5.34.1``|``-minimal`` |``GCCcore/11.3.0``
``5.36.0``|             |``GCCcore/12.1.0``, ``GCCcore/12.2.0``

<a anchor="perl-app-cpanminus"/>
### *perl-app-cpanminus*

cpanm - get, unpack build and install modules from CPAN 

*homepage*: https://github.com/miyagawa/cpanminus

version   |toolchain
----------|----------
``1.7039``|``system``

<a anchor="perl4-corelibs"/>
### *Perl4-CoreLibs*

Libraries historically supplied with Perl 4

*homepage*: https://metacpan.org/pod/Perl4::CoreLibs

version  |versionsuffix   |toolchain
---------|----------------|---------------
``0.003``|``-Perl-5.24.1``|``intel/2017a``

<a anchor="pest++"/>
### *PEST++*

PEST++ is a software suite aimed at supporting  complex numerical models in the decision-support context.  Much focus has been devoted to supporting environmental models  (groundwater, surface water, etc) but these tools are readily  applicable to any computer model. 

*homepage*: https://github.com/usgs/pestpp

version  |toolchain
---------|--------------
``5.0.5``|``foss/2020a``

<a anchor="petsc"/>
### *PETSc*

PETSc, pronounced PET-see (the S is silent), is a suite of data structures and routines for the  scalable (parallel) solution of scientific applications modeled by partial differential equations.

*homepage*: http://www.mcs.anl.gov/petsc

version   |versionsuffix       |toolchain
----------|--------------------|-------------------------------
``3.7.2`` |``-Python-2.7.11``  |``intel/2016a``
``3.7.3`` |``-Python-2.7.11``  |``foss/2016a``
``3.7.5`` |``-downloaded-deps``|``intel/2016b``
``3.8.3`` |``-downloaded-deps``|``foss/2017b``
``3.9.1`` |``-downloaded-deps``|``foss/2018a``
``3.9.3`` |                    |``foss/2018a``, ``intel/2018a``
``3.11.0``|``-downloaded-deps``|``foss/2018b``
``3.11.1``|``-Python-3.7.2``   |``foss/2019a``, ``intel/2019a``
``3.12.4``|``-Python-2.7.16``  |``intel/2019b``
``3.12.4``|``-Python-3.7.4``   |``foss/2019b``, ``intel/2019b``
``3.12.4``|``-Python-3.8.2``   |``foss/2020a``, ``intel/2020a``
``3.14.4``|                    |``foss/2020b``, ``intel/2020b``
``3.15.1``|                    |``foss/2021a``
``3.17.4``|                    |``foss/2022a``

<a anchor="petsc4py"/>
### *petsc4py*

petsc4py are Python bindings for PETSc, the Portable, Extensible Toolchain for Scientific Computation.

*homepage*: https://bitbucket.org/petsc/petsc4py

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``3.9.1`` |``-Python-3.6.4``|``foss/2018a``
``3.12.0``|``-Python-3.7.4``|``foss/2019b``
``3.15.0``|                 |``foss/2021a``

<a anchor="pfind"/>
### *pfind*

Drop-in replacement for find, implemented for using parallel access and MPI.

*homepage*: https://github.com/VI4IO/pfind

version     |toolchain
------------|---------------
``20220613``|``gompi/2021b``

<a anchor="pftoolsv3"/>
### *pftoolsV3*

A suite of tools to build and search generalized profiles (protein and DNA).

*homepage*: https://github.com/sib-swiss/pftools3

version     |toolchain
------------|----------------------------------
``3.2.11``  |``GCCcore/10.3.0``, ``foss/2021a``
``3.2.12``  |``GCCcore/11.2.0``
``20160324``|``foss/2016a``

<a anchor="pfunit"/>
### *pFUnit*

pFUnit is a unit testing framework enabling JUnit-like testing of serial  and MPI-parallel software written in Fortran.

*homepage*: https://github.com/Goddard-Fortran-Ecosystem/pFUnit

version  |toolchain
---------|---------------
``3.2.9``|``gompi/2018b``
``4.2.0``|``gompi/2020b``

<a anchor="pgdspider"/>
### *PGDSpider*

An automated data conversion tool for connecting population genetics and genomics programs

*homepage*: http://cmpg.unibe.ch/software/PGDSpider/

version    |versionsuffix     |toolchain
-----------|------------------|----------
``2.1.0.3``|``-Java-1.7.0_80``|``system``

<a anchor="pgi"/>
### *PGI*

C, C++ and Fortran compilers from The Portland Group - PGI

*homepage*: https://www.pgroup.com/

version  |versionsuffix          |toolchain
---------|-----------------------|----------
``15.7`` |``-GNU-4.9.2-2.25``    |``system``
``15.7`` |``-GNU-4.9.3-2.25``    |``system``
``15.10``|``-GCC-4.9.3-2.25``    |``system``
``16.1`` |``-CDK-GCC-4.9.2-2.25``|``system``
``16.3`` |``-GCC-4.9.3-2.25``    |``system``
``16.4`` |``-GCC-5.3.0-2.26``    |``system``
``16.7`` |``-GCC-5.4.0-2.26``    |``system``
``16.10``|``-GCC-5.4.0-2.26``    |``system``
``17.1`` |``-GCC-6.3.0-2.27``    |``system``
``17.3`` |``-GCC-6.3.0-2.28``    |``system``
``17.4`` |``-GCC-6.4.0-2.28``    |``system``
``17.10``|``-GCC-6.4.0-2.28``    |``system``
``18.1`` |``-GCC-7.2.0-2.29``    |``system``
``18.4`` |``-GCC-6.4.0-2.28``    |``system``
``18.7`` |``-GCC-7.3.0-2.30``    |``system``
``18.10``|``-GCC-6.4.0-2.28``    |``system``
``19.1`` |``-GCC-8.2.0-2.31.1``  |``system``
``19.4`` |``-GCC-8.2.0-2.31.1``  |``system``
``19.7`` |``-GCC-8.3.0-2.32``    |``system``
``19.10``|``-GCC-8.3.0-2.32``    |``system``

<a anchor="phanotate"/>
### *PHANOTATE*

PHANOTATE: a tool to annotate phage genomes

*homepage*: https://github.com/deprekate/PHANOTATE

version     |toolchain
------------|--------------
``20190724``|``foss/2018b``

<a anchor="phantompeakqualtools"/>
### *Phantompeakqualtools*

It computes informative enrichment and quality measures for ChIP-seq/DNase-seq/FAIRE-seq/MNase-seq data.

*homepage*: https://github.com/kundajelab/phantompeakqualtools

version  |toolchain
---------|--------------
``1.2.2``|``foss/2021b``

<a anchor="phase"/>
### *PHASE*

The program PHASE implements a Bayesian statistical  method for reconstructing haplotypes from population genotype data. Documentation: http://stephenslab.uchicago.edu/assets/software/phase/instruct2.1.pdf

*homepage*: http://stephenslab.uchicago.edu/phase/download.html

version  |toolchain
---------|----------
``2.1.1``|``system``

<a anchor="phast"/>
### *PHAST*

PHAST is a freely available software package for comparative and evolutionary genomics.

*homepage*: http://compgen.cshl.edu/phast/

version|toolchain
-------|----------------------------------------------------------
``1.4``|``intel/2017a``
``1.5``|``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``

<a anchor="phewas"/>
### *PheWAS*

Provides an accessible R interface to the phenome wide association study.

*homepage*: https://www.vumc.org/cpm/center-precision-medicine-blog/phewas-r-package

version     |versionsuffix|toolchain
------------|-------------|-------------------------------
``0.12``    |``-R-3.3.3`` |``foss/2016b``, ``intel/2016b``
``0.99.5-2``|``-R-3.6.0`` |``foss/2019a``, ``intel/2019a``

<a anchor="phipack"/>
### *PhiPack*

The PhiPack software package implements (in C) a few tests for recombination and can produce refined incompatibility matrices as well.

*homepage*: http://www.maths.otago.ac.nz/~dbryant/software.html

version       |toolchain
--------------|----------------------------------------
``2016.06.14``|``iccifort/2019.1.144-GCC-8.2.0-2.31.1``

<a anchor="phlat"/>
### *PHLAT*

PHLAT is a bioinformatics algorithm that offers HLA typing at four-digit resolution (or higher)  using genome-wide transcriptome and exome sequencing data over a wide range of read lengths and sequencing depths.

*homepage*: https://sites.google.com/site/phlatfortype/

version|versionsuffix     |toolchain
-------|------------------|--------------
``1.1``|``-Python-2.7.15``|``foss/2018b``

<a anchor="phonemizer"/>
### *phonemizer*

The phonemizer allows simple phonemization of words and texts in many languages. Provides both the phonemize command-line tool and the Python function phonemizer.phonemize. It is using four backends: espeak, espeak-mbrola, festival and segments. 

*homepage*: https://github.com/bootphon/phonemizer

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``2.2.1``|``-Python-3.8.2``|``gompi/2020a``

<a anchor="phono3py"/>
### *phono3py*

phono3py calculates phonon-phonon interaction and related properties using the supercell approach.

*homepage*: https://atztogo.github.io/phono3py/

version      |versionsuffix     |toolchain
-------------|------------------|-------------------------------
``1.12.5.35``|``-Python-2.7.14``|``intel/2017b``
``1.12.7.55``|``-Python-2.7.14``|``foss/2018a``, ``intel/2018a``

<a anchor="phonopy"/>
### *phonopy*

Phonopy is an open source package of phonon calculations based on the supercell approach.

*homepage*: https://atztogo.github.io/phonopy/

version      |versionsuffix     |toolchain
-------------|------------------|-------------------------------
``1.10.1``   |``-Python-2.7.11``|``intel/2016a``
``1.12.2.20``|``-Python-2.7.14``|``intel/2017b``
``1.12.6.66``|``-Python-2.7.14``|``foss/2018a``, ``intel/2018a``
``1.13.0.64``|``-Python-2.7.14``|``intel/2018a``
``1.14.2``   |``-Python-2.7.15``|``intel/2018b``
``2.0.0``    |``-Python-2.7.14``|``intel/2018a``
``2.2.0``    |``-Python-3.7.2`` |``intel/2019a``
``2.7.1``    |``-Python-3.7.4`` |``intel/2019b``
``2.7.1``    |``-Python-3.8.2`` |``intel/2020a``
``2.12.0``   |                  |``foss/2020b``
``2.16.3``   |                  |``foss/2022a``

<a anchor="photontorch"/>
### *photontorch*

Photontorch is a photonic simulator for highly parallel simulation and optimization of photonic circuits in time and frequency domain. Photontorch features CUDA enabled simulation and optimization of photonic circuits. It leverages the deep learning framework PyTorch to view the photonic circuit as essentially a recurrent neural network. This enables the use of native PyTorch optimizers to optimize the (physical) parameters of the circuit. 

*homepage*: https://docs.photontorch.com/

version  |toolchain
---------|----------------------------------
``0.4.1``|``foss/2020b``, ``fosscuda/2020b``

<a anchor="phototonic"/>
### *phototonic*

Phototonic is an image viewer and organizer

*homepage*: https://github.com/oferkv/phototonic

version|toolchain
-------|------------------
``2.1``|``GCCcore/10.3.0``

<a anchor="phylip"/>
### *PHYLIP*

PHYLIP is a free package of programs for inferring phylogenies.

*homepage*: http://evolution.genetics.washington.edu/phylip

version  |toolchain
---------|-------------------------------------------------------------------------
``3.696``|``foss/2016a``, ``intel/2016a``
``3.697``|``GCC/6.4.0-2.28``, ``GCC/9.3.0``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``

<a anchor="phylobayes-mpi"/>
### *PhyloBayes-MPI*

A Bayesian software for phylogenetic reconstruction using mixture models

*homepage*: https://github.com/bayesiancook/pbmpi

version     |toolchain
------------|---------------
``20161021``|``intel/2016b``

<a anchor="phylokit"/>
### *phylokit*

C++ library for high performance phylogenetics

*homepage*: https://github.com/pranjalv123/phylokit

version|toolchain
-------|--------------------
``1.0``|``GCC/8.2.0-2.31.1``

<a anchor="phylonaut"/>
### *phylonaut*

Dynamic programming for phylogenetics applications

*homepage*: https://github.com/pranjalv123/phylonaut

version     |toolchain
------------|---------------
``20190626``|``gompi/2019a``

<a anchor="phylophlan"/>
### *PhyloPhlAn*

PhyloPhlAn is an integrated pipeline for large-scale phylogenetic profiling of genomes and metagenomes. PhyloPhlAn is an accurate, rapid, and easy-to-use method for large-scale microbial genome characterization and phylogenetic analysis at multiple levels of resolution. 

*homepage*: https://github.com/biobakery/phylophlan

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``3.0``  |``-Python-3.8.2``|``foss/2020a``
``3.0.2``|                 |``foss/2021a``

<a anchor="phyml"/>
### *PhyML*

Phylogenetic estimation using (Maximum) Likelihood

*homepage*: https://github.com/stephaneguindon/phyml

version         |toolchain
----------------|--------------
``3.3.20190321``|``foss/2018b``
``3.3.20200621``|``foss/2020b``

<a anchor="phyx"/>
### *phyx*

phyx performs phylogenetics analyses on trees and sequences.

*homepage*: https://github.com/FePhyFoFum/phyx

version |toolchain
--------|--------------
``1.01``|``foss/2019a``

<a anchor="picard"/>
### *picard*

A set of tools (in Java) for working with next generation sequencing data in the BAM format.

*homepage*: https://sourceforge.net/projects/picard

version    |versionsuffix      |toolchain
-----------|-------------------|----------
``1.39``   |                   |``system``
``1.100``  |                   |``system``
``1.109``  |                   |``system``
``1.119``  |                   |``system``
``1.119``  |``-Java-1.7.0_80`` |``system``
``1.120``  |``-Java-1.8.0_66`` |``system``
``1.141``  |``-Java-1.8.0_74`` |``system``
``2.0.1``  |``-Java-1.8.0_66`` |``system``
``2.1.0``  |                   |``system``
``2.1.0``  |``-Java-1.8.0_74`` |``system``
``2.1.1``  |``-Java-1.8.0_112``|``system``
``2.1.1``  |``-Java-1.8.0_74`` |``system``
``2.2.4``  |``-Java-1.8.0_92`` |``system``
``2.6.0``  |``-Java-1.8.0_131``|``system``
``2.10.1`` |``-Java-1.8.0_131``|``system``
``2.18.5`` |``-Java-1.8.0_162``|``system``
``2.18.11``|``-Java-1.8.0_162``|``system``
``2.18.14``|``-Java-1.8``      |``system``
``2.18.17``|``-Java-1.8``      |``system``
``2.18.27``|``-Java-1.8``      |``system``
``2.20.6`` |``-Java-1.8``      |``system``
``2.21.1`` |``-Java-11``       |``system``
``2.21.6`` |``-Java-11``       |``system``
``2.22.1`` |``-Java-11``       |``system``
``2.25.0`` |``-Java-11``       |``system``
``2.25.1`` |``-Java-11``       |``system``
``2.25.5`` |``-Java-13``       |``system``
``2.26.10``|``-Java-15``       |``system``

<a anchor="pigz"/>
### *pigz*

pigz, which stands for parallel implementation of gzip, is a fully  functional replacement for gzip that exploits multiple processors and multiple  cores to the hilt when compressing data. pigz was written by Mark Adler, and  uses the zlib and pthread libraries. 

*homepage*: https://zlib.net/pigz/

version  |toolchain
---------|---------------------------------------------------------------------------------------------------------------------------------
``2.3.3``|``foss/2016b``
``2.3.4``|``GCCcore/6.4.0``
``2.4``  |``GCCcore/10.2.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``foss/2018a``
``2.6``  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``2.7``  |``GCCcore/11.3.0``

<a anchor="pil"/>
### *PIL*

The Python Imaging Library (PIL) adds image processing capabilities to your Python interpreter.  This library supports many file formats, and provides powerful image processing and graphics capabilities.

*homepage*: http://www.pythonware.com/products/pil

version  |versionsuffix                    |toolchain
---------|---------------------------------|-------------------------------
``1.1.7``|``-Python-2.7.11``               |``intel/2016a``
``1.1.7``|``-Python-2.7.11-freetype-2.6.3``|``intel/2016a``
``1.1.7``|``-Python-2.7.12``               |``foss/2016b``, ``intel/2016b``
``1.1.7``|``-Python-2.7.13``               |``intel/2017a``
``1.1.7``|``-Python-2.7.15``               |``foss/2019a``

<a anchor="pileometh"/>
### *PileOMeth*

PileOMeth processes a coordinate-sorted and indexed BAM or CRAM file containing some form of BS-seq alignments.  PileOMeth extracts per-base methylation metrics from them.  PileOMeth requires an indexed fasta file containing the reference genome as well. 

*homepage*: https://github.com/dpryan79/PileOMeth

version   |toolchain
----------|--------------
``0.1.11``|``foss/2016b``

<a anchor="pillow"/>
### *Pillow*

Pillow is the 'friendly PIL fork' by Alex Clark and Contributors.  PIL is the Python Imaging Library by Fredrik Lundh and Contributors.

*homepage*: https://pillow.readthedocs.org/

version  |versionsuffix                    |toolchain
---------|---------------------------------|---------------------------------------------------
``3.2.0``|``-Python-2.7.11``               |``intel/2016a``
``3.2.0``|``-Python-2.7.11-freetype-2.6.3``|``foss/2016a``, ``intel/2016a``
``3.4.2``|``-Python-2.7.12``               |``intel/2016b``
``3.4.2``|``-Python-3.5.2``                |``foss/2016b``, ``intel/2016b``
``3.4.2``|``-Python-3.5.2-freetype-2.6.5`` |``intel/2016b``
``4.1.0``|``-Python-2.7.13``               |``intel/2017a``
``4.1.1``|``-Python-3.6.1``                |``intel/2017a``
``4.2.1``|``-Python-3.6.1``                |``intel/2017a``
``4.3.0``|``-Python-2.7.13``               |``intel/2017a``
``4.3.0``|``-Python-2.7.14``               |``intel/2017b``
``4.3.0``|``-Python-3.6.3``                |``foss/2017b``
``5.0.0``|``-Python-2.7.14``               |``foss/2017b``, ``intel/2017b``, ``intel/2018a``
``5.0.0``|``-Python-3.6.3``                |``foss/2017b``, ``intel/2017b``
``5.0.0``|``-Python-3.6.4``                |``foss/2018a``, ``intel/2018a``
``5.3.0``|``-Python-2.7.15``               |``foss/2018b``
``5.3.0``|``-Python-3.6.6``                |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``6.0.0``|                                 |``GCCcore/8.2.0``
``6.2.1``|                                 |``GCCcore/8.3.0``
``6.2.2``|``-Python-2.7.18``               |``foss/2020b``
``7.0.0``|``-Python-3.8.2``                |``GCCcore/9.3.0``
``8.0.1``|                                 |``GCCcore/10.2.0``
``8.2.0``|                                 |``GCCcore/10.3.0``
``8.3.1``|                                 |``GCCcore/11.2.0``
``8.3.2``|                                 |``GCCcore/11.2.0``
``9.1.0``|                                 |``GCCcore/10.3.0``
``9.1.1``|                                 |``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``9.2.0``|                                 |``GCCcore/10.2.0``

<a anchor="pillow-simd"/>
### *Pillow-SIMD*

Pillow is the 'friendly PIL fork' by Alex Clark and Contributors.  PIL is the Python Imaging Library by Fredrik Lundh and Contributors.

*homepage*: https://github.com/uploadcare/pillow-simd

version        |versionsuffix    |toolchain
---------------|-----------------|---------------------------------------------------
``5.0.0``      |``-Python-3.6.4``|``foss/2018a``, ``intel/2018a``
``5.3.0.post0``|``-Python-3.6.6``|``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``6.0.x.post0``|                 |``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``7.1.2``      |                 |``GCCcore/10.2.0``
``7.1.2``      |``-Python-3.8.2``|``GCCcore/9.3.0``
``8.2.0``      |                 |``GCCcore/10.3.0``
``8.3.1``      |                 |``GCCcore/11.2.0``
``8.3.2``      |                 |``GCCcore/11.2.0``
``9.2.0``      |                 |``GCCcore/11.3.0``

<a anchor="pilon"/>
### *Pilon*

Pilon is an automated genome assembly improvement and variant detection tool

*homepage*: https://github.com/broadinstitute/pilon

version |versionsuffix      |toolchain
--------|-------------------|----------
``1.22``|``-Java-1.8``      |``system``
``1.22``|``-Java-1.8.0_162``|``system``
``1.23``|``-Java-1.8``      |``system``
``1.23``|``-Java-11``       |``system``

<a anchor="pims"/>
### *PIMS*

PIMS is a lazy-loading interface to sequential data with numpy-like slicing.

*homepage*: https://soft-matter.github.io/pims

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.4.1``|``-Python-2.7.14``|``intel/2017b``

<a anchor="pindel"/>
### *Pindel*

Pindel can detect breakpoints of large deletions, medium sized  insertions, inversions, tandem duplications and other structural variants at single-based  resolution from next-gen sequence data. It uses a pattern growth approach to identify the  breakpoints of these variants from paired-end short reads. 

*homepage*: http://gmt.genome.wustl.edu/packages/pindel/

version             |toolchain
--------------------|--------------------------------------------------
``0.2.5b8``         |``foss/2016b``
``0.2.5b9-20170508``|``GCC/11.2.0``, ``GCC/11.3.0``, ``GCC/6.4.0-2.28``

<a anchor="pingouin"/>
### *Pingouin*

Pingouin is an open-source statistical package written in Python 3 and based mostly on Pandas and  NumPy.

*homepage*: https://pingouin-stats.org/

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.3.8``|``-Python-3.7.4``|``foss/2019b``

<a anchor="pint"/>
### *Pint*

Pint is a Python package to define, operate and manipulate physical quantities: the product of a numerical value and a unit of measurement. It allows arithmetic operations between them and conversions from and to different units.

*homepage*: https://github.com/hgrecco/pint

version   |versionsuffix    |toolchain
----------|-----------------|------------------
``0.14``  |``-Python-3.7.4``|``GCCcore/8.3.0``
``0.19.2``|                 |``GCCcore/11.2.0``

<a anchor="pip"/>
### *pip*

The PyPA recommended tool for installing Python packages.

*homepage*: https://pip.pypa.io

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``8.0.2``|``-Python-2.7.11``|``intel/2016a``
``8.1.2``|``-Python-2.7.11``|``foss/2016a``
``8.1.2``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``

<a anchor="pipits"/>
### *PIPITS*

An automated pipeline for analyses of fungal internal transcribed spacer (ITS) sequences  from the Illumina sequencing platform.

*homepage*: https://github.com/hsgweon/pipits

version|versionsuffix    |toolchain
-------|-----------------|--------------
``2.7``|``-Python-3.8.2``|``foss/2020a``
``2.8``|                 |``foss/2021a``

<a anchor="pirs"/>
### *pIRS*

pIRS (profile based Illumina pair-end Reads Simulator) is a program for simulating paired-end reads from a reference genome. It is optimized for simulating reads similar to those generated from the Illumina platform.

*homepage*: https://github.com/galaxy001/pirs

version  |toolchain
---------|---------------
``2.0.2``|``gompi/2019b``

<a anchor="pisces"/>
### *Pisces*

Somatic and germline variant caller for amplicon data. Recommended caller for tumor-only workflows.

*homepage*: https://github.com/Illumina/Pisces

version     |toolchain
------------|-----------------
``5.2.7.47``|``GCCcore/6.4.0``

<a anchor="pisvm"/>
### *piSvM*

piSvM is a parallel implementation of the Support Vector Machine (SVM) algorithm that allows  efficient training and testing on a multiprocessor system.

*homepage*: http://pisvm.sourceforge.net/

version|toolchain
-------|---------------
``1.3``|``intel/2017b``

<a anchor="pisvm-jsc"/>
### *piSvM-JSC*

piSvM is a parallel implementation of the Support Vector Machine (SVM) algorithm that allows  efficient training and testing on a multiprocessor system. This version is a fork of the original PiSvM to increase scalability.

*homepage*: https://github.com/mricherzhagen/pisvm

version         |toolchain
----------------|---------------
``1.2-20150622``|``intel/2017b``

<a anchor="pixman"/>
### *pixman*

Pixman is a low-level software library for pixel manipulation, providing  features such as image compositing and trapezoid rasterization. Important  users of pixman are the cairo graphics library and the X server. 

*homepage*: http://www.pixman.org/

version   |toolchain
----------|--------------------------------------------------------------------------------------------------------------------------------------------
``0.34.0``|``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``0.38.0``|``GCCcore/8.2.0``
``0.38.4``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``0.40.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``0.42.2``|``GCCcore/12.2.0``

<a anchor="pizzly"/>
### *pizzly*

Pizzly is a program for detecting gene fusions from RNA-Seq data of cancer samples.

*homepage*: https://github.com/pmelsted/pizzly

version   |toolchain
----------|--------------
``0.37.3``|``foss/2018b``

<a anchor="pkg-config"/>
### *pkg-config*

pkg-config is a helper tool used when compiling applications and libraries.  It helps you insert the correct compiler options on the command line so an  application can use gcc -o test test.c `pkg-config --libs --cflags glib-2.0`  for instance, rather than hard-coding values on where to find glib (or other  libraries). 

*homepage*: http://www.freedesktop.org/wiki/Software/pkg-config/

version   |toolchain
----------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``0.28``  |``GCC/4.8.2``, ``GCC/4.9.2``, ``GNU/4.9.3-2.25``
``0.29``  |``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``0.29.1``|``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``gimkl/2017a``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``
``0.29.2``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``intel/2017a``, ``system``

<a anchor="pkgconf"/>
### *pkgconf*

pkgconf is a program which helps to configure compiler and linker flags for development libraries.  It is similar to pkg-config from freedesktop.org.

*homepage*: https://github.com/pkgconf/pkgconf

version  |toolchain
---------|--------------------------------------------------
``1.8.0``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``system``
``1.9.3``|``GCCcore/12.2.0``

<a anchor="pkgconfig"/>
### *pkgconfig*

pkgconfig is a Python module to interface with the pkg-config command line tool

*homepage*: https://github.com/matze/pkgconfig

version  |versionsuffix     |toolchain
---------|------------------|---------------------------------------------------
``1.1.0``|``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``1.1.0``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``1.1.0``|``-Python-3.5.1`` |``foss/2016a``
``1.1.0``|``-Python-3.5.2`` |``foss/2016b``, ``intel/2016b``
``1.2.2``|``-Python-2.7.13``|``foss/2017a``, ``intel/2017a``
``1.2.2``|``-Python-2.7.14``|``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``
``1.2.2``|``-Python-3.6.1`` |``foss/2017a``, ``intel/2017a``
``1.2.2``|``-Python-3.6.3`` |``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``
``1.3.1``|``-Python-2.7.14``|``foss/2018a``, ``intel/2018a``
``1.3.1``|``-Python-2.7.15``|``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``1.3.1``|``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``1.3.1``|``-Python-3.6.6`` |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``1.5.1``|``-Python-3.7.4`` |``GCCcore/8.3.0``
``1.5.1``|``-Python-3.8.2`` |``GCCcore/9.3.0``
``1.5.1``|``-python``       |``GCCcore/10.2.0``, ``GCCcore/8.2.0``
``1.5.4``|``-python``       |``GCCcore/10.3.0``
``1.5.5``|``-python``       |``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="plantcv"/>
### *plantcv*

PlantCV: Plant phenotyping using computer vision.

*homepage*: https://pypi.org/project/plantcv/

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``3.8.0``|``-Python-3.8.2``|``foss/2020a``

<a anchor="plascope"/>
### *PlaScope*

Plasmid exploration of bacterial genomes

*homepage*: https://github.com/GuilhemRoyer/PlaScope

version  |toolchain
---------|--------------
``1.3.1``|``foss/2018b``

<a anchor="plasmapy"/>
### *PlasmaPy*

Open source Python ecosystem for plasma research and education

*homepage*: https://www.plasmapy.org

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.3.1``|``-Python-3.8.2``|``foss/2020a``

<a anchor="plast"/>
### *PLAST*

PLAST is a parallel alignment search tool for comparing large protein banks

*homepage*: http://www.irisa.fr/symbiose/projects/plast/

version  |versionsuffix     |toolchain
---------|------------------|--------------
``2.3.1``|``-Java-1.8.0_92``|``foss/2016a``

<a anchor="platanus"/>
### *Platanus*

PLATform for Assembling NUcleotide Sequences

*homepage*: http://platanus.bio.titech.ac.jp/

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.2.1``|``-linux-x86_64``|``system``
``1.2.4``|                 |``foss/2017a``

<a anchor="platypus"/>
### *Platypus*

Platypus is a tool designed for efficient and accurate variant-detection  in high-throughput sequencing data.

*homepage*: http://www.well.ox.ac.uk/platypus

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.8.1``|``-Python-2.7.11``|``intel/2016a``

<a anchor="plc"/>
### *plc*

plc is the public Planck Likelihood Code.  It provides C and Fortran libraries that allow   users to compute the log likelihoods of the temperature,   polarization, and lensing maps.  Optionally, it also provides a python version of this library,   as well as tools to modify the predetermined options for some likelihoods   (e.g. changing the high-ell and low-ell lmin and lmax values of the temperature). 

*homepage*: http://pla.esac.esa.int/pla/#home

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``3.0.1``|``-Python-2.7.15``|``foss/2019a``, ``intel/2018b``
``3.0.1``|``-Python-3.7.4`` |``foss/2019b``

<a anchor="plink"/>
### *PLINK*

Whole-genome association analysis toolset

*homepage*: https://www.cog-genomics.org/plink/2.0/

version                    |toolchain
---------------------------|------------------------------
``1.07``                   |``foss/2016a``, ``foss/2016b``
``1.07-x86_64``            |``system``
``1.9b5``                  |``golf/2020a``
``1.9b_4.1-x86_64``        |``system``
``1.9b_6.17-x86_64``       |``system``
``1.9b_6.21-x86_64``       |``system``
``2.00-alpha1-x86_64``     |``system``
``2.00-alpha2-x86_64``     |``system``
``2.00-alpha2-x86_64_avx2``|``system``
``2.00a2.3``               |``GCC/10.3.0``, ``GCC/11.2.0``
``2.00a2.3_x86_64``        |``system``
``2.00a3.1``               |``GCC/11.2.0``
``2.00a3.6``               |``GCC/11.3.0``

<a anchor="plinkqc"/>
### *plinkQC*

plinkQC is a R/CRAN package for genotype quality control in genetic association studies.  It makes PLINK basic statistics (e.g.missing genotyping rates per individual, allele frequencies per genetic  marker) and relationship functions easily accessible from within R and allows for automatic evaluation of the  results.

*homepage*: https://meyer-lab-cshl.github.io/plinkQC/

version  |versionsuffix|toolchain
---------|-------------|--------------
``0.3.3``|``-R-4.0.0`` |``foss/2020a``

<a anchor="plinkseq"/>
### *PLINKSEQ*

PLINK/SEQ is an open-source C/C++ library for working with human   genetic variation data. The specific focus is to provide a platform for analytic tool   development for variation data from large-scale resequencing and genotyping projects,   particularly whole-exome and whole-genome studies. It is independent of (but designed   to be complementary to) the existing PLINK package. 

*homepage*: https://atgu.mgh.harvard.edu/plinkseq/

version |toolchain
--------|----------------------------------------------------------
``0.10``|``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``

<a anchor="ploticus"/>
### *Ploticus*

Ploticus is a free GPL software utility that can produce various types of plots and graphs

*homepage*: http://ploticus.sourceforge.net/doc/welcome.html

version |toolchain
--------|-----------------
``2.42``|``GCCcore/7.3.0``

<a anchor="plotly"/>
### *plotly*

Easily translate 'ggplot2' graphs to an interactive web-based version and/or  create custom web-based visualizations directly from R.

*homepage*: https://cran.r-project.org/web/packages/plotly

version  |versionsuffix|toolchain
---------|-------------|---------------
``4.7.1``|``-R-3.4.0`` |``intel/2017a``
``4.8.0``|``-R-3.4.4`` |``intel/2018a``

<a anchor="plotly-orca"/>
### *plotly-orca*

Orca is an Electron app that generates images and reports of Plotly things like  plotly.js graphs, dash apps, dashboards from the command line.

*homepage*: https://github.com/plotly/orca

version  |toolchain
---------|-------------------------------------
``1.3.0``|``GCCcore/8.3.0``
``1.3.1``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``

<a anchor="plotly.py"/>
### *plotly.py*

An open-source, interactive graphing library for Python

*homepage*: https://plot.ly/python

version   |toolchain
----------|--------------------------------------
``4.1.0`` |``intel/2019a``
``4.4.1`` |``intel/2019b``
``4.8.1`` |``GCCcore/9.3.0``
``4.14.3``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``5.1.0`` |``GCCcore/10.3.0``
``5.4.0`` |``GCCcore/11.2.0``

<a anchor="plplot"/>
### *PLplot*

PLplot is a cross-platform software package for creating scientific plots whose (UTF-8) plot symbols  and text are limited in practice only by what Unicode-aware system fonts are installed on a user's computer.

*homepage*: http://plplot.sourceforge.net

version   |versionsuffix                   |toolchain
----------|--------------------------------|---------------
``5.11.1``|``-Java-1.7.0_80-Python-2.7.11``|``foss/2016a``
``5.11.1``|``-Java-1.7.0_80-Python-2.7.12``|``intel/2016b``

<a anchor="plumed"/>
### *PLUMED*

PLUMED is an open source library for free energy calculations in molecular systems which  works together with some of the most popular molecular dynamics engines. Free energy calculations can be  performed as a function of many order parameters with a particular  focus on biological problems, using  state of the art methods such as metadynamics, umbrella sampling and Jarzynski-equation based steered MD.  The software, written in C++, can be easily interfaced with both fortran and C/C++ codes. 

*homepage*: https://www.plumed.org

version  |versionsuffix     |toolchain
---------|------------------|------------------------------------------------
``2.2.2``|                  |``intel/2016a``
``2.2.3``|                  |``foss/2016b``, ``intel/2016b``
``2.3.0``|                  |``foss/2016b``, ``foss/2017a``, ``intel/2016b``
``2.3.4``|                  |``intel/2017b``
``2.4.0``|                  |``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``2.4.0``|``-PathCV``       |``intel/2018a``
``2.4.1``|                  |``iomkl/2018a``
``2.4.2``|                  |``foss/2018b``, ``intel/2018b``
``2.5.0``|                  |``intel/2018b``
``2.5.0``|``-Python-2.7.15``|``foss/2018b``, ``fosscuda/2018b``
``2.5.1``|                  |``foss/2019a``, ``intel/2018b``
``2.5.1``|``-PathCV``       |``intel/2018b``
``2.5.2``|``-Python-3.7.2`` |``intel/2019a``
``2.5.3``|``-Python-3.7.4`` |``foss/2019b``, ``intel/2019b``
``2.5b`` |``-Python-2.7.14``|``intel/2018a``
``2.6.0``|``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``2.6.2``|                  |``foss/2020b``, ``intel/2020b``
``2.7.0``|                  |``foss/2020b``
``2.7.2``|                  |``foss/2021a``, ``intel/2021a``
``2.7.3``|                  |``foss/2021b``
``2.8.0``|                  |``foss/2021b``

<a anchor="ply"/>
### *PLY*

PLY is yet another implementation of lex and yacc for Python.

*homepage*: http://www.dabeaz.com/ply/

version |versionsuffix    |toolchain
--------|-----------------|-----------------
``3.11``|``-Python-3.6.4``|``foss/2018a``
``3.11``|``-Python-3.7.4``|``GCCcore/8.3.0``

<a anchor="pmix"/>
### *PMIx*

Process Management for Exascale Environments PMI Exascale (PMIx) represents an attempt to provide an extended version of the PMI standard specifically designed to support clusters up to and including exascale sizes. The overall objective of the project is not to branch the existing pseudo-standard definitions - in fact, PMIx fully supports both of the existing PMI-1 and PMI-2 APIs - but rather to (a) augment and extend those APIs to eliminate some current restrictions that impact scalability, and (b) provide a reference implementation of the PMI-server that demonstrates the desired level of scalability. 

*homepage*: https://pmix.org/

version  |toolchain
---------|-------------------------------------------------------
``1.2.5``|``GCCcore/6.4.0``
``2.1.3``|``GCCcore/7.2.0``, ``GCCcore/7.3.0``
``2.2.1``|``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``2.2.3``|``GCCcore/7.2.0``, ``GCCcore/7.3.0``
``3.0.1``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``3.0.2``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``3.1.1``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``3.1.4``|``GCCcore/8.3.0``
``3.1.5``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``3.2.3``|``GCCcore/10.3.0``
``4.1.0``|``GCCcore/11.2.0``
``4.1.2``|``GCCcore/11.3.0``
``4.2.2``|``GCCcore/12.2.0``

<a anchor="pmx"/>
### *pmx*

pmx (formerly pymacs) is a small bunch of classes to read structure files such as pdb or gro and trajectory data in gromacs xtc format. Over the years it has been extended towards a versatile (bio-) molecular structure manipulation package with some additional functionalities, e.g. gromacs file parsers and scripts for setup and analysis of free energy calculations.

*homepage*: https://github.com/deGrootLab/pmx

version|versionsuffix     |toolchain
-------|------------------|--------------
``2.0``|``-Python-2.7.18``|``foss/2020b``

<a anchor="pnetcdf"/>
### *PnetCDF*

Parallel netCDF: A Parallel I/O Library for NetCDF File Access

*homepage*: https://trac.mcs.anl.gov/projects/parallel-netcdf

version   |toolchain
----------|-------------------------------------------------------------------------------------
``1.8.1`` |``intel/2017a``
``1.9.0`` |``intel/2018a``
``1.10.0``|``foss/2018b``, ``intel/2018b``
``1.12.1``|``gompi/2019b``, ``gompi/2020a``, ``gompic/2019b``, ``gompic/2020a``, ``iimpi/2020a``
``1.12.2``|``gompi/2020b``, ``gompi/2021a``, ``gompic/2020b``, ``iimpi/2021a``
``1.12.3``|``gompi/2021b``, ``gompi/2022a``

<a anchor="pocl"/>
### *pocl*

Pocl is a portable open source (MIT-licensed) implementation of the OpenCL standard

*homepage*: http://portablecl.org

version|toolchain
-------|----------------------------------------------------------
``1.2``|``GCC/7.3.0-2.30``
``1.3``|``GCC/8.2.0-2.31.1``, ``gcccuda/2019a``
``1.4``|``GCC/8.3.0``, ``gcccuda/2019b``
``1.5``|``GCC/9.3.0``
``1.6``|``GCC/10.2.0``, ``gcccuda/2020b``, ``iccifort/2020.4.304``
``1.8``|``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/11.3.0``

<a anchor="poetry"/>
### *poetry*

Python packaging and dependency management made easy

*homepage*: https://python-poetry.org/

version  |versionsuffix    |toolchain
---------|-----------------|-----------------
``1.0.9``|``-Python-3.8.2``|``GCCcore/9.3.0``

<a anchor="polars"/>
### *polars*

Lightning-fast DataFrame library for Rust and Python.

*homepage*: https://pola.rs/

version   |toolchain
----------|--------------
``0.15.6``|``foss/2022a``

<a anchor="polymake"/>
### *polymake*

polymake is open source software for research in polyhedral geometry. It deals with polytopes, polyhedra and fans as well as simplicial complexes, matroids, graphs, tropical hypersurfaces, and other objects.

*homepage*: https://polymake.org

version  |toolchain
---------|--------------
``4.0r1``|``foss/2019b``

<a anchor="pomkl"/>
### *pomkl*

Toolchain with PGI C, C++ and Fortran compilers, alongside Intel MKL & OpenMPI.

*homepage*: http://www.pgroup.com/index.htm

version    |toolchain
-----------|----------
``2016.03``|``system``
``2016.04``|``system``
``2016.09``|``system``

<a anchor="pompi"/>
### *pompi*

Toolchain with PGI C, C++ and Fortran compilers, alongside OpenMPI.

*homepage*: http://www.pgroup.com/index.htm

version    |toolchain
-----------|----------
``2016.03``|``system``
``2016.04``|``system``
``2016.09``|``system``

<a anchor="poppler"/>
### *poppler*

Poppler is a PDF rendering library

*homepage*: https://poppler.freedesktop.org

version    |toolchain
-----------|------------------------------
``0.70.1`` |``foss/2018b``
``0.90.1`` |``GCCcore/8.3.0``
``21.06.1``|``GCC/10.2.0``, ``GCC/10.3.0``
``22.01.0``|``GCC/11.2.0``
``22.12.0``|``GCC/11.3.0``

<a anchor="popscle"/>
### *popscle*

A suite of population scale analysis tools for single-cell genomics data including implementation of Demuxlet / Freemuxlet methods and auxilary tools 

*homepage*: https://github.com/statgen/popscle

version     |toolchain
------------|--------------
``0.1-beta``|``foss/2019b``

<a anchor="popt"/>
### *popt*

Popt is a C library for parsing command line parameters.

*homepage*: http://freecode.com/projects/popt

version |toolchain
--------|-------------------------
``1.14``|``GCC/4.8.2``
``1.16``|``GCC/4.9.2``, ``system``

<a anchor="porechop"/>
### *Porechop*

Porechop is a tool for finding and removing adapters from Oxford Nanopore reads.  Adapters on the ends of reads are trimmed off, and when a read has an adapter in its middle,  it is treated as chimeric and chopped into separate reads. Porechop performs thorough alignments  to effectively find adapters, even at low sequence identity

*homepage*: https://github.com/rrwick/Porechop

version  |versionsuffix    |toolchain
---------|-----------------|--------------------------------------
``0.2.3``|``-Python-3.5.2``|``foss/2016b``
``0.2.4``|                 |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``0.2.4``|``-Python-3.6.6``|``foss/2018b``
``0.2.4``|``-Python-3.7.4``|``intel/2019b``

<a anchor="porefoam"/>
### *porefoam*

Direct pore-scale simulation of single- and two-phase flow through confined media

*homepage*: https://github.com/ImperialCollegeLondon/porefoam

version       |toolchain
--------------|--------------
``2021-09-21``|``foss/2020a``

<a anchor="poretools"/>
### *poretools*

A toolkit for working with nanopore sequencing data from Oxford Nanopore.

*homepage*: https://poretools.readthedocs.io/en/latest/

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.6.0``|``-Python-2.7.14``|``intel/2018a``

<a anchor="portcullis"/>
### *Portcullis*

Portcullis stands for PORTable CULLing of Invalid Splice junctions from pre-aligned RNA-seq data. It is known that RNAseq mapping tools generate many invalid junction predictions, particularly in deep datasets with high coverage over splice sites. In order to address this, instead for creating a new RNAseq mapper, with a focus on SJ accuracy we created a tool that takes in a BAM file generated by an RNAseq mapper of the user's own choice (e.g. Tophat2, Gsnap, STAR2 or HISAT2) as input (i.e. it's portable). It then, analyses and quantifies all splice junctions in the file before, filtering (culling) those which are unlikely to be genuine. Portcullis output's junctions in a variety of formats making it suitable for downstream analysis (such as differential splicing analysis and gene modelling) without additional work. Portcullis can also filter the original BAM file removing alignments associated with bad junctions.

*homepage*: https://github.com/maplesond/portcullis

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.2.2``|``-Python-3.7.4``|``foss/2019b``

<a anchor="postgres-xl"/>
### *Postgres-XL*

Postgres-XL is a horizontally scalable open source SQL database cluster, flexible enough to handle varying database workloads:

*homepage*: http://www.postgres-xl.org

version  |versionsuffix     |toolchain
---------|------------------|---------------
``9.5r1``|``-Python-2.7.11``|``intel/2016a``

<a anchor="postgresql"/>
### *PostgreSQL*

PostgreSQL is a powerful, open source object-relational database system.  It is fully ACID compliant, has full support for foreign keys,  joins, views, triggers, and stored procedures (in multiple languages).  It includes most SQL:2008 data types, including INTEGER,  NUMERIC, BOOLEAN, CHAR, VARCHAR, DATE, INTERVAL, and TIMESTAMP.  It also supports storage of binary large objects, including pictures,  sounds, or video. It has native programming interfaces for C/C++, Java,  .Net, Perl, Python, Ruby, Tcl, ODBC, among others, and exceptional documentation.

*homepage*: https://www.postgresql.org/

version  |versionsuffix     |toolchain
---------|------------------|----------------------------------------------------------------
``9.4.7``|``-Python-2.7.11``|``intel/2016a``
``9.5.2``|``-Python-2.7.11``|``intel/2016a``
``9.6.0``|``-Python-2.7.12``|``intel/2016b``
``9.6.2``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``10.2`` |``-Python-2.7.14``|``intel/2018a``
``10.3`` |                  |``foss/2018b``
``10.3`` |``-Python-2.7.14``|``foss/2017b``, ``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``11.3`` |``-Python-2.7.15``|``GCCcore/8.2.0``
``11.3`` |``-Python-3.7.2`` |``GCCcore/8.2.0``
``12.4`` |                  |``GCCcore/9.3.0``
``13.2`` |                  |``GCCcore/10.2.0``
``13.3`` |                  |``GCCcore/10.3.0``
``13.4`` |                  |``GCCcore/11.2.0``
``14.4`` |                  |``GCCcore/11.3.0``

<a anchor="pot"/>
### *POT*

POT (Python Optimal Transport) is a Python library provide several solvers for optimization problems  related to Optimal Transport for signal, image processing and machine learning.

*homepage*: https://github.com/rflamary/POT

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.5.1``|``-Python-3.6.6``|``intel/2018b``

<a anchor="pov-ray"/>
### *POV-Ray*

The Persistence of Vision Raytracer, or POV-Ray, is a ray tracing program  which generates images from a text-based scene description, and is available for a variety  of computer platforms. POV-Ray is a high-quality, Free Software tool for creating stunning  three-dimensional graphics. The source code is available for those wanting to do their own ports.

*homepage*: https://www.povray.org/

version     |toolchain
------------|---------------------------------------------------------------------------------
``3.7.0.0`` |``intel/2016b``
``3.7.0.7`` |``foss/2017b``, ``foss/2018b``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``
``3.7.0.8`` |``GCC/10.2.0``, ``iccifort/2020.4.304``
``3.7.0.10``|``GCC/11.3.0``

<a anchor="ppanggolin"/>
### *PPanGGOLiN*

PPanGGOLiN is a software suite used to create and manipulate prokaryotic pangenomes from a set of either genomic DNA sequences or provided genome annotations. It is designed to scale up to tens of thousands of genomes. It has the specificity to partition the pangenome using a statistical approach rather than using fixed thresholds which gives it the ability to work with low-quality data such as Metagenomic Assembled Genomes (MAGs) or Single-cell Amplified Genomes (SAGs) thus taking advantage of large scale environmental studies and letting users study the pangenome of uncultivable species. 

*homepage*: https://github.com/labgem/PPanGGOLiN

version    |toolchain
-----------|--------------
``1.1.136``|``foss/2021b``

<a anchor="ppfold"/>
### *PPfold*

PPfold is a new implementation of pfold, written in Java 6.0. It can predict the consensus secondary  structure of RNA alignments through a stochastic context-free grammar coupled to an evolutionary model.  It can also use data from chemical probing experiments to predict RNA secondary structure. PPfold is multithreaded,  and can solve the structure of much longer alignments than pfold.

*homepage*: ihttp://www.daimi.au.dk/~compbio/pfold/downloads.html

version  |versionsuffix     |toolchain
---------|------------------|----------
``3.1.1``|``-Java-1.8.0_66``|``system``

<a anchor="ppl"/>
### *ppl*

The Parma Polyhedra Library (PPL) provides numerical abstractions especially  targeted at applications in the field of analysis and verification of complex systems.

*homepage*: https://www.bugseng.com/parma-polyhedra-library

version|toolchain
-------|-----------------
``1.2``|``GCCcore/6.4.0``

<a anchor="pplacer"/>
### *pplacer*

Pplacer places query sequences on a fixed reference phylogenetic tree to maximize phylogenetic  likelihood or posterior probability according to a reference alignment. Pplacer is designed to be fast, to give  useful information about uncertainty, and to offer advanced visualization and downstream analysis.

*homepage*: http://matsen.fhcrc.org/pplacer/

version        |toolchain
---------------|----------
``1.1.alpha19``|``system``

<a anchor="pplpy"/>
### *pplpy*

This Python package provides a wrapper to the C++ Parma Polyhedra Library (PPL).

*homepage*: https://pypi.org/project/pplpy/

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``0.8.4``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``0.8.4``|``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``

<a anchor="prank"/>
### *PRANK*

PRANK is a probabilistic multiple alignment program for DNA,   codon and amino-acid sequences. PRANK is based on a novel algorithm that treats   insertions correctly and avoids over-estimation of the number of deletion events.

*homepage*: http://wasabiapp.org/software/prank/

version   |toolchain
----------|------------------------------------------------------------------------------
``170427``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/9.3.0``, ``intel/2018a``

<a anchor="prc"/>
### *PRC*

PRC is a stand-alone program for aligning and scoring two profile hidden Markov models.  This can be used to detect remote relationships between profiles more effectively than by doing simple  profile-sequence comparisons. PRC takes into account all transition and emission probabilities in both  hidden Markov models. 

*homepage*: http//supfam.org/PRC/

version  |toolchain
---------|---------------
``1.5.6``|``intel/2018a``

<a anchor="precice"/>
### *preCICE*

preCICE (Precise Code Interaction Coupling Environment) is a coupling library for partitioned multi-physics simulations, including, but not restricted to fluid-structure interaction and conjugate heat transfer simulations. Partitioned means that preCICE couples existing programs (solvers) capable of simulating a subpart of the complete physics involved in a simulation. This allows for the high flexibility that is needed to keep a decent time-to-solution for complex multi-physics scenarios.

*homepage*: https://www.precice.org/

version  |versionsuffix    |toolchain
---------|-----------------|-------------------------------
``2.1.1``|``-Python-3.8.2``|``foss/2020a``
``2.2.0``|``-Python-3.8.2``|``foss/2020a``, ``intel/2020a``

<a anchor="prequal"/>
### *PREQUAL*

A program to identify and mask regions with non-homologous adjacent characters in FASTA files.

*homepage*: https://github.com/simonwhelan/prequal

version |toolchain
--------|------------------
``1.02``|``GCCcore/11.2.0``

<a anchor="preseq"/>
### *preseq*

Software for predicting library complexity and genome coverage in high-throughput sequencing.

*homepage*: https://smithlabresearch.org/software/preseq

version  |toolchain
---------|-------------------------------
``2.0.2``|``foss/2016b``
``2.0.3``|``foss/2018b``, ``intel/2018a``
``3.1.2``|``GCC/10.3.0``, ``GCC/11.2.0``

<a anchor="presto"/>
### *presto*

Presto performs a fast Wilcoxon rank sum test and auROC analysis.

*homepage*: https://github.com/immunogenomics/presto

version           |versionsuffix|toolchain
------------------|-------------|--------------
``1.0.0-20200718``|``-R-4.1.2`` |``foss/2021b``

<a anchor="pretty-yaml"/>
### *pretty-yaml*

PyYAML-based python module to produce pretty and readable YAML-serialized data. This module is for serialization only, see ruamel.yaml module for literate YAML parsing (keeping track of comments, spacing, line/column numbers of values, etc).

*homepage*: https://github.com/mk-fg/pretty-yaml

version    |toolchain
-----------|------------------
``19.12.0``|``GCCcore/8.3.0``
``20.4.0`` |``GCCcore/9.3.0``
``21.10.1``|``GCCcore/10.3.0``

<a anchor="primer3"/>
### *Primer3*

Primer3 is a widely used program for designing PCR primers (PCR = 'Polymerase Chain Reaction').  PCR is an essential and ubiquitous tool in genetics and molecular biology.  Primer3 can also design hybridization probes and sequencing primers.

*homepage*: https://primer3.org/

version  |toolchain
---------|---------------------------------------------------------
``2.3.7``|``intel/2017b``
``2.4.0``|``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2018b``
``2.5.0``|``GCC/9.3.0``

<a anchor="prinseq"/>
### *PRINSEQ*

A bioinformatics tool to PRe-process and show INformation of SEQuence data.

*homepage*: http://prinseq.sourceforge.net

version   |versionsuffix   |toolchain
----------|----------------|--------------
``0.20.4``|``-Perl-5.28.0``|``foss/2018b``
``0.20.4``|``-Perl-5.32.0``|``foss/2020b``
``0.20.4``|``-Perl-5.34.0``|``foss/2021b``

<a anchor="printproto"/>
### *printproto*

X.org PrintProto protocol headers.

*homepage*: http://xorg.freedesktop.org/

version  |toolchain
---------|---------------
``1.0.5``|``intel/2016a``

<a anchor="prisms-pf"/>
### *PRISMS-PF*

PRISMS-PF is a powerful, massively parallel finite element code for conducting phase field and  other related simulations of microstructural evolution.

*homepage*: https://prisms-center.github.io/phaseField

version  |toolchain
---------|-------------------------------
``2.1.1``|``foss/2019a``, ``intel/2019a``
``2.2``  |``foss/2021a``

<a anchor="probabel"/>
### *ProbABEL*

Tool for genome-wide association  analysis of imputed genetic data.

*homepage*: http://www.genabel.org/packages/ProbABEL

version  |toolchain
---------|-----------------------------
``0.5.0``|``GCCcore/9.3.0``, ``system``

<a anchor="prodigal"/>
### *prodigal*

Prodigal (Prokaryotic Dynamic Programming Genefinding Algorithm)    is a microbial (bacterial and archaeal) gene finding program developed    at Oak Ridge National Laboratory and the University of Tennessee.

*homepage*: https://github.com/hyattpd/Prodigal/

version  |toolchain
---------|---------------------------------------------------------------------------------------------------------------------------------------------------------
``2.6.2``|``GCC/4.9.3-binutils-2.25``
``2.6.3``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

<a anchor="profit"/>
### *ProFit*

ProFit (pronounced Pro-Fit, not profit!) is designed to be the ultimate program for  performing least squares fits of two or more protein structures. It performs a very simple and  basic function, but allows as much flexibility as possible in performing this procedure. Thus one  can specify subsets of atoms to be considered, specify zones to be fitted by number, sequence, or  by sequence alignment. 

*homepage*: http://www.bioinf.org.uk/software/profit

version|toolchain
-------|--------------
``3.3``|``GCC/10.3.0``

<a anchor="proj"/>
### *PROJ*

Program proj is a standard Unix filter function which converts geographic longitude and latitude coordinates into cartesian coordinates

*homepage*: https://proj.org

version  |toolchain
---------|---------------------------------------------------------------------------------
``4.9.2``|``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``4.9.3``|``foss/2016b``, ``foss/2017b``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``
``5.0.0``|``foss/2018a``, ``foss/2018b``, ``intel/2018a``, ``intel/2018b``, ``iomkl/2018a``
``6.0.0``|``GCCcore/8.2.0``
``6.2.1``|``GCCcore/8.3.0``
``7.0.0``|``GCCcore/9.3.0``
``7.2.1``|``GCCcore/10.2.0``
``8.0.1``|``GCCcore/10.3.0``
``8.1.0``|``GCCcore/11.2.0``
``9.0.0``|``GCCcore/11.3.0``

<a anchor="projectq"/>
### *ProjectQ*

An open source software framework for quantum computing

*homepage*: https://projectq.ch

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.4.2``|``-Python-3.6.6``|``intel/2018b``

<a anchor="prokka"/>
### *prokka*

Prokka is a software tool for the rapid annotation of prokaryotic genomes.

*homepage*: https://www.vicbioinformatics.com/software.prokka.shtml

version   |versionsuffix     |toolchain
----------|------------------|------------------------------------------------------------------
``1.11``  |``-BioPerl-1.7.0``|``foss/2016b``
``1.13``  |``-BioPerl-1.7.2``|``intel/2018a``
``1.13.4``|                  |``foss/2018b``
``1.13.7``|                  |``gompi/2019a``
``1.14.5``|                  |``gompi/2019a``, ``gompi/2019b``, ``gompi/2020b``, ``gompi/2021a``

<a anchor="prompt-toolkit"/>
### *prompt-toolkit*

prompt_toolkit is a Python library for building powerful interactive command lines and  terminal applications.

*homepage*: https://github.com/jonathanslenders/python-prompt-toolkit

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``1.0.3`` |``-Python-2.7.11``|``foss/2016a``
``1.0.3`` |``-Python-3.5.1`` |``foss/2016a``
``1.0.6`` |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``1.0.6`` |``-Python-3.5.2`` |``intel/2016b``
``1.0.13``|``-Python-2.7.12``|``intel/2016b``

<a anchor="proovread"/>
### *proovread*

PacBio hybrid error correction through iterative short read consensus

*homepage*: https://github.com/BioInf-Wuerzburg/proovread

version   |toolchain
----------|---------------
``2.14.1``|``intel/2017b``

<a anchor="propy"/>
### *propy*

Propy is a protein description software. It allows analyzing sequence-derived structural and physicochemical features, which are very useful in representing and distinguishing proteins or peptides of different structural, functional and interaction properties. These have been widely used in developing methods and software for predicting protein structural and functional classes, protein-protein interactions, drug-target interactions, protein substrates, among others.

*homepage*: https://code.google.com/archive/p/protpy

version|versionsuffix     |toolchain
-------|------------------|--------------
``1.0``|``-Python-2.7.13``|``foss/2017a``

<a anchor="proteinortho"/>
### *Proteinortho*

Proteinortho is a tool to detect orthologous genes within different species.

*homepage*: http://www.bioinf.uni-leipzig.de/Software/proteinortho/

version  |versionsuffix                |toolchain
---------|-----------------------------|--------------
``5.16b``|``-Python-3.6.4-Perl-5.26.1``|``foss/2018a``

<a anchor="prothint"/>
### *ProtHint*

ProtHint is a pipeline for predicting and scoring hints (in the form of introns, start and  stop codons) in the genome of interest by mapping and spliced aligning predicted genes to a database of  reference protein sequences.

*homepage*: https://github.com/gatech-genemark/ProtHint

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------------
``2.4.0``|``-Python-3.7.2``|``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``2.4.0``|``-Python-3.7.4``|``iccifort/2019.5.281``

<a anchor="protobuf"/>
### *protobuf*

Google Protocol Buffers

*homepage*: https://github.com/google/protobuf/

version    |toolchain
-----------|--------------------------------------------------------------------
``2.5.0``  |``GCCcore/10.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.3.0``, ``system``
``2.6.1``  |``system``
``3.0.2``  |``foss/2016a``
``3.2.0``  |``foss/2016b``, ``intel/2016b``
``3.3.0``  |``foss/2016b``, ``intel/2017a``
``3.4.0``  |``GCCcore/6.4.0``, ``intel/2017a``, ``intel/2017b``
``3.5.1``  |``intel/2017b``
``3.6.0``  |``GCCcore/7.3.0``
``3.6.1``  |``GCCcore/7.3.0``
``3.6.1.2``|``GCCcore/8.2.0``
``3.7.1``  |``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``3.10.0`` |``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``3.13.0`` |``GCCcore/9.3.0``
``3.14.0`` |``GCCcore/10.2.0``
``3.17.3`` |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``3.19.4`` |``GCCcore/11.3.0``

<a anchor="protobuf-python"/>
### *protobuf-python*

Python Protocol Buffers runtime library.

*homepage*: https://github.com/google/protobuf/

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------------
``3.0.2`` |``-Python-2.7.11``|``foss/2016a``
``3.0.2`` |``-Python-3.5.1`` |``foss/2016a``
``3.2.0`` |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``3.2.0`` |``-Python-3.5.2`` |``foss/2016b``, ``intel/2016b``
``3.3.0`` |``-Python-2.7.13``|``intel/2017a``
``3.3.0`` |``-Python-3.5.2`` |``foss/2016b``
``3.3.0`` |``-Python-3.6.1`` |``intel/2017a``
``3.4.0`` |``-Python-2.7.13``|``intel/2017a``
``3.4.0`` |``-Python-2.7.14``|``intel/2017b``
``3.4.0`` |``-Python-3.6.1`` |``intel/2017a``
``3.4.0`` |``-Python-3.6.3`` |``intel/2017b``
``3.6.0`` |``-Python-2.7.15``|``fosscuda/2018b``
``3.6.0`` |``-Python-3.6.6`` |``foss/2018b``, ``fosscuda/2018b``
``3.10.0``|``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``
``3.13.0``|``-Python-3.8.2`` |``foss/2020a``, ``fosscuda/2020a``
``3.14.0``|                  |``GCCcore/10.2.0``
``3.17.3``|                  |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``3.19.4``|                  |``GCCcore/11.3.0``

<a anchor="protozero"/>
### *protozero*

Minimalistic protocol buffer decoder and encoder in C++.

*homepage*: https://github.com/mapbox/protozero

version  |toolchain
---------|-----------------
``1.6.8``|``GCCcore/7.3.0``
``1.7.0``|``GCCcore/8.3.0``

<a anchor="prsice"/>
### *PRSice*

PRSice (pronounced 'precise') is a Polygenic Risk  Score software for calculating, applying, evaluating and  plotting the results of polygenic risk scores (PRS) analyses.

*homepage*: https://www.prsice.info/

version   |toolchain
----------|-------------------------------------
``2.2.12``|``GCCcore/8.3.0``
``2.3.1`` |``GCCcore/9.3.0``
``2.3.3`` |``GCCcore/10.3.0``, ``GCCcore/9.3.0``
``2.3.5`` |``GCCcore/11.3.0``

<a anchor="pscom"/>
### *pscom*

ParaStation is a robust and efficient cluster middleware, consisting  of a high-performance communication layer (MPI) and a sophisticated management layer.

*homepage*: http://www.par-tec.com

version     |toolchain
------------|-------------
``5.0.43``  |``GCC/4.8.2``
``5.0.44-1``|``GCC/4.9.2``
``5.0.48-1``|``system``

<a anchor="psi"/>
### *PSI*

PSI4 is an open-source suite of ab initio quantum chemistry programs designed for  efficient, high-accuracy simulations of a variety of molecular properties. We can routinely perform  computations with more than 2500 basis functions running serially or in parallel.

*homepage*: http://www.psicode.org/

version           |versionsuffix        |toolchain
------------------|---------------------|---------------
``4.0b6-20160201``|``-mt-Python-2.7.11``|``intel/2016a``

<a anchor="psi4"/>
### *PSI4*

PSI4 is an open-source suite of ab initio quantum chemistry programs designed for  efficient, high-accuracy simulations of a variety of molecular properties. We can routinely perform  computations with more than 2500 basis functions running serially or in parallel.

*homepage*: http://www.psicode.org/

version  |versionsuffix            |toolchain
---------|-------------------------|---------------
``1.0``  |``-Python-2.7.11``       |``intel/2016a``
``1.0``  |``-mt-Python-2.7.11``    |``intel/2016a``
``1.2.1``|``-Python-2.7.15``       |``intel/2018b``
``1.2.1``|``-Python-2.7.15-maxam8``|``intel/2018b``
``1.3.1``|``-Python-3.7.2``        |``foss/2019a``
``1.3.2``|``-Python-3.7.4``        |``intel/2019b``

<a anchor="psiclass"/>
### *PsiCLASS*

PsiCLASS is a reference-based transcriptome assembler for single or multiple RNA-seq samples.

*homepage*: https://github.com/splicebox/PsiCLASS

version  |toolchain
---------|--------------
``1.0.3``|``GCC/11.2.0``

<a anchor="psipred"/>
### *PSIPRED*

Accurate protein secondary structure prediction

*homepage*: http://bioinf.cs.ucl.ac.uk

version |toolchain
--------|-------------
``4.02``|``GCC/8.3.0``

<a anchor="psmc"/>
### *psmc*

This software package infers population size history from a diploid sequence using the Pairwise Sequentially Markovian Coalescent (PSMC) model.

*homepage*: https://github.com/lh3/psmc

version  |toolchain
---------|------------------------------
``0.6.5``|``foss/2016a``, ``foss/2018a``

<a anchor="psmpi"/>
### *psmpi*

ParaStation MPI is an open source high-performance MPI 3.0 implementation, based on MPICH v3. It provides extra low level communication libraries and integration with  various batch systems for tighter process control. 

*homepage*: https://github.com/ParaStation/psmpi2

version    |versionsuffix|toolchain
-----------|-------------|-------------
``5.1.0-1``|             |``GCC/4.9.2``
``5.1.5-1``|             |``GCC/4.9.3``
``5.1.5-1``|``-mt``      |``GCC/4.9.3``

<a anchor="psmpi2"/>
### *psmpi2*

ParaStation is a robust and efficient cluster middleware, consisting  of a high-performance communication layer (MPI) and a sophisticated management layer.

*homepage*: http://www.par-tec.com

version   |versionsuffix|toolchain
----------|-------------|-------------
``5.0.29``|             |``GCC/4.8.2``
``5.0.29``|``-mt``      |``GCC/4.8.2``

<a anchor="psolver"/>
### *PSolver*

Interpolating scaling function Poisson Solver Library  

*homepage*: http://bigdft.org/devel-doc/d1/d81/group__PSOLVER.html

version  |toolchain
---------|-------------------------------------------------------------------------------------------------
``1.7.6``|``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``
``1.8.3``|``foss/2020b``, ``foss/2021a``, ``foss/2021b``, ``intel/2020b``, ``intel/2021a``, ``intel/2021b``

<a anchor="psortb"/>
### *PSORTb*

PSORTb v3.0.4 is the most precise bacterial localization prediction tool available.

*homepage*: http://psort.org/psortb/index.html

version  |versionsuffix   |toolchain
---------|----------------|--------------
``3.0.4``|``-Perl-5.22.1``|``foss/2016a``

<a anchor="psrecord"/>
### *psrecord*

psrecord is a small utility that uses the psutil library to record the CPU and  memory activity of a process.

*homepage*: https://github.com/astrofrog/psrecord

version|versionsuffix     |toolchain
-------|------------------|---------------
``1.1``|``-Python-2.7.14``|``intel/2018a``
``1.1``|``-Python-2.7.15``|``intel/2018b``
``1.1``|``-Python-3.6.4`` |``intel/2018a``
``1.1``|``-Python-3.6.6`` |``intel/2018b``

<a anchor="pstoedit"/>
### *pstoedit*

pstoedit translates PostScript and PDF graphics into other vector formats

*homepage*: http://pstoedit.net/

version |toolchain
--------|------------------------------------
``3.70``|``GCCcore/6.3.0``, ``GCCcore/6.4.0``

<a anchor="psutil"/>
### *psutil*

A cross-platform process and system utilities module for Python

*homepage*: https://github.com/giampaolo/psutil

version  |versionsuffix     |toolchain
---------|------------------|--------------------------------------
``4.2.0``|``-Python-2.7.11``|``intel/2016a``
``4.3.0``|``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``5.4.3``|``-Python-2.7.14``|``intel/2017b``
``5.4.7``|``-Python-2.7.15``|``foss/2018b``
``5.4.7``|``-Python-3.6.6`` |``foss/2018b``
``5.6.1``|``-Python-2.7.15``|``fosscuda/2018b``
``5.6.3``|                  |``GCCcore/8.2.0``
``5.9.3``|                  |``GCCcore/10.2.0``, ``GCCcore/11.3.0``

<a anchor="psycopg2"/>
### *psycopg2*

Psycopg is the most popular PostgreSQL adapter for the Python programming language.

*homepage*: http://initd.org/psycopg/

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``2.7``  |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``2.8.3``|``-Python-3.7.2`` |``foss/2019a``
``2.8.6``|``-Python-3.8.2`` |``GCCcore/9.3.0``

<a anchor="ptemcee"/>
### *ptemcee*

ptemcee, pronounced "tem-cee", is fork of Daniel Foreman-Mackey's wonderful emcee to implement  parallel tempering more robustly. If you're trying to characterise awkward, multi-model probability  distributions, then ptemcee is your friend.

*homepage*: https://github.com/willvousden/ptemcee

version  |toolchain
---------|--------------
``1.0.0``|``foss/2019a``

<a anchor="ptesfinder"/>
### *PTESFinder*

Post-Transcriptional Exon Shuffling (PTES) Identification Pipeline

*homepage*: https://sourceforge.net/projects/ptesfinder-v1/

version|toolchain
-------|---------------
``1``  |``intel/2017b``

<a anchor="pubtcrs"/>
### *pubtcrs*

This repository contains C++ source code for the TCR clustering  and correlation analyses described in the manuscript "Human T cell receptor  occurrence patterns encode immune history, genetic background, and receptor  specificity" by William S DeWitt III, Anajane Smith, Gary Schoch, John A Hansen,  Frederick A Matsen IV and Philip Bradley, available on bioRxiv.

*homepage*: https://github.com/phbradley/pubtcrs

version     |toolchain
------------|---------------
``20180622``|``intel/2019a``

<a anchor="pugixml"/>
### *pugixml*

pugixml is a light-weight C++ XML processing library

*homepage*: https://pugixml.org/

version   |toolchain
----------|--------------------------------------
``1.11.4``|``GCCcore/10.3.0``
``1.12.1``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="pullseq"/>
### *pullseq*

Utility program for extracting sequences from a fasta/fastq file

*homepage*: https://github.com/bcthomas/pullseq

version  |toolchain
---------|---------------------------------------------------------
``1.0.2``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/7.3.0``

<a anchor="pulp"/>
### *PuLP*

PuLP is an LP modeler written in Python. PuLP can generate MPS or LP files and call GLPK, COIN-OR CLP/CBC, CPLEX, GUROBI, MOSEK, XPRESS, CHOCO, MIPCL, SCIP to solve linear problems. 

*homepage*: https://github.com/coin-or/pulp

version  |toolchain
---------|--------------
``2.5.1``|``foss/2021a``

<a anchor="purge_dups"/>
### *purge_dups*

purge haplotigs and overlaps in an assembly based on read depth

*homepage*: https://github.com/dfguan/purge_dups

version  |toolchain
---------|--------------
``1.2.5``|``foss/2021b``

<a anchor="py"/>
### *py*

library with cross-python path, ini-parsing, io, code, log facilities

*homepage*:  https://pylib.readthedocs.org/

version   |versionsuffix     |toolchain
----------|------------------|--------------
``1.4.31``|``-Python-2.7.11``|``foss/2016a``
``1.4.31``|``-Python-3.5.1`` |``foss/2016a``

<a anchor="py-aiger"/>
### *py-aiger*

A python library for manipulating sequential and combinatorial circuits. This module provides the py-aiger extensions: aiger_bv, aiger_cnf, aiger_ptltl, aiger_coins, aiger_gridworld, aiger_dfa

*homepage*: https://github.com/mvcisback/py-aiger

version   |versionsuffix    |toolchain
----------|-----------------|------------------
``6.1.1`` |``-Python-3.8.2``|``GCCcore/9.3.0``
``6.1.14``|                 |``GCCcore/10.2.0``

<a anchor="py-aiger-bdd"/>
### *py-aiger-bdd*

Aiger to BDD bridge.

*homepage*: https://github.com/mvcisback/py-aiger-bdd

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``3.0.0``|``-Python-3.8.2``|``foss/2020a``

<a anchor="py-cpuinfo"/>
### *py-cpuinfo*

py-cpuinfo gets CPU info with pure Python.

*homepage*: https://github.com/workhorsy/py-cpuinfo

version  |toolchain
---------|----------
``5.0.0``|``system``

<a anchor="pyabc"/>
### *pyABC*

Massively parallel, distributed and scalable ABC-SMC (Approximate Bayesian Computation - Sequential Monte Carlo) for parameter estimation of complex stochastic models.  Implemented in Python with support of the R language. 

*homepage*: https://github.com/icb-dcm/pyabc

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.10.4``|``-Python-3.7.4``|``foss/2019b``

<a anchor="pyamg"/>
### *PyAMG*

PyAMG is a library of Algebraic Multigrid (AMG) solvers with a convenient Python interface.

*homepage*: https://pyamg.github.io

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``3.0.1``|``-Python-2.7.11``|``intel/2016a``
``4.0.0``|                  |``foss/2020b``, ``intel/2020b``
``4.2.3``|                  |``foss/2021a``

<a anchor="pyaps3"/>
### *PyAPS3*

Python 3 Atmospheric Phase Screen

*homepage*: https://github.com/AngeliqueBenoit/pyaps3

version     |versionsuffix    |toolchain
------------|-----------------|--------------
``20190407``|``-Python-3.7.2``|``foss/2019a``

<a anchor="pybedtools"/>
### *pybedtools*

pybedtools wraps and extends BEDTools and offers feature-level manipulations from within Python.

*homepage*: https://daler.github.io/pybedtools

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------------------------------
``0.7.10``|``-Python-2.7.14``|``intel/2017b``, ``intel/2018a``
``0.7.10``|``-Python-3.6.6`` |``foss/2018b``
``0.8.0`` |                  |``foss/2019a``, ``intel/2019a``
``0.8.1`` |                  |``foss/2019b``
``0.8.2`` |                  |``GCC/10.2.0``, ``GCC/11.2.0``, ``iccifort/2020.4.304``
``0.8.2`` |``-Python-2.7.18``|``GCC/10.2.0``
``0.9.0`` |                  |``GCC/11.3.0``

<a anchor="pyberny"/>
### *PyBerny*

PyBerny is an optimizer of molecular geometries with respect to the total energy, using nuclear gradient information.

*homepage*: https://github.com/jhrmnn/pyberny

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.6.2``|``-Python-3.8.2``|``intel/2020a``
``0.6.3``|                 |``foss/2022a``

<a anchor="pybigwig"/>
### *pyBigWig*

A python extension, written in C, for quick access to bigBed  files and access to and creation of bigWig files.

*homepage*: https://github.com/deeptools/pyBigWig

version   |versionsuffix    |toolchain
----------|-----------------|--------------------------------------------------
``0.3.13``|``-Python-3.6.6``|``foss/2018b``
``0.3.17``|                 |``GCCcore/8.2.0``, ``GCCcore/9.3.0``
``0.3.18``|                 |``GCCcore/10.2.0``, ``foss/2021a``, ``foss/2021b``

<a anchor="pybind11"/>
### *pybind11*

pybind11 is a lightweight header-only library that exposes C++ types in Python and vice versa,  mainly to create Python bindings of existing C++ code.

*homepage*: https://pybind11.readthedocs.io

version  |versionsuffix     |toolchain
---------|------------------|------------------
``2.2.4``|``-Python-3.6.4`` |``foss/2018a``
``2.2.4``|``-Python-3.6.6`` |``intel/2018b``
``2.4.3``|``-Python-3.7.4`` |``GCCcore/8.3.0``
``2.4.3``|``-Python-3.8.2`` |``GCCcore/9.3.0``
``2.6.0``|                  |``GCCcore/10.2.0``
``2.6.2``|                  |``GCCcore/10.3.0``
``2.7.1``|                  |``GCCcore/11.2.0``
``2.7.1``|``-Python-2.7.18``|``GCCcore/11.2.0``
``2.9.2``|                  |``GCCcore/11.3.0``

<a anchor="pycairo"/>
### *PyCairo*

Python bindings for the cairo library

*homepage*: https://pycairo.readthedocs.io/

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------------
``1.10.0``|``-Python-2.7.11``|``intel/2016a``
``1.16.1``|``-Python-3.6.3`` |``foss/2017b``
``1.16.2``|``-Python-2.7.14``|``intel/2017b``
``1.18.0``|``-Python-2.7.14``|``intel/2018a``
``1.18.0``|``-Python-2.7.15``|``foss/2018b``
``1.18.0``|``-Python-3.6.6`` |``foss/2018b``
``1.18.0``|``-Python-3.7.2`` |``GCCcore/8.2.0``
``1.18.2``|                  |``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.20.0``|                  |``GCCcore/10.2.0``
``1.20.1``|                  |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``1.21.0``|                  |``GCCcore/11.3.0``

<a anchor="pycalib"/>
### *PyCalib*

Python library for classifier calibration

*homepage*: https://github.com/classifier-calibration/PyCalib

version       |toolchain
--------------|--------------
``0.1.0.dev0``|``foss/2021b``

<a anchor="pyccel"/>
### *pyccel*

Python extension language using accelerators

*homepage*: https://github.com/pyccel/pyccel

version  |toolchain
---------|--------------
``1.7.0``|``foss/2022a``

<a anchor="pycharm"/>
### *PyCharm*

PyCharm Community Edition: Python IDE for Professional Developers

*homepage*: https://www.jetbrains.com/pycharm/

version     |toolchain
------------|----------
``2017.2.3``|``system``
``2019.3.1``|``system``
``2021.1.1``|``system``
``2022.2.2``|``system``

<a anchor="pychemps2"/>
### *PyCheMPS2*

PyCheMPS2 is a python interface to CheMPS2, for compilation without MPI. CheMPS2 is a scientific library which contains a spin-adapted implementation of the density matrix renormalization group (DMRG) for ab initio quantum chemistry.

*homepage*: https://sebwouters.github.io/CheMPS2

version   |toolchain
----------|--------------
``1.8.12``|``foss/2022a``

<a anchor="pycifrw"/>
### *PyCifRW*

PyCIFRW provides support for reading and writing CIF (Crystallographic Information Format) files using Python.

*homepage*: https://bitbucket.org/jamesrhester/pycifrw/src/development

version  |toolchain
---------|-----------------
``4.4.2``|``GCCcore/8.3.0``

<a anchor="pyclone"/>
### *PyClone*

PyClone is a Python package that wraps rclone and provides a threaded interface for  an installation at the host or container level.

*homepage*: https://gitlab.com/ltgiv/pyclone

version     |toolchain
------------|------------------
``2020.9b2``|``GCCcore/10.2.0``

<a anchor="pycma"/>
### *pycma*

A stochastic numerical optimization algorithm for difficult (non-convex, ill-conditioned,  multi-modal, rugged, noisy) optimization problems in continuous search spaces, implemented in Python.

*homepage*: https://github.com/CMA-ES/pycma

version  |toolchain
---------|---------------
``2.7.0``|``intel/2019a``

<a anchor="pycocotools"/>
### *pycocotools*

Official APIs for the MS-COCO dataset

*homepage*: https://pypi.org/project/pycocotools

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------
``2.0.0``|``-Python-3.7.4``|``foss/2019b``, ``fosscuda/2019b``
``2.0.1``|``-Python-3.7.4``|``foss/2019b``, ``fosscuda/2019b``
``2.0.2``|``-Python-3.8.2``|``foss/2020a``, ``fosscuda/2020a``
``2.0.4``|                 |``foss/2021a``

<a anchor="pycodestyle"/>
### *pycodestyle*

pycodestyle is a tool to check your Python code against some of the style conventions in PEP 8.

*homepage*: https://pycodestyle.readthedocs.io

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``2.5.0``|``-Python-3.6.4``|``intel/2018a``

<a anchor="pycogent"/>
### *PyCogent*

PyCogent is a software library for genomic biology. It is a fully  integrated and thoroughly tested framework for: controlling third-party applications;  devising workflows; querying databases; conducting novel probabilistic analyses of  biological sequence evolution; and generating publication quality graphics.

*homepage*: http://pycogent.org

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``1.5.3``|``-Python-2.7.12``|``intel/2016b``
``1.9``  |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``1.9``  |``-Python-2.7.15``|``foss/2018b``

<a anchor="pycoqc"/>
### *pycoQC*

PycoQC computes metrics and generates interactive QC plots for  Oxford Nanopore technologies sequencing data.

*homepage*: https://tleonardi.github.io/pycoQC

version  |toolchain
---------|--------------
``2.5.2``|``foss/2021a``

<a anchor="pycuda"/>
### *PyCUDA*

PyCUDA lets you access Nvidia’s CUDA parallel computation API from Python.

*homepage*: https://mathema.tician.de/software/pycuda

version     |versionsuffix                  |toolchain
------------|-------------------------------|---------------------------------------
``2016.1.2``|``-Python-2.7.12``             |``intel/2016b``
``2017.1.1``|``-CUDA-9.0.176-Python-2.7.14``|``foss/2017b``
``2017.1.1``|``-Python-2.7.14``             |``intel/2018a``
``2018.1``  |``-Python-3.6.4-CUDA-9.1.85``  |``intel/2018a``
``2019.1.2``|``-Python-3.7.4``              |``fosscuda/2019b``, ``intelcuda/2019b``
``2020.1``  |                               |``fosscuda/2020b``

<a anchor="pycurl"/>
### *PycURL*

PycURL is a Python interface to libcurl. PycURL can be used to fetch objects identified by a URL  from a Python program, similar to the urllib Python module. PycURL is mature, very fast, and supports a lot of  features.

*homepage*: http://pycurl.io/

version     |versionsuffix    |toolchain
------------|-----------------|-----------------
``7.43.0.5``|``-Python-3.7.4``|``GCCcore/8.3.0``

<a anchor="pydamage"/>
### *PyDamage*

Pydamage, is a Python software to automate the process of contig damage identification and estimation. After modelling the ancient DNA damage using the C to T transitions, Pydamage uses a likelihood ratio test to discriminate between truly ancient, and modern contigs originating from sample contamination.

*homepage*: https://github.com/maxibor/pydamage

version |toolchain
--------|--------------
``0.70``|``foss/2021a``

<a anchor="pydantic"/>
### *pydantic*

Data validation and settings management using Python type hinting.

*homepage*: https://github.com/samuelcolvin/pydantic

version   |versionsuffix    |toolchain
----------|-----------------|------------------
``1.6.1`` |``-Python-3.7.4``|``GCCcore/8.3.0``
``1.10.2``|                 |``GCCcore/11.2.0``

<a anchor="pydatastream"/>
### *PyDatastream*

Lightweight SOAP client

*homepage*: https://pypi.python.org/pypi/suds-py3

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.5.1``|``-Python-3.6.4``|``intel/2018a``

<a anchor="pydicom"/>
### *pydicom*

Pure python package for DICOM medical file reading and writing.

*homepage*: https://pydicom.github.io/

version  |versionsuffix     |toolchain
---------|------------------|--------------------------------------
``0.9.9``|``-Python-2.7.11``|``intel/2016a``
``1.2.2``|                  |``GCCcore/8.2.0``
``1.4.2``|                  |``GCCcore/8.3.0``
``2.1.2``|                  |``GCCcore/10.2.0``
``2.1.2``|``-Python-3.8.2`` |``GCCcore/9.3.0``
``2.2.2``|                  |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``2.3.0``|                  |``GCCcore/11.3.0``

<a anchor="pydlpoly"/>
### *pydlpoly*

Pydlpoly is a molecular dynamics simulation package which is a modified version of DL-POLY with  a Python language interface.

*homepage*: http://cmc.aci.ruhr-uni-bochum.de/cmc/

version     |versionsuffix     |toolchain
------------|------------------|---------------
``20150225``|``-Python-2.7.12``|``intel/2016b``
``20150225``|``-Python-2.7.13``|``intel/2017a``

<a anchor="pydot"/>
### *pydot*

Python interface to Graphviz's Dot language.

*homepage*: https://github.com/pydot/pydot

version  |toolchain
---------|--------------------------------------
``1.4.1``|``GCCcore/9.3.0``, ``foss/2019b``
``1.4.2``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``

<a anchor="pyega3"/>
### *pyEGA3*

A basic Python-based EGA download client 

*homepage*: https://github.com/EGA-archive/ega-download-client

version   |versionsuffix    |toolchain
----------|-----------------|------------------
``3.0.33``|``-Python-3.7.2``|``GCCcore/8.2.0``
``3.4.0`` |``-Python-3.7.4``|``GCCcore/8.3.0``
``3.4.0`` |``-Python-3.8.2``|``GCCcore/9.3.0``
``4.0.0`` |                 |``GCCcore/11.2.0``

<a anchor="pyenchant"/>
### *pyenchant*

PyEnchant is a spellchecking library for Python, based on the excellent Enchant library.

*homepage*: https://pythonhosted.org/pyenchant/

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.6.8``|``-Python-2.7.13``|``intel/2017a``

<a anchor="pyevtk"/>
### *PyEVTK*

EVTK (Export VTK) package allows exporting data to binary VTK files for visualization and data analysis with any of the visualization packages that support VTK files

*homepage*: https://github.com/paulo-herrera/PyEVTK

version  |toolchain
---------|--------------
``1.4.1``|``foss/2021b``
``2.0.0``|``foss/2021b``

<a anchor="pyfai"/>
### *pyFAI*

Python implementation of fast azimuthal integration.

*homepage*: https://github.com/silx-kit/pyFAI

version   |versionsuffix    |toolchain
----------|-----------------|----------------------------------
``0.19.0``|``-Python-3.7.4``|``foss/2019b``, ``fosscuda/2019b``
``0.20.0``|                 |``foss/2020b``, ``fosscuda/2020b``

<a anchor="pyfaidx"/>
### *pyfaidx*

pyfaidx: efficient pythonic random access to fasta subsequences

*homepage*: https://pypi.python.org/pypi/pyfaidx

version    |versionsuffix    |toolchain
-----------|-----------------|------------------
``0.5.9.5``|                 |``GCCcore/10.2.0``
``0.5.9.5``|``-Python-3.7.4``|``GCCcore/8.3.0``
``0.6.3.1``|                 |``GCCcore/10.3.0``
``0.7.0``  |                 |``GCCcore/11.2.0``

<a anchor="pyfasta"/>
### *pyfasta*

fast, memory-efficient, pythonic (and command-line) access to fasta sequence files

*homepage*: https://pypi.org/project/pyfasta/

version  |toolchain
---------|--------------
``0.5.2``|``foss/2020b``

<a anchor="pyffmpeg"/>
### *PyFFmpeg*

Python FFmpeg wrapper

*homepage*: https://github.com/mhaller/pyffmpeg

version    |versionsuffix     |toolchain
-----------|------------------|----------------
``2.1beta``|``-Python-2.7.10``|``gimkl/2.11.5``
``2.1beta``|``-Python-2.7.11``|``intel/2016a``

<a anchor="pyfftw"/>
### *pyFFTW*

A pythonic wrapper around FFTW, the FFT library, presenting a unified interface  for all the supported transforms.

*homepage*: https://github.com/pyFFTW/pyFFTW

version   |versionsuffix    |toolchain
----------|-----------------|----------------------------------
``0.11.1``|                 |``intel/2019a``
``0.11.1``|``-Python-3.6.6``|``foss/2018b``
``0.12.0``|                 |``foss/2020b``, ``fosscuda/2020b``

<a anchor="pyfits"/>
### *pyfits*

The PyFITS module is a Python library providing access to FITS (Flexible Image Transport System)

*homepage*: https://pythonhosted.org/pyfits/

version|versionsuffix     |toolchain
-------|------------------|---------------
``3.5``|``-Python-2.7.15``|``intel/2018b``

<a anchor="pyfmi"/>
### *PyFMI*

PyFMI is a package for loading and interacting with Functional Mock-Up Units (FMUs),  which are compiled dynamic models compliant with the Functional Mock-Up Interface (FMI)

*homepage*: https://pypi.org/project/PyFMI/

version  |versionsuffix     |toolchain
---------|------------------|---------------
``2.4.0``|``-Python-2.7.15``|``intel/2018b``

<a anchor="pyfoam"/>
### *PyFoam*

A Python library to control OpenFOAM-runs and manipulate OpenFOAM-data.

*homepage*: http://openfoamwiki.net/index.php/Contrib/PyFoam

version   |toolchain
----------|--------------
``2020.5``|``foss/2020b``

<a anchor="pyfr"/>
### *PyFR*

PyFR is an open-source Python based framework for solving advection-diffusion type problems on streaming architectures using the Flux Reconstruction approach of Huynh. The framework is designed to solve a range of governing systems on mixed unstructured grids containing various element types. It is also designed to target a range of hardware platforms via use of an in-built domain specific language derived from the Mako templating engine.

*homepage*: http://www.pyfr.org

version  |versionsuffix                |toolchain
---------|-----------------------------|-------------------
``1.7.6``|``-Python-3.6.4-CUDA-9.1.85``|``intel/2018a``
``1.9.0``|``-Python-3.7.4``            |``intelcuda/2019b``

<a anchor="pyfrag"/>
### *PyFrag*

PyFrag 2019 resolves three main challenges associated with the automatized computational exploration of reaction mechanisms: 1) the management of multiple parallel calculations to automatically find a reaction path; 2) the monitoring of the entire computational process along with the extraction and plotting of relevant information from large amounts of data; and 3) the analysis and presentation of these data in a clear and informative way.  This module provides the Activation Strain Analysis (ASA) Module of PyFrag 2019 

*homepage*: https://pyfragdocument.readthedocs.io/en/latest/includeme.html

version          |versionsuffix|toolchain
-----------------|-------------|---------------
``2019-20220216``|``-ASA``     |``intel/2020b``

<a anchor="pygccxml"/>
### *pygccxml*

Python package for easy C++ declarations navigation.

*homepage*: https://pypi.python.org/pypi/pygccxml

version     |versionsuffix     |toolchain
------------|------------------|--------------
``20160706``|``-Python-2.7.11``|``foss/2016a``
``20160706``|``-Python-3.5.1`` |``foss/2016a``

<a anchor="pygenometracks"/>
### *pyGenomeTracks*

pyGenomeTracks aims to produce high-quality genome browser tracks that are highly customizable.

*homepage*: https://pygenometracks.readthedocs.io

version|toolchain
-------|--------------
``3.7``|``foss/2021b``

<a anchor="pygeos"/>
### *PyGEOS*

PyGEOS is a C/Python library with vectorized geometry functions. The geometry operations are done in  the open-source geometry library GEOS. PyGEOS wraps these operations in NumPy ufuncs providing a performance  improvement when operating on arrays of geometries.

*homepage*: https://pygeos.readthedocs.io

version   |versionsuffix    |toolchain
----------|-----------------|---------------
``0.7.1`` |``-Python-3.7.4``|``foss/2019b``
``0.8``   |``-Python-3.8.2``|``foss/2020a``
``0.10.2``|                 |``intel/2020b``

<a anchor="pygimli"/>
### *pyGIMLi*

pyGIMLi is an open-source multi-method library for solving inverse  and forward tasks related to geophysical problems. Written in C++ and Python,  it offers both efficiency and flexibility allowing you to quickly build  your own robust inversion applications for the geophysical problem at hand.

*homepage*: http://www.pygimli.org/

version     |versionsuffix     |toolchain
------------|------------------|--------------
``20160803``|``-Python-2.7.11``|``foss/2016a``
``20160803``|``-Python-3.5.1`` |``foss/2016a``

<a anchor="pygments"/>
### *Pygments*

Generic syntax highlighter suitable for use in code hosting, forums, wikis or other applications  that need to prettify source code.

*homepage*: https://pygments.org/

version  |versionsuffix     |toolchain
---------|------------------|--------------
``2.1.3``|``-Python-2.7.11``|``foss/2016a``
``2.1.3``|``-Python-3.5.1`` |``foss/2016a``

<a anchor="pygmo"/>
### *pygmo*

pygmo is a scientific Python library for massively parallel optimization.

*homepage*: https://esa.github.io/pygmo2

version   |toolchain
----------|------------------------------
``2.16.1``|``foss/2020b``
``2.18.0``|``foss/2021a``, ``foss/2021b``

<a anchor="pygobject"/>
### *PyGObject*

PyGObject is a Python package which provides bindings for GObject based libraries such as GTK, GStreamer, WebKitGTK, GLib, GIO and many more.

*homepage*: https://pygobject.readthedocs.io/

version   |versionsuffix     |toolchain
----------|------------------|------------------
``2.28.6``|``-Python-2.7.11``|``intel/2016a``
``2.28.6``|``-Python-2.7.14``|``intel/2017b``
``2.28.7``|``-Python-2.7.14``|``intel/2018a``
``2.28.7``|``-Python-2.7.15``|``foss/2018b``
``3.34.0``|``-Python-2.7.16``|``GCCcore/8.3.0``
``3.34.0``|``-Python-3.7.2`` |``GCCcore/8.2.0``
``3.34.0``|``-Python-3.7.4`` |``GCCcore/8.3.0``
``3.42.1``|                  |``GCCcore/11.3.0``

<a anchor="pygraphviz"/>
### *pygraphviz*

PyGraphviz is a Python interface to the Graphviz graph layout and visualization package.  With PyGraphviz you can create, edit, read, write, and draw graphs using Python to access the Graphviz  graph data structure and layout algorithms.

*homepage*: https://pygraphviz.github.io/

version|toolchain
-------|------------------------------
``1.5``|``foss/2019b``
``1.7``|``foss/2020b``, ``foss/2021a``

<a anchor="pygrib"/>
### *pygrib*

Python interface for reading and writing GRIB data

*homepage*: https://jswhit.github.io/pygrib

version  |toolchain
---------|--------------
``2.0.4``|``foss/2019a``

<a anchor="pygtk"/>
### *PyGTK*

PyGTK lets you to easily create programs with a graphical user interface  using the Python programming language.

*homepage*: http://www.pygtk.org/

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------
``2.24.0``|``-Python-2.7.11``|``intel/2016a``
``2.24.0``|``-Python-2.7.14``|``intel/2017b``, ``intel/2018a``
``2.24.0``|``-Python-2.7.15``|``foss/2018b``

<a anchor="pygts"/>
### *PyGTS*

PyGTS is a python package used to construct, manipulate,   and perform computations on triangulated surfaces.   It is a hand-crafted and pythonic binding for the GNU Triangulated Surface (GTS) Library. 

*homepage*: https://sourceforge.net/projects/pygts/

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``0.3.1``|``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``0.3.1``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``0.3.1``|``-Python-2.7.14``|``intel/2018a``

<a anchor="pygwas"/>
### *PyGWAS*

PyGWAS is a library for running Genome Wide Association studies.

*homepage*: https://github.com/timeu/pygwas

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``1.2.0``|``-Python-2.7.11``|``foss/2016a``
``1.3.1``|``-Python-2.7.11``|``foss/2016a``
``1.4.0``|``-Python-2.7.11``|``foss/2016a``
``1.5.0``|``-Python-2.7.11``|``foss/2016a``
``1.6.1``|``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``1.7.1``|``-Python-2.7.13``|``foss/2017a``

<a anchor="pyhdf"/>
### *pyhdf*

Python wrapper around the NCSA HDF version 4 library

*homepage*: https://github.com/fhs/pyhdf

version   |toolchain
----------|--------------
``0.10.1``|``foss/2019a``

<a anchor="pyimagej"/>
### *PyImageJ*

PyImageJ provides a set of wrapper functions for integration between ImageJ2 and Python. It also  supports the original ImageJ API and data structures. A major advantage of this approach is the ability to combine  ImageJ and ImageJ2 with other tools available from the Python software ecosystem, including NumPy, SciPy,  scikit-image, CellProfiler, OpenCV, ITK and many more.

*homepage*: https://pyimagej.readthedocs.io/

version  |toolchain
---------|--------------
``1.3.1``|``foss/2021a``

<a anchor="pyiron"/>
### *pyiron*

An integrated development environment (IDE) for computational materials science.

*homepage*: https://github.com/pyiron/pyiron

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.2.5``|``-Python-3.7.2``|``intel/2019a``

<a anchor="pyke3"/>
### *Pyke3*

Pyke introduces a form of Logic Programming (inspired by Prolog)  to the Python community by providing a knowledge-based inference engine  (expert system) written in 100% Python.

*homepage*: http://sourceforge.net/projects/pyke/

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``1.1.1``|``-Python-3.6.6``|``intel/2018b``

<a anchor="pylift"/>
### *pylift*

pylift is an uplift library that provides, primarily:  (1) Fast uplift modeling implementations and (2) Evaluation tools (UpliftEval class).

*homepage*: https://github.com/df-foundation/pylift

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.1.5``|``-Python-3.7.4``|``foss/2019b``
``0.1.5``|``-Python-3.8.2``|``foss/2020a``

<a anchor="pylint"/>
### *Pylint*

Pylint is a tool that checks for errors in Python code, tries to enforce  a coding standard and looks for code smells. It can also look for certain type errors,  it can recommend suggestions about how particular blocks can be refactored and  can offer you details about the code's complexity.

*homepage*: https://www.pylint.org/

version  |versionsuffix     |toolchain
---------|------------------|------------------------------------------------
``1.9.3``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``, ``iomkl/2018b``
``1.9.5``|``-Python-2.7.15``|``GCCcore/8.2.0``
``1.9.5``|``-Python-2.7.16``|``GCCcore/8.3.0``
``2.7.4``|                  |``GCCcore/10.2.0``

<a anchor="pymatgen"/>
### *pymatgen*

Python Materials Genomics is a robust materials analysis code that defines core object  representations for structures and molecules with support for many electronic structure codes.

*homepage*: https://pypi.python.org/pypi/pymatgen

version       |versionsuffix     |toolchain
--------------|------------------|-------------------------
``3.5.0``     |``-Python-2.7.11``|``intel/2016.02-GCC-4.9``
``4.1.1``     |``-Python-2.7.12``|``intel/2016b``
``4.3.2``     |``-Python-2.7.12``|``intel/2016b``
``4.7.3``     |``-Python-2.7.13``|``intel/2017a``
``2017.10.16``|``-Python-2.7.14``|``intel/2017b``
``2017.10.16``|``-Python-3.6.3`` |``intel/2017b``
``2022.0.4``  |                  |``foss/2020b``

<a anchor="pymatgen-db"/>
### *pymatgen-db*

Pymatgen-db is a database add-on for the Python Materials Genomics (pymatgen)  materials analysis library.

*homepage*: https://pypi.python.org/pypi/pymatgen-db

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.6.5``|``-Python-2.7.13``|``intel/2017a``

<a anchor="pymbar"/>
### *pymbar*

The pymbar package contains the pymbar suite of tools for the analysis of simulated and experimental data with the multistate Bennett acceptance ratio (MBAR) estimator.

*homepage*: http://pymbar.readthedocs.io/en/master/

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``3.0.3``|``-Python-3.6.3``|``intel/2017b``
``3.0.3``|``-Python-3.8.2``|``intel/2020a``

<a anchor="pymc3"/>
### *PyMC3*

Probabilistic Programming in Python:  Bayesian Modeling and Probabilistic Machine Learning with Theano

*homepage*: https://docs.pymc.io/

version   |versionsuffix    |toolchain
----------|-----------------|--------------------------------------------------------------------
``3.8``   |``-Python-3.7.4``|``foss/2019b``, ``intel/2019b``
``3.11.1``|                 |``foss/2021b``, ``fosscuda/2020b``, ``intel/2020b``, ``intel/2021b``

<a anchor="pymca"/>
### *pymca*

The PyMca X-Ray Fluorescence Toolkit, including PyMca5 and fisx.

*homepage*: https://github.com/vasole/pymca

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------
``5.6.3``|                 |``foss/2020b``, ``fosscuda/2020b``
``5.6.3``|``-Python-3.7.4``|``foss/2019b``, ``fosscuda/2019b``

<a anchor="pymemcache"/>
### *pymemcache*

A comprehensive, fast, pure-Python memcached client.

*homepage*: https://github.com/pinterest/pymemcache

version  |versionsuffix    |toolchain
---------|-----------------|-------------------------------
``2.1.1``|``-Python-3.6.4``|``foss/2018a``, ``intel/2018a``

<a anchor="pymol"/>
### *PyMOL*

PyMOL is a Python-enhanced molecular graphics tool. It excels at 3D              visualization of proteins, small molecules, density, surfaces, and trajectories. It also includes molecular editing, ray tracing, and movies. Open Source PyMOL   is free to everyone! 

*homepage*: https://github.com/schrodinger/pymol-open-source

version  |toolchain
---------|--------------
``2.5.0``|``foss/2020b``

<a anchor="pynast"/>
### *PyNAST*

PyNAST is a reimplementation of the NAST sequence aligner, which has  become a popular tool for adding new 16s rRNA sequences to existing 16s rRNA alignments.  This reimplementation is more flexible, faster, and easier to install and maintain than  the original NAST implementation.

*homepage*: https://biocore.github.io/pynast

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``1.2.2``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``1.2.2``|``-Python-2.7.15``|``foss/2018b``

<a anchor="pyobjcryst"/>
### *pyobjcryst*

Python bindings to ObjCryst++, the Object-Oriented Crystallographic Library.

*homepage*: https://github.com/diffpy/pyobjcryst

version        |versionsuffix    |toolchain
---------------|-----------------|---------------
``2.1.0.post2``|``-Python-3.8.2``|``intel/2020a``
``2.2.1``      |                 |``foss/2021b``

<a anchor="pyod"/>
### *PyOD*

PyOD is a comprehensive and scalable Python toolkit for detecting outlying objects in multivariate data.

*homepage*: https://pyod.readthedocs.io

version  |toolchain
---------|-------------------------------
``0.8.7``|``foss/2020b``, ``intel/2020b``

<a anchor="pyomo"/>
### *Pyomo*

Pyomo is a Python-based open-source software package that supports a diverse set of optimization  capabilities for formulating and analyzing optimization models. 

*homepage*: https://www.pyomo.org/

version  |versionsuffix     |toolchain
---------|------------------|------------------------------
``5.5.0``|``-Python-2.7.15``|``foss/2018b``
``5.5.0``|``-Python-3.6.6`` |``foss/2018b``
``6.0.1``|                  |``foss/2020b``, ``foss/2021a``
``6.4.2``|                  |``foss/2022a``

<a anchor="pyopencl"/>
### *PyOpenCL*

PyOpenCL lets you access GPUs and other massively parallel compute devices from Python.

*homepage*: https://mathema.tician.de/software/pyopencl/

version      |versionsuffix    |toolchain
-------------|-----------------|---------------------------------------------------
``2020.2.2`` |``-Python-3.7.4``|``fosscuda/2019b``
``2021.1.2`` |                 |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``
``2021.2.13``|                 |``foss/2021b``

<a anchor="pyopengl"/>
### *PyOpenGL*

PyOpenGL is the most common cross platform Python binding to OpenGL and related APIs.

*homepage*: http://pyopengl.sourceforge.net

version    |versionsuffix     |toolchain
-----------|------------------|---------------------------------------------------------
``3.1.1a1``|                  |``GCCcore/8.2.0``
``3.1.1a1``|``-Python-2.7.11``|``intel/2016a``
``3.1.1a1``|``-Python-2.7.12``|``foss/2016b``
``3.1.1a1``|``-Python-2.7.14``|``foss/2018a``, ``intel/2017b``
``3.1.3b2``|``-Python-2.7.14``|``intel/2018a``
``3.1.3b2``|``-Python-2.7.15``|``foss/2018b``
``3.1.5``  |                  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/8.3.0``
``3.1.6``  |                  |``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="pyparsing"/>
### *pyparsing*

The pyparsing module is an alternative approach to creating and executing simple grammars, vs. the traditional lex/yacc approach, or the use of regular expressions. The pyparsing module provides a library of classes that client code uses to construct the grammar directly in Python code.

*homepage*: https://github.com/pyparsing/pyparsing

version  |versionsuffix     |toolchain
---------|------------------|-----------------
``2.4.6``|``-Python-2.7.16``|``GCCcore/8.3.0``

<a anchor="pyplusplus"/>
### *pyplusplus*

Py++ is a code generator for Boost.Python that simplifies  writing Python bindings of a C/C++ library  The tool is implemented as a Python module which is controlled by a user script.

*homepage*: https://bitbucket.org/ompl/pyplusplus

version     |versionsuffix     |toolchain
------------|------------------|--------------
``20160707``|``-Python-2.7.11``|``foss/2016a``
``20160707``|``-Python-3.5.1`` |``foss/2016a``

<a anchor="pypower"/>
### *PYPOWER*

PYPOWER is a power flow and Optimal Power Flow (OPF) solver. It is a port of MATPOWER to the Python programming language.

*homepage*: https://github.com/rwl/PYPOWER

version   |toolchain
----------|--------------
``5.1.15``|``foss/2020b``

<a anchor="pyproj"/>
### *pyproj*

Python interface to PROJ4 library for cartographic transformations

*homepage*: https://pyproj4.github.io/pyproj

version        |versionsuffix    |toolchain
---------------|-----------------|--------------------------------------
``2.1.3``      |                 |``GCCcore/8.2.0``
``2.4.2``      |``-Python-3.7.4``|``GCCcore/8.3.0``
``2.6.1.post1``|``-Python-3.8.2``|``GCCcore/9.3.0``
``3.0.1``      |                 |``GCCcore/10.2.0``
``3.1.0``      |                 |``GCCcore/10.3.0``
``3.3.1``      |                 |``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``3.4.0``      |                 |``GCCcore/11.3.0``

<a anchor="pypsa"/>
### *PyPSA*

PyPSA is an open source toolbox for simulating and optimising modern power systems that include features such as conventional generators with unit commitment, variable wind and solar generation, storage units, coupling to other energy sectors, and mixed alternating and direct current networks. PyPSA is designed to scale well with large networks and long time series.

*homepage*: https://pypsa.readthedocs.io/

version   |toolchain
----------|--------------
``0.17.1``|``foss/2020b``

<a anchor="pyqstem"/>
### *pyqstem*

QSTEM is a program for quantitative image simulation in electron microscopy, including TEM, STEM and CBED image simulation.  This project interfaces the QSTEM code with Python and the Atomic Simulation Environment (ASE) to provide a single environment for building models, simulating and analysing images.

*homepage*: https://github.com/jacobjma/PyQSTEM

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------
``1.0.3``|``-ASE-3.22.0``  |``foss/2020b``, ``fosscuda/2020b``
``1.0.3``|``-Python-3.6.6``|``foss/2018b``, ``intel/2018b``

<a anchor="pyqt"/>
### *PyQt*

PyQt is a set of Python v2 and v3 bindings for Digia's Qt application framework.

*homepage*: http://www.riverbankcomputing.co.uk/software/pyqt

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``4.11.4``|``-Python-2.7.11``|``intel/2016a``
``4.11.4``|``-Python-2.7.12``|``intel/2016b``
``4.12``  |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``4.12``  |``-Python-2.7.13``|``intel/2017a``
``4.12.1``|``-Python-2.7.14``|``foss/2018a``
``4.12.3``|``-Python-2.7.15``|``fosscuda/2018b``

<a anchor="pyqt5"/>
### *PyQt5*

PyQt5 is a set of Python bindings for v5 of the Qt application framework from The Qt Company.

*homepage*: http://www.riverbankcomputing.co.uk/software/pyqt

version   |versionsuffix     |toolchain
----------|------------------|----------------------------------------------------------
``5.7``   |``-Python-2.7.11``|``foss/2016a``
``5.7.1`` |``-Python-2.7.12``|``intel/2016b``
``5.8.2`` |``-Python-2.7.13``|``intel/2017a``
``5.9.2`` |``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``, ``intel/2018a``
``5.9.2`` |``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``5.11.3``|``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``5.12.1``|``-Python-2.7.15``|``GCCcore/8.2.0``
``5.12.1``|``-Python-3.7.2`` |``GCCcore/8.2.0``
``5.13.2``|``-Python-3.7.4`` |``GCCcore/8.3.0``
``5.15.1``|                  |``GCCcore/10.2.0``
``5.15.1``|``-Python-3.8.2`` |``GCCcore/9.3.0``
``5.15.4``|                  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``5.15.5``|                  |``GCCcore/11.3.0``

<a anchor="pyqtgraph"/>
### *PyQtGraph*

PyQtGraph is a pure-python graphics and GUI library built on PyQt5/PySide2 and numpy.

*homepage*: http://www.pyqtgraph.org/

version   |versionsuffix    |toolchain
----------|-----------------|----------------------------------
``0.10.0``|``-Python-3.6.4``|``intel/2018a``
``0.10.0``|``-Python-3.6.6``|``foss/2018b``, ``intel/2018b``
``0.10.0``|``-Python-3.7.2``|``intel/2019a``
``0.11.0``|``-Python-3.7.4``|``foss/2019b``
``0.11.1``|                 |``foss/2020b``, ``fosscuda/2020b``
``0.12.3``|                 |``foss/2021a``

<a anchor="pyradiomics"/>
### *pyradiomics*

Open-source python package for the extraction of Radiomics features from 2D and 3D   images and binary masks.

*homepage*: https://pyradiomics.readthedocs.io/

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``3.0.1``|                 |``foss/2021a``
``3.0.1``|``-Python-3.7.4``|``foss/2019b``

<a anchor="pyre"/>
### *PyRe*

PyRe (Python Reliability) is a Python module for structural reliability analysis.

*homepage*: https://hackl.science/pyre

version           |versionsuffix    |toolchain
------------------|-----------------|-------------------------------
``5.0.3-20190221``|``-Python-3.7.4``|``foss/2019b``, ``intel/2019b``

<a anchor="pyretis"/>
### *PyRETIS*

PyRETIS is a Python library for rare event molecular simulations with emphasis on methods based  on transition interface sampling and replica exchange transition interface sampling.

*homepage*: http://www.pyretis.org

version  |versionsuffix    |toolchain
---------|-----------------|-------------------------------
``2.1.0``|``-Python-3.6.6``|``intel/2018b``
``2.5.0``|                 |``foss/2020b``, ``intel/2020b``
``2.5.0``|``-Python-3.8.2``|``foss/2020a``, ``intel/2020a``

<a anchor="pyringe"/>
### *pyringe*

Debugger capable of attaching to and injecting code into python processes.

*homepage*: https://github.com/google/pyringe

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.0.2``|``-Python-2.7.11``|``intel/2016a``

<a anchor="pyro-api"/>
### *pyro-api*

Pyro is a flexible, scalable deep probabilistic programming library built on PyTorch.

*homepage*: https://github.com/pyro-ppl/pyro-api

version  |toolchain
---------|------------------
``0.1.2``|``fosscuda/2020b``

<a anchor="pyro-ppl"/>
### *pyro-ppl*

Pyro is a flexible, scalable deep probabilistic programming library built on PyTorch.

*homepage*: https://github.com/pyro-ppl/pyro

version  |toolchain
---------|------------------
``1.5.2``|``fosscuda/2020b``

<a anchor="pyro4"/>
### *Pyro4*

Pyro means PYthon Remote Objects. It is a library that enables you to build applications in which  objects can talk to eachother over the network, with minimal programming effort.

*homepage*: https://pypi.python.org/pypi/Pyro4

version |versionsuffix     |toolchain
--------|------------------|--------------
``4.47``|``-Python-2.7.11``|``foss/2016a``

<a anchor="pyrosetta"/>
### *PyRosetta*

PyRosetta is an interactive Python-based interface to the powerful Rosetta molecular modeling suite. It enables users to design their own custom molecular modeling algorithms using Rosetta sampling methods and energy functions.

*homepage*: https://www.pyrosetta.org/

version          |versionsuffix    |toolchain
-----------------|-----------------|-----------------
``4.release-292``|``-Python-3.7.4``|``GCCcore/8.3.0``

<a anchor="pysam"/>
### *Pysam*

Pysam is a python module for reading and manipulating Samfiles.   It's a lightweight wrapper of the samtools C-API. Pysam also includes an interface for tabix.

*homepage*: https://github.com/pysam-developers/pysam

version     |versionsuffix     |toolchain
------------|------------------|--------------------------------------------------------------------------------------------------------------
``0.8.4``   |``-Python-2.7.12``|``intel/2016b``
``0.9.1.4`` |``-Python-2.7.12``|``foss/2016b``
``0.10.0``  |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``0.12.0.1``|``-Python-2.7.13``|``intel/2017a``
``0.12.0.1``|``-Python-2.7.14``|``intel/2017b``
``0.12.0.1``|``-Python-3.6.3`` |``intel/2017b``
``0.13``    |``-Python-2.7.14``|``intel/2017b``
``0.13.0``  |``-Python-3.6.3`` |``intel/2017b``
``0.14``    |``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``0.14``    |``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``0.14``    |``-Python-3.6.4`` |``intel/2018a``
``0.14.1``  |``-Python-2.7.14``|``foss/2018a``, ``intel/2018a``
``0.14.1``  |``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``0.15.1``  |``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``0.15.1``  |``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``0.15.2``  |                  |``GCC/8.2.0-2.31.1``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``0.15.3``  |                  |``GCC/8.3.0``, ``iccifort/2019.5.281``
``0.16.0.1``|                  |``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/8.3.0``, ``GCC/9.3.0``, ``iccifort/2020.1.217``, ``iccifort/2020.4.304``
``0.16.0.1``|``-Python-2.7.18``|``GCC/10.2.0``
``0.17.0``  |                  |``GCC/11.2.0``
``0.18.0``  |                  |``GCC/11.2.0``
``0.19.1``  |                  |``GCC/11.3.0``

<a anchor="pysamstats"/>
### *pysamstats*

A Python utility for calculating statistics against genome positions based on sequence alignments from a SAM or BAM file.

*homepage*: https://github.com/alimanfoo/pysamstats

version  |toolchain
---------|--------------
``1.1.2``|``foss/2020b``

<a anchor="pysat"/>
### *PySAT*

PySAT is a Python toolkit, which aims at providing a simple and unified interface to a number of state-of-art Boolean satisfiability (SAT) solvers as well as to a variety of cardinality and pseudo-Boolean encodings.

*homepage*: https://pysathq.github.io/

version        |versionsuffix    |toolchain
---------------|-----------------|--------------
``0.1.6.dev11``|``-Python-3.8.2``|``GCC/9.3.0``
``0.1.7.dev1`` |                 |``GCC/10.2.0``

<a anchor="pyscaf"/>
### *pyScaf*

pyScaf orders contigs from genome assemblies utilising several types of information

*homepage*: https://github.com/lpryszcz/pyScaf

version   |versionsuffix     |toolchain
----------|------------------|---------------
``0.12a4``|``-Python-2.7.14``|``intel/2017b``

<a anchor="pyscenic"/>
### *pySCENIC*

pySCENIC is a lightning-fast python implementation of the SCENIC pipeline (Single-Cell rEgulatory Network Inference and Clustering) which enables biologists to infer transcription factors, gene regulatory networks and cell types from single-cell RNA-seq data.

*homepage*: https://github.com/aertslab/pySCENIC

version   |versionsuffix    |toolchain
----------|-----------------|-------------------------------
``0.10.3``|``-Python-3.8.2``|``foss/2020a``, ``intel/2020a``

<a anchor="pyscf"/>
### *PySCF*

PySCF is an open-source collection of electronic structure modules powered by Python.

*homepage*: http://www.pyscf.org

version  |versionsuffix    |toolchain
---------|-----------------|-----------------------------------------------
``1.6.3``|``-Python-3.7.2``|``foss/2019a``
``1.7.6``|                 |``foss/2020b``, ``foss/2021a``, ``gomkl/2021a``

<a anchor="pysheds"/>
### *pysheds*

Simple and fast watershed delineation in python.

*homepage*: https://mattbartos.com/pysheds/

version    |toolchain
-----------|--------------
``0.2.7.1``|``foss/2020b``

<a anchor="pyshp"/>
### *pyshp*

Pure Python read/write support for ESRI Shapefile format

*homepage*: https://github.com/GeospatialPython/pyshp

version   |versionsuffix    |toolchain
----------|-----------------|------------------
``1.2.12``|``-Python-3.6.2``|``foss/2017b``
``2.1.3`` |                 |``GCCcore/10.2.0``

<a anchor="pyside2"/>
### *PySide2*

PySide2 is the official Python module from the Qt for Python project,     which provides access to the complete Qt 5.12+ framework.

*homepage*: https://pypi.python.org/pypi/PySide2

version     |toolchain
------------|------------------
``5.14.2.3``|``GCCcore/10.2.0``

<a anchor="pysndfx"/>
### *pysndfx*

A lightweight Python wrapper for SoX - Sound eXchange.  Supported effects range from EQ and compression to phasers, reverb and pitch shifters.

*homepage*: https://github.com/carlthome/python-audio-effects

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.3.6``|``-Python-3.7.4``|``foss/2019b``

<a anchor="pysolar"/>
### *Pysolar*

Pysolar is a collection of Python libraries for simulating the irradiation  of any point on earth by the sun.

*homepage*: https://pysolar.org/

version|versionsuffix    |toolchain
-------|-----------------|-----------------
``0.7``|``-Python-3.6.3``|``intel/2017b``
``0.7``|``-Python-3.6.4``|``intel/2018a``
``0.8``|``-Python-3.6.6``|``intel/2018b``
``0.8``|``-Python-3.7.2``|``GCCcore/8.2.0``

<a anchor="pyspoa"/>
### *pyspoa*

Python bindings to spoa.

*homepage*: https://github.com/nanoporetech/pyspoa

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------------------
``0.0.4``|``-Python-3.7.4``|``GCC/8.3.0``
``0.0.8``|                 |``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``

<a anchor="pysqlite"/>
### *pysqlite*

pysqlite is an interface to the SQLite 3.x embedded relational database engine.  It is almost fully compliant with the Python database API version 2.0 also exposes the unique features of SQLite.

*homepage*: https://pypi.python.org/pypi/pysqlite

version  |versionsuffix     |toolchain
---------|------------------|--------------
``2.8.2``|``-Python-2.7.11``|``foss/2016a``

<a anchor="pystan"/>
### *PyStan*

Python interface to Stan, a package for Bayesian inference  using the No-U-Turn sampler, a variant of Hamiltonian Monte Carlo.

*homepage*: https://github.com/stan-dev/pystan

version     |versionsuffix    |toolchain
------------|-----------------|-------------------------------
``2.19.0.0``|``-Python-3.6.4``|``intel/2018a``
``2.19.1.1``|                 |``foss/2020b``, ``intel/2020b``
``3.5.0``   |                 |``foss/2021b``

<a anchor="pysteps"/>
### *pysteps*

Pysteps is an open-source and community-driven Python library for probabilistic precipitation nowcasting, i.e. short-term ensemble prediction systems.

*homepage*: https://pysteps.github.io/

version  |toolchain
---------|--------------
``1.7.1``|``foss/2022a``

<a anchor="pystran"/>
### *pystran*

Toolset of dynamical model STRucture ANalysis algorithms

*homepage*: https://stijnvanhoey.github.io/pystran/

version       |versionsuffix     |toolchain
--------------|------------------|---------------
``2017.04.20``|``-Python-2.7.14``|``intel/2017b``

<a anchor="pytables"/>
### *PyTables*

PyTables is a package for managing hierarchical datasets and designed to efficiently and easily cope  with extremely large amounts of data. PyTables is built on top of the HDF5 library, using the Python language and the  NumPy package. It features an object-oriented interface that, combined with C extensions for the performance-critical  parts of the code (generated using Cython), makes it a fast, yet extremely easy to use tool for interactively browse,  process and search very large amounts of data. One important feature of PyTables is that it optimizes memory and disk  resources so that data takes much less space (specially if on-flight compression is used) than other solutions such as  relational or object oriented databases.

*homepage*: https://www.pytables.org

version    |versionsuffix     |toolchain
-----------|------------------|-----------------------------------------------------------------------------------
``3.2.2``  |``-Python-2.7.12``|``foss/2016b``
``3.2.3.1``|``-Python-2.7.12``|``intel/2016b``
``3.3.0``  |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``3.3.0``  |``-Python-3.5.2`` |``intel/2016b``
``3.4.2``  |``-Python-2.7.13``|``foss/2017a``
``3.4.2``  |``-Python-3.6.1`` |``intel/2017a``
``3.4.2``  |``-Python-3.6.3`` |``intel/2017b``
``3.4.2``  |``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``3.4.4``  |``-Python-2.7.15``|``foss/2018b``
``3.4.4``  |``-Python-3.5.1`` |``foss/2016a``
``3.4.4``  |``-Python-3.6.6`` |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``3.5.2``  |                  |``intel/2019a``
``3.5.2``  |``-Python-2.7.14``|``intel/2018a``
``3.5.2``  |``-Python-2.7.16``|``intel/2019b``
``3.5.2``  |``-Python-2.7.18``|``foss/2020b``
``3.6.1``  |                  |``foss/2020b``, ``foss/2021a``, ``foss/2021b``, ``fosscuda/2020b``, ``intel/2020b``
``3.6.1``  |``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``
``3.6.1``  |``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``

<a anchor="pytest"/>
### *pytest*

pytest: simple powerful testing with Python

*homepage*: https://pytest.org

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``3.0.1``|``-Python-2.7.11``|``foss/2016a``
``3.0.1``|``-Python-3.5.1`` |``foss/2016a``
``3.8.0``|``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``3.8.2``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``3.8.2``|``-Python-2.7.15``|``intel/2018b``
``3.8.2``|``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``3.8.2``|``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``4.3.0``|``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``4.4.0``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``4.4.0``|``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``6.0.1``|``-Python-3.7.4`` |``GCCcore/8.3.0``
``7.1.3``|                  |``GCCcore/11.2.0``

<a anchor="pytest-benchmark"/>
### *pytest-benchmark*

A pytest fixture for benchmarking code.

*homepage*: https://github.com/ionelmc/pytest-benchmark

version  |toolchain
---------|------------------
``3.4.1``|``GCCcore/10.2.0``

<a anchor="pytest-xdist"/>
### *pytest-xdist*

xdist: pytest distributed testing plugin  The pytest-xdist plugin extends pytest with some unique test execution modes:      * test run parallelization: if you have multiple CPUs or hosts you       can use those for a combined test run. This allows to speed up       development or to use special resources of remote machines.      * --looponfail: run your tests repeatedly in a subprocess. After         each run pytest waits until a file in your project changes and         then re-runs the previously failing tests. This is repeated         until all tests pass after which again a full run is         performed.      * Multi-Platform coverage: you can specify different Python       interpreters or different platforms and run tests in parallel on       all of them.  Before running tests remotely, pytest efficiently “rsyncs” your program source code to the remote place. All test results are reported back and displayed to your local terminal. You may specify different Python versions and interpreters.

*homepage*: https://github.com/pytest-dev/pytest-xdist

version  |toolchain
---------|--------------------------------------
``2.1.0``|``GCCcore/10.2.0``
``2.3.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``2.5.0``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="pythia"/>
### *PYTHIA*

PYTHIA is a standard tool for the generation of events in high-energy collisions,  comprising a coherent set of physics models for the evolution from a few-body hard  process to a complex multiparticle final state.

*homepage*: http://home.thep.lu.se/~torbjorn/Pythia.html

version  |versionsuffix     |toolchain
---------|------------------|---------------
``8.226``|``-Python-2.7.13``|``intel/2017a``

<a anchor="python"/>
### *Python*

Python is a programming language that lets you work more quickly and integrate your systems  more effectively.

*homepage*: https://python.org/

version   |versionsuffix    |toolchain
----------|-----------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.7.9`` |``-bare``        |``GCC/4.8.4``, ``GCC/4.9.2``
``2.7.10``|                 |``gimkl/2.11.5``
``2.7.10``|``-bare``        |``GCC/4.9.3-2.25``, ``GNU/4.9.3-2.25``
``2.7.11``|                 |``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``
``2.7.11``|``-bare``        |``GCC/4.9.3-2.25``
``2.7.11``|``-libX11-1.6.3``|``intel/2016a``
``2.7.12``|                 |``foss/2016b``, ``intel/2016b``, ``iomkl/2017a``
``2.7.12``|``-bare``        |``GCC/5.4.0-2.26``, ``GCCcore/4.9.3``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``
``2.7.13``|                 |``foss/2017a``, ``intel/2017a``
``2.7.13``|``-bare``        |``GCCcore/6.3.0``
``2.7.14``|                 |``foss/2017b``, ``foss/2018a``, ``fosscuda/2017b``, ``fosscuda/2018a``, ``intel/2017b``, ``intel/2018.01``, ``intel/2018a``, ``intelcuda/2017b``, ``iomkl/2018a``
``2.7.14``|``-bare``        |``GCCcore/6.4.0``
``2.7.15``|                 |``GCCcore/8.2.0``, ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``, ``iomkl/2018b``
``2.7.15``|``-bare``        |``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``2.7.16``|                 |``GCCcore/8.3.0``
``2.7.16``|``-bare``        |``GCCcore/8.3.0``
``2.7.18``|                 |``GCCcore/10.2.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``
``2.7.18``|``-bare``        |``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/9.3.0``
``3.5.1`` |                 |``foss/2016a``, ``intel/2016a``
``3.5.2`` |                 |``foss/2016.04``, ``foss/2016b``, ``intel/2016b``
``3.5.2`` |``-bare``        |``GCC/5.4.0-2.26``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``
``3.6.1`` |                 |``foss/2017a``, ``intel/2017a``
``3.6.2`` |                 |``foss/2017b``, ``intel/2017b``, ``intel/2018.00``
``3.6.3`` |                 |``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intel/2018.01``, ``intelcuda/2017b``
``3.6.4`` |                 |``foss/2017a``, ``foss/2018a``, ``fosscuda/2018a``, ``golf/2018a``, ``iimkl/2018a``, ``intel/2018a``, ``iomkl/2018.02``, ``iomkl/2018a``
``3.6.6`` |                 |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``, ``iomkl/2018b``
``3.7.0`` |                 |``foss/2018b``, ``intel/2018b``, ``iomkl/2018b``
``3.7.2`` |                 |``GCCcore/8.2.0``
``3.7.4`` |                 |``GCCcore/8.3.0``
``3.8.2`` |                 |``GCCcore/9.3.0``
``3.8.6`` |                 |``GCCcore/10.2.0``
``3.9.5`` |                 |``GCCcore/10.3.0``
``3.9.5`` |``-bare``        |``GCCcore/10.3.0``
``3.9.6`` |                 |``GCCcore/11.2.0``
``3.9.6`` |``-bare``        |``GCCcore/11.2.0``
``3.10.4``|                 |``GCCcore/11.3.0``
``3.10.4``|``-bare``        |``GCCcore/11.3.0``
``3.10.8``|                 |``GCCcore/12.2.0``
``3.10.8``|``-bare``        |``GCCcore/12.2.0``

<a anchor="python-docx"/>
### *python-docx*

python-docx is a Python library for creating and updating Microsoft Word (.docx) files

*homepage*: https://python-docx.readthedocs.io/en/latest/

version   |toolchain
----------|------------------
``0.8.11``|``GCCcore/10.2.0``

<a anchor="python-hl7"/>
### *python-hl7*

A simple library for parsing messages of Health Level 7 (HL7) version 2.x into Python objects.

*homepage*: https://github.com/johnpaulett/python-hl7

version  |versionsuffix    |toolchain
---------|-----------------|-----------------
``0.3.4``|``-Python-3.7.4``|``GCCcore/8.3.0``

<a anchor="python-igraph"/>
### *python-igraph*

Python interface to the igraph high performance graph library, primarily aimed at complex network  research and analysis.

*homepage*: https://igraph.org/python

version        |versionsuffix     |toolchain
---------------|------------------|----------------------------------
``0.7.1.post6``|``-Python-2.7.14``|``intel/2017b``
``0.7.1.post6``|``-Python-3.6.6`` |``foss/2018b``
``0.8.0``      |                  |``foss/2019b``, ``foss/2020a``
``0.9.0``      |                  |``foss/2020b``, ``fosscuda/2020b``
``0.9.6``      |                  |``foss/2021a``
``0.9.8``      |                  |``foss/2021b``

<a anchor="python-irodsclient"/>
### *python-irodsclient*

A python API for iRODS

*homepage*: https://github.com/irods/python-irodsclient

version  |toolchain
---------|--------------------------------------
``1.1.4``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``

<a anchor="python-isal"/>
### *python-isal*

Faster zlib and gzip compatible compression and decompression  by providing python bindings for the isa-l library. 

*homepage*: https://github.com/pycompression/python-isal

version   |toolchain
----------|--------------------------------------
``0.11.0``|``GCCcore/10.3.0``
``0.11.1``|``GCCcore/10.2.0``, ``GCCcore/11.2.0``
``1.1.0`` |``GCCcore/11.3.0``

<a anchor="python-levenshtein"/>
### *python-Levenshtein*

Python extension for computing string edit distances and similarities.

*homepage*: https://pypi.org/project/python-Levenshtein/

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.12.0``|``-Python-3.6.6``|``foss/2018b``
``0.12.0``|``-Python-3.7.4``|``foss/2019b``
``0.12.1``|                 |``foss/2020b``

<a anchor="python-libsbml"/>
### *python-libsbml*

LibSBML Python API.

*homepage*: https://sbml.org/

version   |toolchain
----------|--------------
``5.19.7``|``foss/2021a``

<a anchor="python-parasail"/>
### *python-parasail*

Python Bindings for the Parasail C Library

*homepage*: https://github.com/jeffdaily/parasail-python

version   |versionsuffix     |toolchain
----------|------------------|------------------------------------------------------------------
``1.1.12``|``-Python-2.7.14``|``intel/2018a``
``1.1.16``|``-Python-3.6.6`` |``foss/2018b``
``1.2``   |``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``
``1.2.2`` |``-Python-3.8.2`` |``intel/2020a``
``1.2.3`` |``-Python-3.8.2`` |``foss/2020a``
``1.2.4`` |                  |``foss/2020b``, ``foss/2021a``, ``foss/2021b``, ``fosscuda/2020b``

<a anchor="python-telegram-bot"/>
### *python-telegram-bot*

This library provides a pure Python, asynchronous interface for the Telegram Bot API.  It's compatible with Python versions 3.7+.

*homepage*: https://python-telegram-bot.org/

version   |toolchain
----------|------------------
``20.0a0``|``GCCcore/10.2.0``

<a anchor="python-weka-wrapper3"/>
### *python-weka-wrapper3*

Python3 wrapper for the Weka Machine Learning Workbench

*homepage*: https://github.com/fracpete/python-weka-wrapper3

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.1.11``|``-Python-3.7.4``|``foss/2019b``

<a anchor="pythran"/>
### *pythran*

Pythran is an ahead of time compiler for a subset of the Python language, with a focus on scientific computing. It takes a Python module annotated with a few interface description and turns it into a native Python module with the same interface, but (hopefully) faster. 

*homepage*: https://pythran.readthedocs.io

version        |versionsuffix    |toolchain
---------------|-----------------|--------------
``0.9.4.post1``|``-Python-3.7.4``|``foss/2019b``

<a anchor="pytorch"/>
### *PyTorch*

Tensors and Dynamic neural networks in Python with strong GPU acceleration. PyTorch is a deep learning framework that puts Python first.

*homepage*: https://pytorch.org/

version   |versionsuffix                |toolchain
----------|-----------------------------|----------------------------------
``0.3.1`` |``-Python-2.7.14``           |``fosscuda/2017b``
``0.3.1`` |``-Python-3.6.3``            |``fosscuda/2017b``
``0.3.1`` |``-Python-3.6.4``            |``intel/2018a``
``0.3.1`` |``-Python-3.6.4-CUDA-9.1.85``|``foss/2018a``
``0.4.1`` |``-Python-3.6.4``            |``intel/2018a``
``1.0.1`` |``-Python-3.6.6``            |``foss/2018b``, ``fosscuda/2018b``
``1.1.0`` |``-Python-3.7.2``            |``foss/2019a``
``1.2.0`` |``-Python-3.7.2``            |``foss/2019a``, ``fosscuda/2019a``
``1.3.1`` |                             |``fosscuda/2020b``
``1.3.1`` |``-Python-3.7.4``            |``foss/2019b``, ``fosscuda/2019b``
``1.4.0`` |``-Python-3.7.4``            |``foss/2019b``, ``fosscuda/2019b``
``1.6.0`` |``-Python-3.7.4``            |``foss/2019b``, ``fosscuda/2019b``
``1.6.0`` |``-Python-3.7.4-imkl``       |``fosscuda/2019b``
``1.7.1`` |                             |``foss/2020b``, ``fosscuda/2020b``
``1.7.1`` |``-Python-3.7.4``            |``fosscuda/2019b``
``1.7.1`` |``-Python-3.8.2``            |``fosscuda/2020a``
``1.8.1`` |                             |``foss/2020b``, ``fosscuda/2020b``
``1.8.1`` |``-Python-3.7.4``            |``fosscuda/2019b``
``1.9.0`` |                             |``foss/2020b``, ``fosscuda/2020b``
``1.9.0`` |``-imkl``                    |``fosscuda/2020b``
``1.10.0``|                             |``foss/2021a``, ``fosscuda/2020b``
``1.10.0``|``-CUDA-11.3.1``             |``foss/2021a``
``1.11.0``|``-CUDA-11.3.1``             |``foss/2021a``
``1.12.0``|                             |``foss/2022a``
``1.12.0``|``-CUDA-11.7.0``             |``foss/2022a``
``1.12.1``|                             |``foss/2021a``, ``foss/2022a``
``1.12.1``|``-CUDA-11.3.1``             |``foss/2021a``
``1.12.1``|``-CUDA-11.7.0``             |``foss/2022a``

<a anchor="pytorch-geometric"/>
### *PyTorch-Geometric*

PyTorch Geometric (PyG) is a geometric deep learning extension library for PyTorch.

*homepage*: https://github.com/rusty1s/pytorch_geometric

version  |versionsuffix                  |toolchain
---------|-------------------------------|----------------------------------
``1.3.2``|``-Python-3.7.4``              |``foss/2019b``
``1.4.2``|``-Python-3.7.4-PyTorch-1.4.0``|``foss/2019b``
``1.6.3``|                               |``foss/2020b``, ``fosscuda/2020b``
``1.6.3``|``-Python-3.7.4-PyTorch-1.8.1``|``fosscuda/2019b``
``2.0.1``|``-PyTorch-1.9.0``             |``fosscuda/2020b``
``2.1.0``|``-PyTorch-1.12.1-CUDA-11.3.1``|``foss/2021a``

<a anchor="pytorch-lightning"/>
### *PyTorch-Lightning*

PyTorch Lightning is the lightweight PyTorch wrapper for ML researchers.

*homepage*: https://pytorchlightning.ai

version  |versionsuffix   |toolchain
---------|----------------|--------------
``1.5.9``|                |``foss/2021a``
``1.5.9``|``-CUDA-11.3.1``|``foss/2021a``
``1.7.7``|                |``foss/2022a``
``1.7.7``|``-CUDA-11.7.0``|``foss/2022a``
``1.8.4``|                |``foss/2022a``
``1.8.4``|``-CUDA-11.7.0``|``foss/2022a``

<a anchor="pytorch3d"/>
### *PyTorch3D*

PyTorch3D is FAIR's library of reusable components for deep learning with 3D data.

*homepage*: https://pytorch3d.org/

version  |versionsuffix     |toolchain
---------|------------------|------------------
``0.4.0``|``-PyTorch-1.7.1``|``fosscuda/2020b``

<a anchor="pyvcf"/>
### *PyVCF*

A Variant Call Format reader for Python.

*homepage*: https://github.com/jamescasbon/PyVCF

version  |versionsuffix     |toolchain
---------|------------------|-------------
``0.6.8``|``-Python-2.7.16``|``GCC/8.3.0``

<a anchor="pywannier90"/>
### *pyWannier90*

A Wannier90 Python interface for VASP and PySCF

*homepage*: https://github.com/hungpham2017/pyWannier90

version       |toolchain
--------------|-------------------------------
``2021-12-07``|``foss/2021a``, ``gomkl/2021a``

<a anchor="pywavelets"/>
### *PyWavelets*

PyWavelets is open source wavelet transform software for Python.

*homepage*: https://pywavelets.readthedocs.io

version  |versionsuffix    |toolchain
---------|-----------------|-------------------
``1.1.1``|                 |``intelcuda/2020b``
``1.1.1``|``-Python-3.7.4``|``intel/2019b``

<a anchor="pyyaml"/>
### *PyYAML*

PyYAML is a YAML parser and emitter for the Python programming language.

*homepage*: https://github.com/yaml/pyyaml

version  |versionsuffix     |toolchain
---------|------------------|------------------------------------------------------------------------------------
``3.11`` |``-Python-2.7.11``|``intel/2016a``
``3.12`` |                  |``system``
``3.12`` |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``3.12`` |``-Python-2.7.13``|``intel/2017a``
``3.12`` |``-Python-2.7.14``|``foss/2017b``, ``foss/2018a``, ``fosscuda/2017b``, ``intel/2017b``, ``intel/2018a``
``3.12`` |``-Python-3.5.2`` |``intel/2016b``
``3.12`` |``-Python-3.6.1`` |``intel/2017a``
``3.12`` |``-Python-3.6.3`` |``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``
``3.12`` |``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``3.13`` |                  |``system``
``3.13`` |``-Python-2.7.15``|``fosscuda/2018b``, ``intel/2018b``
``3.13`` |``-Python-3.6.6`` |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``5.1``  |                  |``GCCcore/8.2.0``
``5.1.2``|                  |``GCCcore/8.3.0``
``5.3``  |                  |``GCCcore/9.3.0``
``5.3.1``|                  |``GCCcore/10.2.0``
``5.4.1``|                  |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``6.0``  |                  |``GCCcore/11.3.0``, ``GCCcore/12.2.0``

<a anchor="pyzmq"/>
### *PyZMQ*

Python bindings for ZeroMQ

*homepage*: https://www.zeromq.org/bindings:python

version   |versionsuffix          |toolchain
----------|-----------------------|-------------------------------
``15.2.0``|``-Python-2.7.11-zmq4``|``foss/2016a``, ``intel/2016a``
``15.2.0``|``-Python-3.5.1-zmq4`` |``intel/2016a``
``15.3.0``|``-Python-2.7.11-zmq4``|``foss/2016a``
``15.3.0``|``-Python-3.5.1-zmq4`` |``foss/2016a``
``15.4.0``|``-Python-2.7.12-zmq4``|``intel/2016b``
``15.4.0``|``-Python-3.5.2-zmq4`` |``intel/2016b``
``16.0.2``|``-Python-2.7.12-zmq4``|``foss/2016b``, ``intel/2016b``
``16.0.2``|``-Python-2.7.13-zmq4``|``foss/2017a``, ``intel/2017a``
``16.0.2``|``-Python-3.5.2-zmq4`` |``foss/2016b``, ``intel/2016b``
``16.0.3``|``-Python-2.7.14-zmq4``|``intel/2017b``
``17.0.0``|``-Python-2.7.14-zmq4``|``foss/2018a``
``17.0.0``|``-Python-3.6.4-zmq4`` |``foss/2018a``
``18.1.1``|``-Python-3.7.4``      |``GCCcore/8.3.0``
``22.3.0``|                       |``GCCcore/10.3.0``

<a anchor="q"/>
### *Q*


<a href="#q6">Q6</a> - <a href="#qca">QCA</a> - <a href="#qcat">qcat</a> - <a href="#qcg-pilotjob">QCG-PilotJob</a> - <a href="#qcint">qcint</a> - <a href="#qcxms">QCxMS</a> - <a href="#qd">QD</a> - <a href="#qdd">QDD</a> - <a href="#qemu">QEMU</a> - <a href="#qgis">QGIS</a> - <a href="#qhull">Qhull</a> - <a href="#qiime">QIIME</a> - <a href="#qiime2">QIIME2</a> - <a href="#qiskit">Qiskit</a> - <a href="#qjson">QJson</a> - <a href="#qml">QML</a> - <a href="#qpth">qpth</a> - <a href="#qrupdate">qrupdate</a> - <a href="#qscintilla">QScintilla</a> - <a href="#qt">Qt</a> - <a href="#qt5">Qt5</a> - <a href="#qt5webkit">Qt5Webkit</a> - <a href="#qtconsole">Qtconsole</a> - <a href="#qtkeychain">QtKeychain</a> - <a href="#qtltools">QTLtools</a> - <a href="#qtop">qtop</a> - <a href="#qtpy">QtPy</a> - <a href="#qualimap">Qualimap</a> - <a href="#quandl">Quandl</a> - <a href="#quantumespresso">QuantumESPRESSO</a> - <a href="#quast">QUAST</a> - <a href="#quazip">QuaZIP</a> - <a href="#quickff">QuickFF</a> - <a href="#quickpic">QuickPIC</a> - <a href="#quip">Quip</a> - <a href="#quorum">Quorum</a> - <a href="#qutip">QuTiP</a> - <a href="#qwt">Qwt</a> - <a href="#qwtpolar">QwtPolar</a>


<a anchor="q6"/>
### *Q6*

EVB, FEP and LIE simulator.

*homepage*: https://github.com/qusers/Q6

version     |toolchain
------------|---------------
``20180205``|``gompi/2019a``

<a anchor="qca"/>
### *QCA*

Taking a hint from the similarly-named Java Cryptography Architecture,   QCA aims to provide a straightforward and cross-platform crypto API, using Qt datatypes   and conventions. QCA separates the API from the implementation, using plugins known as   Providers. The advantage of this model is to allow applications to avoid linking to or   explicitly depending on any particular cryptographic library. This allows one to easily   change or upgrade crypto implementations without even needing to recompile the   application! QCA should work everywhere Qt does, including Windows/Unix/MacOSX.

*homepage*: http://delta.affinix.com/qca/

version  |toolchain
---------|--------------------------------------------------
``2.1.0``|``foss/2016a``, ``intel/2016b``
``2.1.3``|``GCCcore/8.2.0``, ``foss/2016b``, ``intel/2016b``

<a anchor="qcat"/>
### *qcat*

qcat is a Python command-line tool for demultiplexing Oxford Nanopore reads from FASTQ files

*homepage*: https://github.com/nanoporetech/qcat/releases

version  |versionsuffix    |toolchain
---------|-----------------|-------------------------------
``1.1.0``|``-Python-3.7.4``|``foss/2019b``, ``intel/2019b``
``1.1.0``|``-Python-3.8.2``|``intel/2020a``

<a anchor="qcg-pilotjob"/>
### *QCG-PilotJob*

A python service for easy execution of many tasks inside a single allocation.

*homepage*: https://qcg-pilotjob.readthedocs.org

version   |toolchain
----------|--------------
``0.12.3``|``foss/2021a``

<a anchor="qcint"/>
### *qcint*

libcint is an open source library for analytical Gaussian integrals. qcint is an optimized libcint branch for the x86-64 platform.

*homepage*: http://wiki.sunqm.net/libcint

version   |toolchain
----------|--------------
``3.0.18``|``foss/2019a``

<a anchor="qcxms"/>
### *QCxMS*

QCxMS is a quantum chemical based program to calculate electron ionization (EI)  and collision induced dissociation (CID) mass spectra using Born-Oppenheimer Molecular Dynamics  (BO-MD). It is the successor of the QCEIMS program, in which the EI part is exchanged to x to  account for the greater general applicibility of the program. 

*homepage*: https://xtb-docs.readthedocs.io/en/latest/qcxms_doc/qcxms.html

version  |toolchain
---------|----------
``5.0.3``|``system``

<a anchor="qd"/>
### *QD*

Quad Double computation package

*homepage*: https://github.com/scibuilder/QD

version   |versionsuffix|toolchain
----------|-------------|--------------
``0.8.9`` |             |``foss/2021a``
``2.3.17``|``-20160110``|``NVHPC/21.2``

<a anchor="qdd"/>
### *QDD*

A user-friendly program to select microsatellite markers and design primers  from large sequencing projects.

*homepage*: http://net.imbe.fr/~emeglecz/qdd.html

version  |versionsuffix   |toolchain
---------|----------------|---------------
``3.1.2``|``-Perl-5.28.0``|``intel/2018b``

<a anchor="qemu"/>
### *QEMU*

QEMU is a generic and open source machine emulator and virtualizer.

*homepage*: https://www.qemu.org/

version   |toolchain
----------|---------------
``2.10.1``|``intel/2017b``

<a anchor="qgis"/>
### *QGIS*

QGIS is a user friendly Open Source Geographic Information System (GIS)

*homepage*: http://www.qgis.org/

version    |versionsuffix     |toolchain
-----------|------------------|---------------
``2.14.12``|``-Python-2.7.12``|``intel/2016b``
``2.18.4`` |``-Python-2.7.12``|``foss/2016b``
``3.4.12`` |``-Python-3.7.2`` |``foss/2019a``

<a anchor="qhull"/>
### *Qhull*

Qhull computes the convex hull, Delaunay triangulation, Voronoi diagram,  halfspace intersection about a point, furthest-site Delaunay triangulation,  and furthest-site Voronoi diagram. The source code runs in 2-d, 3-d, 4-d, and  higher dimensions. Qhull implements the Quickhull algorithm for computing the  convex hull. 

*homepage*: http://www.qhull.org

version   |toolchain
----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2015.2``|``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``
``2019.1``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``2020.2``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="qiime"/>
### *QIIME*

QIIME is an open-source bioinformatics pipeline for performing microbiome analysis  from raw DNA sequencing data.

*homepage*: http://qiime.org/

version  |toolchain
---------|----------
``1.9.1``|``system``

<a anchor="qiime2"/>
### *QIIME2*

QIIME is an open-source bioinformatics pipeline for performing microbiome analysis  from raw DNA sequencing data.

*homepage*: http://qiime2.org/

version    |toolchain
-----------|----------
``2017.10``|``system``
``2018.2`` |``system``
``2019.4`` |``system``
``2019.7`` |``system``
``2020.8`` |``system``
``2020.11``|``system``
``2021.8`` |``system``
``2022.8`` |``system``

<a anchor="qiskit"/>
### *Qiskit*

Qiskit is an open-source framework for working with noisy quantum computers  at the level of pulses, circuits, and algorithms.

*homepage*: https://qiskit.org

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.11.1``|``-Python-3.7.2``|``foss/2019a``
``0.12.0``|``-Python-3.7.2``|``foss/2019a``
``0.31.0``|                 |``foss/2021a``

<a anchor="qjson"/>
### *QJson*

QJson is a Qt-based library that maps JSON data to QVariant objects and vice versa.

*homepage*: http://qjson.sourceforge.net/

version  |toolchain
---------|----------------------------------------------------------------------
``0.9.0``|``GCCcore/10.2.0``, ``GCCcore/8.2.0``, ``foss/2016b``, ``intel/2016b``

<a anchor="qml"/>
### *QML*

QML is a Python2/3-compatible toolkit for representation learning of properties  of molecules and solids.

*homepage*: https://www.qmlcode.org

version   |versionsuffix     |toolchain
----------|------------------|---------------
``0.2.10``|``-Python-2.7.13``|``intel/2017a``

<a anchor="qpth"/>
### *qpth*

A fast and differentiable QP solver for PyTorch. 

*homepage*: https://locuslab.github.io/qpth/

version            |versionsuffix    |toolchain
-------------------|-----------------|--------------
``0.0.13-20190626``|``-Python-3.7.2``|``foss/2019a``

<a anchor="qrupdate"/>
### *qrupdate*

qrupdate is a Fortran library for fast updates of QR and Cholesky decompositions.

*homepage*: https://sourceforge.net/projects/qrupdate/

version  |toolchain
---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.1.2``|``GCC/5.4.0-2.26``, ``GCC/8.2.0-2.31.1``, ``GCCcore/10.2.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/6.4.0``, ``GCCcore/8.3.0``, ``foss/2016a``, ``foss/2018a``, ``foss/2018b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``

<a anchor="qscintilla"/>
### *QScintilla*

QScintilla is a port to Qt of Neil Hodgson's Scintilla C++ editor control

*homepage*: https://www.riverbankcomputing.com/software/qscintilla

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``2.9.4`` |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``2.10``  |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``2.11.2``|``-Python-3.7.2`` |``GCCcore/8.2.0``

<a anchor="qt"/>
### *Qt*

Qt is a comprehensive cross-platform C++ application framework.

*homepage*: http://qt.io/

version  |versionsuffix   |toolchain
---------|----------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``3.3.8``|                |``intel/2016a``
``4.8.6``|                |``system``
``4.8.7``|                |``GCCcore/8.2.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``fosscuda/2018b``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``
``4.8.7``|``-GLib-2.48.0``|``foss/2016a``, ``intel/2016a``

<a anchor="qt5"/>
### *Qt5*

Qt is a comprehensive cross-platform C++ application framework.

*homepage*: https://qt.io/

version   |toolchain
----------|------------------------------------------------------------------------------------
``5.6.0`` |``foss/2016a``, ``intel/2016a``
``5.7.0`` |``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``5.7.1`` |``intel/2016b``
``5.8.0`` |``foss/2017a``, ``foss/2017b``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``
``5.9.3`` |``foss/2017b``
``5.9.8`` |``fosscuda/2018b``
``5.10.1``|``foss/2018a``, ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018a``, ``intel/2018b``
``5.11.2``|``foss/2018b``
``5.12.3``|``GCCcore/8.2.0``
``5.13.1``|``GCCcore/8.3.0``
``5.14.1``|``GCCcore/9.3.0``
``5.14.2``|``GCCcore/10.2.0``
``5.15.2``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``5.15.5``|``GCCcore/11.3.0``
``5.15.7``|``GCCcore/12.2.0``

<a anchor="qt5webkit"/>
### *Qt5Webkit*

Qt Port of WebKit. WebKit is an open source web browser engine.

*homepage*: https://github.com/qt/qtwebkit

version           |toolchain
------------------|------------------
``5.212.0-alpha3``|``GCCcore/8.2.0``
``5.212.0-alpha4``|``GCCcore/10.2.0``

<a anchor="qtconsole"/>
### *Qtconsole*

A rich Qt-based console for working with Jupyter kernels, supporting rich media  output, session export, and more. The Qtconsole is a very lightweight application that largely feels like a        terminal, but provides a number of enhancements only possible in a GUI, such as  inline figures, proper multiline editing with syntax highlighting, graphical     calltips, and more.

*homepage*: https://jupyter.org/

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------
``4.7.7``|``-Python-3.7.4``|``foss/2019b``, ``fosscuda/2019b``
``5.0.2``|                 |``GCCcore/10.2.0``
``5.3.2``|                 |``GCCcore/11.2.0``

<a anchor="qtkeychain"/>
### *QtKeychain*

Platform-independent Qt API for storing passwords securely.

*homepage*: https://github.com/frankosterfeld/qtkeychain

version  |toolchain
---------|---------------------------------
``0.9.1``|``GCCcore/8.2.0``, ``foss/2018b``

<a anchor="qtltools"/>
### *QTLtools*

QTLtools is a tool set for molecular QTL discovery and analysis. It allows to go from the raw sequence data to collection of molecular Quantitative Trait Loci (QTLs)  in few easy-to-perform steps.

*homepage*: https://qtltools.github.io/qtltools/

version  |toolchain
---------|---------------
``1.1``  |``intel/2016b``
``1.3.1``|``foss/2020b``

<a anchor="qtop"/>
### *qtop*

qtop is a nifty command-line tool for monitoring queueing systems, esp. PBS/torque.  It tries to fit as much information as possible in your screen's real estate,  by stitching together the output of commands like pbsnodes -a, qstat & qstat -q.  It is possible to write wrappers for other platforms -people have done so for SGE, OAR etc-  or, even examine traces offline and present the sampled information.

*homepage*: http://cern.ch/fotis/QTOP/

version|versionsuffix|toolchain
-------|-------------|----------
``53`` |``-1``       |``system``

<a anchor="qtpy"/>
### *QtPy*

QtPy is a small abstraction layer that lets you write applications using a       single API call to either PyQt or PySide.  It provides support for PyQt5, PyQt4, PySide2 and PySide.

*homepage*: https://github.com/spyder-ide/qtpy

version  |versionsuffix    |toolchain
---------|-----------------|------------------
``1.9.0``|                 |``GCCcore/10.2.0``
``1.9.0``|``-Python-3.7.4``|``GCCcore/8.3.0``
``2.2.1``|                 |``GCCcore/11.2.0``

<a anchor="qualimap"/>
### *Qualimap*

Qualimap 2 is a platform-independent application written in Java and R that provides both  a Graphical User Inteface (GUI) and a command-line interface to facilitate the quality control of  alignment sequencing data and its derivatives like feature counts.

*homepage*: http://qualimap.bioinfo.cipf.es/

version  |versionsuffix|toolchain
---------|-------------|--------------
``2.2.1``|``-R-3.6.0`` |``foss/2019a``
``2.2.1``|``-R-4.0.3`` |``foss/2020b``
``2.2.1``|``-R-4.1.2`` |``foss/2021b``

<a anchor="quandl"/>
### *Quandl*

A Python library for Quandl’s RESTful API.

*homepage*: https://pypi.python.org/pypi/Quandl

version  |versionsuffix    |toolchain
---------|-----------------|------------------------------
``3.4.2``|``-Python-3.6.4``|``intel/2018a``
``3.4.8``|                 |``foss/2019a``
``3.6.1``|                 |``foss/2020b``, ``foss/2021a``

<a anchor="quantumespresso"/>
### *QuantumESPRESSO*

Quantum ESPRESSO  is an integrated suite of computer codes for electronic-structure calculations and materials modeling at the nanoscale. It is based on density-functional theory, plane waves, and pseudopotentials (both norm-conserving and ultrasoft). 

*homepage*: https://www.quantum-espresso.org

version  |versionsuffix|toolchain
---------|-------------|-------------------------------------------------------------------------------------------------
``5.3.0``|             |``intel/2016.02-GCC-4.9``
``5.4.0``|             |``intel/2016.02-GCC-4.9``
``5.4.0``|``-hybrid``  |``foss/2016b``
``6.0``  |             |``intel/2016b``
``6.1``  |             |``intel/2017a``
``6.2``  |             |``intel/2017b``
``6.2.1``|             |``iomkl/2017b``
``6.3``  |             |``foss/2018b``, ``intel/2018b``
``6.4.1``|             |``intel/2019a``
``6.5``  |             |``intel/2019a``, ``intel/2019b``
``6.6``  |             |``foss/2019b``, ``foss/2020a``, ``foss/2020b``, ``intel/2019b``
``6.7``  |             |``foss/2019b``, ``foss/2020b``, ``foss/2021a``, ``intel/2019b``, ``intel/2021a``, ``iomkl/2019b``
``6.8``  |             |``foss/2021a``, ``foss/2021b``, ``intel/2021a``
``7.0``  |             |``foss/2021b``, ``intel/2021b``
``7.1``  |             |``foss/2022a``, ``intel/2022a``

<a anchor="quast"/>
### *QUAST*

QUAST evaluates genome assemblies by computing various metrics. It works both with and without reference genomes. The tool accepts multiple assemblies, thus is suitable for comparison.

*homepage*: https://github.com/ablab/quast

version  |versionsuffix     |toolchain
---------|------------------|----------------------------------------------
``4.6.0``|``-Python-3.5.2`` |``foss/2016b``
``4.6.3``|``-Python-3.6.4`` |``foss/2018a``
``5.0.2``|                  |``foss/2020b``, ``foss/2021a``, ``foss/2021b``
``5.0.2``|``-Python-2.7.15``|``foss/2018b``, ``foss/2019a``
``5.0.2``|``-Python-2.7.18``|``foss/2020b``
``5.0.2``|``-Python-3.7.2`` |``foss/2019a``
``5.0.2``|``-Python-3.8.2`` |``foss/2020a``

<a anchor="quazip"/>
### *QuaZIP*

QuaZIP is the C++ wrapper for Gilles Vollant's ZIP/UNZIP   package (AKA Minizip) using Trolltech's Qt library.

*homepage*: https://stachenov.github.io/quazip

version  |toolchain
---------|-----------------
``0.8.1``|``GCCcore/8.2.0``

<a anchor="quickff"/>
### *QuickFF*

QuickFF is a Python package developed at the Center for Molecular Modeling (CMM) to quickly derive accurate force fields from ab initio calculations.

*homepage*: https://molmod.github.io/QuickFF/

version  |versionsuffix     |toolchain
---------|------------------|---------------
``2.1.4``|``-Python-2.7.12``|``intel/2016b``
``2.2.0``|``-Python-2.7.14``|``intel/2017b``
``2.2.4``|``-Python-3.7.2`` |``intel/2019a``
``2.2.4``|``-Python-3.8.2`` |``intel/2020a``
``2.2.7``|``-Python-3.8.2`` |``intel/2020a``

<a anchor="quickpic"/>
### *QuickPIC*

QuickPIC is a 3D parallel (MPI & OpenMP Hybrid) Quasi-Static PIC code,  which is developed based on the framework UPIC. QuickPIC can efficiently simulate plasma  based accelerator problems. This is the UCLA Plasma Simulation Group's official open-source  repository for QuickPIC.

*homepage*: https://github.com/UCLA-Plasma-Simulation-Group/QuickPIC-OpenSource

version     |toolchain
------------|---------------
``20210224``|``gompi/2021b``

<a anchor="quip"/>
### *Quip*

Quip compresses next-generation sequencing data with extreme prejudice.  It supports input and output in the FASTQ and SAM/BAM formats,  compressing large datasets to as little as 15% of their original size.

*homepage*: http://homes.cs.washington.edu/~dcjones/quip

version  |toolchain
---------|-------------
``1.1.8``|``GCC/4.8.2``

<a anchor="quorum"/>
### *Quorum*

QuorUM is an error corrector for Illumina reads

*homepage*: http://www.genome.umd.edu/quorum.html

version  |toolchain
---------|---------------
``1.1.1``|``intel/2017a``

<a anchor="qutip"/>
### *QuTiP*

QuTiP is open-source software for simulating the dynamics of open quantum systems.

*homepage*: http://qutip.org

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``4.1.0``|``-Python-2.7.12``|``intel/2016b``
``4.3.1``|``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``

<a anchor="qwt"/>
### *Qwt*

The Qwt library contains GUI Components and utility classes which are primarily useful for programs  with a technical background.

*homepage*: https://qwt.sourceforge.io/

version  |toolchain
---------|----------------------------------------------------
``6.1.2``|``intel/2016a``
``6.1.3``|``foss/2016b``, ``intel/2016b``
``6.1.4``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``foss/2018b``
``6.1.5``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``6.2.0``|``GCCcore/10.3.0``

<a anchor="qwtpolar"/>
### *QwtPolar*

The QwtPolar library contains classes for displaying values on a polar coordinate system.

*homepage*: http://qwtpolar.sourceforge.net/

version  |toolchain
---------|----------------------------------------------------------------------
``1.1.1``|``GCCcore/10.2.0``, ``GCCcore/8.2.0``, ``foss/2016b``, ``intel/2016b``

<a anchor="r"/>
### *R*


<a href="#r">R</a> - <a href="#r-bundle-bioconductor">R-bundle-Bioconductor</a> - <a href="#r-inla">R-INLA</a> - <a href="#r-keras">R-keras</a> - <a href="#r-opencv">R-opencv</a> - <a href="#r-tesseract">R-tesseract</a> - <a href="#racon">Racon</a> - <a href="#radeontop">radeontop</a> - <a href="#ragoo">RaGOO</a> - <a href="#ragout">Ragout</a> - <a href="#ragtag">RagTag</a> - <a href="#rampart">rampart</a> - <a href="#randfold">randfold</a> - <a href="#randrproto">randrproto</a> - <a href="#rapidjson">RapidJSON</a> - <a href="#rapidtide">rapidtide</a> - <a href="#rapsearch2">RAPSearch2</a> - <a href="#rascaf">Rascaf</a> - <a href="#raspa2">RASPA2</a> - <a href="#rasterio">rasterio</a> - <a href="#rasterstats">rasterstats</a> - <a href="#ratatosk">Ratatosk</a> - <a href="#raxml">RAxML</a> - <a href="#raxml-ng">RAxML-NG</a> - <a href="#ray-assembler">Ray-assembler</a> - <a href="#ray-project">Ray-project</a> - <a href="#raysect">Raysect</a> - <a href="#rbfopt">RBFOpt</a> - <a href="#rclone">rclone</a> - <a href="#rcorrector">Rcorrector</a> - <a href="#rcppgsl">RcppGSL</a> - <a href="#rcuda">rCUDA</a> - <a href="#rdflib">RDFlib</a> - <a href="#rdkit">RDKit</a> - <a href="#rdp-classifier">RDP-Classifier</a> - <a href="#re2">RE2</a> - <a href="#re2c">re2c</a> - <a href="#reads2snp">Reads2snp</a> - <a href="#reapr">Reapr</a> - <a href="#recon">RECON</a> - <a href="#red">Red</a> - <a href="#redis">Redis</a> - <a href="#redis-py">redis-py</a> - <a href="#redundans">Redundans</a> - <a href="#reframe">ReFrame</a> - <a href="#regionmask">regionmask</a> - <a href="#regtools">RegTools</a> - <a href="#relate">Relate</a> - <a href="#relion">RELION</a> - <a href="#rematch">ReMatCh</a> - <a href="#remora">REMORA</a> - <a href="#renderproto">renderproto</a> - <a href="#repasthpc">RepastHPC</a> - <a href="#repeatmasker">RepeatMasker</a> - <a href="#repeatmodeler">RepeatModeler</a> - <a href="#repeatscout">RepeatScout</a> - <a href="#request">request</a> - <a href="#requests">requests</a> - <a href="#rerconverge">RERconverge</a> - <a href="#resolos">resolos</a> - <a href="#retworkx">retworkx</a> - <a href="#revbayes">RevBayes</a> - <a href="#rgdal">rgdal</a> - <a href="#rgeos">rgeos</a> - <a href="#rgurobi">Rgurobi</a> - <a href="#rhdf5">rhdf5</a> - <a href="#rheotool">RheoTool</a> - <a href="#rickflow">rickflow</a> - <a href="#rinchi">RInChI</a> - <a href="#rioxarray">rioxarray</a> - <a href="#rising">rising</a> - <a href="#rjags">rjags</a> - <a href="#rlcard">RLCard</a> - <a href="#rmath">Rmath</a> - <a href="#rmats-turbo">rMATS-turbo</a> - <a href="#rmblast">RMBlast</a> - <a href="#rna-bloom">RNA-Bloom</a> - <a href="#rna-seqc">RNA-SeQC</a> - <a href="#rnaclust">RNAclust</a> - <a href="#rnacode">RNAcode</a> - <a href="#rnaindel">RNAIndel</a> - <a href="#rnammer">RNAmmer</a> - <a href="#rnaquast">rnaQUAST</a> - <a href="#rnaz">RNAz</a> - <a href="#rnbeads">RnBeads</a> - <a href="#roary">Roary</a> - <a href="#rocm">ROCm</a> - <a href="#rocm-cmake">rocm-cmake</a> - <a href="#rocm-compilersupport">ROCm-CompilerSupport</a> - <a href="#rocm-smi">rocm-smi</a> - <a href="#rocminfo">rocminfo</a> - <a href="#rocr-runtime">ROCR-Runtime</a> - <a href="#roct-thunk-interface">ROCT-Thunk-Interface</a> - <a href="#roi_pac">ROI_PAC</a> - <a href="#rome">ROME</a> - <a href="#root">ROOT</a> - <a href="#root_numpy">root_numpy</a> - <a href="#rootpy">rootpy</a> - <a href="#rosetta">Rosetta</a> - <a href="#rpmrebuild">rpmrebuild</a> - <a href="#rpy2">rpy2</a> - <a href="#rqgis3">RQGIS3</a> - <a href="#rsem">RSEM</a> - <a href="#rseqc">RSeQC</a> - <a href="#rstan">RStan</a> - <a href="#rstanarm">rstanarm</a> - <a href="#rstudio-server">RStudio-Server</a> - <a href="#rtg-tools">RTG-Tools</a> - <a href="#rtree">Rtree</a> - <a href="#ruamel.yaml">ruamel.yaml</a> - <a href="#ruby">Ruby</a> - <a href="#ruby-tk">Ruby-Tk</a> - <a href="#ruffus">ruffus</a> - <a href="#rust">Rust</a>


<a anchor="r"/>
### *R*

R is a free software environment for statistical computing  and graphics.

*homepage*: https://www.r-project.org/

version  |versionsuffix                |toolchain
---------|-----------------------------|---------------------------------------------------
``3.2.3``|                             |``foss/2016a``, ``foss/2016b``, ``intel/2016a``
``3.2.3``|``-bare``                    |``foss/2016a``, ``intel/2016a``
``3.2.3``|``-libX11-1.6.3``            |``intel/2016a``
``3.3.1``|                             |``foss/2016a``, ``foss/2016b``, ``intel/2016b``
``3.3.3``|``-X11-20160819``            |``foss/2016b``, ``intel/2016b``
``3.3.3``|``-X11-20170314``            |``intel/2017a``
``3.4.0``|``-X11-20170314``            |``intel/2017a``
``3.4.1``|``-X11-20160819``            |``foss/2016b``
``3.4.3``|``-X11-20171023``            |``foss/2017b``, ``intel/2017b``
``3.4.3``|``-X11-20171023-HDF5-1.8.19``|``intel/2017b``
``3.4.4``|``-X11-20180131``            |``foss/2018a``, ``intel/2018a``, ``iomkl/2018a``
``3.5.0``|``-X11-20180131``            |``iomkl/2018a``
``3.5.1``|                             |``foss/2018b``, ``intel/2018b``
``3.5.1``|``-Python-2.7.15``           |``foss/2018b``
``3.5.1``|``-bare``                    |``foss/2018b``
``3.6.0``|                             |``foss/2019a``, ``fosscuda/2019a``, ``intel/2019a``
``3.6.2``|                             |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``
``3.6.3``|                             |``foss/2020a``
``4.0.0``|                             |``foss/2020a``, ``fosscuda/2020a``
``4.0.3``|                             |``foss/2020b``, ``fosscuda/2020b``
``4.0.4``|                             |``foss/2020b``, ``fosscuda/2020b``
``4.0.5``|                             |``foss/2020b``, ``fosscuda/2020b``
``4.1.0``|                             |``foss/2021a``
``4.1.2``|                             |``foss/2021b``
``4.2.0``|                             |``foss/2021b``
``4.2.1``|                             |``foss/2022a``

<a anchor="r-bundle-bioconductor"/>
### *R-bundle-Bioconductor*

Bioconductor provides tools for the analysis and coprehension  of high-throughput genomic data.

*homepage*: https://bioconductor.org

version |versionsuffix|toolchain
--------|-------------|-------------------------------
``3.2`` |``-R-3.2.3`` |``foss/2016a``, ``intel/2016a``
``3.3`` |``-R-3.3.1`` |``intel/2016b``
``3.5`` |``-R-3.4.0`` |``intel/2017a``
``3.6`` |``-R-3.4.3`` |``foss/2017b``, ``intel/2017b``
``3.6`` |``-R-3.4.4`` |``intel/2018a``
``3.7`` |``-R-3.5.0`` |``iomkl/2018a``
``3.7`` |``-R-3.5.1`` |``foss/2018b``
``3.8`` |``-R-3.5.1`` |``foss/2018b``
``3.9`` |``-R-3.6.0`` |``foss/2019a``
``3.10``|             |``foss/2019b``
``3.11``|``-R-4.0.0`` |``foss/2020a``
``3.12``|``-R-4.0.3`` |``foss/2020b``
``3.13``|``-R-4.1.0`` |``foss/2021a``
``3.14``|``-R-4.1.2`` |``foss/2021b``
``3.15``|``-R-4.2.0`` |``foss/2021b``
``3.15``|``-R-4.2.1`` |``foss/2022a``

<a anchor="r-inla"/>
### *R-INLA*

R-INLA is a package in R that do approximate Bayesian inference for Latent Gaussian Models.

*homepage*: https://www.r-inla.org

version     |versionsuffix|toolchain
------------|-------------|--------------
``21.05.02``|``-R-4.0.4`` |``foss/2020b``

<a anchor="r-keras"/>
### *R-keras*

Interface to 'Keras' <https://keras.io>, a high-level neural networks 'API'. 

*homepage*: https://cran.r-project.org/web/packages/keras

version    |versionsuffix            |toolchain
-----------|-------------------------|----------------------------------
``2.1.6``  |``-R-3.4.4``             |``foss/2018a``
``2.2.5.0``|``-Python-3.7.2-R-3.6.0``|``foss/2019a``, ``fosscuda/2019a``
``2.2.5.0``|``-Python-3.7.4-R-3.6.2``|``foss/2019b``, ``fosscuda/2019b``
``2.4.0``  |``-R-4.0.4``             |``foss/2020b``, ``fosscuda/2020b``

<a anchor="r-opencv"/>
### *R-opencv*

Experimenting with computer vision and machine learning in R. This package exposes some of the available OpenCV algorithms, such as edge, body or face detection. These can either be applied to analyze static images, or to filter live video footage from a camera device.

*homepage*: https://cran.r-project.org/web/packages/opencv/

version  |versionsuffix|toolchain
---------|-------------|--------------
``0.2.0``|``-R-4.0.0`` |``foss/2020a``

<a anchor="r-tesseract"/>
### *R-tesseract*

The R extension for using tesseract

*homepage*: https://cran.r-project.org/package=tesseract

version|versionsuffix|toolchain
-------|-------------|--------------
``4.0``|``-R-3.5.1`` |``foss/2018b``

<a anchor="racon"/>
### *Racon*

Ultrafast consensus module for raw de novo genome assembly of long uncorrected reads.

*homepage*: https://github.com/lbcb-sci/racon

version   |toolchain
----------|--------------------------------------
``1.3.2`` |``GCCcore/8.2.0``
``1.4.7`` |``GCCcore/8.2.0``, ``gcccuda/2019b``
``1.4.10``|``GCC/7.3.0-2.30``
``1.4.13``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.4.21``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``1.5.0`` |``GCCcore/11.2.0``

<a anchor="radeontop"/>
### *radeontop*

View your GPU utilization, both for the total activity percent and individual blocks.

*homepage*: https://github.com/clbr/radeontop

version|toolchain
-------|------------------
``1.3``|``GCCcore/10.3.0``

<a anchor="ragoo"/>
### *RaGOO*

A tool to order and orient genome assembly contigs via Minimap2 alignments to a reference genome

*homepage*: https://github.com/malonge/RaGOO

version |versionsuffix    |toolchain
--------|-----------------|--------------
``1.11``|``-Python-3.6.6``|``foss/2018b``

<a anchor="ragout"/>
### *Ragout*

Ragout (Reference-Assisted Genome Ordering UTility) is a tool for chromosome-level scaffolding using multiple references. Given initial assembly fragments (contigs/scaffolds) and one or multiple related references (complete or draft), it produces a chromosome-scale assembly (as a set of scaffolds).

*homepage*: https://github.com/fenderglass/Ragout

version|versionsuffix     |toolchain
-------|------------------|--------------
``2.0``|``-Python-2.7.12``|``foss/2016b``
``2.3``|                  |``foss/2020b``

<a anchor="ragtag"/>
### *RagTag*

RagTag is a collection of software tools for scaffolding and improving modern genome assemblies. Tasks include: homology-based misassembly correction, homology-based assembly scaffolding and patching, and scaffold merging. RagTag also provides command line utilities for working with common genome assembly file formats.

*homepage*: https://github.com/malonge/RagTag

version  |toolchain
---------|--------------
``2.0.1``|``foss/2020b``

<a anchor="rampart"/>
### *rampart*

Read Assignment, Mapping, and Phylogenetic Analysis in Real Time.

*homepage*: https://github.com/artic-network/rampart/

version     |versionsuffix    |toolchain
------------|-----------------|--------------
``1.2.0``   |                 |``foss/2020b``
``1.2.0rc3``|``-Python-3.6.6``|``foss/2018b``

<a anchor="randfold"/>
### *randfold*

Minimum free energy of folding randomization test software

*homepage*: http://bioinformatics.psb.ugent.be/software/details/Randfold

version  |toolchain
---------|-------------------------------
``2.0.1``|``foss/2018b``, ``intel/2016b``

<a anchor="randrproto"/>
### *randrproto*

Xrandr protocol and ancillary headers

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version  |toolchain
---------|-------------------------------
``1.5.0``|``foss/2016a``, ``intel/2016a``

<a anchor="rapidjson"/>
### *RapidJSON*

A fast JSON parser/generator for C++ with both SAX/DOM style API

*homepage*: https://rapidjson.org

version  |toolchain
---------|---------------------------------------------------------------------------------------------------------------------------------------
``1.1.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

<a anchor="rapidtide"/>
### *rapidtide*

Rapidtide is a suite of python programs used to perform time delay analysis on functional imaging  data to find time lagged correlations between the voxelwise time series and other time series.

*homepage*: https://rapidtide.readthedocs.io

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``1.8.0``|``-Python-3.7.2``|``intel/2019a``

<a anchor="rapsearch2"/>
### *RAPSearch2*

RAPSearch stands for Reduced Alphabet based Protein similarity Search 

*homepage*: https://omics.informatics.indiana.edu/mg/RAPSearch2

version |toolchain
--------|-------------
``2.24``|``GCC/9.3.0``

<a anchor="rascaf"/>
### *Rascaf*

Rascaf (RnA-seq SCAFfolder) uses continuity and order information from paired-end RNA-seq reads  to improve a draft assembly, particularly in the gene regions.

*homepage*: https://github.com/mourisl/Rascaf

version  |toolchain
---------|---------------
``1.0.2``|``intel/2017a``

<a anchor="raspa2"/>
### *RASPA2*

RASPA is a software package for simulating adsorption and diffusion of molecules in flexible nanoporous materials.

*homepage*: https://iraspa.org/raspa/

version   |versionsuffix     |toolchain
----------|------------------|---------------
``2.0.3`` |``-Python-2.7.12``|``intel/2016b``
``2.0.41``|                  |``foss/2020b``

<a anchor="rasterio"/>
### *rasterio*

Rasterio reads and writes geospatial raster data.

*homepage*: https://github.com/mapbox/rasterio

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``1.1.7`` |``-Python-3.8.2``|``foss/2020a``
``1.2.3`` |                 |``foss/2020b``
``1.2.10``|                 |``foss/2021b``

<a anchor="rasterstats"/>
### *rasterstats*

rasterstats is a Python module for summarizing geospatial raster datasets based on vector geometries.

*homepage*: https://github.com/perrygeo/python-rasterstats

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.15.0``|``-Python-3.8.2``|``foss/2020a``

<a anchor="ratatosk"/>
### *Ratatosk*

Phased hybrid error correction of long reads using colored de Bruijn graphs

*homepage*: https://github.com/DecodeGenetics/Ratatosk

version|toolchain
-------|--------------
``0.4``|``GCC/10.2.0``

<a anchor="raxml"/>
### *RAxML*

RAxML search algorithm for maximum likelihood based inference of phylogenetic trees.

*homepage*: https://github.com/stamatak/standard-RAxML

version   |versionsuffix     |toolchain
----------|------------------|---------------------------------------------------------------------------------------------------------------------
``8.2.4`` |``-hybrid-avx2``  |``foss/2016a``
``8.2.9`` |``-hybrid-avx2``  |``foss/2016a``
``8.2.10``|``-hybrid-avx2``  |``intel/2017a``
``8.2.11``|``-hybrid-avx``   |``foss/2017b``, ``intel/2017b``
``8.2.11``|``-hybrid-avx2``  |``foss/2017b``, ``intel/2017b``, ``intel/2018a``
``8.2.11``|``-hybrid-sse3``  |``foss/2017b``, ``intel/2017b``
``8.2.12``|``-hybrid-avx2``  |``gompi/2020a``, ``gompi/2020b``, ``gompi/2021a``, ``gompi/2021b``, ``iimpi/2019b``, ``intel/2018b``, ``intel/2019a``
``8.2.12``|``-pthreads-avx2``|``GCC/10.2.0``

<a anchor="raxml-ng"/>
### *RAxML-NG*

RAxML-NG is a phylogenetic tree inference tool which uses maximum-likelihood (ML)  optimality criterion. Its search heuristic is based on iteratively performing a series of Subtree  Pruning and Regrafting (SPR) moves, which allows to quickly navigate to the best-known ML tree.

*homepage*: https://github.com/amkozlov/raxml-ng

version  |toolchain
---------|------------------------------
``0.9.0``|``GCC/8.3.0``, ``gompi/2019b``
``1.0.1``|``gompi/2019b``
``1.0.2``|``gompi/2020b``
``1.0.3``|``GCC/10.2.0``
``1.1.0``|``GCC/11.2.0``

<a anchor="ray-assembler"/>
### *Ray-assembler*

Parallel genome assemblies for parallel DNA sequencing

*homepage*: http://denovoassembler.sourceforge.net/

version  |toolchain
---------|---------------
``2.3.1``|``iimpi/2019a``

<a anchor="ray-project"/>
### *Ray-project*

Ray is an open source project that makes it simple to scale any compute-intensive Python workload — from deep learning to production model serving. With a rich set of libraries and integrations built on a flexible distributed execution framework, Ray makes distributed computing easy and accessible to every engineer.

*homepage*: https://www.ray.io/

version   |versionsuffix    |toolchain
----------|-----------------|------------------------------
``0.8.4`` |``-Python-3.7.4``|``foss/2019b``
``1.0.1`` |``-Python-3.7.4``|``fosscuda/2019b``
``1.9.2`` |                 |``foss/2021b``
``1.13.0``|                 |``foss/2021a``, ``foss/2021b``

<a anchor="raysect"/>
### *Raysect*

Raysect is an OOP ray-tracing framework for Python

*homepage*: https://raysect.org

version  |versionsuffix    |toolchain
---------|-----------------|-------------------------------
``0.6.0``|``-Python-3.6.6``|``intel/2018b``
``0.7.1``|                 |``foss/2020b``, ``intel/2020b``

<a anchor="rbfopt"/>
### *RBFOpt*

RBFOpt is a Python library for black-box optimization (also known as derivative-free optimization).

*homepage*: https://github.com/coin-or/rbfopt

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``4.1.1``|                 |``intel/2019a``
``4.1.1``|``-Python-3.6.6``|``intel/2018b``

<a anchor="rclone"/>
### *rclone*

Rclone is a command line program to sync files and directories to and from  a variety of online storage services 

*homepage*: https://rclone.org

version   |versionsuffix|toolchain
----------|-------------|----------
``1.42``  |``-amd64``   |``system``
``1.54.1``|``-amd64``   |``system``
``1.56.0``|``-amd64``   |``system``
``1.57.0``|             |``system``

<a anchor="rcorrector"/>
### *Rcorrector*

Rcorrector(RNA-seq error CORRECTOR) is a kmer-based error correction method for RNA-seq data.

*homepage*: https://github.com/mourisl/Rcorrector

version  |toolchain
---------|---------------
``1.0.2``|``intel/2017a``

<a anchor="rcppgsl"/>
### *RcppGSL*

The 'RcppGSL' package provides an easy-to-use interface  between 'GSL' data structures and R using concepts from 'Rcpp' which is itself  a package that eases the interfaces between R and C++.

*homepage*: https://cran.r-project.org/web/packages/RcppGSL

version  |versionsuffix|toolchain
---------|-------------|--------------
``0.3.8``|``-R-4.0.4`` |``foss/2020b``

<a anchor="rcuda"/>
### *rCUDA*

The rCUDA Framework enables the concurrent usage of CUDA-compatible devices remotely.

*homepage*: http://www.rcuda.net/

version  |versionsuffix                 |toolchain
---------|------------------------------|----------
``4.0.1``|``_linux_64_Ubuntu10.04``     |``system``
``5.0``  |``_linux_64_scientificLinux6``|``system``

<a anchor="rdflib"/>
### *RDFlib*

RDFLib is a Python library for working with RDF, a simple yet powerful language  for representing information.

*homepage*: https://github.com/RDFLib/rdflib

version  |toolchain
---------|--------------------------------------
``4.2.2``|``GCCcore/8.3.0``, ``foss/2019a``
``5.0.0``|``GCCcore/10.2.0``
``6.2.0``|``GCCcore/10.3.0``, ``GCCcore/11.3.0``

<a anchor="rdkit"/>
### *RDKit*

RDKit is a collection of cheminformatics and machine-learning software written in C++ and Python.

*homepage*: https://www.rdkit.org

version      |versionsuffix    |toolchain
-------------|-----------------|-------------------------------
``2018.09.3``|``-Python-3.6.6``|``intel/2018b``
``2019.09.3``|``-Python-3.7.4``|``foss/2019b``
``2020.03.3``|``-Python-3.8.2``|``foss/2020a``, ``intel/2020a``
``2020.09.3``|``-Python-3.7.4``|``foss/2019b``
``2021.03.4``|                 |``foss/2021a``
``2022.03.5``|                 |``foss/2021b``

<a anchor="rdp-classifier"/>
### *RDP-Classifier*

The RDP Classifier is a naive Bayesian classifier that can rapidly and accurately provides taxonomic  assignments from domain to genus, with confidence estimates for each assignment.

*homepage*: http://sourceforge.net/projects/rdp-classifier

version |versionsuffix     |toolchain
--------|------------------|----------
``2.7`` |``-Java-1.7.0_60``|``system``
``2.12``|``-Java-1.8``     |``system``
``2.13``|``-Java-11``      |``system``
``2.13``|``-Java-17``      |``system``

<a anchor="re2"/>
### *RE2*

RE2 is a fast, safe, thread-friendly alternative to backtracking regular expression engines like those used in PCRE, Perl, and Python. It is a C++ library. 

*homepage*: https://github.com/google/re2

version       |toolchain
--------------|--------------------------------------
``2020-07-01``|``GCCcore/8.3.0``
``2021-06-01``|``GCCcore/10.2.0``
``2022-02-01``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``2022-06-01``|``GCCcore/11.3.0``

<a anchor="re2c"/>
### *re2c*

re2c is a free and open-source lexer generator for C and C++. Its main goal is generating fast lexers: at least as fast as their reasonably optimized hand-coded counterparts. Instead of using traditional table-driven approach, re2c encodes the generated finite state automata directly in the form of conditional jumps and comparisons.

*homepage*: https://re2c.org

version  |toolchain
---------|--------------------------------------
``1.1.1``|``GCCcore/8.2.0``
``1.2.1``|``GCCcore/8.3.0``
``1.3``  |``GCCcore/9.3.0``
``2.0.3``|``GCCcore/10.2.0``
``2.1.1``|``GCCcore/10.3.0``
``2.2``  |``GCCcore/11.2.0``, ``GCCcore/11.3.0``
``3.0``  |``GCCcore/12.2.0``

<a anchor="reads2snp"/>
### *Reads2snp*

reads2snp is a SNP and genotype caller: it predicts the genotype of distinct individuals   at distinct positions of a set of sequences based on read mapping / read counts. Its typical input is a bam file.   Its typical output is a vcf file. It is written in C++, based on the bio++ libraries, multi-threaded with openMP,   available under Linux and MacOS

*homepage*: http://kimura.univ-montp2.fr/PopPhyl/index.php?section=tools

version|toolchain
-------|----------
``2.0``|``system``

<a anchor="reapr"/>
### *Reapr*

A tool that evaluates the accuracy of a genome assembly using mapped paired end reads,  without the use of a reference genome for comparison.

*homepage*: https://www.sanger.ac.uk/science/tools/reapr

version   |toolchain
----------|--------------
``1.0.18``|``foss/2019a``

<a anchor="recon"/>
### *RECON*

Patched version of RECON to be used with RepeatModeler.

*homepage*: https://www.repeatmasker.org/RepeatModeler/

version |toolchain
--------|--------------
``1.08``|``GCC/10.2.0``

<a anchor="red"/>
### *Red*

Red (REpeat Detector)

*homepage*: http://toolsmith.ens.utulsa.edu/

version       |toolchain
--------------|----------------------------------------
``2015-05-22``|``iccifort/2019.1.144-GCC-8.2.0-2.31.1``

<a anchor="redis"/>
### *Redis*

Redis is an open source (BSD licensed), in-memory data structure store, used as a database, cache, and message broker. Redis provides data structures such as strings, hashes, lists, sets, sorted sets with range queries, bitmaps, hyperloglogs, geospatial indexes, and streams. Redis has built-in replication, Lua scripting, LRU eviction, transactions, and different levels of on-disk persistence, and provides high availability via Redis Sentinel and automatic partitioning with Redis Cluster.

*homepage*: https://redis.io

version  |toolchain
---------|------------------------------
``6.2.6``|``GCC/10.3.0``, ``GCC/11.2.0``

<a anchor="redis-py"/>
### *redis-py*

The Python interface to the Redis key-value store.

*homepage*: https://github.com/redis/redis-py

version  |toolchain
---------|--------------
``4.3.1``|``foss/2021a``
``4.3.3``|``foss/2021b``

<a anchor="redundans"/>
### *Redundans*

Redundans is a pipeline that assists an assembly of heterozygous/polymorphic genomes.

*homepage*: https://github.com/lpryszcz/redundans

version  |toolchain
---------|---------------
``0.13c``|``intel/2017b``

<a anchor="reframe"/>
### *ReFrame*

ReFrame is a framework for writing regression tests for HPC systems.

*homepage*: https://github.com/eth-cscs/reframe

version   |toolchain
----------|----------
``2.18``  |``system``
``2.19``  |``system``
``2.20``  |``system``
``2.21``  |``system``
``3.0``   |``system``
``3.2``   |``system``
``3.3``   |``system``
``3.4.1`` |``system``
``3.5.0`` |``system``
``3.5.1`` |``system``
``3.5.2`` |``system``
``3.6.2`` |``system``
``3.6.3`` |``system``
``3.7.3`` |``system``
``3.8.0`` |``system``
``3.9.0`` |``system``
``3.9.1`` |``system``
``3.10.1``|``system``
``3.11.0``|``system``
``3.11.1``|``system``
``3.11.2``|``system``
``3.12.0``|``system``

<a anchor="regionmask"/>
### *regionmask*

regionmask creates masks of geographical regions. It determines to which geographic region each grid point belongs.

*homepage*: https://regionmask.readthedocs.io

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.9.0``|                 |``foss/2021b``
``0.9.0``|``-Python-3.8.2``|``foss/2020a``

<a anchor="regtools"/>
### *RegTools*

RegTools is a set of tools that integrate DNA-seq and RNA-seq data to help interpret mutations in a regulatory and splicing context.

*homepage*: https://regtools.readthedocs.org

version  |toolchain
---------|------------------------------
``0.4.2``|``foss/2020b``
``0.5.2``|``foss/2020b``, ``foss/2021b``

<a anchor="relate"/>
### *Relate*

Software for estimating genome-wide genealogies for thousands of samples

*homepage*: https://myersgroup.github.io/relate/

version     |versionsuffix|toolchain
------------|-------------|--------------
``20211123``|``-R-4.0.3`` |``foss/2020b``

<a anchor="relion"/>
### *RELION*

RELION (for REgularised LIkelihood OptimisatioN, pronounce rely-on) is a stand-alone computer  program that employs an empirical Bayesian approach to refinement of (multiple) 3D reconstructions or 2D class  averages in electron cryo-microscopy (cryo-EM).

*homepage*: http://www2.mrc-lmb.cam.ac.uk/relion/index.php/Main_Page

version                |versionsuffix   |toolchain
-----------------------|----------------|------------------------------------------------------------------------------------------------------------
``1.4``                |                |``intel/2016b``
``1.4``                |``-single``     |``intel/2016b``
``2.0.1``              |                |``intel/2016b``
``2.1``                |                |``foss/2017b``, ``foss/2018a``, ``fosscuda/2017b``, ``fosscuda/2018a``, ``intel/2017b``, ``intelcuda/2017b``
``2.1``                |``-CUDA-9.1.85``|``foss/2018a``
``3.0.4``              |                |``foss/2017b``, ``intel/2017b``
``3.0_beta.2018.08.02``|                |``fosscuda/2018a``, ``intel/2018a``

<a anchor="rematch"/>
### *ReMatCh*

Reads mapping against target sequences, checking mapping and consensus sequences production

*homepage*: https://github.com/B-UMMI/ReMatCh

version|versionsuffix     |toolchain
-------|------------------|--------------
``3.2``|``-Python-2.7.12``|``foss/2016b``

<a anchor="remora"/>
### *REMORA*

REsource MOnitoring for Remote Applications

*homepage*: https://github.com/TACC/remora

version  |toolchain
---------|----------------------------------------------------------------
``1.8.2``|``foss/2017a``, ``foss/2018a``, ``intel/2017a``, ``intel/2018a``
``1.8.3``|``gompi/2019a``

<a anchor="renderproto"/>
### *renderproto*

Xrender protocol and ancillary headers

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version |toolchain
--------|------------------------------------------------------------------
``0.11``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2017b``

<a anchor="repasthpc"/>
### *RepastHPC*

The Repast Suite is a family of advanced, free, and open source agent-based modeling and simulation platforms that have collectively been under continuous development for over 15 years:  Repast for High Performance Computing 2.2.0, released on 30 September 2016, is a lean and expert-focused C++-based  modeling system that is designed for use on large computing clusters and supercomputers.

*homepage*: https://repast.github.io/

version  |toolchain
---------|--------------
``2.2.0``|``foss/2016a``

<a anchor="repeatmasker"/>
### *RepeatMasker*

RepeatMasker is a program that screens DNA sequences for interspersed repeats  and low complexity DNA sequences.

*homepage*: https://www.repeatmasker.org/

version     |versionsuffix         |toolchain
------------|----------------------|----------------------------------------------------------
``4.0.8``   |``-Perl-5.26.0-HMMER``|``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``
``4.0.8``   |``-Perl-5.28.0-HMMER``|``intel/2018b``
``4.0.9-p2``|``-HMMER``            |``gompi/2019b``
``4.1.2-p1``|                      |``foss/2020b``

<a anchor="repeatmodeler"/>
### *RepeatModeler*

RepeatModeler is a de novo transposable element (TE) family identification and modeling package. 

*homepage*: https://www.repeatmasker.org/

version   |toolchain
----------|--------------
``2.0.2a``|``foss/2020b``

<a anchor="repeatscout"/>
### *RepeatScout*

De Novo Repeat Finder, Price A.L., Jones N.C. and Pevzner P.A. Developed and  tested with our multiple sequence version of RepeatScout ( 1.0.6 )

*homepage*: https://www.repeatmasker.org/

version  |toolchain
---------|--------------
``1.0.6``|``GCC/10.2.0``

<a anchor="request"/>
### *request*

Request is designed to be the simplest way possible to make http calls.      It supports HTTPS and follows redirects by default.

*homepage*: https://github.com/request/request/

version   |versionsuffix      |toolchain
----------|-------------------|------------------
``2.88.1``|``-nodejs-12.19.0``|``fosscuda/2020b``

<a anchor="requests"/>
### *requests*

Python http for humans

*homepage*: https://pypi.python.org/pypi/requests

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``2.10.0``|``-Python-2.7.11``|``foss/2016a``
``2.10.0``|``-Python-3.5.1`` |``foss/2016a``
``2.11.1``|``-Python-2.7.12``|``intel/2016b``
``2.11.1``|``-Python-3.5.2`` |``intel/2016b``
``2.13.0``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``

<a anchor="rerconverge"/>
### *RERconverge*

RERconverge is a set of software written in R that  estimates the correlation between relative evolutionary rates of  genes and the evolution of a convergent binary or continuous trait  across a phylogeny. 

*homepage*: https://github.com/nclark-lab/RERconverge

version  |versionsuffix|toolchain
---------|-------------|-------------------------------
``0.1.0``|``-R-3.4.3`` |``foss/2017b``, ``intel/2017b``

<a anchor="resolos"/>
### *resolos*

Resolos is a toolkit written in Python for maintaining reproducible environments  for scientific computations. It's main goal is to enable researchers to easily replicate environments through space (running code on HPC environment) and time (environment preservation for long term archival). For installation and detailed usage, check out the documentation.

*homepage*: https://gitlab.unige.ch/resolos/resolos

version  |toolchain
---------|------------------
``0.3.5``|``GCCcore/11.3.0``

<a anchor="retworkx"/>
### *retworkx*

retworkx is a general purpose graph library for python3 written in Rust to take advantage of the performance and safety that Rust provides. It was built as a replacement for qiskit's previous (and current) networkx usage (hence the name) but is designed to provide a high performance general purpose graph library for any python application. The project was originally started to build a faster directed graph to use as the underlying data structure for the DAG at the center of qiskit-terra's transpiler, but it has since grown to cover all the graph usage in Qiskit and other applications.

*homepage*: https://github.com/Qiskit/retworkx

version  |toolchain
---------|--------------
``0.9.0``|``foss/2021a``

<a anchor="revbayes"/>
### *RevBayes*

RevBayes provides an interactive environment for statistical computation in  phylogenetics. It is primarily intended for modeling, simulation, and Bayesian inference in  evolutionary biology, particularly phylogenetics.

*homepage*: https://revbayes.github.io/

version  |toolchain
---------|------------------------------
``1.1.1``|``GCC/10.2.0``, ``GCC/11.2.0``

<a anchor="rgdal"/>
### *rgdal*

Provides bindings to the 'Geospatial' Data Abstraction Library ('GDAL') (>= 1.11.4 and <= 2.5.0) and  access to projection/transformation operations from the 'PROJ.4' library.

*homepage*: https://rgdal.r-forge.r-project.org/

version   |versionsuffix|toolchain
----------|-------------|--------------
``1.4-4`` |``-R-3.6.0`` |``foss/2019a``
``1.4-8`` |``-R-3.6.2`` |``foss/2019b``
``1.4-8`` |``-R-4.0.0`` |``foss/2020a``
``1.5-16``|``-R-4.0.0`` |``foss/2020a``
``1.5-23``|``-R-4.0.4`` |``foss/2020b``
``1.5-23``|``-R-4.1.0`` |``foss/2021a``

<a anchor="rgeos"/>
### *rgeos*

R interface to Geometry Engine - Open Source (GEOS) using the C API for topology operations  on geometries

*homepage*: https://cran.r-project.org/web/packages/rgeos/

version   |versionsuffix|toolchain
----------|-------------|---------------
``0.3-17``|``-R-3.2.3`` |``intel/2016a``
``0.5-1`` |``-R-3.6.0`` |``foss/2019a``
``0.5-2`` |``-R-3.6.2`` |``foss/2019b``
``0.5-5`` |``-R-4.0.0`` |``foss/2020a``
``0.5-5`` |``-R-4.1.0`` |``foss/2021a``

<a anchor="rgurobi"/>
### *Rgurobi*

Gurobi Optimizer 9.5 interface

*homepage*: https://www.gurobi.com

version  |versionsuffix|toolchain
---------|-------------|--------------
``9.1.2``|``-R-4.1.0`` |``foss/2021a``
``9.5.0``|``-R-4.1.0`` |``foss/2021a``

<a anchor="rhdf5"/>
### *rhdf5*

This R/Bioconductor package provides an interface between HDF5 and R.

*homepage*: https://bioconductor.org/packages/release/bioc/html/rhdf5.html

version   |versionsuffix|toolchain
----------|-------------|---------------
``2.16.0``|``-R-3.2.3`` |``intel/2016a``
``2.18.0``|``-R-3.3.1`` |``intel/2016b``

<a anchor="rheotool"/>
### *RheoTool*

RheoTool is an open-source toolbox based on OpenFOAM to simulate Generalized Newtonian Fluids (GNF) and viscoelastic fluids under pressure-driven and/or electrically-driven flows.

*homepage*: https://github.com/fppimenta/rheoTool

version|toolchain
-------|--------------
``5.0``|``foss/2019b``

<a anchor="rickflow"/>
### *rickflow*

Running and Analyzing OpenMM Jobs

*homepage*: https://gitlab.com/Olllom/rickflow

version           |versionsuffix    |toolchain
------------------|-----------------|---------------
``0.7.0``         |``-Python-3.7.4``|``intel/2019b``
``0.7.0-20200529``|``-Python-3.7.4``|``intel/2019b``

<a anchor="rinchi"/>
### *RInChI*

The reaction IUPAC International Chemical Identifier (RInChI TM) is a non-proprietary identifier for chemical reactions that can be used in printed and electronic data sources thus enabling easier linking of diverse data compilations.

*homepage*: https://www.inchi-trust.org/

version |versionsuffix|toolchain
--------|-------------|----------
``1.00``|``-x86_64``  |``system``

<a anchor="rioxarray"/>
### *rioxarray*

geospatial xarray extension powered by rasterio

*homepage*: https://github.com/corteva/rioxarray

version   |versionsuffix    |toolchain
----------|-----------------|---------------
``0.0.12``|``-Python-3.7.2``|``intel/2019a``
``0.0.24``|``-Python-3.7.4``|``foss/2019b``
``0.1.1`` |``-Python-3.8.2``|``foss/2020a``
``0.11.1``|                 |``foss/2021b``

<a anchor="rising"/>
### *rising*

Provides everything needed for high performance data loading and augmentation in PyTorch.

*homepage*: https://rising.rtfd.io/

version  |versionsuffix   |toolchain
---------|----------------|------------------------------
``0.2.2``|                |``foss/2021a``, ``foss/2022a``
``0.2.2``|``-CUDA-11.3.1``|``foss/2021a``
``0.2.2``|``-CUDA-11.7.0``|``foss/2022a``

<a anchor="rjags"/>
### *rjags*

The rjags package is an interface to the JAGS library.

*homepage*: https://cran.r-project.org/web/packages/rjags

version |versionsuffix|toolchain
--------|-------------|----------------------------------
``4-6`` |``-R-3.4.0`` |``intel/2017a``
``4-6`` |``-R-3.4.3`` |``intel/2017b``
``4-8`` |``-R-3.5.1`` |``foss/2018b``
``4-9`` |``-R-3.6.0`` |``foss/2019a``
``4-10``|             |``foss/2019b``
``4-10``|``-R-4.0.0`` |``foss/2020a``
``4-10``|``-R-4.0.3`` |``foss/2020b``, ``fosscuda/2020b``
``4-10``|``-R-4.0.4`` |``foss/2020b``, ``fosscuda/2020b``
``4-10``|``-R-4.0.5`` |``foss/2020b``, ``fosscuda/2020b``
``4-10``|``-R-4.1.0`` |``foss/2021a``
``4-12``|``-R-4.1.2`` |``foss/2021b``

<a anchor="rlcard"/>
### *RLCard*

RLCard is a toolkit for Reinforcement Learning (RL) in card games.

*homepage*: https://www.rlcard.org

version  |toolchain
---------|--------------
``1.0.9``|``foss/2022a``

<a anchor="rmath"/>
### *Rmath*

Rmath is the standalone version of the R math library.  Rmath can be used in your own C/C++ routines.

*homepage*: https://www.r-project.org/

version  |toolchain
---------|---------------
``3.3.1``|``intel/2016b``
``4.0.4``|``foss/2020b``

<a anchor="rmats-turbo"/>
### *rMATS-turbo*

rMATS turbo is the C/Cython version of rMATS (refer to http://rnaseq-mats.sourceforge.net).

*homepage*: https://github.com/Xinglab/rmats-turbo

version  |toolchain
---------|--------------
``4.1.1``|``foss/2020b``

<a anchor="rmblast"/>
### *RMBlast*

RMBlast is a RepeatMasker compatible version of the standard NCBI BLAST suite. The primary  difference between this distribution and the NCBI distribution is the addition of a new program 'rmblastn'  for use with RepeatMasker and RepeatModeler.

*homepage*: http://www.repeatmasker.org/RMBlast.html

version   |versionsuffix     |toolchain
----------|------------------|---------------
``2.2.28``|``-Python-2.7.11``|``foss/2016a``
``2.9.0`` |                  |``gompi/2019b``
``2.10.0``|                  |``gompi/2019b``
``2.11.0``|                  |``gompi/2020b``

<a anchor="rna-bloom"/>
### *RNA-Bloom*

RNA-Bloom is a fast and memory-efficient de novo transcript sequence assembler.

*homepage*: https://github.com/bcgsc/RNA-Bloom

version  |toolchain
---------|--------------
``1.2.3``|``GCC/8.3.0``
``1.4.3``|``GCC/11.2.0``

<a anchor="rna-seqc"/>
### *RNA-SeQC*

Fast, efficient RNA-Seq metrics for quality control and process optimization

*homepage*: https://github.com/getzlab/rnaseqc

version  |versionsuffix      |toolchain
---------|-------------------|------------------------------
``1.1.8``|``-Java-1.8``      |``foss/2018b``
``1.1.8``|``-Java-1.8.0_121``|``foss/2016b``
``2.4.2``|                   |``foss/2021a``, ``foss/2021b``

<a anchor="rnaclust"/>
### *RNAclust*

RNAclust is a perl script summarizing all the single steps required for clustering  of structured RNA motifs, i.e. identifying groups of RNA sequences sharing a secondary structure  motif. It requires as input a multiple FASTA file.

*homepage*: http://www.bioinf.uni-leipzig.de/~kristin/Software/RNAclust/

version|versionsuffix    |toolchain
-------|-----------------|--------------
``1.3``|``-Perl-5.24.0`` |``foss/2016b``
``1.3``|``-Python-3.6.6``|``foss/2018b``

<a anchor="rnacode"/>
### *RNAcode*

RNAcode - Analyze the protein coding potential in multiple sequence alignments

*homepage*: https://wash.github.io/rnacode/

version|toolchain
-------|--------------
``0.3``|``foss/2017a``

<a anchor="rnaindel"/>
### *RNAIndel*

RNAIndel calls coding indels and classifies them into somatic, germline,  and artifact from tumor RNA-Seq data.

*homepage*: https://github.com/stjude/RNAIndel

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.3.0``|``-Python-3.6.6``|``intel/2018b``
``1.0.0``|``-Python-3.6.6``|``intel/2018b``

<a anchor="rnammer"/>
### *RNAmmer*

This is an example description.

*homepage*: https://www.cbs.dtu.dk/services/RNAmmer

version|versionsuffix   |toolchain
-------|----------------|---------------
``1.2``|``-Perl-5.28.0``|``intel/2018b``

<a anchor="rnaquast"/>
### *rnaQUAST*

rnaQUAST is a tool for evaluating RNA-Seq assemblies using reference genome and gene database. In addition, rnaQUAST is also capable of estimating gene database coverage by raw reads and de novo quality assessment using third-party software.

*homepage*: https://cab.spbu.ru/software/rnaquast/

version  |versionsuffix     |toolchain
---------|------------------|--------------
``1.5.2``|``-Python-2.7.15``|``foss/2018b``
``2.2.0``|``-Python-3.8.2`` |``foss/2020a``
``2.2.2``|                  |``foss/2021b``

<a anchor="rnaz"/>
### *RNAz*

RNAz is a program for predicting structurally conserved and thermodynamically stable RNA secondary structures in multiple sequence alignments.

*homepage*: http://www.tbi.univie.ac.at/~wash/RNAz/

version|toolchain
-------|------------------------------
``2.1``|``foss/2016b``, ``foss/2018b``

<a anchor="rnbeads"/>
### *RnBeads*

RnBeads is an R package for comprehensive analysis of DNA methylation data obtained with any experimental protocol that provides single-CpG resolution.

*homepage*: https://rnbeads.org

version  |versionsuffix|toolchain
---------|-------------|--------------
``2.6.0``|``-R-4.0.0`` |``foss/2020a``

<a anchor="roary"/>
### *Roary*

Rapid large-scale prokaryote pan genome analysis

*homepage*: https://github.com/sanger-pathogens/Roary

version   |versionsuffix   |toolchain
----------|----------------|------------------------------
``3.12.0``|                |``system``
``3.12.0``|``-Perl-5.26.1``|``intel/2018a``
``3.13.0``|                |``foss/2020a``, ``foss/2021a``

<a anchor="rocm"/>
### *ROCm*

AMD ROCm is the first open-source software development platform for HPC/Hyperscale-class GPU computing. AMD ROCm brings the UNIX philosophy of choice, minimalism and modular software development to GPU computing.

*homepage*: https://rocmdocs.amd.com/en/latest/

version  |toolchain
---------|------------------
``4.5.0``|``GCCcore/11.2.0``

<a anchor="rocm-cmake"/>
### *rocm-cmake*

ROCM cmake modules provides cmake modules for common build tasks needed for the ROCM software stack

*homepage*: https://github.com/RadeonOpenCompute/rocm-cmake

version  |toolchain
---------|------------------
``4.5.0``|``GCCcore/11.2.0``

<a anchor="rocm-compilersupport"/>
### *ROCm-CompilerSupport*

The compiler support repository provides various Lightning Compiler related services

*homepage*: https://github.com/RadeonOpenCompute/ROCm-CompilerSupport

version  |toolchain
---------|------------------
``4.5.0``|``GCCcore/11.2.0``

<a anchor="rocm-smi"/>
### *rocm-smi*

The ROCm System Management Interface Library, or ROCm SMI library, is part of the Radeon Open Compute ROCm software stack. It is a C library for Linux that provides a user space interface for applications to monitor and control GPU applications.

*homepage*: https://github.com/RadeonOpenCompute/rocm_smi_lib

version  |toolchain
---------|------------------
``4.5.0``|``GCCcore/11.2.0``

<a anchor="rocminfo"/>
### *rocminfo*

ROCm Application for Reporting System Info

*homepage*: https://github.com/RadeonOpenCompute/rocminfo

version  |toolchain
---------|------------------
``4.5.0``|``GCCcore/11.2.0``

<a anchor="rocr-runtime"/>
### *ROCR-Runtime*

The user-mode API interfaces and libraries necessary for host applications to launch compute kernels to available HSA ROCm kernel agents

*homepage*: https://github.com/RadeonOpenCompute/ROCR-Runtime

version  |toolchain
---------|------------------
``4.5.0``|``GCCcore/11.2.0``

<a anchor="roct-thunk-interface"/>
### *ROCT-Thunk-Interface*

The user-mode API interfaces used to interact with the ROCk driver

*homepage*: https://github.com/RadeonOpenCompute/ROCT-Thunk-Interface

version  |toolchain
---------|------------------
``4.5.0``|``GCCcore/11.2.0``

<a anchor="roi_pac"/>
### *ROI_PAC*

Repeat Orbit Interferometry PACkage (ROI_PAC), software for processing synthetic aperture radar data  to produce differential interferograms

*homepage*: http://roipac.org/

version  |versionsuffix   |toolchain
---------|----------------|---------------
``3.0.1``|``-Perl-5.24.1``|``intel/2017a``

<a anchor="rome"/>
### *ROME*

The ROME (Refinement and Optimization via Machine Learning for cryo-EM)  Software package is one of the major research products at the Intel® PCCSB.

*homepage*: https://ipccsb.dfci.harvard.edu/rome

version  |toolchain
---------|-----------------
``1.1.2``|``intel/2019.02``

<a anchor="root"/>
### *ROOT*

The ROOT system provides a set of OO frameworks with all the functionality     needed to handle and analyze large amounts of data in a very efficient way.

*homepage*: http://root.cern.ch/drupal/

version     |versionsuffix     |toolchain
------------|------------------|-------------------------------
``6.10.02`` |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``6.10.04`` |``-Python-2.7.13``|``intel/2017a``
``6.10.08`` |``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``6.14.06`` |``-Python-2.7.15``|``foss/2018b``
``6.14.06`` |``-Python-3.6.6`` |``foss/2018b``
``6.20.04`` |``-Python-3.7.4`` |``foss/2019b``
``6.24.06`` |                  |``foss/2021b``
``v5.34.34``|``-Python-2.7.11``|``intel/2016a``
``v5.34.36``|``-Python-2.7.11``|``intel/2016a``
``v6.06.02``|``-Python-2.7.12``|``intel/2016b``
``v6.08.02``|``-Python-2.7.11``|``foss/2016a``

<a anchor="root_numpy"/>
### *root_numpy*

root_numpy is a Python extension module that provides an efficient interface between ROOT and  NumPy. root_numpy’s internals are compiled C++ and can therefore handle large amounts of data much faster than  equivalent pure Python implementations.

*homepage*: https://scikit-hep.org/root_numpy/

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``4.8.0``|``-Python-3.6.6``|``foss/2018b``

<a anchor="rootpy"/>
### *rootpy*

The rootpy project is a community-driven initiative aiming to provide a more pythonic interface with ROOT on top of the existing PyROOT bindings. Given Python’s reflective and dynamic nature, rootpy also aims to improve ROOT design flaws and supplement existing ROOT functionality. The scientific Python community also offers a multitude of powerful packages such as SciPy, NumPy, matplotlib, scikit-learn, and PyTables, but a suitable interface between them and ROOT has been lacking. rootpy provides the interfaces and conversion mechanisms required to liberate your data and to take advantage of these alternatives if needed.

*homepage*: http://www.rootpy.org/

version  |versionsuffix     |toolchain
---------|------------------|--------------
``0.8.0``|``-Python-2.7.11``|``foss/2016a``
``1.0.1``|``-Python-3.6.6`` |``foss/2018b``

<a anchor="rosetta"/>
### *Rosetta*

Rosetta is the premier software suite for modeling macromolecular structures. As a flexible, multi-purpose application, it includes tools for structure prediction, design, and remodeling of proteins and nucleic acids.

*homepage*: https://www.rosettacommons.org

version          |toolchain
-----------------|--------------
``3.7``          |``foss/2016b``
``2016.13.58602``|``foss/2016a``
``2016.46.59086``|``foss/2016b``

<a anchor="rpmrebuild"/>
### *rpmrebuild*

rpmrebuild is a tool to build an RPM file from a package that has already been     installed in a basic use

*homepage*: http://rpmrebuild.sourceforge.net/

version |toolchain
--------|----------
``2.11``|``system``

<a anchor="rpy2"/>
### *rpy2*

rpy2 is an interface to R running embedded in a Python process.

*homepage*: https://rpy2.bitbucket.io/

version  |versionsuffix     |toolchain
---------|------------------|---------------
``2.7.9``|``-Python-2.7.12``|``foss/2016b``
``2.8.2``|``-Python-2.7.13``|``intel/2017a``
``3.2.6``|``-Python-3.7.4`` |``foss/2019b``
``3.4.5``|                  |``foss/2021b``

<a anchor="rqgis3"/>
### *RQGIS3*

RQGIS3 establishes an interface between R and QGIS3, i.e.,  it allows the user to access QGIS3 functionalities from within R.

*homepage*: https://github.com/r-spatial/RQGIS3

version     |versionsuffix|toolchain
------------|-------------|--------------
``20190903``|``-R-3.6.0`` |``foss/2019a``

<a anchor="rsem"/>
### *RSEM*

RNA-Seq by Expectation-Maximization

*homepage*: https://deweylab.github.io/RSEM/

version   |toolchain
----------|------------------------------------------------
``1.2.26``|``GNU/4.9.3-2.25``
``1.2.30``|``foss/2016a``, ``intel/2016b``
``1.3.0`` |``foss/2016b``, ``intel/2017a``
``1.3.1`` |``foss/2017b``, ``intel/2017b``, ``intel/2018a``
``1.3.2`` |``foss/2018b``
``1.3.3`` |``foss/2019b``

<a anchor="rseqc"/>
### *RSeQC*

RSeQC provides a number of useful modules that can  comprehensively evaluate high throughput sequence data especially RNA-seq  data. Some basic modules quickly inspect sequence quality, nucleotide  composition bias, PCR bias and GC bias, while RNA-seq specific modules  evaluate sequencing saturation, mapped reads distribution, coverage  uniformity, strand specificity, transcript level RNA integrity etc.

*homepage*: http://rseqc.sourceforge.net/

version  |versionsuffix             |toolchain
---------|--------------------------|------------------------------
``2.6.4``|``-Python-2.7.12-R-3.3.1``|``foss/2016b``
``2.6.4``|``-Python-2.7.14``        |``intel/2018a``
``3.0.0``|``-Python-3.6.6``         |``foss/2018b``
``4.0.0``|                          |``foss/2021a``, ``foss/2021b``
``4.0.0``|``-Python-3.8.2``         |``foss/2020a``

<a anchor="rstan"/>
### *RStan*

RStan is the R interface to Stan. Stan is a state-of-the-art   platform for statistical modeling and high-performance statistical computation.

*homepage*: https://github.com/stan-dev/rstan

version   |versionsuffix|toolchain
----------|-------------|-------------------------------
``2.18.2``|``-R-3.4.3`` |``foss/2017b``, ``intel/2017b``
``2.18.2``|``-R-3.5.1`` |``foss/2018b``

<a anchor="rstanarm"/>
### *rstanarm*

Estimates previously compiled regression models using the 'rstan' package, which provides  the R interface to the Stan C++ library for Bayesian estimation.

*homepage*: https://r-forge.r-project.org/projects/rstanarm

version   |versionsuffix|toolchain
----------|-------------|--------------
``2.19.3``|``-R-3.6.2`` |``foss/2019b``

<a anchor="rstudio-server"/>
### *RStudio-Server*

This is the RStudio Server version. RStudio is a set of integrated tools designed to help you be more productive with R.  The server can be started with:   rserver --server-daemonize=0 --www-port=8787  If you need a database config one can be created with:   MYTMP=`mktemp -d` && echo -e "provider=sqlite\ndirectory=${MYTMP}/sqlite" > "${MYTMP}/db.conf" and then used with:   rserver ... --database-config-file="${MYTMP}/db.conf" 

*homepage*: https://www.rstudio.com/

version          |versionsuffix       |toolchain
-----------------|--------------------|------------------
``1.2.5033``     |``-Java-11``        |``fosscuda/2019b``
``1.2.5042``     |``-Java-11``        |``foss/2019b``
``1.3.959``      |``-Java-11-R-4.0.0``|``foss/2020a``
``1.3.1093``     |``-Java-11-R-4.0.0``|``foss/2020a``
``1.4.1717``     |``-Java-11-R-4.1.0``|``foss/2021a``
``2022.07.2+576``|``-Java-11-R-4.2.1``|``foss/2022a``

<a anchor="rtg-tools"/>
### *RTG-Tools*

RTG Tools contains utilities to easily manipulate and accurately compare multiple VCF files,  as well as utilities for processing other common NGS data formats. 

*homepage*: https://www.realtimegenomics.com/products/rtg-tools

version  |versionsuffix|toolchain
---------|-------------|----------
``3.9.1``|``-Java-1.8``|``system``

<a anchor="rtree"/>
### *Rtree*

Rtree is a ctypes Python wrapper of libspatialindex that provides a number of advanced spatial  indexing features for the spatially curious Python user.

*homepage*: https://toblerity.org/rtree/

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.8.3``|``-Python-2.7.14``|``intel/2018a``

<a anchor="ruamel.yaml"/>
### *ruamel.yaml*

ruamel.yaml is a YAML 1.2 loader/dumper package for Python.

*homepage*: https://sourceforge.net/projects/ruamel-yaml

version    |toolchain
-----------|----------------------------------------------------------
``0.17.21``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="ruby"/>
### *Ruby*

Ruby is a dynamic, open source programming language with  a focus on simplicity and productivity. It has an elegant syntax that is  natural to read and easy to write.

*homepage*: https://www.ruby-lang.org

version  |toolchain
---------|------------------------------------------------
``2.1.6``|``system``
``2.3.1``|``intel/2016b``, ``system``
``2.3.3``|``system``
``2.3.4``|``system``
``2.4.2``|``foss/2017b``
``2.5.0``|``foss/2018a``, ``intel/2017a``, ``intel/2017b``
``2.5.1``|``foss/2018a``, ``intel/2018a``
``2.6.1``|``GCCcore/7.3.0``
``2.6.3``|``GCCcore/8.2.0``
``2.7.1``|``GCCcore/8.3.0``
``2.7.2``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``3.0.1``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``3.0.3``|``GCCcore/11.3.0``
``3.0.5``|``GCCcore/11.3.0``

<a anchor="ruby-tk"/>
### *Ruby-Tk*

Ruby Tk interface module using tcltklib

*homepage*: https://github.com/ruby/tk

version  |versionsuffix  |toolchain
---------|---------------|-------------------------------
``0.2.0``|``-Ruby-2.5.1``|``foss/2018a``, ``intel/2018a``

<a anchor="ruffus"/>
### *ruffus*

Ruffus is a Computation Pipeline library for python. It is open-sourced, powerful and user-friendly,  and widely used in science and bioinformatics.

*homepage*: http://www.ruffus.org.uk/

version  |toolchain
---------|--------------
``2.8.4``|``foss/2021b``

<a anchor="rust"/>
### *Rust*

Rust is a systems programming language that runs blazingly fast, prevents segfaults,  and guarantees thread safety.

*homepage*: https://www.rust-lang.org

version   |toolchain
----------|--------------------------------------------------------------------------
``1.8.0`` |``foss/2016a``
``1.12.0``|``foss/2016b``
``1.12.1``|``foss/2016b``
``1.18.0``|``foss/2017a``
``1.21.0``|``foss/2017b``
``1.22.1``|``GCCcore/6.4.0``
``1.29.2``|``GCCcore/7.3.0``
``1.30.1``|``GCCcore/6.4.0``
``1.35.0``|``GCCcore/8.2.0``
``1.37.0``|``GCCcore/8.3.0``
``1.42.0``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.52.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``1.54.0``|``GCCcore/11.2.0``
``1.56.0``|``GCCcore/11.2.0``
``1.60.0``|``GCCcore/10.3.0``, ``GCCcore/11.3.0``
``1.65.0``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``

<a anchor="s"/>
### *S*


<a href="#s-lang">S-Lang</a> - <a href="#s4">S4</a> - <a href="#sage">SAGE</a> - <a href="#sailfish">Sailfish</a> - <a href="#salib">SALib</a> - <a href="#salmon">Salmon</a> - <a href="#salmon-tddft">SALMON-TDDFT</a> - <a href="#sambamba">Sambamba</a> - <a href="#samblaster">samblaster</a> - <a href="#samcef">Samcef</a> - <a href="#samclip">samclip</a> - <a href="#samplot">samplot</a> - <a href="#samtools">SAMtools</a> - <a href="#sansa">sansa</a> - <a href="#sap">SAP</a> - <a href="#sas">SAS</a> - <a href="#satsuma2">Satsuma2</a> - <a href="#savvy">savvy</a> - <a href="#saxon-he">Saxon-HE</a> - <a href="#sbcl">SBCL</a> - <a href="#sbt">sbt</a> - <a href="#scafacos">ScaFaCoS</a> - <a href="#scalapack">ScaLAPACK</a> - <a href="#scalasca">Scalasca</a> - <a href="#scalce">SCALCE</a> - <a href="#scanpy">scanpy</a> - <a href="#scenic">SCENIC</a> - <a href="#scgenefit">scGeneFit</a> - <a href="#scgid">SCGid</a> - <a href="#schrodinger">Schrodinger</a> - <a href="#sciclone">sciClone</a> - <a href="#scientificpython">ScientificPython</a> - <a href="#scikit-allel">scikit-allel</a> - <a href="#scikit-bio">scikit-bio</a> - <a href="#scikit-build">scikit-build</a> - <a href="#scikit-cuda">scikit-cuda</a> - <a href="#scikit-image">scikit-image</a> - <a href="#scikit-learn">scikit-learn</a> - <a href="#scikit-misc">scikit-misc</a> - <a href="#scikit-multilearn">scikit-multilearn</a> - <a href="#scikit-optimize">scikit-optimize</a> - <a href="#scikit-uplift">scikit-uplift</a> - <a href="#sciphi">SCIPhI</a> - <a href="#scipy">scipy</a> - <a href="#scipy-bundle">SciPy-bundle</a> - <a href="#scnorm">SCnorm</a> - <a href="#scoary">Scoary</a> - <a href="#scons">SCons</a> - <a href="#scoop">SCOOP</a> - <a href="#scopeloomr">SCopeLoomR</a> - <a href="#score-p">Score-P</a> - <a href="#scotch">SCOTCH</a> - <a href="#scp">scp</a> - <a href="#scpred">scPred</a> - <a href="#scrappie">Scrappie</a> - <a href="#scvelo">scVelo</a> - <a href="#scvi-tools">scvi-tools</a> - <a href="#scythe">Scythe</a> - <a href="#sdcc">SDCC</a> - <a href="#sdl">SDL</a> - <a href="#sdl2">SDL2</a> - <a href="#sdl2_image">SDL2_image</a> - <a href="#sdl_image">SDL_image</a> - <a href="#sdsl">SDSL</a> - <a href="#seaborn">Seaborn</a> - <a href="#searchgui">SearchGUI</a> - <a href="#secapr">SECAPR</a> - <a href="#seeder">Seeder</a> - <a href="#segemehl">segemehl</a> - <a href="#segmentation-models">segmentation-models</a> - <a href="#seissol">SeisSol</a> - <a href="#selestim">SelEstim</a> - <a href="#selfies">SELFIES</a> - <a href="#sentencepiece">SentencePiece</a> - <a href="#sep">sep</a> - <a href="#sepp">SEPP</a> - <a href="#seq-gen">Seq-Gen</a> - <a href="#seq2hla">seq2HLA</a> - <a href="#seqan">SeqAn</a> - <a href="#seqan3">SeqAn3</a> - <a href="#seqkit">SeqKit</a> - <a href="#seqlib">SeqLib</a> - <a href="#seqmagick">Seqmagick</a> - <a href="#seqprep">SeqPrep</a> - <a href="#seqtk">seqtk</a> - <a href="#serf">Serf</a> - <a href="#setuptools">setuptools</a> - <a href="#seurat">Seurat</a> - <a href="#seuratdata">SeuratData</a> - <a href="#seuratdisk">SeuratDisk</a> - <a href="#seuratwrappers">SeuratWrappers</a> - <a href="#sf">sf</a> - <a href="#sfftk">sfftk</a> - <a href="#shannon">Shannon</a> - <a href="#shap">SHAP</a> - <a href="#shapaar">shapAAR</a> - <a href="#shapeit">SHAPEIT</a> - <a href="#shapeit4">SHAPEIT4</a> - <a href="#shapely">Shapely</a> - <a href="#sharutils">sharutils</a> - <a href="#shasta">Shasta</a> - <a href="#shengbte">ShengBTE</a> - <a href="#shift">shift</a> - <a href="#shore">SHORE</a> - <a href="#short-pair">Short-Pair</a> - <a href="#shovill">shovill</a> - <a href="#shrinkwrap">shrinkwrap</a> - <a href="#shtns">SHTns</a> - <a href="#sibelia">Sibelia</a> - <a href="#sicer2">SICER2</a> - <a href="#sickle">sickle</a> - <a href="#siesta">Siesta</a> - <a href="#signalp">SignalP</a> - <a href="#silx">silx</a> - <a href="#simint">simint</a> - <a href="#simnibs">SimNIBS</a> - <a href="#simpeg">SimPEG</a> - <a href="#simple">SIMPLE</a> - <a href="#simpleelastix">SimpleElastix</a> - <a href="#simpleitk">SimpleITK</a> - <a href="#simpy">simpy</a> - <a href="#simvascular">SimVascular</a> - <a href="#singlem">SingleM</a> - <a href="#singular">Singular</a> - <a href="#sinvict">SiNVICT</a> - <a href="#sionlib">SIONlib</a> - <a href="#sip">SIP</a> - <a href="#sisso">SISSO</a> - <a href="#sisso++">SISSO++</a> - <a href="#skesa">SKESA</a> - <a href="#sketchmap">sketchmap</a> - <a href="#skewer">skewer</a> - <a href="#sklearn-pandas">sklearn-pandas</a> - <a href="#sklearn-som">sklearn-som</a> - <a href="#skorch">skorch</a> - <a href="#slamdunk">SlamDunk</a> - <a href="#slamdunk">slamdunk</a> - <a href="#slatec">SLATEC</a> - <a href="#slepc">SLEPc</a> - <a href="#slepc4py">slepc4py</a> - <a href="#sleuth">sleuth</a> - <a href="#slidingwindow">slidingwindow</a> - <a href="#slim">SLiM</a> - <a href="#slow5tools">slow5tools</a> - <a href="#smafa">smafa</a> - <a href="#smallgenomeutilities">smallgenomeutilities</a> - <a href="#smartdenovo">SMARTdenovo</a> - <a href="#smc++">SMC++</a> - <a href="#smithwaterman">smithwaterman</a> - <a href="#smoldyn">Smoldyn</a> - <a href="#smooth-topk">smooth-topk</a> - <a href="#smrt-link">SMRT-Link</a> - <a href="#smv">SMV</a> - <a href="#snakemake">snakemake</a> - <a href="#snap">SNAP</a> - <a href="#snap-hmm">SNAP-HMM</a> - <a href="#snape-pooled">SNAPE-pooled</a> - <a href="#snaphu">snaphu</a> - <a href="#snappy">snappy</a> - <a href="#snippy">snippy</a> - <a href="#snp-sites">snp-sites</a> - <a href="#snpeff">snpEff</a> - <a href="#snphylo">SNPhylo</a> - <a href="#snpomatic">SNPomatic</a> - <a href="#soapaligner">SOAPaligner</a> - <a href="#soapdenovo-trans">SOAPdenovo-Trans</a> - <a href="#soapdenovo2">SOAPdenovo2</a> - <a href="#soapfuse">SOAPfuse</a> - <a href="#socat">socat</a> - <a href="#soci">SOCI</a> - <a href="#solexaqa++">SolexaQA++</a> - <a href="#sonic">sonic</a> - <a href="#soqt">SoQt</a> - <a href="#sortmerna">SortMeRNA</a> - <a href="#sox">SoX</a> - <a href="#soxt">SoXt</a> - <a href="#spaceranger">SpaceRanger</a> - <a href="#spack">Spack</a> - <a href="#spacy">spaCy</a> - <a href="#spades">SPAdes</a> - <a href="#spaln">spaln</a> - <a href="#spark">Spark</a> - <a href="#sparsehash">sparsehash</a> - <a href="#spatialreg">spatialreg</a> - <a href="#spdlog">spdlog</a> - <a href="#spectra">SpectrA</a> - <a href="#spectral.methods">spectral.methods</a> - <a href="#speech_tools">speech_tools</a> - <a href="#spglib">spglib</a> - <a href="#spglib-python">spglib-python</a> - <a href="#sphinx">Sphinx</a> - <a href="#sphinx-rtd-theme">Sphinx-RTD-Theme</a> - <a href="#spiceypy">SpiceyPy</a> - <a href="#spieceasi">SpiecEasi</a> - <a href="#spladder">SplAdder</a> - <a href="#splash">SPLASH</a> - <a href="#splicemap">SpliceMap</a> - <a href="#split-seq">split-seq</a> - <a href="#splitref">splitRef</a> - <a href="#spm">SPM</a> - <a href="#spoa">spoa</a> - <a href="#spooles">SPOOLES</a> - <a href="#spotpy">SPOTPY</a> - <a href="#sprng">SPRNG</a> - <a href="#spyder">Spyder</a> - <a href="#sqlite">SQLite</a> - <a href="#squeezemeta">SqueezeMeta</a> - <a href="#squidpy">Squidpy</a> - <a href="#sra-toolkit">SRA-Toolkit</a> - <a href="#srprism">SRPRISM</a> - <a href="#srst2">SRST2</a> - <a href="#ssaha2">SSAHA2</a> - <a href="#ssn">SSN</a> - <a href="#sspace_basic">SSPACE_Basic</a> - <a href="#ssw">SSW</a> - <a href="#stacey">STACEY</a> - <a href="#stack">Stack</a> - <a href="#stacks">Stacks</a> - <a href="#stamp">STAMP</a> - <a href="#stamps">StaMPS</a> - <a href="#stampy">Stampy</a> - <a href="#star">STAR</a> - <a href="#star-ccm+">STAR-CCM+</a> - <a href="#star-fusion">STAR-Fusion</a> - <a href="#stars">stars</a> - <a href="#stata">Stata</a> - <a href="#statistics-r">Statistics-R</a> - <a href="#statsmodels">statsmodels</a> - <a href="#steak">STEAK</a> - <a href="#stir">STIR</a> - <a href="#stpipeline">stpipeline</a> - <a href="#stream">STREAM</a> - <a href="#strelka">strelka</a> - <a href="#stringtie">StringTie</a> - <a href="#stripy">stripy</a> - <a href="#strique">STRique</a> - <a href="#structure">Structure</a> - <a href="#strumpack">STRUMPACK</a> - <a href="#suave">suave</a> - <a href="#subread">Subread</a> - <a href="#subversion">Subversion</a> - <a href="#suds">suds</a> - <a href="#suitesparse">SuiteSparse</a> - <a href="#sumaclust">SUMACLUST</a> - <a href="#sumatra">SUMATRA</a> - <a href="#sumo">SUMO</a> - <a href="#sundials">SUNDIALS</a> - <a href="#sunpy">SunPy</a> - <a href="#superlu">SuperLU</a> - <a href="#superlu_dist">SuperLU_DIST</a> - <a href="#supermagic">supermagic</a> - <a href="#supernova">supernova</a> - <a href="#survivor">SURVIVOR</a> - <a href="#svdetect">SVDetect</a> - <a href="#svdquest">SVDquest</a> - <a href="#svg">SVG</a> - <a href="#svist4get">svist4get</a> - <a href="#swarm">swarm</a> - <a href="#swash">SWASH</a> - <a href="#swat+">SWAT+</a> - <a href="#swifter">swifter</a> - <a href="#swig">SWIG</a> - <a href="#swipe">SWIPE</a> - <a href="#swissknife">swissknife</a> - <a href="#symengine">SymEngine</a> - <a href="#symengine-python">SymEngine-python</a> - <a href="#symphony">SYMPHONY</a> - <a href="#sympy">sympy</a> - <a href="#syri">SyRI</a> - <a href="#szip">Szip</a>


<a anchor="s-lang"/>
### *S-Lang*

S-Lang is a multi-platform programmer's library designed to allow  a developer to create robust multi-platform software. It provides facilities  required by interactive applications such as display/screen management, keyboard  input, keymaps, and so on.

*homepage*: http://www.jedsoft.org/slang/

version  |toolchain
---------|-------------
``2.3.0``|``GCC/4.9.2``

<a anchor="s4"/>
### *S4*

S4 stands for Stanford Stratified Structure Solver, a frequency domain code  to solve the linear Maxwell’s equations in layered periodic structures. Internally, it uses  Rigorous Coupled Wave Analysis (RCWA, also called the Fourier Modal Method (FMM)) and the  S-matrix algorithm.

*homepage*: https://web.stanford.edu/group/fan/S4

version           |toolchain
------------------|--------------
``1.1.1-20180610``|``foss/2017b``

<a anchor="sage"/>
### *SAGE*

S.A.G.E. (Statistical Analysis for Genetic  Epidemiology) is free software package containing programs for  use in the genetic analysis of family, pedigree and individual data.

*homepage*: http://darwin.cwru.edu/sage/

version|toolchain
-------|----------
``6.3``|``system``
``6.4``|``system``

<a anchor="sailfish"/>
### *Sailfish*

Sailfish is a software tool that implements a novel, alignment-free algorithm for the estimation of  isoform abundances directly from a set of reference sequences and RNA-seq reads. 

*homepage*: http://www.cs.cmu.edu/~ckingsf/software/sailfish/

version   |versionsuffix     |toolchain
----------|------------------|---------------
``0.10.1``|                  |``gompi/2019b``
``0.10.1``|``-Python-2.7.12``|``foss/2016b``

<a anchor="salib"/>
### *SALib*

Sensitivity Analysis Library in Python (Numpy). Contains Sobol, Morris, Fractional Factorial and FAST methods.

*homepage*: https://SALib.github.io/SALib/

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.1.3``|``-Python-2.7.14``|``intel/2017b``

<a anchor="salmon"/>
### *Salmon*

Salmon is a wicked-fast program to produce a highly-accurate,  transcript-level quantification estimates from RNA-seq data.

*homepage*: https://github.com/COMBINE-lab/salmon

version   |versionsuffix     |toolchain
----------|------------------|--------------------------------
``0.8.2`` |                  |``system``
``0.8.2`` |``-Python-2.7.12``|``foss/2016b``
``0.11.2``|                  |``intel/2018a``
``0.12.0``|                  |``foss/2018b``
``0.14.1``|                  |``foss/2018b``
``0.14.2``|                  |``gompi/2019a``
``1.0.0`` |                  |``gompi/2019a``, ``gompi/2019b``
``1.1.0`` |                  |``gompi/2019b``
``1.2.0`` |                  |``gompi/2019b``
``1.3.0`` |                  |``gompi/2020a``
``1.4.0`` |                  |``GCC/11.2.0``, ``gompi/2020b``

<a anchor="salmon-tddft"/>
### *SALMON-TDDFT*

SALMON is an open-source computer program for ab-initio quantum-mechanical calculations of  electron dynamics at the nanoscale that takes place in various situations of light-matter interactions.  It is based on time-dependent density functional theory, solving time-dependent Kohn-Sham equation in  real time and real space with norm-conserving pseudopotentials.

*homepage*: https://salmon-tddft.jp

version  |toolchain
---------|-------------------------------
``1.2.1``|``foss/2018b``, ``intel/2018b``

<a anchor="sambamba"/>
### *Sambamba*

Sambamba is a high performance modern robust and fast tool  (and library), written in the D programming language, for working with SAM  and BAM files. Current functionality is an important subset of samtools  functionality, including view, index, sort, markdup, and depth.

*homepage*: https://lomereiter.github.io/sambamba/

version  |toolchain
---------|--------------
``0.6.6``|``system``
``0.7.1``|``system``
``0.8.0``|``GCC/10.2.0``
``0.8.2``|``GCC/10.3.0``

<a anchor="samblaster"/>
### *samblaster*

samblaster is a fast and flexible program for marking  duplicates in read-id grouped1 paired-end SAM files.

*homepage*: https://github.com/GregoryFaust/samblaster

version   |toolchain
----------|------------------------------
``0.1.24``|``foss/2018b``
``0.1.26``|``GCC/10.2.0``, ``GCC/10.3.0``

<a anchor="samcef"/>
### *Samcef*

FEM solver solution suite for basic linear structures to advanced, flexible nonlinear mechanisms and  thermal applications.

*homepage*: http://www.plm.automation.siemens.com/en_us/products/lms/samtech/samcef-solver-suite/

version    |toolchain
-----------|----------
``17.0-03``|``system``

<a anchor="samclip"/>
### *samclip*

Filter SAM file for soft and hard clipped alignments.

*homepage*: https://github.com/tseemann/samclip

version  |versionsuffix   |toolchain
---------|----------------|---------------------------------------------------------
``0.2``  |``-Perl-5.28.0``|``GCCcore/7.3.0``
``0.4.0``|                |``GCCcore/10.2.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``

<a anchor="samplot"/>
### *samplot*

Plot structural variant signals from many BAMs and CRAMs. 

*homepage*: https://github.com/ryanlayer/samplot

version  |toolchain
---------|--------------
``1.3.0``|``foss/2021b``

<a anchor="samtools"/>
### *SAMtools*

SAM Tools provide various utilities for manipulating alignments in the SAM format,   including sorting, merging, indexing and generating alignments in a per-position format.

*homepage*: http://www.htslib.org/

version   |versionsuffix    |toolchain
----------|-----------------|---------------------------------------------------------------------------------------------------------------------------------------
``0.1.17``|                 |``intel/2017a``
``0.1.19``|                 |``GCC/10.3.0``, ``foss/2016a``, ``foss/2016b``
``0.1.20``|                 |``GCC/8.3.0``, ``foss/2018b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``, ``intel/2019b``
``1.2``   |                 |``foss/2016b``
``1.3``   |                 |``foss/2016a``, ``intel/2016a``
``1.3.1`` |                 |``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``1.3.1`` |``-HTSlib-1.3.2``|``intel/2016b``
``1.4``   |                 |``foss/2016b``, ``intel/2016b``, ``intel/2017a``
``1.4.1`` |                 |``intel/2017a``
``1.5``   |                 |``foss/2016b``, ``intel/2017a``
``1.6``   |                 |``GCC/6.4.0-2.28``, ``foss/2017a``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``, ``intel/2017b``
``1.7``   |                 |``foss/2018a``, ``intel/2018a``
``1.8``   |                 |``intel/2018a``
``1.9``   |                 |``GCC/6.4.0-2.28``, ``GCC/7.3.0-2.30``, ``GCC/8.2.0-2.31.1``, ``foss/2018b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2018b``
``1.10``  |                 |``GCC/8.2.0-2.31.1``, ``GCC/8.3.0``, ``GCC/9.3.0``, ``iccifort/2019.5.281``
``1.11``  |                 |``GCC/10.2.0``, ``GCC/9.3.0``, ``iccifort/2020.4.304``
``1.12``  |                 |``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/9.3.0``
``1.13``  |                 |``GCC/10.3.0``, ``GCC/11.3.0``
``1.14``  |                 |``GCC/11.2.0``
``1.15``  |                 |``GCC/11.2.0``
``1.15.1``|                 |``GCC/11.2.0``, ``GCC/11.3.0``
``1.16.1``|                 |``GCC/11.2.0``, ``GCC/11.3.0``

<a anchor="sansa"/>
### *sansa*

Structural variant (SV) annotation, a companion to the 'dolly' tool.

*homepage*: https://github.com/dellytools/sansa/

version  |toolchain
---------|---------------
``0.0.7``|``gompi/2020b``

<a anchor="sap"/>
### *SAP*

SAP is a pairwise structure alignment via double dynamic programming

*homepage*: https://github.com/mathbio-nimr-mrc-ac-uk/SAP

version  |toolchain
---------|--------------
``1.1.3``|``GCC/11.3.0``

<a anchor="sas"/>
### *SAS*

SAS is a software suite for advanced analytics, multivariate analyses, business intelligence,  data management, and predictive analytics.

*homepage*: https://www.sas.com

version|toolchain
-------|----------
``9.4``|``system``

<a anchor="satsuma2"/>
### *Satsuma2*

Satsuma2 is an optimised version of Satsuma, a tool to reliably align large and complex DNA sequences providing maximum sensitivity (to find all there is to find), specificity (to only find real homology) and speed (to accommodate the billions of base pairs in vertebrate genomes).

*homepage*: https://github.com/bioinfologics/satsuma2

version     |toolchain
------------|--------------
``20220304``|``GCC/11.3.0``

<a anchor="savvy"/>
### *savvy*

Interface to various variant calling formats.

*homepage*: https://github.com/statgen/savvy

version  |toolchain
---------|--------------------
``1.3.0``|``GCC/8.2.0-2.31.1``

<a anchor="saxon-he"/>
### *Saxon-HE*

Open Source SAXON XSLT processor developed by Saxonica Limited.

*homepage*: http://saxon.sourceforge.net

version     |versionsuffix      |toolchain
------------|-------------------|----------
``9.7.0.4`` |``-Java-1.7.0_79`` |``system``
``9.7.0.21``|``-Java-1.8.0_162``|``system``
``9.9.1.7`` |``-Java-13``       |``system``

<a anchor="sbcl"/>
### *SBCL*

Steel Bank Common Lisp (SBCL) is a high performance Common Lisp compiler. It is open source / free software, with a permissive license. In addition to the compiler and runtime system for ANSI Common Lisp, it provides an interactive environment including a debugger, a statistical profiler, a code coverage tool, and many other extensions.

*homepage*: http://sbcl.sourceforge.net/

version  |toolchain
---------|------------------
``2.0.9``|``GCCcore/9.3.0``
``2.2.1``|``GCCcore/10.3.0``

<a anchor="sbt"/>
### *sbt*

sbt is a build tool for Scala, Java, and more.

*homepage*: http://www.scala-sbt.org/

version   |versionsuffix      |toolchain
----------|-------------------|----------
``1.0.2`` |``-Java-1.8.0_152``|``system``
``1.3.13``|``-Java-1.8``      |``system``

<a anchor="scafacos"/>
### *ScaFaCoS*

ScaFaCoS is a library of scalable fast coulomb solvers.

*homepage*: http://www.scafacos.de/

version  |toolchain
---------|-------------------------------------------------------------------------------
``1.0.1``|``foss/2020a``, ``foss/2020b``, ``foss/2021a``, ``foss/2021b``, ``intel/2020a``

<a anchor="scalapack"/>
### *ScaLAPACK*

The ScaLAPACK (or Scalable LAPACK) library includes a subset of LAPACK routines  redesigned for distributed memory MIMD parallel computers.

*homepage*: https://www.netlib.org/scalapack/

version  |versionsuffix                    |toolchain
---------|---------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------
``2.0.2``|                                 |``gompi/2019b``, ``gompic/2019b``
``2.0.2``|``-BLIS-0.3.2``                  |``gompi/2018b``
``2.0.2``|``-OpenBLAS-0.2.13-LAPACK-3.5.0``|``gmvapich2/1.7.20``
``2.0.2``|``-OpenBLAS-0.2.15-LAPACK-3.6.0``|``gmpich/2016a``, ``gmvapich2/2016a``, ``gompi/2016a``
``2.0.2``|``-OpenBLAS-0.2.18-LAPACK-3.6.0``|``gompi/2016.04``, ``gompi/2016.06``
``2.0.2``|``-OpenBLAS-0.2.18-LAPACK-3.6.1``|``gompi/2016.07``, ``gompi/2016b``
``2.0.2``|``-OpenBLAS-0.2.19-LAPACK-3.6.1``|``gompi/2016.09``
``2.0.2``|``-OpenBLAS-0.2.19-LAPACK-3.7.0``|``gompi/2017a``
``2.0.2``|``-OpenBLAS-0.2.20``             |``gimpi/2017b``, ``gimpi/2018a``, ``gimpic/2017b``, ``gmpich/2017.08``, ``gompi/2017b``, ``gompi/2018a``, ``gompic/2017b``, ``gompic/2018a``
``2.0.2``|``-OpenBLAS-0.3.1``              |``gompi/2018b``, ``gompic/2018b``
``2.0.2``|``-OpenBLAS-0.3.3``              |``gompi/2018.08``
``2.0.2``|``-OpenBLAS-0.3.5``              |``gompi/2019a``, ``gompic/2019a``
``2.1.0``|                                 |``gompi/2020a``, ``gompi/2020b``, ``gompic/2020a``, ``gompic/2020b``
``2.1.0``|``-bf``                          |``gompi/2020a``, ``gompi/2020b``, ``gompi/2021a``, ``iimpi/2020b``
``2.1.0``|``-bl``                          |``gompi/2020b``
``2.1.0``|``-fb``                          |``gompi/2021a``, ``gompi/2021b``
``2.2``  |``-amd``                         |``gompi/2020a``
``2.2.0``|``-fb``                          |``gompi/2022.05``, ``gompi/2022.10``, ``gompi/2022a``, ``gompi/2022b``

<a anchor="scalasca"/>
### *Scalasca*

Scalasca is a software tool that supports the performance optimization of  parallel programs by measuring and analyzing their runtime behavior. The  analysis identifies potential performance bottlenecks -- in particular  those concerning communication and synchronization -- and offers guidance  in exploring their causes. 

*homepage*: https://www.scalasca.org/

version|toolchain
-------|---------------------------------
``2.3``|``foss/2016a``
``2.5``|``gompi/2019a``, ``gompi/2020a``
``2.6``|``gompi/2021a``, ``gompic/2020b``

<a anchor="scalce"/>
### *SCALCE*

SCALCE [skeɪlz] is a FASTQ compression tool that uses locally consistet parsing  to obtain better compression rate. SCALCE has been specifically designed for Illumina reads but it can  handle other technologies (that generate base pair reads) if the read length is the same throughout the file.

*homepage*: http://scalce.sourceforge.net/Home

version|toolchain
-------|-------------
``2.7``|``GCC/4.8.2``

<a anchor="scanpy"/>
### *scanpy*

Scanpy is a scalable toolkit for analyzing single-cell gene expression data built  jointly with anndata. It includes preprocessing, visualization, clustering, trajectory inference  and differential expression testing. The Python-based implementation efficiently deals with  datasets of more than one million cells. 

*homepage*: https://scanpy.readthedocs.io/en/stable/

version  |toolchain
---------|------------------------------
``1.7.2``|``foss/2020b``
``1.8.1``|``foss/2020b``, ``foss/2021a``
``1.8.2``|``foss/2021b``
``1.9.1``|``foss/2021b``

<a anchor="scenic"/>
### *SCENIC*

SCENIC Suite is a set of tools to study and decipher gene regulation.

*homepage*: https://scenic.aertslab.org

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.2.4``|``-R-4.1.0`` |``foss/2021a``

<a anchor="scgenefit"/>
### *scGeneFit*

Python code for genetic marker selection using linear programming.

*homepage*: https://github.com/solevillar/scGeneFit-python

version  |toolchain
---------|--------------
``1.0.2``|``foss/2021a``

<a anchor="scgid"/>
### *SCGid*

A consensus approach to contig filtering and genome prediction from single-cell sequencing libraries

*homepage*: https://github.com/amsesk/SCGid

version  |toolchain
---------|--------------
``0.9b0``|``foss/2021b``

<a anchor="schrodinger"/>
### *Schrodinger*

Schrodinger aims to provide integrated software solutions and services   that truly meet its customers needs. We want to empower researchers around the world to   achieve their goals of improving human health and quality of life through advanced computational   techniques that transform the way chemists design compounds and materials. 

*homepage*: https://www.schrodinger.com/

version   |toolchain
----------|----------
``2020-4``|``system``
``2021-4``|``system``
``2022-1``|``system``
``2022-2``|``system``
``2022-3``|``system``

<a anchor="sciclone"/>
### *sciClone*

An R package for inferring the subclonal architecture of tumors 

*homepage*: https://github.com/genome/sciclone

version|versionsuffix|toolchain
-------|-------------|--------------
``1.1``|``-R-3.5.1`` |``foss/2018b``

<a anchor="scientificpython"/>
### *ScientificPython*

ScientificPython is a collection of Python modules for scientific computing.  It contains support for geometry, mathematical functions, statistics, physical units, IO, visualization,  and parallelization.

*homepage*: http://dirac.cnrs-orleans.fr/ScientificPython

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``2.9.4``|``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``

<a anchor="scikit-allel"/>
### *scikit-allel*

This package provides utilities for exploratory analysis of large scale genetic variation data.  It is based on numpy, scipy and other general-purpose Python scientific libraries.

*homepage*: https://scikit-allel.readthedocs.io/en/latest/

version  |versionsuffix     |toolchain
---------|------------------|--------------
``1.1.8``|``-Python-2.7.13``|``foss/2017a``
``1.2.1``|``-Python-3.8.2`` |``foss/2020a``
``1.3.2``|                  |``foss/2020b``
``1.3.3``|                  |``foss/2021a``

<a anchor="scikit-bio"/>
### *scikit-bio*

scikit-bio is an open-source, BSD-licensed Python 3 package providing data structures, algorithms and educational resources for bioinformatics.

*homepage*: http://scikit-bio.org

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------------------
``0.5.6``|``-Python-3.8.2``|``foss/2020a``
``0.5.7``|                 |``foss/2020b``, ``foss/2021b``, ``foss/2022a``

<a anchor="scikit-build"/>
### *scikit-build*

Scikit-Build, or skbuild, is an improved build system generator for CPython C/C++/Fortran/Cython extensions.

*homepage*: https://scikit-build.readthedocs.io/en/latest

version   |versionsuffix    |toolchain
----------|-----------------|----------------------------------------------------------------------------------------------------------------
``0.10.0``|``-Python-3.6.6``|``foss/2018b``, ``intel/2018b``
``0.10.0``|``-Python-3.8.2``|``foss/2020a``, ``fosscuda/2020a``
``0.11.1``|                 |``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``, ``intelcuda/2020b``
``0.15.0``|                 |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="scikit-cuda"/>
### *scikit-cuda*

SciKit-cuda, a.k.a. skcuda, provides Python interfaces to many of the functions in the CUDA  device/runtime, CUBLAS, CUFFT, and CUSOLVER libraries distributed as part of NVIDIA's CUDA Programming Toolkit.

*homepage*: https://github.com/lebedov/scikit-cuda

version  |versionsuffix    |toolchain
---------|-----------------|------------------
``0.5.3``|``-Python-3.7.4``|``fosscuda/2019b``

<a anchor="scikit-image"/>
### *scikit-image*

scikit-image is a collection of algorithms for image processing.

*homepage*: https://scikit-image.org/

version   |versionsuffix     |toolchain
----------|------------------|------------------------------------------------------------------------
``0.12.3``|``-Python-2.7.11``|``foss/2016a``
``0.12.3``|``-Python-2.7.12``|``intel/2016b``
``0.12.3``|``-Python-3.5.2`` |``foss/2016b``, ``intel/2016b``
``0.13.0``|``-Python-2.7.13``|``intel/2017a``
``0.13.0``|``-Python-3.6.1`` |``intel/2017a``
``0.13.0``|``-Python-3.6.3`` |``foss/2017b``
``0.13.1``|``-Python-2.7.14``|``intel/2017b``
``0.13.1``|``-Python-3.6.3`` |``foss/2017b``
``0.13.1``|``-Python-3.6.4`` |``foss/2018a``
``0.14.0``|``-Python-3.6.4`` |``intel/2018a``
``0.14.1``|``-Python-3.6.6`` |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``0.15.0``|``-Python-3.7.2`` |``foss/2019a``
``0.16.2``|``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``
``0.17.1``|``-Python-3.8.2`` |``foss/2020a``
``0.18.1``|                  |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``, ``intelcuda/2020b``
``0.18.3``|                  |``foss/2021a``
``0.19.1``|                  |``foss/2021b``
``0.19.3``|                  |``foss/2022a``

<a anchor="scikit-learn"/>
### *scikit-learn*

Scikit-learn integrates machine learning algorithms in the tightly-knit scientific Python world, building upon numpy, scipy, and matplotlib. As a machine-learning module, it provides versatile tools for data mining and analysis in any field of science and engineering. It strives to be simple and efficient, accessible to everybody, and reusable in various contexts.

*homepage*: https://scikit-learn.org/stable/index.html

version   |versionsuffix                    |toolchain
----------|---------------------------------|------------------------------------------------------------------------
``0.16.1``|``-Python-2.7.13``               |``foss/2017a``
``0.17.1``|``-Python-2.7.11``               |``foss/2016a``, ``intel/2016a``
``0.17.1``|``-Python-2.7.11-freetype-2.6.3``|``intel/2016a``
``0.17.1``|``-Python-2.7.12``               |``intel/2016b``
``0.17.1``|``-Python-3.5.1``                |``foss/2016a``, ``intel/2016a``
``0.18``  |``-Python-2.7.12``               |``intel/2016b``
``0.18``  |``-Python-3.5.2``                |``intel/2016b``
``0.18.1``|``-Python-2.7.12``               |``foss/2016b``, ``intel/2016b``
``0.18.1``|``-Python-2.7.13``               |``intel/2017a``
``0.18.1``|``-Python-3.5.2``                |``foss/2016b``, ``intel/2016b``
``0.18.1``|``-Python-3.6.1``                |``intel/2017a``
``0.18.2``|``-Python-3.6.1``                |``intel/2017a``
``0.19.0``|``-Python-3.6.1``                |``intel/2017a``
``0.19.1``|``-Python-2.7.13``               |``foss/2017a``
``0.19.1``|``-Python-2.7.14``               |``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``0.19.1``|``-Python-3.6.3``                |``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``0.19.1``|``-Python-3.6.4``                |``foss/2018a``, ``intel/2018a``
``0.20.0``|``-Python-2.7.15``               |``foss/2018b``, ``intel/2018b``
``0.20.0``|``-Python-3.6.6``                |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``0.20.2``|``-Python-2.7.15``               |``foss/2018b``
``0.20.2``|``-Python-3.6.6``                |``foss/2018b``
``0.20.3``|                                 |``foss/2019a``, ``fosscuda/2019a``, ``intel/2019a``
``0.20.4``|``-Python-2.7.16``               |``intel/2019b``
``0.20.4``|``-Python-2.7.18``               |``foss/2020b``
``0.21.3``|``-Python-3.7.4``                |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``
``0.23.1``|``-Python-3.8.2``                |``foss/2020a``, ``fosscuda/2020a``, ``intel/2020a``, ``intelcuda/2020a``
``0.23.2``|                                 |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``, ``intelcuda/2020b``
``0.24.2``|                                 |``foss/2021a``, ``intel/2021a``
``1.0.1`` |                                 |``foss/2021b``, ``intel/2021b``
``1.0.2`` |                                 |``foss/2021b``
``1.1.2`` |                                 |``foss/2022a``, ``intel/2022a``

<a anchor="scikit-misc"/>
### *scikit-misc*

Miscellaneous tools for data analysis and scientific computing

*homepage*: https://github.com/has2k1/scikit-misc

version  |toolchain
---------|--------------
``0.1.4``|``foss/2021a``

<a anchor="scikit-multilearn"/>
### *scikit-multilearn*

Scikit-multilearn is a BSD-licensed library for multi-label classification that is built on top of the well-known scikit-learn ecosystem.

*homepage*: http://scikit.ml

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.2.0``|``-Python-3.6.6``|``foss/2018b``

<a anchor="scikit-optimize"/>
### *scikit-optimize*

Scikit-Optimize, or skopt, is a simple and efficient library to minimize (very) expensive  and noisy black-box functions.

*homepage*: https://scikit-optimize.github.io

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.5.2``|``-Python-3.6.6``|``intel/2018b``
``0.7.4``|``-Python-3.7.4``|``foss/2019b``
``0.8.1``|``-Python-3.8.2``|``foss/2020a``
``0.9.0``|                 |``foss/2021a``

<a anchor="scikit-uplift"/>
### *scikit-uplift*

scikit-uplift is a Python module for classic approaches for uplift modeling built on top of scikit-learn. Uplift prediction aims to estimate the causal impact of a treatment at the individual level. 

*homepage*: https://github.com/maks-sh/scikit-uplift

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.2.0``|``-Python-3.8.2``|``foss/2020a``

<a anchor="sciphi"/>
### *SCIPhI*

Single-cell mutation identification via phylogenetic inference (SCIPhI) is a new approach  to mutation detection in individual tumor cells by leveraging the evolutionary relationship among cells.

*homepage*: https://github.com/cbg-ethz/SCIPhI

version  |toolchain
---------|---------------
``0.1.3``|``intel/2018b``

<a anchor="scipy"/>
### *scipy*

SciPy is a collection of mathematical algorithms and convenience  functions built on the Numpy extension for Python.

*homepage*: https://www.scipy.org

version   |versionsuffix     |toolchain
----------|------------------|----------------------------------
``0.16.0``|``-Python-2.7.12``|``intel/2016b``
``0.17.0``|``-Python-2.7.11``|``intel/2016a``
``0.19.0``|``-Python-3.5.2`` |``intel/2016b``
``1.4.1`` |``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``

<a anchor="scipy-bundle"/>
### *SciPy-bundle*

Bundle of Python packages for scientific software

*homepage*: https://python.org/

version    |versionsuffix     |toolchain
-----------|------------------|------------------------------------------------------------------------
``2019.03``|                  |``foss/2019a``, ``fosscuda/2019a``, ``intel/2019a``, ``intelcuda/2019a``
``2019.10``|``-Python-2.7.16``|``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``
``2019.10``|``-Python-3.7.2`` |``intel/2019a``
``2019.10``|``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``, ``intelcuda/2019b``
``2020.03``|``-Python-2.7.18``|``foss/2020a``, ``fosscuda/2020a``, ``intel/2020a``
``2020.03``|``-Python-3.8.2`` |``foss/2020a``, ``fosscuda/2020a``, ``intel/2020a``, ``intelcuda/2020a``
``2020.11``|                  |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``, ``intelcuda/2020b``
``2020.11``|``-Python-2.7.18``|``foss/2020b``
``2021.05``|                  |``foss/2021a``, ``gomkl/2021a``, ``intel/2021a``
``2021.10``|                  |``foss/2021b``, ``intel/2021b``
``2021.10``|``-Python-2.7.18``|``foss/2021b``
``2022.05``|                  |``foss/2022.05``, ``foss/2022a``, ``intel/2022.05``, ``intel/2022a``

<a anchor="scnorm"/>
### *SCnorm*

This package implements SCnorm — a method to normalize single-cell RNA-seq data.

*homepage*: https://bioconductor.org/packages/SCnorm

version   |versionsuffix|toolchain
----------|-------------|---------------
``0.99.7``|``-R-3.4.0`` |``intel/2017a``

<a anchor="scoary"/>
### *Scoary*

Microbial pan-GWAS using the output from Roary

*homepage*: https://github.com/AdmiralenOla/Scoary

version   |versionsuffix     |toolchain
----------|------------------|---------------
``1.6.16``|                  |``foss/2021a``
``1.6.16``|``-Python-2.7.14``|``intel/2018a``

<a anchor="scons"/>
### *SCons*

SCons is a software construction tool.

*homepage*: https://www.scons.org/

version        |versionsuffix          |toolchain
---------------|-----------------------|---------------------------------------------------------------------------------
``2.4.1``      |``-Python-2.7.11``     |``foss/2016a``
``2.5.0``      |``-Python-2.7.12``     |``foss/2016b``, ``intel/2016b``
``2.5.1``      |                       |``intel/2017a``
``2.5.1``      |``-Python-2.7.12``     |``foss/2016b``, ``intel/2016b``
``2.5.1``      |``-Python-2.7.13``     |``intel/2017a``
``3.0.1``      |``-Python-2.7.14``     |``foss/2017b``, ``foss/2018a``, ``intel/2017b``, ``intel/2018a``, ``iomkl/2018a``
``3.0.1``      |``-Python-2.7.15``     |``foss/2018b``, ``fosscuda/2018b``
``3.0.1``      |``-Python-2.7.15-bare``|``GCCcore/7.3.0``
``3.0.1``      |``-Python-3.6.3``      |``foss/2017b``, ``intel/2017b``
``3.0.1``      |``-Python-3.6.4``      |``foss/2018a``, ``intel/2018a``
``3.0.4``      |``-Python-2.7.15``     |``foss/2018b``
``3.0.5``      |                       |``GCCcore/8.2.0``
``3.0.5``      |``-Python-2.7.15``     |``GCCcore/8.2.0``
``3.0.5``      |``-Python-3.7.2``      |``GCCcore/8.2.0``
``3.1.1``      |                       |``GCCcore/8.3.0``
``3.1.2``      |                       |``GCCcore/9.3.0``
``4.0.1``      |                       |``GCCcore/10.2.0``
``4.1.0.post1``|                       |``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``4.2.0``      |                       |``GCCcore/11.2.0``
``4.4.0``      |                       |``GCCcore/11.3.0``

<a anchor="scoop"/>
### *SCOOP*

SCOOP (Scalable COncurrent Operations in Python) is a distributed task module  allowing concurrent parallel programming on various environments, from heterogeneous grids to supercomputers.

*homepage*: http://code.google.com/p/scoop/

version    |versionsuffix     |toolchain
-----------|------------------|-----------------
``0.6.2``  |``-Python-2.7.12``|``intel/2016b``
``0.7.1.1``|                  |``GCCcore/8.2.0``
``0.7.1.1``|``-Python-2.7.11``|``intel/2016a``
``0.7.1.1``|``-Python-2.7.14``|``intel/2017b``
``0.7.1.1``|``-Python-3.5.1`` |``intel/2016a``

<a anchor="scopeloomr"/>
### *SCopeLoomR*

An R package (compatible with SCope) to create generic .loom files and extend them with other data e.g.: SCENIC regulons, Seurat clusters and markers, ...

*homepage*: https://github.com/aertslab/SCopeLoomR

version   |versionsuffix|toolchain
----------|-------------|--------------
``0.13.0``|``-R-4.1.2`` |``foss/2021b``

<a anchor="score-p"/>
### *Score-P*

The Score-P measurement infrastructure is a highly scalable and easy-to-use  tool suite for profiling, event tracing, and online analysis of HPC  applications. 

*homepage*: https://www.score-p.org

version  |versionsuffix   |toolchain
---------|----------------|--------------------------------------------------
``2.0.1``|                |``foss/2016a``
``6.0``  |                |``gompi/2019a``, ``gompi/2020a``, ``gompic/2019b``
``7.0``  |                |``gompi/2021a``, ``gompic/2020b``
``7.1``  |``-CUDA-11.3.1``|``gompi/2021a``

<a anchor="scotch"/>
### *SCOTCH*

Software package and libraries for sequential and parallel graph partitioning, static mapping, and sparse matrix block ordering, and sequential mesh and hypergraph partitioning.

*homepage*: https://www.labri.fr/perso/pelegrin/scotch/

version            |versionsuffix|toolchain
-------------------|-------------|-----------------------------------------------------------------------------------------------------------------------------------------------------
``5.1.12b_esmumps``|             |``foss/2017b``
``6.0.4``          |             |``foss/2016a``, ``foss/2016b``, ``foss/2017b``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``
``6.0.4``          |``-64bitint``|``foss/2017b``
``6.0.6``          |             |``foss/2018a``, ``foss/2018b``, ``gompi/2019a``, ``iimpi/2019a``, ``intel/2018a``
``6.0.9``          |             |``gompi/2019b``, ``gompi/2020a``, ``iimpi/2019b``, ``iimpi/2020a``, ``iimpic/2019b``
``6.1.0``          |             |``gompi/2020b``, ``gompi/2021a``, ``iimpi/2020b``, ``iimpi/2021a``
``6.1.2``          |             |``gompi/2021b``, ``iimpi/2021b``
``7.0.1``          |             |``gompi/2022a``, ``iimpi/2022a``

<a anchor="scp"/>
### *scp*

The scp.py module uses a paramiko transport to send and recieve files via the scp1 protocol.

*homepage*: https://github.com/jbardin/scp.py

version   |versionsuffix     |toolchain
----------|------------------|---------------
``0.10.2``|``-Python-2.7.12``|``intel/2016b``
``0.13.1``|``-Python-2.7.15``|``intel/2018b``
``0.13.2``|``-Python-2.7.15``|``intel/2018b``

<a anchor="scpred"/>
### *scPred*

scPred package for cell type prediction from scRNA-seq data 

*homepage*: https://github.com/powellgenomicslab/scPred

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.9.2``|``-R-4.1.2`` |``foss/2021b``

<a anchor="scrappie"/>
### *Scrappie*

Scrappie is a technology demonstrator for the Oxford Nanopore Research Algorithms group.

*homepage*: https://github.com/nanoporetech/scrappie

version  |toolchain
---------|---------------
``1.3.2``|``intel/2018a``

<a anchor="scvelo"/>
### *scVelo*

scVelo is a scalable toolkit for estimating and analyzing RNA velocities in single cells using  dynamical modeling.

*homepage*: https://scvelo.org

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.1.24``|``-Python-3.7.4``|``foss/2019b``
``0.2.3`` |                 |``foss/2021a``

<a anchor="scvi-tools"/>
### *scvi-tools*

scvi-tools (single-cell variational inference tools) is a package for probabilistic modeling and analysis of single-cell omics data, built on top of PyTorch and AnnData.

*homepage*: https://github.com/scverse/scvi-tools

version   |versionsuffix   |toolchain
----------|----------------|--------------
``0.16.4``|                |``foss/2021a``
``0.16.4``|``-CUDA-11.3.1``|``foss/2021a``

<a anchor="scythe"/>
### *Scythe*

Scythe uses a Naive Bayesian approach to classify contaminant substrings in  sequence reads. It considers quality information, which can make it robust in  picking out 3'-end adapters, which often include poor quality bases. 

*homepage*: https://github.com/ucdavis-bioinformatics/scythe

version  |toolchain
---------|-----------------
``0.994``|``GCCcore/9.3.0``

<a anchor="sdcc"/>
### *SDCC*

SDCC is a retargettable, optimizing ANSI - C compiler suite that targets  the Intel MCS51 based microprocessors (8031, 8032, 8051, 8052, etc.), Maxim (formerly Dallas)  DS80C390 variants, Freescale (formerly Motorola) HC08 based (hc08, s08) and Zilog Z80 based  MCUs (z80, z180, gbz80, Rabbit 2000/3000, Rabbit 3000A). Work is in progress on supporting the  Microchip PIC16 and PIC18 targets. It can be retargeted for other microprocessors.

*homepage*: http://sdcc.sourceforge.net/

version  |toolchain
---------|----------
``3.3.0``|``system``

<a anchor="sdl"/>
### *SDL*

SDL: Simple DirectMedia Layer, a cross-platform multimedia library

*homepage*: https://www.libsdl.org/

version   |toolchain
----------|-----------------
``1.2.15``|``GCCcore/6.4.0``

<a anchor="sdl2"/>
### *SDL2*

SDL: Simple DirectMedia Layer, a cross-platform multimedia library

*homepage*: http://www.libsdl.org/

version   |toolchain
----------|----------------------------------------------------------------------
``2.0.4`` |``intel/2016b``
``2.0.8`` |``GCCcore/6.4.0``, ``foss/2017b``, ``intel/2017b``, ``intel/2018a``
``2.0.9`` |``GCCcore/8.2.0``, ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``2.0.10``|``GCCcore/8.3.0``
``2.0.14``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``2.0.20``|``GCCcore/11.2.0``
``2.0.22``|``GCCcore/11.3.0``

<a anchor="sdl2_image"/>
### *SDL2_image*

SDL_image is an image file loading library.

*homepage*: https://www.libsdl.org/projects/SDL_image/

version  |toolchain
---------|-----------------
``2.0.3``|``GCCcore/6.4.0``

<a anchor="sdl_image"/>
### *SDL_image*

SDL_image is an image file loading library.

*homepage*: https://www.libsdl.org/projects/SDL_image/release-1.2.html

version   |toolchain
----------|-----------------
``1.2.12``|``GCCcore/6.4.0``

<a anchor="sdsl"/>
### *SDSL*

The Succinct Data Structure Library (SDSL) is a powerful and flexible C++11 library implementing  succinct data structures.

*homepage*: https://github.com/simongog/sdsl-lite

version           |toolchain
------------------|-----------------
``2.1.1-20191211``|``GCCcore/8.3.0``

<a anchor="seaborn"/>
### *Seaborn*

Seaborn is a Python visualization library based on matplotlib.  It provides a high-level interface for drawing attractive statistical graphics. 

*homepage*: https://seaborn.pydata.org/

version   |versionsuffix     |toolchain
----------|------------------|-----------------------------------------------------------------------------------------
``0.7.1`` |``-Python-2.7.12``|``intel/2016b``
``0.8.1`` |``-Python-2.7.13``|``foss/2017a``
``0.8.1`` |``-Python-2.7.14``|``intel/2018a``
``0.9.0`` |``-Python-2.7.14``|``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intel/2018a``, ``intelcuda/2017b``
``0.9.0`` |``-Python-2.7.15``|``foss/2018b``, ``foss/2019a``, ``intel/2018b``, ``intel/2019a``
``0.9.0`` |``-Python-3.6.3`` |``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``0.9.0`` |``-Python-3.6.4`` |``intel/2018a``
``0.9.0`` |``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``0.9.0`` |``-Python-3.7.2`` |``foss/2019a``, ``intel/2019a``
``0.9.1`` |``-Python-2.7.16``|``foss/2019b``
``0.9.1`` |``-Python-2.7.18``|``foss/2020b``
``0.10.0``|``-Python-3.7.4`` |``foss/2019b``, ``intel/2019b``
``0.10.1``|                  |``intel/2020b``
``0.10.1``|``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``0.11.1``|                  |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``
``0.11.2``|                  |``foss/2021a``, ``foss/2021b``
``0.12.1``|                  |``foss/2022a``

<a anchor="searchgui"/>
### *SearchGUI*

SearchGUI is a user-friendly open-source graphical user interface for configuring and  running proteomics identification search engines and de novo sequencing algorithms,  currently supporting X! Tandem, MS-GF+, MS Amanda, MyriMatch, Comet, Tide, Andromeda, OMSSA,  Novor and DirecTag.

*homepage*: https://github.com/compomics/searchgui

version  |versionsuffix      |toolchain
---------|-------------------|----------
``3.3.3``|``-Java-1.8.0_152``|``system``

<a anchor="secapr"/>
### *SECAPR*

SECAPR is a bioinformatics pipeline for the rapid and user-friendly processing of targeted enriched Illumina sequences, from raw reads to alignments

*homepage*: https://github.com/AntonelliLab/seqcap_processor

version   |versionsuffix     |toolchain
----------|------------------|--------------
``1.1.15``|``-Python-2.7.16``|``foss/2019b``

<a anchor="seeder"/>
### *Seeder*

Seeder is a framework for DNA motif discovery.

*homepage*: https://metacpan.org/pod/Seeder

version |versionsuffix   |toolchain
--------|----------------|-----------------
``0.01``|``-Perl-5.28.1``|``GCCcore/8.2.0``

<a anchor="segemehl"/>
### *segemehl*

segemehl is a software to map short sequencer reads to reference genomes.   Unlike other methods, segemehl is able to detect not only mismatches but also insertions   and deletions. Furthermore, segemehl is not limited to a specific read length and is able   to mapprimer- or polyadenylation contaminated reads correctly. segemehl implements a matching   strategy based on enhanced suffix arrays (ESA). Segemehl now supports the SAM format, reads   gziped queries to save both disk and memory space and allows bisulfite sequencing mapping   and split read mapping. 

*homepage*: http://www.bioinf.uni-leipzig.de/Software/segemehl/

version  |toolchain
---------|-----------------------------------------------------------------------------
``0.2.0``|``foss/2016b``, ``intel/2017b``, ``intel/2018a``
``0.3.4``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/8.3.0``, ``foss/2018b``

<a anchor="segmentation-models"/>
### *segmentation-models*

Python library with Neural Networks for Image Segmentation based on Keras and TensorFlow.

*homepage*: https://github.com/qubvel/segmentation_models

version  |versionsuffix    |toolchain
---------|-----------------|----------------------------------
``1.0.1``|``-Python-3.7.4``|``foss/2019b``, ``fosscuda/2019b``

<a anchor="seissol"/>
### *SeisSol*

SeisSol is a software package for simulating wave propagation and dynamic rupture based on  the arbitrary high-order accurate derivative discontinuous Galerkin method (ADER-DG).

*homepage*: http://www.seissol.org

version   |versionsuffix     |toolchain
----------|------------------|--------------
``201703``|``-Python-2.7.15``|``foss/2018b``

<a anchor="selestim"/>
### *SelEstim*

SelEstim is aimed at distinguishing neutral from selected polymorphisms and estimate   the intensity of selection at the latter. The SelEstim model accounts explicitly for positive selection,   and it is assumed that all marker loci in the dataset are responding to selection, to some extent

*homepage*: http://www1.montpellier.inra.fr/CBGP/software/selestim/index.html

version  |versionsuffix    |toolchain
---------|-----------------|----------
``1.1.4``|``-Linux-64bits``|``system``

<a anchor="selfies"/>
### *SELFIES*

Robust representation of semantically constrained graphs, in particular for molecules in chemistry

*homepage*: https://github.com/aspuru-guzik-group/selfies

version  |toolchain
---------|--------------
``2.1.1``|``GCC/11.2.0``

<a anchor="sentencepiece"/>
### *SentencePiece*

Unsupervised text tokenizer for Neural Network-based text generation.

*homepage*: https://github.com/google/sentencepiece

version   |versionsuffix    |toolchain
----------|-----------------|------------------------------
``0.1.85``|``-Python-3.7.4``|``GCC/8.3.0``
``0.1.94``|``-Python-3.8.2``|``GCC/9.3.0``
``0.1.96``|                 |``GCC/10.2.0``, ``GCC/10.3.0``
``0.1.97``|                 |``GCC/11.3.0``

<a anchor="sep"/>
### *sep*

Python and C library for Source Extraction and Photometry.  (this easyconfig provides python library only)

*homepage*: https://sep.readthedocs.io

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``1.0.3``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``

<a anchor="sepp"/>
### *SEPP*

SATe-enabled Phylogenetic Placement - addresses the problem of phylogenetic placement of short reads into reference alignments and trees.

*homepage*: https://github.com/smirarab/sepp

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``4.3.10``|``-Python-3.7.4``|``foss/2019b``
``4.3.10``|``-Python-3.8.2``|``foss/2020a``
``4.4.0`` |                 |``foss/2020b``
``4.5.0`` |                 |``foss/2021a``
``4.5.1`` |                 |``foss/2021b``

<a anchor="seq-gen"/>
### *Seq-Gen*

Seq-Gen is a program that will simulate the evolution of nucleotide or amino acid sequences  along a phylogeny, using common models of the substitution process.

*homepage*: http://tree.bio.ed.ac.uk/software/Seq-Gen

version  |toolchain
---------|---------------
``1.3.4``|``intel/2017b``

<a anchor="seq2hla"/>
### *seq2HLA*

In-silico method written in Python and R to determine HLA genotypes of a sample.  seq2HLA takes standard RNA-Seq sequence reads in fastq format as input,  uses a bowtie index comprising all HLA alleles and outputs the most likely  HLA class I and class II genotypes (in 4 digit resolution),  a p-value for each call, and the expression of each class. 

*homepage*: https://github.com/TRON-Bioinformatics/seq2HLA

version|versionsuffix     |toolchain
-------|------------------|---------------
``2.3``|``-Python-2.7.14``|``intel/2018a``
``2.3``|``-Python-2.7.15``|``foss/2018b``

<a anchor="seqan"/>
### *SeqAn*

SeqAn is an open source C++ library of efficient algorithms and data structures  for the analysis of sequences with the focus on biological data

*homepage*: https://www.seqan.de/

version  |versionsuffix|toolchain
---------|-------------|-----------------------------------------------------------------------------------------------------------------------------------
``1.4.2``|``-library`` |``system``
``2.3.2``|             |``foss/2016b``
``2.4.0``|             |``GCC/8.2.0-2.31.1``, ``GCCcore/10.2.0``, ``GCCcore/11.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``foss/2018b``, ``intel/2018b``

<a anchor="seqan3"/>
### *SeqAn3*

SeqAn is an open source C++ library   of efficient algorithms and data structures for the   analysis of sequences with the focus on biological   data. Our library applies a unique generic design   that guarantees high performance, generality,   extensibility, and integration with other libraries.   SeqAn is easy to use and simplifies the development   of new software tools with a minimal loss of   performance. 

*homepage*: https://github.com/seqan/seqan3

version  |toolchain
---------|----------
``3.0.0``|``system``

<a anchor="seqkit"/>
### *SeqKit*

SeqKit - a cross-platform and ultrafast toolkit for FASTA/Q file manipulation

*homepage*: https://bioinf.shenwei.me/seqkit/

version   |toolchain
----------|----------
``0.8.1`` |``system``
``0.13.2``|``system``
``2.1.0`` |``system``
``2.2.0`` |``system``

<a anchor="seqlib"/>
### *SeqLib*

C++ interface to HTSlib, BWA-MEM and Fermi.

*homepage*: https://github.com/walaj/SeqLib

version  |toolchain
---------|-------------------------------------------------------------
``1.2.0``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/9.3.0``

<a anchor="seqmagick"/>
### *Seqmagick*

We often have to convert between sequence formats and do  little tasks on them, and it's not worth writing scripts for that. Seqmagick  is a kickass little utility built in the spirit of imagemagick to expose the  file format conversion in Biopython in a convenient way. Instead of having a  big mess of scripts, there is one that takes arguments.

*homepage*: https://fhcrc.github.io/seqmagick/

version  |versionsuffix     |toolchain
---------|------------------|--------------
``0.6.1``|``-Python-2.7.11``|``foss/2016a``
``0.6.2``|``-Python-2.7.15``|``foss/2018b``

<a anchor="seqprep"/>
### *SeqPrep*

Tool for stripping adaptors and/or merging paired reads with overlap into single reads.

*homepage*: https://github.com/jstjohn/SeqPrep

version  |toolchain
---------|-----------------
``1.3.2``|``GCCcore/7.3.0``

<a anchor="seqtk"/>
### *seqtk*

Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format.   It seamlessly parses both FASTA and FASTQ files which can also be optionally compressed by gzip.

*homepage*: https://github.com/lh3/seqtk/

version|toolchain
-------|----------------------------------------------------------------------------------------------------------------------------------
``1.2``|``foss/2016b``, ``intel/2017a``
``1.3``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/8.2.0-2.31.1``, ``GCC/8.3.0``, ``GCC/9.3.0``, ``foss/2018a``, ``foss/2018b``

<a anchor="serf"/>
### *Serf*

The serf library is a high performance C-based HTTP client library  built upon the Apache Portable Runtime (APR) library

*homepage*: http://serf.apache.org/

version  |toolchain
---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.3.9``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/9.3.0``, ``foss/2017b``, ``intel/2017b``, ``iomkl/2018a``

<a anchor="setuptools"/>
### *setuptools*

Easily download, build, install, upgrade, and uninstall Python packages

*homepage*: https://pypi.org/project/setuptools

version   |versionsuffix|toolchain
----------|-------------|----------
``1.4.2`` |             |``system``
``41.0.1``|``-py3``     |``system``

<a anchor="seurat"/>
### *Seurat*

Seurat is an R package designed for QC, analysis, and exploration of single cell RNA-seq data.

*homepage*: https://satijalab.org/seurat

version     |versionsuffix|toolchain
------------|-------------|---------------
``1.4.0.16``|``-R-3.4.0`` |``intel/2017a``
``2.3.4``   |``-R-3.4.4`` |``intel/2018a``
``2.3.4``   |``-R-3.5.1`` |``foss/2018b``
``3.1.2``   |``-R-3.6.0`` |``foss/2019a``
``3.1.5``   |``-R-4.0.0`` |``foss/2020a``
``4.0.1``   |``-R-4.0.3`` |``foss/2020b``
``4.0.3``   |``-R-4.0.3`` |``foss/2020b``
``4.2.0``   |``-R-4.2.1`` |``foss/2022a``
``4.3.0``   |``-R-4.1.2`` |``foss/2021b``

<a anchor="seuratdata"/>
### *SeuratData*

SeuratData is a mechanism for distributing datasets in the form of Seurat objects using R's internal package and data management systems.

*homepage*: https://github.com/satijalab/seurat-data

version     |versionsuffix|toolchain
------------|-------------|--------------
``20210514``|``-R-4.0.3`` |``foss/2020b``

<a anchor="seuratdisk"/>
### *SeuratDisk*

Interfaces for HDF5-based Single Cell File Formats

*homepage*: https://github.com/mojaveazure/seurat-disk

version       |versionsuffix|toolchain
--------------|-------------|--------------
``0.0.0.9020``|``-R-4.2.1`` |``foss/2022a``

<a anchor="seuratwrappers"/>
### *SeuratWrappers*

SeuratWrappers is a collection of community-provided methods and extensions for Seurat

*homepage*: https://github.com/satijalab/seurat-wrappers

version     |versionsuffix|toolchain
------------|-------------|--------------
``20210528``|``-R-4.0.3`` |``foss/2020b``

<a anchor="sf"/>
### *sf*

Support for simple features, a standardized way to encode spatial vector data. Binds to GDAL for reading and writing data, to GEOS for geometrical operations, and to PROJ for projection conversions and datum transformations.

*homepage*: https://cran.r-project.org/package=sf

version  |versionsuffix            |toolchain
---------|-------------------------|--------------
``0.9-5``|``-R-4.0.0-Python-3.8.2``|``foss/2020a``

<a anchor="sfftk"/>
### *sfftk*

sfftk is a set of utilities that facilitate creation, conversion and             modification of Electron Microscopy Data Bank - Segmentation File Format         (EMDB-SFF) files. EMDB-SFF is an open, community-driven file format to handle    annotated segmentations and subtomogram averages that facilitates segmentation   file interchange. It is written in Python and provides both a command-line suite of commands and a Python API.

*homepage*: https://emdb-empiar.github.io/EMDB-SFF/

version  |toolchain
---------|--------------
``0.7.4``|``foss/2021a``

<a anchor="shannon"/>
### *Shannon*

Shannon is a program for assembling transcripts from RNA-Seq data

*homepage*: https://sreeramkannan.github.io/Shannon/

version     |versionsuffix     |toolchain
------------|------------------|---------------
``20170511``|``-Python-2.7.13``|``intel/2017a``

<a anchor="shap"/>
### *SHAP*

SHAP (SHapley Additive exPlanations) is a game theoretic approach to explain the output of any  machine learning model. It connects optimal credit allocation with local explanations using the classic Shapley  values from game theory and their related extensions.

*homepage*: https://github.com/slundberg/shap

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.35.0``|``-Python-3.7.4``|``foss/2019b``

<a anchor="shapaar"/>
### *shapAAR*

An R package for the extraction, analysis and classification of (not only) archaeological objects  from scanned images.

*homepage*: https://github.com/ISAAKiel/shapAAR

version           |versionsuffix|toolchain
------------------|-------------|--------------
``0.1.0-20180425``|``-R-3.6.0`` |``foss/2019a``

<a anchor="shapeit"/>
### *SHAPEIT*

SHAPEIT is a fast and accurate method for estimation of  haplotypes (aka phasing) from genotype or sequencing data.

*homepage*: https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html

version   |versionsuffix  |toolchain
----------|---------------|----------
``2.r837``|``.GLIBCv2.12``|``system``
``2.r904``|``.glibcv2.17``|``system``

<a anchor="shapeit4"/>
### *SHAPEIT4*

SHAPEIT4 is a fast and accurate method for estimation of haplotypes (aka phasing) for SNP array and high coverage sequencing data. 

*homepage*: https://odelaneau.github.io/shapeit4/

version  |toolchain
---------|----------------------------------------------
``4.1.3``|``foss/2019b``
``4.2.0``|``foss/2019b``, ``foss/2020a``, ``foss/2020b``
``4.2.2``|``foss/2020b``, ``foss/2021a``

<a anchor="shapely"/>
### *Shapely*

Shapely is a BSD-licensed Python package for manipulation and analysis of planar geometric objects. It is based on the widely deployed GEOS (the engine of PostGIS) and JTS (from which GEOS is ported) libraries.

*homepage*: https://github.com/Toblerity/Shapely

version        |versionsuffix    |toolchain
---------------|-----------------|-------------------------------------------------------
``1.7.0``      |``-Python-3.7.2``|``foss/2019a``
``1.7.0``      |``-Python-3.7.4``|``GCC/8.3.0``, ``iccifort/2019.5.281``
``1.7.1``      |``-Python-3.8.2``|``GCC/9.3.0``
``1.8.1.post1``|                 |``GCC/11.2.0``
``1.8.2``      |                 |``foss/2021b``, ``foss/2022a``
``1.8a1``      |                 |``GCC/10.2.0``, ``GCC/10.3.0``, ``iccifort/2020.4.304``

<a anchor="sharutils"/>
### *sharutils*

GNU shar makes so-called shell archives out of many files, preparing them for transmission by  electronic mail services, while unshar helps unpacking shell archives after reception.

*homepage*: https://www.gnu.org/software/sharutils/

version |toolchain
--------|-----------------
``4.15``|``GCCcore/6.3.0``

<a anchor="shasta"/>
### *Shasta*

The goal of the Shasta long read assembler is to rapidly produce accurate assembled sequence using DNA reads generated by Oxford Nanopore flow cells as input. Computational methods used by the Shasta assembler include: Using a run-length representation of the read sequence. This makes the assembly process more resilient to errors in homopolymer repeat counts, which are the most common type of errors in Oxford Nanopore reads. Using in some phases of the computation a representation of the read sequence based on markers, a fixed subset of short k-mers (k ≈ 10). 

*homepage*: https://github.com/chanzuckerberg/shasta

version  |toolchain
---------|--------------
``0.8.0``|``foss/2020b``

<a anchor="shengbte"/>
### *ShengBTE*

A solver for the Boltzmann transport equation for phonons.

*homepage*: https://bitbucket.org/sousaw/shengbte

version  |toolchain
---------|--------------
``1.1.1``|``foss/2021a``

<a anchor="shift"/>
### *shift*

Shift is a framework for Self-Healing Independent File Transfer that provides high performance and resilience for local and remote transfers through a variety of techniques. 

*homepage*: http://people.nas.nasa.gov/~kolano/projects/shift.html

version|toolchain
-------|---------------
``4.0``|``intel/2016a``

<a anchor="shore"/>
### *SHORE*

SHORE, for Short Read, is a mapping and analysis pipeline for short read data produced on the Illumina platform. 

*homepage*: http://1001genomes.org/software/shore.html

version  |toolchain
---------|--------------
``0.9.3``|``foss/2016a``

<a anchor="short-pair"/>
### *Short-Pair*

Sensitive Short Read Homology Search for Paired-End Reads

*homepage*: https://sourceforge.net/projects/short-pair

version     |versionsuffix     |toolchain
------------|------------------|--------------
``20170125``|                  |``foss/2021b``
``20170125``|``-Python-2.7.15``|``foss/2018b``

<a anchor="shovill"/>
### *shovill*

Faster SPAdes assembly of Illumina reads

*homepage*: https://github.com/tseemann/shovill

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.9.0``|``-Python-2.7.14``|``foss/2018a``
``1.0.4``|``-Python-2.7.15``|``foss/2018b``
``1.1.0``|                  |``gompi/2021b``

<a anchor="shrinkwrap"/>
### *shrinkwrap*

A std::streambuf wrapper for compression formats.

*homepage*: https://github.com/jonathonl/shrinkwrap

version       |toolchain
--------------|------------------
``1.0.0-beta``|``GCCcore/8.2.0``
``1.1.0``     |``GCCcore/10.2.0``

<a anchor="shtns"/>
### *SHTns*

Spherical Harmonic Transform library aimed at high performance numerical simulations in spherical geometries.

*homepage*: https://bitbucket.org/nschaeff/shtns

version  |toolchain
---------|--------------
``2.7``  |``foss/2021b``
``3.5.1``|``foss/2021b``

<a anchor="sibelia"/>
### *Sibelia*

Sibelia: A comparative genomics tool: It assists biologists in analysing   the genomic variations that correlate with pathogens, or the genomic changes that help   microorganisms adapt in different environments. Sibelia will also be helpful for the   evolutionary and genome rearrangement studies for multiple strains of microorganisms.

*homepage*: http://bioinf.spbau.ru/en/sibelia

version  |toolchain
---------|------------------------------
``3.0.6``|``foss/2016b``
``3.0.7``|``foss/2018b``, ``foss/2020b``

<a anchor="sicer2"/>
### *SICER2*

Redesigned and improved ChIP-seq broad peak calling tool SICER

*homepage*: https://zanglab.github.io/SICER2/

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.0.3``|``-Python-3.8.2``|``foss/2020a``

<a anchor="sickle"/>
### *sickle*

Windowed Adaptive Trimming for fastq files using quality 

*homepage*: https://github.com/najoshi/sickle

version |toolchain
--------|-------------------------------
``1.33``|``foss/2017a``, ``intel/2018a``

<a anchor="siesta"/>
### *Siesta*

SIESTA is both a method and its computer program implementation, to perform efficient electronic  structure calculations and ab initio molecular dynamics simulations of molecules and solids.

*homepage*: http://departments.icmab.es/leem/siesta

version        |versionsuffix|toolchain
---------------|-------------|------------------------------------------------------------------------------------------------
``4.0``        |             |``foss/2017b``, ``intel/2017a``
``4.0.1``      |             |``intel/2017a``
``4.1-b2``     |             |``intel/2017a``
``4.1-b3``     |             |``intel/2017a``
``4.1-b4``     |             |``foss/2018b``, ``intel/2018b``
``4.1-MaX-1.0``|             |``intel/2019b``
``4.1-MaX-1.0``|``-PEXSI``   |``intel/2019b``
``4.1.5``      |             |``foss/2020a``, ``foss/2021a``, ``foss/2021b``, ``foss/2022a``, ``intel/2020a``, ``intel/2022a``

<a anchor="signalp"/>
### *SignalP*

SignalP predicts the presence  and  location of signal peptide  cleavage sites in  amino acid  sequences  from  different organisms

*homepage*: https://services.healthtech.dtu.dk/software.php

version |versionsuffix        |toolchain
--------|---------------------|--------------
``5.0b``|``-Linux``           |``system``
``6.0g``|``-fast``            |``foss/2022a``
``6.0g``|``-fast-CUDA-11.7.0``|``foss/2022a``

<a anchor="silx"/>
### *silx*

The silx project provides a collection of Python packages to support the  development of data assessment, reduction and analysis applications at synchrotron radiation facilities.

*homepage*: http://www.silx.org/

version   |versionsuffix    |toolchain
----------|-----------------|----------------------------------
``0.13.2``|``-Python-3.7.4``|``foss/2019b``, ``fosscuda/2019b``
``0.14.0``|                 |``foss/2020b``, ``fosscuda/2020b``

<a anchor="simint"/>
### *simint*

Simint is a vectorized implementation of the Obara-Saika (OS)  method of calculating electron repulsion integrals. Speedup is gained by  vectorizing the primitive loop of the OS algorithm, with additional vectorization  and optimizations left to the compiler.

*homepage*: https://www.bennyp.org/research/simint/

version|versionsuffix           |toolchain
-------|------------------------|----------------------------------
``0.7``|``-lmax-5-vec-avx-psi4``|``GCC/11.2.0``, ``GCCcore/11.2.0``

<a anchor="simnibs"/>
### *SimNIBS*

SimNIBS is a free and open source software package for the Simulation of Non-invasive Brain Stimulation.

*homepage*: https://simnibs.github.io/simnibs

version  |toolchain
---------|--------------
``3.2.4``|``foss/2020b``

<a anchor="simpeg"/>
### *SimPEG*

Simulation and Parameter Estimation in Geophysics: a python package for simulation and gradient based parameter estimation in the context of geophysical applications.

*homepage*: https://simpeg.xyz

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``0.3.1`` |``-Python-2.7.12``|``intel/2016b``
``0.14.1``|``-Python-3.8.2`` |``intel/2020a``
``0.18.1``|                  |``foss/2021b``, ``intel/2021b``

<a anchor="simple"/>
### *SIMPLE*

Single-particle IMage Processing Linux Engine SIMPLE is an open-source software package for analysis of cryogenic transmission electron microscopy (cryo-EM)  movies of single-particles (Single-Particle Analysis, SPA).

*homepage*: https://simplecryoem.com/

version  |toolchain
---------|--------------
``2.5``  |``foss/2018a``
``3.0.0``|``foss/2020b``

<a anchor="simpleelastix"/>
### *SimpleElastix*

Multi-lingual medical image registration library.

*homepage*: https://simpleelastix.github.io

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``0.10.0``|``-Python-3.6.3``|``foss/2017b``
``0.10.0``|``-Python-3.6.4``|``foss/2018a``
``1.1.0`` |``-Python-3.7.4``|``foss/2019b``

<a anchor="simpleitk"/>
### *SimpleITK*

SimpleITK is a simplified programming interface to the algorithms and data structures of  the Insight Toolkit (ITK).

*homepage*: https://www.simpleitk.org

version    |versionsuffix     |toolchain
-----------|------------------|------------------------------
``1.1.0``  |``-Python-2.7.15``|``foss/2018b``
``1.1.0``  |``-Python-3.6.4`` |``foss/2018a``
``1.1.0``  |``-Python-3.6.6`` |``foss/2018b``
``1.2.4``  |``-Python-3.7.4`` |``foss/2019b``
``2.1.0``  |                  |``foss/2020b``
``2.1.1``  |                  |``foss/2021a``
``2.1.1.2``|                  |``foss/2021b``, ``foss/2022a``

<a anchor="simpy"/>
### *simpy*

SimPy is a process-based discrete-event simulation framework based on standard Python.

*homepage*: https://simpy.readthedocs.io

version   |versionsuffix    |toolchain
----------|-----------------|---------------
``3.0.11``|                 |``intel/2019a``
``3.0.11``|``-Python-3.6.6``|``intel/2018b``

<a anchor="simvascular"/>
### *SimVascular*

SimVascular is an open source software suite for cardiovascular simulation, providing a complete  pipeline from medical image data to 3D model construction, meshing, and blood flow simulation.

*homepage*: http://simvascular.github.io/

version      |toolchain
-------------|---------------
``2.16.0406``|``intel/2016b``

<a anchor="singlem"/>
### *SingleM*

SingleM is a tool to find the abundances of discrete operational taxonomic units (OTUs) directly  from shotgun metagenome data, without heavy reliance on reference sequence databases.

*homepage*: https://github.com/wwood/singlem

version   |versionsuffix     |toolchain
----------|------------------|---------------
``0.12.1``|``-Python-2.7.15``|``intel/2019a``

<a anchor="singular"/>
### *Singular*

Singular is a computer algebra system for polynomial computations, with special emphasis on commutative and non-commutative algebra, algebraic geometry, and singularity theory.

*homepage*: https://www.singular.uni-kl.de/

version  |toolchain
---------|--------------------------------
``4.1.2``|``GCC/8.2.0-2.31.1``, ``system``

<a anchor="sinvict"/>
### *SiNVICT*

SiNVICT is a tool for the detection of SNVs and indels from cfDNA/ctDNA samples obtained  by ultra-deep sequencing.

*homepage*: https://github.com/sfu-compbio/sinvict

version         |toolchain
----------------|-------------
``1.0-20180817``|``GCC/9.3.0``

<a anchor="sionlib"/>
### *SIONlib*

SIONlib is a scalable I/O library for parallel access to task-local files.  The library not only supports writing and reading binary data to or from  several thousands of processors into a single or a small number of physical  files, but also provides global open and close functions to access SIONlib  files in parallel. This package provides a stripped-down installation of  SIONlib for use with performance tools (e.g., Score-P), with renamed symbols  to avoid conflicts when an application using SIONlib itself is linked against  a tool requiring a different SIONlib version. 

*homepage*: https://www.fz-juelich.de/ias/jsc/EN/Expertise/Support/Software/SIONlib/_node.html

version  |versionsuffix|toolchain
---------|-------------|----------------------------------------------------------------------------
``1.6.1``|             |``foss/2016a``
``1.6.1``|``-tools``   |``foss/2016a``
``1.7.1``|             |``foss/2017a``
``1.7.1``|``-tools``   |``foss/2017a``
``1.7.4``|``-tools``   |``GCCcore/8.2.0``
``1.7.6``|``-tools``   |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.7.7``|``-tools``   |``GCCcore/11.3.0``

<a anchor="sip"/>
### *SIP*

SIP is a tool that makes it very easy to create Python bindings for C and C++ libraries.

*homepage*: http://www.riverbankcomputing.com/software/sip/

version    |versionsuffix     |toolchain
-----------|------------------|----------------------------------------------------------------
``4.18``   |``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``4.18.1`` |``-Python-2.7.11``|``foss/2016a``
``4.18.1`` |``-Python-2.7.12``|``intel/2016b``
``4.19``   |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``4.19.2`` |``-Python-2.7.13``|``intel/2017a``
``4.19.8`` |``-Python-2.7.14``|``foss/2017b``, ``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``4.19.8`` |``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``4.19.13``|``-Python-2.7.15``|``fosscuda/2018b``

<a anchor="sisso"/>
### *SISSO*

A data-driven method combining symbolic regression and compressed sensing toward accurate &  interpretable models.

*homepage*: https://github.com/rouyang2017/SISSO

version         |toolchain
----------------|---------------
``3.0.2``       |``iimpi/2021b``
``3.1-20220324``|``iimpi/2021b``

<a anchor="sisso++"/>
### *SISSO++*

C++ implementation of SISSO with built in Python bindings for an efficient python interface

*homepage*: https://gitlab.com/sissopp_developers/sissopp

version|toolchain
-------|--------------
``1.1``|``foss/2021b``

<a anchor="skesa"/>
### *SKESA*

SKESA is a de-novo sequence read assembler for cultured single isolate genomes based on DeBruijn graphs.

*homepage*: https://github.com/ncbi/SKESA

version  |versionsuffix     |toolchain
---------|------------------|---------------
``2.2``  |                  |``foss/2018a``
``2.3.0``|                  |``foss/2018b``
``2.4.0``|``_saute.1.3.0_1``|``gompi/2021b``

<a anchor="sketchmap"/>
### *sketchmap*

Sketch-map is a dimensionality reduction algorithm that is particularly well suited to examining  the high-dimensionality data that is routinely produced in atomistic simulations.

*homepage*: http://sketchmap.org/

version     |toolchain
------------|---------------
``20170130``|``intel/2016b``

<a anchor="skewer"/>
### *skewer*

skewer implements the bit-masked k-difference matching algorithm dedicated to the task of adapter  trimming and it is specially designed for processing next-generation sequencing (NGS) paired-end sequences.

*homepage*: https://github.com/relipmoc/skewer

version  |toolchain
---------|---------------
``0.2.2``|``intel/2016b``

<a anchor="sklearn-pandas"/>
### *sklearn-pandas*

This module provides a bridge between Scikit-Learn's machine learning methods and pandas-style Data Frames. In particular, it provides a way to map DataFrame columns to transformations, which are later recombined into features.

*homepage*: https://github.com/scikit-learn-contrib/sklearn-pandas

version  |toolchain
---------|--------------
``2.2.0``|``foss/2021b``

<a anchor="sklearn-som"/>
### *sklearn-som*

A simple, planar self-organizing map with methods similar to clustering methods in Scikit Learn.

*homepage*: https://github.com/rileypsmith/sklearn-som

version  |toolchain
---------|--------------
``1.1.0``|``foss/2020b``

<a anchor="skorch"/>
### *skorch*

A scikit-learn compatible neural network library that wraps PyTorch.

*homepage*: https://skorch.readthedocs.io/

version   |versionsuffix      |toolchain
----------|-------------------|--------------
``0.11.0``|``-PyTorch-1.10.0``|``foss/2021a``

<a anchor="slamdunk"/>
### *SlamDunk*

SlamDunk is a novel, fully automated software tool for automated, robust, scalable and reproducible SLAMseq data analysis.

*homepage*: https://t-neumann.github.io/slamdunk

version  |toolchain
---------|--------------
``0.4.3``|``foss/2021b``

<a anchor="slamdunk"/>
### *slamdunk*

SlamDunk is a novel, fully automated software tool for automated, robust, scalable  and reproducible SLAMseq data analysis.

*homepage*: https://t-neumann.github.io/slamdunk/

version  |toolchain
---------|--------------
``0.4.3``|``foss/2021b``

<a anchor="slatec"/>
### *SLATEC*

SLATEC Common Mathematical Library, a comprehensive software library containing     over 1400 general purpose mathematical and statistical routines written in Fortran 77.

*homepage*: https://www.netlib.org/slatec/

version|toolchain
-------|-------------------------------------------------------------------------
``4.1``|``GCC/6.4.0-2.28``, ``GCC/8.3.0``, ``iccifort/2018.1.163-GCC-6.4.0-2.28``

<a anchor="slepc"/>
### *SLEPc*

SLEPc (Scalable Library for Eigenvalue Problem Computations) is a software library for the solution  of large scale sparse eigenvalue problems on parallel computers. It is an extension of PETSc and can be used for  either standard or generalized eigenproblems, with real or complex arithmetic. It can also be used for computing a  partial SVD of a large, sparse, rectangular matrix, and to solve quadratic eigenvalue problems.

*homepage*: https://www.grycap.upv.es/slepc/

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``3.8.3`` |                  |``foss/2017b``
``3.9.2`` |                  |``foss/2018a``
``3.11.0``|                  |``foss/2018b``
``3.12.2``|``-Python-2.7.16``|``intel/2019b``
``3.12.2``|``-Python-3.7.4`` |``foss/2019b``, ``intel/2019b``
``3.12.2``|``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``3.14.2``|                  |``foss/2020b``
``3.15.1``|                  |``foss/2021a``
``3.17.2``|                  |``foss/2022a``

<a anchor="slepc4py"/>
### *slepc4py*

Python bindings for SLEPc, the Scalable Library for Eigenvalue Problem Computations.

*homepage*: https://bitbucket.org/slepc/slepc4py

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``3.9.0`` |``-Python-3.6.4``|``foss/2018a``
``3.12.0``|``-Python-3.7.4``|``foss/2019b``
``3.15.1``|                 |``foss/2021a``

<a anchor="sleuth"/>
### *sleuth*

Investigate RNA-Seq transcript abundance from kallisto and perform differential expression analysis.

*homepage*: http://pachterlab.github.io/sleuth

version   |versionsuffix|toolchain
----------|-------------|---------------
``0.29.0``|``-R-3.4.0`` |``intel/2017a``
``0.30.0``|``-R-3.5.1`` |``foss/2018b``

<a anchor="slidingwindow"/>
### *slidingwindow*

slidingwindow is a simple little Python library for computing a set of windows into a larger dataset,  designed for use with image-processing algorithms that utilise a sliding window to break the processing up  into a series of smaller chunks.

*homepage*: https://github.com/adamrehn/slidingwindow

version   |versionsuffix     |toolchain
----------|------------------|---------------
``0.0.13``|``-Python-2.7.15``|``intel/2018b``
``0.0.13``|``-Python-3.6.6`` |``intel/2018b``

<a anchor="slim"/>
### *SLiM*

SLiM is an evolutionary simulation framework that combines a powerful engine for population  genetic simulations with the capability of modeling arbitrarily complex evolutionary scenarios.

*homepage*: https://messerlab.org/slim

version|toolchain
-------|-------------
``3.4``|``GCC/9.3.0``

<a anchor="slow5tools"/>
### *slow5tools*

slow5tools is a toolkit for converting (FAST5 <-> SLOW5), compressing, viewing, indexing  and manipulating data in SLOW5 format.

*homepage*: https://hasindu2008.github.io/slow5tools

version  |toolchain
---------|---------------
``0.4.0``|``gompi/2021b``

<a anchor="smafa"/>
### *smafa*

Smafa attempts to align or cluster pre-aligned biological sequences, handling sequences  which are all the same length.

*homepage*: https://github.com/wwood/smafa

version  |toolchain
---------|-----------------
``0.4.0``|``GCCcore/8.2.0``

<a anchor="smallgenomeutilities"/>
### *smallgenomeutilities*

The smallgenomeutilities are a collection of scripts that is  useful for dealing and manipulating NGS data of small viral genomes. They  are written in Python 3 with a small number of dependencies.

*homepage*: https://github.com/cbg-ethz/smallgenomeutilities

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.2.1``|``-Python-3.6.6``|``foss/2018b``

<a anchor="smartdenovo"/>
### *SMARTdenovo*

SMARTdenovo is a de novo assembler for PacBio and Oxford Nanopore (ONT) data. It produces an assembly from all-vs-all raw read alignments without an error correction stage. It also provides tools to generate accurate consensus sequences, though a platform dependent consensus polish tools (e.g. Quiver for PacBio or Nanopolish for ONT) are still required for higher accuracy.

*homepage*: https://github.com/ruanjue/smartdenovo

version     |toolchain
------------|--------------
``20180219``|``foss/2018b``

<a anchor="smc++"/>
### *SMC++*

SMC++ is a program for estimating the size history of populations from whole genome sequence data.

*homepage*: https://github.com/popgenmethods/smcpp

version   |toolchain
----------|--------------
``1.15.4``|``foss/2022a``

<a anchor="smithwaterman"/>
### *smithwaterman*

smith-waterman-gotoh alignment algorithm.

*homepage*: https://github.com/ekg/smithwaterman

version     |toolchain
------------|-----------------------------------------------------------------------------
``20160702``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``

<a anchor="smoldyn"/>
### *Smoldyn*

Smoldyn is a computer program for cell-scale biochemical simulations. It simulates each molecule of  interest individually to capture natural stochasticity and to yield nanometer-scale spatial resolution. It treats  other molecules implicitly, enabling it to simulate hundreds of thousands of molecules over several minutes of real  time. Simulated molecules diffuse, react, are confined by surfaces, and bind to membranes much as they would in a real  biological system. 

*homepage*: http://www.smoldyn.org/

version |toolchain
--------|--------------
``2.48``|``foss/2016a``

<a anchor="smooth-topk"/>
### *smooth-topk*

Smooth Loss Functions for Deep Top-k Classification

*homepage*: https://github.com/oval-group/smooth-topk

version         |versionsuffix   |toolchain
----------------|----------------|--------------
``1.0-20210817``|                |``foss/2021a``
``1.0-20210817``|``-CUDA-11.3.1``|``foss/2021a``

<a anchor="smrt-link"/>
### *SMRT-Link*

PacBio’s open-source SMRT Analysis software suite is designed for use with Single Molecule,   Real-Time (SMRT) Sequencing data. You can analyze, visualize, and manage your data through an intuitive GUI   or command-line interface. You can also integrate SMRT Analysis in your existing data workflow through   the extensive set of APIs provided

*homepage*: https://www.pacb.com/support/software-downloads/

version        |versionsuffix      |toolchain
---------------|-------------------|----------
``5.1.0.26412``|``-cli-tools-only``|``system``
``6.0.0.47841``|``-cli-tools-only``|``system``

<a anchor="smv"/>
### *SMV*

Smokeview is a visualization program that displays output of FDS and CFAST simulations.

*homepage*: https://github.com/firemodels/smv

version   |toolchain
----------|-----------------------
``6.7.17``|``iccifort/2020.4.304``

<a anchor="snakemake"/>
### *snakemake*

The Snakemake workflow management system is a tool to create reproducible and scalable data analyses.

*homepage*: https://snakemake.readthedocs.io

version   |versionsuffix    |toolchain
----------|-----------------|-------------------------------
``5.2.2`` |``-Python-3.6.4``|``intel/2018a``
``5.2.4`` |``-Python-3.6.6``|``foss/2018b``, ``intel/2018b``
``5.7.1`` |``-Python-3.7.2``|``foss/2019a``
``5.26.1``|``-Python-3.8.2``|``intel/2020a``
``6.1.0`` |                 |``foss/2020b``
``6.6.1`` |                 |``foss/2021a``
``6.10.0``|                 |``foss/2021b``

<a anchor="snap"/>
### *SNAP*

Scalable Nucleotide Alignment Program -- a fast and accurate read aligner for  high-throughput sequencing data

*homepage*: http://snap.cs.berkeley.edu/

version       |toolchain
--------------|---------------
``1.0beta.23``|``intel/2017b``

<a anchor="snap-hmm"/>
### *SNAP-HMM*

SNAP is a general purpose gene finding program suitable for both eukaryotic and prokaryotic genomes.  SNAP is an acroynm for Semi-HMM-based Nucleic Acid Parser.

*homepage*: https://korflab.github.io/

version       |toolchain
--------------|-------------------------------------------------------------------------
``2013-11-29``|``GCC/6.4.0-2.28``, ``GCC/8.3.0``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``
``20190603``  |``GCC/10.2.0``

<a anchor="snape-pooled"/>
### *SNAPE-pooled*

SNAPE-pooled computes the probability distribution for the frequency of the minor allele  in a certain population, at a certain position in the genome.

*homepage*: https://code.google.com/archive/p/snape-pooled/

version|toolchain
-------|--------------
``r32``|``foss/2016a``

<a anchor="snaphu"/>
### *snaphu*

SNAPHU is an implementation of the Statistical-cost, Network-flow Algorithm for Phase Unwrapping  proposed by Chen and Zebker

*homepage*: https://web.stanford.edu/group/radar/softwareandlinks/sw/snaphu/

version  |toolchain
---------|---------------------------------------------------
``1.4.2``|``GCCcore/6.3.0``, ``intel/2016b``, ``intel/2017a``

<a anchor="snappy"/>
### *snappy*

Snappy is a compression/decompression library. It does not aim for maximum compression, or compatibility with any other compression library; instead, it aims for very high speeds and reasonable compression.

*homepage*: https://github.com/google/snappy

version  |toolchain
---------|------------------------------------------------------------------------------------------------------------
``1.1.2``|``GCC/4.9.2``
``1.1.3``|``GCC/4.9.3``, ``GCC/4.9.3-2.25``
``1.1.6``|``system``
``1.1.7``|``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``intel/2017a``, ``intel/2017b``
``1.1.8``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/9.3.0``
``1.1.9``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``

<a anchor="snippy"/>
### *snippy*

Rapid haploid variant calling and core genome alignment

*homepage*: https://github.com/tseemann/snippy

version  |versionsuffix            |toolchain
---------|-------------------------|--------------
``4.4.1``|``-Perl-5.28.0``         |``foss/2018b``
``4.6.0``|                         |``GCC/10.2.0``
``4.6.0``|``-Java-13-Python-3.8.2``|``GCC/9.3.0``
``4.6.0``|``-R-4.1.2``             |``foss/2021b``

<a anchor="snp-sites"/>
### *snp-sites*

Finds SNP sites from a multi-FASTA alignment file.

*homepage*: https://sanger-pathogens.github.io/snp-sites/

version  |toolchain
---------|-----------------------------------------------------------------------------
``2.5.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``

<a anchor="snpeff"/>
### *snpEff*

SnpEff is a variant annotation and effect prediction tool.  It annotates and predicts the effects of genetic variants (such as amino acid changes).

*homepage*: https://pcingola.github.io/SnpEff/

version |versionsuffix            |toolchain
--------|-------------------------|------------------
``3.6`` |``-Java-1.7.0_80``       |``system``
``4.1d``|``-Java-1.7.0_80``       |``system``
``4.3t``|``-Java-1.8``            |``system``
``5.0`` |``-Java-13``             |``system``
``5.0`` |``-Java-13-Python-3.8.2``|``GCCcore/9.3.0``
``5.0e``|``-Java-11``             |``GCCcore/11.2.0``
``5.0e``|``-Java-13``             |``GCCcore/10.2.0``

<a anchor="snphylo"/>
### *SNPhylo*

SNPhylo: a pipeline to generate a phylogenetic tree from huge SNP data

*homepage*: http://chibba.pgml.uga.edu/snphylo/

version     |versionsuffix             |toolchain
------------|--------------------------|-------------------------------
``20140701``|                          |``foss/2016a``, ``intel/2016a``
``20160204``|``-Python-2.7.14-R-3.4.3``|``foss/2017b``, ``intel/2017b``

<a anchor="snpomatic"/>
### *SNPomatic*

High throughput sequencing technologies generate large amounts of short reads.  Mapping these to a reference sequence consumes large amounts of processing  time and memory, and read mapping errors can lead to noisy or incorrect  alignments. SNP-o-matic is a fast, memory-efficient, and stringent read  mapping tool offering a variety of analytical output functions, with an  emphasis on genotyping. 

*homepage*: https://github.com/magnusmanske/snpomatic

version|toolchain
-------|-----------------
``1.0``|``GCCcore/9.3.0``

<a anchor="soapaligner"/>
### *SOAPaligner*

SOAPaligner/soap2 is a member of the SOAP (Short Oligonucleotide   Analysis Package). It is an updated version of SOAP software for short oligonucleotide   alignment. 

*homepage*: http://soap.genomics.org.cn/soapaligner.html

version |versionsuffix    |toolchain
--------|-----------------|----------
``2.21``|``_Linux-x86_64``|``system``

<a anchor="soapdenovo-trans"/>
### *SOAPdenovo-Trans*

SOAPdenovo-Trans is a de novo transcriptome assembler basing on the SOAPdenovo framework, adapt to alternative splicing and different expression level among transcripts. 

*homepage*: http://soap.genomics.org.cn/SOAPdenovo-Trans.html

version  |toolchain
---------|---------------
``1.0.4``|``intel/2017a``

<a anchor="soapdenovo2"/>
### *SOAPdenovo2*

SOAPdenovo is a novel short-read assembly method that can build a   de novo draft assembly for human-sized genomes. The program is specially designed to   assemble Illumina short reads. It creates new opportunities for building reference   sequences and carrying out accurate analyses of unexplored genomes in a cost effective way.   SOAPdenovo2 is the successor of SOAPdenovo.

*homepage*: http://soap.genomics.org.cn/index.html

version |toolchain
--------|-------------------------------------------------------------------------------------------
``r240``|``GCC/5.4.0-2.26``
``r241``|``GCC/6.4.0-2.28``, ``foss/2018a``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``, ``intel/2018a``

<a anchor="soapfuse"/>
### *SOAPfuse*

SOAPfuse is an open source tool developed for genome-wide detection of fusion  transcripts from paired-end RNA-Seq data.

*homepage*: https://sourceforge.net/p/soapfuse/wiki/Home/

version |versionsuffix   |toolchain
--------|----------------|--------------
``1.27``|``-Perl-5.24.0``|``foss/2016b``
``1.27``|``-Perl-5.28.0``|``foss/2018b``

<a anchor="socat"/>
### *socat*

socat is a relay for bidirectional data transfer between two independent data channels.

*homepage*: http://www.dest-unreach.org/socat

version    |toolchain
-----------|-----------------
``1.7.3.3``|``GCCcore/8.2.0``

<a anchor="soci"/>
### *SOCI*

SOCI is a database access library for C++ that makes the illusion of embedding SQL queries in the  regular C++ code, staying entirely within the Standard C++.

*homepage*: http://soci.sourceforge.net/

version  |toolchain
---------|---------------------------------
``4.0.1``|``GCC/10.2.0``, ``GCCcore/9.3.0``
``4.0.2``|``GCC/10.3.0``
``4.0.3``|``GCC/11.2.0``, ``GCC/11.3.0``

<a anchor="solexaqa++"/>
### *SolexaQA++*

SolexaQA calculates sequence quality statistics and creates visual representations  of data quality for second-generation sequencing data. Originally developed for the Illumina system  (historically known as “Solexa”), SolexaQA now also supports Ion Torrent and 454 data.

*homepage*: http://solexaqa.sourceforge.net

version  |toolchain
---------|--------------
``3.1.5``|``foss/2016b``

<a anchor="sonic"/>
### *sonic*

Sonic is a simple algorithm for speeding up or slowing down speech.  However, it's optimized for speed ups of over 2X, unlike previous algorithms for changing speech rate.  The Sonic library is a very simple ANSI C library that is designed to easily be integrated into streaming voice applications, like TTS back ends. 

*homepage*: https://github.com/espeak-ng/sonic

version     |toolchain
------------|---------------
``20180202``|``gompi/2020a``

<a anchor="soqt"/>
### *SoQt*

SoQt is a library which provides the glue between Systems in  Motion's Coin high-level 3D visualization library and the Qt 2D user interface  library. 

*homepage*: https://coin3d.github.io/SoQt/html/

version  |toolchain
---------|------------------------------
``1.6.0``|``GCC/10.3.0``, ``GCC/11.2.0``

<a anchor="sortmerna"/>
### *SortMeRNA*

SortMeRNA is a biological sequence analysis tool for filtering, mapping and OTU-picking NGS reads.

*homepage*: http://bioinfo.lifl.fr/RNA/sortmerna/

version|toolchain
-------|--------------
``2.1``|``foss/2016a``

<a anchor="sox"/>
### *SoX*

Sound eXchange, the Swiss Army knife of audio manipulation

*homepage*: http://sox.sourceforge.net/

version   |toolchain
----------|---------------------------------
``14.4.2``|``GCC/8.3.0``, ``GCCcore/11.3.0``

<a anchor="soxt"/>
### *SoXt*

SoXt is an Xt/Motif glue library for Coin. It can also be used  on top of the SGI or TGS implementation of Open Inventor, and is designed to  be source code compatible with SGI's InventorXt library. 

*homepage*: https://coin3d.github.io/SoXt/html/

version  |toolchain
---------|--------------
``1.4.0``|``GCC/11.2.0``

<a anchor="spaceranger"/>
### *SpaceRanger*

Space Ranger is a set of analysis pipelines that process Visium spatial RNA-seq output and brightfield microscope images in order to detect tissue, align reads, generate feature-spot matrices, perform clustering and gene expression analysis, and place spots in spatial context on the slide image. 

*homepage*: https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/what-is-space-ranger

version  |toolchain
---------|--------------
``1.1.0``|``GCC/9.3.0``
``1.2.2``|``GCC/9.3.0``
``1.3.0``|``GCC/10.3.0``
``1.3.1``|``GCC/11.2.0``
``2.0.0``|``GCC/11.2.0``

<a anchor="spack"/>
### *Spack*

Spack is a package manager for supercomputers, Linux, and macOS. It makes installing scientific  software easy. With Spack, you can build a package with multiple versions, configurations, platforms, and compilers,  and all of these builds can coexist on the same machine.

*homepage*: https://spack.io/

version   |toolchain
----------|----------
``0.10.0``|``system``
``0.11.2``|``system``
``0.12.1``|``system``
``0.16.2``|``system``
``0.17.0``|``system``
``0.17.2``|``system``

<a anchor="spacy"/>
### *spaCy*

Industrial-strength Natural Language Processing (NLP) in Python.

*homepage*: https://spacy.io/

version  |toolchain
---------|--------------
``3.4.4``|``foss/2022a``

<a anchor="spades"/>
### *SPAdes*

Genome assembler for single-cell and isolates data sets

*homepage*: http://bioinf.spbau.ru/en/spades

version   |versionsuffix     |toolchain
----------|------------------|----------------------------------------------
``3.9.0`` |                  |``foss/2016a``, ``foss/2016b``
``3.10.1``|                  |``foss/2016b``, ``foss/2017a``
``3.11.1``|                  |``foss/2017b``, ``foss/2018a``
``3.12.0``|                  |``foss/2016b``, ``foss/2018a``, ``foss/2018b``
``3.13.0``|                  |``GCC/10.3.0``, ``foss/2018b``
``3.13.1``|                  |``GCC/8.2.0-2.31.1``
``3.14.0``|``-Python-3.7.2`` |``GCC/8.2.0-2.31.1``
``3.14.0``|``-Python-3.7.4`` |``GCC/8.3.0``
``3.14.1``|``-Python-3.8.2`` |``GCC/9.3.0``
``3.15.2``|                  |``GCC/10.2.0``
``3.15.2``|``-Python-2.7.18``|``GCC/10.2.0``
``3.15.3``|                  |``GCC/10.3.0``, ``GCC/11.2.0``

<a anchor="spaln"/>
### *spaln*

Spaln (space-efficient spliced alignment) is a stand-alone program that maps  and aligns a set of cDNA or protein sequences onto a whole genomic sequence in a single job.

*homepage*: https://github.com/ogotoh/spaln

version   |toolchain
----------|----------------------------------------
``2.3.3c``|``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``2.4.03``|``iccifort/2019.5.281``
``2.4.12``|``GCC/10.2.0``, ``GCC/11.2.0``

<a anchor="spark"/>
### *Spark*

Spark is Hadoop MapReduce done in memory

*homepage*: https://spark.apache.org

version  |versionsuffix                              |toolchain
---------|-------------------------------------------|----------------------------------
``1.3.0``|                                           |``system``
``1.4.1``|                                           |``system``
``1.5.0``|                                           |``system``
``1.6.0``|                                           |``system``
``1.6.1``|                                           |``system``
``2.0.0``|                                           |``system``
``2.0.2``|                                           |``system``
``2.2.0``|``-Hadoop-2.6-Java-1.8.0_144``             |``system``
``2.2.0``|``-Hadoop-2.6-Java-1.8.0_152``             |``system``
``2.2.0``|``-Hadoop-2.6-Java-1.8.0_152-Python-3.6.3``|``intel/2017b``
``2.3.0``|``-Hadoop-2.7-Java-1.8.0_162``             |``system``
``2.4.0``|``-Hadoop-2.7-Java-1.8``                   |``system``
``2.4.0``|``-Hadoop-2.7-Java-1.8-Python-3.6.6``      |``intel/2018b``
``2.4.0``|``-Python-2.7.15``                         |``foss/2018b``, ``intel/2018b``
``2.4.0``|``-Python-3.6.6``                          |``intel/2018b``
``2.4.5``|``-Python-3.7.4-Java-1.8``                 |``intel/2019b``
``3.0.0``|``-Python-2.7.15``                         |``foss/2018b``, ``intel/2018b``
``3.1.1``|                                           |``foss/2020b``, ``fosscuda/2020b``
``3.1.1``|``-Python-3.8.2``                          |``foss/2020a``
``3.2.1``|                                           |``foss/2021b``

<a anchor="sparsehash"/>
### *sparsehash*

An extremely memory-efficient hash_map implementation. 2 bits/entry overhead!  The SparseHash library contains several hash-map implementations, including  implementations that optimize for space or speed. 

*homepage*: https://github.com/sparsehash/sparsehash

version  |toolchain
---------|------------------------------------------------------------------------------------------------------------------------------
``2.0.2``|``foss/2016a``
``2.0.3``|``GCCcore/5.4.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``foss/2016b``, ``intel/2017a``
``2.0.4``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``

<a anchor="spatialreg"/>
### *spatialreg*

A collection of all the estimation functions for spatial cross-sectional models (on lattice/areal  data using spatial weights matrices) contained up to now in 'spdep', 'sphet' and 'spse'.

*homepage*: https://cran.r-project.org/package=spatialreg

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.1-5``|``-R-3.6.2`` |``foss/2019b``
``1.1-8``|``-R-4.1.0`` |``foss/2021a``

<a anchor="spdlog"/>
### *spdlog*

Very fast, header-only/compiled, C++ logging library.

*homepage*: https://github.com/gabime/spdlog

version  |toolchain
---------|------------------
``1.9.2``|``GCCcore/10.3.0``

<a anchor="spectra"/>
### *SpectrA*

Spectra stands for Sparse Eigenvalue Computation Toolkit as a Redesigned ARPACK. It is a C++  library for large scale eigenvalue problems, built on top of Eigen, an open source linear algebra library.

*homepage*: https://spectralib.org/

version  |toolchain
---------|--------------------------------------
``1.0.0``|``GCC/10.2.0``
``1.0.1``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="spectral.methods"/>
### *spectral.methods*

Contains some implementations of Singular Spectrum Analysis (SSA) for the gapfilling and  spectral decomposition of time series.

*homepage*: https://cran.r-project.org/web/packages/spectral.methods

version      |versionsuffix|toolchain
-------------|-------------|---------------
``0.7.2.133``|``-R-3.4.3`` |``intel/2017b``
``0.7.2.133``|``-R-3.4.4`` |``intel/2018a``

<a anchor="speech_tools"/>
### *speech_tools*

The Edinburgh Speech Tools Library is a collection of C++ class, functions and related programs for manipulating the sorts of objects used in speech processing. It includes support for reading and writing waveforms, parameter files (LPC, Ceptra, F0) in various formats and converting between them. It also includes support for linguistic type objects and support for various label files and ngrams (with smoothing). 

*homepage*: ['http://festvox.org/festival/']

version  |toolchain
---------|-----------------
``2.5.0``|``GCCcore/9.3.0``

<a anchor="spglib"/>
### *spglib*

Spglib is a library for finding and handling crystal symmetries written in C.

*homepage*: https://atztogo.github.io/spglib/

version   |toolchain
----------|-------------------------
``1.9.2`` |``intel/2016.02-GCC-4.9``
``1.9.9`` |``intel/2017b``
``1.16.1``|``GCCcore/10.2.0``
``1.16.2``|``GCCcore/10.3.0``

<a anchor="spglib-python"/>
### *spglib-python*

Spglib for Python. Spglib is a library for finding and handling crystal symmetries written in C.

*homepage*: https://pypi.python.org/pypi/spglib

version         |versionsuffix     |toolchain
----------------|------------------|--------------------------------------------------------
``1.9.4.2``     |``-Python-2.7.12``|``intel/2016b``
``1.9.5``       |``-Python-2.7.12``|``intel/2016b``
``1.9.9.38``    |``-Python-2.7.13``|``intel/2017a``
``1.10.0.2``    |``-Python-2.7.14``|``intel/2017b``
``1.10.0.2``    |``-Python-3.6.3`` |``intel/2017b``
``1.14.1.post0``|``-Python-3.7.2`` |``intel/2019a``
``1.16.0``      |                  |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``
``1.16.0``      |``-Python-3.7.4`` |``fosscuda/2019b``, ``intel/2019b``, ``intelcuda/2019b``
``1.16.0``      |``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``1.16.1``      |                  |``foss/2021a``, ``gomkl/2021a``, ``intel/2021a``
``1.16.3``      |                  |``foss/2021b``, ``intel/2021b``
``2.0.0``       |                  |``foss/2022a``, ``intel/2022a``

<a anchor="sphinx"/>
### *Sphinx*

Sphinx is a tool that makes it easy to create intelligent and beautiful documentation.  It was originally created for the new Python documentation, and it has excellent facilities for the   documentation of Python projects, but C/C++ is already supported as well, and it is planned to add   special support for other languages as well.

*homepage*: http://sphinx.pocoo.org/

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``1.4.8``|``-Python-2.7.11``|``foss/2016a``
``1.4.8``|``-Python-3.5.1`` |``foss/2016a``
``1.8.1``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``1.8.1``|``-Python-2.7.15``|``foss/2018b``
``1.8.1``|``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``1.8.1``|``-Python-3.6.4`` |``foss/2018a``
``1.8.1``|``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``1.8.3``|``-Python-3.6.4`` |``intel/2018a``

<a anchor="sphinx-rtd-theme"/>
### *Sphinx-RTD-Theme*

Sphinx theme designed to provide a great reader experience  for documentation users on both desktop and mobile devices.

*homepage*: https://sphinx-rtd-theme.readthedocs.io

version  |toolchain
---------|------------------
``1.1.1``|``GCCcore/10.2.0``

<a anchor="spiceypy"/>
### *SpiceyPy*

SpiceyPy is a Python wrapper for the NAIF C SPICE Toolkit (N65)

*homepage*: https://github.com/AndrewAnnex/SpiceyPy

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``1.1.0``|``-Python-2.7.12``|``intel/2016b``
``1.1.1``|``-Python-3.6.1`` |``intel/2017a``
``2.1.0``|``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``2.1.0``|``-Python-3.6.4`` |``foss/2018a``

<a anchor="spieceasi"/>
### *SpiecEasi*

Sparse InversE Covariance estimation for Ecological Association and Statistical Inference

*homepage*: https://github.com/zdk123/SpiecEasi

version     |versionsuffix|toolchain
------------|-------------|---------------
``1.0.0``   |``-R-3.4.4`` |``intel/2018a``
``20160830``|``-R-3.3.1`` |``intel/2016b``

<a anchor="spladder"/>
### *SplAdder*

Splicing Adder is a toolbox for alternative splicing analysis based on RNA-Seq alignment data. Briefly, the software takes a given annotation and RNA-Seq read alignments in standardized formats, transforms the annotation into a splicing graph representation, augments the splicing graph with additional information extracted from the read data, extracts alternative splicing events from the graph and quantifies the events based on the alignment data.

*homepage*: https://github.com/ratschlab/spladder

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``2.4.2``|``-Python-3.8.2``|``foss/2020a``

<a anchor="splash"/>
### *SPLASH*

SPLASH is a free and open source visualisation tool  for Smoothed Particle Hydrodynamics (SPH) simulations.

*homepage*: http://users.monash.edu.au/~dprice/splash/index.html

version  |toolchain
---------|--------------
``2.8.0``|``foss/2018b``

<a anchor="splicemap"/>
### *SpliceMap*

SpliceMap is a de novo splice junction discovery and alignment tool.  It offers high sensitivity and support for arbitrary RNA-seq read lengths.

*homepage*: https://web.stanford.edu/group/wonglab/SpliceMap/

version    |toolchain
-----------|------------------
``3.3.5.2``|``GCC/7.3.0-2.30``

<a anchor="split-seq"/>
### *split-seq*

Analysis tools for split-seq.

*homepage*: https://github.com/yjzhang/split-seq-pipeline

version     |versionsuffix    |toolchain
------------|-----------------|--------------
``20190717``|``-Python-3.6.6``|``foss/2018b``

<a anchor="splitref"/>
### *splitRef*

splitRef splits a reference haplotype file into smaller files   with subsets of markers. The current version is a pre-release. 

*homepage*: http://www.sph.umich.edu/csg/yli/splitRef/index.html

version  |toolchain
---------|----------
``0.0.2``|``system``

<a anchor="spm"/>
### *SPM*

SPM (Statistical Parametric Mapping) refers to the construction  and assessment of spatially extended statistical processes used to test hypo-  theses about functional imaging data. These ideas have been instantiated in  software that is called SPM.  The SPM software package has been designed for the analysis of brain imaging  data sequences. The sequences can be a series of images from different cohorts,  or time-series from the same subject. The current release is designed for the  analysis of fMRI, PET, SPECT, EEG and MEG. 

*homepage*: https://www.fil.ion.ucl.ac.uk/spm

version       |versionsuffix    |toolchain
--------------|-----------------|----------
``12.5_r7771``|``-MATLAB-2021a``|``system``
``12.5_r7771``|``-MATLAB-2021b``|``system``

<a anchor="spoa"/>
### *spoa*

Spoa (SIMD POA) is a c++ implementation of the partial order alignment (POA) algorithm  which is used to generate consensus sequences

*homepage*: https://github.com/rvaser/spoa

version  |toolchain
---------|----------------------------------------------
``3.0.1``|``GCC/7.3.0-2.30``
``3.4.0``|``GCC/10.2.0``
``4.0.0``|``GCC/8.3.0``
``4.0.7``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``

<a anchor="spooles"/>
### *SPOOLES*

SPOOLES is a library for solving sparse real and complex linear  systems of equations, written in the C language using object oriented design. 

*homepage*: https://netlib.org/linalg/spooles/spooles.2.2.html

version|toolchain
-------|---------------
``2.2``|``gompi/2021a``

<a anchor="spotpy"/>
### *SPOTPY*

SPOTPY is a Python framework that enables the use of Computational optimization techniques for calibration, uncertainty and sensitivity analysis techniques of almost every (environmental-) model.

*homepage*: https://spotpy.readthedocs.io

version   |toolchain
----------|---------------
``1.5.14``|``intel/2021b``

<a anchor="sprng"/>
### *SPRNG*

Scalable Parallel Pseudo Random Number Generators Library

*homepage*: http://www.sprng.org/

version |toolchain
--------|--------------
``2.0b``|``foss/2016a``

<a anchor="spyder"/>
### *Spyder*

Spyder is an interactive Python development environment providing MATLAB-like features in a simple  and light-weighted software.

*homepage*: https://github.com/spyder-ide/spyder

version  |versionsuffix     |toolchain
---------|------------------|---------------
``3.1.4``|``-Python-2.7.13``|``intel/2017a``
``3.3.1``|``-Python-3.6.4`` |``foss/2018a``
``3.3.2``|``-Python-3.6.6`` |``foss/2018b``
``4.1.5``|``-Python-3.7.2`` |``foss/2019a``
``4.1.5``|``-Python-3.8.2`` |``foss/2020a``

<a anchor="sqlite"/>
### *SQLite*

SQLite: SQL Database Engine in a C Library

*homepage*: http://www.sqlite.org/

version     |toolchain
------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------
``3.8.8.1`` |``GCC/4.8.4``, ``GCC/4.9.2``
``3.8.10.2``|``GCC/4.9.3-2.25``, ``GNU/4.9.3-2.25``, ``gimkl/2.11.5``
``3.9.2``   |``GCC/4.9.3-2.25``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``
``3.13.0``  |``GCC/4.9.3-2.25``, ``GCC/5.4.0-2.26``, ``GCCcore/6.3.0``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``, ``intel/2016b``
``3.14.1``  |``GCCcore/4.9.3``
``3.17.0``  |``GCCcore/6.3.0``
``3.20.1``  |``GCCcore/6.4.0``
``3.21.0``  |``GCCcore/6.4.0``
``3.23.0``  |``GCCcore/6.4.0``
``3.24.0``  |``GCCcore/7.2.0``, ``GCCcore/7.3.0``
``3.26.0``  |``GCCcore/8.2.0``
``3.27.2``  |``GCCcore/8.2.0``
``3.29.0``  |``GCCcore/8.3.0``
``3.31.1``  |``GCCcore/9.3.0``
``3.33.0``  |``GCCcore/10.2.0``
``3.35.4``  |``GCCcore/10.3.0``
``3.36``    |``GCCcore/11.2.0``
``3.38.3``  |``GCCcore/11.3.0``
``3.39.4``  |``GCCcore/12.2.0``

<a anchor="squeezemeta"/>
### *SqueezeMeta*

SqueezeMeta is a full automatic pipeline for metagenomics/metatranscriptomics,  covering all steps of the analysis.

*homepage*: https://github.com/jtamames/SqueezeMeta

version  |versionsuffix     |toolchain
---------|------------------|--------------
``0.4.3``|``-Python-2.7.15``|``foss/2018b``
``1.0.0``|``-Python-2.7.15``|``foss/2018b``
``1.5.0``|                  |``foss/2021b``

<a anchor="squidpy"/>
### *Squidpy*

Squidpy is a tool for the analysis and visualization of spatial molecular data.

*homepage*: https://squidpy.readthedocs.io

version  |toolchain
---------|--------------
``1.2.2``|``foss/2021b``

<a anchor="sra-toolkit"/>
### *SRA-Toolkit*

The SRA Toolkit, and the source-code SRA System Development  Kit (SDK), will allow you to programmatically access data housed within SRA  and convert it from the SRA format

*homepage*: https://www.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?view=software

version    |versionsuffix      |toolchain
-----------|-------------------|---------------
``2.3.5``  |``-centos_linux64``|``system``
``2.5.4-1``|``-centos_linux64``|``system``
``2.5.7``  |``-centos_linux64``|``system``
``2.8.2-1``|``-centos_linux64``|``system``
``2.9.0``  |``-centos_linux64``|``system``
``2.9.2``  |``-ubuntu64``      |``system``
``2.9.4``  |``-centos_linux64``|``system``
``2.9.6-1``|``-centos_linux64``|``system``
``2.10.4`` |                   |``gompi/2019b``
``2.10.5`` |``-centos_linux64``|``system``
``2.10.8`` |                   |``gompi/2020a``
``2.10.9`` |                   |``gompi/2020b``
``3.0.0``  |``-centos_linux64``|``system``

<a anchor="srprism"/>
### *SRPRISM*

Single Read Paired Read Indel Substitution Minimizer

*homepage*: https://github.com/ncbi/SRPRISM

version  |versionsuffix|toolchain
---------|-------------|------------------
``3.0.0``|             |``foss/2018b``
``3.1.1``|``-Java-11`` |``GCCcore/8.2.0``
``3.1.2``|             |``GCCcore/10.2.0``

<a anchor="srst2"/>
### *SRST2*

Short Read Sequence Typing for Bacterial Pathogens

*homepage*: https://katholt.github.io/srst2

version           |versionsuffix     |toolchain
------------------|------------------|--------------
``0.2.0-20210620``|``-Python-2.7.18``|``foss/2020b``

<a anchor="ssaha2"/>
### *SSAHA2*

SSAHA2 (Sequence Search and Alignment by Hashing Algorithm) is a   pairwise sequence alignment program designed for the efficient mapping of sequencing   reads onto genomic reference sequences. SSAHA2 reads of most sequencing platforms   (ABI-Sanger, Roche 454, Illumina-Solexa) and a range of output formats (SAM, CIGAR, PSL etc.)   are supported. A pile-up pipeline for analysis and genotype calling is available as   a separate package. 

*homepage*: http://www.sanger.ac.uk/resources/software/ssaha2/

version  |versionsuffix|toolchain
---------|-------------|----------
``2.5.5``|``-i686``    |``system``
``2.5.5``|``-x86_64``  |``system``

<a anchor="ssn"/>
### *SSN*

Spatial statistical modeling and prediction for data on stream networks, including models based on in-stream distance. Models are created using moving average constructions. Spatial linear models, including explanatory variables, can be fit with (restricted) maximum likelihood. Mapping and other graphical functions are included.

*homepage*: https://cran.r-project.org/web/packages/SSN

version   |versionsuffix|toolchain
----------|-------------|--------------
``1.1.14``|``-R-3.6.0`` |``foss/2019a``

<a anchor="sspace_basic"/>
### *SSPACE_Basic*

SSPACE Basic, SSAKE-based Scaffolding of Pre-Assembled Contigs after Extension

*homepage*: https://github.com/nsoranzo/sspace_basic

version  |versionsuffix     |toolchain
---------|------------------|---------------
``2.1.1``|``-Perl-5.24.1``  |``intel/2017a``
``2.1.1``|``-Perl-5.26.0``  |``intel/2017b``
``2.1.1``|``-Python-2.7.18``|``GCC/10.2.0``

<a anchor="ssw"/>
### *SSW*

SSW is a fast implementation of the Smith-Waterman algorithm, which uses the  Single-Instruction Multiple-Data (SIMD) instructions to parallelize the algorithm at the  instruction level. SSW library provides an API that can be flexibly used by programs written in  C, C++ and other languages. We also provide a software that can do protein and genome alignment  directly. Current version of our implementation is ~50 times faster than an ordinary  Smith-Waterman. It can return the Smith-Waterman score, alignment location and traceback path  (cigar) of the optimal alignment accurately; and return the sub-optimal alignment score and  location heuristically. 

*homepage*: https://github.com/mengyao/Complete-Striped-Smith-Waterman-Library

version|toolchain
-------|-----------------------------------------------------------------------------
``1.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``

<a anchor="stacey"/>
### *STACEY*

The BEAST2 package STACEY can be used for species delimitation and species tree estimation, based on the multispecies coalescent model.

*homepage*: http://www.indriid.com/software.html

version  |toolchain
---------|--------------
``1.2.5``|``GCC/10.2.0``

<a anchor="stack"/>
### *Stack*

Stack is a cross-platform program for developing Haskell projects. It is intended for Haskellers both new and experienced.

*homepage*: https://docs.haskellstack.org

version  |versionsuffix|toolchain
---------|-------------|----------
``2.3.3``|``-x86_64``  |``system``

<a anchor="stacks"/>
### *Stacks*

Stacks is a software pipeline for building loci from short-read sequences, such as those generated on  the Illumina platform. Stacks was developed to work with restriction enzyme-based data, such as RAD-seq,  for the purpose of building genetic maps and conducting population genomics and phylogeography. 

*homepage*: https://catchenlab.life.illinois.edu/stacks/

version       |toolchain
--------------|------------------------------------------------------------------------------
``1.40``      |``foss/2016a``
``1.42``      |``foss/2016a``
``1.44``      |``foss/2016a``
``1.45``      |``foss/2016a``
``1.46``      |``intel/2017a``
``1.47``      |``foss/2016a``
``1.48``      |``intel/2017b``, ``intel/2018b``
``2.0``       |``foss/2018a``, ``intel/2018a``
``2.0Beta7c`` |``intel/2017b``
``2.0Beta8c`` |``intel/2017b``
``2.0Beta9``  |``intel/2018a``
``2.0Beta10a``|``foss/2018a``
``2.2``       |``foss/2018a``
``2.3b``      |``foss/2018a``
``2.3e``      |``foss/2018b``
``2.5``       |``iccifort/2019.5.281``
``2.41``      |``GCC/8.2.0-2.31.1``, ``foss/2018b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``2.53``      |``foss/2019b``, ``iccifort/2019.5.281``
``2.54``      |``foss/2020a``
``2.62``      |``foss/2022a``

<a anchor="stamp"/>
### *STAMP*

STAMP is a tool for characterizing similarities between transcription factor binding motifs

*homepage*: http://www.benoslab.pitt.edu/stamp/

version|toolchain
-------|--------------------------------
``1.2``|``intel/2016a``
``1.3``|``intel/2016a``, ``intel/2016b``

<a anchor="stamps"/>
### *StaMPS*

A software package to extract ground displacements from time series of synthetic aperture radar  (SAR) acquisitions.

*homepage*: http://homepages.see.leeds.ac.uk/~earahoo/stamps/

version  |versionsuffix   |toolchain
---------|----------------|---------------
``3.3b1``|``-Perl-5.24.1``|``intel/2017a``

<a anchor="stampy"/>
### *Stampy*

Stampy is a package for the mapping of short reads from illumina sequencing machines onto  a reference genome.

*homepage*: http://www.well.ox.ac.uk/stampy

version   |versionsuffix     |toolchain
----------|------------------|---------------
``1.0.31``|``-Python-2.7.12``|``intel/2016b``
``1.0.32``|``-Python-2.7.14``|``intel/2018a``

<a anchor="star"/>
### *STAR*

STAR aligns RNA-seq reads to a reference genome using uncompressed suffix arrays.

*homepage*: https://github.com/alexdobin/STAR

version                 |toolchain
------------------------|--------------------------------------------------------------
``2.4.2a``              |``foss/2018b``
``2.5.0a``              |``GNU/4.9.3-2.25``
``2.5.2a``              |``foss/2016a``
``2.5.2b``              |``intel/2016b``
``2.5.3a``              |``GCC/8.3.0``, ``GCC/9.3.0``, ``intel/2017a``, ``intel/2017b``
``2.5.4b``              |``foss/2016b``, ``foss/2017b``, ``intel/2017b``
``2.6.0c``              |``foss/2018a``, ``intel/2018a``
``2.6.1c``              |``foss/2018b``
``2.7.0d``              |``foss/2018b``
``2.7.0f``              |``GCC/8.2.0-2.31.1``, ``foss/2018b``
``2.7.1a``              |``GCC/8.2.0-2.31.1``, ``foss/2018b``
``2.7.2b``              |``GCC/8.3.0``
``2.7.3a``              |``GCC/8.3.0``, ``GCC/9.3.0``
``2.7.4a``              |``GCC/9.3.0``
``2.7.5b``              |``GCC/9.3.0``
``2.7.6a``              |``GCC/10.2.0``, ``GCC/9.3.0``
``2.7.7a``              |``GCC/10.2.0``, ``GCC/9.3.0``
``2.7.8a``              |``GCC/10.2.0``
``2.7.9a``              |``GCC/10.3.0``, ``GCC/11.2.0``
``2.7.10a_alpha_220601``|``GCC/10.3.0``

<a anchor="star-ccm+"/>
### *STAR-CCM+*

STAR-CCM+ is a multiphysics computational fluid dynamics (CFD) software for the simulation of products operating under real-world conditions.

*homepage*: https://mdx.plm.automation.siemens.com/star-ccm-plus

version      |versionsuffix|toolchain
-------------|-------------|----------
``13.04.011``|             |``system``
``17.02.008``|             |``system``
``17.02.008``|``-r8``      |``system``
``17.04.008``|             |``system``
``17.04.008``|``-r8``      |``system``
``17.06.007``|             |``system``
``17.06.007``|``-r8``      |``system``

<a anchor="star-fusion"/>
### *STAR-Fusion*

STAR-Fusion uses the STAR aligner to identify candidate fusion transcripts  supported by Illumina reads. STAR-Fusion further processes the output generated  by the STAR aligner to map junction reads and spanning reads to a reference  annotation set.

*homepage*: https://github.com/STAR-Fusion/STAR-Fusion

version  |versionsuffix                |toolchain
---------|-----------------------------|--------------------
``1.5.0``|``-Perl-5.28.0``             |``foss/2018b``
``1.6.0``|``-Perl-5.28.1-Python-3.7.2``|``GCC/8.2.0-2.31.1``
``1.8.1``|``-Python-3.7.4``            |``foss/2019b``

<a anchor="stars"/>
### *stars*

Reading, manipulating, writing and plotting spatiotemporal arrays (raster and vector data cubes)  in R, using GDAL bindings provided by sf, and NetCDF bindings by ncmeta and RNetCDF.

*homepage*: https://cran.r-project.org/package=stars

version  |versionsuffix            |toolchain
---------|-------------------------|--------------
``0.4-3``|``-R-4.0.0-Python-3.8.2``|``foss/2020a``

<a anchor="stata"/>
### *Stata*

Stata is a complete, integrated statistical software package that provides everything you need  for data analysis, data management, and graphics.

*homepage*: https://www.stata.com/

version|versionsuffix|toolchain
-------|-------------|----------
``15`` |             |``system``
``16`` |``-legacy``  |``system``
``17`` |             |``system``

<a anchor="statistics-r"/>
### *Statistics-R*

Perl interface with the R statistical program

*homepage*: https://metacpan.org/pod/Statistics::R

version |toolchain
--------|--------------
``0.34``|``foss/2020a``

<a anchor="statsmodels"/>
### *statsmodels*

Statsmodels is a Python module that provides classes and functions  for the estimation of many different statistical models, as well as for conducting   statistical tests, and statistical data exploration.

*homepage*: https://www.statsmodels.org

version   |versionsuffix     |toolchain
----------|------------------|---------------------------------------------------
``0.6.1`` |``-Python-2.7.13``|``foss/2017a``
``0.6.1`` |``-Python-3.5.1`` |``intel/2016a``
``0.6.1`` |``-Python-3.5.2`` |``intel/2016b``
``0.8.0`` |``-Python-2.7.13``|``intel/2017a``
``0.9.0`` |``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``0.9.0`` |``-Python-2.7.16``|``intel/2019b``
``0.9.0`` |``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``0.10.1``|                  |``foss/2019a``
``0.11.0``|``-Python-3.7.4`` |``foss/2019b``, ``intel/2019b``
``0.11.1``|``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``0.12.1``|                  |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``
``0.12.2``|                  |``foss/2021a``
``0.13.1``|                  |``foss/2021b``, ``foss/2022a``, ``intel/2021b``

<a anchor="steak"/>
### *STEAK*

Detects integrations of any sort in high-throughput  sequencing (HTS) data. STEAK was built for validating and discovering  transposable element (TE) and retroviral integrations in a variety of  HTS data. The software performs on both single-end (SE) and paired-end ( PE) libraries and on a variety of HTS sequencing strategies. It can  be applied to a broad range of research interests and clinical uses  such as population genetic studies and detecting polymorphic integrations.

*homepage*: https://omictools.com/steak-tool

version       |versionsuffix     |toolchain
--------------|------------------|--------------
``2019.09.12``|                  |``foss/2021b``
``2019.09.12``|``-Python-2.7.16``|``foss/2019b``

<a anchor="stir"/>
### *STIR*

Software for Tomographic Image Reconstruction

*homepage*: http://stir.sourceforge.net/

version|toolchain
-------|---------------
``3.0``|``intel/2018a``

<a anchor="stpipeline"/>
### *stpipeline*

The ST Pipeline contains the tools and scripts needed to process  and analyze the raw files generated with the Spatial Transcriptomics method  in FASTQ format to generated datasets for down-stream analysis. The ST  pipeline can also be used to process single cell data as long as a file  with barcodes identifying each cell is provided. The ST Pipeline can also  process RNA-Seq datasets generated with or without UMIs.

*homepage*: https://github.com/SpatialTranscriptomicsResearch/st_pipeline

version  |versionsuffix     |toolchain
---------|------------------|--------------
``1.7.3``|``-Python-2.7.15``|``foss/2018b``
``1.7.6``|``-Python-3.7.2`` |``foss/2019a``
``1.7.6``|``-Python-3.7.4`` |``foss/2019b``

<a anchor="stream"/>
### *STREAM*

The STREAM benchmark is a simple synthetic benchmark program that measures sustainable  memory bandwidth (in MB/s) and the corresponding computation rate for simple vector kernels.

*homepage*: https://www.cs.virginia.edu/stream/

version |toolchain
--------|------------------------------------------------------------------------------------------------------------------
``5.10``|``GCC/7.3.0-2.30``, ``GCC/8.2.0-2.31.1``, ``GCC/9.3.0``, ``iccifort/2020.1.217``, ``intel/2016b``, ``intel/2018b``

<a anchor="strelka"/>
### *strelka*

Strelka2 is a fast and accurate small variant caller  optimized for analysis of germline variation in small cohorts and  somatic variation in tumor/normal sample pairs.

*homepage*: https://github.com/Illumina/strelka

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``2.9.7`` |                  |``intel/2018a``
``2.9.9`` |                  |``foss/2018b``
``2.9.10``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``

<a anchor="stringtie"/>
### *StringTie*

StringTie is a fast and highly efficient assembler of RNA-Seq alignments into potential transcripts

*homepage*: https://ccb.jhu.edu/software/stringtie/

version   |toolchain
----------|----------------------------------
``1.3.0`` |``intel/2016b``
``1.3.3`` |``GCCcore/6.4.0``, ``intel/2017a``
``1.3.3b``|``foss/2016b``
``1.3.5`` |``GCCcore/8.2.0``, ``foss/2018b``
``2.0.3`` |``GCCcore/7.3.0``
``2.1.0`` |``foss/2018b``
``2.1.1`` |``GCC/8.3.0``
``2.1.3`` |``GCC/8.3.0``, ``GCC/9.3.0``
``2.1.4`` |``GCC/8.3.0``, ``GCC/9.3.0``
``2.1.7`` |``GCC/10.3.0``
``2.2.1`` |``GCC/11.2.0``

<a anchor="stripy"/>
### *stripy*

A Python interface to TRIPACK and STRIPACK Fortran code for (constrained) triangulation in Cartesian  coordinates and on a sphere. Stripy is an object-oriented package and includes routines from SRFPACK and SSRFPACK for  interpolation (nearest neighbor, linear and hermite cubic) and to evaluate derivatives (Renka 1996a,b and 1997a,b).

*homepage*: https://github.com/underworldcode/stripy

version  |toolchain
---------|--------------
``2.1.0``|``foss/2021a``

<a anchor="strique"/>
### *STRique*

STRique is a python package to analyze repeat expansion and methylation states of short tandem repeats (STR) in Oxford Nanopore Technology (ONT) long read sequencing data.

*homepage*: https://github.com/giesselmann/STRique

version  |toolchain
---------|--------------
``0.4.2``|``foss/2021b``

<a anchor="structure"/>
### *Structure*

The program structure is a free software package for using multi-locus genotype data to investigate  population structure.

*homepage*: https://web.stanford.edu/group/pritchardlab/structure.html

version  |toolchain
---------|-------------------------------------------------------------------------------------
``2.3.4``|``GCC/8.2.0-2.31.1``, ``iccifort/2019.3.199-GCC-8.3.0-2.32``, ``iccifort/2019.5.281``

<a anchor="strumpack"/>
### *STRUMPACK*

STRUMPACK - STRUctured Matrix PACKage - Fast linear solvers and preconditioner  for both dense and sparse systems using low-rank structured factorization with randomized sampling.

*homepage*: https://fastmath-scidac.llnl.gov/software/strumpack.html

version  |toolchain
---------|-------------------------------
``6.1.0``|``foss/2020b``, ``intel/2020b``

<a anchor="suave"/>
### *suave*

suave is an interactive web application to visualize read depth ratios between two samples and the structural variants of one of the samples (typically the "case" sample in a case/control setup such as tumor/normal comparison).

*homepage*: https://github.com/dellytools/suave

version     |toolchain
------------|--------------
``20160529``|``foss/2020b``

<a anchor="subread"/>
### *Subread*

High performance read alignment, quantification and mutation discovery

*homepage*: http://subread.sourceforge.net/

version     |toolchain
------------|---------------------------------------------
``1.5.0-p1``|``foss/2016a``, ``foss/2016b``
``1.6.3``   |``foss/2018b``
``1.6.4``   |``foss/2018b``
``2.0.0``   |``GCC/7.3.0-2.30``, ``GCC/8.3.0``
``2.0.2``   |``GCC/10.2.0``
``2.0.3``   |``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/9.3.0``

<a anchor="subversion"/>
### *Subversion*

Subversion is an open source version control system.

*homepage*: http://subversion.apache.org/

version   |toolchain
----------|--------------------------------------
``1.9.7`` |``iomkl/2018a``
``1.9.9`` |``GCCcore/7.3.0``
``1.10.0``|``foss/2017b``, ``intel/2017b``
``1.12.0``|``GCCcore/8.2.0``
``1.14.0``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``1.14.1``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``

<a anchor="suds"/>
### *suds*

Lightweight SOAP client

*homepage*: https://pypi.python.org/pypi/suds-py3

version    |versionsuffix    |toolchain
-----------|-----------------|---------------
``1.3.3.0``|``-Python-3.6.4``|``intel/2018a``

<a anchor="suitesparse"/>
### *SuiteSparse*

SuiteSparse is a collection of libraries manipulate sparse matrices.

*homepage*: https://faculty.cse.tamu.edu/davis/suitesparse.html

version   |versionsuffix               |toolchain
----------|----------------------------|-------------------------------------------------------------------------------------------------
``4.5.1`` |``-METIS-5.1.0``            |``foss/2016a``, ``intel/2016a``
``4.5.2`` |``-METIS-5.1.0``            |``foss/2016a``, ``intel/2016a``
``4.5.3`` |``-METIS-5.1.0``            |``foss/2016a``, ``foss/2016b``, ``intel/2016b``
``4.5.3`` |``-ParMETIS-4.0.3``         |``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``4.5.5`` |``-METIS-5.1.0``            |``foss/2017a``, ``intel/2017a``
``4.5.5`` |``-ParMETIS-4.0.3``         |``foss/2017a``, ``foss/2017b``, ``intel/2017a``, ``intel/2017b``
``4.5.6`` |``-METIS-5.1.0``            |``foss/2017b``
``5.1.2`` |``-METIS-5.1.0``            |``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``
``5.1.2`` |``-ParMETIS-4.0.3``         |``foss/2017b``, ``intel/2017b``
``5.4.0`` |``-METIS-5.1.0``            |``foss/2019a``, ``intel/2018b``, ``intel/2019a``
``5.6.0`` |``-METIS-5.1.0``            |``foss/2019b``, ``intel/2019b``
``5.7.1`` |``-METIS-5.1.0``            |``foss/2020a``, ``intel/2020a``
``5.8.1`` |``-METIS-5.1.0``            |``foss/2020b``, ``intel/2020b``
``5.10.1``|``-METIS-5.1.0``            |``foss/2021a``, ``foss/2021b``
``5.10.1``|``-METIS-5.1.0-CUDA-11.3.1``|``foss/2021a``
``5.13.0``|``-METIS-5.1.0``            |``foss/2022a``

<a anchor="sumaclust"/>
### *SUMACLUST*

SUMATRA and SUMACLUST: fast and exact comparison and clustering of sequences.

*homepage*: http://metabarcoding.org/sumatra

version   |toolchain
----------|--------------
``1.0.20``|``foss/2016a``

<a anchor="sumatra"/>
### *SUMATRA*

SUMATRA and SUMACLUST: fast and exact comparison and clustering of sequences.

*homepage*: http://metabarcoding.org/sumatra

version   |toolchain
----------|--------------
``1.0.20``|``foss/2016a``

<a anchor="sumo"/>
### *SUMO*

Simulation of Urban MObility" (SUMO) is an open source, highly portable, microscopic and continuous traffic simulation package designed to handle large networks. It allows for intermodal simulation including pedestrians and comes with a large set of tools for scenario creation.

*homepage*: https://www.eclipse.org/sumo/

version   |versionsuffix    |toolchain
----------|-----------------|-----------------
``1.3.1`` |``-Python-3.7.4``|``GCCcore/8.3.0``
``1.7.0`` |``-Python-3.8.2``|``foss/2020a``
``1.12.0``|                 |``foss/2021b``
``1.14.1``|                 |``foss/2021b``

<a anchor="sundials"/>
### *SUNDIALS*

SUNDIALS: SUite of Nonlinear and DIfferential/ALgebraic Equation Solvers

*homepage*: https://computation.llnl.gov/projects/sundials

version  |toolchain
---------|----------------------------------------------------------------------------------------------------------------------------------
``2.6.2``|``intel/2016b``, ``intel/2018b``
``2.7.0``|``foss/2016b``, ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``
``5.1.0``|``foss/2019b``, ``intel/2019b``
``5.7.0``|``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``
``6.3.0``|``foss/2021b``, ``foss/2022a``

<a anchor="sunpy"/>
### *SunPy*

The community-developed, free and open-source solar data analysis environment for Python.

*homepage*: https://sunpy.org/

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.1.3``|``-Python-3.7.4``|``foss/2019b``

<a anchor="superlu"/>
### *SuperLU*

SuperLU is a general purpose library for the  direct solution of large, sparse, nonsymmetric systems  of linear equations on high performance machines.

*homepage*: https://crd-legacy.lbl.gov/~xiaoye/SuperLU/

version  |toolchain
---------|--------------------------------------------------------------------------------
``5.1.1``|``foss/2016a``, ``intel/2016a``
``5.2.1``|``foss/2017b``, ``intel/2017b``
``5.2.2``|``foss/2020a``, ``intel/2020a``
``5.3.0``|``foss/2020b``, ``foss/2021a``, ``foss/2022a``, ``intel/2020b``, ``intel/2022a``

<a anchor="superlu_dist"/>
### *SuperLU_DIST*

SuperLU is a general purpose library for the direct solution of large, sparse, nonsymmetric systems  of linear equations on high performance machines.

*homepage*: https://crd-legacy.lbl.gov/~xiaoye/SuperLU/

version  |versionsuffix      |toolchain
---------|-------------------|-------------------------------
``5.4.0``|``-trisolve-merge``|``intel/2020a``
``6.4.0``|                   |``foss/2020a``, ``intel/2020a``
``8.1.0``|                   |``foss/2022a``

<a anchor="supermagic"/>
### *supermagic*

Very simple MPI sanity code. Nothing more, nothing less.

*homepage*: https://github.com/hpc/supermagic

version     |toolchain
------------|-------------------------------
``20170824``|``foss/2017a``, ``gompi/2019b``

<a anchor="supernova"/>
### *supernova*

Supernova is a software package for de novo assembly from Chromium Linked-Reads  that are made from a single whole-genome library from an individual DNA source

*homepage*: https://support.10xgenomics.com/de-novo-assembly/software/overview/latest/welcome

version  |toolchain
---------|----------
``2.0.1``|``system``
``2.1.1``|``system``

<a anchor="survivor"/>
### *SURVIVOR*

Toolset for SV simulation, comparison and filtering

*homepage*: https://github.com/fritzsedlazeck/SURVIVOR

version              |toolchain
---------------------|--------------
``1.0.7-19-ged1ca51``|``GCC/11.2.0``

<a anchor="svdetect"/>
### *SVDetect*

SVDetect is a application for the isolation and the type prediction of intra- and inter-chromosomal  rearrangements from paired-end/mate-pair sequencing data provided by the high-throughput sequencing technologies.  This tool aims to identifying structural variations with both clustering and sliding-window strategies, and  helping in their visualization at the genome scale.

*homepage*: http://svdetect.sourceforge.net/Site/Home.html

version |versionsuffix   |toolchain
--------|----------------|----------------------------------------------------------
``0.8b``|``-Perl-5.26.0``|``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``

<a anchor="svdquest"/>
### *SVDquest*

SVDquartets-based species trees

*homepage*: https://github.com/pranjalv123/SVDquest

version     |toolchain
------------|---------------
``20190627``|``gompi/2019a``

<a anchor="svg"/>
### *SVG*

Perl binding for SVG

*homepage*: https://metacpan.org/pod/SVG

version |versionsuffix   |toolchain
--------|----------------|--------------
``2.84``|``-Perl-5.30.0``|``foss/2019b``

<a anchor="svist4get"/>
### *svist4get*

Svist4get is a simple bioinformatics tool for visualization of  genomic signal tracks in user-defined genomic windows, either arbitrary selected  by genomic coordinates or anchored to particular transcripts or genes.

*homepage*: https://github.com/art-egorov/svist4get

version  |toolchain
---------|--------------
``1.3.1``|``foss/2020b``

<a anchor="swarm"/>
### *swarm*

A robust and fast clustering method for amplicon-based studies 

*homepage*: https://github.com/torognes/swarm

version  |toolchain
---------|--------------
``2.2.2``|``foss/2018b``

<a anchor="swash"/>
### *SWASH*

SWASH is a general-purpose numerical tool for simulating unsteady, non-hydrostatic, free-surface,  rotational flow and transport phenomena in coastal waters as driven by waves, tides, buoyancy and wind forces.

*homepage*: http://swash.sourceforge.net/

version |versionsuffix|toolchain
--------|-------------|--------------------------------
``3.14``|``-mpi``     |``intel/2016b``, ``intel/2017a``
``4.01``|``-mpi``     |``intel/2017a``

<a anchor="swat+"/>
### *SWAT+*

The Soil & Water Assessment Tool (SWAT) is a small watershed to river basin-scale model used to simulate the quality and quantity of surface and ground water and predict the environmental impact of land use, land management practices, and climate change. In order to face present and future challenges in water resources modeling SWAT code has undergone major modifications over the past few years, resulting in SWAT+, a completely revised version of the model. SWAT+ provides a more flexible spatial representation of interactions and processes within a watershed.

*homepage*: https://swatplus.gitbook.io

version   |toolchain
----------|-----------------------
``60.4.1``|``GCC/9.3.0``
``60.5.1``|``iccifort/2020.4.304``

<a anchor="swifter"/>
### *swifter*

A package which efficiently applies any function to a pandas dataframe or series in the fastest available manner.

*homepage*: https://github.com/jmcarpenter2/swifter

version  |toolchain
---------|----------------------------------
``1.0.9``|``foss/2020b``, ``fosscuda/2020b``

<a anchor="swig"/>
### *SWIG*

SWIG is a software development tool that connects programs written in C and C++ with  a variety of high-level programming languages.

*homepage*: http://www.swig.org/

version   |versionsuffix               |toolchain
----------|----------------------------|------------------------------------------------------------------------------
``3.0.8`` |``-Python-2.7.11``          |``foss/2016a``, ``intel/2016a``
``3.0.8`` |``-Python-3.5.1``           |``foss/2016a``
``3.0.10``|``-Python-2.7.12``          |``foss/2016b``, ``intel/2016b``
``3.0.10``|``-Python-2.7.12-PCRE-8.39``|``intel/2016b``
``3.0.11``|``-Python-2.7.12``          |``foss/2016b``, ``intel/2016b``
``3.0.12``|                            |``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``3.0.12``|``-Python-2.7.13``          |``intel/2017a``
``3.0.12``|``-Python-2.7.14``          |``foss/2017b``, ``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``3.0.12``|``-Python-2.7.14-bare``     |``GCCcore/6.4.0``
``3.0.12``|``-Python-2.7.15``          |``GCCcore/8.2.0``, ``foss/2018b``, ``intel/2018b``
``3.0.12``|``-Python-3.6.1``           |``intel/2017a``
``3.0.12``|``-Python-3.6.2``           |``foss/2017b``, ``intel/2018.00``
``3.0.12``|``-Python-3.6.3``           |``foss/2017b``, ``intel/2017b``, ``intel/2018.01``
``3.0.12``|``-Python-3.6.4``           |``foss/2018a``, ``intel/2018a``, ``iomkl/2018a``
``3.0.12``|``-Python-3.6.6``           |``foss/2018b``, ``intel/2018b``
``3.0.12``|``-Python-3.7.2``           |``GCCcore/8.2.0``
``4.0.1`` |                            |``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``4.0.2`` |                            |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="swipe"/>
### *SWIPE*

Smith-Waterman database searches with inter-sequence SIMD parallelisation

*homepage*: https://github.com/torognes/swipe

version  |toolchain
---------|--------------
``2.1.1``|``GCC/10.3.0``

<a anchor="swissknife"/>
### *swissknife*

Perl module for reading and writing UniProtKB data in plain text format.

*homepage*: https://sourceforge.net/projects/swissknife/

version |toolchain
--------|-----------------
``1.80``|``GCCcore/8.3.0``

<a anchor="symengine"/>
### *SymEngine*

SymEngine is a standalone fast C++ symbolic manipulation library

*homepage*: https://github.com/symengine/symengine

version  |versionsuffix|toolchain
---------|-------------|--------------------
``0.3.0``|``-20181006``|``intel/2018a``
``0.4.0``|             |``GCC/8.2.0-2.31.1``
``0.7.0``|             |``GCC/10.3.0``

<a anchor="symengine-python"/>
### *SymEngine-python*

Python wrappers to the C++ library SymEngine, a fast C++ symbolic manipulation library.

*homepage*: https://github.com/symengine/symengine.py

version  |toolchain
---------|--------------
``0.7.2``|``GCC/10.3.0``

<a anchor="symphony"/>
### *SYMPHONY*

SYMPHONY is an open-source solver for mixed-integer linear programs (MILPs) written in C.

*homepage*: https://projects.coin-or.org/SYMPHONY

version   |toolchain
----------|--------------
``5.6.16``|``foss/2018b``

<a anchor="sympy"/>
### *sympy*

SymPy is a Python library for symbolic mathematics. It aims to  become a full-featured computer algebra system (CAS) while keeping the code as  simple as possible in order to be comprehensible and easily extensible. SymPy  is written entirely in Python and does not require any external libraries.

*homepage*: https://sympy.org/

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``1.0``   |``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``1.3``   |``-Python-2.7.14``|``intel/2018a``
``1.3``   |``-Python-2.7.15``|``foss/2018b``
``1.3``   |``-Python-3.6.4`` |``intel/2018a``
``1.3``   |``-Python-3.6.6`` |``foss/2018b``
``1.4``   |                  |``foss/2019a``, ``intel/2019a``
``1.5.1`` |``-Python-3.7.4`` |``foss/2019b``
``1.6.2`` |``-Python-3.8.2`` |``foss/2020a``
``1.7.1`` |                  |``foss/2020b``, ``intel/2020b``
``1.8``   |                  |``foss/2021a``
``1.9``   |                  |``foss/2021b``
``1.10.1``|                  |``foss/2022a``
``1.11.1``|                  |``foss/2022a``

<a anchor="syri"/>
### *SyRI*

Synteny and Rearrangement Identifier (SyRI).

*homepage*: https://schneebergerlab.github.io/syri/

version|toolchain
-------|--------------
``1.4``|``foss/2021a``

<a anchor="szip"/>
### *Szip*

Szip compression software, providing lossless compression of scientific data 

*homepage*: https://www.hdfgroup.org/doc_resource/SZIP/

version  |toolchain
---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.1``  |``GCC/4.8.1``, ``GCCcore/5.4.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``gimkl/2.11.5``, ``gimkl/2017a``, ``gmpolf/2017.10``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``intel/2017.01``, ``intel/2017a``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``
``2.1.1``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

<a anchor="t"/>
### *T*


<a href="#tabix">tabix</a> - <a href="#tabixpp">tabixpp</a> - <a href="#taco">taco</a> - <a href="#tagdust">TagDust</a> - <a href="#taglib">TagLib</a> - <a href="#taiyaki">Taiyaki</a> - <a href="#talon">TALON</a> - <a href="#talys">TALYS</a> - <a href="#tamkin">TAMkin</a> - <a href="#taxator-tk">taxator-tk</a> - <a href="#tba">TBA</a> - <a href="#tbb">tbb</a> - <a href="#tbl2asn">tbl2asn</a> - <a href="#tcc">TCC</a> - <a href="#tcl">Tcl</a> - <a href="#tclap">TCLAP</a> - <a href="#tcsh">tcsh</a> - <a href="#tecplot360ex">tecplot360ex</a> - <a href="#telemac-mascaret">TELEMAC-MASCARET</a> - <a href="#telescope">Telescope</a> - <a href="#teneto">Teneto</a> - <a href="#tensorboard">tensorboard</a> - <a href="#tensorboardx">tensorboardX</a> - <a href="#tensorflow">TensorFlow</a> - <a href="#tensorflow-datasets">TensorFlow-Datasets</a> - <a href="#tensorflow-probability">tensorflow-probability</a> - <a href="#tensorrt">TensorRT</a> - <a href="#terastructure">terastructure</a> - <a href="#termcolor">termcolor</a> - <a href="#tesla-deployment-kit">Tesla-Deployment-Kit</a> - <a href="#tesseract">tesseract</a> - <a href="#testpath">testpath</a> - <a href="#tetgen">TetGen</a> - <a href="#tetoolkit">TEToolkit</a> - <a href="#tetranscripts">TEtranscripts</a> - <a href="#texinfo">texinfo</a> - <a href="#texlive">texlive</a> - <a href="#text-csv">Text-CSV</a> - <a href="#tfea">TFEA</a> - <a href="#theano">Theano</a> - <a href="#themispy">ThemisPy</a> - <a href="#theta">THetA</a> - <a href="#thurstonianirt">thurstonianIRT</a> - <a href="#ticcutils">TiCCutils</a> - <a href="#tidybayes">tidybayes</a> - <a href="#tidymodels">tidymodels</a> - <a href="#tika">Tika</a> - <a href="#timbl">TiMBL</a> - <a href="#time">time</a> - <a href="#tinker">TINKER</a> - <a href="#tinydb">TinyDB</a> - <a href="#tinyxml">TinyXML</a> - <a href="#tk">Tk</a> - <a href="#tkinter">Tkinter</a> - <a href="#tm-align">TM-align</a> - <a href="#tmae">tMAE</a> - <a href="#tmap">tmap</a> - <a href="#tmux">tmux</a> - <a href="#tn93">TN93</a> - <a href="#tobias">TOBIAS</a> - <a href="#tofu">ToFu</a> - <a href="#togl">Togl</a> - <a href="#tokenizers">tokenizers</a> - <a href="#tombo">Tombo</a> - <a href="#toml-fortran">TOML-Fortran</a> - <a href="#topaz">topaz</a> - <a href="#tophat">TopHat</a> - <a href="#torchaudio">torchaudio</a> - <a href="#torchdata">torchdata</a> - <a href="#torchinfo">torchinfo</a> - <a href="#torchsampler">torchsampler</a> - <a href="#torchtext">torchtext</a> - <a href="#torchvision">torchvision</a> - <a href="#totalview">TotalView</a> - <a href="#tqdm">tqdm</a> - <a href="#tracer">Tracer</a> - <a href="#transcriptclean">TranscriptClean</a> - <a href="#transdecoder">TransDecoder</a> - <a href="#transformers">Transformers</a> - <a href="#transrate">Transrate</a> - <a href="#travis">travis</a> - <a href="#travis-analyzer">TRAVIS-Analyzer</a> - <a href="#treatsens">treatSens</a> - <a href="#treemix">TreeMix</a> - <a href="#treeshrink">TreeShrink</a> - <a href="#trf">TRF</a> - <a href="#triangle">Triangle</a> - <a href="#trilinos">Trilinos</a> - <a href="#trim_galore">Trim_Galore</a> - <a href="#trimal">trimAl</a> - <a href="#trimesh">trimesh</a> - <a href="#trimmomatic">Trimmomatic</a> - <a href="#trinity">Trinity</a> - <a href="#triqs">TRIQS</a> - <a href="#triqs-cthyb">TRIQS-cthyb</a> - <a href="#triqs-dft_tools">TRIQS-dft_tools</a> - <a href="#triqs-tprf">TRIQS-tprf</a> - <a href="#trust">TRUST</a> - <a href="#trust4">TRUST4</a> - <a href="#trycycler">Trycycler</a> - <a href="#tsne">tsne</a> - <a href="#turbovnc">TurboVNC</a> - <a href="#tvb">TVB</a> - <a href="#tvb-data">tvb-data</a> - <a href="#tvb-deps">TVB-deps</a> - <a href="#tvb-framework">tvb-framework</a> - <a href="#tvb-library">tvb-library</a> - <a href="#twl-ninja">TWL-NINJA</a> - <a href="#typing-extensions">typing-extensions</a>


<a anchor="tabix"/>
### *tabix*

Generic indexer for TAB-delimited genome position files 

*homepage*: http://samtools.sourceforge.net

version  |toolchain
---------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------
``0.2.6``|``GCCcore/10.2.0``, ``GCCcore/11.2.0``, ``GCCcore/5.4.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.3.0``, ``foss/2016a``, ``intel/2016a``, ``intel/2016b``

<a anchor="tabixpp"/>
### *tabixpp*

C++ wrapper to tabix indexer

*homepage*: https://github.com/ekg/tabixpp

version  |toolchain
---------|-------------------------------------------------------------
``1.1.0``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/9.3.0``

<a anchor="taco"/>
### *taco*

Multi-sample transcriptome assembly from RNA-Seq

*homepage*: http://tacorna.github.io

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.5.1``|``-Python-2.7.12``|``intel/2016b``

<a anchor="tagdust"/>
### *TagDust*

Raw sequences produced by next generation sequencing (NGS) machines may contain adapter,  linker, barcode and fingerprint sequences. TagDust2 is a program to extract and correctly label the  sequences to be mapped in downstream pipelines.

*homepage*: http://tagdust.sourceforge.net/

version |toolchain
--------|-------------------------------------
``2.33``|``GCCcore/10.2.0``, ``GCCcore/8.3.0``

<a anchor="taglib"/>
### *TagLib*

TagLib is a library for reading and editing the meta-data of several popular audio formats.

*homepage*: https://taglib.org/

version   |toolchain
----------|-----------------
``1.11.1``|``GCCcore/8.2.0``

<a anchor="taiyaki"/>
### *Taiyaki*

Taiyaki is research software for training models for basecalling Oxford Nanopore reads.

*homepage*: https://github.com/nanoporetech/taiyaki

version           |versionsuffix                  |toolchain
------------------|-------------------------------|----------------------------------
``5.1.0-20200617``|``-Python-3.7.2-PyTorch-1.2.0``|``foss/2019a``, ``fosscuda/2019a``

<a anchor="talon"/>
### *TALON*

TALON is a Python package for identifying and quantifying known and novel genes/isoforms in long-read transcriptome data sets. TALON is technology-agnostic in that it works from mapped SAM files, allowing data from different sequencing platforms (i.e. PacBio and Oxford Nanopore) to be analyzed side by side.

*homepage*: https://github.com/mortazavilab/TALON

version|toolchain
-------|--------------
``5.0``|``foss/2020b``

<a anchor="talys"/>
### *TALYS*

TALYS is a nuclear reaction program.

*homepage*: https://tendl.web.psi.ch/tendl_2019/talys.html

version |toolchain
--------|-------------------------------------
``1.95``|``GCCcore/10.3.0``, ``GCCcore/9.3.0``

<a anchor="tamkin"/>
### *TAMkin*

TAMkin is a post-processing toolkit for normal mode analysis,  thermochemistry and reaction kinetics. It uses a Hessian computation from a  standard computational chemistry program as its input.

*homepage*: https://molmod.github.io/tamkin/

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.0.9``|``-Python-2.7.11``|``intel/2016a``
``1.2.4``|``-Python-2.7.14``|``intel/2017b``
``1.2.6``|``-Python-3.7.2`` |``intel/2019a``
``1.2.6``|``-Python-3.8.2`` |``intel/2020a``

<a anchor="taxator-tk"/>
### *taxator-tk*

A set of programs for the taxonomic analysis of nucleotide sequence data

*homepage*: https://github.com/fungs/taxator-tk

version  |toolchain
---------|-----------------------------------------------
``1.3.3``|``GCC/10.2.0``, ``foss/2018b``, ``gompi/2019a``

<a anchor="tba"/>
### *TBA*

TBA (a Transcription factor Binding Analysis): TBA is a multi-functional machine learning tool for identifying  transcription factors associated with genomic features

*homepage*: https://github.com/jenhantao/tba

version|toolchain
-------|--------------
``1.0``|``foss/2020b``

<a anchor="tbb"/>
### *tbb*

Intel Threading Building Blocks (Intel TBB)  is a widely used, award-winning C++ template library for creating reliable,  portable, and scalable parallel applications.  Use Intel TBB for a simple and rapid way of developing robust task-based  parallel applications that scale to available processor cores, are compatible  with multiple environments, and are easier to maintain.  Intel TBB is the most proficient way to implement future-proof parallel applications  that tap into the power and performance of multicore and manycore hardware platforms.

*homepage*: https://software.intel.com/en-us/articles/intel-tbb/

version       |toolchain
--------------|----------------------------------------------------------
``4.0.0.233`` |``system``
``4.0.5.339`` |``system``
``4.3.6.211`` |``system``
``4.4.2.152`` |``system``
``2017.2.132``|``system``
``2017.4.174``|``system``
``2017.6.196``|``system``
``2017_U5``   |``GCCcore/5.4.0``, ``foss/2016b``, ``intel/2017a``
``2017_U6``   |``GCCcore/6.3.0``, ``intel/2017a``
``2018_U1``   |``GCCcore/6.4.0``
``2018_U2``   |``GCCcore/6.4.0``
``2018_U3``   |``GCCcore/6.4.0``
``2018_U5``   |``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``2019_U4``   |``GCCcore/8.2.0``
``2019_U9``   |``GCCcore/8.3.0``
``2020.1``    |``GCCcore/9.3.0``
``2020.2``    |``GCCcore/8.3.0``
``2020.3``    |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``2021.4.0``  |``GCCcore/11.2.0``
``2021.5.0``  |``GCCcore/11.3.0``

<a anchor="tbl2asn"/>
### *tbl2asn*

Tbl2asn is a command-line program that automates the creation of  sequence records for submission to GenBank

*homepage*: https://www.ncbi.nlm.nih.gov/genbank/tbl2asn2/

version     |versionsuffix|toolchain
------------|-------------|----------
``25.8``    |``-linux64`` |``system``
``20180227``|``-linux64`` |``system``
``20200302``|``-linux64`` |``system``
``20220427``|``-linux64`` |``system``

<a anchor="tcc"/>
### *TCC*

The Tiny C Compiler (aka TCC, tCc, or TinyCC) is an x86 and x86-64 C compiler  created by Fabrice Bellard. It is designed to work for slow computers with little disk space  and can run shebang style !/usr/bin/tcc . TCC is distributed under the LGPL. TCC claims to  implement all of ANSI C (C89/C90),[1] much of the new ISO C99 standard,  and many GNU C extensions including inline assembly.

*homepage*: http://bellard.org/tcc/

version   |toolchain
----------|----------
``0.9.26``|``system``

<a anchor="tcl"/>
### *Tcl*

Tcl (Tool Command Language) is a very powerful but easy to learn dynamic  programming language, suitable for a very wide range of uses, including web  and desktop applications, networking, administration, testing and many more. 

*homepage*: https://www.tcl.tk/

version   |toolchain
----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``8.5.19``|``foss/2017a``
``8.6.3`` |``GCC/4.8.4``, ``GCC/4.9.2``
``8.6.4`` |``GCC/4.9.3-2.25``, ``GNU/4.9.3-2.25``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``
``8.6.5`` |``GCC/4.9.3-2.25``, ``GCC/5.4.0-2.26``, ``GCCcore/6.3.0``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``, ``intel/2016b``
``8.6.6`` |``GCCcore/4.9.3``, ``GCCcore/6.3.0``
``8.6.7`` |``GCCcore/6.4.0``
``8.6.8`` |``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``
``8.6.9`` |``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``8.6.10``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``8.6.11``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``8.6.12``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``

<a anchor="tclap"/>
### *TCLAP*

TCLAP is a small, flexible library that provides a simple interface for defining and accessing command line arguments. It was intially inspired by the user friendly CLAP libary.

*homepage*: http://tclap.sourceforge.net/

version  |toolchain
---------|--------------------------------------
``1.2.2``|``GCCcore/8.3.0``
``1.2.4``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``1.2.5``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``

<a anchor="tcsh"/>
### *tcsh*

Tcsh is an enhanced, but completely compatible version of the Berkeley UNIX C shell (csh).  It is a command language interpreter usable both as an interactive login shell and a shell script command  processor. It includes a command-line editor, programmable word completion, spelling correction, a history  mechanism, job control and a C-like syntax.

*homepage*: https://www.tcsh.org

version    |toolchain
-----------|-------------------------------------------------------
``6.19.00``|``intel/2016a``
``6.20.00``|``GCCcore/5.4.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``6.22.02``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``6.22.03``|``GCCcore/10.2.0``
``6.22.04``|``GCCcore/10.3.0``
``6.24.01``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="tecplot360ex"/>
### *tecplot360ex*

Quickly plot and animate your CFD results exactly the way you want. Analyze complex solutions, arrange multiple layouts, and communicate your results with professional images and animations.

*homepage*: http://www.tecplot.com/products/tecplot-360/

version    |toolchain
-----------|----------
``linux64``|``system``

<a anchor="telemac-mascaret"/>
### *TELEMAC-MASCARET*

TELEMAC-MASCARET is an integrated suite of solvers for use in the field of free-surface flow. Having been used in the context of many studies throughout the world, it has become one of the major standards in its field.

*homepage*: http://www.opentelemac.org

version  |toolchain
---------|--------------
``8p3r1``|``foss/2021b``

<a anchor="telescope"/>
### *Telescope*

Single locus resolution of Transposable ELEment expression using next-generation sequencing.

*homepage*: https://github.com/mlbendall/telescope

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.0.3``|``-Python-3.7.4``|``foss/2019b``

<a anchor="teneto"/>
### *Teneto*

Teneto is package for deriving, analysing and plotting temporal network representations. Additional tools for temporal network analysis with neuroimaging contexts.

*homepage*: https://teneto.readthedocs.io

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.5.1``|``-Python-3.8.2``|``foss/2020a``

<a anchor="tensorboard"/>
### *tensorboard*

TensorBoard is a suite of web applications for inspecting and understanding your TensorFlow runs and graphs.

*homepage*: https://github.com/tensorflow/tensorboard

version   |toolchain
----------|--------------
``2.8.0`` |``foss/2021a``
``2.10.0``|``foss/2022a``

<a anchor="tensorboardx"/>
### *tensorboardX*

Tensorboard for PyTorch.

*homepage*: https://github.com/lanpa/tensorboardX

version  |versionsuffix     |toolchain
---------|------------------|----------------------------------
``2.0``  |``-Python-3.7.4`` |``foss/2019b``
``2.1``  |``-PyTorch-1.7.1``|``fosscuda/2020b``
``2.2``  |``-PyTorch-1.7.1``|``foss/2020b``, ``fosscuda/2020b``
``2.5.1``|                  |``foss/2022a``

<a anchor="tensorflow"/>
### *TensorFlow*

An open-source software library for Machine Intelligence

*homepage*: https://www.tensorflow.org/

version   |versionsuffix                |toolchain
----------|-----------------------------|---------------------------------------------------
``0.12.1``|``-Python-2.7.12``           |``foss/2016b``, ``intel/2016b``
``0.12.1``|``-Python-3.5.2``            |``foss/2016b``, ``intel/2016b``
``1.0.1`` |``-Python-2.7.12``           |``intel/2016b``
``1.0.1`` |``-Python-3.5.2``            |``intel/2016b``
``1.1.0`` |``-Python-2.7.13``           |``intel/2017a``
``1.1.0`` |``-Python-3.6.1``            |``intel/2017a``
``1.2.0`` |``-Python-3.6.1``            |``intel/2017a``
``1.2.1`` |``-GPU-Python-3.5.2``        |``foss/2016b``
``1.2.1`` |``-Python-3.5.2``            |``foss/2016b``
``1.3.0`` |``-Python-2.7.13``           |``intel/2017a``
``1.3.0`` |``-Python-3.6.1``            |``intel/2017a``
``1.3.0`` |``-Python-3.6.3``            |``intel/2017b``
``1.4.0`` |``-Python-3.6.3``            |``foss/2017b``, ``intel/2017b``
``1.4.1`` |``-Python-3.6.3``            |``foss/2017b``
``1.5.0`` |``-Python-3.6.3``            |``foss/2017b``, ``intel/2017b``
``1.6.0`` |``-Python-3.6.4``            |``foss/2018a``, ``intel/2018a``
``1.6.0`` |``-Python-3.6.4-CUDA-9.1.85``|``foss/2018a``
``1.7.0`` |``-Python-3.6.4``            |``foss/2018a``
``1.7.0`` |``-Python-3.6.4-CUDA-9.1.85``|``foss/2018a``
``1.8.0`` |``-Python-2.7.14``           |``foss/2017b``, ``fosscuda/2017b``
``1.8.0`` |``-Python-3.6.3``            |``foss/2017b``, ``fosscuda/2017b``
``1.8.0`` |``-Python-3.6.4``            |``foss/2018a``, ``fosscuda/2018a``, ``intel/2018a``
``1.10.0``|``-Python-2.7.15``           |``fosscuda/2018b``
``1.10.1``|``-Python-2.7.15``           |``fosscuda/2018b``
``1.10.1``|``-Python-3.6.6``            |``foss/2018b``
``1.11.0``|``-Python-3.6.6``            |``foss/2018b``
``1.12.0``|``-Python-2.7.15``           |``fosscuda/2018b``
``1.12.0``|``-Python-3.6.6``            |``foss/2018b``, ``fosscuda/2018b``
``1.13.1``|``-Python-3.6.6``            |``foss/2018b``
``1.13.1``|``-Python-3.7.2``            |``foss/2019a``, ``fosscuda/2019a``
``1.14.0``|``-Python-3.7.2``            |``foss/2019a``, ``fosscuda/2019a``
``1.15.0``|``-Python-3.7.4``            |``foss/2019b``, ``fosscuda/2019b``
``1.15.2``|``-Python-3.7.4``            |``foss/2019b``, ``fosscuda/2019b``
``1.15.5``|``-Python-3.7.4``            |``fosscuda/2019b``
``1.15.5``|``-Python-3.7.4-nompi``      |``fosscuda/2019b``
``2.0.0`` |``-Python-3.7.2``            |``foss/2019a``
``2.0.0`` |``-Python-3.7.4``            |``fosscuda/2019b``
``2.0.1`` |``-Python-3.7.2``            |``foss/2019a``
``2.0.1`` |``-Python-3.7.4``            |``fosscuda/2019b``
``2.1.0`` |``-Python-3.7.4``            |``foss/2019b``, ``fosscuda/2019b``
``2.2.0`` |``-Python-3.7.4``            |``foss/2019b``, ``fosscuda/2019b``
``2.2.2`` |``-Python-3.7.4``            |``foss/2019b``
``2.2.3`` |                             |``foss/2020b``
``2.3.1`` |``-Python-3.7.4``            |``foss/2019b``, ``fosscuda/2019b``
``2.3.1`` |``-Python-3.8.2``            |``foss/2020a``, ``fosscuda/2020a``
``2.4.1`` |                             |``foss/2020b``, ``fosscuda/2020b``
``2.4.1`` |``-Python-3.7.4``            |``fosscuda/2019b``
``2.4.4`` |                             |``foss/2021a``
``2.5.0`` |                             |``foss/2020b``, ``fosscuda/2020b``
``2.5.0`` |``-Python-3.7.4``            |``fosscuda/2019b``
``2.5.3`` |                             |``foss/2021a``
``2.5.3`` |``-CUDA-11.3.1``             |``foss/2021a``
``2.6.0`` |                             |``foss/2021a``
``2.6.0`` |``-CUDA-11.3.1``             |``foss/2021a``
``2.7.1`` |                             |``foss/2021b``
``2.7.1`` |``-CUDA-11.4.1``             |``foss/2021b``
``2.8.4`` |                             |``foss/2021b``

<a anchor="tensorflow-datasets"/>
### *TensorFlow-Datasets*

TensorFlow Datasets is a collection of datasets ready to use, with TensorFlow or other Python ML frameworks, such as Jax. All datasets are exposed as tf.data.Datasets , enabling easy-to-use and high-performance input pipelines. 

*homepage*: https://www.tensorflow.org/datasets

version  |versionsuffix   |toolchain
---------|----------------|--------------
``4.7.0``|``-CUDA-11.3.1``|``foss/2021a``

<a anchor="tensorflow-probability"/>
### *tensorflow-probability*

TensorFlow Probability (TFP) is a library for probabilistic reasoning and statistical analysis.

*homepage*: https://www.tensorflow.org/probability

version   |versionsuffix    |toolchain
----------|-----------------|----------------------------------
``0.9.0`` |``-Python-3.7.4``|``foss/2019b``
``0.12.1``|                 |``foss/2020b``, ``fosscuda/2020b``
``0.16.0``|                 |``foss/2021b``

<a anchor="tensorrt"/>
### *TensorRT*

NVIDIA TensorRT is a platform for high-performance deep learning inference

*homepage*: https://developer.nvidia.com/tensorrt

version    |versionsuffix     |toolchain
-----------|------------------|------------------
``4.0.1.6``|``-Python-2.7.15``|``fosscuda/2018b``

<a anchor="terastructure"/>
### *terastructure*

TeraStructure is a new algorithm to fit Bayesian models of genetic variation in human  populations on tera-sample-sized data sets (10^12 observed genotypes, i.e., 1M individuals at 1M SNPs).  This package provides a scalable, multi-threaded C++ implementation that can be run on a single computer. 

*homepage*: https://github.com/StoreyLab/terastructure

version|toolchain
-------|-------------
``1.0``|``GCC/8.3.0``

<a anchor="termcolor"/>
### *termcolor*

Termcolor is a header-only C++ library for printing colored  messages to the terminal.

*homepage*: https://termcolor.readthedocs.io/

version  |toolchain
---------|----------
``2.0.0``|``system``

<a anchor="tesla-deployment-kit"/>
### *Tesla-Deployment-Kit*

The Tesla Deployment Kit is a set of tools provided primarily for the NVIDIA Tesla  range of GPUs. They aim to empower users to better manage their NVIDIA GPUs by providing a broad  range of functionalities. The kit contains:  * NVIDIA Management Library (NVML),  * Tesla Deployment Kit - Linux Edition (Aug 1st, 2013)

*homepage*: https://developer.nvidia.com/tesla-deployment-kit

version     |toolchain
------------|----------
``5.319.43``|``system``

<a anchor="tesseract"/>
### *tesseract*

Tesseract is an optical character recognition engine

*homepage*: https://github.com/tesseract-ocr/tesseract

version  |toolchain
---------|-------------------------------------
``4.0.0``|``GCCcore/7.3.0``
``4.1.0``|``GCCcore/10.3.0``, ``GCCcore/8.2.0``
``5.3.0``|``GCCcore/11.3.0``

<a anchor="testpath"/>
### *testpath*

Test utilities for code working with files and commands

*homepage*: https://github.com/jupyter/testpath

version|versionsuffix     |toolchain
-------|------------------|-------------------------------
``0.3``|``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``0.3``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``0.3``|``-Python-3.5.1`` |``foss/2016a``
``0.3``|``-Python-3.5.2`` |``intel/2016b``

<a anchor="tetgen"/>
### *TetGen*

A Quality Tetrahedral Mesh Generator and a 3D Delaunay Triangulator 

*homepage*: http://wias-berlin.de/software/index.jsp?id=TetGen

version  |toolchain
---------|------------------
``1.5.0``|``GCCcore/6.4.0``
``1.6.0``|``GCCcore/10.2.0``

<a anchor="tetoolkit"/>
### *TEToolkit*

Tools for estimating differential enrichment of Transposable  Elements and other highly repetitive regions

*homepage*: http://hammelllab.labsites.cshl.edu/software/#TEToolkit

version  |versionsuffix     |toolchain
---------|------------------|--------------
``1.5.1``|``-Python-2.7.11``|``foss/2016a``

<a anchor="tetranscripts"/>
### *TEtranscripts*

TEtranscripts and TEcount takes RNA-seq (and similar data) and annotates reads to both genes & transposable elements. TEtranscripts then performs differential analysis using DESeq2.

*homepage*: https://github.com/mhammell-laboratory/TEtranscripts

version  |toolchain
---------|--------------
``2.2.0``|``foss/2020a``

<a anchor="texinfo"/>
### *texinfo*

Texinfo is the official documentation format of the GNU project.

*homepage*: https://www.gnu.org/software/texinfo/

version  |toolchain
---------|--------------------------------------------------------
``4.13a``|``system``
``5.2``  |``GCC/4.8.2``
``6.4``  |``GCCcore/5.4.0``
``6.5``  |``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``6.6``  |``GCCcore/8.2.0``
``6.7``  |``GCCcore/10.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``6.8``  |``GCCcore/11.2.0``

<a anchor="texlive"/>
### *texlive*

TeX is a typesetting language. Instead of visually formatting your text, you enter your manuscript  text intertwined with TeX commands in a plain text file. You then run TeX to produce formatted output, such as a  PDF file. Thus, in contrast to standard word processors, your document is a separate file that does not pretend to  be a representation of the final typeset output, and so can be easily edited and manipulated.

*homepage*: https://tug.org

version     |toolchain
------------|------------------------------
``20200406``|``GCCcore/8.3.0``
``20210324``|``GCC/10.3.0``, ``GCC/11.2.0``
``20220321``|``GCC/11.2.0``

<a anchor="text-csv"/>
### *Text-CSV*

Text-CSV parser

*homepage*: https://metacpan.org/pod/Text::CSV

version |versionsuffix   |toolchain
--------|----------------|--------------
``1.33``|``-Perl-5.22.1``|``foss/2016a``

<a anchor="tfea"/>
### *TFEA*

Transcription Factor Enrichment Analysis 

*homepage*: https://github.com/Dowell-Lab/TFEA

version  |versionsuffix     |toolchain
---------|------------------|--------------
``1.1.4``|``-muMerge-1.1.0``|``foss/2020b``

<a anchor="theano"/>
### *Theano*

Theano is a Python library that allows you to define, optimize, and evaluate mathematical expressions involving multi-dimensional arrays efficiently.

*homepage*: https://deeplearning.net/software/theano

version  |versionsuffix     |toolchain
---------|------------------|------------------------------------------------------------------------------------
``0.8.2``|``-Python-2.7.11``|``foss/2016a``
``0.8.2``|``-Python-3.5.1`` |``foss/2016a``
``0.8.2``|``-Python-3.5.2`` |``intel/2016b``
``0.9.0``|``-Python-2.7.13``|``intel/2017a``
``0.9.0``|``-Python-3.6.1`` |``intel/2017a``
``1.0.0``|``-Python-2.7.14``|``intel/2017b``
``1.0.0``|``-Python-3.6.3`` |``intel/2017b``
``1.0.1``|``-Python-3.6.3`` |``foss/2017b``, ``intel/2017b``
``1.0.2``|``-Python-2.7.14``|``fosscuda/2017b``, ``intelcuda/2017b``
``1.0.2``|``-Python-2.7.15``|``fosscuda/2018b``
``1.0.2``|``-Python-3.6.3`` |``fosscuda/2017b``, ``intelcuda/2017b``
``1.0.2``|``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``1.0.3``|``-Python-3.6.6`` |``foss/2018b``, ``fosscuda/2018b``
``1.0.4``|                  |``foss/2019a``, ``fosscuda/2019a``
``1.0.4``|``-Python-3.6.4`` |``foss/2018a``
``1.0.4``|``-Python-3.6.6`` |``foss/2018b``, ``fosscuda/2018b``
``1.0.4``|``-Python-3.7.4`` |``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``
``1.0.4``|``-Python-3.8.2`` |``foss/2020a``
``1.1.2``|``-PyMC``         |``foss/2020b``, ``foss/2021b``, ``fosscuda/2020b``, ``intel/2020b``, ``intel/2021b``

<a anchor="themispy"/>
### *ThemisPy*

A header-only C++ library for L-BFGS and L-BFGS-B algorithms

*homepage*: https://github.com/aeb/ThemisPy

version  |toolchain
---------|--------------
``0.3.0``|``foss/2021a``

<a anchor="theta"/>
### *THetA*

Tumor Heterogeneity Analysis (THetA) and THetA2 are algorithms  that estimate the tumor purity and clonal/subclonal copy number aberrations  directly from high-throughput DNA sequencing data.

*homepage*: https://github.com/raphael-group/THetA

version|versionsuffix     |toolchain
-------|------------------|--------------
``0.7``|``-Python-2.7.15``|``foss/2018b``

<a anchor="thurstonianirt"/>
### *thurstonianIRT*

Fit Thurstonian IRT models in R using Stan, lavaan, or Mplus

*homepage*: https://github.com/paul-buerkner/thurstonianIRT

version  |versionsuffix|toolchain
---------|-------------|--------------
``0.9.0``|``-R-3.6.0`` |``foss/2019a``

<a anchor="ticcutils"/>
### *TiCCutils*

TiCC utils is a collection of generic C++ software which is used in a lot of programs produced at  Tilburg centre for Cognition and Communication (TiCC) at Tilburg University and  Centre for Dutch Language and Speech at University of Antwerp.

*homepage*: https://github.com/LanguageMachines/ticcutils

version |toolchain
--------|--------------------------------
``0.11``|``foss/2016a``
``0.21``|``iimpi/2019a``, ``intel/2018b``

<a anchor="tidybayes"/>
### *tidybayes*

Compose data for and extract, manipulate, and visualize posterior draws from Bayesian models ('JAGS', 'Stan', 'rstanarm', 'brms', 'MCMCglmm', 'coda', ...) in a tidy data format. 

*homepage*: https://cran.r-project.org/package=tidybayes

version  |versionsuffix|toolchain
---------|-------------|--------------
``2.1.1``|``-R-4.0.0`` |``foss/2020a``

<a anchor="tidymodels"/>
### *tidymodels*

The tidy modeling "verse" is a collection of packages for modeling and statistical analysis that share the underlying design philosophy, grammar, and data structures of the tidyverse.

*homepage*: https://cran.r-project.org/package=tidymodels

version  |versionsuffix|toolchain
---------|-------------|--------------
``0.1.0``|``-R-4.0.0`` |``foss/2020a``

<a anchor="tika"/>
### *Tika*

The Apache Tika toolkit detects and extracts metadata and text from over a thousand different  file types (such as PPT, XLS, and PDF).

*homepage*: https://tika.apache.org/

version |toolchain
--------|----------
``1.16``|``system``

<a anchor="timbl"/>
### *TiMBL*

TiMBL (Tilburg Memory Based Learner)  is an open source software package implementing several memory-based learning algorithms,  among which IB1-IG, an implementation of k-nearest neighbor classification with feature weighting suitable for  symbolic feature spaces, and IGTree, a decision-tree approximation of IB1-IG. All implemented algorithms have in  common that they store some representation of the training set explicitly in memory. During testing, new cases are  classified by extrapolation from the most similar stored cases.

*homepage*: http://ilk.uvt.nl/timbl/

version   |toolchain
----------|--------------------------------
``6.4.7`` |``foss/2016a``
``6.4.13``|``iimpi/2019a``, ``intel/2018b``

<a anchor="time"/>
### *time*

The `time' command runs another program, then displays information about the resources used by that  program, collected by the system while the program was running.

*homepage*: https://www.gnu.org/software/time/

version|toolchain
-------|---------------------------------------------------------------------------------------------------------------------------------------
``1.7``|``system``
``1.9``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/7.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

<a anchor="tinker"/>
### *TINKER*

The TINKER molecular modeling software is a complete and general package for molecular mechanics  and dynamics, with some special features for biopolymers.

*homepage*: https://dasher.wustl.edu/tinker

version  |toolchain
---------|--------------
``8.6.1``|``foss/2018b``
``8.7.2``|``foss/2019b``
``8.8.1``|``foss/2020a``

<a anchor="tinydb"/>
### *TinyDB*

TinyDB is a lightweight document oriented database optimized for your happiness :) It's written in  pure Python and has no external dependencies. The target are small apps that would be blown away by a SQL-DB or an  external database server.

*homepage*: https://tinydb.readthedocs.io/

version   |toolchain
----------|-----------------
``3.15.2``|``GCCcore/8.3.0``

<a anchor="tinyxml"/>
### *TinyXML*

TinyXML is a simple, small, minimal, C++ XML parser that can be easily integrating  into other programs. It reads XML and creates C++ objects representing the XML document. The  objects can be manipulated, changed, and saved again as XML. 

*homepage*: https://sourceforge.net/projects/tinyxml

version  |toolchain
---------|--------------------------------------
``2.6.2``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="tk"/>
### *Tk*

Tk is an open source, cross-platform widget toolchain that provides a library of basic elements for  building a graphical user interface (GUI) in many different programming languages.

*homepage*: https://www.tcl.tk/

version   |versionsuffix    |toolchain
----------|-----------------|------------------------------------------------------------------------------------------------------------------------------------------------
``8.6.3`` |``-no-X11``      |``GCC/4.8.4``, ``GCC/4.9.2``
``8.6.4`` |                 |``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``
``8.6.4`` |``-libX11-1.6.3``|``intel/2016a``
``8.6.4`` |``-no-X11``      |``GCC/4.9.3-2.25``, ``GNU/4.9.3-2.25``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``
``8.6.5`` |                 |``GCC/5.4.0-2.26``, ``GCCcore/6.3.0``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``, ``intel/2016b``
``8.6.6`` |                 |``foss/2017a``, ``intel/2017a``
``8.6.7`` |                 |``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``8.6.8`` |                 |``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``foss/2018a``, ``iomkl/2018a``
``8.6.9`` |                 |``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``8.6.10``|                 |``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``8.6.11``|                 |``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``8.6.12``|                 |``GCCcore/11.3.0``

<a anchor="tkinter"/>
### *Tkinter*

Tkinter module, built with the Python buildsystem

*homepage*: https://python.org/

version   |versionsuffix     |toolchain
----------|------------------|---------------------------------------------------------------------------------------------------------
``2.7.13``|``-Python-2.7.13``|``foss/2017a``, ``intel/2017a``
``2.7.14``|``-Python-2.7.14``|``foss/2017b``, ``foss/2018a``, ``fosscuda/2017b``, ``intel/2017b``, ``intel/2018a``, ``intelcuda/2017b``
``2.7.15``|                  |``GCCcore/8.2.0``
``2.7.15``|``-Python-2.7.15``|``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``2.7.16``|                  |``GCCcore/8.3.0``
``2.7.18``|                  |``GCCcore/10.2.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``
``3.6.1`` |``-Python-3.6.1`` |``foss/2017a``, ``intel/2017a``
``3.6.2`` |``-Python-3.6.2`` |``foss/2017b``
``3.6.3`` |``-Python-3.6.3`` |``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``3.6.4`` |``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``, ``iomkl/2018.02``, ``iomkl/2018a``
``3.6.6`` |``-Python-3.6.6`` |``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``, ``iomkl/2018b``
``3.7.2`` |                  |``GCCcore/8.2.0``
``3.7.4`` |                  |``GCCcore/8.3.0``
``3.8.2`` |                  |``GCCcore/9.3.0``
``3.8.6`` |                  |``GCCcore/10.2.0``
``3.9.5`` |                  |``GCCcore/10.3.0``
``3.9.6`` |                  |``GCCcore/11.2.0``
``3.10.4``|                  |``GCCcore/11.3.0``

<a anchor="tm-align"/>
### *TM-align*

This package unifies protein structure alignment and RNA structure alignment  into the standard TM-align program for single chain structure alignment, MM-align program for  multi-chain structure alignment, and TM-score program for sequence dependent structure superposition.

*homepage*: https://zhanglab.ccmb.med.umich.edu/RNA-align/TMalign/

version     |toolchain
------------|-------------------------------
``20180426``|``foss/2018b``, ``intel/2019a``
``20190822``|``GCC/11.3.0``

<a anchor="tmae"/>
### *tMAE*

Package containing functions to:      perform a negative binomial test on allele-specific counts     add gnomAD minor allele frequencies     MAplot (FC vs total counts) of allele-specific counts and results     allelic counts (ALT vs REF)

*homepage*: https://github.com/mumichae/tMAE

version  |versionsuffix|toolchain
---------|-------------|--------------
``1.0.0``|``-R-4.0.3`` |``foss/2020b``
``1.0.1``|``-R-4.1.2`` |``foss/2021b``

<a anchor="tmap"/>
### *tmap*

tmap is a very fast visualization library for large, high-dimensional data sets. Currently, tmap is available for Python. tmap's graph layouts are based on the OGDF library.

*homepage*: https://tmap.gdb.tools

version     |toolchain
------------|--------------
``20220502``|``GCC/11.2.0``

<a anchor="tmux"/>
### *tmux*

tmux is a terminal multiplexer: it enables a number of terminals to be created, accessed, and controlled from a single screen. tmux may be detached from a screen and continue running in the background, then later reattached.

*homepage*: https://github.com/tmux/tmux/

version |toolchain
--------|--------------------------------------------------
``1.9a``|``GCC/4.9.2``
``2.2`` |``GCCcore/4.9.3``
``2.3`` |``GCC/5.4.0-2.26``, ``system``
``3.1c``|``GCCcore/8.3.0``
``3.2`` |``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``3.2a``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``system``
``3.3a``|``GCCcore/11.3.0``

<a anchor="tn93"/>
### *TN93*

This is a simple program meant to compute pairwise distances between  aligned nucleotide sequences in sequential FASTA format using the Tamura Nei 93 distance.

*homepage*: https://github.com/veg/tn93

version  |toolchain
---------|------------------
``1.0.7``|``GCCcore/10.3.0``

<a anchor="tobias"/>
### *TOBIAS*

TOBIAS is a collection of command-line bioinformatics tools  for performing footprinting analysis on ATAC-seq data.

*homepage*: https://github.com/loosolab/TOBIAS

version    |toolchain
-----------|--------------
``0.12.12``|``foss/2021b``
``0.14.0`` |``foss/2020b``

<a anchor="tofu"/>
### *ToFu*

Tomography for Fusion.

*homepage*: https://github.com/ToFuProject/tofu

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``1.3.17``|``-Python-2.7.14``|``foss/2018a``
``1.3.17``|``-Python-3.6.4`` |``foss/2018a``
``1.4.0`` |``-Python-2.7.14``|``foss/2018a``, ``intel/2018a``
``1.4.0`` |``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``1.4.1`` |``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``1.5.0`` |                  |``foss/2020b``, ``intel/2020b``

<a anchor="togl"/>
### *Togl*

A Tcl/Tk widget for OpenGL rendering.

*homepage*: https://sourceforge.net/projects/togl/

version|toolchain
-------|---------------------------------------------------------
``2.0``|``GCCcore/10.2.0``, ``GCCcore/11.3.0``, ``GCCcore/8.3.0``

<a anchor="tokenizers"/>
### *tokenizers*

Fast State-of-the-Art Tokenizers optimized for Research and Production

*homepage*: https://github.com/huggingface/tokenizers

version   |toolchain
----------|------------------
``0.12.1``|``GCCcore/10.3.0``

<a anchor="tombo"/>
### *Tombo*

Tombo is a suite of tools primarily for the identification of modified nucleotides  from raw nanopore sequencing data.

*homepage*: https://github.com/nanoporetech/tombo

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.5.1``|``-Python-3.7.4``|``foss/2019b``

<a anchor="toml-fortran"/>
### *TOML-Fortran*

TOML parser for Fortran projects

*homepage*: https://github.com/toml-f/toml-f

version  |toolchain
---------|---------------------------------------
``0.2.2``|``GCC/10.2.0``, ``iccifort/2020.4.304``
``0.3.1``|``GCC/11.2.0``

<a anchor="topaz"/>
### *topaz*

A pipeline for particle detection in cryo-electron microscopy  images using convolutional neural networks trained from positive and unlabeled  examples. Topaz also includes methods for micrograph denoising using deep de-  noising models. 

*homepage*: https://github.com/tbepler/topaz

version  |versionsuffix   |toolchain
---------|----------------|--------------
``0.2.5``|                |``foss/2021a``
``0.2.5``|``-CUDA-11.3.1``|``foss/2021a``

<a anchor="tophat"/>
### *TopHat*

TopHat is a fast splice junction mapper for RNA-Seq reads.

*homepage*: http://ccb.jhu.edu/software/tophat/

version  |versionsuffix     |toolchain
---------|------------------|--------------------------------------------------------------------------------
``2.1.0``|``-Python-2.7.14``|``intel/2018a``
``2.1.1``|                  |``foss/2016a``, ``foss/2016b``, ``foss/2017b``, ``intel/2017a``, ``intel/2017b``
``2.1.2``|                  |``foss/2018b``, ``gompi/2019b``, ``iimpi/2019b``
``2.1.2``|``-Python-2.7.18``|``GCC/10.2.0``

<a anchor="torchaudio"/>
### *torchaudio*

Data manipulation and transformation for audio signal processing, powered by PyTorch 

*homepage*: https://github.com/pytorch/audio

version  |versionsuffix                  |toolchain
---------|-------------------------------|----------------------------------
``0.5.0``|``-Python-3.7.4-PyTorch-1.4.0``|``foss/2019b``, ``fosscuda/2019b``

<a anchor="torchdata"/>
### *torchdata*

TorchData is a prototype library of common modular data loading primitives for easily constructing flexible and performant data pipelines." 

*homepage*: https://github.com/pytorch/data

version  |versionsuffix                  |toolchain
---------|-------------------------------|--------------
``0.3.0``|``-PyTorch-1.11.0-CUDA-11.3.1``|``foss/2021a``

<a anchor="torchinfo"/>
### *torchinfo*

" Torchinfo provides information complementary to what is provided by print(your_model) in PyTorch, similar to Tensorflow's model.summary() API to view the visualization of the model, which is helpful while debugging your network. 

*homepage*: https://github.com/TylerYep/torchinfo

version  |versionsuffix     |toolchain
---------|------------------|----------------------------------
``1.5.2``|``-PyTorch-1.7.1``|``foss/2020b``, ``fosscuda/2020b``

<a anchor="torchsampler"/>
### *torchsampler*

A (PyTorch) imbalanced dataset sampler for oversampling low classes and undersampling high frequent ones.

*homepage*: https://github.com/ufoym/imbalanced-dataset-sampler

version  |versionsuffix   |toolchain
---------|----------------|--------------
``0.1.2``|                |``foss/2022a``
``0.1.2``|``-CUDA-11.7.0``|``foss/2022a``

<a anchor="torchtext"/>
### *torchtext*

Data loaders and abstractions for text and NLP

*homepage*: https://github.com/pytorch/text

version   |versionsuffix                  |toolchain
----------|-------------------------------|----------------------------------
``0.5.0`` |``-PyTorch-1.4.0-Python-3.7.4``|``fosscuda/2019b``
``0.7.0`` |``-Python-3.7.4-PyTorch-1.6.0``|``foss/2019b``, ``fosscuda/2019b``
``0.8.1`` |``-PyTorch-1.7.1``             |``fosscuda/2020b``
``0.9.1`` |``-PyTorch-1.8.1``             |``fosscuda/2020b``
``0.10.0``|``-PyTorch-1.9.0``             |``fosscuda/2020b``

<a anchor="torchvision"/>
### *torchvision*

Datasets, Transforms and Models specific to Computer Vision

*homepage*: https://github.com/pytorch/vision

version   |versionsuffix                       |toolchain
----------|------------------------------------|----------------------------------
``0.2.0`` |``-Python-3.6.4``                   |``intel/2018a``
``0.2.0`` |``-Python-3.6.4-CUDA-9.1.85``       |``foss/2018a``
``0.2.2`` |``-Python-3.6.6``                   |``foss/2018b``, ``fosscuda/2018b``
``0.3.0`` |``-Python-3.7.2``                   |``foss/2019a``
``0.4.2`` |``-PyTorch-1.3.1``                  |``fosscuda/2020b``
``0.4.2`` |``-Python-3.7.4``                   |``foss/2019b``, ``fosscuda/2019b``
``0.5.0`` |``-Python-3.7.4-PyTorch-1.4.0``     |``fosscuda/2019b``
``0.7.0`` |``-Python-3.7.4-PyTorch-1.6.0``     |``foss/2019b``, ``fosscuda/2019b``
``0.7.0`` |``-Python-3.7.4-PyTorch-1.6.0-imkl``|``fosscuda/2019b``
``0.8.2`` |``-PyTorch-1.7.1``                  |``foss/2020b``, ``fosscuda/2020b``
``0.8.2`` |``-Python-3.7.4-PyTorch-1.7.1``     |``fosscuda/2019b``
``0.9.1`` |``-PyTorch-1.8.1``                  |``fosscuda/2020b``
``0.9.1`` |``-Python-3.7.4-PyTorch-1.8.1``     |``fosscuda/2019b``
``0.10.0``|``-PyTorch-1.9.0``                  |``fosscuda/2020b``
``0.11.1``|                                    |``foss/2021a``
``0.11.1``|``-CUDA-11.3.1``                    |``foss/2021a``
``0.11.3``|                                    |``foss/2021a``
``0.11.3``|``-CUDA-11.3.1``                    |``foss/2021a``
``0.12.0``|``-PyTorch-1.11.0-CUDA-11.3.1``     |``foss/2021a``
``0.13.1``|                                    |``foss/2022a``
``0.13.1``|``-CUDA-11.7.0``                    |``foss/2022a``

<a anchor="totalview"/>
### *TotalView*

TotalView is a GUI-based source code defect analysis tool that gives you unprecedented  control over processes and thread execution and visibility into program state and variables. It allows  you to debug one or many processes and/or threads in a single window with complete control over program  execution. This allows you to set breakpoints, stepping line by line through the code on a single thread,  or with coordinated groups of processes or threads, and run or halt arbitrary sets of processes or threads.  You can reproduce and troubleshoot difficult problems that can occur in concurrent programs that take  advantage of threads, OpenMP, MPI, GPUs or coprocessors.

*homepage*: http://www.roguewave.com/products/totalview.aspx

version     |versionsuffix    |toolchain
------------|-----------------|----------
``8.11.0-0``|``-linux-x86-64``|``system``
``8.11.0-2``|``-linux-x86-64``|``system``
``8.12.0-0``|``-linux-x86-64``|``system``

<a anchor="tqdm"/>
### *tqdm*

A fast, extensible progress bar for Python and CLI

*homepage*: https://github.com/tqdm/tqdm

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``4.24.0``|``-Python-2.7.15``|``foss/2018b``
``4.24.0``|``-Python-3.5.1`` |``foss/2016a``
``4.29.0``|``-Python-3.6.4`` |``intel/2018a``
``4.32.1``|                  |``GCCcore/8.2.0``
``4.41.1``|                  |``GCCcore/8.3.0``
``4.41.1``|``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``4.47.0``|                  |``GCCcore/9.3.0``
``4.51.0``|``-Python-3.8.2`` |``intel/2020a``
``4.56.2``|                  |``GCCcore/10.2.0``
``4.60.0``|                  |``GCCcore/10.2.0``
``4.61.1``|                  |``GCCcore/10.3.0``
``4.61.2``|                  |``GCCcore/10.3.0``
``4.62.3``|                  |``GCCcore/11.2.0``
``4.64.0``|                  |``GCCcore/11.3.0``

<a anchor="tracer"/>
### *Tracer*

Tracer is a graphical tool for visualization and diagnostics of MCMC output.

*homepage*: https://github.com/beast-dev/tracer

version  |toolchain
---------|-----------------
``1.7.1``|``GCCcore/8.2.0``

<a anchor="transcriptclean"/>
### *TranscriptClean*

TranscriptClean is a Python program that corrects mismatches, microindels, and noncanonical splice junctions in long reads that have been mapped to the genome.

*homepage*: https://github.com/mortazavilab/TranscriptClean

version  |toolchain
---------|--------------
``2.0.2``|``foss/2020b``

<a anchor="transdecoder"/>
### *TransDecoder*

TransDecoder identifies candidate coding regions within transcript sequences,  such as those generated by de novo RNA-Seq transcript assembly using Trinity,  or constructed based on RNA-Seq alignments to the genome using  Tophat and Cufflinks.

*homepage*: https://github.com/TransDecoder/TransDecoder/wiki

version  |versionsuffix   |toolchain
---------|----------------|---------------
``2.1.0``|``-Perl-5.24.1``|``intel/2017a``

<a anchor="transformers"/>
### *Transformers*

State-of-the-art Natural Language Processing for PyTorch and TensorFlow 2.0

*homepage*: https://github.com/huggingface/transformers

version   |versionsuffix    |toolchain
----------|-----------------|--------------
``4.2.1`` |``-Python-3.8.2``|``foss/2020a``
``4.20.1``|                 |``foss/2021a``
``4.21.1``|                 |``foss/2021b``
``4.24.0``|                 |``foss/2022a``

<a anchor="transrate"/>
### *Transrate*

Transrate is software for de-novo transcriptome assembly quality analysis.  It examines your assembly in detail and compares it to experimental evidence  such as the sequencing reads, reporting quality scores for contigs and  assemblies. This allows you to choose between assemblers and parameters,  filter out the bad contigs from an assembly, and help decide when  to stop trying to improve the assembly.

*homepage*: http://hibberdlab.com/transrate/

version  |toolchain
---------|----------
``1.0.3``|``system``

<a anchor="travis"/>
### *travis*

Travis CI Client (CLI and Ruby library)

*homepage*: https://github.com/travis-ci/travis.rb

version  |versionsuffix  |toolchain
---------|---------------|----------
``1.8.2``|``-Ruby-2.3.1``|``system``
``1.8.4``|``-Ruby-2.3.3``|``system``

<a anchor="travis-analyzer"/>
### *TRAVIS-Analyzer*

TRAVIS is a free tool for analyzing and visualizing trajectories from all kinds of Molecular Dynamics or Monte Carlo simulations.

*homepage*: http://www.travis-analyzer.de/

version   |toolchain
----------|--------------
``210521``|``GCC/10.3.0``

<a anchor="treatsens"/>
### *treatSens*

Utilities to investigate sensitivity to unmeasured confounding in parametric models  with either binary or continuous treatment.

*homepage*: https://github.com/vdorie/treatSens

version         |versionsuffix|toolchain
----------------|-------------|--------------
``3.0-20201002``|``-R-4.0.0`` |``foss/2020a``

<a anchor="treemix"/>
### *TreeMix*

TreeMix is a method for inferring the patterns of population splits and mixtures in the history of a  set of populations.

*homepage*: http://bitbucket.org/nygcresearch/treemix

version |toolchain
--------|-----------------------------------------------
``1.13``|``GCC/10.3.0``, ``GCC/11.2.0``, ``intel/2018a``

<a anchor="treeshrink"/>
### *TreeShrink*

TreeShrink is an algorithm for detecting abnormally long branches in one or more phylogenetic trees.

*homepage*: https://github.com/uym2/TreeShrink

version  |versionsuffix    |toolchain
---------|-----------------|--------------------
``1.3.2``|``-Python-3.7.2``|``GCC/8.2.0-2.31.1``

<a anchor="trf"/>
### *TRF*

Tandem Repeats Finder: a program to analyze DNA sequences.

*homepage*: https://tandem.bu.edu/trf/trf.html

version   |versionsuffix|toolchain
----------|-------------|-------------------------------------------------------------------------
``4.09``  |``-linux64`` |``system``
``4.09.1``|             |``GCC/10.3.0``, ``GCCcore/10.2.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``

<a anchor="triangle"/>
### *Triangle*

Triangle generates exact Delaunay triangulations, constrained Delaunay triangulations,  conforming Delaunay triangulations, Voronoi diagrams, and high-quality triangular meshes.  The latter can be generated with no small or large angles,  and are thus suitable for finite element analysis.

*homepage*: http://www.cs.cmu.edu/~quake/triangle.html

version|toolchain
-------|----------------------------------------------------------------------------------------------------
``1.6``|``GCCcore/6.4.0``, ``foss/2016a``, ``foss/2018a``, ``intel/2016b``, ``intel/2017a``, ``intel/2018a``

<a anchor="trilinos"/>
### *Trilinos*

The Trilinos Project is an effort to develop algorithms and enabling technologies  within an object-oriented software framework for the solution of large-scale, complex multi-physics  engineering and scientific problems. A unique design feature of Trilinos is its focus on packages.

*homepage*: https://trilinos.org

version    |versionsuffix     |toolchain
-----------|------------------|-------------------------------
``12.12.1``|``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``12.12.1``|``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``12.12.1``|``-Python-3.7.4`` |``foss/2019b``

<a anchor="trim_galore"/>
### *Trim_Galore*

Trim Galore! is a wrapper script to automate quality and adapter trimming as well as quality control, with some added functionality to remove biased methylation positions for RRBS sequence files (for directional, non-directional  (or paired-end) sequencing).

*homepage*: https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/

version  |versionsuffix            |toolchain
---------|-------------------------|-------------------------------
``0.4.2``|                         |``foss/2016b``
``0.4.4``|                         |``foss/2016b``, ``intel/2017a``
``0.5.0``|                         |``foss/2018b``
``0.5.0``|``-Python-3.6.6``        |``intel/2018b``
``0.6.0``|``-Python-2.7.15``       |``foss/2018b``
``0.6.0``|``-Python-3.6.6``        |``foss/2018b``
``0.6.1``|``-Python-2.7.15``       |``foss/2018b``
``0.6.1``|``-Python-3.6.6``        |``foss/2018b``
``0.6.2``|``-Java-11``             |``GCCcore/8.2.0``
``0.6.2``|``-Python-2.7.15``       |``foss/2018b``
``0.6.2``|``-Python-3.6.6``        |``foss/2018b``
``0.6.5``|``-Java-11-Python-3.7.4``|``GCCcore/8.3.0``
``0.6.6``|``-Python-2.7.18``       |``GCC/10.2.0``
``0.6.6``|``-Python-3.8.2``        |``GCCcore/9.3.0``
``0.6.7``|                         |``GCCcore/10.3.0``

<a anchor="trimal"/>
### *trimAl*

A tool for automated alignment trimming in large-scale phylogenetic analyses

*homepage*: http://trimal.cgenomics.org/

version  |toolchain
---------|----------------------------------------------------------------------------
``1.4.1``|``GCC/8.2.0-2.31.1``, ``GCCcore/10.3.0``, ``GCCcore/9.3.0``, ``intel/2018a``

<a anchor="trimesh"/>
### *trimesh*

Trimesh is a Python (2.7- 3.3+) library for loading and using triangular meshes with an emphasis on watertight meshes. The goal of the library is to provide a fully featured Trimesh object which allows for easy manipulation and analysis, in the style of the excellent Polygon object in the Shapely library.

*homepage*: https://trimsh.org/

version   |toolchain
----------|--------------
``3.17.1``|``foss/2022a``

<a anchor="trimmomatic"/>
### *Trimmomatic*

Trimmomatic performs a variety of useful trimming tasks for illumina   paired-end and single ended data.The selection of trimming steps and their associated   parameters are supplied on the command line. 

*homepage*: http://www.usadellab.org/cms/?page=trimmomatic

version |versionsuffix      |toolchain
--------|-------------------|----------
``0.32``|``-Java-1.7.0_80`` |``system``
``0.36``|``-Java-1.8.0_92`` |``system``
``0.38``|``-Java-1.8``      |``system``
``0.38``|``-Java-1.8.0_162``|``system``
``0.39``|``-Java-1.8``      |``system``
``0.39``|``-Java-11``       |``system``

<a anchor="trinity"/>
### *Trinity*

Trinity represents a novel method for the efficient and robust de novo reconstruction  of transcriptomes from RNA-Seq data. Trinity combines three independent software modules: Inchworm,  Chrysalis, and Butterfly, applied sequentially to process large volumes of RNA-Seq reads.

*homepage*: https://trinityrnaseq.github.io

version   |versionsuffix     |toolchain
----------|------------------|----------------------------------------
``2.2.0`` |                  |``foss/2016a``
``2.4.0`` |                  |``foss/2017a``
``2.5.1`` |                  |``intel/2017a``, ``intel/2017b``
``2.6.6`` |                  |``intel/2018a``
``2.8.4`` |                  |``foss/2018b``
``2.8.5`` |                  |``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``2.8.5`` |``-Java-11``      |``GCC/8.3.0``
``2.9.1`` |``-Python-2.7.16``|``foss/2019b``
``2.9.1`` |``-Python-3.7.4`` |``foss/2019b``
``2.10.0``|``-Python-3.7.4`` |``foss/2019b``

<a anchor="triqs"/>
### *TRIQS*

TRIQS (Toolbox for Research on Interacting Quantum Systems) is a  scientific project providing a set of C++ and Python libraries to  develop new tools for the study of interacting quantum systems. 

*homepage*: https://triqs.github.io/triqs

version  |versionsuffix     |toolchain
---------|------------------|--------------
``2.2.0``|``-Python-2.7.15``|``foss/2019a``
``2.2.1``|``-Python-2.7.15``|``foss/2019a``
``3.0.0``|``-Python-3.8.2`` |``foss/2020a``
``3.1.1``|                  |``foss/2022a``

<a anchor="triqs-cthyb"/>
### *TRIQS-cthyb*

TRIQS (Toolbox for Research on Interacting Quantum Systems) is a  scientific project providing a set of C++ and Python libraries to  develop new tools for the study of interacting quantum systems.   cthyb = continuous-time hybridisation-expansion quantum Monte Carlo   The TRIQS-based hybridization-expansion solver allows to solve the  generic problem of a quantum impurity embedded in a conduction bath  for an arbitrary local interaction vertex. The “impurity” can be any  set of orbitals, on one or several atoms. 

*homepage*: https://triqs.github.io/cthyb/

version  |versionsuffix     |toolchain
---------|------------------|--------------
``2.2.0``|``-Python-2.7.15``|``foss/2019a``
``3.0.0``|``-Python-3.8.2`` |``foss/2020a``
``3.1.0``|                  |``foss/2022a``

<a anchor="triqs-dft_tools"/>
### *TRIQS-dft_tools*

TRIQS (Toolbox for Research on Interacting Quantum Systems) is a  scientific project providing a set of C++ and Python libraries to  develop new tools for the study of interacting quantum systems.   This TRIQS-based-based application is aimed at ab-initio calculations  for correlated materials, combining realistic DFT band-structure calculation   with the dynamical mean-field theory. Together with the necessary tools to  perform the DMFT self-consistency loop for realistic multi-band problems,  the package provides a full-fledged charge self-consistent interface to the  Wien2K package. In addition, if Wien2k is not available, it provides a generic  interface for one-shot DFT+DMFT calculations, where only the single-particle  Hamiltonian in orbital space has to be provided. 

*homepage*: https://triqs.github.io/dft_tools/

version  |versionsuffix     |toolchain
---------|------------------|--------------
``2.2.0``|``-Python-2.7.15``|``foss/2019a``
``3.0.0``|``-Python-3.8.2`` |``foss/2020a``
``3.1.0``|                  |``foss/2022a``

<a anchor="triqs-tprf"/>
### *TRIQS-tprf*

TRIQS (Toolbox for Research on Interacting Quantum Systems) is a  scientific project providing a set of C++ and Python libraries to  develop new tools for the study of interacting quantum systems.   TPRF is a TRIQS-based two-particle response function tool box that  implements basic operations for higher order response functions such  as inversion, products, the random phase approximation, the bethe  salpeter equation (in the local vertex approximation), etc..   The aim is to provide efficient (C++/OpenMP/MPI) implementations of  the basic operations needed to compute the two-particle response in  the different two-particle channels (particle-hole, particle-particle). 

*homepage*: https://triqs.github.io/tprf

version  |versionsuffix     |toolchain
---------|------------------|--------------
``2.2.0``|``-Python-2.7.15``|``foss/2019a``
``3.0.0``|``-Python-3.8.2`` |``foss/2020a``
``3.1.1``|                  |``foss/2022a``

<a anchor="trust"/>
### *TRUST*

Tcr Receptor Utilities for Solid Tissue (TRUST) is a computational tool to analyze  TCR and BCR sequences using unselected RNA sequencing data, profiled from solid tissues, including tumors. 

*homepage*: https://bitbucket.org/liulab/trust

version  |versionsuffix     |toolchain
---------|------------------|---------------
``3.0.2``|``-Python-2.7.14``|``intel/2018a``

<a anchor="trust4"/>
### *TRUST4*

Tcr Receptor Utilities for Solid Tissue (TRUST) is a computational tool to analyze TCR and BCR sequences using unselected RNA sequencing data, profiled from solid tissues, including tumors. TRUST4 performs de novo assembly on V, J, C genes including the hypervariable complementarity-determining region 3 (CDR3) and reports consensus of BCR/TCR sequences. TRUST4 then realigns the contigs to IMGT reference gene sequences to report the corresponding information. TRUST4 supports both single-end and paired-end sequencing data with any read length.

*homepage*: https://github.com/liulab-dfci/TRUST4

version    |toolchain
-----------|--------------
``1.0.5.1``|``system``
``1.0.6``  |``GCC/11.2.0``
``1.0.7``  |``GCC/11.3.0``

<a anchor="trycycler"/>
### *Trycycler*

Trycycler is a tool for generating consensus long-read assemblies for bacterial genomes.

*homepage*: https://github.com/rrwick/Trycycler

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``0.3.3``|``-Python-3.8.2``|``foss/2020a``
``0.5.2``|                 |``foss/2021a``
``0.5.3``|                 |``foss/2021b``

<a anchor="tsne"/>
### *tsne*

Python library containing T-SNE algorithms.

*homepage*: https://github.com/danielfrg/tsne

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.1.8``|``-Python-2.7.16``|``intel/2019b``

<a anchor="turbovnc"/>
### *TurboVNC*

TurboVNC is a derivative of VNC (Virtual Network Computing) that is tuned to provide  peak performance for 3D and video workloads.

*homepage*: https://www.turbovnc.org

version  |toolchain
---------|------------------
``2.2.3``|``GCCcore/8.2.0``
``3.0.1``|``GCCcore/11.3.0``

<a anchor="tvb"/>
### *TVB*

The Virtual Brain will deliver the first open simulation of the human brain  based on individual large-scale connectivity.

*homepage*: http://thevirtualbrain.org

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.4.1``|``-Python-2.7.11``|``intel/2016a``
``1.5``  |``-Python-2.7.11``|``intel/2016a``

<a anchor="tvb-data"/>
### *tvb-data*

The Virtual Brain Project (TVB Project) has the purpose of offering some modern tools to the  Neurosciences community, for computing, simulating and analyzing functional and structural data of human brains.  Various demonstration data for use with TVB.

*homepage*: http://www.thevirtualbrain.org/

version     |versionsuffix     |toolchain
------------|------------------|---------------
``1.5``     |``-Python-2.7.11``|``intel/2016a``
``20150915``|``-Python-2.7.11``|``intel/2016a``

<a anchor="tvb-deps"/>
### *TVB-deps*

Bundle of dependency Python packages for TVB (The Virtual Brain)

*homepage*: https://thevirtualbrain.org

version     |versionsuffix     |toolchain
------------|------------------|---------------
``20160618``|``-Python-2.7.11``|``intel/2016a``

<a anchor="tvb-framework"/>
### *tvb-framework*

The Virtual Brain Project (TVB Project) has the purpose of offering some modern tools to the  Neurosciences community, for computing, simulating and analyzing functional and structural data of human brains.  TVB Scientific Library is the most important scientific contribution of TVB Project.

*homepage*: http://www.thevirtualbrain.org/

version     |versionsuffix     |toolchain
------------|------------------|---------------
``1.5``     |``-Python-2.7.11``|``intel/2016a``
``20150921``|``-Python-2.7.11``|``intel/2016a``

<a anchor="tvb-library"/>
### *tvb-library*

The Virtual Brain Project (TVB Project) has the purpose of offering some modern tools to the  Neurosciences community, for computing, simulating and analyzing functional and structural data of human brains.  TVB Scientific Library is the most important scientific contribution of TVB Project.

*homepage*: http://www.thevirtualbrain.org/

version     |versionsuffix     |toolchain
------------|------------------|---------------
``1.5``     |``-Python-2.7.11``|``intel/2016a``
``20150922``|``-Python-2.7.11``|``intel/2016a``

<a anchor="twl-ninja"/>
### *TWL-NINJA*

Nearly Infinite Neighbor Joining Application.

*homepage*: https://github.com/TravisWheelerLab/NINJA

version              |toolchain
---------------------|--------------
``0.97-cluster_only``|``GCC/10.2.0``

<a anchor="typing-extensions"/>
### *typing-extensions*

Typing Extensions - Backported and Experimental Type Hints for Python

*homepage*: https://github.com/python/typing/blob/master/typing_extensions/README.rst

version     |versionsuffix    |toolchain
------------|-----------------|------------------
``3.7.4.3`` |                 |``GCCcore/10.2.0``
``3.7.4.3`` |``-Python-3.7.4``|``GCCcore/8.3.0``
``3.7.4.3`` |``-Python-3.8.2``|``GCCcore/9.3.0``
``3.10.0.0``|                 |``GCCcore/10.3.0``
``3.10.0.2``|                 |``GCCcore/11.2.0``
``4.3.0``   |                 |``GCCcore/11.3.0``
``4.4.0``   |                 |``GCCcore/10.3.0``

<a anchor="u"/>
### *U*


<a href="#ucc">UCC</a> - <a href="#ucc-cuda">UCC-CUDA</a> - <a href="#uclust">UCLUST</a> - <a href="#ucx">UCX</a> - <a href="#ucx-cuda">UCX-CUDA</a> - <a href="#ucx-py">ucx-py</a> - <a href="#udocker">udocker</a> - <a href="#udunits">UDUNITS</a> - <a href="#ufl">UFL</a> - <a href="#umap-learn">umap-learn</a> - <a href="#umi-tools">UMI-tools</a> - <a href="#umi4cpackage">umi4cPackage</a> - <a href="#umis">umis</a> - <a href="#unafold">UNAFold</a> - <a href="#uncertainty-calibration">uncertainty-calibration</a> - <a href="#unicycler">Unicycler</a> - <a href="#unidecode">Unidecode</a> - <a href="#unimap">unimap</a> - <a href="#units">units</a> - <a href="#unixodbc">unixODBC</a> - <a href="#unrar">unrar</a> - <a href="#unzip">UnZip</a> - <a href="#uqtk">UQTk</a> - <a href="#usearch">USEARCH</a> - <a href="#usher">UShER</a> - <a href="#uspex">USPEX</a> - <a href="#utf8proc">utf8proc</a> - <a href="#util-linux">util-linux</a>


<a anchor="ucc"/>
### *UCC*

UCC (Unified Collective Communication) is a collective communication operations API and library that is flexible, complete, and  feature-rich for current and emerging programming models and runtimes. 

*homepage*: https://www.openucx.org/

version  |toolchain
---------|------------------
``1.0.0``|``GCCcore/11.3.0``
``1.1.0``|``GCCcore/12.2.0``

<a anchor="ucc-cuda"/>
### *UCC-CUDA*

UCC (Unified Collective Communication) is a collective communication operations API and library that is flexible, complete, and  feature-rich for current and emerging programming models and runtimes.  This module adds the UCC CUDA support. 

*homepage*: https://www.openucx.org/

version  |versionsuffix   |toolchain
---------|----------------|------------------
``1.0.0``|``-CUDA-11.7.0``|``GCCcore/11.3.0``

<a anchor="uclust"/>
### *UCLUST*

UCLUST: Extreme high-speed sequence clustering, alignment and database search.

*homepage*: https://www.drive5.com/usearch/index.html

version    |versionsuffix  |toolchain
-----------|---------------|----------
``1.2.22q``|``-i86linux64``|``system``

<a anchor="ucx"/>
### *UCX*

Unified Communication X An open-source production grade communication framework for data centric and high-performance applications 

*homepage*: http://www.openucx.org/

version     |versionsuffix   |toolchain
------------|----------------|-------------------------------------------------------
``1.3.1``   |                |``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``1.5.0``   |                |``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``1.5.0rc1``|``-hpcx``       |``GCCcore/8.2.0``
``1.5.1``   |                |``GCCcore/8.2.0``
``1.6.1``   |                |``GCCcore/8.3.0``
``1.8.0``   |                |``GCCcore/9.3.0``
``1.8.0``   |``-CUDA-11.0.2``|``GCCcore/9.3.0``
``1.9.0``   |                |``GCCcore/10.2.0``
``1.9.0``   |``-CUDA-11.1.1``|``GCCcore/10.2.0``
``1.9.0``   |``-CUDA-11.2.1``|``GCCcore/10.2.0``
``1.10.0``  |                |``GCCcore/10.3.0``
``1.11.0``  |                |``GCCcore/11.2.0``
``1.11.2``  |                |``GCCcore/11.2.0``
``1.12.1``  |                |``GCCcore/11.3.0``
``1.13.1``  |                |``GCCcore/12.2.0``

<a anchor="ucx-cuda"/>
### *UCX-CUDA*

Unified Communication X An open-source production grade communication framework for data centric and high-performance applications  This module adds the UCX CUDA support. 

*homepage*: http://www.openucx.org/

version   |versionsuffix   |toolchain
----------|----------------|------------------
``1.10.0``|``-CUDA-11.3.1``|``GCCcore/10.3.0``
``1.11.0``|``-CUDA-11.4.1``|``GCCcore/11.2.0``
``1.11.2``|``-CUDA-11.4.1``|``GCCcore/11.2.0``
``1.12.1``|``-CUDA-11.7.0``|``GCCcore/11.3.0``
``1.13.1``|``-CUDA-12.0.0``|``GCCcore/12.2.0``

<a anchor="ucx-py"/>
### *ucx-py*

UCX-Py is the Python interface for UCX, a low-level high-performance networking library. UCX and UCX-Py supports several transport methods including InfiniBand and NVLink while still using traditional networking protocols like TCP.

*homepage*: https://ucx-py.readthedocs.io

version   |toolchain
----------|--------------
``0.21.0``|``foss/2020b``

<a anchor="udocker"/>
### *udocker*

A basic user tool to execute simple docker containers in batch  or interactive systems without root privileges.

*homepage*: https://github.com/indigo-dc/udocker

version  |versionsuffix     |toolchain
---------|------------------|---------------
``1.1.1``|``-Python-2.7.14``|``foss/2017b``
``1.1.3``|``-Python-2.7.14``|``intel/2018a``

<a anchor="udunits"/>
### *UDUNITS*

UDUNITS supports conversion of unit specifications between formatted and binary forms,  arithmetic manipulation of units, and conversion of values between compatible scales of measurement.

*homepage*: https://www.unidata.ucar.edu/software/udunits/

version   |toolchain
----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``2.2.20``|``foss/2016a``, ``intel/2016b``
``2.2.24``|``intel/2017a``
``2.2.25``|``foss/2017b``, ``intel/2017b``
``2.2.26``|``GCCcore/10.2.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``foss/2018a``, ``foss/2018b``, ``foss/2020a``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``, ``iomkl/2018b``
``2.2.28``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="ufl"/>
### *UFL*

The Unified Form Language (UFL) is a domain specific language   for declaration of finite element discretizations of variational forms.   More precisely, it defines a flexible interface for choosing finite element   spaces and defining expressions for weak forms in a notation close to   mathematical notation.

*homepage*: https://bitbucket.org/fenics-project/ufl

version     |versionsuffix    |toolchain
------------|-----------------|--------------
``2018.1.0``|``-Python-3.6.4``|``foss/2018a``
``2019.1.0``|``-Python-3.7.4``|``foss/2019b``

<a anchor="umap-learn"/>
### *umap-learn*

Uniform Manifold Approximation and Projection (UMAP) is a dimension reduction technique  that can be used for visualisation similarly to t-SNE, but also for general non-linear  dimension reduction. 

*homepage*: https://umap-learn.readthedocs.io/en/latest/

version  |toolchain
---------|------------------------------
``0.4.6``|``fosscuda/2020b``
``0.5.3``|``foss/2021a``, ``foss/2021b``

<a anchor="umi-tools"/>
### *UMI-tools*

Tools for handling Unique Molecular Identifiers in NGS data sets

*homepage*: https://umi-tools.readthedocs.io

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.0.0``|``-Python-3.6.6``|``foss/2018b``
``1.0.1``|``-Python-3.7.4``|``foss/2019b``
``1.0.1``|``-Python-3.8.2``|``foss/2020a``

<a anchor="umi4cpackage"/>
### *umi4cPackage*

umi4cPackage is a processing and analysis pipeline for UMI-4C experiment.

*homepage*: https://github.com/tanaylab/umi4cpackage

version     |versionsuffix|toolchain
------------|-------------|--------------
``20200116``|``-R-4.0.0`` |``foss/2020a``

<a anchor="umis"/>
### *umis*

Package for estimating UMI counts in Transcript Tag Counting data.

*homepage*: https://github.com/vals/umis

version  |toolchain
---------|---------------
``1.0.3``|``intel/2019a``

<a anchor="unafold"/>
### *UNAFold*

The UNAFold package contains several programs for performing energy minimization and partition  function calculations on nucleic acid sequences.

*homepage*: http://mfold.rna.albany.edu/?q=DINAMelt/software

version|versionsuffix   |toolchain
-------|----------------|---------------
``3.8``|``-Perl-5.24.1``|``intel/2017a``

<a anchor="uncertainty-calibration"/>
### *uncertainty-calibration*

Python library to measure the calibration error of models, including confidence intervals computed by Bootstrap resampling, and code to recalibrate models.

*homepage*: https://github.com/p-lambda/verified_calibration

version  |toolchain
---------|--------------
``0.0.9``|``foss/2021b``

<a anchor="unicycler"/>
### *Unicycler*

Unicycler is an assembly pipeline for bacterial genomes. It can assemble Illumina-only read sets  where it functions as a SPAdes-optimiser. It can also assembly long-read-only sets (PacBio or Nanopore)  where it runs a miniasm+Racon pipeline. 

*homepage*: https://github.com/rrwick/Unicycler

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.4.7``|``-Python-3.7.2``|``foss/2019a``
``0.4.8``|``-Python-3.8.2``|``gompi/2020a``
``0.4.9``|                 |``gompi/2021a``
``0.5.0``|                 |``gompi/2021b``

<a anchor="unidecode"/>
### *Unidecode*

Python library for lossy ASCII transliterations of Unicode text (port of Text::Unidecode Perl module)

*homepage*: https://github.com/avian2/unidecode

version  |versionsuffix    |toolchain
---------|-----------------|-----------------
``1.1.1``|``-Python-3.7.4``|``GCCcore/8.3.0``

<a anchor="unimap"/>
### *unimap*

Unimap is a fork of minimap2 optimized for assembly-to-reference alignment. It integrates the minigraph chaining algorithm and can align through long INDELs (up to 100kb by default) much faster than minimap2. Unimap is a better fit for resolving segmental duplications and is recommended over minimap2 for alignment between high-quality assemblies.  Unimap does not replace minimap2 for other types of alignment. It drops the support of multi-part index and short-read mapping. Its long-read alignment is different from minimap2 but is not necessarily better. Unimap is more of a specialized minimap2 at the moment.

*homepage*: https://github.com/lh3/unimap

version|toolchain
-------|------------------
``0.1``|``GCCcore/10.2.0``

<a anchor="units"/>
### *units*

GNU Units converts quantities expressed in various systems of measurement to their equivalents in other systems of measurement. Like many similar programs, it can handle multiplicative scale changes. It can also handle nonlinear conversions such as Fahrenheit to Celsius or wire gauge, and it can convert from and to sums of units, such as converting between meters and feet plus inches.

*homepage*: https://www.gnu.org/software/units/

version |toolchain
--------|-----------------
``2.19``|``GCCcore/8.2.0``

<a anchor="unixodbc"/>
### *unixODBC*

unixODBC provides a uniform interface between application and database driver

*homepage*: https://www.unixodbc.org

version  |toolchain
---------|--------------
``2.3.7``|``foss/2018b``

<a anchor="unrar"/>
### *unrar*

RAR is a powerful archive manager.

*homepage*: https://www.rarlab.com/

version  |toolchain
---------|------------------
``5.6.1``|``GCCcore/7.3.0``
``5.7.3``|``GCCcore/8.2.0``
``6.0.2``|``GCCcore/10.2.0``

<a anchor="unzip"/>
### *UnZip*

UnZip is an extraction utility for archives compressed in .zip format (also called "zipfiles"). Although highly compatible both with PKWARE's PKZIP and PKUNZIP utilities for MS-DOS and with Info-ZIP's own Zip program, our primary objectives have been portability and non-MSDOS functionality.

*homepage*: http://www.info-zip.org/UnZip.html

version|toolchain
-------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``6.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``system``

<a anchor="uqtk"/>
### *UQTk*

The UQ Toolkit (UQTk) is a collection of libraries and tools  for the quantification of uncertainty in numerical model predictions.

*homepage*: https://www.sandia.gov/UQToolkit/

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``3.1.0``|``-Python-3.7.4``|``intel/2019b``

<a anchor="usearch"/>
### *USEARCH*

USEARCH is a unique sequence analysis tool which offers search and clustering algorithms that are  often orders of magnitude faster than BLAST.

*homepage*: http://www.drive5.com/usearch/index.html

version            |versionsuffix  |toolchain
-------------------|---------------|----------
``5.2.236``        |``-i86linux32``|``system``
``5.2.236-6.1.544``|``-i86linux32``|``system``
``6.1.544``        |``-i86linux32``|``system``
``11.0.667``       |``-i86linux32``|``system``

<a anchor="usher"/>
### *UShER*

UShER is now a package consisting of a family of programs for  rapid phylogenetic analyses, particularly suitable for the SARS-CoV-2 genomes.

*homepage*: https://github.com/yatisht/usher

version  |toolchain
---------|---------------
``0.4.1``|``gompi/2020b``
``0.5.0``|``gompi/2021b``

<a anchor="uspex"/>
### *USPEX*

USPEX (Universal Structure Predictor: Evolutionary Xtallography...  and in Russian "uspekh" means "success" - owing to the high success rate and many  useful results produced by this method) is a method developed by the Oganov  laboratory since 2004. The problem of crystal structure prediction is very old and  does, in fact, constitute the central problem of theoretical crystal chemistry.  USPEX can also be used for finding low-energy metastable phases, as well as stable  structures of nanoparticles, surface reconstructions, molecular packings in organic  crystals, and for searching for materials with desired physical (mechanical,  electronic) properties. The USPEX code is based on an efficient evolutionary  algorithm developed by A.R. Oganov's group, but also has options for using  alternative methods (random sampling, metadynamics, corrected particle swarm  optimization algorithms). USPEX is interfaced with many ab initio codes, such as  VASP, SIESTA, GULP, Quantum Espresso, CP2K, CASTEP, LAMMPS, and so on. 

*homepage*: http://uspex-team.org/uspex/overview

version  |versionsuffix     |toolchain
---------|------------------|---------------
``9.4.4``|``-Python-2.7.12``|``intel/2016b``

<a anchor="utf8proc"/>
### *utf8proc*

utf8proc is a small, clean C library that provides Unicode normalization, case-folding,  and other operations for data in the UTF-8 encoding.

*homepage*: https://github.com/JuliaStrings/utf8proc

version  |toolchain
---------|--------------------------------------------------------
``2.2.0``|``GCCcore/6.4.0``
``2.3.0``|``GCCcore/8.2.0``
``2.5.0``|``GCCcore/10.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``2.6.1``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``
``2.7.0``|``GCCcore/11.3.0``

<a anchor="util-linux"/>
### *util-linux*

Set of Linux utilities

*homepage*: https://www.kernel.org/pub/linux/utils/util-linux

version   |toolchain
----------|--------------------------------------
``2.27.1``|``foss/2016a``, ``intel/2016a``
``2.28``  |``foss/2016a``, ``intel/2016a``
``2.28.1``|``intel/2016b``
``2.29``  |``foss/2016b``, ``intel/2016b``
``2.29.2``|``GCCcore/6.3.0``, ``intel/2017a``
``2.30``  |``GCCcore/6.4.0``
``2.30.1``|``GCCcore/6.3.0``, ``foss/2017a``
``2.31``  |``GCCcore/6.4.0``
``2.31.1``|``GCCcore/6.4.0``
``2.32``  |``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``2.32.1``|``GCCcore/7.3.0``
``2.33``  |``GCCcore/8.2.0``
``2.34``  |``GCCcore/8.3.0``
``2.35``  |``GCCcore/9.3.0``
``2.36``  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``
``2.37``  |``GCCcore/11.2.0``
``2.38``  |``GCCcore/11.3.0``
``2.38.1``|``GCCcore/12.2.0``

<a anchor="v"/>
### *V*


<a href="#v8">V8</a> - <a href="#vala">Vala</a> - <a href="#valgrind">Valgrind</a> - <a href="#vampir">Vampir</a> - <a href="#vampire">Vampire</a> - <a href="#vardict">VarDict</a> - <a href="#variant_tools">variant_tools</a> - <a href="#variantmetacaller">VariantMetaCaller</a> - <a href="#varscan">VarScan</a> - <a href="#vartools">vartools</a> - <a href="#vasp">VASP</a> - <a href="#vatools">VAtools</a> - <a href="#vawk">vawk</a> - <a href="#vbz-compression">VBZ-Compression</a> - <a href="#vcf-kit">VCF-kit</a> - <a href="#vcflib">vcflib</a> - <a href="#vcfnp">vcfnp</a> - <a href="#vcftools">VCFtools</a> - <a href="#vegas">VEGAS</a> - <a href="#velocyto">velocyto</a> - <a href="#velvet">Velvet</a> - <a href="#vep">VEP</a> - <a href="#verifybamid">verifyBamID</a> - <a href="#verse">VERSE</a> - <a href="#vesta">VESTA</a> - <a href="#viennarna">ViennaRNA</a> - <a href="#vim">Vim</a> - <a href="#virsorter">VirSorter</a> - <a href="#virtualgl">VirtualGL</a> - <a href="#virtuoso-opensource">Virtuoso-opensource</a> - <a href="#vispy">VisPy</a> - <a href="#vmd">VMD</a> - <a href="#vmtk">VMTK</a> - <a href="#voltools">voltools</a> - <a href="#vorbis-tools">vorbis-tools</a> - <a href="#voro++">Voro++</a> - <a href="#vsc-base">vsc-base</a> - <a href="#vsc-install">vsc-install</a> - <a href="#vsc-mympirun">vsc-mympirun</a> - <a href="#vsc-mympirun-scoop">vsc-mympirun-scoop</a> - <a href="#vsc-processcontrol">vsc-processcontrol</a> - <a href="#vsearch">VSEARCH</a> - <a href="#vt">vt</a> - <a href="#vtk">VTK</a> - <a href="#vtune">VTune</a> - <a href="#vv">VV</a> - <a href="#vxl">VXL</a>


<a anchor="v8"/>
### *V8*

R interface to Google's open source JavaScript engine

*homepage*: https://cran.r-project.org/web/packages/V8/

version  |versionsuffix|toolchain
---------|-------------|-------------------------------
``2.2``  |``-R-3.5.1`` |``foss/2018b``
``2.3``  |``-R-3.6.0`` |``foss/2019a``, ``intel/2019a``
``3.2.0``|``-R-3.6.2`` |``foss/2019b``
``3.4.0``|``-R-4.0.0`` |``foss/2020a``

<a anchor="vala"/>
### *Vala*

Vala is a programming language using modern high level abstractions without imposing additional runtime requirements and without using a different ABI compared to applications and libraries written in C.

*homepage*: https://wiki.gnome.org/Projects/Vala

version   |toolchain
----------|------------------
``0.52.4``|``GCCcore/10.3.0``

<a anchor="valgrind"/>
### *Valgrind*

Valgrind: Debugging and profiling tools

*homepage*: https://valgrind.org

version   |toolchain
----------|---------------------------------------------------------------------------------
``3.11.0``|``foss/2016a``, ``intel/2016a``
``3.13.0``|``foss/2017b``, ``foss/2018a``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``
``3.14.0``|``foss/2018b``
``3.16.1``|``gompi/2019b``, ``gompi/2020a``, ``gompi/2020b``, ``iimpi/2020a``
``3.17.0``|``gompi/2021a``
``3.18.1``|``gompi/2021b``, ``iimpi/2021b``
``3.19.0``|``gompi/2022a``
``3.20.0``|``gompi/2022a``

<a anchor="vampir"/>
### *Vampir*

The Vampir software tool provides an easy-to-use framework that enables  developers to quickly display and analyze arbitrary program behavior at any level of detail.  The tool suite implements optimized event analysis algorithms and customizable displays that  enable fast and interactive rendering of very complex performance monitoring data.

*homepage*: http://www.vampir.eu

version  |versionsuffix|toolchain
---------|-------------|----------
``8.4.1``|             |``system``
``8.4.1``|``-demo``    |``system``

<a anchor="vampire"/>
### *Vampire*

The Vampire Theorem Prover.

*homepage*: https://vprover.github.io/

version  |toolchain
---------|------------------
``4.5.1``|``GCCcore/10.2.0``

<a anchor="vardict"/>
### *VarDict*

VarDict is an ultra sensitive variant caller for both single  and paired sample variant calling from BAM files.

*homepage*: https://github.com/AstraZeneca-NGS/VarDict

version  |versionsuffix   |toolchain
---------|----------------|--------------
``1.5.7``|``-Perl-5.28.0``|``foss/2018b``

<a anchor="variant_tools"/>
### *variant_tools*

Variant tools is a software tool for the manipulation, annotation, selection, simulation, and  analysis of variants in the context of next-gen sequencing analysis. Unlike some other tools used for Next-Gen  sequencing analysis, variant tools is project based and provides a whole set of tools to manipulate and analyze  genetic variants.

*homepage*: https://vatlab.github.io/vat-docs/

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``3.1.3``|``-Python-3.7.4``|``foss/2019b``

<a anchor="variantmetacaller"/>
### *VariantMetaCaller*

VariantMetaCaller automatically integrates variant calling pipelines into a better performing  overall model that also predicts accurate variant probabilities.

*homepage*: http://bioinformatics.mit.bme.hu/VariantMetaCaller/

version|toolchain
-------|---------------
``1.0``|``intel/2017a``

<a anchor="varscan"/>
### *VarScan*

Variant calling and somatic mutation/CNV detection for next-generation sequencing data

*homepage*: https://github.com/dkoboldt/varscan

version  |versionsuffix     |toolchain
---------|------------------|----------
``2.3.6``|``-Java-1.7.0_80``|``system``
``2.4.1``|``-Java-1.7.0_80``|``system``
``2.4.4``|``-Java-1.8``     |``system``
``2.4.4``|``-Java-11``      |``system``

<a anchor="vartools"/>
### *vartools*

Command line utility that provides tools for processing   and analyzing astronomical time series data.

*homepage*: https://www.astro.princeton.edu/~jhartman/vartools.html

version |toolchain
--------|-------------------------------
``1.35``|``foss/2016b``, ``intel/2016b``

<a anchor="vasp"/>
### *VASP*

The Vienna Ab initio Simulation Package (VASP) is a computer program for atomic scale materials modelling, e.g. electronic structure calculations and quantum-mechanical molecular dynamics, from first principles.

*homepage*: http://www.vasp.at

version  |toolchain
---------|-------------------------
``5.4.1``|``intel/2016.02-GCC-4.9``

<a anchor="vatools"/>
### *VAtools*

VAtools is a python package that includes several tools to annotate VCF files  with data from other tools. 

*homepage*: https://vatools.readthedocs.io/en/latest/

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``3.0.1``|``-Python-3.6.6``|``foss/2018b``

<a anchor="vawk"/>
### *vawk*

An awk-like VCF parser  

*homepage*: https://github.com/cc2qe/vawk

version  |versionsuffix     |toolchain
---------|------------------|------------------
``0.0.1``|``-Python-2.7.18``|``GCCcore/10.2.0``

<a anchor="vbz-compression"/>
### *VBZ-Compression*

VBZ compression HDF5 plugin for nanopolish

*homepage*: https://github.com/nanoporetech/vbz_compression

version  |toolchain
---------|---------------
``1.0.1``|``gompi/2020b``

<a anchor="vcf-kit"/>
### *VCF-kit*

VCF-kit is a command-line based collection of utilities for performing analysis on  Variant Call Format (VCF) files.

*homepage*: https://github.com/AndersenLab/VCF-kit

version  |versionsuffix     |toolchain
---------|------------------|---------------
``0.1.6``|``-Python-2.7.15``|``intel/2018b``

<a anchor="vcflib"/>
### *vcflib*

vcflib provides methods to manipulate and interpret sequence variation as it can be  described by VCF. The Variant Call Format (VCF) is a flat-file, tab-delimited textual format intended  to concisely describe reference-indexed genetic variations between individuals.

*homepage*: https://github.com/vcflib/vcflib

version  |versionsuffix    |toolchain
---------|-----------------|------------------------------
``1.0.2``|                 |``GCC/10.2.0``, ``GCC/10.3.0``
``1.0.2``|``-Python-3.8.2``|``GCC/9.3.0``
``1.0.3``|``-R-4.1.0``     |``foss/2021a``
``1.0.3``|``-R-4.1.2``     |``foss/2021b``

<a anchor="vcfnp"/>
### *vcfnp*

Load data from a VCF (variant call format) file into numpy arrays,   and (optionally) from there into an HDF5 file.

*homepage*: https://github.com/alimanfoo/vcfnp

version  |versionsuffix     |toolchain
---------|------------------|--------------
``2.3.0``|``-Python-2.7.11``|``foss/2016a``

<a anchor="vcftools"/>
### *VCFtools*

The aim of VCFtools is to provide   easily accessible methods for working with complex   genetic variation data in the form of VCF files.

*homepage*: https://vcftools.github.io

version   |versionsuffix   |toolchain
----------|----------------|---------------------------------------------------------------------------------------------------------------------
``0.1.14``|``-Perl-5.22.1``|``foss/2016a``, ``intel/2016a``
``0.1.15``|``-Perl-5.24.0``|``foss/2016b``
``0.1.15``|``-Perl-5.26.0``|``foss/2017b``, ``intel/2017b``
``0.1.15``|``-Perl-5.26.1``|``foss/2018a``
``0.1.16``|                |``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/11.3.0``, ``GCC/8.3.0``, ``GCC/9.3.0``, ``iccifort/2019.5.281``
``0.1.16``|``-Perl-5.28.0``|``foss/2018b``, ``intel/2018b``

<a anchor="vegas"/>
### *VEGAS*

VEGAS (Versatile Gene-based Association Study) is  a free program for performing gene-based tests for association using  the results from genetic association studies 

*homepage*: http://gump.qimr.edu.au/VEGAS/

version   |toolchain
----------|----------
``0.8.27``|``system``

<a anchor="velocyto"/>
### *velocyto*

Velocyto is a library for the analysis of RNA velocity.

*homepage*: https://velocyto.org/velocyto.py/

version    |versionsuffix    |toolchain
-----------|-----------------|---------------
``0.17.17``|                 |``foss/2021a``
``0.17.17``|``-Python-3.8.2``|``intel/2020a``

<a anchor="velvet"/>
### *Velvet*

Sequence assembler for very short reads

*homepage*: http://www.ebi.ac.uk/~zerbino/velvet/

version   |versionsuffix   |toolchain
----------|----------------|-------------------------------------------------------------
``1.2.10``|``-mt-kmer_191``|``GCC/11.2.0``, ``GCC/8.3.0``, ``foss/2018a``, ``foss/2018b``
``1.2.10``|``-mt-kmer_37`` |``intel/2017a``

<a anchor="vep"/>
### *VEP*

Variant Effect Predictor (VEP) determines the effect of your  variants (SNPs, insertions, deletions, CNVs or structural variants) on genes,  transcripts, and protein sequence, as well as regulatory regions.  Includes EnsEMBL-XS, which provides pre-compiled replacements for frequently  used routines in VEP.

*homepage*: https://www.ensembl.org/info/docs/tools/vep

version  |versionsuffix   |toolchain
---------|----------------|-------------------------------
``93.4`` |``-Perl-5.26.1``|``intel/2018a``
``94.0`` |``-Perl-5.28.0``|``foss/2018b``
``94.5`` |``-Perl-5.26.0``|``foss/2017b``, ``intel/2017b``
``95.0`` |``-Perl-5.28.0``|``foss/2018b``
``96.0`` |``-Perl-5.28.1``|``foss/2019a``
``103.1``|                |``GCC/10.2.0``
``105``  |                |``GCC/11.2.0``
``107``  |                |``GCC/11.3.0``

<a anchor="verifybamid"/>
### *verifyBamID*

verifyBamID is a software that verifies whether the reads in particular file match previously known   genotypes for an individual (or group of individuals), and checks whether the reads are contaminated as a mixture of   two samples. verifyBamID can detect sample contamination and swaps when external genotypes are available. When   external genotypes are not available, verifyBamID still robustly detects sample swaps. 

*homepage*: http://csg.sph.umich.edu/kang/verifyBamID/index.html

version  |toolchain
---------|--------------
``1.1.3``|``foss/2016a``

<a anchor="verse"/>
### *VERSE*

A versatile and efficient RNA-Seq read counting tool

*homepage*: https://github.com/qinzhu/VERSE

version  |toolchain
---------|--------------
``0.1.5``|``foss/2016b``

<a anchor="vesta"/>
### *VESTA*

VESTA is a 3D visualization program for structured models,    volumetric data such as electron/nuclear densities,    and crystal morphologies. 

*homepage*: http://www.jp-minerals.org/vesta

version  |versionsuffix|toolchain
---------|-------------|----------
``3.5.8``|``-gtk3``    |``system``

<a anchor="viennarna"/>
### *ViennaRNA*

The Vienna RNA Package consists of a C code library and several stand-alone programs for the prediction and comparison of RNA secondary structures.

*homepage*: https://www.tbi.univie.ac.at/RNA/

version   |versionsuffix     |toolchain
----------|------------------|-------------------------------
``2.2.3`` |                  |``intel/2016b``
``2.3.4`` |                  |``foss/2016b``
``2.3.5`` |                  |``intel/2017a``
``2.4.10``|``-Python-2.7.15``|``foss/2018b``, ``intel/2018b``
``2.4.11``|``-Python-3.6.6`` |``foss/2018b``
``2.4.14``|``-Python-3.6.6`` |``foss/2018b``
``2.5.0`` |                  |``foss/2021b``
``2.5.1`` |                  |``foss/2021b``

<a anchor="vim"/>
### *Vim*

Vim is an advanced text editor that seeks to provide the power   of the de-facto Unix editor 'Vi', with a more complete feature set. 

*homepage*: http://www.vim.org

version     |versionsuffix     |toolchain
------------|------------------|------------------
``8.0``     |``-Python-2.7.11``|``foss/2016a``
``8.1.0483``|``-Python-2.7.15``|``foss/2018b``
``8.1.1209``|``-Python-3.7.2`` |``GCCcore/8.2.0``
``9.0.0950``|                  |``GCCcore/11.3.0``

<a anchor="virsorter"/>
### *VirSorter*

VirSorter: mining viral signal from microbial genomic data.

*homepage*: https://github.com/simroux/VirSorter

version     |versionsuffix   |toolchain
------------|----------------|--------------
``20160601``|``-Perl-5.22.1``|``foss/2016a``

<a anchor="virtualgl"/>
### *VirtualGL*

VirtualGL is an open source toolkit that gives any Linux or  Unix remote display software the ability to run OpenGL applications with full  hardware acceleration.

*homepage*: https://virtualgl.org/

version  |toolchain
---------|-----------------
``2.6.1``|``foss/2018b``
``2.6.2``|``GCCcore/9.3.0``
``3.0``  |``GCC/11.2.0``

<a anchor="virtuoso-opensource"/>
### *Virtuoso-opensource*

Virtuoso is a high-performance and scalable Multi-Model RDBMS, Data Integration Middleware, Linked Data Deployment, and HTTP Application Server Platform.

*homepage*: https://github.com/openlink/virtuoso-opensource

version    |toolchain
-----------|--------------
``7.2.6.1``|``GCC/10.3.0``

<a anchor="vispy"/>
### *VisPy*

VisPy is a high-performance interactive 2D/3D data visualization library  leveraging the computational power of modern Graphics Processing Units (GPUs) through the  OpenGL library to display very large datasets.

*homepage*: https://vispy.org

version  |toolchain
---------|-------------------------------
``0.6.6``|``foss/2020b``, ``intel/2020b``

<a anchor="vmd"/>
### *VMD*

VMD is a molecular visualization program for displaying, animating, and analyzing large biomolecular  systems using 3-D graphics and built-in scripting.

*homepage*: https://www.ks.uiuc.edu/Research/vmd

version     |versionsuffix     |toolchain
------------|------------------|----------------------------------
``1.9.3``   |``-Python-2.7.12``|``intel/2016b``
``1.9.3``   |``-Python-2.7.14``|``foss/2017b``, ``intel/2018a``
``1.9.3``   |``-Python-2.7.15``|``intel/2018b``
``1.9.4a43``|``-Python-3.7.4`` |``fosscuda/2019b``
``1.9.4a51``|                  |``foss/2020b``, ``fosscuda/2020b``
``1.9.4a57``|                  |``foss/2022a``

<a anchor="vmtk"/>
### *VMTK*

vmtk is a collection of libraries and tools for 3D reconstruction, geometric analysis, mesh generation and surface data analysis for image-based modeling of blood vessels. 

*homepage*: https://vmtk.org

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.4.0``|``-Python-3.7.4``|``foss/2019b``

<a anchor="voltools"/>
### *voltools*

CUDA-accelerated numpy 3D affine transformations

*homepage*: https://github.com/the-lay/voltools

version  |toolchain
---------|--------------
``0.4.2``|``foss/2020b``

<a anchor="vorbis-tools"/>
### *vorbis-tools*

Command-line tools for creating and playing Ogg Vorbis files.

*homepage*: https://xiph.org/vorbis/

version  |toolchain
---------|----------------------------------------------------------
``1.4.2``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``

<a anchor="voro++"/>
### *Voro++*

Voro++ is a software library for carrying out three-dimensional computations of the Voronoi tessellation. A distinguishing feature of the Voro++ library is that it carries out cell-based calculations, computing the Voronoi cell for each particle individually. It is particularly well-suited for applications that rely on cell-based statistics, where features of Voronoi cells (eg. volume, centroid, number of faces) can be used to analyze a system of particles.

*homepage*: http://math.lbl.gov/voro++/

version  |toolchain
---------|------------------------------------------------------------------------------------------------------------------------------------------------------------------
``0.4.6``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/6.4.0``, ``GCCcore/9.3.0``, ``foss/2016a``, ``foss/2019b``, ``intel/2016a``, ``intel/2019b``

<a anchor="vsc-base"/>
### *vsc-base*

Basic Python libraries used by UGent's HPC group

*homepage*: http://hpcugent.github.com/vsc-base/

version   |versionsuffix     |toolchain
----------|------------------|---------------
``1.6.9`` |                  |``system``
``1.7.3`` |                  |``system``
``2.0.4`` |                  |``system``
``2.1.2`` |                  |``system``
``2.4.2`` |                  |``system``
``2.4.17``|``-Python-2.7.11``|``intel/2016a``
``2.5.1`` |                  |``system``
``2.5.1`` |``-Python-2.7.11``|``intel/2016a``
``2.5.1`` |``-Python-2.7.12``|``intel/2016b``
``2.5.8`` |                  |``system``
``2.8.0`` |                  |``system``
``2.8.1`` |                  |``system``
``2.8.3`` |                  |``system``

<a anchor="vsc-install"/>
### *vsc-install*

vsc-install provides shared setuptools functions and classes for Python libraries  developed by UGent's HPC group

*homepage*: https://pypi.python.org/pypi/vsc-install

version    |versionsuffix     |toolchain
-----------|------------------|---------------
``0.9.18`` |``-Python-2.7.11``|``intel/2016a``
``0.10.6`` |                  |``system``
``0.10.6`` |``-Python-2.7.11``|``intel/2016a``
``0.10.11``|``-Python-2.7.11``|``intel/2016a``
``0.10.11``|``-Python-2.7.12``|``intel/2016b``
``0.10.25``|                  |``system``
``0.10.26``|                  |``system``
``0.11.1`` |                  |``system``
``0.11.2`` |                  |``system``

<a anchor="vsc-mympirun"/>
### *vsc-mympirun*

mympirun is a tool to make it easier for users of HPC clusters to run MPI programs with good performance.

*homepage*: https://github.com/hpcugent/vsc-mympirun

version    |versionsuffix                     |toolchain
-----------|----------------------------------|---------------
``3.2.1``  |                                  |``system``
``3.3.0``  |                                  |``system``
``3.4.2``  |                                  |``system``
``3.4.2``  |``-Python-2.7.11-vsc-base-2.4.17``|``intel/2016a``
``3.4.2``  |``-Python-2.7.11-vsc-base-2.5.1`` |``intel/2016a``
``3.4.2``  |``-vsc-base-2.4.2``               |``system``
``3.4.3``  |``-Python-2.7.12``                |``intel/2016b``
``4.0.0``  |                                  |``system``
``4.0.0b0``|                                  |``system``
``4.0.1``  |                                  |``system``
``4.0.2``  |                                  |``system``
``4.1.4``  |                                  |``system``
``4.1.5``  |                                  |``system``
``4.1.6``  |                                  |``system``
``4.1.8``  |                                  |``system``
``4.1.9``  |                                  |``system``
``5.1.0``  |                                  |``system``
``5.2.0``  |                                  |``system``
``5.2.2``  |                                  |``system``
``5.2.3``  |                                  |``system``
``5.2.4``  |                                  |``system``
``5.2.5``  |                                  |``system``
``5.2.6``  |                                  |``system``
``5.2.7``  |                                  |``system``
``5.2.9``  |                                  |``system``
``5.2.10`` |                                  |``system``
``5.2.11`` |                                  |``system``

<a anchor="vsc-mympirun-scoop"/>
### *vsc-mympirun-scoop*

VSC-tools is a set of Python libraries and scripts that are commonly used within HPC-UGent.

*homepage*: http://hpcugent.github.com/vsc-mympirun/

version  |versionsuffix     |toolchain
---------|------------------|---------------
``3.4.1``|``-Python-2.7.12``|``intel/2016b``

<a anchor="vsc-processcontrol"/>
### *vsc-processcontrol*

vsc-processcontrol is a module to abstract process control like scheduler settings and affinity  from actual implementations like vsc.affinity and psutil.

*homepage*: http://hpcugent.github.com/vsc-processcontrol/

version|versionsuffix      |toolchain
-------|-------------------|----------
``1.0``|                   |``system``
``1.0``|``-vsc-base-2.1.2``|``system``

<a anchor="vsearch"/>
### *VSEARCH*

VSEARCH supports de novo and reference based chimera detection,  clustering, full-length and prefix dereplication, rereplication,  reverse complementation, masking, all-vs-all pairwise global alignment,  exact and global alignment searching, shuffling, subsampling and sorting.  It also supports FASTQ file analysis, filtering,  conversion and merging of paired-end reads.

*homepage*: https://github.com/torognes/vsearch

version   |toolchain
----------|----------------------------------------
``2.9.1`` |``foss/2018b``
``2.13.4``|``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``2.15.0``|``GCC/9.3.0``
``2.18.0``|``GCC/10.2.0``
``2.21.1``|``GCC/10.3.0``

<a anchor="vt"/>
### *vt*

A tool set for short variant discovery in genetic sequence data.

*homepage*: http://genome.sph.umich.edu/wiki/Vt

version    |toolchain
-----------|-----------------------------------------------------------------------------
``0.57721``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``GCC/9.3.0``, ``foss/2018b``

<a anchor="vtk"/>
### *VTK*

The Visualization Toolkit (VTK) is an open-source, freely available software system for  3D computer graphics, image processing and visualization. VTK consists of a C++ class library and several  interpreted interface layers including Tcl/Tk, Java, and Python. VTK supports a wide variety of visualization  algorithms including: scalar, vector, tensor, texture, and volumetric methods; and advanced modeling techniques  such as: implicit modeling, polygon reduction, mesh smoothing, cutting, contouring, and Delaunay triangulation.

*homepage*: https://www.vtk.org

version      |versionsuffix     |toolchain
-------------|------------------|---------------------------------------------------
``6.3.0``    |``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``6.3.0``    |``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``7.0.0``    |``-Python-2.7.12``|``intel/2016b``
``7.1.0``    |``-Python-2.7.12``|``intel/2016b``
``7.1.1``    |``-Python-2.7.13``|``intel/2017a``
``8.0.1``    |``-Python-2.7.14``|``foss/2017b``, ``intel/2017b``
``8.1.0``    |``-Python-2.7.14``|``foss/2018a``, ``intel/2018a``
``8.1.0``    |``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``8.1.1``    |``-Python-2.7.14``|``intel/2018a``
``8.1.1``    |``-Python-2.7.15``|``foss/2018b``
``8.1.1``    |``-Python-3.6.6`` |``foss/2018b``, ``fosscuda/2018b``
``8.2.0``    |                  |``foss/2021a``
``8.2.0``    |``-Python-2.7.15``|``foss/2019a``
``8.2.0``    |``-Python-3.7.2`` |``foss/2019a``, ``intel/2019a``
``8.2.0``    |``-Python-3.7.4`` |``foss/2019b``
``8.2.0``    |``-Python-3.8.2`` |``foss/2020a``, ``fosscuda/2020a``, ``intel/2020a``
``9.0.1``    |                  |``foss/2020b``, ``foss/2021a``
``9.1.0``    |                  |``foss/2021b``
``9.2.0.rc2``|                  |``foss/2022a``
``9.2.2``    |                  |``foss/2022a``

<a anchor="vtune"/>
### *VTune*

Intel VTune Amplifier XE is the premier performance profiler for C, C++, C#, Fortran,  Assembly and Java.

*homepage*: https://software.intel.com/en-us/vtune

version          |toolchain
-----------------|----------
``2013_update6`` |``system``
``2013_update8`` |``system``
``2013_update9`` |``system``
``2013_update10``|``system``
``2013_update11``|``system``
``2013_update12``|``system``
``2016_update3`` |``system``
``2017``         |``system``
``2017_update1`` |``system``
``2017_update2`` |``system``
``2017_update3`` |``system``
``2018_update1`` |``system``
``2018_update2`` |``system``
``2018_update3`` |``system``
``2019_update2`` |``system``
``2019_update3`` |``system``
``2019_update5`` |``system``
``2020_update3`` |``system``
``2021.6.0``     |``system``
``2021.9.0``     |``system``
``2022.0.0``     |``system``
``2022.2.0``     |``system``
``2022.3.0``     |``system``

<a anchor="vv"/>
### *VV*

VV is an open-source and cross platform image viewer, designed for fast and  simple visualization of spatio-temporal images: 2D, 2D+t, 3D and 3D+t (or 4D)  images. Only the command-line (clitk) tools are build.

*homepage*: https://www.creatis.insa-lyon.fr/rio/vv

version       |versionsuffix    |toolchain
--------------|-----------------|--------------
``2018.09.19``|``-Python-3.6.6``|``foss/2018b``

<a anchor="vxl"/>
### *VXL*

A multi-platform collection of C++ software libraries for Computer Vision and Image Understanding.

*homepage*: https://sf.net/projects/vxl

version   |toolchain
----------|--------------
``1.18.0``|``foss/2018a``

<a anchor="w"/>
### *W*


<a href="#wandb">wandb</a> - <a href="#wannier90">Wannier90</a> - <a href="#wanniertools">WannierTools</a> - <a href="#wayland">Wayland</a> - <a href="#waylandpp">Waylandpp</a> - <a href="#wct">WCT</a> - <a href="#wcwidth">wcwidth</a> - <a href="#webin-cli">webin-cli</a> - <a href="#webkitgtk+">WebKitGTK+</a> - <a href="#websocket++">WebSocket++</a> - <a href="#weka">WEKA</a> - <a href="#wget">wget</a> - <a href="#wgsim">wgsim</a> - <a href="#wham">WHAM</a> - <a href="#whatshap">WhatsHap</a> - <a href="#wheel">wheel</a> - <a href="#wien2k">WIEN2k</a> - <a href="#wildmagic">WildMagic</a> - <a href="#winnowmap">Winnowmap</a> - <a href="#wisecondorx">WisecondorX</a> - <a href="#wisexome">WISExome</a> - <a href="#wkhtmltopdf">wkhtmltopdf</a> - <a href="#worker">worker</a> - <a href="#wpebackend-fdo">wpebackend-fdo</a> - <a href="#wps">WPS</a> - <a href="#wrf">WRF</a> - <a href="#wrf-fire">WRF-Fire</a> - <a href="#wrf-python">wrf-python</a> - <a href="#wtdbg2">wtdbg2</a> - <a href="#wxpropertygrid">wxPropertyGrid</a> - <a href="#wxpython">wxPython</a> - <a href="#wxwidgets">wxWidgets</a>


<a anchor="wandb"/>
### *wandb*

CLI and Python API for Weights and Biases (wandb), a tool for visualizing and tracking your machine learning experiments.

*homepage*: https://www.wandb.com/

version   |versionsuffix    |toolchain
----------|-----------------|------------------
``0.8.30``|``-Python-3.7.4``|``gcccuda/2019b``
``0.13.4``|                 |``GCCcore/11.3.0``
``0.13.6``|                 |``GCC/11.3.0``

<a anchor="wannier90"/>
### *Wannier90*

A tool for obtaining maximally-localised Wannier functions

*homepage*: http://www.wannier.org

version    |versionsuffix|toolchain
-----------|-------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.2``    |             |``intel/2016.02-GCC-4.9``
``2.0.1``  |             |``intel/2016.02-GCC-4.9``, ``intel/2016a``
``2.0.1.1``|``-abinit``  |``intel/2018b``
``2.1.0``  |             |``foss/2017b``, ``intel/2017a``, ``intel/2017b``
``3.0.0``  |             |``intel/2018b``
``3.1.0``  |             |``foss/2020b``, ``foss/2021a``, ``foss/2021b``, ``foss/2022a``, ``gomkl/2021a``, ``intel/2020a``, ``intel/2020b``, ``intel/2021a``, ``intel/2021b``, ``intel/2022a``

<a anchor="wanniertools"/>
### *WannierTools*

WannierTools is an open source software that studies the physical properties of given tight-binding  model.

*homepage*: http://www.wanniertools.com

version  |toolchain
---------|---------------
``2.3.0``|``intel/2018a``
``2.5.1``|``intel/2020b``

<a anchor="wayland"/>
### *Wayland*

Wayland is a project to define a protocol for a compositor to talk to  its clients as well as a library implementation of the protocol.  The  compositor can be a standalone display server running on Linux kernel  modesetting and evdev input devices, an X application, or a wayland  client itself.  The clients can be traditional applications, X servers  (rootless or fullscreen) or other display servers. 

*homepage*: https://wayland.freedesktop.org/

version   |toolchain
----------|--------------------------------------
``1.20.0``|``GCCcore/11.3.0``
``1.21.0``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="waylandpp"/>
### *Waylandpp*

Wayland is an object oriented display protocol, which features request and events. Requests can be seen as method calls on certain objects, whereas events can be seen as signals of an object. This makes the Wayland protocol a perfect candidate for a C++ binding.  The goal of this library is to create such a C++ binding for Wayland using the most modern C++ technology currently available, providing an easy to use C++ API to Wayland.

*homepage*: https://github.com/NilsBrause/waylandpp

version  |toolchain
---------|--------------------------------------
``1.0.0``|``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="wct"/>
### *WCT*

NOAA's Weather and Climate Toolkit (WCT) is free, platform independent software  distributed from NOAA's National Centers for Environmental Information (NCEI).  The WCT allows the visualization and data export of weather and climate data,  including Radar, Satellite and Model data. The WCT also provides access to  weather/climate web services provided from NCEI and other organizations. 

*homepage*: https://www.ncdc.noaa.gov/wct/index.php

version  |versionsuffix|toolchain
---------|-------------|----------
``4.6.0``|``-Java-11`` |``system``

<a anchor="wcwidth"/>
### *wcwidth*

wcwidth is a low-level Python library to simplify Terminal emulation.

*homepage*: https://github.com/jquast/wcwidth

version  |versionsuffix     |toolchain
---------|------------------|-------------------------------
``0.1.7``|``-Python-2.7.11``|``foss/2016a``
``0.1.7``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``0.1.7``|``-Python-3.5.1`` |``foss/2016a``
``0.1.7``|``-Python-3.5.2`` |``intel/2016b``

<a anchor="webin-cli"/>
### *webin-cli*

The Webin command line submission interface can be used to validate,  upload and submit files to the European Nucleotide Archive (ENA)

*homepage*: https://github.com/enasequence/webin-cli

version  |toolchain
---------|----------
``1.8.9``|``system``

<a anchor="webkitgtk+"/>
### *WebKitGTK+*

WebKitGTK+ is a full-featured port of the WebKit rendering engine, suitable for projects requiring any kind of web integration, from hybrid HTML/CSS applications to full-fledged web browsers. It offers WebKit's full functionality and is useful in a wide range of systems from desktop computers to embedded systems like phones, tablets, and televisions.

*homepage*: https://webkitgtk.org/

version   |toolchain
----------|-----------------------------
``2.24.1``|``GCC/8.2.0-2.31.1``
``2.27.4``|``GCC/10.3.0``, ``GCC/8.3.0``
``2.37.1``|``GCC/11.2.0``

<a anchor="websocket++"/>
### *WebSocket++*

WebSocket++ is an open source (BSD license) header only C++ library  that implements RFC6455 The WebSocket Protocol. 

*homepage*: https://www.zaphoyd.com/websocketpp/

version  |toolchain
---------|---------------
``0.8.1``|``gompi/2019a``

<a anchor="weka"/>
### *WEKA*

Weka is a collection of machine learning algorithms for data mining tasks.   The algorithms can either be applied directly to a dataset or called from your own Java code.   Weka contains tools for data pre-processing, classification, regression, clustering,   association rules, and visualization. It is also well-suited for developing new machine   learning schemes.

*homepage*: http://www.cs.waikato.ac.nz/ml/weka/index.html

version   |versionsuffix     |toolchain
----------|------------------|----------
``3.6.12``|``-Java-1.7.0_80``|``system``
``3.7.0`` |``-Java-1.7.0_80``|``system``

<a anchor="wget"/>
### *wget*

GNU Wget is a free software package for retrieving files using HTTP, HTTPS and FTP,  the most widely-used Internet protocols. It is a non-interactive commandline tool,  so it may easily be called from scripts, cron jobs, terminals without X-Windows support, etc.

*homepage*: https://www.gnu.org/software/wget

version   |toolchain
----------|-------------------------------------
``1.17.1``|``foss/2016a``, ``foss/2016b``
``1.19.4``|``GCCcore/6.4.0``
``1.20.1``|``GCCcore/7.3.0``, ``GCCcore/8.3.0``
``1.20.3``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``1.21.1``|``GCCcore/10.3.0``
``1.21.2``|``GCCcore/11.2.0``

<a anchor="wgsim"/>
### *wgsim*

Wgsim is a small tool for simulating sequence reads from a reference genome.

*homepage*: https://github.com/lh3/wgsim/

version     |toolchain
------------|------------------------------
``20111017``|``GCC/10.2.0``, ``GCC/11.2.0``

<a anchor="wham"/>
### *WHAM*

An implementation of WHAM:  the Weighted Histogram Analysis Method

*homepage*: http://membrane.urmc.rochester.edu/?page_id=126

version     |versionsuffix|toolchain
------------|-------------|---------------
``2.0.9.1`` |             |``intel/2019a``
``2.0.9.1`` |``-kj_mol``  |``intel/2019a``
``2.0.10.2``|             |``intel/2020a``
``2.0.10.2``|``-kj_mol``  |``intel/2020a``

<a anchor="whatshap"/>
### *WhatsHap*

WhatsHap is a software for phasing genomic variants using DNA sequencing reads, also called read-based phasing or haplotype assembly. It is especially suitable for long reads, but works also well with short reads.

*homepage*: https://whatshap.readthedocs.io

version|toolchain
-------|------------------------------
``1.1``|``foss/2020b``, ``foss/2021a``
``1.4``|``foss/2021b``

<a anchor="wheel"/>
### *wheel*

A built-package format for Python.

*homepage*: https://pypi.python.org/pypi/wheel

version   |versionsuffix     |toolchain
----------|------------------|------------------------------------------------------------------------
``0.29.0``|``-Python-2.7.11``|``foss/2016a``
``0.29.0``|``-Python-3.5.1`` |``foss/2016a``
``0.30.0``|``-Python-2.7.14``|``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``0.30.0``|``-Python-3.6.3`` |``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``0.30.0``|``-Python-3.6.4`` |``foss/2018a``, ``intel/2018a``
``0.31.0``|``-Python-2.7.14``|``fosscuda/2018a``, ``intel/2018a``
``0.31.0``|``-Python-3.6.4`` |``foss/2018a``, ``fosscuda/2018a``
``0.31.1``|``-Python-2.7.15``|``fosscuda/2018b``
``0.31.1``|``-Python-3.6.6`` |``foss/2018b``, ``fosscuda/2018b``

<a anchor="wien2k"/>
### *WIEN2k*

The program package WIEN2k allows to perform electronic structure calculations of solids using density functional theory (DFT). It is based on the full-potential (linearized) augmented plane-wave ((L)APW) + local orbitals (lo) method, one among the most accurate schemes for band structure calculations. WIEN2k is an all-electron scheme including relativistic effects and has many features.

*homepage*: http://www.wien2k.at/

version |toolchain
--------|------------------------------------------------
``17.1``|``foss/2018a``, ``gimkl/2017a``, ``intel/2018a``
``18.1``|``foss/2018a``, ``gimkl/2017a``, ``intel/2018a``
``19.1``|``intel/2019a``
``19.2``|``intel/2020b``
``21.1``|``intel/2021a``

<a anchor="wildmagic"/>
### *WildMagic*

Wild Magic 5.17

*homepage*: https://www.geometrictools.com/index.html

version |toolchain
--------|----------------------------------
``5.17``|``GCCcore/10.3.0``, ``foss/2018b``

<a anchor="winnowmap"/>
### *Winnowmap*

Winnowmap is a long-read mapping algorithm, and a result of our exploration  into superior minimizer sampling techniques.

*homepage*: https://github.com/marbl/Winnowmap

version|toolchain
-------|-------------
``1.0``|``GCC/8.3.0``

<a anchor="wisecondorx"/>
### *WisecondorX*

WisecondorX -- an evolved WISECONDOR

*homepage*: https://github.com/CenterForMedicalGeneticsGhent/WisecondorX

version  |versionsuffix    |toolchain
---------|-----------------|--------------
``1.1.6``|``-Python-3.8.2``|``foss/2020a``

<a anchor="wisexome"/>
### *WISExome*

A within-sample comparison approach to detect copy number variations in whole exome sequencing data

*homepage*: https://github.com/VUmcCGP/wisexome

version     |versionsuffix     |toolchain
------------|------------------|---------------
``20180517``|``-Python-2.7.14``|``intel/2018a``

<a anchor="wkhtmltopdf"/>
### *wkhtmltopdf*

wkhtmltopdf and wkhtmltoimage are open source (LGPLv3) command line tools to render   HTML into PDF and various image formats using the Qt WebKit rendering engine. These run entirely headless   and do not require a display or display service.

*homepage*: http://wkhtmltopdf.org/

version   |versionsuffix    |toolchain
----------|-----------------|----------
``0.12.3``|``-Linux-x86_64``|``system``

<a anchor="worker"/>
### *worker*

The Worker framework has been developed to help deal with parameter exploration experiments  that would otherwise result in many jobs, forcing the user resort to scripting to retain her sanity;  see also https://vscentrum.be/neutral/documentation/cluster-doc/running-jobs/worker-framework.

*homepage*: https://github.com/gjbex/worker

version   |versionsuffix   |toolchain
----------|----------------|------------------------------
``1.6.4`` |                |``intel/2016a``
``1.6.5`` |                |``intel/2016a``
``1.6.6`` |                |``intel/2016b``
``1.6.7`` |``-intel-2016b``|``system``
``1.6.7`` |``-intel-2017a``|``system``
``1.6.7`` |``-intel-2017b``|``system``
``1.6.8`` |``-intel-2018a``|``system``
``1.6.8`` |``-intel-2018b``|``system``
``1.6.11``|                |``intel/2019b``
``1.6.12``|                |``foss/2019a``, ``foss/2021b``
``1.6.13``|                |``iimpi/2021b``

<a anchor="wpebackend-fdo"/>
### *wpebackend-fdo*

WPE WebKit allows embedders to create simple and performant  systems based on Web platform technologies. It is a WebKit port designed with  flexibility and hardware acceleration in mind, leveraging common 3D graphics  APIs for best performance.

*homepage*: https://wpewebkit.org/

version   |toolchain
----------|------------------
``1.13.1``|``GCCcore/11.2.0``

<a anchor="wps"/>
### *WPS*

WRF Preprocessing System (WPS) for WRF. The Weather Research and Forecasting (WRF) Model is  a next-generation mesoscale numerical weather prediction system designed to serve both operational  forecasting and atmospheric research needs.

*homepage*: http://www.wrf-model.org

version  |versionsuffix|toolchain
---------|-------------|---------------
``3.8.0``|``-dmpar``   |``intel/2016a``
``3.9.1``|``-dmpar``   |``foss/2020b``
``4.0.1``|``-dmpar``   |``intel/2018b``
``4.0.2``|``-dmpar``   |``foss/2018b``
``4.1``  |``-dmpar``   |``intel/2019b``
``4.2``  |``-dmpar``   |``foss/2020b``
``4.3.1``|``-dmpar``   |``foss/2021a``
``4.4``  |``-dmpar``   |``foss/2022a``

<a anchor="wrf"/>
### *WRF*

The Weather Research and Forecasting (WRF) Model is a next-generation mesoscale  numerical weather prediction system designed to serve both operational forecasting and atmospheric  research needs.

*homepage*: https://www.wrf-model.org

version    |versionsuffix|toolchain
-----------|-------------|----------------------------------------------------------------
``3.8.0``  |``-dmpar``   |``intel/2016a``, ``intel/2016b``
``3.9.1.1``|``-dmpar``   |``foss/2020a``, ``foss/2020b``, ``intel/2020a``, ``intel/2020b``
``4.0.1``  |``-dmpar``   |``intel/2018b``
``4.0.2``  |``-dmpar``   |``foss/2018b``
``4.1.3``  |``-dm+sm``   |``intel/2019b``
``4.1.3``  |``-dmpar``   |``foss/2019b``, ``foss/2020a``, ``intel/2019b``
``4.2.2``  |``-dmpar``   |``foss/2020b``
``4.3``    |``-dmpar``   |``foss/2021a``
``4.4``    |``-dmpar``   |``foss/2022a``

<a anchor="wrf-fire"/>
### *WRF-Fire*

WRF-Fire combines the Weather Research and Forecasting model (WRF) with a fire code implementing a  surface fire behavior model, called SFIRE, based on semi-empirical formulas calculate the rate of spread of the fire  line (the interface between burning and unignited fuel) based on fuel properties, wind velocities from WRF, and  terrain slope. The fire spread is implemented by the level set method.

*homepage*: http://www.openwfm.org/wiki/WRF-SFIRE

version     |versionsuffix|toolchain
------------|-------------|---------------
``20170221``|``-dmpar``   |``intel/2016b``

<a anchor="wrf-python"/>
### *wrf-python*

A collection of diagnostic and interpolation routines for use with output from  the Weather Research and Forecasting (WRF-ARW) Model.

*homepage*: https://wrf-python.readthedocs.io

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``1.2.0``|``-Python-3.6.4``|``intel/2018a``
``1.3.1``|``-Python-3.6.6``|``intel/2018b``

<a anchor="wtdbg2"/>
### *wtdbg2*

Wtdbg2 is a de novo sequence assembler for long noisy reads produced by PacBio  or Oxford Nanopore Technologies (ONT). It assembles raw reads without error  correction and then builds the consensus from intermediate assembly output. 

*homepage*: https://github.com/ruanjue/wtdbg2

version|toolchain
-------|-------------------------------------
``2.3``|``GCC/7.3.0-2.30``
``2.5``|``GCCcore/11.2.0``, ``GCCcore/9.3.0``

<a anchor="wxpropertygrid"/>
### *wxPropertyGrid*

wxPropertyGrid is a property sheet control for wxWidgets. In other words, it is  a specialized two-column grid for editing properties such as strings, numbers, flagsets, string arrays, and colours.

*homepage*: http://wxpropgrid.sourceforge.net/

version   |toolchain
----------|-------------
``1.4.15``|``GCC/4.9.2``

<a anchor="wxpython"/>
### *wxPython*

Wraps the wxWidgets C++ toolkit and provides access to the user interface portions of the wxWidgets  API, enabling Python applications to have a native GUI on Windows, Macs or Unix systems, with a native look and feel  and requiring very little (if any) platform specific code.

*homepage*: https://www.wxpython.org/

version        |versionsuffix     |toolchain
---------------|------------------|-------------------------------
``3.0.2.0``    |``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``3.0.2.0``    |``-Python-2.7.12``|``intel/2016b``
``3.0.2.0``    |``-Python-2.7.13``|``intel/2017a``
``3.0.2.0``    |``-Python-2.7.14``|``intel/2017b``
``3.0.2.0``    |``-Python-2.7.15``|``foss/2018b``
``4.0.4``      |``-Python-2.7.15``|``GCC/8.2.0-2.31.1``
``4.0.4``      |``-Python-3.7.2`` |``GCC/8.2.0-2.31.1``
``4.0.7.post2``|``-Python-3.7.4`` |``GCC/8.3.0``
``4.1.1``      |                  |``foss/2021a``

<a anchor="wxwidgets"/>
### *wxWidgets*

wxWidgets is a C++ library that lets developers create applications for Windows, Mac OS X, Linux and other platforms with a single code base. It has popular language bindings for Python, Perl, Ruby and many other languages, and unlike other cross-platform toolkits, wxWidgets gives applications a truly native look and feel because it uses the platform's native API rather than emulating the GUI.

*homepage*: https://www.wxwidgets.org

version  |toolchain
---------|------------------------------
``3.0.3``|``foss/2018a``
``3.0.4``|``GCC/8.2.0-2.31.1``
``3.1.3``|``GCC/8.3.0``
``3.1.4``|``GCC/10.2.0``
``3.1.5``|``GCC/10.3.0``, ``GCC/11.2.0``
``3.2.1``|``GCC/11.3.0``

<a anchor="x"/>
### *X*


<a href="#x11">X11</a> - <a href="#x13as">x13as</a> - <a href="#x264">x264</a> - <a href="#x265">x265</a> - <a href="#xalt">XALT</a> - <a href="#xarray">xarray</a> - <a href="#xbitmaps">xbitmaps</a> - <a href="#xcb-proto">xcb-proto</a> - <a href="#xcb-util">xcb-util</a> - <a href="#xcb-util-image">xcb-util-image</a> - <a href="#xcb-util-keysyms">xcb-util-keysyms</a> - <a href="#xcb-util-renderutil">xcb-util-renderutil</a> - <a href="#xcb-util-wm">xcb-util-wm</a> - <a href="#xcell">xCell</a> - <a href="#xcfun">XCFun</a> - <a href="#xcrysden">XCrySDen</a> - <a href="#xerces-c++">Xerces-C++</a> - <a href="#xesmf">xESMF</a> - <a href="#xextproto">xextproto</a> - <a href="#xf86vidmodeproto">xf86vidmodeproto</a> - <a href="#xgboost">XGBoost</a> - <a href="#xgrafix">XGrafix</a> - <a href="#xineramaproto">xineramaproto</a> - <a href="#xkeyboardconfig">XKeyboardConfig</a> - <a href="#xlsxwriter">XlsxWriter</a> - <a href="#xmds2">XMDS2</a> - <a href="#xmipp">Xmipp</a> - <a href="#xmitgcm">xmitgcm</a> - <a href="#xml-compile">XML-Compile</a> - <a href="#xml-libxml">XML-LibXML</a> - <a href="#xml-parser">XML-Parser</a> - <a href="#xmlf90">xmlf90</a> - <a href="#xmlsec">XMLSec</a> - <a href="#xmlstarlet">XMLStarlet</a> - <a href="#xonsh">xonsh</a> - <a href="#xorg-macros">xorg-macros</a> - <a href="#xplor-nih">XPLOR-NIH</a> - <a href="#xprop">xprop</a> - <a href="#xproto">xproto</a> - <a href="#xsd">XSD</a> - <a href="#xtandem">XTandem</a> - <a href="#xtb">xtb</a> - <a href="#xtensor">xtensor</a> - <a href="#xtrans">xtrans</a> - <a href="#xvfb">Xvfb</a> - <a href="#xxd">xxd</a> - <a href="#xxhash">xxHash</a> - <a href="#xz">XZ</a>


<a anchor="x11"/>
### *X11*

The X Window System (X11) is a windowing system for bitmap displays

*homepage*: https://www.x.org

version     |toolchain
------------|--------------------------------------------------
``20160819``|``GCCcore/5.4.0``, ``foss/2016b``, ``intel/2016b``
``20170129``|``GCCcore/6.3.0``, ``gimkl/2017a``
``20170314``|``GCCcore/6.3.0``
``20171023``|``GCCcore/6.4.0``
``20180131``|``GCCcore/6.4.0``
``20180604``|``GCCcore/7.3.0``
``20190311``|``GCCcore/8.2.0``
``20190717``|``GCCcore/8.3.0``
``20200222``|``GCCcore/9.3.0``
``20201008``|``GCCcore/10.2.0``
``20210518``|``GCCcore/10.3.0``
``20210802``|``GCCcore/11.2.0``
``20220504``|``GCCcore/11.3.0``
``20221110``|``GCCcore/12.2.0``

<a anchor="x13as"/>
### *x13as*

X-13ARIMA-SEATS is seasonal adjustment software produced, distributed,  and maintained by the Census Bureau.  Features of X-13ARIMA-SEATS include:     - Extensive time series modeling and model selection capabilities for        linear regression models with ARIMA errors (regARIMA models);     - The capability to generate ARIMA model-based seasonal adjustment        using a version of the SEATS software originally developed by Victor        Gómez and Agustín Maravall at the Bank of Spain, as well as nonparametric        adjustments from the X-11 procedure;     - Diagnostics of the quality and stability of the adjustments        achieved under the options selected;     - The ability to efficiently process many series at once. 

*homepage*: https://www.census.gov/data/software/x13as.About_X-13.html

version    |toolchain
-----------|--------------------------------------
``1-1-b59``|``GCCcore/10.2.0``, ``GCCcore/11.2.0``

<a anchor="x264"/>
### *x264*

x264 is a free software library and application for encoding video streams  into the H.264/MPEG-4 AVC compression format, and is released under the  terms of the GNU GPL. 

*homepage*: https://www.videolan.org/developers/x264.html

version     |toolchain
------------|---------------------------------
``20160114``|``gimkl/2.11.5``, ``intel/2016a``
``20160430``|``foss/2016a``, ``intel/2016a``
``20160614``|``foss/2016b``, ``intel/2016b``
``20170406``|``gimkl/2017a``
``20170721``|``GCCcore/6.4.0``
``20170913``|``intel/2017a``
``20171217``|``foss/2017b``, ``intel/2017b``
``20180128``|``GCCcore/6.4.0``
``20180325``|``GCCcore/6.4.0``
``20181203``|``GCCcore/7.3.0``
``20190413``|``GCCcore/8.2.0``
``20190925``|``GCCcore/8.3.0``
``20191217``|``GCCcore/9.3.0``
``20201026``|``GCCcore/10.2.0``
``20210414``|``GCCcore/10.3.0``
``20210613``|``GCCcore/11.2.0``
``20220620``|``GCCcore/11.3.0``

<a anchor="x265"/>
### *x265*

x265 is a free software library and application for encoding video streams  into the H.265 AVC compression format, and is released under the terms of  the GNU GPL. 

*homepage*: https://x265.org/

version|toolchain
-------|----------------------------------------------------------
``2.4``|``foss/2016b``
``2.5``|``intel/2017a``
``2.6``|``GCCcore/6.4.0``, ``intel/2017b``
``2.7``|``GCCcore/6.4.0``
``2.9``|``GCCcore/7.3.0``
``3.0``|``GCCcore/8.2.0``
``3.2``|``GCCcore/8.3.0``
``3.3``|``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``3.5``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="xalt"/>
### *XALT*

XALT 2 is a tool to allow a site to track user executables   and library usage on a cluster. When installed it can tell a site what are   the top executables by Node-Hours or by the number of users or the number   of times it is run. XALT 2 also tracks library usage as well. XALT 2 can   also track package use by R, MATLAB or Python. It tracks both MPI and   non-MPI programs.

*homepage*: https://github.com/xalt/xalt

version  |toolchain
---------|----------
``2.8.4``|``system``

<a anchor="xarray"/>
### *xarray*

xarray (formerly xray) is an open source project and Python package that aims to bring  the labeled data power of pandas to the physical sciences, by providing N-dimensional variants of the  core pandas data structures.

*homepage*: https://github.com/pydata/xarray

version     |versionsuffix     |toolchain
------------|------------------|---------------------------------------------------
``0.9.5``   |``-Python-2.7.13``|``intel/2017a``
``0.9.5``   |``-Python-3.6.1`` |``intel/2017a``
``0.9.6``   |``-Python-2.7.13``|``intel/2017a``
``0.10.0``  |``-Python-2.7.14``|``intel/2017b``
``0.10.0``  |``-Python-3.6.3`` |``intel/2017b``
``0.10.3``  |``-Python-3.6.4`` |``intel/2018a``
``0.10.4``  |``-Python-3.6.4`` |``intel/2018a``
``0.10.8``  |``-Python-3.6.4`` |``intel/2018a``
``0.12.1``  |``-Python-3.6.6`` |``foss/2018b``, ``intel/2018b``
``0.13.0``  |``-Python-3.7.2`` |``intel/2019a``
``0.15.1``  |``-Python-3.7.4`` |``foss/2019b``, ``intel/2019b``
``0.16.1``  |``-Python-3.8.2`` |``foss/2020a``, ``intel/2020a``
``0.16.2``  |                  |``foss/2020b``, ``fosscuda/2020b``, ``intel/2020b``
``0.19.0``  |                  |``foss/2021a``
``0.20.1``  |                  |``foss/2021b``, ``intel/2021b``
``2022.6.0``|                  |``foss/2022a``, ``intel/2022a``
``2022.9.0``|                  |``foss/2022a``

<a anchor="xbitmaps"/>
### *xbitmaps*

provides bitmaps for x

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version  |toolchain
---------|-------------------------------------------
``1.1.1``|``foss/2016a``, ``intel/2016a``, ``system``
``1.1.2``|``system``

<a anchor="xcb-proto"/>
### *xcb-proto*

The X protocol C-language Binding (XCB) is a replacement for Xlib featuring a small footprint, latency hiding, direct access to the protocol, improved threading support, and extensibility.

*homepage*: https://xcb.freedesktop.org/

version |toolchain
--------|----------
``1.11``|``system``
``1.13``|``system``

<a anchor="xcb-util"/>
### *xcb-util*

The xcb-util package provides additional extensions to the XCB library,  many that were previously found in Xlib, but are not part of core X protocol

*homepage*: http://xcb.freedesktop.org/

version  |toolchain
---------|-------------------------------
``0.4.0``|``foss/2016a``, ``intel/2016a``

<a anchor="xcb-util-image"/>
### *xcb-util-image*

The xcb-util-image package provides additional extensions to the XCB library.

*homepage*: http://xcb.freedesktop.org/

version  |toolchain
---------|-------------------------------
``0.4.0``|``foss/2016a``, ``intel/2016a``

<a anchor="xcb-util-keysyms"/>
### *xcb-util-keysyms*

The xcb-util-keysyms package contains a library for  handling standard X key constants and conversion to/from keycodes.

*homepage*: http://xcb.freedesktop.org/

version  |toolchain
---------|-------------------------------
``0.4.0``|``foss/2016a``, ``intel/2016a``

<a anchor="xcb-util-renderutil"/>
### *xcb-util-renderutil*

The xcb-util-renderutil package provides additional extensions to the XCB library.

*homepage*: http://xcb.freedesktop.org/

version  |toolchain
---------|-------------------------------
``0.3.9``|``foss/2016a``, ``intel/2016a``

<a anchor="xcb-util-wm"/>
### *xcb-util-wm*

The xcb-util-wm package contains libraries which provide client and window-manager helpers  for EWMH and ICCCM.

*homepage*: http://xcb.freedesktop.org/

version  |toolchain
---------|-------------------------------
``0.4.1``|``foss/2016a``, ``intel/2016a``

<a anchor="xcell"/>
### *xCell*

xCell is a gene signatures-based method learned from thousands of pure cell types from various sources.

*homepage*: https://github.com/dviraran/xCell

version |versionsuffix|toolchain
--------|-------------|--------------
``1.12``|``-R-3.5.1`` |``foss/2018b``

<a anchor="xcfun"/>
### *XCFun*

XCFun is a library of DFT exchange-correlation (XC) functionals.  It is based on automatic differentiation and can therefore generate arbitrary order  derivatives of these functionals. 

*homepage*: http://dftlibs.org/xcfun/

version     |versionsuffix     |toolchain
------------|------------------|----------------------------------------------------------
``2.1.0``   |                  |``GCCcore/9.3.0``
``2.1.1``   |                  |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.3.0``
``20180122``|``-Python-2.7.14``|``intel/2017b``
``20190127``|``-Python-3.7.2`` |``foss/2019a``

<a anchor="xcrysden"/>
### *XCrySDen*

XCrySDen is a crystalline and molecular structure visualisation program aiming at display of isosurfaces and contours, which can be superimposed on crystalline structures and interactively rotated and manipulated. It also possesses some tools for analysis of properties in reciprocal space such as interactive selection of k-paths in the Brillouin zone for the band-structure plots, and visualisation of Fermi surfaces. 

*homepage*: http://www.xcrysden.org/

version   |toolchain
----------|--------------------------------------------------------------------------------
``1.5.60``|``intel/2019a``
``1.6.2`` |``foss/2019b``, ``foss/2020b``, ``foss/2022a``, ``intel/2019b``, ``intel/2022a``

<a anchor="xerces-c++"/>
### *Xerces-C++*

Xerces-C++ is a validating XML parser written in a portable subset of C++. Xerces-C++ makes it easy to give your application the ability to read and write XML data. A shared library is provided for parsing, generating, manipulating, and validating XML documents using the DOM, SAX, and SAX2 APIs.

*homepage*: https://xerces.apache.org/xerces-c/

version  |toolchain
---------|-----------------------------------------------------------------------------
``3.1.4``|``GCCcore/6.4.0``
``3.2.0``|``GCCcore/7.3.0``
``3.2.2``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``3.2.3``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/9.3.0``

<a anchor="xesmf"/>
### *xESMF*

xESMF: Universal Regridder for Geospatial Data

*homepage*: https://xesmf.readthedocs.io

version  |versionsuffix    |toolchain
---------|-----------------|---------------
``0.3.0``|                 |``intel/2020b``
``0.3.0``|``-Python-3.8.2``|``foss/2020a``

<a anchor="xextproto"/>
### *xextproto*

XExtProto protocol headers.

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version  |toolchain
---------|------------------------------------------------------------------
``7.3.0``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2017b``

<a anchor="xf86vidmodeproto"/>
### *xf86vidmodeproto*

X11 XFree86 video mode extension protocol headers.

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version  |toolchain
---------|-------------------------------
``2.3.1``|``foss/2016a``, ``intel/2016a``

<a anchor="xgboost"/>
### *XGBoost*

XGBoost is an optimized distributed gradient boosting library designed to be highly efficient,  flexible and portable.

*homepage*: https://github.com/dmlc/xgboost

version     |versionsuffix      |toolchain
------------|-------------------|---------------
``0.6a2``   |``-Python-2.7.12`` |``intel/2016b``
``0.6a2``   |``-Python-3.5.2``  |``intel/2016b``
``0.6a2``   |``-Python-3.6.1``  |``intel/2017a``
``0.72.1``  |``-Python-2.7.14`` |``intel/2017b``
``0.90``    |``-Python-3.7.4``  |``foss/2019b``
``1.2.0``   |``-Python-3.8.2``  |``foss/2020a``
``1.5.0``   |                   |``foss/2021a``
``1.7.2``   |                   |``foss/2022a``
``1.7.2``   |``-CUDA-11.7.0``   |``foss/2022a``
``20171120``|``-Java-1.8.0_152``|``intel/2017b``

<a anchor="xgrafix"/>
### *XGrafix*

A graphics library and controller for physics simulation programs. 3-d surface plots, scatter plots, 2-d line plots.

*homepage*: https://ptsg.egr.msu.edu/

version |toolchain
--------|-----------------
``2.41``|``GCCcore/9.3.0``

<a anchor="xineramaproto"/>
### *xineramaproto*

X protocol and ancillary headers for xinerama

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version  |toolchain
---------|-------------------------------
``1.2.1``|``foss/2016a``, ``intel/2016a``

<a anchor="xkeyboardconfig"/>
### *XKeyboardConfig*

The non-arch keyboard configuration database for X Window.  The goal is to provide the consistent, well-structured,  frequently released open source of X keyboard configuration data  for X Window System implementations (free, open source and commercial).  The project is targeted to XKB-based systems.

*homepage*: http://www.freedesktop.org/wiki/Software/XKeyboardConfig/

version |toolchain
--------|-------------------------------
``2.17``|``foss/2016a``, ``intel/2016a``

<a anchor="xlsxwriter"/>
### *XlsxWriter*

A Python module for creating Excel XLSX files

*homepage*: https://xlsxwriter.readthedocs.io/

version  |toolchain
---------|------------------
``1.4.0``|``GCCcore/10.2.0``
``1.4.4``|``GCCcore/10.3.0``
``3.0.2``|``GCCcore/11.2.0``

<a anchor="xmds2"/>
### *XMDS2*

The purpose of XMDS2 is to simplify the process of creating simulations that  solve systems of initial-value first-order partial and ordinary differential equations.

*homepage*: http://www.xmds.org/

version  |versionsuffix     |toolchain
---------|------------------|--------------
``2.2.3``|``-Python-2.7.15``|``foss/2018b``

<a anchor="xmipp"/>
### *Xmipp*

Scipion is an image processing framework to obtain 3D models of macromolecular complexes using Electron Microscopy (3DEM). It integrates several software packages and presents an unified interface for both biologists and developers. Scipion allows to execute workflows combining different software tools, while taking care of formats and conversions. Additionally, all steps are tracked and can be reproduced later on.  

*homepage*: https://github.com/I2PC/scipion/wiki

version           |versionsuffix     |toolchain
------------------|------------------|----------------------------------
``3.19.04-Apollo``|``-Python-2.7.15``|``foss/2019a``, ``fosscuda/2019a``
``3.22.07-Helios``|                  |``foss/2022a``
``3.22.07-Helios``|``-CUDA-11.7.0``  |``foss/2022a``

<a anchor="xmitgcm"/>
### *xmitgcm*

xmitgcm is a python package for reading MITgcm binary  MDS files into xarray data structures. By storing data in dask arrays,  xmitgcm enables parallel, out-of-core analysis of MITgcm output data.

*homepage*: https://github.com/MITgcm/xmitgcm

version  |toolchain
---------|--------------
``0.5.2``|``foss/2022a``

<a anchor="xml-compile"/>
### *XML-Compile*

Perl module for compilation based XML processing

*homepage*: https://metacpan.org/pod/XML::Compile

version |toolchain
--------|------------------
``1.63``|``GCCcore/11.2.0``

<a anchor="xml-libxml"/>
### *XML-LibXML*

Perl binding for libxml2

*homepage*: https://metacpan.org/pod/distribution/XML-LibXML/LibXML.pod

version   |versionsuffix   |toolchain
----------|----------------|----------------------------------------------------------
``2.0132``|``-Perl-5.24.0``|``foss/2016b``, ``intel/2016b``
``2.0132``|``-Perl-5.24.1``|``intel/2017a``
``2.0132``|``-Perl-5.26.0``|``foss/2017b``, ``intel/2017b``
``2.0132``|``-Perl-5.26.1``|``GCCcore/6.4.0``
``2.0132``|``-Perl-5.28.0``|``GCCcore/7.3.0``
``2.0200``|``-Perl-5.28.1``|``GCCcore/8.2.0``
``2.0201``|                |``GCCcore/8.3.0``
``2.0205``|                |``GCCcore/9.3.0``
``2.0206``|                |``GCCcore/10.2.0``
``2.0207``|                |``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="xml-parser"/>
### *XML-Parser*

This is a Perl extension interface to James Clark's XML parser, expat.

*homepage*: https://search.cpan.org/~toddr/XML-Parser-2.46/

version    |versionsuffix   |toolchain
-----------|----------------|--------------------------------------------------------------------------------------
``2.41``   |``-Perl-5.20.3``|``intel/2016a``
``2.44``   |``-Perl-5.22.1``|``foss/2016a``, ``intel/2016a``
``2.44_01``|``-Perl-5.24.0``|``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``foss/2016b``, ``gimkl/2017a``, ``intel/2016b``
``2.44_01``|``-Perl-5.24.1``|``GCCcore/6.3.0``, ``intel/2017a``
``2.44_01``|``-Perl-5.26.0``|``GCCcore/6.4.0``
``2.44_01``|``-Perl-5.26.1``|``GCCcore/6.4.0``
``2.44_01``|``-Perl-5.28.0``|``GCCcore/7.3.0``
``2.46``   |``-Perl-5.32.1``|``GCCcore/10.3.0``
``2.46``   |``-Perl-5.34.1``|``GCCcore/11.3.0``

<a anchor="xmlf90"/>
### *xmlf90*

xmlf90 is a basic XML parsing library written in Fortran.

*homepage*: https://launchpad.net/xmlf90

version  |toolchain
---------|------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.5.3``|``foss/2016b``, ``foss/2017a``
``1.5.4``|``GCC/10.2.0``, ``GCC/10.3.0``, ``GCC/11.2.0``, ``iccifort/2019.5.281``, ``iccifort/2020.4.304``, ``intel-compilers/2021.2.0``, ``intel-compilers/2021.4.0``

<a anchor="xmlsec"/>
### *XMLSec*

XML Security Library is a C library based on LibXML2,  supporting major XML security standards.

*homepage*: https://www.aleksey.com/xmlsec/index.html

version   |toolchain
----------|-----------------
``1.2.26``|``GCCcore/6.4.0``

<a anchor="xmlstarlet"/>
### *XMLStarlet*

Command line XML tool

*homepage*: http://xmlstar.sourceforge.net

version  |toolchain
---------|---------------------------------
``1.6.1``|``GCCcore/6.4.0``, ``foss/2016a``

<a anchor="xonsh"/>
### *xonsh*

Xonsh is a Python-ish, BASHwards-looking shell language and command prompt.

*homepage*: https://xon.sh/

version  |toolchain
---------|---------------
``0.3.2``|``intel/2016a``

<a anchor="xorg-macros"/>
### *xorg-macros*

X.org macros utilities.

*homepage*: https://cgit.freedesktop.org/xorg/util/macros

version   |toolchain
----------|------------------------------------------------------------------------------------------------------------------------------------
``1.19.0``|``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``
``1.19.1``|``GCCcore/6.3.0``, ``GCCcore/6.4.0``
``1.19.2``|``GCCcore/10.2.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.2.0``, ``GCCcore/9.3.0``
``1.19.3``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/12.2.0``

<a anchor="xplor-nih"/>
### *XPLOR-NIH*

A System for X-ray Crystallography and NMR

*homepage*: https://nmr.cit.nih.gov/xplor-nih/

version|versionsuffix    |toolchain
-------|-----------------|----------
``3.4``|``-Linux_x86_64``|``system``

<a anchor="xprop"/>
### *xprop*

The xprop utility is for displaying window and font properties in an X server.  One window or font is selected using the command line arguments or possibly  in the case of a window, by clicking on the desired window. A list of  properties is then given, possibly with formatting information.

*homepage*: https://www.x.org/wiki/

version  |versionsuffix    |toolchain
---------|-----------------|-----------------------------------------------------------------------------------------------------------------------
``1.2.2``|                 |``GCCcore/5.4.0``, ``GCCcore/6.4.0``, ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``
``1.2.2``|``-X11-20180131``|``GCCcore/6.4.0``
``1.2.3``|                 |``GCCcore/7.3.0``
``1.2.4``|                 |``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.2.5``|                 |``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="xproto"/>
### *xproto*

X protocol and ancillary headers

*homepage*: https://www.freedesktop.org/wiki/Software/xlibs

version   |toolchain
----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``7.0.28``|``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``7.0.29``|``intel/2016a``
``7.0.31``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

<a anchor="xsd"/>
### *XSD*

CodeSynthesis XSD is an open-source, cross-platform W3C XML Schema to C++ data binding compiler.

*homepage*: https://www.codesynthesis.com/products/xsd/

version  |toolchain
---------|-----------------
``4.0.0``|``GCCcore/8.2.0``

<a anchor="xtandem"/>
### *XTandem*

X!Tandem open source is software that can match tandem mass  spectra with peptide sequences, in a process that has come to be known as  protein identification.

*homepage*: https://www.thegpm.org/tandem

version       |toolchain
--------------|----------------------------------------------------------
``17.02.01.4``|``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``

<a anchor="xtb"/>
### *xtb*

Semiempirical Extended Tight-Binding Program Package.

*homepage*: https://xtb-docs.readthedocs.io

version         |versionsuffix    |toolchain
----------------|-----------------|-------------------------------
``6.2.2-hotfix``|``-Python-3.6.3``|``intel/2017b``
``6.2.3``       |                 |``foss/2019b``
``6.4.1``       |                 |``foss/2021b``, ``intel/2021a``

<a anchor="xtensor"/>
### *xtensor*

xtensor is a C++ library meant for numerical analysis with multi-dimensional array expressions.

*homepage*: https://github.com/xtensor-stack/xtensor

version   |toolchain
----------|--------------
``0.24.0``|``foss/2021b``

<a anchor="xtrans"/>
### *xtrans*

xtrans includes a number of routines to make X implementations transport-independent;  at time of writing, it includes support for UNIX sockets, IPv4, IPv6, and DECnet. 

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

version  |toolchain
---------|---------------------------------------------------------------------
``1.3.5``|``GCCcore/11.3.0``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``

<a anchor="xvfb"/>
### *Xvfb*

Xvfb is an X server that can run on machines with no display hardware and no physical input devices.  It emulates a dumb framebuffer using virtual memory.

*homepage*: https://www.x.org/releases/X11R7.6/doc/man/man1/Xvfb.1.xhtml

version    |toolchain
-----------|-------------------------------------
``1.20.8`` |``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``1.20.9`` |``GCCcore/10.2.0``, ``GCCcore/9.3.0``
``1.20.11``|``GCCcore/10.3.0``
``1.20.13``|``GCCcore/11.2.0``
``21.1.3`` |``GCCcore/11.3.0``

<a anchor="xxd"/>
### *xxd*

xxd is part of the VIM package and this will only install xxd, not vim! xxd converts to/from hexdumps of binary files.

*homepage*: https://www.vim.org

version     |toolchain
------------|----------------------------------------------------------
``8.2.4220``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``

<a anchor="xxhash"/>
### *xxHash*

xxHash is an extremely fast non-cryptographic hash algorithm, working at RAM speed limit.

*homepage*: https://cyan4973.github.io/xxHash

version  |toolchain
---------|------------------
``0.8.0``|``GCCcore/11.2.0``

<a anchor="xz"/>
### *XZ*

xz: XZ utilities

*homepage*: https://tukaani.org/xz/

version  |versionsuffix      |toolchain
---------|-------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------
``5.0.5``|                   |``GCC/4.8.2``
``5.2.0``|                   |``GCC/4.9.2``
``5.2.2``|                   |``GCC/4.9.2``, ``GCC/5.4.0-2.26``, ``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``5.2.2``|``-gettext-0.19.7``|``foss/2016a``, ``intel/2016a``
``5.2.3``|                   |``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``gimkl/2017a``
``5.2.4``|                   |``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.2.0``
``5.2.5``|                   |``GCCcore/10.1.0``, ``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/9.3.0``
``5.2.7``|                   |``GCCcore/12.2.0``

<a anchor="y"/>
### *Y*


<a href="#yacs">YACS</a> - <a href="#yade">Yade</a> - <a href="#yaff">yaff</a> - <a href="#yambo">Yambo</a> - <a href="#yaml-cpp">yaml-cpp</a> - <a href="#yank">YANK</a> - <a href="#yaps">YAPS</a> - <a href="#yasm">Yasm</a> - <a href="#yaxt">YAXT</a> - <a href="#yices">Yices</a>


<a anchor="yacs"/>
### *YACS*

YACS was created as a lightweight library to define and manage system configurations, such as those commonly found in software designed for scientific experimentation. These "configurations" typically cover concepts like hyperparameters used in training a machine learning model or configurable model hyperparameters, such as the depth of a convolutional neural network.

*homepage*: https://github.com/rbgirshick/yacs

version  |toolchain
---------|-------------------------------------------------------------------------------------------------
``0.1.8``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/8.3.0``

<a anchor="yade"/>
### *Yade*

Yade is an extensible open-source framework for discrete numerical models,   focused on Discrete Element Method. The computation parts are written in c++ using flexible object model,   allowing independent implementation of new alogrithms and interfaces.   Python is used for rapid and concise scene construction, simulation control, postprocessing and debugging. 

*homepage*: https://yade-dem.org/doc/

version     |versionsuffix     |toolchain
------------|------------------|-------------------------------
``1.20.0``  |``-Python-2.7.11``|``foss/2016a``, ``intel/2016a``
``2016.06a``|``-Python-2.7.12``|``foss/2016b``, ``intel/2016b``
``2017.01a``|``-Python-2.7.12``|``intel/2016b``
``2018.02b``|``-Python-2.7.14``|``intel/2018a``

<a anchor="yaff"/>
### *yaff*

Yaff stands for 'Yet another force field'. It is a pythonic force-field code.

*homepage*: https://molmod.github.io/yaff/

version             |versionsuffix                 |toolchain
--------------------|------------------------------|------------------------------------------------
``1.0.develop.2.15``|``-Python-2.7.12-HDF5-1.8.18``|``intel/2016b``
``1.1.2``           |``-Python-2.7.13``            |``intel/2017a``
``1.1.3``           |``-Python-2.7.13``            |``intel/2017a``
``1.4.2``           |``-Python-2.7.14``            |``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``1.4.2``           |``-Python-3.6.3``             |``intel/2017b``
``1.4.5``           |``-Python-2.7.15``            |``intel/2018b``
``1.5.0``           |``-Python-2.7.15``            |``intel/2018b``
``1.5.0``           |``-Python-3.7.2``             |``intel/2019a``
``1.6.0``           |                              |``foss/2020b``
``1.6.0``           |``-Python-3.7.2``             |``intel/2019a``
``1.6.0``           |``-Python-3.7.4``             |``foss/2019b``, ``intel/2019b``
``1.6.0``           |``-Python-3.8.2``             |``foss/2020a``, ``intel/2020a``

<a anchor="yambo"/>
### *Yambo*

Yambo is a FORTRAN/C code for Many-Body calculations in solid state and molecular physics.  Yambo relies on the Kohn-Sham wavefunctions generated by two DFT public codes: abinit, and PWscf.

*homepage*: http://www.yambo-code.org

version  |toolchain
---------|-------------------------
``3.4.2``|``intel/2016.02-GCC-4.9``
``5.0.4``|``intel/2021a``

<a anchor="yaml-cpp"/>
### *yaml-cpp*

yaml-cpp is a YAML parser and emitter in C++ matching the YAML 1.2 spec

*homepage*: https://github.com/jbeder/yaml-cpp

version  |toolchain
---------|------------------
``0.6.3``|``GCCcore/8.3.0``
``0.7.0``|``GCCcore/11.3.0``

<a anchor="yank"/>
### *YANK*

A GPU-accelerated Python framework for exploring algorithms for alchemical free energy calculations

*homepage*: http://getyank.org/

version   |versionsuffix    |toolchain
----------|-----------------|---------------
``0.25.2``|``-Python-3.8.2``|``intel/2020a``

<a anchor="yaps"/>
### *YAPS*

YAPS - Yet Another Positioning Solver

*homepage*: https://github.com/baktoft/yaps

version  |versionsuffix|toolchain
---------|-------------|---------------
``1.1.0``|``-R-3.5.1`` |``foss/2018b``
``1.1.0``|``-R-3.6.0`` |``intel/2019a``

<a anchor="yasm"/>
### *Yasm*

Yasm: Complete rewrite of the NASM assembler with BSD license

*homepage*: http://www.tortall.net/projects/yasm/

version  |toolchain
---------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.3.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``gimkl/2.11.5``, ``gimkl/2017a``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``

<a anchor="yaxt"/>
### *YAXT*

Yet Another eXchange Tool

*homepage*: https://www.dkrz.de/redmine/projects/yaxt

version    |toolchain
-----------|-------------------------------------------------
``0.5.1``  |``intel/2016b``, ``intel/2017a``, ``intel/2017b``
``0.6.0``  |``intel/2018a``, ``intel/2018b``, ``iomkl/2018b``
``0.6.2``  |``foss/2018b``, ``gompi/2019b``, ``iimpi/2019b``
``0.9.0``  |``gompi/2020b``
``0.9.1``  |``gompi/2021a``
``0.9.2``  |``iimpi/2021b``
``0.9.2.1``|``gompi/2021b``

<a anchor="yices"/>
### *Yices*

Yices 2 is an SMT solver that decides the satisfiability of formulas containing uninterpreted  function symbols with equality, real and integer arithmetic, bitvectors, scalar types, and tuples. Yices 2 supports  both linear and nonlinear arithmetic.

*homepage*: https://yices.csl.sri.com/index.html

version  |toolchain
---------|------------------
``2.6.2``|``GCCcore/10.2.0``

<a anchor="z"/>
### *Z*


<a href="#z3">Z3</a> - <a href="#zarr">zarr</a> - <a href="#zeromq">ZeroMQ</a> - <a href="#zfp">zfp</a> - <a href="#zgoubi">Zgoubi</a> - <a href="#zinger">zingeR</a> - <a href="#zip">Zip</a> - <a href="#zlib">zlib</a> - <a href="#zlib-ng">zlib-ng</a> - <a href="#zlibbioc">zlibbioc</a> - <a href="#zopfli">Zopfli</a> - <a href="#zpaq">ZPAQ</a> - <a href="#zsh">zsh</a> - <a href="#zstd">zstd</a>


<a anchor="z3"/>
### *Z3*

Z3 is a theorem prover from Microsoft Research. 

*homepage*: https://github.com/Z3Prover/z3

version   |toolchain
----------|------------------------------------
``4.8.9`` |``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``4.8.10``|``GCCcore/10.2.0``
``4.8.11``|``GCCcore/10.3.0``
``4.8.12``|``GCCcore/11.2.0``
``4.8.16``|``GCCcore/11.3.0``
``4.10.2``|``GCCcore/11.3.0``

<a anchor="zarr"/>
### *zarr*

Zarr is a Python package providing an implementation of compressed, chunked, N-dimensional arrays,  designed for use in parallel computing.

*homepage*: https://zarr.readthedocs.io/en/stable/

version   |versionsuffix     |toolchain
----------|------------------|------------------------------
``2.1.4`` |``-Python-2.7.13``|``foss/2017a``
``2.4.0`` |``-Python-3.8.2`` |``foss/2020a``
``2.8.1`` |                  |``foss/2020b``
``2.10.1``|                  |``foss/2021a``
``2.13.3``|                  |``foss/2021b``, ``foss/2022a``

<a anchor="zeromq"/>
### *ZeroMQ*

ZeroMQ looks like an embeddable networking library but acts like a concurrency framework.  It gives you sockets that carry atomic messages across various transports like in-process,  inter-process, TCP, and multicast. You can connect sockets N-to-N with patterns like fanout,  pub-sub, task distribution, and request-reply. It's fast enough to be the fabric for clustered  products. Its asynchronous I/O model gives you scalable multicore applications, built as asynchronous  message-processing tasks. It has a score of language APIs and runs on most operating systems.

*homepage*: https://www.zeromq.org/

version  |toolchain
---------|---------------------------------------------------------------------------------------------------------
``4.1.4``|``foss/2016a``, ``intel/2016a``
``4.1.5``|``intel/2016b``
``4.2.0``|``foss/2016b``, ``intel/2016b``
``4.2.2``|``foss/2017a``, ``foss/2017b``, ``fosscuda/2017b``, ``intel/2017a``, ``intel/2017b``, ``intelcuda/2017b``
``4.2.5``|``foss/2018a``, ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018a``, ``intel/2018b``
``4.3.2``|``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``4.3.3``|``GCCcore/10.2.0``
``4.3.4``|``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``

<a anchor="zfp"/>
### *zfp*

zfp is a compressed format for representing multidimensional floating-point and integer arrays. zfp provides compressed-array classes that support high throughput read and write random access to individual array elements. zfp also supports serial and parallel (OpenMP and CUDA) compression of whole arrays, e.g., for applications that read and write large data sets to and from disk.

*homepage*: https://github.com/LLNL/zfp

version  |toolchain
---------|-------------------------------------
``0.5.5``|``GCCcore/10.2.0``
``1.0.0``|``GCCcore/10.3.0``, ``GCCcore/9.3.0``

<a anchor="zgoubi"/>
### *Zgoubi*

Zgoubi is a ray-tracing code in use for charged particle beam dynamics simulations.  It can simulate beam dynamics in a large variety of machines and optical systems.

*homepage*: https://zgoubi.sourceforge.io

version  |toolchain
---------|------------------
``6.0.2``|``GCCcore/10.3.0``

<a anchor="zinger"/>
### *zingeR*

Zero-Inflated Negative binomial Gene Expression in R

*homepage*: https://github.com/statOmics/zingeR

version     |versionsuffix|toolchain
------------|-------------|--------------
``20180131``|``-R-3.5.1`` |``foss/2018b``

<a anchor="zip"/>
### *Zip*

Zip is a compression and file packaging/archive utility. Although highly compatible both with PKWARE's PKZIP and PKUNZIP utilities for MS-DOS and with Info-ZIP's own UnZip, our primary objectives have been portability and other-than-MSDOS functionality

*homepage*: http://www.info-zip.org/Zip.html

version|toolchain
-------|----------------------------------------------------------------------------------------------------------------------------------------------------------
``3.0``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.2.0``, ``GCCcore/11.3.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.3.0``

<a anchor="zlib"/>
### *zlib*

zlib is designed to be a free, general-purpose, legally unencumbered -- that is,  not covered by any patents -- lossless data-compression library for use on virtually any  computer hardware and operating system.

*homepage*: http://www.zlib.net/

version   |toolchain
----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
``1.2.7`` |``GCC/4.8.1``, ``GCC/4.8.2``
``1.2.8`` |``GCC/4.8.2``, ``GCC/4.8.3``, ``GCC/4.8.4``, ``GCC/4.9.2``, ``GCC/4.9.2-binutils-2.25``, ``GCC/4.9.3``, ``GCC/4.9.3-2.25``, ``GCC/4.9.3-binutils-2.25``, ``GCC/5.1.0-binutils-2.25``, ``GCCcore/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/4.9.4``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.1.0``, ``GCCcore/6.2.0``, ``GCCcore/6.3.0``, ``GNU/4.9.3-2.25``, ``foss/2016.04``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``intel/2017.01``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
``1.2.10``|``system``
``1.2.11``|``FCC/4.5.0``, ``GCCcore/10.1.0``, ``GCCcore/10.2.0``, ``GCCcore/10.3.0``, ``GCCcore/11.1.0``, ``GCCcore/11.2.0``, ``GCCcore/5.4.0``, ``GCCcore/5.5.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.1.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/7.4.0``, ``GCCcore/8.1.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/8.4.0``, ``GCCcore/9.1.0``, ``GCCcore/9.2.0``, ``GCCcore/9.3.0``, ``GCCcore/9.4.0``, ``GCCcore/system``, ``gimkl/2017a``, ``system``
``1.2.12``|``GCCcore/11.3.0``, ``GCCcore/12.1.0``, ``GCCcore/12.2.0``, ``GCCcore/9.5.0``, ``system``

<a anchor="zlib-ng"/>
### *zlib-ng*

zlib data compression library for the next generation systems

*homepage*: https://github.com/zlib-ng/zlib-ng

version  |toolchain
---------|------------------
``2.0.5``|``GCCcore/10.2.0``
``2.0.6``|``GCCcore/10.3.0``

<a anchor="zlibbioc"/>
### *zlibbioc*

This package uses the source code of zlib-1.2.5 to create libraries for systems that do not have  these available via other means.

*homepage*: https://bioconductor.org/packages/release/bioc/html/zlibbioc.html

version   |versionsuffix|toolchain
----------|-------------|---------------
``1.18.0``|``-R-3.2.3`` |``intel/2016a``
``1.20.0``|``-R-3.3.1`` |``intel/2016b``

<a anchor="zopfli"/>
### *Zopfli*

Zopfli Compression Algorithm is a compression library programmed in C to perform very good, but slow, deflate or zlib compression.

*homepage*: https://github.com/google/zopfli

version  |toolchain
---------|--------------------------------------
``1.0.3``|``GCCcore/10.2.0``, ``GCCcore/10.3.0``

<a anchor="zpaq"/>
### *ZPAQ*

zpaq is a free and open source (GPL v3) incremental,  journaling command-line archiver for Windows, Linux and Mac OS/X

*homepage*: http://mattmahoney.net/dc/zpaq.html

version |toolchain
--------|-------------
``7.00``|``GCC/4.8.2``

<a anchor="zsh"/>
### *zsh*

Zsh is a shell designed for interactive use, although it is also a powerful scripting language.

*homepage*: https://www.zsh.org/

version  |toolchain
---------|-------------------------
``5.1.1``|``GNU/4.9.3-2.25``
``5.2``  |``foss/2016b``
``5.8``  |``GCC/8.3.0``, ``system``

<a anchor="zstd"/>
### *zstd*

Zstandard is a real-time compression algorithm, providing high compression ratios.  It offers a very wide range of compression/speed trade-off, while being backed by a very fast decoder.  It also offers a special mode for small data, called dictionary compression, and can create dictionaries  from any sample set.

*homepage*: https://facebook.github.io/zstd

version  |toolchain
---------|----------------------------------------------------
``1.3.4``|``foss/2016b``
``1.4.0``|``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``foss/2018b``
``1.4.4``|``GCCcore/8.3.0``, ``GCCcore/9.3.0``
``1.4.5``|``GCCcore/10.2.0``
``1.4.9``|``GCCcore/10.3.0``
``1.5.0``|``GCCcore/11.2.0``
``1.5.2``|``GCCcore/11.3.0``, ``GCCcore/12.2.0``
